{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pip install requests\n",
    "pip install lxml\n",
    "pip install pandas\n",
    "pip install openpyxl\n",
    "pip install cobra\n",
    "pip install cobrakbase\n",
    "# for windows only, download the file from https://www.lfd.uci.edu/~gohlke/pythonlibs/#pyeda firstly\n",
    "# pip install pyeda-0.28.0-cp37-cp37m-win_amd64.whl  \n",
    "pip install modelseedpy\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "org_name = 'mko'\n",
    "org_full_name = 'Methylomonas_koyamae'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "org_name = 'mbry'\n",
    "org_full_name = 'Methylocystis_bryophila'"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# build model from modelseed"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Auto build model from genome by modelseed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import modelseedpy\n",
    "import sys\n",
    "from modelseedpy import KBaseMediaPkg\n",
    "from modelseedpy import FBAHelper, MSBuilder, MSGenome\n",
    "from modelseedpy import RastClient\n",
    "import os\n",
    "import cobra\n",
    "os.environ[\"HOME\"] = \"1\"\n",
    "import cobrakbase\n",
    "\n",
    "# download genome from NCBI https://www.ncbi.nlm.nih.gov/data-hub/genome/GCA_027925445.1/\n",
    "for root, dirs, files in os.walk(org_full_name+'_genome'):\n",
    "    if 'protein.faa' in files:\n",
    "        genome_path = root+'\\\\protein.faa'\n",
    "\n",
    "genome = MSGenome.from_fasta(genome_path,split=' ')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rast = RastClient()\n",
    "rast.annotate_genome(genome)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "kbase_api = cobrakbase.KBaseAPI('5IS2FWVJ5FJZCHJ6KDTDHUUZLIXKTPQR')\n",
    "kb_template = kbase_api.get_from_ws(\"GramNegModelTemplateV4\",\"NewKBaseModelTemplates\")\n",
    "media1 = kbase_api.get_from_ws('Carbon-D-Glucose', 'KBaseMedia')\n",
    "media2 = kbase_api.get_from_ws('C-acetate', 'KBaseMedia')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "kb_template = kbase_api.get_from_ws(\"GramNegModelTemplateV4\",\"NewKBaseModelTemplates\")\n",
    "driftmodel = MSBuilder.build_metabolic_model(org_full_name, genome, template = kb_template, allow_all_non_grp_reactions = True)\n",
    "driftmodel\n",
    "cobra.io.write_sbml_model(driftmodel,filename=org_name+'_model\\\\'+org_name+'_drift_model_from_modelseed.xml')\n",
    "\n",
    "'''\n",
    "or build this model by Kbase\n",
    "\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "modelseedpy.helpers - WARNING - downloading data file to: c:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\modelseedpy//data/templates/template_core.json\n",
      "WARNING:modelseedpy.helpers:downloading data file to: c:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\modelseedpy//data/templates/template_core.json\n"
     ]
    }
   ],
   "source": [
    "from cobrakbase.core.kbasefba.newmodeltemplate_builder import NewModelTemplateBuilder\n",
    "from modelseedpy.helpers import get_template, get_classifier\n",
    "template_core = get_template('template_core')\n",
    "template_gramneg = get_template('template_gram_neg')\n",
    "template_core = NewModelTemplateBuilder.from_dict(template_core, None).build()\n",
    "template_gramneg = NewModelTemplateBuilder.from_dict(template_gramneg, None).build()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_builder = MSBuilder(genome, template_gramneg)\n",
    "base_model = model_builder.build('model', '0', allow_all_non_grp_reactions=True, annotate_with_rast=False)\n",
    "from modelseedpy.core.msbuilder import build_biomass, core_atp\n",
    "base_model.add_reactions([build_biomass('bio2', base_model, template_core, core_atp, '0')])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Initialize new model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>Methylomonas_koyamae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>275915d1f40</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td></td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model Methylomonas_koyamae at 0x275915d1f40>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import cobra\n",
    "\n",
    "new_model = cobra.Model(org_full_name)\n",
    "new_model"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## KEGG ID mapping to modelseed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "done!\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "#org_name = 'mbry'\n",
    "df_reaction = pd.read_excel(org_name+'_model\\\\'+org_name+'_clean_kegg_reaction.xlsx', index_col=1)  \n",
    "titlename= ['entry_name','pathway_title','reaction_type','reaction_substrates','reaction_products']\n",
    "df_keggID_modelseedID = pd.read_csv('reaction_keggID_modelseedID.txt', sep='\\t')\n",
    "mapping_reaction = pd.merge(df_reaction,df_keggID_modelseedID,how='left',on='Kegg_ID').drop_duplicates(subset=['Kegg_ID'], keep='first')[['ModelSeed_ID','Kegg_ID']+titlename]\n",
    "mapping_reaction.to_excel(org_name+'_model\\\\'+org_name+'_mapping_reaction.xlsx', sheet_name='Sheet1', header=True)\n",
    "\n",
    "print('done!')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## add reactions from drift model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import cobra\n",
    "import pandas as pd\n",
    "'''\n",
    "org_name = 'mbry'\n",
    "org_full_name = 'Methylocystis_bryophila'\n",
    "drift_model_path = org_name+'_model\\\\ModelSeed_model_'+org_full_name+'.xml'\n",
    "drift_model = cobra.io.read_sbml_model(drift_model_path)\n",
    "new_model = cobra.io.read_sbml_model(org_name+'_model\\\\'+org_name+'_blank.xml')\n",
    "'''\n",
    "\n",
    "mapping_reaction = pd.read_excel(org_name+'_model\\\\'+org_name+'_mapping_reaction.xlsx', index_col=0)  \n",
    "modelseed_id_list = list(mapping_reaction['ModelSeed_ID'].dropna(axis = 0, how = 'any'))  # reaction ID that need to be added\n",
    "\n",
    "added_rxn_id_list = []\n",
    "unadded_rxn_id_list = []\n",
    "for rxn_id in modelseed_id_list:\n",
    "    try:\n",
    "        reaction = drift_model.reactions.get_by_id(rxn_id+'_c0')\n",
    "        new_model.add_reactions([reaction])\n",
    "    except:\n",
    "        unadded_rxn_id_list.append(rxn_id)\n",
    "        continue\n",
    "    added_rxn_id_list.append(rxn_id)\n",
    "'''\n",
    "bioreaction = drift_model.reactions.get_by_id('bio1_biomass')\n",
    "new_model.add_reactions([bioreaction])\n",
    "new_model.objective = new_model.problem.Objective(bioreaction.flux_expression, direction='max')\n",
    "'''\n",
    "print(str(len(added_rxn_id_list)) + ' / ' + str(len(modelseed_id_list)) + ' reactions and biomass reaction are added, which are copied from drift model')\n",
    "\n",
    "\n",
    "#cobra.io.write_sbml_model(new_model,filename=org_name+'_copyfrom_DriftModel.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['rxn40277',\n",
       " 'rxn40276',\n",
       " 'rxn40037',\n",
       " 'rxn40273',\n",
       " 'rxn40254',\n",
       " 'rxn40065',\n",
       " 'rxn39352',\n",
       " 'rxn40102',\n",
       " 'rxn40015',\n",
       " 'rxn40016',\n",
       " 'rxn40018',\n",
       " 'rxn40019',\n",
       " 'rxn40020',\n",
       " 'rxn40014',\n",
       " 'rxn39964',\n",
       " 'rxn40041',\n",
       " 'rxn40040']"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "unadded_rxn_id_list"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### add reactions not included in drift model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "======================== \n",
      " 433 reactions begin to added \n",
      "\n",
      "rxn15116 added\n",
      "rxn00536 added\n",
      "rxn15494 added\n",
      "rxn15364 added\n",
      "rxn15249 added\n",
      "rxn15694 added\n",
      "rxn15314 added\n",
      "rxn06120 added\n",
      "rxn15989 added\n",
      "rxn13974 added\n",
      "rxn00175 added\n",
      "rxn15298 added\n",
      "rxn00250 added\n",
      "rxn06109 added\n",
      "rxn15271 added\n",
      "rxn29919 added\n",
      "rxn00778 added\n",
      "rxn08647 added\n",
      "rxn03643 added\n",
      "rxn16622 added\n",
      "rxn00211 added\n",
      "rxn01042 added\n",
      "rxn15081 added\n",
      "rxn15270 added\n",
      "rxn02100 added\n",
      "rxn01008 added\n",
      "rxn07578 added\n",
      "rxn07576 added\n",
      "rxn07577 added\n",
      "rxn05336 added\n",
      "rxn05342 added\n",
      "rxn05340 added\n",
      "rxn05338 added\n",
      "rxn05341 added\n",
      "rxn05337 added\n",
      "rxn05339 added\n",
      "rxn00533 added\n",
      "rxn08766 added\n",
      "rxn05322 added\n",
      "rxn05326 added\n",
      "rxn05325 added\n",
      "rxn05327 added\n",
      "rxn05324 added\n",
      "rxn05323 added\n",
      "rxn05328 added\n",
      "rxn07455 added\n",
      "rxn00947 added\n",
      "rxn06237 added\n",
      "rxn05994 added\n",
      "rxn06238 added\n",
      "rxn05830 added\n",
      "rxn17757 added\n",
      "rxn05996 added\n",
      "rxn05871 added\n",
      "rxn14029 added\n",
      "rxn11642 added\n",
      "rxn11643 added\n",
      "rxn16584 added\n",
      "rxn07335 added\n",
      "rxn05004 added\n",
      "rxn16585 added\n",
      "rxn11156 added\n",
      "rxn11994 added\n",
      "rxn40277 added\n",
      "rxn40276 added\n",
      "rxn01917 added\n",
      "rxn01637 added\n",
      "rxn00560 added\n",
      "rxn05231 added\n",
      "rxn01649 added\n",
      "rxn01137 added\n",
      "rxn00916 added\n",
      "rxn01138 added\n",
      "rxn02449 added\n",
      "rxn01548 added\n",
      "rxn00514 added\n",
      "rxn01962 added\n",
      "rxn01139 added\n",
      "rxn01549 added\n",
      "rxn00307 added\n",
      "rxn03039 added\n",
      "rxn05114 added\n",
      "rxn05115 added\n",
      "rxn00140 added\n",
      "rxn02518 added\n",
      "rxn00063 added\n",
      "rxn00301 added\n",
      "rxn04783 added\n",
      "rxn00238 added\n",
      "rxn00134 added\n",
      "rxn01358 added\n",
      "rxn06075 added\n",
      "rxn00119 added\n",
      "rxn01360 added\n",
      "rxn06076 added\n",
      "rxn01517 added\n",
      "rxn14299 added\n",
      "rxn01367 added\n",
      "rxn00463 added\n",
      "rxn01704 added\n",
      "rxn00107 added\n",
      "rxn01219 added\n",
      "rxn00364 added\n",
      "rxn01648 added\n",
      "rxn01146 added\n",
      "rxn01366 added\n",
      "rxn00272 added\n",
      "rxn00503 added\n",
      "rxn00555 added\n",
      "rxn00509 added\n",
      "rxn00424 added\n",
      "rxn00422 added\n",
      "rxn00541 added\n",
      "rxn01833 added\n",
      "rxn04142 added\n",
      "rxn06044 added\n",
      "rxn00275 added\n",
      "rxn02928 added\n",
      "rxn40037 added\n",
      "rxn05957 added\n",
      "rxn05091 added\n",
      "rxn03057 added\n",
      "rxn01021 added\n",
      "rxn03052 added\n",
      "rxn00143 added\n",
      "rxn05107 added\n",
      "rxn00426 added\n",
      "rxn01757 added\n",
      "rxn01906 added\n",
      "rxn02229 added\n",
      "rxn15090 added\n",
      "rxn01303 added\n",
      "rxn04934 added\n",
      "rxn00738 added\n",
      "rxn16503 added\n",
      "rxn16511 added\n",
      "rxn05035 added\n",
      "rxn07435 added\n",
      "rxn07431 added\n",
      "rxn07433 added\n",
      "rxn07434 added\n",
      "rxn07430 added\n",
      "rxn07432 added\n",
      "rxn15510 added\n",
      "rxn05109 added\n",
      "rxn02751 added\n",
      "rxn00735 added\n",
      "rxn03435 added\n",
      "rxn15467 added\n",
      "rxn03436 added\n",
      "rxn15511 added\n",
      "rxn03068 added\n",
      "rxn08043 added\n",
      "rxn02749 added\n",
      "rxn15021 added\n",
      "rxn02011 added\n",
      "rxn11968 added\n",
      "rxn01418 added\n",
      "rxn01729 added\n",
      "rxn06112 added\n",
      "rxn02196 added\n",
      "rxn02826 added\n",
      "rxn02090 added\n",
      "rxn15949 added\n",
      "rxn15950 added\n",
      "rxn00853 added\n",
      "rxn00291 added\n",
      "rxn00467 added\n",
      "rxn05947 added\n",
      "rxn00183 added\n",
      "rxn02356 added\n",
      "rxn03071 added\n",
      "rxn03424 added\n",
      "rxn05776 added\n",
      "rxn15507 added\n",
      "rxn03426 added\n",
      "rxn00867 added\n",
      "rxn03195 added\n",
      "rxn03135 added\n",
      "rxn03345 added\n",
      "rxn01836 added\n",
      "rxn03333 added\n",
      "rxn03346 added\n",
      "rxn02087 added\n",
      "rxn02993 added\n",
      "rxn02397 added\n",
      "rxn03342 added\n",
      "rxn01715 added\n",
      "rxn01903 added\n",
      "rxn01004 added\n",
      "rxn01156 added\n",
      "rxn04598 added\n",
      "rxn04597 added\n",
      "rxn00489 added\n",
      "rxn04653 added\n",
      "rxn04650 added\n",
      "rxn02747 added\n",
      "rxn03801 added\n",
      "rxn03800 added\n",
      "rxn15998 added\n",
      "rxn07909 added\n",
      "rxn07908 added\n",
      "rxn03359 added\n",
      "rxn03358 added\n",
      "rxn02081 added\n",
      "rxn01564 added\n",
      "rxn02219 added\n",
      "rxn11962 added\n",
      "rxn00490 added\n",
      "rxn02476 added\n",
      "rxn16553 added\n",
      "rxn11571 added\n",
      "rxn06781 added\n",
      "rxn06421 added\n",
      "rxn03373 added\n",
      "rxn16206 added\n",
      "rxn16213 added\n",
      "rxn02525 added\n",
      "rxn01377 added\n",
      "rxn24067 added\n",
      "rxn25763 added\n",
      "rxn25680 added\n",
      "rxn01860 added\n",
      "rxn02139 added\n",
      "rxn06227 added\n",
      "rxn02008 added\n",
      "rxn06281 added\n",
      "rxn06168 added\n",
      "rxn12003 added\n",
      "rxn00198 added\n",
      "rxn00186 added\n",
      "rxn06401 added\n",
      "rxn12049 added\n",
      "rxn06272 added\n",
      "rxn06403 added\n",
      "rxn12204 added\n",
      "rxn06095 added\n",
      "rxn06096 added\n",
      "rxn15532 added\n",
      "rxn06165 added\n",
      "rxn40273 added\n",
      "rxn02284 added\n",
      "rxn00298 added\n",
      "rxn00016 added\n",
      "rxn15435 added\n",
      "rxn04703 added\n",
      "rxn16543 added\n",
      "rxn01750 added\n",
      "rxn06000 added\n",
      "rxn40254 added\n",
      "rxn40065 added\n",
      "rxn22648 added\n",
      "rxn13990 added\n",
      "rxn03919 added\n",
      "rxn03918 added\n",
      "rxn03916 added\n",
      "rxn03439 added\n",
      "rxn03182 added\n",
      "rxn16613 added\n",
      "rxn16611 added\n",
      "rxn03140 added\n",
      "rxn02286 added\n",
      "rxn04308 added\n",
      "rxn15315 added\n",
      "rxn39352 added\n",
      "rxn06140 added\n",
      "rxn00983 added\n",
      "rxn00765 added\n",
      "rxn06139 added\n",
      "rxn06671 added\n",
      "rxn06219 added\n",
      "rxn06362 added\n",
      "rxn01073 added\n",
      "rxn00616 added\n",
      "rxn06080 added\n",
      "rxn06043 added\n",
      "rxn06090 added\n",
      "rxn05970 added\n",
      "rxn06803 added\n",
      "rxn00758 added\n",
      "rxn06376 added\n",
      "rxn06091 added\n",
      "rxn06045 added\n",
      "rxn00176 added\n",
      "rxn00226 added\n",
      "rxn00227 added\n",
      "rxn16082 added\n",
      "rxn16084 added\n",
      "rxn06926 added\n",
      "rxn14054 added\n",
      "rxn14089 added\n",
      "rxn15990 added\n",
      "rxn04726 added\n",
      "rxn04736 added\n",
      "rxn00921 added\n",
      "rxn01034 added\n",
      "rxn03870 added\n",
      "rxn05851 added\n",
      "rxn00934 added\n",
      "rxn00682 added\n",
      "rxn00908 added\n",
      "rxn00675 added\n",
      "rxn00986 added\n",
      "rxn03061 added\n",
      "rxn02169 added\n",
      "rxn00868 added\n",
      "rxn01731 added\n",
      "rxn00684 added\n",
      "rxn01601 added\n",
      "rxn00907 added\n",
      "rxn06538 added\n",
      "rxn14120 added\n",
      "rxn16360 added\n",
      "rxn14207 added\n",
      "rxn07849 added\n",
      "rxn06294 added\n",
      "rxn10613 added\n",
      "rxn07848 added\n",
      "rxn07847 added\n",
      "rxn15964 added\n",
      "rxn03644 added\n",
      "rxn01539 added\n",
      "rxn40102 added\n",
      "rxn07292 added\n",
      "rxn02485 added\n",
      "rxn23042 added\n",
      "rxn00436 added\n",
      "rxn15951 added\n",
      "rxn02988 added\n",
      "rxn02154 added\n",
      "rxn01669 added\n",
      "rxn01265 added\n",
      "rxn00076 added\n",
      "rxn00105 added\n",
      "rxn02176 added\n",
      "rxn02128 added\n",
      "rxn15466 added\n",
      "rxn06023 added\n",
      "rxn06864 added\n",
      "rxn00792 added\n",
      "rxn14045 added\n",
      "rxn02277 added\n",
      "rxn16573 added\n",
      "rxn21864 added\n",
      "rxn16385 added\n",
      "rxn40015 added\n",
      "rxn40016 added\n",
      "rxn40018 added\n",
      "rxn40019 added\n",
      "rxn40020 added\n",
      "rxn40014 added\n",
      "rxn15454 added\n",
      "rxn02986 added\n",
      "rxn16231 added\n",
      "rxn16232 added\n",
      "rxn16806 added\n",
      "rxn39964 added\n",
      "rxn16824 added\n",
      "rxn07436 added\n",
      "rxn10543 added\n",
      "rxn16574 added\n",
      "rxn16583 added\n",
      "rxn01323 added\n",
      "rxn05120 added\n",
      "rxn00056 added\n",
      "rxn02287 added\n",
      "rxn02056 added\n",
      "rxn02774 added\n",
      "rxn06887 added\n",
      "rxn03542 added\n",
      "rxn06979 added\n",
      "rxn03540 added\n",
      "rxn01084 added\n",
      "rxn03538 added\n",
      "rxn03150 added\n",
      "rxn05121 added\n",
      "rxn05054 added\n",
      "rxn04384 added\n",
      "rxn14070 added\n",
      "rxn08352 added\n",
      "rxn04113 added\n",
      "rxn09037 added\n",
      "rxn16686 added\n",
      "rxn07469 added\n",
      "rxn07468 added\n",
      "rxn16635 added\n",
      "rxn16594 added\n",
      "rxn16514 added\n",
      "rxn00498 added\n",
      "rxn05070 added\n",
      "rxn05071 added\n",
      "rxn14358 added\n",
      "rxn00109 added\n",
      "rxn00567 added\n",
      "rxn05801 added\n",
      "rxn14010 added\n",
      "rxn00388 added\n",
      "rxn00102 added\n",
      "rxn01350 added\n",
      "rxn40041 added\n",
      "rxn16341 added\n",
      "rxn40040 added\n",
      "rxn14050 added\n",
      "rxn06440 added\n",
      "rxn06439 added\n",
      "rxn06444 added\n",
      "rxn06432 added\n",
      "rxn06437 added\n",
      "rxn06435 added\n",
      "rxn06438 added\n",
      "rxn06936 added\n",
      "rxn06446 added\n",
      "rxn06443 added\n",
      "rxn06449 added\n",
      "rxn06300 added\n",
      "rxn06442 added\n",
      "rxn06441 added\n",
      "rxn06447 added\n",
      "rxn06448 added\n",
      "rxn06280 added\n",
      "rxn10534 added\n",
      "rxn11595 added\n",
      "rxn11747 added\n",
      "rxn03402 added\n",
      "rxn00504 added\n",
      "rxn09188 added\n",
      "rxn02358 added\n",
      "rxn03072 added\n",
      "rxn05982 added\n",
      "rxn02285 added\n",
      "rxn01667 added\n",
      "rxn12649 added\n",
      "rxn05890 added\n",
      "\n",
      " 433 / 433 reactions added successfully \n",
      " =====================\n",
      "100 / 805 reactions added succesfully\n",
      "200 / 805 reactions added succesfully\n",
      "300 / 805 reactions added succesfully\n",
      "400 / 805 reactions added succesfully\n",
      "500 / 805 reactions added succesfully\n",
      "600 / 805 reactions added succesfully\n",
      "700 / 805 reactions added succesfully\n",
      "800 / 805 reactions added succesfully\n",
      "805 / 805 reactions added succesfully\n"
     ]
    }
   ],
   "source": [
    "import cobra\n",
    "import pandas as pd\n",
    "\n",
    "import modelseedpy\n",
    "modelseed_path = 'C:\\\\Users\\\\vickenlee\\\\ModelSEEDDatabase'\n",
    "modelseed = modelseedpy.biochem.from_local(modelseed_path)\n",
    "\n",
    "\n",
    "print('======================== \\n %d reactions begin to added \\n'%len(unadded_rxn_id_list))\n",
    "\n",
    "i,j = 0,0\n",
    "for rxn_id in unadded_rxn_id_list:\n",
    "    reaction_type = mapping_reaction.query('ModelSeed_ID == \"'+ rxn_id +'\"')['reaction_type'].values[0]\n",
    "    if reaction_type == 'reversible':\n",
    "        direction = 'reversible'\n",
    "    elif reaction_type == 'irreversible':\n",
    "        direction = 'forward'\n",
    "    try:\n",
    "        add_ms_reaction(new_model, rxn_id = rxn_id, modelseed = modelseed, compartment = 'c0', direction = direction)\n",
    "        print(rxn_id+' added')\n",
    "        i+=1\n",
    "    except:\n",
    "        print(rxn_id+' added wrong !!!')\n",
    "        j+=1\n",
    "\n",
    "print('\\n %d / %d reactions added successfully \\n ====================='%(i,i+j))\n",
    "\n",
    "for metabolite in new_model.metabolites:\n",
    "    metaid = [meta for meta in metabolite.id.split('_') if meta.startswith('cpd')]\n",
    "    if metaid:\n",
    "        ms_metabolite = modelseed.get_seed_compound(metaid[0])\n",
    "        if  ms_metabolite.formula == ms_metabolite.formula:\n",
    "            metabolite.formula = ms_metabolite.formula\n",
    "        metabolite.charge = ms_metabolite.data['charge']\n",
    "\n",
    "# add gene\n",
    "mapping_reaction = pd.read_excel(org_name+'_mapping_reaction.xlsx', index_col=0)  \n",
    "i=1\n",
    "for reaction in new_model.reactions:\n",
    "    rxnid = [rxn for rxn in reaction.id.split('_') if rxn.startswith('rxn')]\n",
    "    if rxnid:\n",
    "        gene_list_str = '( '+mapping_reaction.query('ModelSeed_ID == \"'+ rxnid[0] +'\"')['entry_name'].values[0].replace(' ', ' or ')+' )'\n",
    "        reaction.gene_reaction_rule = gene_list_str\n",
    "    if i%100 == 0:\n",
    "        print('%d / %d reactions added succesfully'%(i,len(new_model.reactions)))\n",
    "    i+=1\n",
    "print('%d / %d reactions added succesfully'%(len(new_model.reactions),len(new_model.reactions)))\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "cobra.io.write_sbml_model(new_model,filename=org_name+'_copyfrom_DriftModel_and_add_from_ModelSEED.xml')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.8"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "4fcda1b261581924d09de579bd4b132cd3d21b247043ee13c49095974b82ed9f"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
