{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "if not os.path.exists('BIGG'):\n",
    "    os.makedirs('BIGG')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Modify origin json file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# modified from http://bigg.ucsd.edu/static/namespace/universal_model.json to a cobrapy json file\n",
    "\n",
    "def add_to_dir(my_dir, key, value):\n",
    "    if key in my_dir:\n",
    "        if isinstance(my_dir[key], list):\n",
    "            my_dir[key].append(value)\n",
    "        else:\n",
    "            my_dir[key] = [my_dir[key], value]\n",
    "    else:\n",
    "        my_dir[key] = value\n",
    "\n",
    "\n",
    "import json\n",
    "\n",
    "with open('BIGG\\\\universal_model.json', 'r') as f:\n",
    "    data = json.load(f)\n",
    "\n",
    "for item in data['metabolites']+data['reactions']:\n",
    "    annolist = item.get('annotation')\n",
    "    if annolist:\n",
    "        annodir={}\n",
    "        for anno in annolist:\n",
    "            spli = anno[1].split('/')\n",
    "            key = spli[-2]\n",
    "            value = spli[-1]\n",
    "            add_to_dir(annodir, key, value)\n",
    "        item['annotation'] = annodir\n",
    "    else:\n",
    "        item['annotation'] = {}\n",
    "\n",
    "with open('BIGG\\\\universal_model_modified.json', 'w') as f:\n",
    "    json.dump(data, f)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Get relation betweent other database and BIGG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>bigg_universal</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>2806eac56a0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>15638</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>28301</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td></td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model bigg_universal at 0x2806eac56a0>"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import cobra\n",
    "biggunimodel = cobra.io.load_json_model('BIGG\\\\universal_model_modified.json')\n",
    "#cobra.io.write_sbml_model(biggunimodel,\"BIGG\\\\universal_model_modified.xml\")\n",
    "biggunimodel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get the list of other database reactions with a certain annotation\n",
    "\n",
    "def add_to_dir(my_dir, key, value):\n",
    "    if key in my_dir:\n",
    "        if isinstance(my_dir[key], list):\n",
    "            my_dir[key].append(value)\n",
    "        else:\n",
    "            my_dir[key] = [my_dir[key], value]\n",
    "    else:\n",
    "        my_dir[key] = value\n",
    "\n",
    "        \n",
    "KEGG_BIGG_dir = {}\n",
    "for rxn in biggunimodel.reactions:\n",
    "    kegganno = rxn.annotation.get('kegg.reaction')\n",
    "    if kegganno:\n",
    "        if type(kegganno) is list:\n",
    "            for i in kegganno:\n",
    "                add_to_dir(KEGG_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(KEGG_BIGG_dir, kegganno, rxn.id)\n",
    "\n",
    "EC_BIGG_dir = {}\n",
    "for rxn in biggunimodel.reactions:\n",
    "    ecanno = rxn.annotation.get('ec-code')\n",
    "    if ecanno:\n",
    "        if type(ecanno) is list:\n",
    "            for i in ecanno:\n",
    "                add_to_dir(EC_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(EC_BIGG_dir, ecanno, rxn.id)\n",
    "\n",
    "cyc_BIGG_dir = {}\n",
    "for rxn in biggunimodel.reactions:\n",
    "    cycanno = rxn.annotation.get('biocyc')\n",
    "    if cycanno:\n",
    "        if type(cycanno) is list:\n",
    "            for i in cycanno:\n",
    "                add_to_dir(cyc_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(cyc_BIGG_dir, cycanno, rxn.id)\n",
    "\n",
    "seed_BIGG_dir = {}\n",
    "for rxn in biggunimodel.reactions:\n",
    "    seedanno = rxn.annotation.get('seed.reaction')\n",
    "    if seedanno:\n",
    "        if type(seedanno) is list:\n",
    "            for i in seedanno:\n",
    "                add_to_dir(seed_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(seed_BIGG_dir, seedanno, rxn.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "def select_rxn(biggunimodel, rxnlist):\n",
    "    if not isinstance(rxnlist,list):\n",
    "        unirxn = biggunimodel.reactions.get_by_id(rxnlist)\n",
    "        comlist=[]\n",
    "        for key, value in unirxn.metabolites.items():\n",
    "            comlist.append(key.id.split('_')[-1])\n",
    "        if set(comlist).issubset(set(['c','e','p'])):\n",
    "            return rxnlist\n",
    "        else:\n",
    "            return None\n",
    "    else:\n",
    "        newrxnlist_c = []\n",
    "        newrxnlist_p = []\n",
    "        newrxnlist_e = []\n",
    "        for rxn in rxnlist:\n",
    "            unirxn = biggunimodel.reactions.get_by_id(rxn)\n",
    "            comlist=[]\n",
    "            for key, value in unirxn.metabolites.items():\n",
    "                comlist.append(key.id.split('_')[-1])\n",
    "            if set(comlist).issubset(set(['c'])):\n",
    "                newrxnlist_c.append(rxn)\n",
    "            elif set(comlist).issubset(set(['c','p'])):\n",
    "                newrxnlist_p.append(rxn)\n",
    "            elif set(comlist).issubset(set(['c','e','p'])):\n",
    "                newrxnlist_e.append(rxn)\n",
    "        if len(newrxnlist_c) > 0:\n",
    "            return min(newrxnlist_c, key=len)\n",
    "        elif len(newrxnlist_p) > 0:\n",
    "            return min(newrxnlist_p, key=len)\n",
    "        elif len(newrxnlist_p) > 0:\n",
    "                return min(newrxnlist_e, key=len)\n",
    "        else:\n",
    "            return None\n",
    "        \n",
    "def kegg2bigg(keggid,biggunimodel=biggunimodel,KEGG_BIGG_dir=KEGG_BIGG_dir):\n",
    "    if KEGG_BIGG_dir.get(keggid):\n",
    "        selr = select_rxn(biggunimodel, KEGG_BIGG_dir.get(keggid))\n",
    "        if selr:\n",
    "            if type(selr) is list:\n",
    "                print(keggid)\n",
    "                for i in selr:\n",
    "                    print('     http://bigg.ucsd.edu/universal/reactions/'+i)\n",
    "                    #print('KEGG_BIGG_dir_selected['+'\\''+keggid+'\\''+'] = '+'\\''+i+'\\'')\n",
    "                #print(keggid, selr)\n",
    "                return selr\n",
    "            elif type(selr) is str:\n",
    "                return selr     \n",
    "    else:\n",
    "        return None        \n",
    "\n",
    "def bigg2kegg(biggid,biggunimodel=biggunimodel):\n",
    "    try:\n",
    "        rxn = biggunimodel.reactions.get_by_id(biggid)\n",
    "    except:\n",
    "        return ''\n",
    "    if rxn.annotation.get('kegg.reaction'):\n",
    "        return rxn.annotation.get('kegg.reaction')\n",
    "    else:\n",
    "        return ''  \n",
    "    \n",
    "def seed2bigg(seedid,biggunimodel=biggunimodel,seed_BIGG_dir=seed_BIGG_dir):\n",
    "    if seed_BIGG_dir.get(seedid):\n",
    "        selr = select_rxn(biggunimodel, seed_BIGG_dir.get(seedid))\n",
    "        if selr:\n",
    "            if type(selr) is list:\n",
    "                print(seedid, selr)\n",
    "                return selr\n",
    "            elif type(selr) is str:\n",
    "                return selr     \n",
    "    else:\n",
    "        return None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "KEGG_BIGG_dir_selected = {}\n",
    "for key, value in KEGG_BIGG_dir.items():\n",
    "    new_value = kegg2bigg(key)\n",
    "    if new_value:\n",
    "        KEGG_BIGG_dir_selected[key] = new_value"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "with open('KEGG\\\\KEGG_BIGG_dir_selected.json', 'w') as f:\n",
    "    f.write(json.dumps(KEGG_BIGG_dir_selected))\n",
    "with open('BIGG\\\\KEGG_BIGG_dir_selected.json', 'w') as f:\n",
    "    f.write(json.dumps(KEGG_BIGG_dir_selected))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rxn09806 ['2DHPtm', '2DHPtn']\n",
      "rxn09802 ['34HPPt2p', '34HPPt2m']\n",
      "rxn09869 ['34HPPt2p', '34HPPt2m']\n",
      "rxn09793 ['4H2OGLTtp', '4H2OGLTtm']\n",
      "rxn09868 ['4H2OGLTtp', '4H2OGLTtm']\n",
      "rxn09648 ['6PGLter', '6PGLthr']\n",
      "rxn13213 ['ACCOAtn', 'ACCOAgt', 'ACCOAtr', 'ACCOAts', 'ACCOAtm_1']\n",
      "rxn09865 ['ASPGLU2m', 'ASPGLUtp']\n",
      "rxn13231 ['ASPGLU2m', 'ASPGLUtp']\n",
      "rxn09528 ['ATPS3m', 'ATPS3g', 'ATPS3v']\n",
      "rxn09529 ['ATPS3m', 'ATPS3g', 'ATPS3v']\n",
      "rxn09530 ['ATPS3m', 'ATPS3g', 'ATPS3v']\n",
      "rxn13766 ['ATPS3m', 'ATPS3g', 'ATPS3v']\n",
      "rxn09780 ['ATPtm_H', 'ATPtp_H']\n",
      "rxn09863 ['ATPtm_H', 'ATPtp_H']\n",
      "rxn09779 ['CITtap', 'CITtam']\n",
      "rxn09862 ['CITtap', 'CITtam']\n",
      "rxn09777 ['CITtcm', 'CITtcp']\n",
      "rxn09861 ['CITtcm', 'CITtcp']\n",
      "rxn13290 ['FRDPtm', 'FRDPtc', 'FRDPth', 'FRDPtcr', 'FRDPtr']\n",
      "rxn13332 ['IPDPtm', 'IPDPtr', 'IPDPtx', 'IPDPth']\n",
      "rxn13407 ['TRDRDtp', 'r1441']\n",
      "rxn09483 ['ASPOcm_1', 'ASPOcm']\n",
      "rxn09804 ['2OMPH_5tn', '2OMPH_5tm']\n",
      "rxn09829 ['2OMPH_5tn', '2OMPH_5tm']\n",
      "rxn09817 ['DHORtn', 'DHORStm']\n",
      "rxn12647 ['DM_datp_m', 'DATPtn', 'DM_datp_n', 'DATPtm_cho']\n",
      "rxn09836 ['MALOAAth', 'MALOAAtm', 'MALOAAtp']\n",
      "rxn00011 ['PDHam1hi', 'PDHam1mi']\n",
      "rxn00544 ['PDCh', 'PDCm']\n",
      "rxn02271 ['DHRT_2mbcoa', 'r0604']\n",
      "rxn09774 ['r0995', 'CRNCARtm', 'CRNCARtp']\n",
      "rxn09859 ['r0995', 'CRNCARtm', 'CRNCARtp']\n",
      "rxn13314 ['HISt6', 'r2416']\n",
      "rxn02342 ['PDHam2hi', 'PDHam2mi']\n",
      "rxn00738 ['ACCAHh', 'ACCAHm']\n",
      "rxn01523 ['URO', 'URAOX']\n",
      "rxn02867 ['DHRT_ivcoa', 'r0656']\n",
      "rxn06586 ['DHRT_ivcoa', 'r0656']\n",
      "rxn03430 ['RE3421M', 'RE3421R']\n"
     ]
    }
   ],
   "source": [
    "seed_BIGG_dir_selected = {}\n",
    "for key, value in seed_BIGG_dir.items():\n",
    "    new_value = seed2bigg(key)\n",
    "    if new_value:\n",
    "        seed_BIGG_dir_selected[key] = new_value"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "del_list = ['rxn09806','rxn09802','rxn09869','rxn09793','rxn09868','rxn09648','rxn13213','rxn09865',\n",
    "            'rxn13231','rxn09528','rxn09529','rxn09530','rxn13766','rxn09780','rxn09863','rxn09779',\n",
    "            'rxn09862','rxn09777','rxn09861','rxn13290','rxn13332','rxn13407','rxn09483','rxn09804',\n",
    "            'rxn09829','rxn09817','rxn12647','rxn09836','rxn00011','rxn00544','rxn02271','rxn09774',\n",
    "            'rxn09859','rxn13314','rxn02342','rxn00738','rxn01523','rxn02867','rxn06586','rxn03430']\n",
    "for key in del_list:\n",
    "    del seed_BIGG_dir_selected[key]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('BIGG\\\\SEED_BIGG_dir_selected.json', 'w') as f:\n",
    "    f.write(json.dumps(seed_BIGG_dir_selected))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Draw BIGG label to KEGG pathway map pictures"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\lix\\AppData\\Local\\Temp\\ipykernel_17980\\3013797931.py:4: FutureWarning: The squeeze argument has been deprecated and will be removed in a future version. Append .squeeze(\"columns\") to the call to squeeze.\n",
      "\n",
      "\n",
      "  a = K2B.pd.read_csv('KEGG\\\\Reaction_in_pathway_count.csv',index_col=0,squeeze=True)\n"
     ]
    }
   ],
   "source": [
    "import myModules.KEGG2BIGG as K2B\n",
    "unikegglist = list(set(K2B.KEGG_pathways_reactions['R_ID'].tolist()))\n",
    "mappedinBIGG = [r for r in unikegglist if K2B.KEGG_BIGG_dir_selected.get(r)]\n",
    "a = K2B.pd.read_csv('KEGG\\\\Reaction_in_pathway_count.csv',index_col=0,squeeze=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "map00230 104 / 153 Purine metabolism\n",
      "map00240 73 / 99 Pyrimidine metabolism\n",
      "map00270 60 / 102 Cysteine and methionine metabolism\n",
      "map00520 49 / 146 Amino sugar and nucleotide sugar metabolism\n",
      "map00860 49 / 171 Porphyrin metabolism\n",
      "map00330 48 / 95 Arginine and proline metabolism\n",
      "map00630 47 / 83 Glyoxylate and dicarboxylate metabolism\n",
      "map00620 45 / 82 Pyruvate metabolism\n",
      "map00260 44 / 72 Glycine, serine and threonine metabolism\n",
      "map00680 39 / 105 Methane metabolism\n",
      "map00040 34 / 77 Pentose and glucuronate interconversions\n",
      "map00250 33 / 46 Alanine, aspartate and glutamate metabolism\n",
      "map00760 33 / 75 Nicotinate and nicotinamide metabolism\n",
      "map00720 32 / 57 Carbon fixation pathways in prokaryotes\n",
      "map00790 31 / 69 Folate biosynthesis\n",
      "map00350 31 / 100 Tyrosine metabolism\n",
      "map00650 31 / 69 Butanoate metabolism\n",
      "map00640 30 / 64 Propanoate metabolism\n",
      "map00071 29 / 46 Fatty acid degradation\n",
      "map00400 28 / 48 Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "map00670 27 / 31 One carbon pool by folate\n",
      "map00052 26 / 53 Galactose metabolism\n",
      "map00562 26 / 61 Inositol phosphate metabolism\n",
      "map00380 26 / 101 Tryptophan metabolism\n",
      "map00564 25 / 90 Glycerophospholipid metabolism\n",
      "map00362 25 / 90 Benzoate degradation\n",
      "map00770 25 / 35 Pantothenate and CoA biosynthesis\n",
      "map00970 25 / 32 Aminoacyl-tRNA biosynthesis\n",
      "map00051 24 / 82 Fructose and mannose metabolism\n",
      "map00310 24 / 70 Lysine degradation\n",
      "map00010 23 / 55 Glycolysis / Gluconeogenesis\n",
      "map00541 23 / 92 O-Antigen nucleotide sugar biosynthesis\n",
      "map00360 22 / 62 Phenylalanine metabolism\n",
      "map00500 22 / 77 Starch and sucrose metabolism\n",
      "map00340 22 / 53 Histidine metabolism\n",
      "map00220 21 / 31 Arginine biosynthesis\n",
      "map00480 20 / 54 Glutathione metabolism\n",
      "map00030 19 / 56 Pentose phosphate pathway\n",
      "map00053 19 / 71 Ascorbate and aldarate metabolism\n",
      "map00280 19 / 51 Valine, leucine and isoleucine degradation\n",
      "map00710 19 / 26 Carbon fixation in photosynthetic organisms\n",
      "map00622 19 / 36 Xylene degradation\n",
      "map00061 19 / 68 Fatty acid biosynthesis\n",
      "map00920 18 / 52 Sulfur metabolism\n",
      "map00062 18 / 35 Fatty acid elongation\n",
      "map00290 17 / 25 Valine, leucine and isoleucine biosynthesis\n",
      "map00740 17 / 29 Riboflavin metabolism\n",
      "map00410 16 / 35 beta-Alanine metabolism\n",
      "map00130 16 / 73 Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "map00300 16 / 37 Lysine biosynthesis\n",
      "map00910 16 / 36 Nitrogen metabolism\n",
      "map00900 16 / 55 Terpenoid backbone biosynthesis\n",
      "map00020 15 / 27 Citrate cycle (TCA cycle)\n",
      "map00750 15 / 39 Vitamin B6 metabolism\n",
      "map00561 15 / 48 Glycerolipid metabolism\n",
      "map00540 14 / 49 Lipopolysaccharide biosynthesis\n",
      "map00100 13 / 66 Steroid biosynthesis\n",
      "map00470 12 / 68 D-Amino acid metabolism\n",
      "map00780 11 / 26 Biotin metabolism\n",
      "map01220 10 / 243 Degradation of aromatic compounds\n",
      "map00550 9 / 25 Peptidoglycan biosynthesis\n",
      "map00627 9 / 103 Aminobenzoate degradation\n",
      "map00521 9 / 24 Streptomycin biosynthesis\n",
      "map00600 8 / 44 Sphingolipid metabolism\n",
      "map01040 8 / 50 Biosynthesis of unsaturated fatty acids\n",
      "map00140 8 / 122 Steroid hormone biosynthesis\n",
      "map00983 8 / 51 Drug metabolism - other enzymes\n",
      "map00430 7 / 27 Taurine and hypotaurine metabolism\n",
      "map00730 7 / 33 Thiamine metabolism\n",
      "map00460 7 / 53 Cyanoamino acid metabolism\n",
      "map00960 7 / 71 Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "map00643 7 / 23 Styrene degradation\n",
      "map00982 6 / 80 Drug metabolism - cytochrome P450\n",
      "map00332 6 / 29 Carbapenem biosynthesis\n",
      "map00623 6 / 50 Toluene degradation\n",
      "map00909 5 / 87 Sesquiterpenoid and triterpenoid biosynthesis\n",
      "map00450 5 / 23 Selenocompound metabolism\n",
      "map01053 5 / 14 Biosynthesis of siderophore group nonribosomal peptides\n",
      "map00791 4 / 27 Atrazine degradation\n",
      "map00830 4 / 29 Retinol metabolism\n",
      "map00523 4 / 60 Polyketide sugar unit biosynthesis\n",
      "map00950 4 / 138 Isoquinoline alkaloid biosynthesis\n",
      "map00660 4 / 37 C5-Branched dibasic acid metabolism\n",
      "map00906 4 / 142 Carotenoid biosynthesis\n",
      "map00940 4 / 78 Phenylpropanoid biosynthesis\n",
      "map00591 4 / 16 Linoleic acid metabolism\n",
      "map00785 4 / 17 Lipoic acid metabolism\n",
      "map00261 3 / 27 Monobactam biosynthesis\n",
      "map00120 3 / 52 Primary bile acid biosynthesis\n",
      "map00966 3 / 61 Glucosinolate biosynthesis\n",
      "map00621 3 / 30 Dioxin degradation\n",
      "map00590 3 / 72 Arachidonic acid metabolism\n",
      "map00999 3 / 134 Biosynthesis of various plant secondary metabolites\n",
      "map00440 2 / 56 Phosphonate and phosphinate metabolism\n",
      "map00930 2 / 27 Caprolactam degradation\n",
      "map00525 2 / 35 Acarbose and validamycin biosynthesis\n",
      "map00401 2 / 31 Novobiocin biosynthesis\n",
      "map00552 2 / 46 Teichoic acid biosynthesis\n",
      "map00513 1 / 27 Various types of N-glycan biosynthesis\n",
      "map00902 1 / 66 Monoterpenoid biosynthesis\n",
      "map00980 1 / 104 Metabolism of xenobiotics by cytochrome P450\n",
      "map00510 1 / 37 N-Glycan biosynthesis\n",
      "map00903 1 / 47 Limonene degradation\n",
      "map00405 1 / 30 Phenazine biosynthesis\n",
      "map00965 1 / 26 Betalain biosynthesis\n",
      "map00624 1 / 114 Polycyclic aromatic hydrocarbon degradation\n",
      "map01057 1 / 127 Biosynthesis of type II polyketide products\n",
      "map00901 1 / 82 Indole alkaloid biosynthesis\n",
      "map01055 1 / 13 Biosynthesis of vancomycin group antibiotics\n",
      "map00402 1 / 8 Benzoxazinoid biosynthesis\n",
      "map00998 1 / 81 Biosynthesis of various antibiotics\n",
      "map00311 1 / 17 Penicillin and cephalosporin biosynthesis\n",
      "map00524 1 / 77 Neomycin, kanamycin and gentamicin biosynthesis\n",
      "map00625 1 / 50 Chloroalkane and chloroalkene degradation\n",
      "map00254 1 / 29 Aflatoxin biosynthesis\n"
     ]
    }
   ],
   "source": [
    "b = K2B.pathway_count(mappedinBIGG,a)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from PIL import Image, ImageFont, ImageDraw\n",
    "\n",
    "def drawtext(draw,text,coords_list):\n",
    "    box = draw.textbbox((0,0),text, font=font)\n",
    "    w = box[2] - box[0]\n",
    "    h = box[3] - box[1]\n",
    "    if type(coords_list) == list:\n",
    "        coords_num = [2*int(i) for i in coords_list]\n",
    "        #draw.rectangle(coords_num, fill=(255, 192, 203), outline=(5, 5, 5),width=1)\n",
    "        #draw.text(((coords_num[0]+coords_num[2])/2-w/2,(coords_num[3]+coords_num[1])/2-h/2), text, font=font,fill=(0, 0, 0))\n",
    "        draw.text(((coords_num[0]+coords_num[2])/2-w/2,coords_num[3]), text, font=font,fill=(218, 155, 0))\n",
    "    else:\n",
    "        print('coords_list should be a list')\n",
    "\n",
    "\n",
    "map_list=[key for key in b.keys()]\n",
    "#map_list = ['map00680']\n",
    "font = ImageFont.truetype(\"arial.ttf\", 15)\n",
    "for mapp in map_list:\n",
    "    map_reactions = K2B.KEGG_pathways_reactions[K2B.KEGG_pathways_reactions['pw_ID']==mapp]\n",
    "    inBIGG_map_reactions = [r for r in mappedinBIGG if r in map_reactions['R_ID'].to_list()]\n",
    "    img = Image.open(\"KEGG\\\\maps\\\\mapasR\\\\%s_asR.png\"%mapp)\n",
    "    draw = ImageDraw.Draw(img)\n",
    "    for r in inBIGG_map_reactions:\n",
    "        text = K2B.KEGG_BIGG_dir_selected.get(r)\n",
    "        findresult = map_reactions[map_reactions['R_ID']==r]['coords'].to_list()\n",
    "        for i in findresult:\n",
    "            coords_list = i.split(',')\n",
    "            if len(coords_list) == 4:\n",
    "                drawtext(draw,text,coords_list) \n",
    "    img.save('KEGG\\\\maps\\\\mapedinBIGGandR\\\\%s_biggandR.png'%mapp)\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.11"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
