{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "org_name = 'mko'\n",
    "org_full_name = 'Methylomonas_koyamae'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "org_name = 'mbry'\n",
    "org_full_name = 'Methylocystis_bryophila'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "org_name = 'mca'\n",
    "org_full_name = 'Methylococcus_capsulatus'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "org_name = 'mca'"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Initialize new model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>Methylocystis_bryophila</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>1d2f82f29a0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td></td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model Methylocystis_bryophila at 0x1d2f82f29a0>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import cobra\n",
    "\n",
    "new_model = cobra.Model(org_full_name)\n",
    "new_model"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## import universal model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "def add_to_dir(my_dir, key, value):\n",
    "    if key in my_dir:\n",
    "        if isinstance(my_dir[key], list):\n",
    "            my_dir[key].append(value)\n",
    "        else:\n",
    "            my_dir[key] = [my_dir[key], value]\n",
    "    else:\n",
    "        my_dir[key] = value\n",
    "\n",
    "\n",
    "import json\n",
    "\n",
    "with open('universal_model.json', 'r') as f:\n",
    "    data = json.load(f)\n",
    "\n",
    "for item in data['metabolites']+data['reactions']:\n",
    "    annolist = item.get('annotation')\n",
    "    if annolist:\n",
    "        annodir={}\n",
    "        for anno in annolist:\n",
    "            spli = anno[1].split('/')\n",
    "            key = spli[-2]\n",
    "            value = spli[-1]\n",
    "            add_to_dir(annodir, key, value)\n",
    "        item['annotation'] = annodir\n",
    "    else:\n",
    "        item['annotation'] = {}\n",
    "\n",
    "with open('universal_model_modified.json', 'w') as f:\n",
    "    json.dump(data, f)\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>bigg_universal</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>152f7abb640</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>15638</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>28301</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td></td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model bigg_universal at 0x152f7abb640>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "unimodel = cobra.io.load_json_model('other_model\\\\universal_model_modified.json')\n",
    "#cobra.io.write_sbml_model(unimodel,\"universal_model.xml\")\n",
    "unimodel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "def add_to_dir(my_dir, key, value):\n",
    "    if key in my_dir:\n",
    "        if isinstance(my_dir[key], list):\n",
    "            my_dir[key].append(value)\n",
    "        else:\n",
    "            my_dir[key] = [my_dir[key], value]\n",
    "    else:\n",
    "        my_dir[key] = value\n",
    "\n",
    "        \n",
    "KEGG_BIGG_dir = {}\n",
    "for rxn in unimodel.reactions:\n",
    "    kegganno = rxn.annotation.get('kegg.reaction')\n",
    "    if kegganno:\n",
    "        if type(kegganno) is list:\n",
    "            for i in kegganno:\n",
    "                add_to_dir(KEGG_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(KEGG_BIGG_dir, kegganno, rxn.id)\n",
    "\n",
    "EC_BIGG_dir = {}\n",
    "for rxn in unimodel.reactions:\n",
    "    ecanno = rxn.annotation.get('ec-code')\n",
    "    if ecanno:\n",
    "        if type(ecanno) is list:\n",
    "            for i in ecanno:\n",
    "                add_to_dir(EC_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(EC_BIGG_dir, ecanno, rxn.id)\n",
    "\n",
    "cyc_BIGG_dir = {}\n",
    "for rxn in unimodel.reactions:\n",
    "    cycanno = rxn.annotation.get('biocyc')\n",
    "    if cycanno:\n",
    "        if type(cycanno) is list:\n",
    "            for i in cycanno:\n",
    "                add_to_dir(cyc_BIGG_dir, i, rxn.id)\n",
    "        else:\n",
    "            add_to_dir(cyc_BIGG_dir, cycanno, rxn.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/inchi/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/bigg.reaction/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/metanetx.reaction/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/metanetx.reaction/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "http://identifiers.org/metanetx.reaction/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>iMcBath</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>152f7bc8ca0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>879</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>913</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>730</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>1.0*BIOMASS_Mcapsulatus - 1.0*BIOMASS_Mcapsulatus_reverse_6a067</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>cytosol, extracellular, periplasm, inner membrane</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model iMcBath at 0x152f7bc8ca0>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "iMcBath_model = cobra.io.read_sbml_model('other_model\\\\iMcBath.xml')\n",
    "iMcBath_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "iMcBath_rxnlist = [rxn.id for rxn in iMcBath_model.reactions]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "with open('MethylococcusCapsulatusMetabolismMap.json', 'r') as f:\n",
    "    data = json.load(f)\n",
    "\n",
    "mappediMcrxnlist = []\n",
    "for key, value in data[1]['reactions'].items():\n",
    "    mappediMcrxnlist.append(value['bigg_id'])\n",
    "\n",
    "mappediMcrxnlist = list(set(mappediMcrxnlist))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## build by carveme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "for root, dirs, files in os.walk('genome\\\\'+org_full_name+'_genome'):\n",
    "    print('1')\n",
    "    #if 'protein.faa' in files:\n",
    "    #    genome_path = root+dirs+'\\\\protein.faa'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'root' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[9], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m root\n",
      "\u001b[1;31mNameError\u001b[0m: name 'root' is not defined"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "genome_path = 'C:\\\\Users\\\\lix\\\\Documents\\\\GitHub\\\\BuildGEM\\\\\\genome\\\\Methylomonas_koyamae_genome\\\\ncbi_dataset\\\\data\\\\GCF_019669905.1\\\\protein.faa'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "cmd1='carve '+genome_path+' --gapfill M9 --mediadb Database\\media_db.tsv --output ' + org_name+'_model\\\\'+org_name+'_carveme_model.xml'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "cmd1='carve '+genome_path+' --output ' + org_name +'_carveme_model.xml'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "carve C:\\Users\\lix\\Documents\\GitHub\\BuildGEM\\\\genome\\Methylomonas_koyamae_genome\\ncbi_dataset\\data\\GCF_019669905.1\\protein.faa --output mko_carveme_model.xml\n"
     ]
    }
   ],
   "source": [
    "print(cmd1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.system(cmd1)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Please run the printed code in Terminal"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "carveme_model = cobra.io.read_sbml_model(org_name+'_model\\\\'+org_name+'_carveme_model.xml')\n",
    "cobra.io.save_json_model(carveme_model, org_name+'_model\\\\'+org_name+'_carveme_model.json')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## build by RAVEN in matlab"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "``` matlab\n",
    "mbry_kegg_model = getKEGGModelForOrganism('mbry');\n",
    "mko_kegg_model = getKEGGModelForOrganism('mko');\n",
    "mca_kegg_model = getKEGGModelForOrganism('mca');\n",
    "\n",
    "mbry=[mbry_kegg_model.rxns,mbry_kegg_model.rxnNames,mbry_kegg_model.eccodes,mbry_kegg_model.grRules,mbry_kegg_model.subSystems];\n",
    "mko=[mko_kegg_model.rxns,mko_kegg_model.rxnNames,mko_kegg_model.eccodes,mko_kegg_model.grRules,mko_kegg_model.subSystems];\n",
    "mca=[mca_kegg_model.rxns,mca_kegg_model.rxnNames,mca_kegg_model.eccodes,mca_kegg_model.grRules,mca_kegg_model.subSystems];\n",
    "\n",
    "\n",
    "data = mca;\n",
    "x=size(data,1);\n",
    "\n",
    "fid=fopen('mca_raven.txt','w');\n",
    "for i=1:x\n",
    "    for j=1:5\n",
    "        a = data{i,j};\n",
    "        if iscell(a)\n",
    "            aa = a{1};\n",
    "            for m=1:length(a)-1\n",
    "                aa= strcat(aa,\";\",a{m+1});\n",
    "            end\n",
    "            a = aa(1:length(aa));\n",
    "        end\n",
    "        fprintf(fid,'%s\\t',a);\n",
    "    end\n",
    "    fprintf(fid,'%s\\n',\"\");\n",
    "end\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# import RAVEN database\n",
    "\n",
    "import pandas as pd\n",
    "rxn_raven_df =pd.DataFrame(columns=['id', 'name','eccode','reversibility','subsystem','subsystemid'])\n",
    "\n",
    "with open('Database\\\\keggrxn_fromRAVENnew.txt', 'r',encoding='utf-8-sig') as f:\n",
    "    for line in f.readlines():\n",
    "        spli = line.split('\\t')\n",
    "        if \";\" in spli[4]:\n",
    "            subsytem = spli[4].split(';')\n",
    "            subsytemid = spli[5].split(';')\n",
    "            for i in range(len(subsytem)):\n",
    "                rxn_raven_df.loc[len(rxn_raven_df.index)]=[spli[0],spli[1],spli[2],spli[3],subsytem[i],subsytemid[i]]\n",
    "        else:\n",
    "            rxn_raven_df.loc[len(rxn_raven_df.index)]=spli\n",
    "f.close()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define function to print kegg rxn\n",
    "\n",
    "def printkeggrxn(keggid,rxn_raven_df=rxn_raven_df):\n",
    "    search = rxn_raven_df[rxn_raven_df['id']==keggid]\n",
    "    if len(search.index) == 0:\n",
    "        print(keggid)\n",
    "        print('     no rxn')\n",
    "    else:\n",
    "        print(keggid+': '+search.iloc[0]['name'])\n",
    "        if search.iloc[0]['subsystemid'].strip() == '':\n",
    "            print('     https://www.kegg.jp/entry/'+keggid)\n",
    "        else:\n",
    "            for i in range(len(search.index)):\n",
    "                sysid = search.iloc[i]['subsystemid'].strip()\n",
    "                sysname = search.iloc[i]['subsystem'].strip()\n",
    "                print('     https://www.kegg.jp/pathway/%s+%s      %s'%(sysid,keggid,sysname))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# import organism specific kegg rxn and genes\n",
    "\n",
    "import pandas as pd\n",
    "spec_rxn_raven_df =pd.DataFrame(columns=['id', 'name','eccode','genes','subsystem'])\n",
    "\n",
    "with open(org_name+'_model\\\\'+org_name+'_raven.txt', 'r') as f:\n",
    "    for line in f:\n",
    "        line = line[:-2].split('\\t')\n",
    "        if ';' in line[2]:\n",
    "            ecs = line[2].split(';')\n",
    "            for ec in ecs:\n",
    "                spec_rxn_raven_df.loc[len(spec_rxn_raven_df)] = [line[0], line[1], ec, line[3],line[4]]\n",
    "        else:\n",
    "            spec_rxn_raven_df.loc[len(spec_rxn_raven_df)] = line\n",
    "f.close()\n",
    "\n",
    "spec_kegglist = sorted(set(list(spec_rxn_raven_df['id'])),key=list(spec_rxn_raven_df['id']).index)\n",
    "\n",
    "#rxn_raven_df = pd.read_csv(org_name+'_model\\\\'+org_name+'_raven.txt', sep='\\t', header=None, index_col=None, names=['id', 'name','eccode','genes'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# import Mca kegg rxn and genes\n",
    "\n",
    "Mca_rxn_raven_df =pd.DataFrame(columns=['id', 'name','eccode','genes','subsystem'])\n",
    "\n",
    "with open('other_model\\\\mca_raven.txt', 'r') as f:\n",
    "    for line in f:\n",
    "        line = line[:-2].split('\\t')\n",
    "        if ';' in line[2]:\n",
    "            ecs = line[2].split(';')\n",
    "            for ec in ecs:\n",
    "                Mca_rxn_raven_df.loc[len(Mca_rxn_raven_df)] = [line[0], line[1], ec, line[3],line[4]]\n",
    "        else:\n",
    "            Mca_rxn_raven_df.loc[len(Mca_rxn_raven_df)] = line\n",
    "f.close()\n",
    "\n",
    "Mca_kegglist = sorted(set(list(Mca_rxn_raven_df['id'])),key=list(Mca_rxn_raven_df['id']).index)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Model does not contain SBML fbc package information.\n",
      "'' is not a valid SBML 'SId'.\n",
      "SBML package 'layout' not supported by cobrapy, information is not parsed\n",
      "SBML package 'render' not supported by cobrapy, information is not parsed\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00498_c \"Adenosine diphosphoglucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00498_c \"Adenosine diphosphoglucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00008_c \"Adenosine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00008_c \"Adenosine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00369_c \"Starch\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00369_c \"Starch\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00009_c \"Phosphoric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00009_c \"Phosphoric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00103_c \"D-Glucose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00103_c \"D-Glucose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01165_c \"L-Glutamate gamma-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01165_c \"L-Glutamate gamma-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00001_c \"water\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00001_c \"water\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00080_c \"Hydron\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00080_c \"Hydron\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03912_c \"1-Pyrroline-5-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03912_c \"1-Pyrroline-5-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00093_c \"D-Glycerol 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00093_c \"D-Glycerol 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00111_c \"3-Hydroxy-2-oxopropyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00111_c \"3-Hydroxy-2-oxopropyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00063_c \"Cytidine triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00063_c \"Cytidine triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C08362_c \"(9Z)-Hexadecenoicacid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C08362_c \"(9Z)-Hexadecenoicacid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00005_c \"Reduced nicotinamide adenine dinucleotide phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00005_c \"Reduced nicotinamide adenine dinucleotide phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00013_c \"Diphosphoric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00013_c \"Diphosphoric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00006_c \"Nicotinamide adenine dinucleotide phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00006_c \"Nicotinamide adenine dinucleotide phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPG161161_c \"phosphatidyl glycerol with palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPG161161_c \"phosphatidyl glycerol with palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00055_c \"Cytidylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00055_c \"Cytidylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06424_c \"Tetradecanoicacid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06424_c \"Tetradecanoicacid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPG161140_c \"phosphatidyl glycerol with myristic and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPG161140_c \"phosphatidyl glycerol with myristic and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00249_c \"Hexadecanoicacid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00249_c \"Hexadecanoicacid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPG160161_c \"phosphatidyl glycerol with palmitic acid and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPG160161_c \"phosphatidyl glycerol with palmitic acid and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00065_c \"Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00065_c \"Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00011_c \"Carbon dioxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00011_c \"Carbon dioxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPE161161_c \"phosphatidyl ethanol amine with palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPE161161_c \"phosphatidyl ethanol amine with palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPE161140_c \"phosphatidyl ethanol amine with myristic and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPE161140_c \"phosphatidyl ethanol amine with myristic and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPE160161_c \"phosphatidyl ethanol amine with palmitic acid and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPE160161_c \"phosphatidyl ethanol amine with palmitic acid and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00019_c \"S-Adenosyl-L-methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00019_c \"S-Adenosyl-L-methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00021_c \"S-Adenosylhomocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00021_c \"S-Adenosylhomocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPC161161_c \"phosphatidyl choline with palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPC161161_c \"phosphatidyl choline with palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPC161140_c \"phosphatidyl choline with myristic and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPC161140_c \"phosphatidyl choline with myristic and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPC160161_c \"phosphatidyl choline with palmitic acid and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CPC160161_c \"phosphatidyl choline with palmitic acid and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00116_c \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00116_c \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CCL161161_c \"cardiolipin with palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CCL161161_c \"cardiolipin with palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CCL161140_c \"cardiolipin with myristic and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CCL161140_c \"cardiolipin with myristic and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CCL160161_c \"cardiolipin with palmitic acid and palmitoleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_CCL160161_c \"cardiolipin with palmitic acid and palmitoleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00004_c \"Reduced nicotinamide adenine dinucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00004_c \"Reduced nicotinamide adenine dinucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00007_c \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00007_c \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00003_c \"Nicotinamide adenine dinucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00003_c \"Nicotinamide adenine dinucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15781_c \"24-Methylenecholesterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15781_c \"24-Methylenecholesterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C21106_c \"24-epi-Campesterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C21106_c \"24-epi-Campesterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00075_c \"Uridine triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00075_c \"Uridine triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C21027_c \"N-Acetylmuramicacidalpha-1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C21027_c \"N-Acetylmuramicacidalpha-1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01050_c \"UDP-N-acetylmuramic acid;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01050_c \"UDP-N-acetylmuramic acid;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00002_c \"ATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00002_c \"ATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02713_c \"N-Acetylmuramate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02713_c \"N-Acetylmuramate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C21015_c \"gamma-L-Glutamyl-L-2-aminobutyrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C21015_c \"gamma-L-Glutamyl-L-2-aminobutyrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00037_c \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00037_c \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C21016_c \"Ophthalmate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C21016_c \"Ophthalmate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00025_c \"L-Glutamic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00025_c \"L-Glutamic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02356_c \"(S)-2-Aminobutyric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02356_c \"(S)-2-Aminobutyric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00026_c \"Oxoglutaric acid;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00026_c \"Oxoglutaric acid;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00109_c \"Alpha-Ketobutyrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00109_c \"Alpha-Ketobutyrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01172_c \"Beta-D-Glucose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01172_c \"Beta-D-Glucose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01236_c \"6-Phospho-D-glucono-1,5-lactone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01236_c \"6-Phospho-D-glucono-1,5-lactone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00010_c \"Coenzyme A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00010_c \"Coenzyme A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00022_c \"pyruvic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00022_c \"pyruvic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02869_c \"Oxidizedflavodoxin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02869_c \"Oxidizedflavodoxin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00024_c \"Acetyl coenzyme A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00024_c \"Acetyl coenzyme A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02745_c \"Reducedflavodoxin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02745_c \"Reducedflavodoxin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11453_c \"2-C-Methyl-D-erythritol 2,4-cyclodiphosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11453_c \"2-C-Methyl-D-erythritol 2,4-cyclodiphosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11811_c \"(E)-4-Hydroxy-3-methylbut-2-en-1-yl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11811_c \"(E)-4-Hydroxy-3-methylbut-2-en-1-yl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20247_c \"2-(2-Carboxy-4-methylthiazol-5-yl)ethyl phosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20247_c \"2-(2-Carboxy-4-methylthiazol-5-yl)ethyl phosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04752_c \"4-Amino-2-methyl-5-diphosphomethylpyrimidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04752_c \"4-Amino-2-methyl-5-diphosphomethylpyrimidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01081_c \"Thiamine phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01081_c \"Thiamine phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01209_c \"Malonyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01209_c \"Malonyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05744_c \"Acetoacetyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05744_c \"Acetoacetyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00345_c \"6-Phospho-D-gluconate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00345_c \"6-Phospho-D-gluconate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00199_c \"D-Ribulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00199_c \"D-Ribulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00125_c \"Cytochrome c3+\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00125_c \"Cytochrome c3+\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00192_c \"Hydroxylamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00192_c \"Hydroxylamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00126_c \"Reduced cytochrome c\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00126_c \"Reduced cytochrome c\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00088_c \"Nitrite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00088_c \"Nitrite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00320_c \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00320_c \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00059_c \"Sulfuric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00059_c \"Sulfuric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00441_c \"L-Aspartic 4-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00441_c \"L-Aspartic 4-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20258_c \"(2S,4S)-4-Hydroxy-2,3,4,5-tetrahydrodipicolinate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20258_c \"(2S,4S)-4-Hydroxy-2,3,4,5-tetrahydrodipicolinate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19845_c \"7-Hydroxy-7-oxoheptanoyl-[acyl-carrier protein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19845_c \"7-Hydroxy-7-oxoheptanoyl-[acyl-carrier protein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00041_c \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00041_c \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00229_c \"Acyl-carrier protein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00229_c \"Acyl-carrier protein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01092_c \"8-Amino-7-oxononanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01092_c \"8-Amino-7-oxononanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20378_c \"Enoylpimeloyl-[acyl-carrier protein] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20378_c \"Enoylpimeloyl-[acyl-carrier protein] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19846_c \"Pimelyl-[acp] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19846_c \"Pimelyl-[acp] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20377_c \"3-Hydroxypimeloyl-[acyl-carrier protein] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20377_c \"3-Hydroxypimeloyl-[acyl-carrier protein] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20376_c \"3-Ketopimeloyl-[acyl-carrier protein] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20376_c \"3-Ketopimeloyl-[acyl-carrier protein] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20375_c \"Glutaryl-[acyl-carrier protein] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20375_c \"Glutaryl-[acyl-carrier protein] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20374_c \"Enoylglutaryl-[acyl-carrier protein] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20374_c \"Enoylglutaryl-[acyl-carrier protein] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20373_c \"3-Hydroxyglutaryl-[acp] methyl ester;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20373_c \"3-Hydroxyglutaryl-[acp] methyl ester;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20372_c \"3-Ketoglutaryl-[acyl-carrier protein] methyl ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20372_c \"3-Ketoglutaryl-[acyl-carrier protein] methyl ester\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19673_c \"Malonyl-[acp] methyl ester;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19673_c \"Malonyl-[acp] methyl ester;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00288_c \"Bicarbonate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00288_c \"Bicarbonate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01594_c \"Linamarin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01594_c \"Linamarin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00221_c \"Beta-D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00221_c \"Beta-D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02659_c \"2-Hydroxyisobutyronitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02659_c \"2-Hydroxyisobutyronitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C08334_c \"Lotaustralin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C08334_c \"Lotaustralin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18796_c \"2-Hydroxy-2-methylbutanenitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C18796_c \"2-Hydroxy-2-methylbutanenitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05143_c \"(S)-4-Hydroxymandelonitrile beta-D-glucoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05143_c \"(S)-4-Hydroxymandelonitrile beta-D-glucoside\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03742_c \"(S)-4-Hydroxymandelonitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03742_c \"(S)-4-Hydroxymandelonitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20239_c \"6-Carboxytetrahydropterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20239_c \"6-Carboxytetrahydropterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00014_c \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00014_c \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C20248_c \"7-Carboxy-7-deazaguanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C20248_c \"7-Carboxy-7-deazaguanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15996_c \"7-Cyano-7-deazaguanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15996_c \"7-Cyano-7-deazaguanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04895_c \"7,8-Dihydroneopterin 3'-triphosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04895_c \"7,8-Dihydroneopterin 3'-triphosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00084_c \"Acetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00084_c \"Acetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00536_c \"Inorganic triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00536_c \"Inorganic triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06019_c \"D-arabino-6-Phospho-hex-3-ulose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06019_c \"D-arabino-6-Phospho-hex-3-ulose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05345_c \"beta-D-Fructose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05345_c \"beta-D-Fructose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19879_c \"D-glycero-alpha-D-manno-Heptose 1,7-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19879_c \"D-glycero-alpha-D-manno-Heptose 1,7-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19880_c \"D-glycero-alpha-D-manno-Heptose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19880_c \"D-glycero-alpha-D-manno-Heptose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05382_c \"altro-Heptulose 7-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05382_c \"altro-Heptulose 7-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19878_c \"D-glycero-alpha-D-manno-Heptose 7-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19878_c \"D-glycero-alpha-D-manno-Heptose 7-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00132_c \"Methanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00132_c \"Methanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C19787_c \"S-Methyl-5'-thioinosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C19787_c \"S-Methyl-5'-thioinosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04188_c \"S-Methyl-5-thio-D-ribose 1-phosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04188_c \"S-Methyl-5-thio-D-ribose 1-phosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00262_c \"Hypoxanthine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00262_c \"Hypoxanthine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00170_c \"Thiomethyladenosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00170_c \"Thiomethyladenosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00007_p \"C00007[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00390_p \"C00390[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01438_p \"C01438[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00132_p \"C00132[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00399_p \"C00399[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00044_c \"Guanosine 5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00044_c \"Guanosine 5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05198_c \"5'-Deoxyadenosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05198_c \"5'-Deoxyadenosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C21310_c \"(8S)-3',8-Cyclo-7,8-dihydroguanosine5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C21310_c \"(8S)-3',8-Cyclo-7,8-dihydroguanosine5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00073_c \"Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00073_c \"Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00173_c \"Acyl-[acyl-carrier protein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00173_c \"Acyl-[acyl-carrier protein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00681_c \"Lysophosphatidic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00681_c \"Lysophosphatidic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18902_c \"Methylseleninate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C18902_c \"Methylseleninate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05703_c \"Methylselenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05703_c \"Methylselenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04489_c \"5-Methyltetrahydropteroyltri-L-glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04489_c \"5-Methyltetrahydropteroyltri-L-glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05698_c \"Selenohomocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05698_c \"Selenohomocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05335_c \"L-Selenomethionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05335_c \"L-Selenomethionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04144_c \"Tetrahydropteroyltri-L-glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04144_c \"Tetrahydropteroyltri-L-glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00448_c \"(2E,6E)-Farnesyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00448_c \"(2E,6E)-Farnesyl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00129_c \"delta3-Methyl-3-butenyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00129_c \"delta3-Methyl-3-butenyl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04146_c \"all-trans-Octaprenyl diphosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04146_c \"all-trans-Octaprenyl diphosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06753_c \"Glycol chlorohydrin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06753_c \"Glycol chlorohydrin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18233_c \"FerricytochromecL\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06754_c \"2-Chloroethanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06754_c \"2-Chloroethanal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18234_c \"FerrocytochromecL\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00469_c \"Ethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00469_c \"Ethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15532_c \"N-Acetyl-L-citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15532_c \"N-Acetyl-L-citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00033_c \"Acetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00033_c \"Acetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00327_c \"Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00327_c \"Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01217_c \"Tetrahydromethanopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01217_c \"Tetrahydromethanopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04377_c \"N5,N10-Methylenetetrahydromethanopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04377_c \"N5,N10-Methylenetetrahydromethanopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18174_c \"Carboxynorspermidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C18174_c \"Carboxynorspermidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03375_c \"Norspermidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03375_c \"Norspermidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18172_c \"Carboxyspermidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C18172_c \"Carboxyspermidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00315_c \"Spermidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00315_c \"Spermidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02962_c \"D-Allose6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02962_c \"D-Allose6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18096_c \"D-Allulose6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C18096_c \"D-Allulose6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00140_c \"2-Acetamido-2-deoxy-D-glucose;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00140_c \"2-Acetamido-2-deoxy-D-glucose;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04501_c \"N-Acetyl-alpha-D-glucosamine 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04501_c \"N-Acetyl-alpha-D-glucosamine 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17562_c \"2-Polyprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17562_c \"2-Polyprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00399_c \"Ubiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00399_c \"Ubiquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17561_c \"2-Polyprenyl-3-methyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17561_c \"2-Polyprenyl-3-methyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17560_c \"2-Polyprenyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17560_c \"2-Polyprenyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17552_c \"2-Polyprenyl-6-methoxyphenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17552_c \"2-Polyprenyl-6-methoxyphenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17551_c \"3-Polyprenylcatechol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17551_c \"3-Polyprenylcatechol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17401_c \"Cobalt-factor III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17401_c \"Cobalt-factor III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11539_c \"Cobalt-precorrin 3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11539_c \"Cobalt-precorrin 3\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00138_c \"Reducedferredoxin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00138_c \"Reducedferredoxin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00139_c \"Oxidizedferredoxin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00139_c \"Oxidizedferredoxin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00095_c \"D-Fructose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00095_c \"D-Fructose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00089_c \"Sucrose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00089_c \"Sucrose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06006_c \"(S)-2-Hydroxy-2-ethyl-3-oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06006_c \"(S)-2-Hydroxy-2-ethyl-3-oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00092_c \"Glucose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00092_c \"Glucose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00118_c \"Glyceraldehyde 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00118_c \"Glyceraldehyde 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00085_c \"D-Fructose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00085_c \"D-Fructose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00279_c \"D-Erythrose 4-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00279_c \"D-Erythrose 4-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00091_c \"Succinyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00091_c \"Succinyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01137_c \"Decarboxy-AdoMet\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01137_c \"Decarboxy-AdoMet\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01672_c \"Cadaverine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01672_c \"Cadaverine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16565_c \"Aminopropylcadaverine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16565_c \"Aminopropylcadaverine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00235_c \"Prenyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00235_c \"Prenyl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01001_c \"Formylmethanofuran\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01001_c \"Formylmethanofuran\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00058_c \"Formic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00058_c \"Formic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00862_c \"Methanofuran\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00862_c \"Methanofuran\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04330_c \"N5,N10-Methenyltetrahydromethanopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04330_c \"N5,N10-Methenyltetrahydromethanopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00067_c \"Formaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00067_c \"Formaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16391_c \"Trinitrotoluene;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16391_c \"Trinitrotoluene;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16393_c \"2-Hydroxylamino-4,6-dinitrotoluene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16393_c \"2-Hydroxylamino-4,6-dinitrotoluene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16396_c \"2,4-Diamino-6-nitrotoluene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16396_c \"2,4-Diamino-6-nitrotoluene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00282_c \"Hydrogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00282_c \"Hydrogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16399_c \"2,4-Diamino-6-hydroxylaminotoluene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16399_c \"2,4-Diamino-6-hydroxylaminotoluene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16243_c \"Cobalt-precorrin5B\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16243_c \"Cobalt-precorrin5B\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11542_c \"Cobalt-precorrin6\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11542_c \"Cobalt-precorrin6\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16221_c \"(2E)-Octadecenoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16221_c \"(2E)-Octadecenoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04088_c \"Octadecanoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04088_c \"Octadecanoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16220_c \"(R)-3-Hydroxyoctadecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16220_c \"(R)-3-Hydroxyoctadecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16219_c \"3-Oxostearoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16219_c \"3-Oxostearoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05764_c \"Hexadecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05764_c \"Hexadecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05754_c \"trans-Dec-2-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05754_c \"trans-Dec-2-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04180_c \"cis-Dec-3-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04180_c \"cis-Dec-3-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16675_c \"7-Aminomethyl-7-deazaguanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16675_c \"7-Aminomethyl-7-deazaguanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05107_c \"7-Dehydrodesmosterol;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05107_c \"7-Dehydrodesmosterol;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01164_c \"Cholesta-5,7-dien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01164_c \"Cholesta-5,7-dien-3beta-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05108_c \"14-Demethyllanosterol;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05108_c \"14-Demethyllanosterol;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15915_c \"4,4-Dimethyl-5alpha-cholesta-8-en-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15915_c \"4,4-Dimethyl-5alpha-cholesta-8-en-3beta-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05437_c \"5alpha-Cholesta-8,24-dien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05437_c \"5alpha-Cholesta-8,24-dien-3beta-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03845_c \"Cholestenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03845_c \"Cholestenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01789_c \"Campesterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01789_c \"Campesterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C08821_c \"Isofucosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C08821_c \"Isofucosterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01753_c \"beta-Sitosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01753_c \"beta-Sitosterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15809_c \"2-Iminoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15809_c \"2-Iminoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00027_c \"Hydrogen peroxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00027_c \"Hydrogen peroxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15672_c \"HemeO\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15672_c \"HemeO\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15670_c \"HemeA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15670_c \"HemeA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00032_c \"Heme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00032_c \"Heme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00049_c \"L-Aspartic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00049_c \"L-Aspartic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05840_c \"Iminosuccinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05840_c \"Iminosuccinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15667_c \"5-Carboxyamino-1-(5-phospho-D-ribosyl)imidazole\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15667_c \"5-Carboxyamino-1-(5-phospho-D-ribosyl)imidazole\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04751_c \"5-Amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04751_c \"5-Amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03373_c \"5-Amino-1-(5-phospho-D-ribosyl)imidazole\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03373_c \"5-Amino-1-(5-phospho-D-ribosyl)imidazole\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15650_c \"2,3-Diketo-5-methylthiopentyl-1-phosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15650_c \"2,3-Diketo-5-methylthiopentyl-1-phosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15606_c \"1,2-Dihydroxy-5-(methylthio)pent-1-en-3-one\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15606_c \"1,2-Dihydroxy-5-(methylthio)pent-1-en-3-one\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04582_c \"S-Methyl-5-thio-D-ribulose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04582_c \"S-Methyl-5-thio-D-ribulose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01180_c \"2-Oxo-4-methylthiobutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01180_c \"2-Oxo-4-methylthiobutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00237_c \"Carbon monoxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00237_c \"Carbon monoxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C08276_c \"3-Methylthiopropionate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C08276_c \"3-Methylthiopropionate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00751_c \"Supraene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00751_c \"Supraene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06309_c \"Hopan-22-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06309_c \"Hopan-22-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06310_c \"Hop-22(29)-ene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06310_c \"Hop-22(29)-ene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03194_c \"(R)-1-Aminopropan-2-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03194_c \"(R)-1-Aminopropan-2-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06507_c \"Adenosylcobyrinatehexaamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06507_c \"Adenosylcobyrinatehexaamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06508_c \"Adenosylcobinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06508_c \"Adenosylcobinamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15556_c \"2-Hydroxy-3-oxobutyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15556_c \"2-Hydroxy-3-oxobutyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04454_c \"5-Amino-6-(5-phospho-D-ribitylamino)uracil\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04454_c \"5-Amino-6-(5-phospho-D-ribitylamino)uracil\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04732_c \"5-Amino-6-(1-D-ribitylamino)uracil;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04732_c \"5-Amino-6-(1-D-ribitylamino)uracil;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05774_c \"Cobinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05774_c \"Cobinamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00143_c \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00143_c \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00440_c \"5-Methyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00440_c \"5-Methyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11537_c \"(2R)-3-Sulfolactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11537_c \"(2R)-3-Sulfolactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05528_c \"3-Sulfopyruvic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05528_c \"3-Sulfopyruvic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05966_c \"(5Z,8Z,11Z,13E)-(15S)-15-Hydroperoxyicosa-5,8,11,13-tetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05966_c \"(5Z,8Z,11Z,13E)-(15S)-15-Hydroperoxyicosa-5,8,11,13-tetraenoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00051_c \"Glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00051_c \"Glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00127_c \"Oxidized glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00127_c \"Oxidized glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04742_c \"(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04742_c \"(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05356_c \"(5S,6E,8Z,11Z,14Z)-5-Hydroperoxyicosa-6,8,11,14-tetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05356_c \"(5S,6E,8Z,11Z,14Z)-5-Hydroperoxyicosa-6,8,11,14-tetraenoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04805_c \"5-Hydroxyeicosatetraenoate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04805_c \"5-Hydroxyeicosatetraenoate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02909_c \"2-Hydroxymethylnaphthalene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02909_c \"2-Hydroxymethylnaphthalene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C14099_c \"2-Naphthalenecarboxaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C14099_c \"2-Naphthalenecarboxaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C14089_c \"1-Naphthalenemethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C14089_c \"1-Naphthalenemethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C14090_c \"1-Formylnaphthalene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C14090_c \"1-Formylnaphthalene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03263_c \"Coproporphyrinogen III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03263_c \"Coproporphyrinogen III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01079_c \"Protoporphyrinogen IX\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01079_c \"Protoporphyrinogen IX\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C13309_c \"Demethylphylloquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C13309_c \"Demethylphylloquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02059_c \"Vitamin K1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02059_c \"Vitamin K1\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00423_c \"Trans-Cinnamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00423_c \"Trans-Cinnamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C12622_c \"cis-3-(3-Carboxyethenyl)-3,5-cyclohexadiene-1,2-diol;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C12622_c \"cis-3-(3-Carboxyethenyl)-3,5-cyclohexadiene-1,2-diol;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05629_c \"3-Phenylpropionic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05629_c \"3-Phenylpropionic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11588_c \"cis-3-(2-Carboxy-ethyl)-3,5-cyclo-hexadiene-1,2-diol;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11588_c \"cis-3-(2-Carboxy-ethyl)-3,5-cyclo-hexadiene-1,2-diol;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00035_c \"Guanosine diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00035_c \"Guanosine diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06509_c \"Adenosylcobinamide phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06509_c \"Adenosylcobinamide phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04122_c \"(R)-1-Aminopropan-2-yl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04122_c \"(R)-1-Aminopropan-2-yl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11907_c \"4,6-Dideoxy-4-oxo-dTDP-D-glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11907_c \"4,6-Dideoxy-4-oxo-dTDP-D-glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00688_c \"dTDP-4-oxo-L-rhamnose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00688_c \"dTDP-4-oxo-L-rhamnose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00842_c \"DTDP-alpha-D-glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00842_c \"DTDP-alpha-D-glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04574_c \"ditrans,octacis-Undecaprenyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04574_c \"ditrans,octacis-Undecaprenyl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05814_c \"2-Octaprenyl-3-methyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05814_c \"2-Octaprenyl-3-methyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05815_c \"2-Octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05815_c \"2-Octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11437_c \"1-Deoxy-D-xylulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11437_c \"1-Deoxy-D-xylulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11638_c \"1-Amino-3-(phosphohydroxy)propan-2-one\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11638_c \"1-Amino-3-(phosphohydroxy)propan-2-one\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00627_c \"Pyridoxol 5'-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00627_c \"Pyridoxol 5'-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05773_c \"Cobyrinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05773_c \"Cobyrinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00064_c \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00064_c \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06504_c \"Cobyrinic acid a,c-diamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06504_c \"Cobyrinic acid a,c-diamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11545_c \"Cobalt-precorrin-8\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11545_c \"Cobalt-precorrin-8\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11540_c \"Cobalt-precorrin 4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11540_c \"Cobalt-precorrin 4\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16242_c \"Cobalt-precorrin 5A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16242_c \"Cobalt-precorrin 5A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11538_c \"Cobalt-sirohydrochlorin;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11538_c \"Cobalt-sirohydrochlorin;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05778_c \"Sirohydrochlorin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05778_c \"Sirohydrochlorin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00175_c \"Cobalt ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00175_c \"Cobalt ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01943_c \"Obtusifoliol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01943_c \"Obtusifoliol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11508_c \"4alpha-Methyl-5alpha-ergosta-8,14,24(28)-trien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11508_c \"4alpha-Methyl-5alpha-ergosta-8,14,24(28)-trien-3beta-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00061_c \"Flavin mononucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00061_c \"Flavin mononucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01847_c \"Reduced FMN\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01847_c \"Reduced FMN\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01189_c \"Lathosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01189_c \"Lathosterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05439_c \"5alpha-Cholesta-7,24-dien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05439_c \"5alpha-Cholesta-7,24-dien-3beta-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01222_c \"GDP-4-dehydro-6-deoxy-alpha-D-mannose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01222_c \"GDP-4-dehydro-6-deoxy-alpha-D-mannose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00325_c \"GDP-beta-L-fucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00325_c \"GDP-beta-L-fucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11434_c \"2-C-Methyl-D-erythritol 4-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11434_c \"2-C-Methyl-D-erythritol 4-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06055_c \"4-(Phosphonooxy)-L-threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06055_c \"4-(Phosphonooxy)-L-threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C07335_c \"2-Amino-3-oxo-4-phosphonooxybutyrate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C07335_c \"2-Amino-3-oxo-4-phosphonooxybutyrate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00043_c \"UDP-N-acetylglucosamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00043_c \"UDP-N-acetylglucosamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04851_c \"MurAc(oyl-L-Ala-D-gamma-Glu-L-Lys-D-Ala-D-Ala)-diphospho-undecaprenol;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04851_c \"MurAc(oyl-L-Ala-D-gamma-Glu-L-Lys-D-Ala-D-Ala)-diphospho-undecaprenol;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00015_c \"Uridine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00015_c \"Uridine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05893_c \"GlcNAc-(1->4)-MurNAc(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05893_c \"GlcNAc-(1->4)-MurNAc(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11472_c \"D-glycero-beta-D-manno-Heptose 1,7-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11472_c \"D-glycero-beta-D-manno-Heptose 1,7-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C07838_c \"D-glycero-beta-D-manno-Heptose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C07838_c \"D-glycero-beta-D-manno-Heptose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C07836_c \"D-glycero-beta-D-manno-Heptose 7-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C07836_c \"D-glycero-beta-D-manno-Heptose 7-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06397_c \"ADP-D-glycero-beta-D-manno-heptose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06397_c \"ADP-D-glycero-beta-D-manno-heptose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01724_c \"Lanosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01724_c \"Lanosterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11455_c \"4,4-Dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11455_c \"4,4-Dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11436_c \"2-Phospho-4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11436_c \"2-Phospho-4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11435_c \"4-(Cytidine 5'-diphospho)-2-C-methyl-D-erythritol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11435_c \"4-(Cytidine 5'-diphospho)-2-C-methyl-D-erythritol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17556_c \"ditrans,octacis-Undecaprenyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17556_c \"ditrans,octacis-Undecaprenyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04882_c \"UDP-N-acetylmuramoyl-L-alanyl-D-gamma-glutamyl-6-carboxy-L-lysyl-D-alanyl-D-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04882_c \"UDP-N-acetylmuramoyl-L-alanyl-D-gamma-glutamyl-6-carboxy-L-lysyl-D-alanyl-D-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05897_c \"Undecaprenyl-diphospho-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05897_c \"Undecaprenyl-diphospho-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00105_c \"5'Uridylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00105_c \"5'Uridylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04702_c \"UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04702_c \"UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysyl-D-alanyl-D-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00156_c \"Hydroxybenzenecarboxylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00156_c \"Hydroxybenzenecarboxylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05809_c \"3-Octaprenyl-4-hydroxybenzoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05809_c \"3-Octaprenyl-4-hydroxybenzoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04442_c \"2-Dehydro-3-deoxy-D-gluconate 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04442_c \"2-Dehydro-3-deoxy-D-gluconate 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C09814_c \"Cyanobenzene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C09814_c \"Cyanobenzene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00180_c \"Benzoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00180_c \"Benzoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11355_c \"4-Amino-4-deoxychorismate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11355_c \"4-Amino-4-deoxychorismate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00568_c \"P-Aminobenzoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00568_c \"P-Aminobenzoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01548_c \"Ethyne\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01548_c \"Ethyne\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06547_c \"Ethylene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06547_c \"Ethylene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06156_c \"D-Glucosamine 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06156_c \"D-Glucosamine 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06612_c \"cis-3-Chloroallyl alcohol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06612_c \"cis-3-Chloroallyl alcohol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C16348_c \"cis-3-Chloro-2-propenal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C16348_c \"cis-3-Chloro-2-propenal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06611_c \"trans-3-Chloroallyl alcohol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06611_c \"trans-3-Chloroallyl alcohol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06613_c \"trans-3-Chloro-2-propenal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06613_c \"trans-3-Chloro-2-propenal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06506_c \"Adenosylcobyrinic acid a,c-diamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06506_c \"Adenosylcobyrinic acid a,c-diamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06399_c \"Hydrogenobyrinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06399_c \"Hydrogenobyrinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06503_c \"Hydrogenobyrinic acid a,c-diamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06503_c \"Hydrogenobyrinic acid a,c-diamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06510_c \"Adenosylcobinamide-GDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06510_c \"Adenosylcobinamide-GDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05775_c \"N1-(alpha-D-Ribosyl)-5,6-dimethylbenzimidazole\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05775_c \"N1-(alpha-D-Ribosyl)-5,6-dimethylbenzimidazole\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00144_c \"Guanylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00144_c \"Guanylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00194_c \"Funacomide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00194_c \"Funacomide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06505_c \"Cob(I)yrinic acid a,c-diamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06505_c \"Cob(I)yrinic acid a,c-diamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00697_c \"Nitrogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00697_c \"Nitrogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06407_c \"Precorrin4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06407_c \"Precorrin4\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06416_c \"Precorrin5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06416_c \"Precorrin5\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06406_c \"Precorrin 3B\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06406_c \"Precorrin 3B\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06408_c \"Precorrin8X\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06408_c \"Precorrin8X\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06319_c \"Precorrin 6B\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06319_c \"Precorrin 6B\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06056_c \"Hydroxythreonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06056_c \"Hydroxythreonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06054_c \"(3R)-3-Hydroxy-2-oxo-4-phosphonooxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06054_c \"(3R)-3-Hydroxy-2-oxo-4-phosphonooxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04121_c \"CMP-3-deoxy-D-manno-octulosonate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04121_c \"CMP-3-deoxy-D-manno-octulosonate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06024_c \"3-Deoxy-D-manno-octulosonyl-lipid IV(A);\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06024_c \"3-Deoxy-D-manno-octulosonyl-lipid IV(A);\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06025_c \"KDO2-lipid IV(A)\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06025_c \"KDO2-lipid IV(A)\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06010_c \"(2S)-2-Hydroxy-2-methyl-3-oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06010_c \"(2S)-2-Hydroxy-2-methyl-3-oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04181_c \"2-Oxo-3-hydroxyisovalerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04181_c \"2-Oxo-3-hydroxyisovalerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06007_c \"(2R,3R)-2,3-Dihydroxy-3-methylpentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06007_c \"(2R,3R)-2,3-Dihydroxy-3-methylpentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00671_c \"(3S)-3-Methyl-2-oxopentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00671_c \"(3S)-3-Methyl-2-oxopentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C14463_c \"(R)-3-Hydroxy-3-methyl-2-oxopentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C14463_c \"(R)-3-Hydroxy-3-methyl-2-oxopentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05947_c \"L-erythro-4-Hydroxyglutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05947_c \"L-erythro-4-Hydroxyglutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00036_c \"Oxalacetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00036_c \"Oxalacetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05946_c \"D-4-Hydroxy-2-oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05946_c \"D-4-Hydroxy-2-oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00402_c \"D-Aspartic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00402_c \"D-Aspartic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05923_c \"2,5-Diaminopyrimidine nucleoside triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05923_c \"2,5-Diaminopyrimidine nucleoside triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06148_c \"2,5-Diamino-6-(5'-triphosphoryl-3',4'-trihydroxy-2'-oxopentyl)-amino-4-oxopyrimidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06148_c \"2,5-Diamino-6-(5'-triphosphoryl-3',4'-trihydroxy-2'-oxopentyl)-amino-4-oxopyrimidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05922_c \"Formamidopyrimidine nucleoside triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05922_c \"Formamidopyrimidine nucleoside triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05898_c \"Undecaprenyl-diphospho-N-acetylmuramoyl-(N-acetylglucosamine)-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05898_c \"Undecaprenyl-diphospho-N-acetylmuramoyl-(N-acetylglucosamine)-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05847_c \"Polyprenyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05847_c \"Polyprenyl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05848_c \"4-Hydroxy-3-polyprenylbenzoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05848_c \"4-Hydroxy-3-polyprenylbenzoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05818_c \"2-Demethylmenaquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05818_c \"2-Demethylmenaquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00828_c \"Vitamin K2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00828_c \"Vitamin K2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05813_c \"2-Octaprenyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05813_c \"2-Octaprenyl-6-methoxy-1,4-benzoquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05812_c \"2-Octaprenyl-6-methoxyphenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05812_c \"2-Octaprenyl-6-methoxyphenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05811_c \"2-Octaprenyl-6-hydroxyphenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05811_c \"2-Octaprenyl-6-hydroxyphenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05810_c \"2-Octaprenylphenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05810_c \"2-Octaprenylphenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05807_c \"2-Polyprenylphenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05807_c \"2-Polyprenylphenol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05766_c \"Uroporphyrinogen I\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05766_c \"Uroporphyrinogen I\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05768_c \"Coproporphyrinogen I\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05768_c \"Coproporphyrinogen I\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05763_c \"trans-Hexadec-2-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05763_c \"trans-Hexadec-2-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05761_c \"Tetradecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05761_c \"Tetradecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05762_c \"3-Oxohexadecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05762_c \"3-Oxohexadecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05760_c \"trans-Tetradec-2-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05760_c \"trans-Tetradec-2-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05757_c \"(R)-3-Hydroxydodecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05757_c \"(R)-3-Hydroxydodecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05758_c \"trans-Dodec-2-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05758_c \"trans-Dodec-2-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05756_c \"3-Oxododecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05756_c \"3-Oxododecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05755_c \"Decanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05755_c \"Decanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05752_c \"Octanoyl-[acyl-carrier protein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05752_c \"Octanoyl-[acyl-carrier protein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05753_c \"3-Oxodecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05753_c \"3-Oxodecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05751_c \"trans-Oct-2-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05751_c \"trans-Oct-2-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05749_c \"Hexanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05749_c \"Hexanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05750_c \"3-Oxooctanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05750_c \"3-Oxooctanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05748_c \"trans-Hex-2-enoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05748_c \"trans-Hex-2-enoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05747_c \"(R)-3-Hydroxyhexanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05747_c \"(R)-3-Hydroxyhexanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05746_c \"3-Oxohexanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05746_c \"3-Oxohexanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05745_c \"Butyryl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05745_c \"Butyryl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05699_c \"L-Selenocystathionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05699_c \"L-Selenocystathionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05697_c \"Selenic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05697_c \"Selenic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05686_c \"Adenosine-5'-phosphoselenate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05686_c \"Adenosine-5'-phosphoselenate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05576_c \"3,4-Dihydroxyphenylethyleneglycol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05576_c \"3,4-Dihydroxyphenylethyleneglycol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05577_c \"3,4-Dihydroxyphenylglycolaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05577_c \"3,4-Dihydroxyphenylglycolaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05378_c \"beta-D-Fructose 1,6-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05378_c \"beta-D-Fructose 1,6-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05223_c \"Dodecanoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05223_c \"Dodecanoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05759_c \"3-Oxotetradecanoyl-[acp]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05759_c \"3-Oxotetradecanoyl-[acp]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04919_c \"Lipid A disaccharide bisphosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04919_c \"Lipid A disaccharide bisphosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04932_c \"Lipid A disaccharide;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04932_c \"Lipid A disaccharide;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04896_c \"5-(5-Phospho-D-ribosylaminoformimino)-1-(5-phosphoribosyl)-imidazole-4-carboxamide;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04896_c \"5-(5-Phospho-D-ribosylaminoformimino)-1-(5-phosphoribosyl)-imidazole-4-carboxamide;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04916_c \"5-[(5-Phospho-1-deoxy-D-ribulos-1-ylamino)methylideneamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04916_c \"5-[(5-Phospho-1-deoxy-D-ribulos-1-ylamino)methylideneamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04874_c \"7,8-Dihydroneopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04874_c \"7,8-Dihydroneopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00993_c \"D-Ala-D-Ala\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00993_c \"D-Ala-D-Ala\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04877_c \"UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimelate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04877_c \"UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimelate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04824_c \"2,3-Bis(3-hydroxytetradecanoyl)-alpha-D-glucosaminyl 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04824_c \"2,3-Bis(3-hydroxytetradecanoyl)-alpha-D-glucosaminyl 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04652_c \"UDP-2,3-bis(beta-hydroxymyristoyl)-D-glucosamine;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04652_c \"UDP-2,3-bis(beta-hydroxymyristoyl)-D-glucosamine;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04778_c \"alpha-Ribazole 5'-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04778_c \"alpha-Ribazole 5'-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04823_c \"1-(5'-Phosphoribosyl)-5-amino-4-(N-succinocarboxamide)-imidazole;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04823_c \"1-(5'-Phosphoribosyl)-5-amino-4-(N-succinocarboxamide)-imidazole;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04738_c \"UDP-3-O-(beta-hydroxymyristoyl)-N-acetylglucosamine;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04738_c \"UDP-3-O-(beta-hydroxymyristoyl)-N-acetylglucosamine;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06022_c \"UDP-3-O-[(3R)-3-hydroxymyristoyl]-alpha-D-glucosamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06022_c \"UDP-3-O-[(3R)-3-hydroxymyristoyl]-alpha-D-glucosamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05892_c \"UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-gamma-D-glutamyl-L-lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05892_c \"UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-gamma-D-glutamyl-L-lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04688_c \"beta-Hydroxymyristyl-[acyl-carrier protein];\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04688_c \"beta-Hydroxymyristyl-[acyl-carrier protein];\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00234_c \"10-Formyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00234_c \"10-Formyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04677_c \"5-Amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04677_c \"5-Amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04734_c \"5-Formamido-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04734_c \"5-Formamido-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00101_c \"Tetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00101_c \"Tetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00122_c \"trans-Butenedioic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00122_c \"trans-Butenedioic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04666_c \"D-erythro-Imidazole-glycerol phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04666_c \"D-erythro-Imidazole-glycerol phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04633_c \"(3R)-3-Hydroxypalmitoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04633_c \"(3R)-3-Hydroxypalmitoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04620_c \"(3R)-3-Hydroxyoctanoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04620_c \"(3R)-3-Hydroxyoctanoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04619_c \"(3R)-3-Hydroxydecanoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04619_c \"(3R)-3-Hydroxydecanoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04618_c \"(3R)-3-Hydroxybutanoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04618_c \"(3R)-3-Hydroxybutanoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04556_c \"4-Amino-5-phosphomethyl-2-methylpyrimidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04556_c \"4-Amino-5-phosphomethyl-2-methylpyrimidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04421_c \"N-Succinyl-L-2,6-diaminopimelate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04421_c \"N-Succinyl-L-2,6-diaminopimelate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04462_c \"N-Succinyl-L-2-amino-6-oxopimelate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04462_c \"N-Succinyl-L-2-amino-6-oxopimelate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04376_c \"5'-Phosphoribosyl-N-formylglycinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04376_c \"5'-Phosphoribosyl-N-formylglycinamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04640_c \"2-(Formamido)-N1-(5-phospho-D-ribosyl)acetamidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04640_c \"2-(Formamido)-N1-(5-phospho-D-ribosyl)acetamidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04332_c \"6,7-Dimethyl-8-(D-ribityl)lumazine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04332_c \"6,7-Dimethyl-8-(D-ribityl)lumazine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04272_c \"(2R)-2,3-Dihydroxy-3-methylbutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04272_c \"(2R)-2,3-Dihydroxy-3-methylbutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00141_c \"2-Keto-3-methylbutyric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00141_c \"2-Keto-3-methylbutyric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04246_c \"But-2-enoyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04246_c \"But-2-enoyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04411_c \"2-D-threo-Hydroxy-3-carboxy-isocaproate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04411_c \"2-D-threo-Hydroxy-3-carboxy-isocaproate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04236_c \"3-Carboxy-4-methyl-2-oxopentanoate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04236_c \"3-Carboxy-4-methyl-2-oxopentanoate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04593_c \"Methylisocitric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04593_c \"Methylisocitric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04225_c \"cis-2-Methylaconitate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04225_c \"cis-2-Methylaconitate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00155_c \"Homocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00155_c \"Homocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04079_c \"N-Pantothenoylcysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04079_c \"N-Pantothenoylcysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04352_c \"N-[(R)-4'-Phosphopantothenoyl]-L-cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04352_c \"N-[(R)-4'-Phosphopantothenoyl]-L-cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00119_c \"5-Phosphoribosyl 1-pyrophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00119_c \"5-Phosphoribosyl 1-pyrophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04051_c \"5-Amino-4-imidazolecarboxyamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04051_c \"5-Amino-4-imidazolecarboxyamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03972_c \"2,3,4,5-Tetrahydrodipicolinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03972_c \"2,3,4,5-Tetrahydrodipicolinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03939_c \"Acetyl-[acyl-carrierprotein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03939_c \"Acetyl-[acyl-carrierprotein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03838_c \"Glycinamide ribonucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03838_c \"Glycinamide ribonucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00445_c \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00445_c \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03722_c \"Quinolinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03722_c \"Quinolinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03684_c \"6-Pyruvoyl-5,6,7,8-tetrahydropterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03684_c \"6-Pyruvoyl-5,6,7,8-tetrahydropterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00097_c \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00097_c \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03492_c \"D-4'-Phosphopantothenate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03492_c \"D-4'-Phosphopantothenate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00020_c \"Adenosine 5'-monophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00020_c \"Adenosine 5'-monophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01005_c \"Dexfosfoserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01005_c \"Dexfosfoserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03232_c \"3-Phosphohydroxypyruvic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03232_c \"3-Phosphohydroxypyruvic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03114_c \"Dimethylbenzimidazole\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03114_c \"Dimethylbenzimidazole\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01185_c \"Nicotinic acid ribonucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01185_c \"Nicotinic acid ribonucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00253_c \"Nicotinic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00253_c \"Nicotinic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03090_c \"5-Phosphoribosylamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03090_c \"5-Phosphoribosylamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02741_c \"Phosphoribosyl-AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02741_c \"Phosphoribosyl-AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02739_c \"Phosphoribosyl-ATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02739_c \"Phosphoribosyl-ATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02631_c \"2-Isopropylmaleate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02631_c \"2-Isopropylmaleate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05670_c \"Beta-Aminopropionitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05670_c \"Beta-Aminopropionitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06114_c \"gamma-Glutamyl-3-aminopropiononitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06114_c \"gamma-Glutamyl-3-aminopropiononitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02512_c \"L-beta-Cyanoalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02512_c \"L-beta-Cyanoalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05711_c \"Gamma-Glutamyl-beta-cyanoalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05711_c \"Gamma-Glutamyl-beta-cyanoalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02504_c \"3-Carboxy-3-hydroxyisocaproate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02504_c \"3-Carboxy-3-hydroxyisocaproate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02463_c \"Dihydrosirohydrochlorin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02463_c \"Dihydrosirohydrochlorin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05772_c \"Precorrin 3A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05772_c \"Precorrin 3A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02320_c \"R-S-Glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02320_c \"R-S-Glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05729_c \"R-S-Cysteinylglycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05729_c \"R-S-Cysteinylglycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02226_c \"Methylmaleic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02226_c \"Methylmaleic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06032_c \"D-erythro-3-Methylmalate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06032_c \"D-erythro-3-Methylmalate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02612_c \"D-Citramalate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02612_c \"D-Citramalate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03313_c \"Phytonadiol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03313_c \"Phytonadiol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05109_c \"24,25-Dihydrolanosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05109_c \"24,25-Dihydrolanosterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05688_c \"L-Selenocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05688_c \"L-Selenocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01528_c \"Selenide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01528_c \"Selenide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05684_c \"Selenite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05684_c \"Selenite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01455_c \"Toluene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01455_c \"Toluene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01467_c \"3-Cresol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01467_c \"3-Cresol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05714_c \"Alpha-Aminopropiononitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05714_c \"Alpha-Aminopropiononitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01401_c \"Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01401_c \"Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01345_c \"2'-Deoxyinosine 5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01345_c \"2'-Deoxyinosine 5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06196_c \"2'-Deoxyinosine 5'-phosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06196_c \"2'-Deoxyinosine 5'-phosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01344_c \"2'-Deoxyinosine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01344_c \"2'-Deoxyinosine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04302_c \"N-(5-Phospho-D-ribosyl)anthranilate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04302_c \"N-(5-Phospho-D-ribosyl)anthranilate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01302_c \"1-(2-Carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01302_c \"1-(2-Carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03506_c \"Indole-3-glycerol phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03506_c \"Indole-3-glycerol phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01300_c \"2-Amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01300_c \"2-Amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00266_c \"Hydroxyacetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00266_c \"Hydroxyacetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04807_c \"7,8-Dihydropterin pyrophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04807_c \"7,8-Dihydropterin pyrophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01279_c \"4-Amino-2-methyl-5-pyrimidinemethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01279_c \"4-Amino-2-methyl-5-pyrimidinemethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01274_c \"5-Formyl-5,6,7,8-tetrahydromethanopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01274_c \"5-Formyl-5,6,7,8-tetrahydromethanopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00074_c \"Phosphoenolpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00074_c \"Phosphoenolpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03175_c \"Shikimate 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03175_c \"Shikimate 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01269_c \"O5-(1-Carboxyvinyl)-3-phosphoshikimate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01269_c \"O5-(1-Carboxyvinyl)-3-phosphoshikimate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01304_c \"2,5-Diamino-6-(1-D-ribosylamino)pyrimidin-4(3H)-one 5'-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01304_c \"2,5-Diamino-6-(1-D-ribosylamino)pyrimidin-4(3H)-one 5'-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01268_c \"5-Amino-6-(5-phosphoribosylamino)uracil\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01268_c \"5-Amino-6-(5-phosphoribosylamino)uracil\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01267_c \"Imidazole-acetol phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01267_c \"Imidazole-acetol phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04133_c \"N-Acetyl-L-glutamyl 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04133_c \"N-Acetyl-L-glutamyl 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01250_c \"2-Acetamido-5-oxopentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01250_c \"2-Acetamido-5-oxopentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01241_c \"Phosphatidyl-N-methylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01241_c \"Phosphatidyl-N-methylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04308_c \"Phosphatidyl-N-dimethylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04308_c \"Phosphatidyl-N-dimethylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04494_c \"Guanosine 5'-triphosphate,3'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04494_c \"Guanosine 5'-triphosphate,3'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01228_c \"Guanosine 5'-diphosphate,3'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01228_c \"Guanosine 5'-diphosphate,3'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01187_c \"3-Deoxyoctulosonic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01187_c \"3-Deoxyoctulosonic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04478_c \"2-Dehydro-3-deoxy-D-octonate 8-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04478_c \"2-Dehydro-3-deoxy-D-octonate 8-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05841_c \"Beta-D-Ribosylnicotinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05841_c \"Beta-D-Ribosylnicotinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03287_c \"L-Glutamate 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03287_c \"L-Glutamate 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04281_c \"3-Hydroxy-L-1-pyrroline-5-carboxylate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04281_c \"3-Hydroxy-L-1-pyrroline-5-carboxylate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01157_c \"trans-4-Hydroxy-L-proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01157_c \"trans-4-Hydroxy-L-proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01134_c \"D-Pantetheine 4'-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01134_c \"D-Pantetheine 4'-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01112_c \"D-Arabinose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01112_c \"D-Arabinose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01100_c \"L-Histidinol phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01100_c \"L-Histidinol phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00081_c \"Inosine tripolyphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00081_c \"Inosine tripolyphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01097_c \"D-Tagatose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01097_c \"D-Tagatose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03785_c \"D-Tagatose 1,6-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03785_c \"D-Tagatose 1,6-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00104_c \"Inosine diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00104_c \"Inosine diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00112_c \"Cytidine diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00112_c \"Cytidine diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04425_c \"S-Adenosyl-4-methylthio-2-oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04425_c \"S-Adenosyl-4-methylthio-2-oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01037_c \"7,8-Diaminopelargonic acid;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01037_c \"7,8-Diaminopelargonic acid;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04327_c \"4-Methyl-5-(2-phosphoethyl)thiazole\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04327_c \"4-Methyl-5-(2-phosphoethyl)thiazole\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01063_c \"Pimeloyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01063_c \"Pimeloyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01054_c \"(S)-2,3-Epoxysqualene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01054_c \"(S)-2,3-Epoxysqualene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01051_c \"Uroporphyrinogen III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01051_c \"Uroporphyrinogen III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01212_c \"UDP-N-acetyl-alpha-D-muramoyl-L-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01212_c \"UDP-N-acetyl-alpha-D-muramoyl-L-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04631_c \"UDP-N-acetylglucosamine enolpyruvate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04631_c \"UDP-N-acetylglucosamine enolpyruvate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01909_c \"Desthiobiotin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01909_c \"Desthiobiotin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01024_c \"Hydroxymethylbilane\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01024_c \"Hydroxymethylbilane\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01182_c \"D-Ribulose 1,5-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01182_c \"D-Ribulose 1,5-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00988_c \"Phosphoglycolic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00988_c \"Phosphoglycolic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00197_c \"3-Phosphoglycerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00197_c \"3-Phosphoglycerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00979_c \"O3-Acetyl-L-serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00979_c \"O3-Acetyl-L-serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05824_c \"S-Sulfo-L-cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05824_c \"S-Sulfo-L-cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00094_c \"Sulfurous acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00094_c \"Sulfurous acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00957_c \"3-Mercaptopyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00957_c \"3-Mercaptopyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02938_c \"3-Indoleacetonitrile;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02938_c \"3-Indoleacetonitrile;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00954_c \"Indoleacetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00954_c \"Indoleacetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00944_c \"Porphobilinogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00944_c \"Porphobilinogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02637_c \"3-Dehydroshikimate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02637_c \"3-Dehydroshikimate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04691_c \"3-Deoxy-D-arabino-heptulosonic acid 7-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04691_c \"3-Deoxy-D-arabino-heptulosonic acid 7-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00921_c \"7,8-Dihydropteroate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00921_c \"7,8-Dihydropteroate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00900_c \"2-Acetolactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00900_c \"2-Acetolactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04039_c \"2,3-Dihydroxy-isovaleric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04039_c \"2,3-Dihydroxy-isovaleric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00882_c \"3'-Dephospho-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00882_c \"3'-Dephospho-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03360_c \"4-Nitrophenyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03360_c \"4-Nitrophenyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00870_c \"4-Hydroxynitrobenzene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00870_c \"4-Hydroxynitrobenzene\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00864_c \"Pantothenate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00864_c \"Pantothenate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00860_c \"L-Histidinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00860_c \"L-Histidinol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01929_c \"Histidinal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01929_c \"Histidinal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00857_c \"Deamido-NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00857_c \"Deamido-NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02823_c \"Cyanocobalamin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02823_c \"Cyanocobalamin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00177_c \"Cyanide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00177_c \"Cyanide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00853_c \"Cob(I)alamin;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00853_c \"Cob(I)alamin;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C08325_c \"Amygdalin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C08325_c \"Amygdalin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00031_c \"D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00031_c \"D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00844_c \"Prunasin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00844_c \"Prunasin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00831_c \"Pantetheine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00831_c \"Pantetheine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05819_c \"Reduced vitamin K2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05819_c \"Reduced vitamin K2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00810_c \"(R)-3-Hydroxybutan-2-one\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00810_c \"(R)-3-Hydroxybutan-2-one\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00791_c \"Creatinine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00791_c \"Creatinine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02565_c \"N-Methylimidazolidine-2,4-dione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02565_c \"N-Methylimidazolidine-2,4-dione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03428_c \"Presqualene diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03428_c \"Presqualene diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00750_c \"Spermine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00750_c \"Spermine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C14818_c \"Ferrous ion;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C14818_c \"Ferrous ion;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00748_c \"Siroheme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00748_c \"Siroheme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00692_c \"UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00692_c \"UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00680_c \"Meso-Diaminoheptanedioate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00680_c \"Meso-Diaminoheptanedioate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00217_c \"D-Glutamic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00217_c \"D-Glutamic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03319_c \"DTDP-beta-L-rhamnose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03319_c \"DTDP-beta-L-rhamnose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00668_c \"alpha-D-Glucose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00668_c \"alpha-D-Glucose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00666_c \"LL-2,6-Diaminopimelic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00666_c \"LL-2,6-Diaminopimelic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00042_c \"Succinic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00042_c \"Succinic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00078_c \"Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00078_c \"Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00700_c \"Xanthosine 5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00700_c \"Xanthosine 5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00655_c \"(9-D-Ribosylxanthine)-5'-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00655_c \"(9-D-Ribosylxanthine)-5'-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01762_c \"Xanthosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01762_c \"Xanthosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00624_c \"N-Acetyl-L-glutamic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00624_c \"N-Acetyl-L-glutamic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00606_c \"3-Sulfino-L-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00606_c \"3-Sulfino-L-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05527_c \"3-Sulfinylpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05527_c \"3-Sulfinylpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01094_c \"D-Fructose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01094_c \"D-Fructose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00577_c \"D-Glyceraldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00577_c \"D-Glyceraldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00561_c \"Mandelonitrile\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00561_c \"Mandelonitrile\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00556_c \"Benzylalcohol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00556_c \"Benzylalcohol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C07086_c \"Benzeneacetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C07086_c \"Benzeneacetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00582_c \"Phenylacetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00582_c \"Phenylacetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03451_c \"S-D-Lactoylglutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03451_c \"S-D-Lactoylglutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00546_c \"2-Oxopropanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00546_c \"2-Oxopropanal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00526_c \"2'-Deoxyuridine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00526_c \"2'-Deoxyuridine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00106_c \"Uracil\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00106_c \"Uracil\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00672_c \"2-Deoxy-alpha-D-ribose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00672_c \"2-Deoxy-alpha-D-ribose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00522_c \"Pantoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00522_c \"Pantoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00099_c \"Beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00099_c \"Beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00966_c \"2-Dehydropantoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00966_c \"2-Dehydropantoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00506_c \"3-Sulfoalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00506_c \"3-Sulfoalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00182_c \"Glycogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00182_c \"Glycogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00493_c \"Shikimic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00493_c \"Shikimic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00491_c \"L-Cystine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00491_c \"L-Cystine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01962_c \"S-Mercapto-L-cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01962_c \"S-Mercapto-L-cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00490_c \"Itaconate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00490_c \"Itaconate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00531_c \"Itaconyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00531_c \"Itaconyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03273_c \"5-Oxopentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03273_c \"5-Oxopentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00489_c \"Glutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00489_c \"Glutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00463_c \"Indole\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00463_c \"Indole\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01346_c \"2'-Deoxyuridine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01346_c \"2'-Deoxyuridine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00460_c \"2'-Deoxyuridine 5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00460_c \"2'-Deoxyuridine 5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00459_c \"Deoxythymidine 5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00459_c \"Deoxythymidine 5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00705_c \"2'-Deoxycytidine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00705_c \"2'-Deoxycytidine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00458_c \"2'-Deoxycytidine 5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00458_c \"2'-Deoxycytidine 5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00455_c \"beta-Nicotinamide mononucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00455_c \"beta-Nicotinamide mononucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03150_c \"N-Ribosylnicotinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03150_c \"N-Ribosylnicotinamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03479_c \"Folinic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03479_c \"Folinic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03082_c \"4-Phospho-L-aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03082_c \"4-Phospho-L-aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00437_c \"N2-Acetyl-L-ornithine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00437_c \"N2-Acetyl-L-ornithine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00077_c \"L-Ornithine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00077_c \"L-Ornithine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03741_c \"L-Glutamate 1-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03741_c \"L-Glutamate 1-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00430_c \"5-Amino-4-oxovaleric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00430_c \"5-Amino-4-oxovaleric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00040_c \"Acyl coenzyme A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00040_c \"Acyl coenzyme A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00416_c \"Phosphatidic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00416_c \"Phosphatidic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00641_c \"L-1,2-Diacylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00641_c \"L-1,2-Diacylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00415_c \"Dihydrofolic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00415_c \"Dihydrofolic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00504_c \"Folic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00504_c \"Folic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00407_c \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00407_c \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15602_c \"Quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15602_c \"Quinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C15603_c \"Reduced quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C15603_c \"Reduced quinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00390_c \"Ubiquinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00390_c \"Ubiquinol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00385_c \"Xanthine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00385_c \"Xanthine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00718_c \"1,4-alpha-D-Glucan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00718_c \"1,4-alpha-D-Glucan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00365_c \"Deoxyuridylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00365_c \"Deoxyuridylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00364_c \"Thymidylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00364_c \"Thymidylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00363_c \"Deoxythymidine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00363_c \"Deoxythymidine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00362_c \"Deoxyguanosine monophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00362_c \"Deoxyguanosine monophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00361_c \"2'-Deoxyguanosine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00361_c \"2'-Deoxyguanosine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00360_c \"Deoxyadenosine monophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00360_c \"Deoxyadenosine monophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00559_c \"2'-Deoxyadenosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00559_c \"2'-Deoxyadenosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00352_c \"D-Glucosamine phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00352_c \"D-Glucosamine phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00350_c \"L-1-Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00350_c \"L-1-Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02737_c \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02737_c \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00344_c \"Phosphatidylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00344_c \"Phosphatidylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00269_c \"CDP-diacylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00269_c \"CDP-diacylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05980_c \"Diphosphatidylglycerol;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05980_c \"Diphosphatidylglycerol;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03892_c \"Phosphatidylglycerophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03892_c \"Phosphatidylglycerophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00342_c \"Reduced thioredoxin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00342_c \"Reduced thioredoxin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00343_c \"Thioredoxin sulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00343_c \"Thioredoxin sulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00053_c \"3'-Phosphoadenylyl sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00053_c \"3'-Phosphoadenylyl sulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00054_c \"Adenosine 3',5'-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00054_c \"Adenosine 3',5'-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00206_c \"2'-Deoxyadenosine 5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00206_c \"2'-Deoxyadenosine 5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00341_c \"Geranyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00341_c \"Geranyl diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00337_c \"Dihydro-L-orotic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00337_c \"Dihydro-L-orotic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00438_c \"N-Carbamoyl-L-aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00438_c \"N-Carbamoyl-L-aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00331_c \"Indolepyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00331_c \"Indolepyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00637_c \"Indoleacetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00637_c \"Indoleacetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00330_c \"Deoxyguanosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00330_c \"Deoxyguanosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03406_c \"L-Arginosuccinic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03406_c \"L-Arginosuccinic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00322_c \"2-Oxoadipic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00322_c \"2-Oxoadipic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00527_c \"Glutaryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00527_c \"Glutaryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01755_c \"Thiocyanic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01755_c \"Thiocyanic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00134_c \"Butane-1,4-diamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00134_c \"Butane-1,4-diamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00311_c \"Isocitric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00311_c \"Isocitric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00417_c \"cis-Aconitic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00417_c \"cis-Aconitic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05715_c \"4-Amino-4-cyanobutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05715_c \"4-Amino-4-cyanobutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00302_c \"Glutaminic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00302_c \"Glutaminic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01103_c \"Orotidine 5'-phosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01103_c \"Orotidine 5'-phosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00295_c \"Uracil-6-carboxylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00295_c \"Uracil-6-carboxylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00286_c \"Deoxyguanosine triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00286_c \"Deoxyguanosine triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00231_c \"D-Xylulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00231_c \"D-Xylulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00275_c \"D-Mannose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00275_c \"D-Mannose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00636_c \"alpha-D-Mannose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00636_c \"alpha-D-Mannose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00267_c \"Alpha-D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00267_c \"Alpha-D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00263_c \"L-Homoserine;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00263_c \"L-Homoserine;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01118_c \"O-Succinylhomoserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01118_c \"O-Succinylhomoserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01102_c \"O-Phospho-L-homoserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01102_c \"O-Phospho-L-homoserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00256_c \"D-Lactic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00256_c \"D-Lactic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00254_c \"Prephenic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00254_c \"Prephenic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01179_c \"p-Hydroxyphenylpyruvic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01179_c \"p-Hydroxyphenylpyruvic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00251_c \"Chorismate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00251_c \"Chorismate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00579_c \"Dihydrothioctamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00579_c \"Dihydrothioctamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00248_c \"Thioctic acid amide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00248_c \"Thioctic acid amide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00239_c \"2'-Deoxycytidine 5'-monophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00239_c \"2'-Deoxycytidine 5'-monophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00881_c \"Deoxycytidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00881_c \"Deoxycytidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05928_c \"(6R)-10-Formyltetrahydropteroyldiglutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05928_c \"(6R)-10-Formyltetrahydropteroyldiglutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00233_c \"2-Oxoisocaproate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00233_c \"2-Oxoisocaproate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00117_c \"Ribose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00117_c \"Ribose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00083_c \"Malonyl coenzyme A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00083_c \"Malonyl coenzyme A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00227_c \"Acetyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00227_c \"Acetyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02265_c \"D-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02265_c \"D-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00166_c \"2-Oxo-3-phenylpropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00166_c \"2-Oxo-3-phenylpropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00214_c \"Thymidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00214_c \"Thymidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00178_c \"Thymine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00178_c \"Thymine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00631_c \"2-Phospho-(R)-glycerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00631_c \"2-Phospho-(R)-glycerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00258_c \"Glyceric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00258_c \"Glyceric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00236_c \"D-Glycerate 1,3-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00236_c \"D-Glycerate 1,3-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01233_c \"Glycerophosphoethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01233_c \"Glycerophosphoethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00189_c \"2-Hydroxyethylamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00189_c \"2-Hydroxyethylamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00188_c \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00188_c \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00187_c \"Cholesterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00187_c \"Cholesterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01802_c \"Desmosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01802_c \"Desmosterol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06206_c \"Benzoyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06206_c \"Benzoyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02376_c \"5-Methylcytosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02376_c \"5-Methylcytosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00147_c \"6-Aminopurine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00147_c \"6-Aminopurine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03089_c \"S-Methyl-5-thio-D-ribose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03089_c \"S-Methyl-5-thio-D-ribose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00169_c \"Carbamoyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00169_c \"Carbamoyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00167_c \"UDP-alpha-D-glucuronate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00167_c \"UDP-alpha-D-glucuronate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00617_c \"UDPgalacturonate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00617_c \"UDPgalacturonate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00190_c \"UDP-alpha-D-xylose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00190_c \"UDP-alpha-D-xylose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00163_c \"Propionic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00163_c \"Propionic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05983_c \"Propionyl-adenosine monophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05983_c \"Propionyl-adenosine monophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02876_c \"Propionyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02876_c \"Propionyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00160_c \"Hydroxyacetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00160_c \"Hydroxyacetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00158_c \"2-Hydroxytricarballylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00158_c \"2-Hydroxytricarballylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00157_c \"Phosphatidylcholine (is normally +1 charged but at ph 7 loses H+)\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00157_c \"Phosphatidylcholine (is normally +1 charged but at ph 7 loses H+)\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00283_c \"Hydrogen sulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00283_c \"Hydrogen sulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02291_c \"Cystathionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02291_c \"Cystathionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00154_c \"Palmitoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00154_c \"Palmitoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00153_c \"Nicotinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00153_c \"Nicotinamide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00149_c \"Malic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00149_c \"Malic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04348_c \"(3S)-3-Carboxy-3-hydroxypropionyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04348_c \"(3S)-3-Carboxy-3-hydroxypropionyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00148_c \"2-Pyrrolidinecarboxylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00148_c \"2-Pyrrolidinecarboxylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00242_c \"Guanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00242_c \"Guanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00387_c \"Guanosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00387_c \"Guanosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00183_c \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00183_c \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C04006_c \"Inositol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C04006_c \"Inositol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00137_c \"Inositol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00137_c \"Inositol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03546_c \"myo-Inositol4-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03546_c \"myo-Inositol4-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01177_c \"Inositol1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01177_c \"Inositol1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00135_c \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00135_c \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00388_c \"Histamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00388_c \"Histamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00436_c \"N-Carbamoylputrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00436_c \"N-Carbamoylputrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00133_c \"D-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00133_c \"D-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18233_p \"C18233[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C18234_p \"C18234[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01438_c \"Methane\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01438_c \"Methane\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00131_c \"dATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00131_c \"dATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00130_c \"5'-IMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00130_c \"5'-IMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03794_c \"Adenylosuccinic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03794_c \"Adenylosuccinic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00294_c \"Inosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00294_c \"Inosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00244_c \"Nitric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00244_c \"Nitric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00123_c \"(2S)-alpha-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00123_c \"(2S)-alpha-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00062_c \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00062_c \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02514_c \"3-Fumarylpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02514_c \"3-Fumarylpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C17023_c \"Sulfurdonor\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C17023_c \"Sulfurdonor\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05359_c \"Electron\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05359_c \"Electron\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00120_c \"Biotin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00120_c \"Biotin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05921_c \"Biotinyl-5'-AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05921_c \"Biotinyl-5'-AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00108_c \"2-Aminobenzoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00108_c \"2-Aminobenzoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00354_c \"D-Fructose 1,6-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00354_c \"D-Fructose 1,6-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00620_c \"D-Ribose 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00620_c \"D-Ribose 1-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00301_c \"Adenosine diphosphate ribose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00301_c \"Adenosine diphosphate ribose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00670_c \"Glycerophosphocholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00670_c \"Glycerophosphocholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00114_c \"Bilineurine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00114_c \"Bilineurine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00184_c \"Glycerone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00184_c \"Glycerone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00380_c \"Cytosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00380_c \"Cytosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00299_c \"Uridine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00299_c \"Uridine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C09332_c \"Tetrahydropteroyldiglutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C09332_c \"Tetrahydropteroyldiglutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00048_c \"Glyoxylic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00048_c \"Glyoxylic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00100_c \"Propionyl coenzyme A\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00100_c \"Propionyl coenzyme A\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06027_c \"(2R,3S)-2-Methyl-3-carboxy-3-hydroxypropanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C06027_c \"(2R,3S)-2-Methyl-3-carboxy-3-hydroxypropanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01419_c \"L-Cysteinylglycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01419_c \"L-Cysteinylglycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00669_c \"Gamma-Glutamylcysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00669_c \"Gamma-Glutamylcysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00096_c \"GDP-alpha-D-mannose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00096_c \"GDP-alpha-D-mannose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00029_c \"Uridine diphosphate glucose;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00029_c \"Uridine diphosphate glucose;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02336_c \"Fructose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02336_c \"Fructose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00082_c \"Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00082_c \"Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00232_c \"4-Oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00232_c \"4-Oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00079_c \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00079_c \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00826_c \"Pretyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00826_c \"Pretyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00068_c \"Thiamin diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00068_c \"Thiamin diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05381_c \"3-Carboxy-1-hydroxypropyl-ThPP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05381_c \"3-Carboxy-1-hydroxypropyl-ThPP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00168_c \"3-Hydroxypyruvic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00168_c \"3-Hydroxypyruvic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00152_c \"2-Aminosuccinamic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00152_c \"2-Aminosuccinamic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00179_c \"Agmatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00179_c \"Agmatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00255_c \"Riboflavin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00255_c \"Riboflavin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00224_c \"5'-Adenylyl sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00224_c \"5'-Adenylyl sulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00475_c \"Cytidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00475_c \"Cytidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01127_c \"4-Hydroxy-2-oxoglutaric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01127_c \"4-Hydroxy-2-oxoglutaric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00047_c \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00047_c \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00203_c \"UDP-N-acetyl-alpha-D-galactosamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00203_c \"UDP-N-acetyl-alpha-D-galactosamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05993_c \"Acetyladenylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05993_c \"Acetyladenylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C02191_c \"Porphyrinogen IX\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C02191_c \"Porphyrinogen IX\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00533_c \"Nitrogen monoxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00533_c \"Nitrogen monoxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00887_c \"Nitrous oxide;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00887_c \"Nitrous oxide;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00052_c \"UDP-galactose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00052_c \"UDP-galactose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01251_c \"Homocitric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01251_c \"Homocitric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C03539_c \"S-Ribosyl-L-homocysteine;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C03539_c \"S-Ribosyl-L-homocysteine;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00212_c \"Adenosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00212_c \"Adenosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00016_c \"Flavin adenine dinucleotide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00016_c \"Flavin adenine dinucleotide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11038_c \"2'-Deoxy-5-hydroxymethylcytidine-5'-diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11038_c \"2'-Deoxy-5-hydroxymethylcytidine-5'-diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C11039_c \"2'-Deoxy-5-hydroxymethylcytidine-5'-triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C11039_c \"2'-Deoxy-5-hydroxymethylcytidine-5'-triphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01353_c \"Carbonicacid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01353_c \"Carbonicacid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01260_c \"P1,P4-Bis(5'-adenosyl)tetraphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01260_c \"P1,P4-Bis(5'-adenosyl)tetraphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00931_c \"Porphobilinogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00931_c \"Porphobilinogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01000_c \"Ferrocytochromec2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00997_c \"Ferricytochromec2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C14819_c \"Ferric ion;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C14819_c \"Ferric ion;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00208_c \"Maltose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00208_c \"Maltose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00185_c \"1-beta-D-Glucopyranosyl-4-D-glucopyranose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00185_c \"1-beta-D-Glucopyranosyl-4-D-glucopyranose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01674_c \"N,N'-Diacetylchitobiose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C01674_c \"N,N'-Diacetylchitobiose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C05125_c \"2-(alpha-Hydroxyethyl)thiamine diphosphate;\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C05125_c \"2-(alpha-Hydroxyethyl)thiamine diphosphate;\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00080_p \"C00080[p]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00305_c \"Magnesiumcation\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00305_c \"Magnesiumcation\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00076_c \"Calciumcation\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00076_c \"Calciumcation\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00070_c \"Copper\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00070_c \"Copper\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00023_c \"Iron\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00023_c \"Iron\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01203_c \"C01203[c]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00712_c \"(9Z)-Octadecenoicacid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00712_c \"(9Z)-Octadecenoicacid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00940_c \"2-Oxoglutaramate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00940_c \"2-Oxoglutaramate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00022_e \"C00022[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00009_e \"C00009[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00007_e \"C00007[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00088_e \"C00088[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00244_e \"C00244[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00014_e \"C00014[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00697_e \"C00697[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00256_e \"C00256[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00032_e \"C00032[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00001_e \"C00001[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00058_e \"C00058[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00469_e \"C00469[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00011_e \"C00011[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C01438_e \"C01438[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00305_e \"C00305[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00175_e \"C00175[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00076_e \"C00076[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00070_e \"C00070[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00023_e \"C00023[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00033_e \"C00033[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00059_e \"C00059[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00042_e \"C00042[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CCL160161_e \"CCL160161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CCL161140_e \"CCL161140[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CCL161161_e \"CCL161161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPC160161_e \"CPC160161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPC161140_e \"CPC161140[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPC161161_e \"CPC161161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPE160161_e \"CPE160161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPE161140_e \"CPE161140[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPE161161_e \"CPE161161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPG160161_e \"CPG160161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPG161140_e \"CPG161140[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_CPG161161_e \"CPG161161[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06706_e \"C06706[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06706_c \"Lipo-oligosaccharide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C06025_e \"C06025[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00134_e \"C00134[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00315_e \"C00315[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00086_e \"C00086[e]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00086_c \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00086_c \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00058_p \"Formic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00058_p \"Formic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00022_p \"pyruvic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00022_p \"pyruvic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00009_p \"Phosphoric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00009_p \"Phosphoric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00088_p \"Nitrite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00088_p \"Nitrite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00244_p \"Nitric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00244_p \"Nitric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00014_p \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00014_p \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00697_p \"Nitrogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00697_p \"Nitrogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00256_p \"D-Lactic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00256_p \"D-Lactic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00032_p \"Heme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00032_p \"Heme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00001_p \"water\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00001_p \"water\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00469_p \"Ethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00469_p \"Ethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00011_p \"Carbon dioxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00011_p \"Carbon dioxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00305_p \"Magnesiumcation\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00305_p \"Magnesiumcation\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00175_p \"Cobalt ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00175_p \"Cobalt ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00076_p \"Calciumcation\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00076_p \"Calciumcation\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00070_p \"Copper\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00070_p \"Copper\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00023_p \"Iron\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00023_p \"Iron\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00033_p \"Acetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00033_p \"Acetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00059_p \"Sulfuric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00059_p \"Sulfuric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00042_p \"Succinic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00042_p \"Succinic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00041_e \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00041_e \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00152_e \"2-Aminosuccinamic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00152_e \"2-Aminosuccinamic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00049_e \"L-Aspartic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00049_e \"L-Aspartic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00097_e \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00097_e \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00025_e \"L-Glutamic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00025_e \"L-Glutamic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00064_e \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00064_e \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00183_e \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00183_e \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00957_e \"3-Mercaptopyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00957_e \"3-Mercaptopyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00141_e \"2-Keto-3-methylbutyric acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00141_e \"2-Keto-3-methylbutyric acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00036_e \"Oxalacetic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00036_e \"Oxalacetic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00940_e \"2-Oxoglutaramate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00940_e \"2-Oxoglutaramate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00132_e \"Methanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00132_e \"Methanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00067_e \"Formaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00067_e \"Formaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_C00067_p \"Formaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_C00067_p \"Formaldehyde\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Starch_synthesis \"Starch Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Starch_degredation \"Starch Degredation Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Rspont2 \"Spontanous Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPG161161 \"Phosphatidylglycerol Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPG161140 \"Phosphatidylglycerol Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPG160161 \"Phosphatidylglycerol Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPE161161 \"Phosphatidylethanol Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPE161140 \"Phosphatidylethanol Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPE160161 \"Phosphatidylethanol Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPC161161 \"PhosphatidylCholine Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPC161140 \"PhosphatidylCholine Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPC160161 \"PhosphatidylCholine Synthesis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RCL161161 \"Cardiolipin Syntheis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RCL161140 \"Cardiolipin Syntheis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RCL160161 \"Cardiolipin Syntheis Reaction\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RC08362 \"Fatty Acid Synthesis Reation\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R11096>\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R11096>\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R11025 \"UTP:N-acetyl-alpha-D-muramate-1-phosphate uridylyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R11025 \"UTP:N-acetyl-alpha-D-muramate-1-phosphate uridylyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R11024 \"ATP:N-acetylmuramate 1-phosphotransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R11024 \"ATP:N-acetylmuramate 1-phosphotransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10994 \"gamma-L-glutamyl-L-2-aminobutyrate:glycine ligase (ADP-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10994 \"gamma-L-glutamyl-L-2-aminobutyrate:glycine ligase (ADP-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10993 \"L-glutamate:2-aminobutanoate gamma-ligase (ADP-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10993 \"L-glutamate:2-aminobutanoate gamma-ligase (ADP-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10991 \"2-aminobutanoate:2-oxoglutarate aminotransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10991 \"2-aminobutanoate:2-oxoglutarate aminotransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10907 \"beta-D-glucose-6-phosphate:NAD+ 1-oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10907 \"beta-D-glucose-6-phosphate:NAD+ 1-oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10866 \"pyruvate:flavodoxin 2-oxidoreductase (CoA-acetylating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10866 \"pyruvate:flavodoxin 2-oxidoreductase (CoA-acetylating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10859 \"(E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized flavodoxin oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10859 \"(E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized flavodoxin oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10712 \"2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:2-(2-carboxy-4-methylthiazol-5-yl)ethyl phosphate 2-methyl-4-aminopyrimidine-5-methenyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10712 \"2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:2-(2-carboxy-4-methylthiazol-5-yl)ethyl phosphate 2-methyl-4-aminopyrimidine-5-methenyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10707 \"acetyl-CoA:malonyl-[acyl-carrier protein] C-acetyltransferase (decarboxylating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10707 \"acetyl-CoA:malonyl-[acyl-carrier protein] C-acetyltransferase (decarboxylating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10221 \"6-phospho-D-gluconate:NAD+ 2-oxidoreductase (decarboxylating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10221 \"6-phospho-D-gluconate:NAD+ 2-oxidoreductase (decarboxylating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10164 \"hydroxylamine:ferricytochrome-c oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10164 \"hydroxylamine:ferricytochrome-c oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10151>\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10151>\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10147 \"L-aspartate-4-semialdehyde hydro-lyase [adding pyruvate and cyclizing; (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate-forming]\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10147 \"L-aspartate-4-semialdehyde hydro-lyase [adding pyruvate and cyclizing; (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate-forming]\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10124 \"pimeloyl-[acyl-carrier protein]:L-alanine C-pimeloyltransferase (decarboxylating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10124 \"pimeloyl-[acyl-carrier protein]:L-alanine C-pimeloyltransferase (decarboxylating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10122 \"pimelyl-[acyl-carrier protein]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10122 \"pimelyl-[acyl-carrier protein]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10121 \"3-hydroxypimeloyl-[acp]-methyl-ester hydro-lyase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10121 \"3-hydroxypimeloyl-[acp]-methyl-ester hydro-lyase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10120 \"3-hydroxypimeloyl-[acp]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10120 \"3-hydroxypimeloyl-[acp]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10119 \"glutaryl-[acp]-methyl-ester:malonyl-[acyl-carrier protein] C-acyltransferase (decarboxylating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10119 \"glutaryl-[acp]-methyl-ester:malonyl-[acyl-carrier protein] C-acyltransferase (decarboxylating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10118 \"glutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10118 \"glutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10117 \"3-hydroxyglutaryl-[acp]-methyl-ester hydro-lyase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10117 \"3-hydroxyglutaryl-[acp]-methyl-ester hydro-lyase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10116 \"3-hydroxyglutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10116 \"3-hydroxyglutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10115 \"malonyl-[acyl-carrier protein]-methyl-ester:malonyl-[acyl-carrier protein]  C-acyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10115 \"malonyl-[acyl-carrier protein]-methyl-ester:malonyl-[acyl-carrier protein]  C-acyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10092 \"carbonate hydro-lyase (carbon-dioxide-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10092 \"carbonate hydro-lyase (carbon-dioxide-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10040 \"beta-D-glucoside glucohydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10040 \"beta-D-glucoside glucohydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10039 \"beta-D-glucoside glucohydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10039 \"beta-D-glucoside glucohydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10035 \"beta-D-glucoside glucohydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10035 \"beta-D-glucoside glucohydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R10002 \"6-carboxy-5,6,7,8-tetrahydropterin ammonia-lyase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R10002 \"6-carboxy-5,6,7,8-tetrahydropterin ammonia-lyase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09978 \"7-carboxy-7-carbaguanine:ammonia ligase (ADP-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09978 \"7-carboxy-7-carbaguanine:ammonia ligase (ADP-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09959 \"7,8-dihydroneopterin 3'-triphosphate acetaldehyde-lyase (6-carboxy-5,6,7,8-tetrahydropterin and triphosphate-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09959 \"7,8-dihydroneopterin 3'-triphosphate acetaldehyde-lyase (6-carboxy-5,6,7,8-tetrahydropterin and triphosphate-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09780 \"D-arabino-hex-3-ulose-6-phosphate isomerase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09780 \"D-arabino-hex-3-ulose-6-phosphate isomerase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09771 \"D-glycero-alpha-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09771 \"D-glycero-alpha-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09769 \"D-glycero-alpha-D-manno-heptose 7-phosphate aldose-ketose-isomerase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09769 \"D-glycero-alpha-D-manno-heptose 7-phosphate aldose-ketose-isomerase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09725 \"pimeloyl-[acyl-carrier protein]-methyl-ester hydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09725 \"pimeloyl-[acyl-carrier protein]-methyl-ester hydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09668 \"S-methyl-5'-thioinosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09668 \"S-methyl-5'-thioinosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09660 \"S-methyl-5'-thioadenosine amidohydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09660 \"S-methyl-5'-thioadenosine amidohydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09543 \"S-adenosyl-L-methionine:malonyl-[acyl-carrier protein] O-methyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09543 \"S-adenosyl-L-methionine:malonyl-[acyl-carrier protein] O-methyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09518 \"methane,quinol:oxygen oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09518 \"methane,quinol:oxygen oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09394 \"GTP 3',8-cyclase [(8S)-3',8-cyclo-7,8-dihydroguanosine 5'-triphosphate-forming]\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09394 \"GTP 3',8-cyclase [(8S)-3',8-cyclo-7,8-dihydroguanosine 5'-triphosphate-forming]\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09380 \"acyl-ACP:sn-glycerol-3-phosphate 1-O-acyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09380 \"acyl-ACP:sn-glycerol-3-phosphate 1-O-acyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09372 \"methaneselenol:NADP+ oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09372 \"methaneselenol:NADP+ oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09365 \"5-methyltetrahydropteroyltri-L-glutamate:L-selenohomocysteine Se-methyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09365 \"5-methyltetrahydropteroyltri-L-glutamate:L-selenohomocysteine Se-methyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09248 \"(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 5 isopentenyl units)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09248 \"(2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 5 isopentenyl units)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09128 \"2-chloroethanol:cytochrome c oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09128 \"2-chloroethanol:cytochrome c oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09127 \"ethanol:cytochrome c oxidoreductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09127 \"ethanol:cytochrome c oxidoreductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09107 \"N-acetyl-L-citrulline amidohydrolase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09107 \"N-acetyl-L-citrulline amidohydrolase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09099 \"5,10-methylenetetrahydromethanopterin:glycine hydroxymethyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09099 \"5,10-methylenetetrahydromethanopterin:glycine hydroxymethyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09082 \"carboxynorspermidine carboxy-lyase (norspermidine-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09082 \"carboxynorspermidine carboxy-lyase (norspermidine-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09081 \"carboxyspermidine carboxy-lyase (spermidine-forming)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09081 \"carboxyspermidine carboxy-lyase (spermidine-forming)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R09030 \"D-allose-6-phosphate aldose-ketose-isomerase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R09030 \"D-allose-6-phosphate aldose-ketose-isomerase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08968 \"ATP:N-acetyl-D-glucosamine 1-phosphotransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08968 \"ATP:N-acetyl-D-glucosamine 1-phosphotransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08781 \"S-adenosyl-L-methionine:3-hydroxy-2-methoxy-5-methyl-6-(all-trans-polyprenyl)-1,4-benzoquinone 3-O-methyltransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08781 \"S-adenosyl-L-methionine:3-hydroxy-2-methoxy-5-methyl-6-(all-trans-polyprenyl)-1,4-benzoquinone 3-O-methyltransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08775>\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08775>\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08774>\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08774>\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08773>\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08773>\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08769 \"S-adenosyl-L-methionine:3-polyprenylcatechol 1-O-methyltransferase;\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08769 \"S-adenosyl-L-methionine:3-polyprenylcatechol 1-O-methyltransferase;\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08716 \"cobalt-precorrin 3 dehydrogenase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08689 \"(E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized ferredoxin oxidoreductase (hydrating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08689 \"(E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized ferredoxin oxidoreductase (hydrating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08676>\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08648 \"pyruvate:2-oxobutanoate acetaldehydetransferase (decarboxylating)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08648 \"pyruvate:2-oxobutanoate acetaldehydetransferase (decarboxylating)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08639 \"alpha-D-glucose 1,6-phosphomutase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08639 \"alpha-D-glucose 1,6-phosphomutase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08575 \"sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08575 \"sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R08549 \"2-Oxoglutarate dehydrogenase complex\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R08549 \"2-Oxoglutarate dehydrogenase complex\">\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[7], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m mcoosmodel \u001b[39m=\u001b[39m cobra\u001b[39m.\u001b[39;49mio\u001b[39m.\u001b[39;49mread_sbml_model(\u001b[39m'\u001b[39;49m\u001b[39mother_model\u001b[39;49m\u001b[39m\\\\\u001b[39;49;00m\u001b[39mnewmethylococcus.xml\u001b[39;49m\u001b[39m'\u001b[39;49m)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\sbml.py:458\u001b[0m, in \u001b[0;36mread_sbml_model\u001b[1;34m(filename, number, f_replace, **kwargs)\u001b[0m\n\u001b[0;32m    456\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m    457\u001b[0m     doc \u001b[39m=\u001b[39m _get_doc_from_filename(filename)\n\u001b[1;32m--> 458\u001b[0m     \u001b[39mreturn\u001b[39;00m _sbml_to_model(doc, number\u001b[39m=\u001b[39mnumber, f_replace\u001b[39m=\u001b[39mf_replace, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mkwargs)\n\u001b[0;32m    459\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mIOError\u001b[39;00m \u001b[39mas\u001b[39;00m e:\n\u001b[0;32m    460\u001b[0m     \u001b[39mraise\u001b[39;00m e\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\sbml.py:956\u001b[0m, in \u001b[0;36m_sbml_to_model\u001b[1;34m(doc, number, f_replace, set_missing_bounds, **kwargs)\u001b[0m\n\u001b[0;32m    954\u001b[0m             \u001b[39mif\u001b[39;00m f_replace \u001b[39mand\u001b[39;00m F_GENE \u001b[39min\u001b[39;00m f_replace:\n\u001b[0;32m    955\u001b[0m                 gpr \u001b[39m=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39m \u001b[39m\u001b[39m\"\u001b[39m\u001b[39m.\u001b[39mjoin(f_replace[F_GENE](t) \u001b[39mfor\u001b[39;00m t \u001b[39min\u001b[39;00m gpr\u001b[39m.\u001b[39msplit(\u001b[39m\"\u001b[39m\u001b[39m \u001b[39m\u001b[39m\"\u001b[39m))\n\u001b[1;32m--> 956\u001b[0m         cobra_reaction\u001b[39m.\u001b[39mgpr \u001b[39m=\u001b[39m GPR\u001b[39m.\u001b[39;49mfrom_string(gpr)\n\u001b[0;32m    958\u001b[0m cobra_model\u001b[39m.\u001b[39madd_reactions(reactions)\n\u001b[0;32m    960\u001b[0m \u001b[39m# Objective\u001b[39;00m\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\core\\gene.py:378\u001b[0m, in \u001b[0;36mGPR.from_string\u001b[1;34m(cls, string_gpr)\u001b[0m\n\u001b[0;32m    373\u001b[0m         warn(\n\u001b[0;32m    374\u001b[0m             \u001b[39m\"\u001b[39m\u001b[39mMalformed gene_reaction_rule \u001b[39m\u001b[39m'\u001b[39m\u001b[39m{}\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m.\u001b[39mformat(escaped_str),\n\u001b[0;32m    375\u001b[0m             \u001b[39mSyntaxWarning\u001b[39;00m,\n\u001b[0;32m    376\u001b[0m         )\n\u001b[0;32m    377\u001b[0m         \u001b[39mreturn\u001b[39;00m gpr\n\u001b[1;32m--> 378\u001b[0m gpr \u001b[39m=\u001b[39m \u001b[39mcls\u001b[39;49m(tree)\n\u001b[0;32m    379\u001b[0m gpr\u001b[39m.\u001b[39mupdate_genes()\n\u001b[0;32m    380\u001b[0m \u001b[39mreturn\u001b[39;00m gpr\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\core\\gene.py:306\u001b[0m, in \u001b[0;36mGPR.__init__\u001b[1;34m(self, gpr_from, **kwargs)\u001b[0m\n\u001b[0;32m    304\u001b[0m     cleaner\u001b[39m.\u001b[39mvisit(gpr_from)\n\u001b[0;32m    305\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_genes \u001b[39m=\u001b[39m deepcopy(cleaner\u001b[39m.\u001b[39mgene_set)\n\u001b[1;32m--> 306\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mbody \u001b[39m=\u001b[39m deepcopy(gpr_from\u001b[39m.\u001b[39;49mbody)\n\u001b[0;32m    307\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39meval()\n\u001b[0;32m    308\u001b[0m \u001b[39melse\u001b[39;00m:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "    \u001b[1;31m[... skipping similar frames: deepcopy at line 146 (1 times)]\u001b[0m\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:205\u001b[0m, in \u001b[0;36m_deepcopy_list\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    203\u001b[0m append \u001b[39m=\u001b[39m y\u001b[39m.\u001b[39mappend\n\u001b[0;32m    204\u001b[0m \u001b[39mfor\u001b[39;00m a \u001b[39min\u001b[39;00m x:\n\u001b[1;32m--> 205\u001b[0m     append(deepcopy(a, memo))\n\u001b[0;32m    206\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "    \u001b[1;31m[... skipping similar frames: _deepcopy_dict at line 230 (7 times), _reconstruct at line 270 (7 times), deepcopy at line 172 (7 times), deepcopy at line 146 (7 times)]\u001b[0m\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "    \u001b[1;31m[... skipping similar frames: deepcopy at line 146 (1 times)]\u001b[0m\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:205\u001b[0m, in \u001b[0;36m_deepcopy_list\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    203\u001b[0m append \u001b[39m=\u001b[39m y\u001b[39m.\u001b[39mappend\n\u001b[0;32m    204\u001b[0m \u001b[39mfor\u001b[39;00m a \u001b[39min\u001b[39;00m x:\n\u001b[1;32m--> 205\u001b[0m     append(deepcopy(a, memo))\n\u001b[0;32m    206\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:205\u001b[0m, in \u001b[0;36m_deepcopy_list\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    203\u001b[0m append \u001b[39m=\u001b[39m y\u001b[39m.\u001b[39mappend\n\u001b[0;32m    204\u001b[0m \u001b[39mfor\u001b[39;00m a \u001b[39min\u001b[39;00m x:\n\u001b[1;32m--> 205\u001b[0m     append(deepcopy(a, memo))\n\u001b[0;32m    206\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    170\u001b[0m                 y \u001b[39m=\u001b[39m x\n\u001b[0;32m    171\u001b[0m             \u001b[39melse\u001b[39;00m:\n\u001b[1;32m--> 172\u001b[0m                 y \u001b[39m=\u001b[39m _reconstruct(x, memo, \u001b[39m*\u001b[39;49mrv)\n\u001b[0;32m    174\u001b[0m \u001b[39m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[0;32m    175\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m x:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[1;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[0;32m    268\u001b[0m \u001b[39mif\u001b[39;00m state \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m    269\u001b[0m     \u001b[39mif\u001b[39;00m deep:\n\u001b[1;32m--> 270\u001b[0m         state \u001b[39m=\u001b[39m deepcopy(state, memo)\n\u001b[0;32m    271\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mhasattr\u001b[39m(y, \u001b[39m'\u001b[39m\u001b[39m__setstate__\u001b[39m\u001b[39m'\u001b[39m):\n\u001b[0;32m    272\u001b[0m         y\u001b[39m.\u001b[39m__setstate__(state)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m--> 146\u001b[0m     y \u001b[39m=\u001b[39m copier(x, memo)\n\u001b[0;32m    147\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    148\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39missubclass\u001b[39m(\u001b[39mcls\u001b[39m, \u001b[39mtype\u001b[39m):\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[1;34m(x, memo, deepcopy)\u001b[0m\n\u001b[0;32m    228\u001b[0m memo[\u001b[39mid\u001b[39m(x)] \u001b[39m=\u001b[39m y\n\u001b[0;32m    229\u001b[0m \u001b[39mfor\u001b[39;00m key, value \u001b[39min\u001b[39;00m x\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[39m=\u001b[39m deepcopy(value, memo)\n\u001b[0;32m    231\u001b[0m \u001b[39mreturn\u001b[39;00m y\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\copy.py:142\u001b[0m, in \u001b[0;36mdeepcopy\u001b[1;34m(x, memo, _nil)\u001b[0m\n\u001b[0;32m    139\u001b[0m \u001b[39mif\u001b[39;00m y \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m _nil:\n\u001b[0;32m    140\u001b[0m     \u001b[39mreturn\u001b[39;00m y\n\u001b[1;32m--> 142\u001b[0m \u001b[39mcls\u001b[39m \u001b[39m=\u001b[39m \u001b[39mtype\u001b[39;49m(x)\n\u001b[0;32m    144\u001b[0m copier \u001b[39m=\u001b[39m _deepcopy_dispatch\u001b[39m.\u001b[39mget(\u001b[39mcls\u001b[39m)\n\u001b[0;32m    145\u001b[0m \u001b[39mif\u001b[39;00m copier \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n",
      "\u001b[1;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "mcoosmodel = cobra.io.read_sbml_model('other_model\\\\newmethylococcus.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.write_sbml_model(mcoosmodel, 'other_model\\\\newmethylococcus_new.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h3>Objective</h3><p>1.0 Biomass_reaction = 0.2311190276729317</p><h4>Uptake</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>C00023_e</td>\n",
       "      <td>EX_C00023</td>\n",
       "      <td>0.001093</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00070_e</td>\n",
       "      <td>EX_C00070</td>\n",
       "      <td>0.001852</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00076_e</td>\n",
       "      <td>EX_C00076</td>\n",
       "      <td>0.001852</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00175_e</td>\n",
       "      <td>EX_C00175</td>\n",
       "      <td>0.0001852</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00305_e</td>\n",
       "      <td>EX_C00305</td>\n",
       "      <td>0.01296</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C01438_e</td>\n",
       "      <td>EX_CH4</td>\n",
       "      <td>18.5</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00032_e</td>\n",
       "      <td>EX_Heme</td>\n",
       "      <td>4.075E-05</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00007_e</td>\n",
       "      <td>EX_O2</td>\n",
       "      <td>23.25</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00009_e</td>\n",
       "      <td>EX_Pi</td>\n",
       "      <td>0.3316</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00088_e</td>\n",
       "      <td>EX_nitrite</td>\n",
       "      <td>2.126</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00059_e</td>\n",
       "      <td>EX_sulfate</td>\n",
       "      <td>0.04193</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table><h4>Secretion</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>C00011_e</td>\n",
       "      <td>EX_CO2</td>\n",
       "      <td>-8.786</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00001_e</td>\n",
       "      <td>EX_H2O</td>\n",
       "      <td>-29.39</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<cobra.summary.model_summary.ModelSummary at 0x2210b80fcd0>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mcoosmodel.summary()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'' is not a valid SBML 'SId'.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Objective</h3><p>1.0 Biomass_reaction = 0.010995735691106626</p><h4>Uptake</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>C00023_e</td>\n",
       "      <td>EX_C00023</td>\n",
       "      <td>5.199E-05</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00070_e</td>\n",
       "      <td>EX_C00070</td>\n",
       "      <td>8.812E-05</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00076_e</td>\n",
       "      <td>EX_C00076</td>\n",
       "      <td>8.812E-05</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00175_e</td>\n",
       "      <td>EX_C00175</td>\n",
       "      <td>8.812E-06</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00305_e</td>\n",
       "      <td>EX_C00305</td>\n",
       "      <td>0.0006168</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C01438_e</td>\n",
       "      <td>EX_CH4</td>\n",
       "      <td>3.327</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00032_e</td>\n",
       "      <td>EX_Heme</td>\n",
       "      <td>1.939E-06</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00007_e</td>\n",
       "      <td>EX_O2</td>\n",
       "      <td>6</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00009_e</td>\n",
       "      <td>EX_Pi</td>\n",
       "      <td>0.01578</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00088_e</td>\n",
       "      <td>EX_nitrite</td>\n",
       "      <td>0.1012</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00059_e</td>\n",
       "      <td>EX_sulfate</td>\n",
       "      <td>0.001995</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table><h4>Secretion</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>C00033_e</td>\n",
       "      <td>EX_Ac</td>\n",
       "      <td>-3.519E-05</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00011_e</td>\n",
       "      <td>EX_CO2</td>\n",
       "      <td>-2.865</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00080_c</td>\n",
       "      <td>EX_H</td>\n",
       "      <td>-0.0749</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00001_e</td>\n",
       "      <td>EX_H2O</td>\n",
       "      <td>-6.292</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<cobra.summary.model_summary.ModelSummary at 0x1d29d2586d0>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model = cobra.io.read_sbml_model('other_model\\\\newmethylococcus_new.xml')\n",
    "model.reactions.EX_CH4.upper_bound = 0\n",
    "model.reactions.EX_CH4.lower_bound = -1000\n",
    "model.reactions.EX_O2.upper_bound = -6\n",
    "model.reactions.EX_O2.lower_bound = -6\n",
    "#model.reactions.EX_CH4.upper_bound = -2.9\n",
    "model.optimize()\n",
    "model.summary()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "solution = model.optimize()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.010995735691106283\n"
     ]
    }
   ],
   "source": [
    "print(solution.objective_value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'Metabolite' object has no attribute 'flux'",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[16], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m model\u001b[39m.\u001b[39;49mmetabolites\u001b[39m.\u001b[39;49mC00033_e\u001b[39m.\u001b[39;49mflux\n",
      "\u001b[1;31mAttributeError\u001b[0m: 'Metabolite' object has no attribute 'flux'"
     ]
    }
   ],
   "source": [
    "model.metabolites.C00033_e.summary()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "EX_Ac , Exchange Reaction\n",
      "\t C00033_e --> \n",
      "\t\t C00033_e \t C00033[e] \t -1.0\n"
     ]
    }
   ],
   "source": [
    "printmyrxn(model, 'EX_Ac')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h3>Objective</h3><p>1.0 EX_Ac = 1.8666666666666714</p><h4>Uptake</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>C01438_e</td>\n",
       "      <td>EX_CH4</td>\n",
       "      <td>4.867</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00007_e</td>\n",
       "      <td>EX_O2</td>\n",
       "      <td>6</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table><h4>Secretion</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>C00033_e</td>\n",
       "      <td>EX_Ac</td>\n",
       "      <td>-1.867</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00011_e</td>\n",
       "      <td>EX_CO2</td>\n",
       "      <td>-1.133</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00080_c</td>\n",
       "      <td>EX_H</td>\n",
       "      <td>-1.867</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>C00001_e</td>\n",
       "      <td>EX_H2O</td>\n",
       "      <td>-6</td>\n",
       "      <td>0</td>\n",
       "      <td>0.00%</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<cobra.summary.model_summary.ModelSummary at 0x221122eae50>"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.optimize()\n",
    "model.summary()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "mcoosmodel_rxnlist = [rxn.id for rxn in mcoosmodel.reactions]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Rspont2 pink\n",
      "RPG161161 pink\n",
      "RPG161140 pink\n",
      "RPG160161 pink\n",
      "RPE161161 pink\n",
      "RPE161140 pink\n",
      "RPE160161 pink\n",
      "RPC161161 pink\n",
      "RPC161140 pink\n",
      "RPC160161 pink\n",
      "RCL161161 pink\n",
      "RCL161140 pink\n",
      "RCL160161 pink\n",
      "RC08362 pink\n",
      "R11096 pink\n",
      "R11025 pink\n",
      "R11024 pink\n",
      "R10994 pink\n",
      "R10993 pink\n",
      "R10991 pink\n",
      "R10907 pink\n",
      "R10866 pink\n",
      "R10859 pink\n",
      "R10712 pink\n",
      "R10707 pink\n",
      "R10221 pink\n",
      "R10164 pink\n",
      "R10151 pink\n",
      "R10147 pink\n",
      "R10124 pink\n",
      "R10122 pink\n",
      "R10121 pink\n",
      "R10120 pink\n",
      "R10119 pink\n",
      "R10118 pink\n",
      "R10117 pink\n",
      "R10116 pink\n",
      "R10115 pink\n",
      "R10092 pink\n",
      "R10040 pink\n",
      "R10039 pink\n",
      "R10035 pink\n",
      "R10002 pink\n",
      "R09978 pink\n",
      "R09959 pink\n",
      "R09780 pink\n",
      "R09771 pink\n",
      "R09769 pink\n",
      "R09725 pink\n",
      "R09668 pink\n",
      "R09660 pink\n",
      "R09543 pink\n",
      "R09518 pink\n",
      "R09394 pink\n",
      "R09380 pink\n",
      "R09372 pink\n",
      "R09365 pink\n",
      "R09248 pink\n",
      "R09128 pink\n",
      "R09127 pink\n",
      "R09107 pink\n",
      "R09099 pink\n",
      "R09082 pink\n",
      "R09081 pink\n",
      "R09030 pink\n",
      "R08968 pink\n",
      "R08781 pink\n",
      "R08775 pink\n",
      "R08774 pink\n",
      "R08773 pink\n",
      "R08769 pink\n",
      "R08716 pink\n",
      "R08689 pink\n",
      "R08676 pink\n",
      "R08648 pink\n",
      "R08639 pink\n",
      "R08575 pink\n",
      "R08549 pink\n",
      "R08359 pink\n",
      "R08210 pink\n",
      "R08060 pink\n",
      "R08059 pink\n",
      "R08058 pink\n",
      "R08042 pink\n",
      "R08034 pink\n",
      "R07773 pink\n",
      "R07765 pink\n",
      "R07764 pink\n",
      "R07763 pink\n",
      "R07762 pink\n",
      "R07639 pink\n",
      "R07605 pink\n",
      "R07507 pink\n",
      "R07499 pink\n",
      "R07498 pink\n",
      "R07493 pink\n",
      "R07488 pink\n",
      "R07463 pink\n",
      "R07412 pink\n",
      "R07411 pink\n",
      "R07410 pink\n",
      "R07407 pink\n",
      "R07405 pink\n",
      "R07404 pink\n",
      "R07395 pink\n",
      "R07392 pink\n",
      "R07364 pink\n",
      "R07363 pink\n",
      "R07323 pink\n",
      "R07322 pink\n",
      "R07302 pink\n",
      "R07281 pink\n",
      "R07280 pink\n",
      "R07268 pink\n",
      "R07168 pink\n",
      "R07136 pink\n",
      "R07035 pink\n",
      "R07034 pink\n",
      "R06927 pink\n",
      "R06917 pink\n",
      "R06895 pink\n",
      "R06859 pink\n",
      "R06783 pink\n",
      "R06782 pink\n",
      "R06558 pink\n",
      "R06529 pink\n",
      "R06514 pink\n",
      "R06513 pink\n",
      "R06447 pink\n",
      "R06146 pink\n",
      "R05884 pink\n",
      "R05838 pink\n",
      "R05815 pink\n",
      "R05814 pink\n",
      "R05810 pink\n",
      "R05809 pink\n",
      "R05808 pink\n",
      "R05807 pink\n",
      "R05731 pink\n",
      "R05705 pink\n",
      "R05703 pink\n",
      "R05692 pink\n",
      "R05688 pink\n",
      "R05681 pink\n",
      "R05662 pink\n",
      "R05647 pink\n",
      "R05646 pink\n",
      "R05645 pink\n",
      "R05644 pink\n",
      "R05640 pink\n",
      "R05639 pink\n",
      "R05637 pink\n",
      "R05636 pink\n",
      "R05634 pink\n",
      "R05633 pink\n",
      "R05630 pink\n",
      "R05629 pink\n",
      "R05627 pink\n",
      "R05615 pink\n",
      "R05614 pink\n",
      "R05605 pink\n",
      "R05591 pink\n",
      "R05553 pink\n",
      "R05496 pink\n",
      "R05339 pink\n",
      "R05338 pink\n",
      "R05332 pink\n",
      "R05234 pink\n",
      "R05233 pink\n",
      "R05225 pink\n",
      "R05224 pink\n",
      "R05223 pink\n",
      "R05222 pink\n",
      "R05221 pink\n",
      "R05220 pink\n",
      "R05185 pink\n",
      "R05181 pink\n",
      "R05180 pink\n",
      "R05177 pink\n",
      "R05149 pink\n",
      "R05086 pink\n",
      "R05085 pink\n",
      "R05074 pink\n",
      "R05071 pink\n",
      "R05070 pink\n",
      "R05069 pink\n",
      "R05068 pink\n",
      "R05053 pink\n",
      "R05052 pink\n",
      "R05048 pink\n",
      "R05046 pink\n",
      "R05032 pink\n",
      "R05000 pink\n",
      "R04993 pink\n",
      "R04990 pink\n",
      "R04989 pink\n",
      "R04988 pink\n",
      "R04986 pink\n",
      "R04985 pink\n",
      "R04972 pink\n",
      "R04970 pink\n",
      "R04969 pink\n",
      "R04968 pink\n",
      "R04967 pink\n",
      "R04966 pink\n",
      "R04965 pink\n",
      "R04964 pink\n",
      "R04963 pink\n",
      "R04962 pink\n",
      "R04961 pink\n",
      "R04960 pink\n",
      "R04959 pink\n",
      "R04958 pink\n",
      "R04957 pink\n",
      "R04956 pink\n",
      "R04955 pink\n",
      "R04954 pink\n",
      "R04953 pink\n",
      "R04952 pink\n",
      "R04941 pink\n",
      "R04929 pink\n",
      "R04880 pink\n",
      "R04779 pink\n",
      "R04726 pink\n",
      "R04725 pink\n",
      "R04724 pink\n",
      "R04658 pink\n",
      "R04657 pink\n",
      "R04640 pink\n",
      "R04639 pink\n",
      "R04620 pink\n",
      "R04617 pink\n",
      "R04606 pink\n",
      "R04594 pink\n",
      "R04591 pink\n",
      "R04587 pink\n",
      "R04573 pink\n",
      "R04568 pink\n",
      "R04567 pink\n",
      "R04566 pink\n",
      "R04560 pink\n",
      "R04559 pink\n",
      "R04558 pink\n",
      "R04550 pink\n",
      "R04549 pink\n",
      "R04544 pink\n",
      "R04543 pink\n",
      "R04537 pink\n",
      "R04536 pink\n",
      "R04535 pink\n",
      "R04534 pink\n",
      "R04533 pink\n",
      "R04509 pink\n",
      "R04475 pink\n",
      "R04463 pink\n",
      "R04457 pink\n",
      "R04441 pink\n",
      "R04440 pink\n",
      "R04430 pink\n",
      "R04429 pink\n",
      "R04428 pink\n",
      "R04426 pink\n",
      "R04425 pink\n",
      "R04420 pink\n",
      "R04405 pink\n",
      "R04391 pink\n",
      "R04378 pink\n",
      "R04365 pink\n",
      "R04355 pink\n",
      "R04326 pink\n",
      "R04325 pink\n",
      "R04292 pink\n",
      "R04286 pink\n",
      "R04231 pink\n",
      "R04230 pink\n",
      "R04208 pink\n",
      "R04199 pink\n",
      "R04198 pink\n",
      "R04173 pink\n",
      "R04148 pink\n",
      "R04144 pink\n",
      "R04037 pink\n",
      "R04035 pink\n",
      "R04001 pink\n",
      "R03971 pink\n",
      "R03970 pink\n",
      "R03968 pink\n",
      "R03948 pink\n",
      "R03947 pink\n",
      "R03916 pink\n",
      "R03898 pink\n",
      "R03896 pink\n",
      "R03816 pink\n",
      "R03689 pink\n",
      "R03599 pink\n",
      "R03596 pink\n",
      "R03562 pink\n",
      "R03542 pink\n",
      "R03531 pink\n",
      "R03530 pink\n",
      "R03509 pink\n",
      "R03508 pink\n",
      "R03504 pink\n",
      "R03503 pink\n",
      "R03472 pink\n",
      "R03471 pink\n",
      "R03464 pink\n",
      "R03460 pink\n",
      "R03459 pink\n",
      "R03458 pink\n",
      "R03457 pink\n",
      "R03443 pink\n",
      "R03424 pink\n",
      "R03409 pink\n",
      "R03390 pink\n",
      "R03351 pink\n",
      "R03350 pink\n",
      "R03348 pink\n",
      "R03346 pink\n",
      "R03321 pink\n",
      "R03313 pink\n",
      "R03293 pink\n",
      "R03291 pink\n",
      "R03269 pink\n",
      "R03254 pink\n",
      "R03243 pink\n",
      "R03239 pink\n",
      "R03238 pink\n",
      "R03237 pink\n",
      "R03236 pink\n",
      "R03231 pink\n",
      "R03223 pink\n",
      "R03220 pink\n",
      "R03210 pink\n",
      "R03199 pink\n",
      "R03197 pink\n",
      "R03194 pink\n",
      "R03193 pink\n",
      "R03192 pink\n",
      "R03191 pink\n",
      "R03182 pink\n",
      "R03165 pink\n",
      "R03140 pink\n",
      "R03132 pink\n",
      "R03105 pink\n",
      "R03093 pink\n",
      "R03084 pink\n",
      "R03083 pink\n",
      "R03067 pink\n",
      "R03066 pink\n",
      "R03051 pink\n",
      "R03035 pink\n",
      "R03024 pink\n",
      "R03018 pink\n",
      "R03015 pink\n",
      "R03013 pink\n",
      "R03012 pink\n",
      "R03005 pink\n",
      "R02999 pink\n",
      "R02985 pink\n",
      "R02971 pink\n",
      "R02964 pink\n",
      "R02948 pink\n",
      "R02922 pink\n",
      "R02874 pink\n",
      "R02872 pink\n",
      "R02869 pink\n",
      "R02864 pink\n",
      "R02788 pink\n",
      "R02783 pink\n",
      "R02777 pink\n",
      "R02740 pink\n",
      "R02739 pink\n",
      "R02736 pink\n",
      "R02735 pink\n",
      "R02734 pink\n",
      "R02722 pink\n",
      "R02720 pink\n",
      "R02719 pink\n",
      "R02649 pink\n",
      "R02619 pink\n",
      "R02568 pink\n",
      "R02558 pink\n",
      "R02550 pink\n",
      "R02539 pink\n",
      "R02530 pink\n",
      "R02484 pink\n",
      "R02473 pink\n",
      "R02472 pink\n",
      "R02433 pink\n",
      "R02421 pink\n",
      "R02413 pink\n",
      "R02412 pink\n",
      "R02408 pink\n",
      "R02404 pink\n",
      "R02401 pink\n",
      "R02340 pink\n",
      "R02331 pink\n",
      "R02328 pink\n",
      "R02326 pink\n",
      "R02325 pink\n",
      "R02323 pink\n",
      "R02322 pink\n",
      "R02301 pink\n",
      "R02300 pink\n",
      "R02291 pink\n",
      "R02283 pink\n",
      "R02282 pink\n",
      "R02272 pink\n",
      "R02241 pink\n",
      "R02240 pink\n",
      "R02237 pink\n",
      "R02236 pink\n",
      "R02235 pink\n",
      "R02199 pink\n",
      "R02164 pink\n",
      "R02161 pink\n",
      "R02142 pink\n",
      "R02111 pink\n",
      "R02102 pink\n",
      "R02101 pink\n",
      "R02100 pink\n",
      "R02098 pink\n",
      "R02094 pink\n",
      "R02093 pink\n",
      "R02090 pink\n",
      "R02088 pink\n",
      "R02060 pink\n",
      "R02056 pink\n",
      "R02055 pink\n",
      "R02036 pink\n",
      "R02035 pink\n",
      "R02030 pink\n",
      "R02029 pink\n",
      "R02024 pink\n",
      "R02021 pink\n",
      "R02019 pink\n",
      "R02018 pink\n",
      "R02017 pink\n",
      "R02016 pink\n",
      "R02003 pink\n",
      "R01993 pink\n",
      "R01974 pink\n",
      "R01968 pink\n",
      "R01954 pink\n",
      "R01933 pink\n",
      "R01931 pink\n",
      "R01920 pink\n",
      "R01900 pink\n",
      "R01887 pink\n",
      "R01870 pink\n",
      "R01868 pink\n",
      "R01867 pink\n",
      "R01858 pink\n",
      "R01857 pink\n",
      "R01830 pink\n",
      "R01827 pink\n",
      "R01826 pink\n",
      "R01819 pink\n",
      "R01818 pink\n",
      "R01801 pink\n",
      "R01800 pink\n",
      "R01799 pink\n",
      "R01786 pink\n",
      "R01777 pink\n",
      "R01775 pink\n",
      "R01773 pink\n",
      "R01771 pink\n",
      "R01736 pink\n",
      "R01728 pink\n",
      "R01724 pink\n",
      "R01716 pink\n",
      "R01715 pink\n",
      "R01714 pink\n",
      "R01706 pink\n",
      "R01698 pink\n",
      "R01665 pink\n",
      "R01664 pink\n",
      "R01658 pink\n",
      "R01655 pink\n",
      "R01654 pink\n",
      "R01652 pink\n",
      "R01641 pink\n",
      "R01626 pink\n",
      "R01621 pink\n",
      "R01600 pink\n",
      "R01582 pink\n",
      "R01570 pink\n",
      "R01569 pink\n",
      "R01547 pink\n",
      "R01530 pink\n",
      "R01529 pink\n",
      "R01528 pink\n",
      "R01518 pink\n",
      "R01514 pink\n",
      "R01512 pink\n",
      "R01492 pink\n",
      "R01470 pink\n",
      "R01466 pink\n",
      "R01457 pink\n",
      "R01421 pink\n",
      "R01411 pink\n",
      "R01401 pink\n",
      "R01398 pink\n",
      "R01397 pink\n",
      "R01385 pink\n",
      "R01384 pink\n",
      "R01373 pink\n",
      "R01354 pink\n",
      "R01353 pink\n",
      "R01334 pink\n",
      "R01325 pink\n",
      "R01320 pink\n",
      "R01302 pink\n",
      "R01288 pink\n",
      "R01286 pink\n",
      "R01280 pink\n",
      "R01268 pink\n",
      "R01256 pink\n",
      "R01251 pink\n",
      "R01248 pink\n",
      "R01231 pink\n",
      "R01230 pink\n",
      "R01229 pink\n",
      "R01227 pink\n",
      "R01226 pink\n",
      "R01224 pink\n",
      "R01221 pink\n",
      "R01216 pink\n",
      "R01214 pink\n",
      "R01213 pink\n",
      "R01209 pink\n",
      "R01196 pink\n",
      "R01187 pink\n",
      "R01186 pink\n",
      "R01185 pink\n",
      "R01167 pink\n",
      "R01163 pink\n",
      "R01152 pink\n",
      "R01150 pink\n",
      "R01148 pink\n",
      "R01146 pink\n",
      "R01142 pink\n",
      "R01138 pink\n",
      "R01137 pink\n",
      "R01135 pink\n",
      "R01132 pink\n",
      "R01130 pink\n",
      "R01127 pink\n",
      "R01126 pink\n",
      "R01106 pink\n",
      "R01090 pink\n",
      "R01086 pink\n",
      "R01085 pink\n",
      "R01083 pink\n",
      "R01082 pink\n",
      "R01078 pink\n",
      "R01074 pink\n",
      "R01073 pink\n",
      "R01072 pink\n",
      "R01071 pink\n",
      "R01070 pink\n",
      "R01068 pink\n",
      "R01067 pink\n",
      "R01061 pink\n",
      "R01057 pink\n",
      "R01056 pink\n",
      "R01054 pink\n",
      "R01049 pink\n",
      "R01030 pink\n",
      "R01015 pink\n",
      "R01011 pink\n",
      "R00996 pink\n",
      "R00994 pink\n",
      "R00986 pink\n",
      "R00985 pink\n",
      "R00974 pink\n",
      "R00966 pink\n",
      "R00965 pink\n",
      "R00963 pink\n",
      "R00959 pink\n",
      "R00948 pink\n",
      "R00946 pink\n",
      "R00945 pink\n",
      "R00943 pink\n",
      "R00942 pink\n",
      "R00941 pink\n",
      "R00940 pink\n",
      "R00939 pink\n",
      "R00937 pink\n",
      "R00936 pink\n",
      "R00934 pink\n",
      "R00926 pink\n",
      "R00899 pink\n",
      "R00897 pink\n",
      "R00896 pink\n",
      "R00894 pink\n",
      "R00888 pink\n",
      "R00885 pink\n",
      "R00858 pink\n",
      "R00851 pink\n",
      "R00847 pink\n",
      "R00844 pink\n",
      "R00842 pink\n",
      "R00835 pink\n",
      "R00806 pink\n",
      "R00802 pink\n",
      "R00801 pink\n",
      "R00787 pink\n",
      "R00782 pink\n",
      "R00771 pink\n",
      "R00768 pink\n",
      "R00756 pink\n",
      "R00754 pink\n",
      "R00742 pink\n",
      "R00734 pink\n",
      "R00722 pink\n",
      "R00720 pink\n",
      "R00714 pink\n",
      "R00713 pink\n",
      "R00709 pink\n",
      "R00704 pink\n",
      "R00702 pink\n",
      "R00694 pink\n",
      "R00691 pink\n",
      "R00674 pink\n",
      "R00669 pink\n",
      "R00667 pink\n",
      "R00662 pink\n",
      "R00660 pink\n",
      "R00658 pink\n",
      "R00621 pink\n",
      "R00617 pink\n",
      "R00588 pink\n",
      "R00586 pink\n",
      "R00585 pink\n",
      "R00582 pink\n",
      "R00578 pink\n",
      "R00575 pink\n",
      "R00573 pink\n",
      "R00571 pink\n",
      "R00570 pink\n",
      "R00568 pink\n",
      "R00566 pink\n",
      "R00549 pink\n",
      "R00548 pink\n",
      "R00529 pink\n",
      "R00528 pink\n",
      "R00519 pink\n",
      "R00515 pink\n",
      "R00512 pink\n",
      "R00511 pink\n",
      "R00509 pink\n",
      "R00508 pink\n",
      "R00497 pink\n",
      "R00494 pink\n",
      "R00489 pink\n",
      "R00483 pink\n",
      "R00481 pink\n",
      "R00480 pink\n",
      "R00475 pink\n",
      "R00473 pink\n",
      "R00470 pink\n",
      "R00451 pink\n",
      "R00430 pink\n",
      "R00429 pink\n",
      "R00428 pink\n",
      "R00426 pink\n",
      "R00425 pink\n",
      "R00424 pink\n",
      "R00418 pink\n",
      "R00416 pink\n",
      "R00405 pink\n",
      "R00401 pink\n",
      "R00396 pink\n",
      "R00372 pink\n",
      "R00369 pink\n",
      "R00357 pink\n",
      "R00355 pink\n",
      "R00351 pink\n",
      "R00342 pink\n",
      "R00336 pink\n",
      "R00332 pink\n",
      "R00330 pink\n",
      "R00317 pink\n",
      "R00316 pink\n",
      "R00315 pink\n",
      "R00310 pink\n",
      "R00299 pink\n",
      "R00294 pink\n",
      "R00291 pink\n",
      "R00289 pink\n",
      "R00287 pink\n",
      "R00286 pink\n",
      "R00274 pink\n",
      "R00271 pink\n",
      "R00260 pink\n",
      "R00259 pink\n",
      "R00257 pink\n",
      "R00256 pink\n",
      "R00253 pink\n",
      "R00248 pink\n",
      "R00243 pink\n",
      "R00239 pink\n",
      "R00236 pink\n",
      "R00235 pink\n",
      "R00226 pink\n",
      "R00220 pink\n",
      "R00217 pink\n",
      "R00214 pink\n",
      "R00209 pink\n",
      "R00206 pink\n",
      "R00200 pink\n",
      "R00194 pink\n",
      "R00192 pink\n",
      "R00190 pink\n",
      "R00185 pink\n",
      "R00183 pink\n",
      "R00178 pink\n",
      "R00177 pink\n",
      "R00161 pink\n",
      "R00158 pink\n",
      "R00156 pink\n",
      "R00149 pink\n",
      "R00148 pink\n",
      "R00139 pink\n",
      "R00137 pink\n",
      "R00132 pink\n",
      "R00130 pink\n",
      "R00127 pink\n",
      "R00125 pink\n",
      "R00114 pink\n",
      "R00112 pink\n",
      "R00104 pink\n",
      "R00093 pink\n",
      "R00087 pink\n",
      "R00086 pink\n",
      "R00084 pink\n",
      "R00082 pink\n",
      "R00078 pink\n",
      "R00066 pink\n",
      "R00036 pink\n",
      "R00028 pink\n",
      "R00026 pink\n",
      "R00022 pink\n",
      "R00014 pink\n",
      "R00006 pink\n",
      "R00004 pink\n",
      "R_formylTHMPT_formate pink\n",
      "R08159 pink\n",
      "R03370 pink\n",
      "R02814 pink\n",
      "R00258 pink\n",
      "R00485 pink\n",
      "R00576 pink\n",
      "R01215 pink\n",
      "R00345 pink\n",
      "R02164_ub pink\n"
     ]
    }
   ],
   "source": [
    "for rxn in mcoosmodel_rxnlist:\n",
    "    if rxn.startswith('R'):\n",
    "        print(rxn,'pink')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "650\n",
      "=====================\n",
      "R01070: beta-D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00010+R01070      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R01070      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R01070      Fructose and mannose metabolism\n",
      "R00754: ethanol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00754      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00754      Pyruvate metabolism\n",
      "R00014: pyruvate:thiamin diphosphate acetaldehydetransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00010+R00014      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R00014      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00014      Pyruvate metabolism\n",
      "R00200: ATP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00200      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00200      Pyruvate metabolism\n",
      "R00658: 2-phospho-D-glycerate hydro-lyase (phosphoenolpyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00010+R00658      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00680+R00658      Methane metabolism\n",
      "R01061: D-glyceraldehyde-3-phosphate:NAD+ oxidoreductase (phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00010+R01061      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00710+R01061      Carbon fixation in photosynthetic organisms\n",
      "R01015: D-glyceraldehyde-3-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01015      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00051+R01015      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00562+R01015      Inositol phosphate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R01015      Carbon fixation in photosynthetic organisms\n",
      "R04779: ATP:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R04779      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R04779      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R04779      Fructose and mannose metabolism\n",
      "R02740: alpha-D-Glucose 6-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02740      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R02740      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00520+R02740      Amino sugar and nucleotide sugar metabolism\n",
      "R00959: alpha-D-Glucose 1-phosphate 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00959      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00052+R00959      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00959      Amino sugar and nucleotide sugar metabolism\n",
      "R03321: beta-D-Glucose 6-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R03321      Glycolysis / Gluconeogenesis\n",
      "R01600: ATP:beta-D-glucose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01600      Glycolysis / Gluconeogenesis\n",
      "R02739: alpha-D-Glucose 6-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02739      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R02739      Pentose phosphate pathway\n",
      "R01786: ATP:alpha-D-glucose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01786      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00052+R01786      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01786      Amino sugar and nucleotide sugar metabolism\n",
      "R01512: ATP:3-phospho-D-glycerate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01512      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00710+R01512      Carbon fixation in photosynthetic organisms\n",
      "R09127: ethanol:cytochrome c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R09127      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R09127      Pyruvate metabolism\n",
      "R01196: pyruvate:ferredoxin 2-oxidoreductase (CoA-acetylating)\n",
      "     https://www.kegg.jp/pathway/rn00010+R01196      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R01196      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R01196      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R01196      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01196      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01196      Carbon fixation pathways in prokaryotes\n",
      "R00235: Acetate:CoA ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00010+R00235      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00630+R00235      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00235      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00235      Carbon fixation pathways in prokaryotes\n",
      "R01518: D-phosphoglycerate 2,3-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01518      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00260+R01518      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01518      Methane metabolism\n",
      "R00206: ATP:pyruvate,phosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00206      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00206      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00206      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00206      Carbon fixation pathways in prokaryotes\n",
      "R00621: \n",
      "     https://www.kegg.jp/pathway/rn00020+R00621      Citrate cycle (TCA cycle)\n",
      "R00405: Succinate:CoA ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00405      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00640+R00405      Propanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00405      Carbon fixation pathways in prokaryotes\n",
      "R00709: isocitrate:NAD+ oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00709      Citrate cycle (TCA cycle)\n",
      "R01082: (S)-malate hydro-lyase (fumarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R01082      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R01082      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01082      Carbon fixation pathways in prokaryotes\n",
      "R01900: isocitrate hydro-lyase (cis-aconitate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R01900      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00630+R01900      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01900      Carbon fixation pathways in prokaryotes\n",
      "R01325: citrate hydro-lyase (cis-aconitate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R01325      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00630+R01325      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01325      Carbon fixation pathways in prokaryotes\n",
      "R00351: acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00351      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00351      Glyoxylate and dicarboxylate metabolism\n",
      "R00342: (S)-malate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00020+R00342      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00342      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00342      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00342      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00342      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00342      Carbon fixation pathways in prokaryotes\n",
      "R02164: succinate:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00020+R02164      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R02164      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R02164      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R02164      Carbon fixation pathways in prokaryotes\n",
      "R05605: 2-dehydro-3-deoxy-6-phospho-D-gluconate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R05605      Pentose phosphate pathway\n",
      "R01049: ATP:D-ribose-5-phosphate diphosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01049      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00230+R01049      Purine metabolism\n",
      "R01641: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01641      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00710+R01641      Carbon fixation in photosynthetic organisms\n",
      "R01056: D-ribose-5-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01056      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00710+R01056      Carbon fixation in photosynthetic organisms\n",
      "R01529: D-Ribulose-5-phosphate 3-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01529      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00040+R01529      Pentose and glucuronate interconversions\n",
      "     https://www.kegg.jp/pathway/rn00710+R01529      Carbon fixation in photosynthetic organisms\n",
      "R01830: beta-D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01830      Pentose phosphate pathway\n",
      "R01528: 6-phospho-D-gluconate:NADP+ 2-oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00030+R01528      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00480+R01528      Glutathione metabolism\n",
      "R02035: 6-Phospho-D-glucono-1,5-lactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00030+R02035      Pentose phosphate pathway\n",
      "R02736: beta-D-glucose-6-phosphate:NADP+ 1-oxoreductase\n",
      "     https://www.kegg.jp/pathway/rn00030+R02736      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00480+R02736      Glutathione metabolism\n",
      "R02036: 6-Phospho-D-gluconate hydro-lyase(2-dehydro-3-deoxy-6-phospho-D-gluconate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R02036      Pentose phosphate pathway\n",
      "R01827: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01827      Pentose phosphate pathway\n",
      "R01057: D-Ribose 1,5-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01057      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00230+R01057      Purine metabolism\n",
      "R01621: D-xylulose 5-phosphate D-glyceraldehyde-3-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R01621      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00710+R01621      Carbon fixation in photosynthetic organisms\n",
      "R05338: D-arabino-hex-3-ulose-6-phosphate formaldehyde-lyase (D-ribulose-5-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R05338      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00680+R05338      Methane metabolism\n",
      "R09780: D-arabino-hex-3-ulose-6-phosphate isomerase\n",
      "     https://www.kegg.jp/pathway/rn00030+R09780      Pentose phosphate pathway\n",
      "R10221: 6-phospho-D-gluconate:NAD+ 2-oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00030+R10221      Pentose phosphate pathway\n",
      "R00289: UTP:alpha-D-glucose-1-phosphate uridylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00040+R00289      Pentose and glucuronate interconversions\n",
      "     https://www.kegg.jp/pathway/rn00052+R00289      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00500+R00289      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00289      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00289      O-Antigen nucleotide sugar biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00561+R00289      Glycerolipid metabolism\n",
      "R00286: UDP-glucose:NAD+ 6-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00040+R00286      Pentose and glucuronate interconversions\n",
      "     https://www.kegg.jp/pathway/rn00053+R00286      Ascorbate and aldarate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00286      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00286      O-Antigen nucleotide sugar biosynthesis\n",
      "R01819: D-mannose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00051+R01819      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01819      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R01819      O-Antigen nucleotide sugar biosynthesis\n",
      "R01818: D-mannose 6-phosphate 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00051+R01818      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01818      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R01818      O-Antigen nucleotide sugar biosynthesis\n",
      "R05692: GDP-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing)\n",
      "     https://www.kegg.jp/pathway/rn00051+R05692      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R05692      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R05692      O-Antigen nucleotide sugar biosynthesis\n",
      "R00888: GDP-alpha-D-mannose 4,6-hydro-lyase (GDP-4-dehydro-alpha-D-rhamnose-forming);\n",
      "     https://www.kegg.jp/pathway/rn00051+R00888      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00888      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00888      O-Antigen nucleotide sugar biosynthesis\n",
      "R00885: GTP:alpha-D-mannose-1-phosphate guanylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00051+R00885      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00885      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00885      O-Antigen nucleotide sugar biosynthesis\n",
      "R02568: D-fructose 1-phosphate D-glyceraldehyde-3-phosphate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00051+R02568      Fructose and mannose metabolism\n",
      "R09030: D-allose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00051+R09030      Fructose and mannose metabolism\n",
      "R03236: ATP:D-tagatose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03236      Galactose metabolism\n",
      "R00801: sucrose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00052+R00801      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00500+R00801      Starch and sucrose metabolism\n",
      "R00291: UDP-glucose 4-epimerase;\n",
      "     https://www.kegg.jp/pathway/rn00052+R00291      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00291      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00291      O-Antigen nucleotide sugar biosynthesis\n",
      "R01385: UDP-glucuronate 4-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00053+R01385      Ascorbate and aldarate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01385      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R01385      O-Antigen nucleotide sugar biosynthesis\n",
      "R07765: octadecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R07765      Fatty acid biosynthesis\n",
      "R07764: \n",
      "     https://www.kegg.jp/pathway/rn00061+R07764      Fatty acid biosynthesis\n",
      "R07762: \n",
      "     https://www.kegg.jp/pathway/rn00061+R07762      Fatty acid biosynthesis\n",
      "R07763: \n",
      "     https://www.kegg.jp/pathway/rn00061+R07763      Fatty acid biosynthesis\n",
      "R04968: Tetradecanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04968      Fatty acid biosynthesis\n",
      "R04726: dodecanoyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04726      Fatty acid biosynthesis\n",
      "R04963: Decanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04963      Fatty acid biosynthesis\n",
      "R04960: Octanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04960      Fatty acid biosynthesis\n",
      "R04957: hexanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04957      Fatty acid biosynthesis\n",
      "R04952: butyryl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04952      Fatty acid biosynthesis\n",
      "R04969: hexadecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04969      Fatty acid biosynthesis\n",
      "R04543: (3R)-3-hydroxypalmitoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04543      Fatty acid biosynthesis\n",
      "R04966: tetradecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04966      Fatty acid biosynthesis\n",
      "R04566: (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04566      Fatty acid biosynthesis\n",
      "R04724: dodecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04724      Fatty acid biosynthesis\n",
      "R04964: (3R)-3-hydroxydodecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04964      Fatty acid biosynthesis\n",
      "R04961: decanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04961      Fatty acid biosynthesis\n",
      "R04534: (3R)-3-hydroxydecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04534      Fatty acid biosynthesis\n",
      "R04958: octanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04958      Fatty acid biosynthesis\n",
      "R04536: (3R)-3-hydroxyoctanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04536      Fatty acid biosynthesis\n",
      "R04955: hexanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04955      Fatty acid biosynthesis\n",
      "R04953: (3R)-3-hydroxyhexanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04953      Fatty acid biosynthesis\n",
      "R04429: butyryl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04429      Fatty acid biosynthesis\n",
      "R04428: (3R)-3-hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04428      Fatty acid biosynthesis\n",
      "R04533: (3R)-3-hydroxybutanoyl-[acyl-carrier protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04533      Fatty acid biosynthesis\n",
      "R01626: Malonyl-CoA:[acyl-carrier-protein] S-malonyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01626      Fatty acid biosynthesis\n",
      "R00742: acetyl-CoA:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00061+R00742      Fatty acid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00254+R00742      Aflatoxin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00742      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00640+R00742      Propanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00742      Carbon fixation pathways in prokaryotes\n",
      "R04355: Acyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04355      Fatty acid biosynthesis\n",
      "R10707: acetyl-CoA:malonyl-[acyl-carrier protein] C-acetyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R10707      Fatty acid biosynthesis\n",
      "R04954: (3R)-3-hydroxyhexanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04954      Fatty acid biosynthesis\n",
      "R04537: (3R)-3-hydroxyoctanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04537      Fatty acid biosynthesis\n",
      "R04535: (3R)-3-hydroxydecanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04535      Fatty acid biosynthesis\n",
      "R04965: (3R)-3-hydroxydodecanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04965      Fatty acid biosynthesis\n",
      "R04568: (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04568      Fatty acid biosynthesis\n",
      "R04544: (3R)-3-hydroxypalmitoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04544      Fatty acid biosynthesis\n",
      "R07639: decenoyl-[acyl-carrier-protein] Delta2-trans-Delta3-cis-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00061+R07639      Fatty acid biosynthesis\n",
      "R01280: Palmitate:CoA ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00061+R01280      Fatty acid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00071+R01280      Fatty acid degradation\n",
      "R07493: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07493      Steroid biosynthesis\n",
      "R01457: cholesterol:NADP+ Delta24-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R01457      Steroid biosynthesis\n",
      "R07507: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07507      Steroid biosynthesis\n",
      "R05703: lanosterol D24-reductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R05703      Steroid biosynthesis\n",
      "R07498: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07498      Steroid biosynthesis\n",
      "R07499: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07499      Steroid biosynthesis\n",
      "R07488: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07488      Steroid biosynthesis\n",
      "R05731: obtusifoliol,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R05731      Steroid biosynthesis\n",
      "R03689: lanosterol delta24-reductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R03689      Steroid biosynthesis\n",
      "R05640: lanosterol,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R05640      Steroid biosynthesis\n",
      "R02874: squalene,NADPH-hemoprotein:oxygen oxidoreductase (2,3-epoxidizing)\n",
      "     https://www.kegg.jp/pathway/rn00100+R02874      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R02874      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R11096: \n",
      "     https://www.kegg.jp/pathway/rn00100+R11096      Steroid biosynthesis\n",
      "R08769: S-adenosyl-L-methionine:2-polyprenyl-6-hydroxyphenol 6-O-methyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00130+R08769      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R04985: 4-hydroxy-3-polyprenylbenzoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00130+R04985      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R05000: polyisopentenylpyrolinate:4-hydroxybenzoate nonaprenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R05000      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R01302: chorismate pyruvate-lyase (4-hydroxybenzoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00130+R01302      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R02964: NADH:menaquinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00130+R02964      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R03816: NADH:phylloquinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00130+R03816      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R00253: L-Glutamate:ammonia ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00253      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00253      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00253      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00253      Nitrogen metabolism\n",
      "R01398: Carbamoyl-phosphate:L-ornithine carbamoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R01398      Arginine biosynthesis\n",
      "R01954: L-Citrulline:L-aspartate ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R01954      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R01954      Alanine, aspartate and glutamate metabolism\n",
      "R01086: 2-(Nomega-L-arginino)succinate arginine-lyase (fumarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R01086      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R01086      Alanine, aspartate and glutamate metabolism\n",
      "R00259: acetyl-CoA:L-glutamate N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00259      Arginine biosynthesis\n",
      "R02649: ATP:N-acetyl-L-glutamate 5-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R02649      Arginine biosynthesis\n",
      "R03443: N-acetyl-L-glutamate-5-semialdehyde:NADP+ 5-oxidoreductase (phosphrylating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R03443      Arginine biosynthesis\n",
      "R02283: N2-Acetyl-L-ornithine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R02283      Arginine biosynthesis\n",
      "R00669: N2-Acetyl-L-ornithine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00669      Arginine biosynthesis\n",
      "R02282: N2-Acetyl-L-ornithine:L-glutamate N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R02282      Arginine biosynthesis\n",
      "R00243: L-glutamate:NAD+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00243      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00243      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00430+R00243      Taurine and hypotaurine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00243      Nitrogen metabolism\n",
      "R09107: N-acetyl-L-citrulline amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R09107      Arginine biosynthesis\n",
      "R00355: L-Aspartate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00355      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00355      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00355      Carbon fixation in photosynthetic organisms\n",
      "R02019: 2'-deoxyguanosine 5'-diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02019      Purine metabolism\n",
      "R01072: 5-phosphoribosylamine:diphosphate phospho-alpha-D-ribosyltransferase (glutamate-amidating)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01072      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01072      Alanine, aspartate and glutamate metabolism\n",
      "R04144: 5-Phospho-D-ribosylamine:glycine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04144      Purine metabolism\n",
      "R04208: 2-(Formamido)-N1-(5-phosphoribosyl)acetamidine cyclo-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04208      Purine metabolism\n",
      "R04560: 10-Formyltetrahydrofolate:5'-phosphoribosyl-5-amino-4-imidazolecarboxamide formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04560      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R04560      One carbon pool by folate\n",
      "R04559: 1-(5'-Phosphoribosyl)-5-amino-4-(N-succinocarboxamide)-imidazole AMP-lyase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04559      Purine metabolism\n",
      "R04591: 1-(5-Phosphoribosyl)-5-amino-4-carboxyimidazole:L-aspartate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04591      Purine metabolism\n",
      "R01127: IMP 1,2-hydrolase (decyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01127      Purine metabolism\n",
      "R02017: 2'-deoxyadenosine 5'-diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02017      Purine metabolism\n",
      "R00127: ATP:AMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00127      Purine metabolism\n",
      "R01083: N6-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01083      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01083      Alanine, aspartate and glutamate metabolism\n",
      "R01135: IMP:L-aspartate ligase (GDP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01135      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01135      Alanine, aspartate and glutamate metabolism\n",
      "R00183: adenosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00183      Purine metabolism\n",
      "R00190: AMP:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00190      Purine metabolism\n",
      "R02088: 2'-deoxyadenosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02088      Purine metabolism\n",
      "R01126: inosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01126      Purine metabolism\n",
      "R00722: ATP:IDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00722      Purine metabolism\n",
      "R01130: IMP:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01130      Purine metabolism\n",
      "R01230: Xanthosine-5'-phosphate:ammonia ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01230      Purine metabolism\n",
      "R01132: IMP:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01132      Purine metabolism\n",
      "R01227: guanosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01227      Purine metabolism\n",
      "R02142: XMP:pyrophosphate phosphoribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02142      Purine metabolism\n",
      "R02719: xanthosine 5'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02719      Purine metabolism\n",
      "R01229: GMP:diphosphate 5-phospho-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01229      Purine metabolism\n",
      "R00332: ATP:GMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00332      Purine metabolism\n",
      "R00336: guanosine-3',5'-bis(diphosphate) 3'-diphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00336      Purine metabolism\n",
      "R03409: guanosine 3'-diphosphate 5'-triphosphate 5'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R03409      Purine metabolism\n",
      "R02090: ATP:dGMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02090      Purine metabolism\n",
      "R01857: ATP:dGDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01857      Purine metabolism\n",
      "R01968: 2'-deoxyguanosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01968      Purine metabolism\n",
      "R00429: ATP:GTP 3'-diphosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00429      Purine metabolism\n",
      "R00125: P1,P4-bis(5'-adenosyl)-tetraphosphate nucleotidebisphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00125      Purine metabolism\n",
      "R00529: ATP:sulfate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00529      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00261+R00529      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00920+R00529      Sulfur metabolism\n",
      "R00509: ATP:adenylylsulfate 3'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00509      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R00509      Sulfur metabolism\n",
      "R04463: 5'-Phosphoribosylformylglycinamide:L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04463      Purine metabolism\n",
      "R00720: Inosine 5'-triphosphate pyrophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00720      Purine metabolism\n",
      "R02720: XTP pyrophosphohydrolase;\n",
      "     https://www.kegg.jp/pathway/rn00230+R02720      Purine metabolism\n",
      "R01054: ADP-ribose ribophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01054      Purine metabolism\n",
      "R01547: ATP:dAMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01547      Purine metabolism\n",
      "R03530: ATP:dIDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R03530      Purine metabolism\n",
      "R04378: 1-(5'-Phosphoribosyl)-5-amino-4-imidazolecarboxamide:pyrophosphate phosphoribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04378      Purine metabolism\n",
      "R07404: 5-amino-1-(5-phospho-D-ribosyl)imidazole:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R07404      Purine metabolism\n",
      "R07405: 5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole carboxymutase\n",
      "     https://www.kegg.jp/pathway/rn00230+R07405      Purine metabolism\n",
      "R03531: 2'-deoxyinosine-5'-triphosphate pyrophosphohydrolase;\n",
      "     https://www.kegg.jp/pathway/rn00230+R03531      Purine metabolism\n",
      "R00087: ATP diphosphohydrolase (diphosphate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00230+R00087      Purine metabolism\n",
      "R00426: GTP diphosphohydrolase (diphosphate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00230+R00426      Purine metabolism\n",
      "R00330: ATP:GDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00330      Purine metabolism\n",
      "R01137: ATP:dADP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01137      Purine metabolism\n",
      "R00185: ATP:adenosine 5'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00185      Purine metabolism\n",
      "R02018: 2'-deoxyuridine 5'-diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02018      Pyrimidine metabolism\n",
      "R00158: ATP:UMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00158      Pyrimidine metabolism\n",
      "R00575: HCO3-:L-glutamine amido-ligase (ADP-forming, carbamate-phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00575      Pyrimidine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R00575      Alanine, aspartate and glutamate metabolism\n",
      "R01397: carbamoyl-phosphate:L-aspartate carbamoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01397      Pyrimidine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01397      Alanine, aspartate and glutamate metabolism\n",
      "R01993: (S)-dihydroorotate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01993      Pyrimidine metabolism\n",
      "R01867: (S)-dihydroorotate:fumarate oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01867      Pyrimidine metabolism\n",
      "R01870: Orotidine-5'-phosphate:diphosphate phospho-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01870      Pyrimidine metabolism\n",
      "R00156: ATP:UDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00156      Pyrimidine metabolism\n",
      "R00571: UTP:ammonia ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00571      Pyrimidine metabolism\n",
      "R00966: UMP:diphosphate phospho-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00966      Pyrimidine metabolism\n",
      "R00963: uridine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00963      Pyrimidine metabolism\n",
      "R00570: ATP:CDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00570      Pyrimidine metabolism\n",
      "R00511: cytidine-5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00511      Pyrimidine metabolism\n",
      "R00974: Cytosine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00974      Pyrimidine metabolism\n",
      "R02024: 2'-deoxycytidine diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02024      Pyrimidine metabolism\n",
      "R02326: ATP:dCDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02326      Pyrimidine metabolism\n",
      "R02325: dCTP aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02325      Pyrimidine metabolism\n",
      "R01664: 2'-deoxycytidine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01664      Pyrimidine metabolism\n",
      "R02331: ATP:dUDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02331      Pyrimidine metabolism\n",
      "R02098: ATP:dUMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02098      Pyrimidine metabolism\n",
      "R02484: deoxyuridine:orthophosphate 2-deoxy-D-ribosyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00240+R02484      Pyrimidine metabolism\n",
      "R02101: 5,10-Methylenetetrahydrofolate:dUMP C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02101      Pyrimidine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R02101      One carbon pool by folate\n",
      "R02093: ATP:dTDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02093      Pyrimidine metabolism\n",
      "R02094: ATP:dTMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02094      Pyrimidine metabolism\n",
      "R01569: thymidylate 5'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01569      Pyrimidine metabolism\n",
      "R01570: thymidine:phosphate deoxy-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01570      Pyrimidine metabolism\n",
      "R01411: 5-Methylcytosine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01411      Pyrimidine metabolism\n",
      "R01868: (S)-dihydroorotate:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01868      Pyrimidine metabolism\n",
      "R00965: orotidine-5'-phosphate carboxy-lyase (UMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00965      Pyrimidine metabolism\n",
      "R00662: uridine triphosphate pyrophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00662      Pyrimidine metabolism\n",
      "R02102: 2'-deoxyuridine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02102      Pyrimidine metabolism\n",
      "R00139: ATP:2'-deoxy-5-hydroxymethylcytidine-5'-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00139      Pyrimidine metabolism\n",
      "R00515: CTP diphosphohydrolase (diphosphate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00240+R00515      Pyrimidine metabolism\n",
      "R02100: dUTP nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02100      Pyrimidine metabolism\n",
      "R01665: ATP:dCMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01665      Pyrimidine metabolism\n",
      "R00512: ATP:CMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00512      Pyrimidine metabolism\n",
      "R00578: L-aspartate:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00578      Alanine, aspartate and glutamate metabolism\n",
      "R00357: L-Aspartic acid:oxygen oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00357      Alanine, aspartate and glutamate metabolism\n",
      "R00369: L-Alanine:glyoxylate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00369      Alanine, aspartate and glutamate metabolism\n",
      "R00396: L-alanine:NAD+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00396      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00430+R00396      Taurine and hypotaurine metabolism\n",
      "R00713: succinate-semialdehyde:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00713      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00350+R00713      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R00713      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00760+R00713      Nicotinate and nicotinamide metabolism\n",
      "R00114: L-glutamate:NADP+ oxidoreductase (transaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00114      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00114      Nitrogen metabolism\n",
      "R00768: L-glutamine:D-fructose-6-phosphate isomerase (deaminating);\n",
      "     https://www.kegg.jp/pathway/rn00250+R00768      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00768      Amino sugar and nucleotide sugar metabolism\n",
      "R00714: succinate-semialdehyde:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00714      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00350+R00714      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R00714      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00760+R00714      Nicotinate and nicotinamide metabolism\n",
      "R01514: ATP:(R)-glycerate 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01514      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00561+R01514      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R01514      Glyoxylate and dicarboxylate metabolism\n",
      "R00588: L-Serine:glyoxylate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00588      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00588      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00588      Methane metabolism\n",
      "R00585: L-Serine:pyruvate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00585      Glycine, serine and threonine metabolism\n",
      "R00480: ATP:L-aspartate 4-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00480      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00261+R00480      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00270+R00480      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R00480      Lysine biosynthesis\n",
      "R04173: 3-Phosphoserine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R04173      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R04173      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R04173      Methane metabolism\n",
      "R00582: O-phospho-L-serine phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00582      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00582      Methane metabolism\n",
      "R00945: 5,10-Methylenetetrahydrofolate:glycine hydroxymethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00945      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00460+R00945      Cyanoamino acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00945      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R00945      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00680+R00945      Methane metabolism\n",
      "R00996: L-threonine ammonia-lyase (2-oxobutanoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R00996      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00290+R00996      Valine, leucine and isoleucine biosynthesis\n",
      "R01466: O-phospho-L-homoserine phosphate-lyase (adding water;L-threonine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R01466      Glycine, serine and threonine metabolism\n",
      "R02291: L-Aspartate-4-semialdehyde:NADP+ oxidoreductase (phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00260+R02291      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R02291      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R02291      Lysine biosynthesis\n",
      "R01773: L-Homoserine:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01773      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R01773      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R01773      Lysine biosynthesis\n",
      "R00220: L-serine ammonia-lyase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00220      Glycine, serine and threonine metabolism\n",
      "R01800: CDP-diacylglycerol:L-serine 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01800      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R01800      Glycerophospholipid metabolism\n",
      "R02722: L-serine hydro-lyase [adding 1-C-(indol-3-yl)glycerol 3-phosphate; L-tryptophan and glyceraldehyde-3-phosphate-forming]\n",
      "     https://www.kegg.jp/pathway/rn00260+R02722      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00400+R02722      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R00372: Glycine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00372      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00372      Glyoxylate and dicarboxylate metabolism\n",
      "R04198: 2,3,4,5-tetrahydrodipicolinate:NAD+ 4-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00261+R04198      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00300+R04198      Lysine biosynthesis\n",
      "R10147: L-aspartate-4-semialdehyde hydro-lyase [adding pyruvate and cyclizing; (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate-forming]\n",
      "     https://www.kegg.jp/pathway/rn00261+R10147      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00300+R10147      Lysine biosynthesis\n",
      "R01288: O4-succinyl-L-homoserine:hydrogen sulfide S-(3-amino-3-carboxypropyl)transferase;\n",
      "     https://www.kegg.jp/pathway/rn00270+R01288      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R01288      Sulfur metabolism\n",
      "R07364: 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one:oxygen oxidoreductase (formate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R07364      Cysteine and methionine metabolism\n",
      "R07363: 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one:oxygen oxidoreductase (formate- and CO-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R07363      Cysteine and methionine metabolism\n",
      "R07392: S-methyl-5-thio-D-ribulose-1-phosphate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00270+R07392      Cysteine and methionine metabolism\n",
      "R04420: 5-methylthio-5-deoxy-D-ribose-1-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04420      Cysteine and methionine metabolism\n",
      "R01401: methylthioadenosine methylthioribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01401      Cysteine and methionine metabolism\n",
      "R01920: S-adenosylmethioninamine:putrescine 3-aminopropyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01920      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R01920      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00480+R01920      Glutathione metabolism\n",
      "R00178: S-adenosyl-L-methionine carboxy-lyase [S-adenosyl 3-(methylsulfanyl)propylamine-forming]\n",
      "     https://www.kegg.jp/pathway/rn00270+R00178      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R00178      Arginine and proline metabolism\n",
      "R00946: 5-methyltetrahydrofolate:L-homocysteine S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00946      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R00946      One carbon pool by folate\n",
      "R01286: L-cystathionine L-homocysteine-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R01286      Cysteine and methionine metabolism\n",
      "R04405: 5-Methyltetrahydropteroyltri-L-glutamate:L-homocysteine S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04405      Cysteine and methionine metabolism\n",
      "R00177: ATP:L-methionine S-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00177      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00999+R00177      Biosynthesis of various plant secondary metabolites\n",
      "R00192: S-Adenosyl-L-homocysteine hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00192      Cysteine and methionine metabolism\n",
      "R00194: S-Adenosyl-L-homocysteine homocysteinylribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00194      Cysteine and methionine metabolism\n",
      "R07395: 5-(methylthio)-2,3-dioxopentyl-phosphate phosphohydrolase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00270+R07395      Cysteine and methionine metabolism\n",
      "R00897: O3-acetyl-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00270+R00897      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R00897      Sulfur metabolism\n",
      "R00586: acetyl-CoA:L-serine O-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00586      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R00586      Sulfur metabolism\n",
      "     https://www.kegg.jp/pathway/rn00998+R00586      Biosynthesis of various antibiotics\n",
      "R03132: O-acetyl-L-serine:thiosulfate 2-amino-2-carboxyethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R03132      Cysteine and methionine metabolism\n",
      "R02433: L-cysteate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R02433      Cysteine and methionine metabolism\n",
      "R02619: 3-sulfino-L-alanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R02619      Cysteine and methionine metabolism\n",
      "R00782: L-cysteine hydrogen-sulfide-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R00782      Cysteine and methionine metabolism\n",
      "R03105: 3-mercaptopyruvate:sulfide sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R03105      Cysteine and methionine metabolism\n",
      "R00896: L-cysteine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00896      Cysteine and methionine metabolism\n",
      "R01777: Succinyl-CoA:L-homoserine O-succinyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01777      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R01777      Sulfur metabolism\n",
      "R07136: (2R)-3-sulfolactate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00270+R07136      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R07136      Methane metabolism\n",
      "R10991: 2-aminobutanoate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R10991      Cysteine and methionine metabolism\n",
      "R10993: L-glutamate:2-aminobutanoate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R10993      Cysteine and methionine metabolism\n",
      "R10994: gamma-L-glutamyl-L-2-aminobutyrate:glycine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R10994      Cysteine and methionine metabolism\n",
      "R09660: S-methyl-5'-thioadenosine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R09660      Cysteine and methionine metabolism\n",
      "R09668: S-methyl-5'-thioinosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R09668      Cysteine and methionine metabolism\n",
      "R02408: L-cystine thiocysteine-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R02408      Cysteine and methionine metabolism\n",
      "R01214: L-Valine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00280+R01214      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00290+R01214      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00770+R01214      Pantothenate and CoA biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00966+R01214      Glucosinolate biosynthesis\n",
      "R02199: L-Isoleucine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00280+R02199      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00290+R02199      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00966+R02199      Glucosinolate biosynthesis\n",
      "R01090: L-Leucine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00280+R01090      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00290+R01090      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00966+R01090      Glucosinolate biosynthesis\n",
      "R03898: 2-methylmaleate hydratase;\n",
      "     https://www.kegg.jp/pathway/rn00290+R03898      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R03898      C5-Branched dibasic acid metabolism\n",
      "R00994: (2R,3S)-3-methylmalate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00290+R00994      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R00994      C5-Branched dibasic acid metabolism\n",
      "R05068: (R)-2,3-Dihydroxy-3-methylpentanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00290+R05068      Valine, leucine and isoleucine biosynthesis\n",
      "R03968: 2-Isopropylmalate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R03968      Valine, leucine and isoleucine biosynthesis\n",
      "R04426: (2R,3S)-3-isopropylmalate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00290+R04426      Valine, leucine and isoleucine biosynthesis\n",
      "R04001: 3-Isopropylmalate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R04001      Valine, leucine and isoleucine biosynthesis\n",
      "R01213: acetyl-CoA:3-methyl-2-oxobutanoate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming);\n",
      "     https://www.kegg.jp/pathway/rn00290+R01213      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R01213      Pyruvate metabolism\n",
      "R04441: (R)-2,3-Dihydroxy-3-methylbutanoate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R04441      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00770+R04441      Pantothenate and CoA biosynthesis\n",
      "R05069: (S)-2-Aceto-2-hydroxybutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00290+R05069      Valine, leucine and isoleucine biosynthesis\n",
      "R05071: (S)-2-Acetolactate methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00290+R05071      Valine, leucine and isoleucine biosynthesis\n",
      "R05070: (R)-2,3-Dihydroxy-3-methylpentanoate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R05070      Valine, leucine and isoleucine biosynthesis\n",
      "R04440: (R)-2,3-Dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00290+R04440      Valine, leucine and isoleucine biosynthesis\n",
      "R08648: pyruvate:2-oxobutanoate acetaldehydetransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00290+R08648      Valine, leucine and isoleucine biosynthesis\n",
      "R03896: (R)-2-Methylmalate hydro-lyase (2-methylmaleate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00290+R03896      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R03896      C5-Branched dibasic acid metabolism\n",
      "R00226: pyruvate:pyruvate acetaldehydetransferase (decarboxylating);\n",
      "     https://www.kegg.jp/pathway/rn00290+R00226      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00650+R00226      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R00226      C5-Branched dibasic acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00770+R00226      Pantothenate and CoA biosynthesis\n",
      "R04617: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminoheptanedioate:D-alanyl-D-alanine ligase(ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00300+R04617      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00550+R04617      Peptidoglycan biosynthesis\n",
      "R04365: Succinyl-CoA:2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R04365      Lysine biosynthesis\n",
      "R04475: N-Succinyl-L-2,6-diaminoheptanedioate:2-oxoglutarate amino-transferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R04475      Lysine biosynthesis\n",
      "R02734: N-Succinyl-LL-2,6-diaminoheptanedioate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00300+R02734      Lysine biosynthesis\n",
      "R02735: LL-2,6-Diaminoheptanedioate 2-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00300+R02735      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00470+R02735      D-Amino acid metabolism\n",
      "R00451: meso-2,6-diaminoheptanedioate carboxy-lyase (L-lysine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00300+R00451      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00470+R00451      D-Amino acid metabolism\n",
      "R02788: UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:(L)-meso-2,6-diaminoheptanedioate gamma-ligase (ADP-forming);\n",
      "     https://www.kegg.jp/pathway/rn00300+R02788      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00550+R02788      Peptidoglycan biosynthesis\n",
      "R01933: 2-oxoadipate dehydrogenase complex\n",
      "     https://www.kegg.jp/pathway/rn00310+R01933      Lysine degradation\n",
      "     https://www.kegg.jp/pathway/rn00380+R01933      Tryptophan metabolism\n",
      "R02401: glutarate-semialdehyde:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00310+R02401      Lysine degradation\n",
      "R02922: Creatinine iminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R02922      Arginine and proline metabolism\n",
      "R09081: carboxyspermidine carboxy-lyase (spermidine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00330+R09081      Arginine and proline metabolism\n",
      "R09082: carboxynorspermidine carboxy-lyase (norspermidine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00330+R09082      Arginine and proline metabolism\n",
      "R00566: L-arginine carboxy-lyase (agmatine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00330+R00566      Arginine and proline metabolism\n",
      "R01152: N-Carbamoylputrescine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01152      Arginine and proline metabolism\n",
      "R00667: L-Ornithine:2-oxo-acid aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R00667      Arginine and proline metabolism\n",
      "R01248: L-Proline:NAD+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01248      Arginine and proline metabolism\n",
      "R00239: ATP:L-glutamate 5-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R00239      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00332+R00239      Carbapenem biosynthesis\n",
      "R03313: L-glutamate-5-semialdehyde:NADP+ 5-oxidoreductase (phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00330+R03313      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00332+R03313      Carbapenem biosynthesis\n",
      "R03291: trans-4-Hydroxy-L-proline:NAD+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03291      Arginine and proline metabolism\n",
      "R05052: L-erythro-4-hydroxyglutamate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R05052      Arginine and proline metabolism\n",
      "R05053: D-aspartate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R05053      Arginine and proline metabolism\n",
      "R01071: 1-(5-phospho-D-ribosyl)-ATP:diphosphate phospho-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00340+R01071      Histidine metabolism\n",
      "R01163: L-histidinal:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00340+R01163      Histidine metabolism\n",
      "R03012: L-Histidinol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03012      Histidine metabolism\n",
      "R03243: 5-Amino-2-oxopentanoate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03243      Histidine metabolism\n",
      "R03457: D-erythro-1-(Imidazol-4-yl)glycerol 3-phosphate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03457      Histidine metabolism\n",
      "R04640: N-(5'-phospho-D-ribosylformimino)-5-amino-1- (5''-phospho-D-ribosyl)-4-imidazolecarboxamide ketol-isomerase;\n",
      "     https://www.kegg.jp/pathway/rn00340+R04640      Histidine metabolism\n",
      "R04037: 1-(5-phospho-D-ribosyl)-AMP 1,6-hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00340+R04037      Histidine metabolism\n",
      "R04035: Phosphoribosyl-ATP pyrophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00340+R04035      Histidine metabolism\n",
      "R04558: 5-[(5-phospho-1-deoxy-D-ribulos-1-ylamino)methylideneamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate-lyase (L-glutamine-hydrolysing; 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00340+R04558      Histidine metabolism\n",
      "R01167: L-histidine carboxy-lyase (histamine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00340+R01167      Histidine metabolism\n",
      "R04880: 3,4-dihydroxyphenylethyleneglycol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00350+R04880      Tyrosine metabolism\n",
      "R00734: L-tyrosine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R00734      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00350+R00734      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00400+R00734      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00401+R00734      Novobiocin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00950+R00734      Isoquinoline alkaloid biosynthesis\n",
      "R01085: 3-fumarylpyruvate fumarylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00350+R01085      Tyrosine metabolism\n",
      "R00694: L-Phenylalanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00360+R00694      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00400+R00694      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00960+R00694      Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "R01582: D-Phenylalanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00360+R01582      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00470+R01582      D-Amino acid metabolism\n",
      "R06783: trans-cinnamate,NADH:oxygen oxidoreductase (2,3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00360+R06783      Phenylalanine metabolism\n",
      "R06782: 3-phenylpropanoate,NADH:oxygen oxidoreductase (2,3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00360+R06782      Phenylalanine metabolism\n",
      "R01826: Phosphoenolpyruvate:D-erythrose-4-phosphate C-(1-carboxyvinyl)transferase (phosphate hydrolysing, 2-carboxy-2-oxoethyl-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01826      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01714: 5-O-(1-Carboxyvinyl)-3-phosphoshikimate phosphate-lyase (chorismate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01714      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02340: (1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate D-glyceraldehyde-3-phosphate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00400+R02340      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00402+R02340      Benzoxazinoid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00999+R02340      Biosynthesis of various plant secondary metabolites\n",
      "R00674: L-serine hydro-lyase (adding indole; L-tryptophan-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00674      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03508: 1-(2-Carboxyphenylamino)-1-deoxy-D-ribulose-5-phosphate carboxy-lyase(cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00400+R03508      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03509: N-(5-Phospho-beta-D-ribosyl)anthranilate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00400+R03509      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01073: N-(5-Phospho-D-ribosyl)anthranilate:pyrophosphate phosphoribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00400+R01073      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R00985: chorismate pyruvate-lyase (amino-accepting; anthranilate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00985      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01715: Chorismate pyruvatemutase\n",
      "     https://www.kegg.jp/pathway/rn00400+R01715      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01373: prephenate hydro-lyase (decarboxylating; phenylpyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01373      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R00691: L-arogenate hydro-lyase (decarboxylating; L-phenylalanine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00691      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01728: prephenate:NAD+ oxidoreductase(decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01728      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00401+R01728      Novobiocin biosynthesis\n",
      "R03460: phosphoenolpyruvate:3-phosphoshikimate 5-O-(1-carboxyvinyl)-transferase\n",
      "     https://www.kegg.jp/pathway/rn00400+R03460      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02412: ATP:shikimate 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00400+R02412      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02413: Shikimate:NADP+ 3-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00400+R02413      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03084: 3-Dehydroquinate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00400+R03084      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03083: 2-Dehydro-3-deoxy-D-arabino-heptonate 7-phosphate phosphate-lyase (cyclyzing)\n",
      "     https://www.kegg.jp/pathway/rn00400+R03083      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02473: (R)-Pantoate:beta-alanine ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00410+R02473      beta-Alanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00770+R02473      Pantothenate and CoA biosynthesis\n",
      "R00489: L-aspartate 1-carboxy-lyase (beta-alanine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00410+R00489      beta-Alanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00770+R00489      Pantothenate and CoA biosynthesis\n",
      "R00315: ATP:acetate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00430+R00315      Taurine and hypotaurine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00620+R00315      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00315      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00315      Carbon fixation pathways in prokaryotes\n",
      "R03599: L-selenocysteine selenide-lyase (L-alanine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00450+R03599      Selenocompound metabolism\n",
      "R04929: ATP:selenate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00450+R04929      Selenocompound metabolism\n",
      "R04941: Selenocystathionine L-homocysteine-lyase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00450+R04941      Selenocompound metabolism\n",
      "R09365: 5-methyltetrahydropteroyltri-L-glutamate:L-selenohomocysteine Se-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00450+R09365      Selenocompound metabolism\n",
      "R03970: (5-glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03970      Cyanoamino acid metabolism\n",
      "R10040: beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R10040      Cyanoamino acid metabolism\n",
      "R03971: gamma-glutamyl-beta-aminopropiononitrile amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03971      Cyanoamino acid metabolism\n",
      "R10039: beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R10039      Cyanoamino acid metabolism\n",
      "R10035: beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R10035      Cyanoamino acid metabolism\n",
      "R02558: (R)-Prunasin beta-D-glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R02558      Cyanoamino acid metabolism\n",
      "R02985: amygdalin beta-glucosidase\n",
      "     https://www.kegg.jp/pathway/rn00460+R02985      Cyanoamino acid metabolism\n",
      "R00260: glutamate racemase\n",
      "     https://www.kegg.jp/pathway/rn00470+R00260      D-Amino acid metabolism\n",
      "R01150: D-alanine:D-alanine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00470+R01150      D-Amino acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R01150      Peptidoglycan biosynthesis\n",
      "R00401: alanine racemase\n",
      "     https://www.kegg.jp/pathway/rn00470+R00401      D-Amino acid metabolism\n",
      "R02783: UDP-N-acetylmuramoyl-L-alanine:D-glutamate ligase(ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00470+R02783      D-Amino acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R02783      Peptidoglycan biosynthesis\n",
      "R01148: D-Alanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R01148      D-Amino acid metabolism\n",
      "R03916: (5-glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R03916      Glutathione metabolism\n",
      "R00274: glutathione:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00274      Glutathione metabolism\n",
      "R00497: gamma-L-glutamyl-L-cysteine:glycine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00497      Glutathione metabolism\n",
      "R00894: L-glutamate:L-cysteine gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00894      Glutathione metabolism\n",
      "R00494: glutathione gamma-glutamylaminopeptidase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00494      Glutathione metabolism\n",
      "R00899: L-cysteinylglycine dipeptidase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00899      Glutathione metabolism\n",
      "R08359: S-adenosylmethioninamine:cadaverine 3-aminopropyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R08359      Glutathione metabolism\n",
      "R00948: ATP:alpha-D-glucose-1-phosphate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00948      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00948      Amino sugar and nucleotide sugar metabolism\n",
      "R08639: alpha-D-glucose 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00500+R08639      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00521+R08639      Streptomycin biosynthesis\n",
      "R00299: ATP:D-glucose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00299      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00521+R00299      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00524+R00299      Neomycin, kanamycin and gentamicin biosynthesis\n",
      "R00771: D-glucose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00771      Starch and sucrose metabolism\n",
      "R00028: maltose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00028      Starch and sucrose metabolism\n",
      "R02111: 1,4-alpha-D-Glucan:orthophosphate alpha-D-glucosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R02111      Starch and sucrose metabolism\n",
      "R02421\n",
      "     no rxn\n",
      "R00416: UTP:N-acetyl-alpha-D-glucosamine-1-phosphate uridylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00416      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00416      O-Antigen nucleotide sugar biosynthesis\n",
      "R02060: D-Glucosamine 1-phosphate 1,6-phosphomutase;\n",
      "     https://www.kegg.jp/pathway/rn00520+R02060      Amino sugar and nucleotide sugar metabolism\n",
      "R05332: Acetyl-CoA:D-glucosamine-1-phosphate N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R05332      Amino sugar and nucleotide sugar metabolism\n",
      "R00660: Phosphoenolpyruvate:UDP-N-acetyl-D-glucosamine 1-carboxyvinyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00660      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R00660      Peptidoglycan biosynthesis\n",
      "R03191: UDP-N-acetylmuramate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00520+R03191      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R03191      Peptidoglycan biosynthesis\n",
      "R00022: chitobiose N-acetylglucosaminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00022      Amino sugar and nucleotide sugar metabolism\n",
      "R01384: UDP-D-glucuronate carboxy-lyase (UDP-D-xylose-forming)\n",
      "     https://www.kegg.jp/pathway/rn00520+R01384      Amino sugar and nucleotide sugar metabolism\n",
      "R08968: ATP:N-acetyl-D-glucosamine 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R08968      Amino sugar and nucleotide sugar metabolism\n",
      "R11024: ATP:N-acetyl-D-muramate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R11024      Amino sugar and nucleotide sugar metabolism\n",
      "R11025: UTP:N-acetyl-alpha-D-muramate-1-phosphate uridylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R11025      Amino sugar and nucleotide sugar metabolism\n",
      "R01187: 1D-myo-inositol 3-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00521+R01187      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00562+R01187      Inositol phosphate metabolism\n",
      "R02777: dTDP-beta-L-rhamnose:NADP+ 4-oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00521+R02777      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R02777      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R02777      O-Antigen nucleotide sugar biosynthesis\n",
      "R06514: dTDP-4-dehydro-6-deoxy-alpha-D-glucose 3,5-epimerase;\n",
      "     https://www.kegg.jp/pathway/rn00521+R06514      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R06514      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R06514      O-Antigen nucleotide sugar biosynthesis\n",
      "R06513: dTDPglucose 4,6-hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00521+R06513      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R06513      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00525+R06513      Acarbose and validamycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R06513      O-Antigen nucleotide sugar biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn01055+R06513      Biosynthesis of vancomycin group antibiotics\n",
      "R02328: dTTP:alpha-D-glucose-1-phosphate thymidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00521+R02328      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R02328      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00525+R02328      Acarbose and validamycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R02328      O-Antigen nucleotide sugar biosynthesis\n",
      "R03351: CTP:3-deoxy-D-manno-octulosonate cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R03351      Lipopolysaccharide biosynthesis\n",
      "R03350: 3-Deoxy-D-manno-octulosonate-8-phosphate 8-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R03350      Lipopolysaccharide biosynthesis\n",
      "R04657: ATP:2,3,2',3'-tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-beta-D-1,6-glucosaminyl-alpha-phosphate 4-O'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04657      Lipopolysaccharide biosynthesis\n",
      "R04606: UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine:2,3-bis-(3-hydroxytetradecanoyl)-alpha-D-glucosaminyl-1-phosphate 2,3-bis(3-hydroxytetradecanoyl)-glucosaminyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04606      Lipopolysaccharide biosynthesis\n",
      "R05645: D-glycero-beta-D-manno-heptose-7-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05645      Lipopolysaccharide biosynthesis\n",
      "R05647: D-glycero-beta-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05647      Lipopolysaccharide biosynthesis\n",
      "R05646: ATP:D-glycero-beta-D-manno-heptose 7-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05646      Lipopolysaccharide biosynthesis\n",
      "R05644: ATP:D-glycero-beta-D-manno-heptose 1-phosphate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05644      Lipopolysaccharide biosynthesis\n",
      "R05074: CMP-3-deoxy-D-manno-oct-2-ulosonate:(KDO)-lipid IVA 3-deoxy-D-manno-oct-2-ulosonate transferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05074      Lipopolysaccharide biosynthesis\n",
      "R04658: CMP-3-deoxy-D-manno-oct-2-ulosonate:lipid IVA 3-deoxy-D-manno-oct-2-ulosonate transferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04658      Lipopolysaccharide biosynthesis\n",
      "R04549: UDP-2,3-bis[(3R)-3-hydroxymyristoyl]-alpha-D-glucosamine 2,3-bis[(3R)-3-hydroxymyristoyl]-beta-D-glucosaminyl 1-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04549      Lipopolysaccharide biosynthesis\n",
      "R03254: phosphoenolpyruvate:D-arabinose-5-phosphate C-(1-carboxyvinyl)transferase (phosphate-hydrolysing, 2-carboxy-2-oxoethyl-forming);\n",
      "     https://www.kegg.jp/pathway/rn00540+R03254      Lipopolysaccharide biosynthesis\n",
      "R04550: (3R)-3-hydroxymyristoyl-[acyl-carrier protein]:UDP-3-O-[(3R)-3-hydroxymyristoyl]-alpha-D-glucosamine N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04550      Lipopolysaccharide biosynthesis\n",
      "R04587: UDP-3-O-[(3R)-3-hydroxymyristoyl]-N-acetylglucosamine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04587      Lipopolysaccharide biosynthesis\n",
      "R04567: (R)-3-Hydroxytetradecanoly-[acyl-carrier-protein]:UDP-N-acetyl-glucosamine 3-O-(3-hydroxytetradecanoyl)transferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04567      Lipopolysaccharide biosynthesis\n",
      "R09771: D-glycero-alpha-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R09771      Lipopolysaccharide biosynthesis\n",
      "R09769: D-glycero-alpha-D-manno-heptose 7-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R09769      Lipopolysaccharide biosynthesis\n",
      "R01530: D-arabinose-5-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R01530      Lipopolysaccharide biosynthesis\n",
      "R05627: undecaprenyl-diphosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05627      Peptidoglycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00552+R05627      Teichoic acid biosynthesis\n",
      "R05662: UDP-N-acetyl-D-glucosamine:N-acetyl-alpha-D-muramyl(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol 4-beta-N-acetylglucosaminlytransferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05662      Peptidoglycan biosynthesis\n",
      "R05629: UDPMurAc(oyl-L-Ala-D-gamma-Glu-L-Lys-D-Ala-D-Ala):undecaprenyl-phosphate phospho-N-acetylmuramoyl-pentapeptide-transferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05629      Peptidoglycan biosynthesis\n",
      "R04573: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine:alanyl-D-alanine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00550+R04573      Peptidoglycan biosynthesis\n",
      "R03193: UDP-N-acetylmuramate:L-alanine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00550+R03193      Peptidoglycan biosynthesis\n",
      "R05630: UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine:undecaprenyl-phosphate phospho-N-acetylmuramoyl-pentapeptide-transferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05630      Peptidoglycan biosynthesis\n",
      "R05032: UDP-N-acetyl-D-glucosamine:undecaprenyl-diphospho-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine 4-beta-N-acetylglucosaminlytransferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05032      Peptidoglycan biosynthesis\n",
      "R06447: (2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 8 isopentenyl units)\n",
      "     https://www.kegg.jp/pathway/rn00550+R06447      Peptidoglycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00900+R06447      Terpenoid backbone biosynthesis\n",
      "R02241: acyl-CoA:1-acyl-sn-glycerol-3-phosphate 2-O-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02241      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R02241      Glycerophospholipid metabolism\n",
      "R02240: ATP:1,2-diacylglycerol 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02240      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R02240      Glycerophospholipid metabolism\n",
      "R01186: 1D-myo-inositol 4-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00562+R01186      Inositol phosphate metabolism\n",
      "R01185: 1D-myo-inositol 1-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00562+R01185      Inositol phosphate metabolism\n",
      "R01470: sn-Glycero-3-phosphoethanolamine glycerophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01470      Glycerophospholipid metabolism\n",
      "R00842: sn-Glycerol-3-phosphate:NAD+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00564+R00842      Glycerophospholipid metabolism\n",
      "R02029: Phosphatidylglycerophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02029      Glycerophospholipid metabolism\n",
      "R01799: CTP:phosphatidate cytidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01799      Glycerophospholipid metabolism\n",
      "R02055: Phsophatidyl-L-serine carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02055      Glycerophospholipid metabolism\n",
      "R01320: S-adenosyl-L-methionine:phosphatidyl-N-dimethylethanolamine N-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01320      Glycerophospholipid metabolism\n",
      "R01030: sn-Glycero-3-phosphocholine glycerophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01030      Glycerophospholipid metabolism\n",
      "R03424: S-adenosyl-L-methionine:phosphatidyl-N-methylethanolamine N-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R03424      Glycerophospholipid metabolism\n",
      "R02056: S-adenosyl-L-methionine:phosphatidylethanolamine N-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02056      Glycerophospholipid metabolism\n",
      "R01801: CDP-diacylglycerol:sn-glycerol-3-phosphate 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01801      Glycerophospholipid metabolism\n",
      "R02030: CDP-diacylglycerol:phosphatidylglycerol 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02030      Glycerophospholipid metabolism\n",
      "R02530: (R)-S-Lactoylglutathione methylglyoxal-lyase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00620+R02530      Pyruvate metabolism\n",
      "R01736: (R)-S-Lactoylglutathione hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00620+R01736      Pyruvate metabolism\n",
      "R00214: (S)-malate:NAD+ oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00214      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00214      Carbon fixation in photosynthetic organisms\n",
      "R00236: acetyl adenylate:CoA acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00236      Pyruvate metabolism\n",
      "R00316: ATP:acetate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00316      Pyruvate metabolism\n",
      "R00271: acetyl-CoA:2-oxoglutarate C-acetyltransferase (thioester-hydrolysing, carboxymethyl forming)\n",
      "     https://www.kegg.jp/pathway/rn00300+R00271      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00271      Pyruvate metabolism\n",
      "R00317: Acetyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00317      Pyruvate metabolism\n",
      "R10866: pyruvate:flavodoxin 2-oxidoreductase (CoA-acetylating)\n",
      "     https://www.kegg.jp/pathway/rn00620+R10866      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R10866      Carbon fixation pathways in prokaryotes\n",
      "R03562: Toluene,NADPH:oxogen oxidoreductase (3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00623+R03562      Toluene degradation\n",
      "R02550: toluene, ferredoxin:oxygen oxidoreductase (methyl-hydroxylating);\n",
      "     https://www.kegg.jp/pathway/rn00623+R02550      Toluene degradation\n",
      "R05496: reduced ferredoxin:acetylene oxidoreductase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00625+R05496      Chloroalkane and chloroalkene degradation\n",
      "R05233: trans-3-Chloro-2-propene-1-ol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00625+R05233      Chloroalkane and chloroalkene degradation\n",
      "R05234: cis-3-chloro-2-propene-1-ol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00625+R05234      Chloroalkane and chloroalkene degradation\n",
      "R09128: 2-chloroethanol:cytochrome c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00625+R09128      Chloroalkane and chloroalkene degradation\n",
      "R06917: 1-hydroxymethylnaphthalene:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00626+R06917      Naphthalene degradation\n",
      "R06927: (2-naphthyl)methanol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00626+R06927      Naphthalene degradation\n",
      "R01421: Benzoyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R01421      Aminobenzoate degradation\n",
      "R00519: formate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00630+R00519      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00519      Methane metabolism\n",
      "R00473: L-malyl-CoA glyoxylate-lyase (acetyl-CoA-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00473      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00473      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00473      Carbon fixation pathways in prokaryotes\n",
      "R01256: (S)-malate:CoA ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R01256      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01256      Methane metabolism\n",
      "R03140: D-Ribulose 1,5-bisphosphate oxygenase\n",
      "     https://www.kegg.jp/pathway/rn00630+R03140      Glyoxylate and dicarboxylate metabolism\n",
      "R01334: 2-phosphoglycolate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00630+R01334      Glyoxylate and dicarboxylate metabolism\n",
      "R00934: L-erythro-3-methylmalyl-CoA glyoxylate-lyase (propanoyl-CoA-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00934      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R00934      C5-Branched dibasic acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00934      Carbon fixation pathways in prokaryotes\n",
      "R01221: glycine:NAD+ 2-oxidoreductase (tetrahydrofolate-methylene-adding);\n",
      "     https://www.kegg.jp/pathway/rn00630+R01221      Glyoxylate and dicarboxylate metabolism\n",
      "R00470: 4-hydroxy-2-oxoglutarate glyoxylate-lyase (pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00470      Glyoxylate and dicarboxylate metabolism\n",
      "R08034: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08034      Nitrotoluene degradation\n",
      "R08042: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08042      Nitrotoluene degradation\n",
      "R00926: Propionyladenylate:CoA propionyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R00926      Propanoate metabolism\n",
      "R01354: ATP:propanoate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R01354      Propanoate metabolism\n",
      "R01353: ATP:propanoate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R01353      Propanoate metabolism\n",
      "R04425: (2S,3R)-3-Hydroxybutane-1,2,3-tricarboxylate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00640+R04425      Propanoate metabolism\n",
      "R02948: (S)-2-hydroxy-2-methyl-3-oxobutanoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00650+R02948      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R02948      C5-Branched dibasic acid metabolism\n",
      "R02404: Itaconate:CoA ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00660+R02404      C5-Branched dibasic acid metabolism\n",
      "R00936: 5,6,7,8-tetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00936      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00936      Folate biosynthesis\n",
      "R02235: dihydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R02235      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R02235      Folate biosynthesis\n",
      "R07168: 5-methyltetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R07168      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R07168      Carbon fixation pathways in prokaryotes\n",
      "R01655: 5,10-Methenyltetrahydrofolate 5-hydrolase (decyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00670+R01655      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R01655      Carbon fixation pathways in prokaryotes\n",
      "R02301: 5-Formyltetrahydrofolate cyclo-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00670+R02301      One carbon pool by folate\n",
      "R02300: S-Aminomethyldihydrolipoylprotein:(6S)-tetrahydrofolate aminomethyltransferase (ammonia-forming)\n",
      "     https://www.kegg.jp/pathway/rn00670+R02300      One carbon pool by folate\n",
      "R00943: Formate:tetrahydrofolate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00670+R00943      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R00943      Carbon fixation pathways in prokaryotes\n",
      "R00937: 5,6,7,8-tetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00937      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00937      Folate biosynthesis\n",
      "R01142: Methane,NADH:oxygen oxidoreductase (hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00680+R01142      Methane metabolism\n",
      "R09518: methane,quinol:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R09518      Methane metabolism\n",
      "R01146: methanol:ferricytochrome-c oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00680+R01146      Methane metabolism\n",
      "R03464: 5,10-Methenyltetrahydromethanopterin 10-hydrolase (decyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00680+R03464      Methane metabolism\n",
      "R03390: Formylmethanofuran:5,6,7,8-tetrahydromethanopterin 5-formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R03390      Methane metabolism\n",
      "R08060: \n",
      "     https://www.kegg.jp/pathway/rn00680+R08060      Methane metabolism\n",
      "R03015: formylmethanofuran:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R03015      Methane metabolism\n",
      "R08059: \n",
      "     https://www.kegg.jp/pathway/rn00680+R08059      Methane metabolism\n",
      "R08058: 5,6,7,8-tetrahydromethanopterin hydro-lyase (formaldehyde-adding, 5,10-methylenetetrahydromethanopterin-forming)\n",
      "     https://www.kegg.jp/pathway/rn00680+R08058      Methane metabolism\n",
      "R09099: 5,10-methylenetetrahydromethanopterin:glycine hydroxymethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R09099      Methane metabolism\n",
      "R00756: ATP:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00756      Methane metabolism\n",
      "R05339: D-arabino-hex-3-ulose-6-phosphate isomerase\n",
      "     https://www.kegg.jp/pathway/rn00680+R05339      Methane metabolism\n",
      "R01068: D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00680+R01068      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R01068      Carbon fixation in photosynthetic organisms\n",
      "R00617: ATP:thiamin-phosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R00617      Thiamine metabolism\n",
      "R07463: glycine:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00730+R07463      Thiamine metabolism\n",
      "R04509: ATP:4-amino-2-methyl-5-phosphomethylpyrimidine phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R04509      Thiamine metabolism\n",
      "R03471: ATP:4-amino-5-hydroxymethyl-2-methylpyrimidine 5-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R03471      Thiamine metabolism\n",
      "R05636: 1-Deoxy-D-xylulose-5-phosphate pyruvate-lyase (carboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00730+R05636      Thiamine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00900+R05636      Terpenoid backbone biosynthesis\n",
      "R03223: 2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:4-methyl-5-(2-phosphoethyl)-thiazole 2-methyl-4-aminopyridine-5-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R03223      Thiamine metabolism\n",
      "R03472: 5-amino-1-(5-phospho-D-ribosyl)imidazole formate-lyase (decarboxylating, 4-amino-2-methyl-5-phosphomethylpyrimidine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00730+R03472      Thiamine metabolism\n",
      "R10712: 2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:2-(2-carboxy-4-methylthiazol-5-yl)ethyl phosphate 2-methyl-4-aminopyrimidine-5-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R10712      Thiamine metabolism\n",
      "R07281: D-ribulose 5-phosphate formate-lyase (L-3,4-dihydroxybutan-2-one 4-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00740+R07281      Riboflavin metabolism\n",
      "R00161: ATP:FMN adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00161      Riboflavin metabolism\n",
      "R00549: ATP:riboflavin 5'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00549      Riboflavin metabolism\n",
      "R00425: GTP 7,8-8,9-dihydrolase (diphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00740+R00425      Riboflavin metabolism\n",
      "     https://www.kegg.jp/pathway/rn00790+R00425      Folate biosynthesis\n",
      "R03459: 2,5-Diamino-6-hydroxy-4-(5-phosphoribosylamino)-pyrimidine 2-aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R03459      Riboflavin metabolism\n",
      "R03458: 5-amino-6-(5-phosphoribitylamino)uracil:NADP+ 1'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00740+R03458      Riboflavin metabolism\n",
      "R04457: 5-amino-6-(D-ribitylamino)uracil butanedionetransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R04457      Riboflavin metabolism\n",
      "R00066: 6,7-Dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl)lumazine 2,3-butanediyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00066      Riboflavin metabolism\n",
      "R00548: riboflavin-5-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00548      Riboflavin metabolism\n",
      "R05086: O-Phospho-4-hydroxy-L-threonine phospho-lyase (adding water)\n",
      "     https://www.kegg.jp/pathway/rn00750+R05086      Vitamin B6 metabolism\n",
      "R05681: NAD-dependent threonine 4-phosphate dehydrogenase\n",
      "     https://www.kegg.jp/pathway/rn00750+R05681      Vitamin B6 metabolism\n",
      "R05085: O-Phospho-4-hydroxy-L-threonine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00750+R05085      Vitamin B6 metabolism\n",
      "R05838: pyridoxine 5'-phosphate synthase\n",
      "     https://www.kegg.jp/pathway/rn00750+R05838      Vitamin B6 metabolism\n",
      "R00481: L-aspartate:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00481      Nicotinate and nicotinamide metabolism\n",
      "R04292: glycerone phosphate:iminosuccinate alkyltransferase (cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00760+R04292      Nicotinate and nicotinamide metabolism\n",
      "R03348: Nicotinate-nucleotide:pyrophosphate phosphoribosyltransferase (carboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00760+R03348      Nicotinate and nicotinamide metabolism\n",
      "R03005: ATP:nicotinamide-nucleotide adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00760+R03005      Nicotinate and nicotinamide metabolism\n",
      "R00257: Deamido-NAD+:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00760+R00257      Nicotinate and nicotinamide metabolism\n",
      "R02322: nicotinamide-D-ribonucleotide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R02322      Nicotinate and nicotinamide metabolism\n",
      "R01724: 5-phospho-alpha-D-ribose 1-diphosphate:nicotinate ligase (ADP, diphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00760+R01724      Nicotinate and nicotinamide metabolism\n",
      "R01268: Nicotinamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R01268      Nicotinate and nicotinamide metabolism\n",
      "R00137: ATP:nicotinamide-nucleotide adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00137      Nicotinate and nicotinamide metabolism\n",
      "R00104: ATP:NAD+ 2'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00104      Nicotinate and nicotinamide metabolism\n",
      "R00112: NADPH:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00112      Nicotinate and nicotinamide metabolism\n",
      "R02323: nicotinamide ribonucleotide phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R02323      Nicotinate and nicotinamide metabolism\n",
      "R03346: nicotinate D-ribonucleotide phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R03346      Nicotinate and nicotinamide metabolism\n",
      "R00130: ATP:dephospho-CoA 3'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R00130      Pantothenate and CoA biosynthesis\n",
      "R02971: ATP:pantothenate 4'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R02971      Pantothenate and CoA biosynthesis\n",
      "R03269: N-[(R)-4'-Phosphopantothenoyl]-L-cysteine carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03269      Pantothenate and CoA biosynthesis\n",
      "R04391: ATP:pantothenate 4'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R04391      Pantothenate and CoA biosynthesis\n",
      "R04231: (R)-4'-phosphopantothenate:L-cysteine ligase\n",
      "     https://www.kegg.jp/pathway/rn00770+R04231      Pantothenate and CoA biosynthesis\n",
      "R03018: ATP:pantothenate 4'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03018      Pantothenate and CoA biosynthesis\n",
      "R02472: (R)-pantoate:NADP+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00770+R02472      Pantothenate and CoA biosynthesis\n",
      "R01226: 5,10-Methylenetetrahydrofolate:3-methyl-2-oxobutanoate hydroxymethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R01226      Pantothenate and CoA biosynthesis\n",
      "R03035: ATP:pantetheine-4'-phosphate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03035      Pantothenate and CoA biosynthesis\n",
      "R01074: biotin:CoA ligase (AMP-forming);\n",
      "     https://www.kegg.jp/pathway/rn00780+R01074      Biotin metabolism\n",
      "R01078: dethiobiotin:sulfur sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R01078      Biotin metabolism\n",
      "R03182: 7,8-Diaminononanoate:carbon-dioxide cyclo-ligase\n",
      "     https://www.kegg.jp/pathway/rn00780+R03182      Biotin metabolism\n",
      "R03231: S-Adenosyl-L-methionine:8-amino-7-oxononanoate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R03231      Biotin metabolism\n",
      "R03210: pimeloyl-CoA:L-alanine C-carboxyhexanoyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00780+R03210      Biotin metabolism\n",
      "R09725: pimeloyl-[acyl-carrier protein]-methyl-ester hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00780+R09725      Biotin metabolism\n",
      "R10124: pimeloyl-[acyl-carrier protein]:L-alanine C-pimeloyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00780+R10124      Biotin metabolism\n",
      "R09543: S-adenosyl-L-methionine:malonyl-[acyl-carrier protein] O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R09543      Biotin metabolism\n",
      "R10116: 3-hydroxyglutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10116      Biotin metabolism\n",
      "R10117: 3-hydroxyglutaryl-[acp]-methyl-ester hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10117      Biotin metabolism\n",
      "R10118: glutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10118      Biotin metabolism\n",
      "R10119: glutaryl-[acp]-methyl-ester:malonyl-[acyl-carrier protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00780+R10119      Biotin metabolism\n",
      "R10120: 3-hydroxypimeloyl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10120      Biotin metabolism\n",
      "R10121: 3-hydroxypimeloyl-[acp]-methyl-ester hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10121      Biotin metabolism\n",
      "R10122: pimelyl-[acyl-carrier protein]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10122      Biotin metabolism\n",
      "R10115: malonyl-[acyl-carrier protein]-methyl-ester:malonyl-[acyl-carrier protein] C-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10115      Biotin metabolism\n",
      "R01716: chorismate:L-glutamine aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R01716      Folate biosynthesis\n",
      "R05553: 4-amino-4-deoxychorismate pyruvate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R05553      Folate biosynthesis\n",
      "R00942: Tetrahydrofolate:L-glutamate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R00942      Folate biosynthesis\n",
      "R02237: 7,8-dihydropteroate:L-glutamate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R02237      Folate biosynthesis\n",
      "R03067: (7,8-dihydropterin-6-yl)methyl diphosphate:4-aminobenzoate 2-amino-4-hydroxy-7,8-dihydropteridine-6-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R03067      Folate biosynthesis\n",
      "R03066: 6-(hydroxymethyl)-7,8-dihydropterin:4-aminobenzoate 2-amino-4-hydroxydihydropteridine-6-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R03066      Folate biosynthesis\n",
      "R03503: ATP:6-hydroxymethyl-7,8-dihydropterin 6'-diphosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R03503      Folate biosynthesis\n",
      "R03504: 7,8-dihydroneopterin glycolaldehyde-lyase [6-(hydroxymethyl)-7,8-dihydropterin-forming]\n",
      "     https://www.kegg.jp/pathway/rn00790+R03504      Folate biosynthesis\n",
      "R04639: 2-amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl) dihydropteridine triphosphate hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00790+R04639      Folate biosynthesis\n",
      "R05048: 2,5-diaminopyrimidine nucleoside triphosphate mutase\n",
      "     https://www.kegg.jp/pathway/rn00790+R05048      Folate biosynthesis\n",
      "R05046: formamidopyrimidine nucleoside triphosphate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00790+R05046      Folate biosynthesis\n",
      "R04620: 7,8-dihydroneopterin-3'-triphosphate phosphohydrolase (alkaline optimum)\n",
      "     https://www.kegg.jp/pathway/rn00790+R04620      Folate biosynthesis\n",
      "R04286: 2-amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl) dihydropteridine triphosphate lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R04286      Folate biosynthesis\n",
      "R09394: GTP 3',8-cyclase [(8S)-3',8-cyclo-7,8-dihydroguanosine 5'-triphosphate-forming]\n",
      "     https://www.kegg.jp/pathway/rn00790+R09394      Folate biosynthesis\n",
      "R09959: 7,8-dihydroneopterin 3'-triphosphate acetaldehyde-lyase (6-carboxy-5,6,7,8-tetrahydropterin and triphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09959      Folate biosynthesis\n",
      "R10002: 6-carboxy-5,6,7,8-tetrahydropterin ammonia-lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R10002      Folate biosynthesis\n",
      "R09978: 7-carboxy-7-carbaguanine:ammonia ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09978      Folate biosynthesis\n",
      "R07605: 7-aminomethyl-7-carbaguanine:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R07605      Folate biosynthesis\n",
      "R00428: GTP 8,9-hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00790+R00428      Folate biosynthesis\n",
      "R06895: coproporphyrinogen-III:S-adenosyl-L-methionine oxidoreductase(decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R06895      Porphyrin metabolism\n",
      "R03220: Coproporphyrinogen:oxygen oxidoreductase(decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R03220      Porphyrin metabolism\n",
      "R07411: (2E,6E)-farnesyl-diphosphate:protoheme IX farnesyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07411      Porphyrin metabolism\n",
      "R00310: protoheme ferro-lyase (protoporphyrin-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R00310      Porphyrin metabolism\n",
      "R00078: Fe(II):oxygen oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00860+R00078      Porphyrin metabolism\n",
      "R03197: Uroporphyrinogen-III carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03197      Porphyrin metabolism\n",
      "R03165: Hydroxymethylbilane hydro-lyase(cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00860+R03165      Porphyrin metabolism\n",
      "R03194: S-adenosyl-L-methionine:uroporphyrinogen-III C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03194      Porphyrin metabolism\n",
      "R04972: Uroporphyrinogen I carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R04972      Porphyrin metabolism\n",
      "R00084: porphobilinogen:(4-[2-carboxyethyl]-3-[carboxymethyl]pyrrol-2-yl)methyltransferase (hydrolysing);\n",
      "     https://www.kegg.jp/pathway/rn00860+R00084      Porphyrin metabolism\n",
      "R00036: 5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing; porphobilinogen-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R00036      Porphyrin metabolism\n",
      "R02272: (S)-4-Amino-5-oxopentanoate 4,5-aminomutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R02272      Porphyrin metabolism\n",
      "R02864: S-Adenosyl-L-methionine:uroporphyrin-III C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R02864      Porphyrin metabolism\n",
      "R03947: S-adenosyl-L-methionine:uroporphyrin-III C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03947      Porphyrin metabolism\n",
      "R05807: sirohydrochlorin cobalt-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05807      Porphyrin metabolism\n",
      "R05808: S-adenosyl-L-methionine:cobalt-factor-II C20-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05808      Porphyrin metabolism\n",
      "R05810: cobalt-precorrin-4 methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05810      Porphyrin metabolism\n",
      "R03948: S-adenosyl-L-methionine:precorrin-4 C20-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03948      Porphyrin metabolism\n",
      "R05180: S-adenosyl-L-methionine:precorrin-3B C17-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05180      Porphyrin metabolism\n",
      "R05181: S-adenosyl-L-methionine:precorrin-4 C11 methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05181      Porphyrin metabolism\n",
      "R05177: precorrin 8X 11,12-methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05177      Porphyrin metabolism\n",
      "R05224: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05224      Porphyrin metabolism\n",
      "R05814: precorrin-8 11,12-methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05814      Porphyrin metabolism\n",
      "R05815: cobyrinate:L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05815      Porphyrin metabolism\n",
      "R05220: ATP:cob(I)yrinic acid-a,c-diamide Cobeta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05220      Porphyrin metabolism\n",
      "R05225: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05225      Porphyrin metabolism\n",
      "R07268: ATP:cobinamide Cobeta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07268      Porphyrin metabolism\n",
      "R05221: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05221      Porphyrin metabolism\n",
      "R01492: ATP:cob(I)alamin Co-beta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R01492      Porphyrin metabolism\n",
      "R05222: GTP:adenosylcobinamide-phosphate guanylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05222      Porphyrin metabolism\n",
      "R05223: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05223      Porphyrin metabolism\n",
      "R04594: \n",
      "     https://www.kegg.jp/pathway/rn00860+R04594      Porphyrin metabolism\n",
      "R04148: Nicotinate-nucleotide:dimethylbenzimidazole phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R04148      Porphyrin metabolism\n",
      "R07412: ferroheme o:acceptor C-81-oxidoreductase (heme a-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07412      Porphyrin metabolism\n",
      "R07302: adenosylcobyric acid:(R)-1-aminopropan-2-ol ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07302      Porphyrin metabolism\n",
      "R06529: adenosylcobyric acid:(R)-1-aminopropan-2-yl phosphate ligase\n",
      "     https://www.kegg.jp/pathway/rn00860+R06529      Porphyrin metabolism\n",
      "R05149: S-Adenosyl-L-methionine:1-precorrin-6Y C5,15-methyltransferase (C-12-decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05149      Porphyrin metabolism\n",
      "R07773: S-adenosyl-L-methionine:cobalt-precorrin-5B (C1)-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07773      Porphyrin metabolism\n",
      "R05688: 1-Deoxy-D-xylulose-5-phosphate isomeroreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05688      Terpenoid backbone biosynthesis\n",
      "R05633: CTP: 2-C-Methyl-D-erythritol 4-phosphate cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05633      Terpenoid backbone biosynthesis\n",
      "R05634: ATP:4-(Cytidine 5'-diphospho)-2-C-methyl-D-erythritol 2-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05634      Terpenoid backbone biosynthesis\n",
      "R05637: 2-Phospho-4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol CMP-lyase (cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00900+R05637      Terpenoid backbone biosynthesis\n",
      "R08689: (E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized ferredoxin oxidoreductase (hydrating)\n",
      "     https://www.kegg.jp/pathway/rn00900+R08689      Terpenoid backbone biosynthesis\n",
      "R08210: dimethylallyl diphosphate:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R08210      Terpenoid backbone biosynthesis\n",
      "R05884: isopentenyl-diphosphate:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05884      Terpenoid backbone biosynthesis\n",
      "R01658: Dimethylallyl-diphosphate:isopentenyl-diphosphate dimethylallyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R01658      Terpenoid backbone biosynthesis\n",
      "R02003: Geranyl-diphosphate:isopentenyl-diphosphate geranyltrans-transferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R02003      Terpenoid backbone biosynthesis\n",
      "R09248: (2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 5 isopentenyl units)\n",
      "     https://www.kegg.jp/pathway/rn00900+R09248      Terpenoid backbone biosynthesis\n",
      "R10859: (E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized flavodoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R10859      Terpenoid backbone biosynthesis\n",
      "R00702: farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00100+R00702      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00906+R00702      Carotenoid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R00702      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R07322: squalene mutase (cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00909+R07322      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R07323: squalene mutase (cyclizing);\n",
      "     https://www.kegg.jp/pathway/rn00909+R07323      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R10164: hydroxylamine:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R10164      Nitrogen metabolism\n",
      "R00787: ammonia:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00787      Nitrogen metabolism\n",
      "R05185: reduced ferredoxin:dinitrogen oxidoreductase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00910+R05185      Nitrogen metabolism\n",
      "R00294: nitrous-oxide:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00294      Nitrogen metabolism\n",
      "R00148: ammonia,ubiquinol:oxygen oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00910+R00148      Nitrogen metabolism\n",
      "R10092: carbonate hydro-lyase (carbon-dioxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00910+R10092      Nitrogen metabolism\n",
      "R00858: hydrogen-sulfide:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00858      Sulfur metabolism\n",
      "R00508: 3'-phospho-5'-adenylyl sulfate 3'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00508      Sulfur metabolism\n",
      "R02021: adenosine 3',5'-bisphosphate,sulfite:oxidized-thioredoxin oxidoreductase (3'-phosphoadenosine-5'-phosphosulfate -forming)\n",
      "     https://www.kegg.jp/pathway/rn00920+R02021      Sulfur metabolism\n",
      "R01931: thiosulfate:cyanide sulfurtranserase\n",
      "     https://www.kegg.jp/pathway/rn00920+R01931      Sulfur metabolism\n",
      "R10151: \n",
      "     https://www.kegg.jp/pathway/rn00920+R10151      Sulfur metabolism\n",
      "R10907: beta-D-glucose-6-phosphate:NAD+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00030+R10907      Pentose phosphate pathway\n",
      "R03237: CTP:D-tagatose 6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03237      Galactose metabolism\n",
      "R04970: hexadecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04970      Fatty acid biosynthesis\n",
      "R04967: tetradecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04967      Fatty acid biosynthesis\n",
      "R04725: dodecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04725      Fatty acid biosynthesis\n",
      "R04962: decanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04962      Fatty acid biosynthesis\n",
      "R04959: octanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04959      Fatty acid biosynthesis\n",
      "R04956: hexanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04956      Fatty acid biosynthesis\n",
      "R04430: butyryl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04430      Fatty acid biosynthesis\n",
      "R00248: L-glutamate:NADP+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00248      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00248      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00248      Nitrogen metabolism\n",
      "R01231: Xanthosine-5'-phosphate:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01231      Purine metabolism\n",
      "R00573: UTP:L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00573      Pyrimidine metabolism\n",
      "R01775: L-homoserine:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01775      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R01775      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R01775      Lysine biosynthesis\n",
      "R04199: 2,3,4,5-tetrahydrodipicolinate:NADP+ 4-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00261+R04199      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00300+R04199      Lysine biosynthesis\n",
      "R01251: L-Proline:NADP+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01251      Arginine and proline metabolism\n",
      "R03293: trans-4-Hydroxy-L-proline:NADP+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03293      Arginine and proline metabolism\n",
      "R00986: Chorismate pyruvate-lyase (amino-accepting)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00986      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00405+R00986      Phenazine biosynthesis\n",
      "R03192: UDP-N-acetylmuramate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00520+R03192      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R03192      Peptidoglycan biosynthesis\n",
      "R00844: sn-Glycerol-3-phosphate:NADP+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00564+R00844      Glycerophospholipid metabolism\n",
      "R00939: 5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00939      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00939      Folate biosynthesis\n",
      "R02236: dihydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R02236      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R02236      Folate biosynthesis\n",
      "R00940: 5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00940      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00940      Folate biosynthesis\n",
      "R06558: \n",
      "     https://www.kegg.jp/pathway/rn00860+R06558      Porphyrin metabolism\n",
      "R03370: stearoyl-[acyl-carrier protein],reduced-ferredoxin:oxygen oxidoreductase (9,10 cis-dehydrogenating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R03370      Fatty acid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn01040+R03370      Biosynthesis of unsaturated fatty acids\n",
      "R03238: UTP:D-tagatose 6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03238      Galactose metabolism\n",
      "R03239: ITP:D-tagatose 6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03239      Galactose metabolism\n"
     ]
    }
   ],
   "source": [
    "oldinmodel = [r for r in oldkegglist if r in mcoosmodel_rxnlist]\n",
    "print(len(oldinmodel))\n",
    "print('=====================')\n",
    "for r in oldinmodel:\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "211\n",
      "=====================\n",
      "R00746: Ethanol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00746      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00746      Pyruvate metabolism\n",
      "R03270: \n",
      "     https://www.kegg.jp/pathway/rn00010+R03270      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R03270      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R03270      Pyruvate metabolism\n",
      "R02569: acetyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02569      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R02569      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R02569      Pyruvate metabolism\n",
      "R07618: enzyme N6-(dihydrolipoyl)lysine:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R07618      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R07618      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00280+R07618      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00620+R07618      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00640+R07618      Propanoate metabolism\n",
      "R02073: diphosphate:beta-D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02073      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R02073      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R02073      Fructose and mannose metabolism\n",
      "R03316: \n",
      "     https://www.kegg.jp/pathway/rn00020+R03316      Citrate cycle (TCA cycle)\n",
      "R02570: succinyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-succinyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00020+R02570      Citrate cycle (TCA cycle)\n",
      "R01430: D-xylose:NADP+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00040+R01430      Pentose and glucuronate interconversions\n",
      "R00867: ATP:D-fructose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00051+R00867      Fructose and mannose metabolism\n",
      "R01105\n",
      "     no rxn\n",
      "R02767: (3R)-3-hydroxyacyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R02767      Fatty acid biosynthesis\n",
      "R01403: acyl-[acyl-carrier-protein]:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01403      Fatty acid biosynthesis\n",
      "R02768: acyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R02768      Fatty acid biosynthesis\n",
      "R00390: long-chain-fatty-acid:CoA ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00061+R00390      Fatty acid biosynthesis\n",
      "R10208: (3R)-3-hydroxyacyl-[acyl-carrier protein] hydro-lyase (trans-2-enoyl-[acyl-carrier protein]-forming)\n",
      "     https://www.kegg.jp/pathway/rn00061+R10208      Fatty acid biosynthesis\n",
      "R01404: acyl-[acyl-carrier protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01404      Fatty acid biosynthesis\n",
      "R02281: Alkane,reduced-rubredoxin:oxygen 1-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00071+R02281      Fatty acid degradation\n",
      "R00623: primary_alcohol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00071+R00623      Fatty acid degradation\n",
      "R02000: rubredoxin:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00071+R02000      Fatty acid degradation\n",
      "R01347: Fatty acid,reduced rubredoxin:oxygen oxidoreductase (omega-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00071+R01347      Fatty acid degradation\n",
      "R12323: lanosterol,reduced ferredoxin:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R12323      Steroid biosynthesis\n",
      "R12657: S-adenosyl-L-methionine:3-demethylubiquinol 3-O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R12657      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R09736: S-adenosyl-L-methione:demethylmenaquinol methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R09736      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R12658: \n",
      "     https://www.kegg.jp/pathway/rn00130+R12658      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R09737: S-adenosyl-L-methionine:2-methoxy-6-all-trans-polyprenyl-1,4-benzoquinol 5-C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R09737      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R12067: 6-methoxy-3-methyl-2-(all-trans-polyprenyl)-1,4-benzoquinol,acceptor:oxygen oxidoreductase (5-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00130+R12067      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R00089: ATP diphosphate-lyase (cyclizing; 3',5'-cyclic-AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R00089      Purine metabolism\n",
      "R00182: AMP phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00182      Purine metabolism\n",
      "R02297: xanthosine:orthophosphate ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02297      Purine metabolism\n",
      "R01560: Adenosine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01560      Purine metabolism\n",
      "R01561: adenosine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01561      Purine metabolism\n",
      "R01676: Guanine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01676      Purine metabolism\n",
      "R02147: guanosine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02147      Purine metabolism\n",
      "R01562: adenosine 3'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01562      Purine metabolism\n",
      "R02148: guanosine 3'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02148      Purine metabolism\n",
      "R00434: GTP diphosphate-lyase (cyclizing; 3',5'-cyclic-GMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R00434      Purine metabolism\n",
      "R00191: adenosine 3',5'-phosphate 5'-nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00191      Purine metabolism\n",
      "R06974: formate:N1-(5-phospho-beta-D-ribosyl)glycinamide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R06974      Purine metabolism\n",
      "R00331: ATP:nucleoside-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00331      Purine metabolism\n",
      "R12851: adenylosuccinate lyase\n",
      "     https://www.kegg.jp/pathway/rn00230+R12851      Purine metabolism\n",
      "R12852: guanylate kinase\n",
      "     https://www.kegg.jp/pathway/rn00230+R12852      Purine metabolism\n",
      "R12853: nucleoside-diphosphate kinase\n",
      "     https://www.kegg.jp/pathway/rn00230+R12853      Purine metabolism\n",
      "R01863: inosine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01863      Purine metabolism\n",
      "R00510: cytidine-5'-monophosphate phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00510      Pyrimidine metabolism\n",
      "R01877: uridine 3'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01877      Pyrimidine metabolism\n",
      "R02370: cytidine 3'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02370      Pyrimidine metabolism\n",
      "R11323: dTTP diphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R11323      Pyrimidine metabolism\n",
      "R02296: cytidine:orthophosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02296      Pyrimidine metabolism\n",
      "R12956: thymidine 5'-monophosphate phosphodeoxyribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R12956      Pyrimidine metabolism\n",
      "R12955: uridine 5'-monophosphate phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R12955      Pyrimidine metabolism\n",
      "R01876: uridine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01876      Pyrimidine metabolism\n",
      "R03425: glycine:lipoylprotein oxidoreductase (decarboxylating and acceptor-aminomethylating)\n",
      "     https://www.kegg.jp/pathway/rn00260+R03425      Glycine, serine and threonine metabolism\n",
      "R04125: [protein]-S8-aminomethyldihydrolipoyllysine:tetrahydrofolate aminomethyltransferase (ammonia-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R04125      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R04125      One carbon pool by folate\n",
      "R03815: dihydrolipoylprotein:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R03815      Glycine, serine and threonine metabolism\n",
      "R07650: L-2,4-diaminobutanoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00260+R07650      Glycine, serine and threonine metabolism\n",
      "R01402: S-methyl-5'-thioadenosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01402      Cysteine and methionine metabolism\n",
      "R04858: S-adenosyl-L-methionine:DNA (cytosine-5-)-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04858      Cysteine and methionine metabolism\n",
      "R01776: Acetyl-CoA:L-homoserine O-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01776      Cysteine and methionine metabolism\n",
      "R07274: O-phospho-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R07274      Cysteine and methionine metabolism\n",
      "R00193: S-Adenosyl-L-homocysteine aminohydrolase\n",
      "     https://www.kegg.jp/entry/R00193\n",
      "R05066: 3-hydroxy-2-methylpropanoate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00280+R05066      Valine, leucine and isoleucine degradation\n",
      "R01939: L-2-aminoadipate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R01939      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00310+R01939      Lysine degradation\n",
      "R02851: D-Lysine:2-oxocarboxylate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00310+R02851      Lysine degradation\n",
      "     https://www.kegg.jp/pathway/rn00470+R02851      D-Amino acid metabolism\n",
      "R03180: 4-Guanidinobutanamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03180      Arginine and proline metabolism\n",
      "R00135\n",
      "     no rxn\n",
      "R03342: 3-(4-Hydroxyphenyl)pyruvate keto-enol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00350+R03342      Tyrosine metabolism\n",
      "R01378: Phenylpyruvate keto-enol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00360+R01378      Phenylalanine metabolism\n",
      "R02540: 2-phenylacetamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00360+R02540      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00643+R02540      Styrene degradation\n",
      "R03470: 4-Carboxymuconolactone carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00362+R03470      Benzoate degradation\n",
      "R03028: glutaconyl-1-CoA carboxy-lyase;\n",
      "     https://www.kegg.jp/pathway/rn00362+R03028      Benzoate degradation\n",
      "     https://www.kegg.jp/pathway/rn00650+R03028      Butanoate metabolism\n",
      "R03096: Indole-3-acetamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00380+R03096      Tryptophan metabolism\n",
      "R10597: chorismate pyruvate-lyase (3-hydroxybenzoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R10597      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01687: (5-L-glutamyl)-peptide:taurine 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00430+R01687      Taurine and hypotaurine metabolism\n",
      "R04773: Selenomethionine:tRNAMet ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00450+R04773      Selenocompound metabolism\n",
      "R02924: D-Arginine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R02924      D-Amino acid metabolism\n",
      "R02459: D-Ornithine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R02459      D-Amino acid metabolism\n",
      "R13097: \n",
      "     https://www.kegg.jp/pathway/rn00470+R13097      D-Amino acid metabolism\n",
      "R04951: \n",
      "     https://www.kegg.jp/pathway/rn00480+R04951      Glutathione metabolism\n",
      "R00251: 5-oxo-L-proline amidohydrolase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00251      Glutathione metabolism\n",
      "R01262: glutathione:L-amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R01262      Glutathione metabolism\n",
      "R03522: RX:glutathione R-transferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R03522      Glutathione metabolism\n",
      "R12578: glutathione:hydroperoxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00480+R12578      Glutathione metabolism\n",
      "R02887: 1,4-beta-D-Glucan glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R02887      Starch and sucrose metabolism\n",
      "R00306: cellobiose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00306      Starch and sucrose metabolism\n",
      "R00760: ATP:D-fructose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00760      Starch and sucrose metabolism\n",
      "R03527: D-Glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R03527      Starch and sucrose metabolism\n",
      "R02889\n",
      "     no rxn\n",
      "R02110: 1,4-alpha-D-Glucan:1,4-alpha-D-glucan 6-alpha-D-(1,4-alpha-D-glucano)-transferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R02110      Starch and sucrose metabolism\n",
      "R11310: beta-D-glucose 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00500+R11310      Starch and sucrose metabolism\n",
      "R05196: 1,4-alpha-D-Glucan:1,4-alpha-D-glucan 4-alpha-D-glycosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R05196      Starch and sucrose metabolism\n",
      "R09994: alpha-maltose 1-phosphate:(1->4)-alpha-D-glucan 4-alpha-D-maltosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R09994      Starch and sucrose metabolism\n",
      "R11261: \n",
      "     https://www.kegg.jp/pathway/rn00500+R11261      Starch and sucrose metabolism\n",
      "R11530: ADP-alpha-D-glucose:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (configuration-retaining)\n",
      "     https://www.kegg.jp/pathway/rn00500+R11530      Starch and sucrose metabolism\n",
      "R09304: \n",
      "     https://www.kegg.jp/pathway/rn00514+R09304      Other types of O-glycan biosynthesis\n",
      "R07620: \n",
      "     https://www.kegg.jp/pathway/rn00514+R07620      Other types of O-glycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00515+R07620      Mannose type O-glycan biosynthesis\n",
      "R02334\n",
      "     no rxn\n",
      "R03920: ATP:D-fructose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R03920      Amino sugar and nucleotide sugar metabolism\n",
      "R01206\n",
      "     no rxn\n",
      "R10460: GDP-4-amino-4,6-dideoxy-alpha-D-mannose:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R10460      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R10460      O-Antigen nucleotide sugar biosynthesis\n",
      "R06285: \n",
      "     https://www.kegg.jp/pathway/rn00540+R06285      Lipopolysaccharide biosynthesis\n",
      "R12859: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12859      Teichoic acid biosynthesis\n",
      "R02250: triacylglycerol acylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02250      Glycerolipid metabolism\n",
      "R02687: 1,2-diacyl-sn-glycerol acylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02687      Glycerolipid metabolism\n",
      "R01041: glycerol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01041      Glycerolipid metabolism\n",
      "R01012: phosphoenolpyruvate:glycerone phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01012      Glycerolipid metabolism\n",
      "R12241: acyl phosphoate:sn-glycerol 3-phosphate acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R12241      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R12241      Glycerophospholipid metabolism\n",
      "R12240: acyl-[acyl-carrier protein]:phosphate acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R12240      Glycerolipid metabolism\n",
      "R00848: sn-Glycerol-3-phosphate:(acceptor) 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00564+R00848      Glycerophospholipid metabolism\n",
      "R12212\n",
      "     no rxn\n",
      "R05666: toluene,NADH:oxygen oxidoreductase (4-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00623+R05666      Toluene degradation\n",
      "R01238: 4-hydroxybenzoate carboxy-lyase (phenol-forming)\n",
      "     https://www.kegg.jp/pathway/rn00627+R01238      Aminobenzoate degradation\n",
      "R05590: Acylamide aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R05590      Aminobenzoate degradation\n",
      "R00476: glycolate:acceptor 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00630+R00476      Glyoxylate and dicarboxylate metabolism\n",
      "R08017: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08017      Nitrotoluene degradation\n",
      "R05551: \n",
      "     https://www.kegg.jp/pathway/rn00643+R05551      Styrene degradation\n",
      "R01220: 5,10-methylenetetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R01220      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R01220      Carbon fixation pathways in prokaryotes\n",
      "R03940: 10-Formyltetrahydrofolate:L-methionyl-tRNA N-formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00670+R03940      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00970+R03940      Aminoacyl-tRNA biosynthesis\n",
      "R00134: formate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00134      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00134      Carbon fixation pathways in prokaryotes\n",
      "R10902: tyramine:L-glutamate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00680+R10902      Methane metabolism\n",
      "R10935: \n",
      "     https://www.kegg.jp/pathway/rn00680+R10935      Methane metabolism\n",
      "R11039: [5-(aminomethyl)furan-3-yl]methyl phosphate kinase\n",
      "     https://www.kegg.jp/pathway/rn00680+R11039      Methane metabolism\n",
      "R11040: (4-{4-[2-(gamma-L-glutamylamino)ethyl]phenoxymethyl}furan-2-yl)methanamine synthase\n",
      "     https://www.kegg.jp/pathway/rn00680+R11040      Methane metabolism\n",
      "R11319: ADP:thiamin-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R11319      Thiamine metabolism\n",
      "R02135: thiamin monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00730+R02135      Thiamine metabolism\n",
      "R10247: 1-deoxy-D-xylulose 5-phosphate:thiol sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R10247      Thiamine metabolism\n",
      "R07460: L-cysteine:[enzyme]-cysteine sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R07460      Thiamine metabolism\n",
      "R00615: Thiamin diphosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00730+R00615      Thiamine metabolism\n",
      "R09083: \n",
      "     https://www.kegg.jp/pathway/rn00740+R09083      Riboflavin metabolism\n",
      "R11225: dimethylallyl-phosphate:FMNH2 prenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R11225      Riboflavin metabolism\n",
      "R12896: prenyl diphosphate diphosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R12896      Riboflavin metabolism\n",
      "R03004: Deamino-NAD+ nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R03004      Nicotinate and nicotinamide metabolism\n",
      "R00103: NAD+ phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00103      Nicotinate and nicotinamide metabolism\n",
      "R10633: \n",
      "     https://www.kegg.jp/pathway/rn00760+R10633      Nicotinate and nicotinamide metabolism\n",
      "R03036: Dephospho-CoA nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03036      Pantothenate and CoA biosynthesis\n",
      "R04439: (R)-2,3-dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00770+R04439      Pantothenate and CoA biosynthesis\n",
      "R01625: CoA:apo-[acyl-carrier-protein] pantetheinephosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R01625      Pantothenate and CoA biosynthesis\n",
      "R05145: Biotinyl-5'-AMP:apo-[carboxylase] ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00780+R05145      Biotin metabolism\n",
      "R12934: \n",
      "     https://www.kegg.jp/pathway/rn00780+R12934      Biotin metabolism\n",
      "R12428: octanoyl-[acp]:Glycine-cleavage-system-H N-octanoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12428      Lipoic acid metabolism\n",
      "R12423: [lipoyl-carrier protein E2]-N6-(octanoyl)-L-lysine:[Fe-S] cluster scaffold protein carrying a [4Fe-4S]2+ cluster sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12423      Lipoic acid metabolism\n",
      "R12427: octanoyl-[acp]:lipoyl-carrier-protein-E2 N-octanoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12427      Lipoic acid metabolism\n",
      "R12424: [glycine cleavage system H]-N6-octanoyl-L-lysine:[Fe-S] cluster scaffold protein carrying a [4Fe-4S]2+ cluster sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12424      Lipoic acid metabolism\n",
      "R04241: tetrahydropteroyl-gamma-polyglutamate:L-glutamate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R04241      Folate biosynthesis\n",
      "R09395: thiocarboxylated molybdopterin synthase:cyclic pyranopterin phosphate sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R09395      Folate biosynthesis\n",
      "R09594: tetrahydromethanopterin:alpha-L-glutamate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09594      Folate biosynthesis\n",
      "R11372: (8S)-3',8-cyclo-7,8-dihydroguanosine 5'-triphosphate lyase (cyclic pyranopterin phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R11372      Folate biosynthesis\n",
      "R09726: ATP:molybdopterin adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R09726      Folate biosynthesis\n",
      "R09735: adenylyl-molybdopterin:molybdate molybdate transferase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09735      Folate biosynthesis\n",
      "R11581: GTP:molybdenum cofactor guanylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11581      Folate biosynthesis\n",
      "R04734: 4a-hydroxytetrahydrobiopterin hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R04734      Folate biosynthesis\n",
      "R11037: 7,8-dihydroneopterin:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11037      Folate biosynthesis\n",
      "R11073: 7,8-dihydroneopterin 2'-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11073      Folate biosynthesis\n",
      "R11765: tetrahydrobiopterin:NADP+ oxidoreductase (7,8-dihydrobiopterin-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R11765      Folate biosynthesis\n",
      "R11102: 5-phospho-alpha-D-ribose-1-diphosphate:4-hydroxybenzoate 5-phospho-beta-D-ribofuranosyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00790+R11102      Folate biosynthesis\n",
      "R01812: 5,6,7,8-tetrahydrobiopterin:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R01812      Folate biosynthesis\n",
      "R12644: 7,8-dihydrobiopterin:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R12644      Folate biosynthesis\n",
      "R04109: L-glutamate-semialdehyde: NADP+ oxidoreductase(L-glutamyl-tRNA(Glu)-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R04109      Porphyrin metabolism\n",
      "R05578: L-glutamate:tRNA(Glu) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05578      Porphyrin metabolism\n",
      "     https://www.kegg.jp/pathway/rn00970+R05578      Aminoacyl-tRNA biosynthesis\n",
      "R07775: S-adenosyl-L-methionine:cobalt-precorrin-7 C5-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07775      Porphyrin metabolism\n",
      "R07774: S-adenosyl-L-methionine:cobalt-precorrin-6B C15-methyltransferase (C12-decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07774      Porphyrin metabolism\n",
      "R11329: Fe-coproporphyrin-III ferro-lyase (coproporphyrin-III-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11329      Porphyrin metabolism\n",
      "R11580: S-adenosyl-L-methionine:cobalt-factor III 17-methyltransferase (ring contracting)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11580      Porphyrin metabolism\n",
      "R11816: protoheme,donor:oxygen oxidoreductase (biliverdin-IX-delta-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11816      Porphyrin metabolism\n",
      "R11817: protoheme,donor:oxygen oxidoreductase (biliverdin-IX-beta-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11817      Porphyrin metabolism\n",
      "R12184: ATP:Cob(II)yrinate diamide Cobeta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R12184      Porphyrin metabolism\n",
      "R12605: \n",
      "     https://www.kegg.jp/pathway/rn00860+R12605      Porphyrin metabolism\n",
      "R02061: trans,trans-Farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R02061      Terpenoid backbone biosynthesis\n",
      "R09845: CAAX protease\n",
      "     https://www.kegg.jp/pathway/rn00900+R09845      Terpenoid backbone biosynthesis\n",
      "R12895: \n",
      "     https://www.kegg.jp/pathway/rn00900+R12895      Terpenoid backbone biosynthesis\n",
      "R09793: presqualene-diphosphate diphosphate-lyase (15-cis-4,4'-diapophytoene-forming)\n",
      "     https://www.kegg.jp/pathway/rn00906+R09793      Carotenoid biosynthesis\n",
      "R00798: nitrite:acceptor oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00798      Nitrogen metabolism\n",
      "R09513: methanesulfonate,NADH:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R09513      Sulfur metabolism\n",
      "R01851: Methanethiol:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R01851      Sulfur metabolism\n",
      "R10152: sulfide:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R10152      Sulfur metabolism\n",
      "R09499: hydrogen-sulfide:flavocytochrome c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R09499      Sulfur metabolism\n",
      "R13051: daidzein 7-O-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13051      Degradation of flavonoids\n",
      "R13052: genistein 7-O-beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13052      Degradation of flavonoids\n",
      "R13081: naringenin 7-O-beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13081      Degradation of flavonoids\n",
      "R13065: quercetin 3-O-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13065      Degradation of flavonoids\n",
      "R03656: L-Isoleucine:tRNA(Ile) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03656      Aminoacyl-tRNA biosynthesis\n",
      "R03655: L-Histidine:tRNA(His) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03655      Aminoacyl-tRNA biosynthesis\n",
      "R03660: L-Phenylalanine:tRNA(Ala) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03660      Aminoacyl-tRNA biosynthesis\n",
      "R03646: L-Arginine:tRNA(Arg) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03646      Aminoacyl-tRNA biosynthesis\n",
      "R03650: L-Cysteine:tRNA(Cys) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03650      Aminoacyl-tRNA biosynthesis\n",
      "R03654: glycine:tRNA(Gly) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03654      Aminoacyl-tRNA biosynthesis\n",
      "R05577: L-Aspartate:tRNA(Asp) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R05577      Aminoacyl-tRNA biosynthesis\n",
      "R03662: L-Serine:tRNA(Ser) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03662      Aminoacyl-tRNA biosynthesis\n",
      "R03659: L-Methionine:tRNAMet ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03659      Aminoacyl-tRNA biosynthesis\n",
      "R03665: L-Valine:tRNAVal ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03665      Aminoacyl-tRNA biosynthesis\n",
      "R03038: L-Alanine:tRNA(Ala) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03038      Aminoacyl-tRNA biosynthesis\n",
      "R03658: L-lysine:tRNALys ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03658      Aminoacyl-tRNA biosynthesis\n",
      "R03657: L-Leucine:tRNA(Leu) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03657      Aminoacyl-tRNA biosynthesis\n",
      "R03663: L-Threonine:tRNA(Thr) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03663      Aminoacyl-tRNA biosynthesis\n",
      "R03664: L-Tryptophan -tRNA(Trp) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03664      Aminoacyl-tRNA biosynthesis\n",
      "R02918: L-Tyrosine:tRNA(Tyr) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R02918      Aminoacyl-tRNA biosynthesis\n",
      "R03661: L-proline:tRNA(Pro) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03661      Aminoacyl-tRNA biosynthesis\n",
      "R03905: Glu-tRNA(Gln):L-glutamine amido-ligase(ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03905      Aminoacyl-tRNA biosynthesis\n",
      "R03651: L-Glutamate:tRNA(Gln) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03651      Aminoacyl-tRNA biosynthesis\n",
      "R04212: Asp-tRNA(Asn):L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R04212      Aminoacyl-tRNA biosynthesis\n",
      "R08218: L-serine:tRNASec ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R08218      Aminoacyl-tRNA biosynthesis\n",
      "R04998: beta-D-Glucosyl-2-coumarinate glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00999+R04998      Biosynthesis of various plant secondary metabolites\n",
      "R12885: \n",
      "     https://www.kegg.jp/pathway/rn00999+R12885      Biosynthesis of various plant secondary metabolites\n",
      "R01274: palmitoyl-CoA hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00062+R01274      Fatty acid elongation\n",
      "     https://www.kegg.jp/pathway/rn01040+R01274      Biosynthesis of unsaturated fatty acids\n",
      "R02222: steroyl-CoA,hydrogen-donor:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn01040+R02222      Biosynthesis of unsaturated fatty acids\n",
      "R12173: palmitoyl-CoA,ferrocytochrome-b5:oxygen oxidoreductase (9,10-dehydrogenating)\n",
      "     https://www.kegg.jp/pathway/rn01040+R12173      Biosynthesis of unsaturated fatty acids\n",
      "R01344: D-alanine:2-oxo acid aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R01344      D-Amino acid metabolism\n",
      "R02320: NTP:pyruvate O2-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00620+R02320      Pyruvate metabolism\n",
      "R08014: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08014      Nitrotoluene degradation\n",
      "R10634: \n",
      "     https://www.kegg.jp/pathway/rn00760+R10634      Nicotinate and nicotinamide metabolism\n",
      "R00789: ammonia:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00789      Nitrogen metabolism\n",
      "R12048: \n",
      "     https://www.kegg.jp/pathway/rn01040+R12048      Biosynthesis of unsaturated fatty acids\n"
     ]
    }
   ],
   "source": [
    "oldnotinmodel = [r for r in oldkegglist if r not in mcoosmodel_rxnlist]\n",
    "print(len(oldnotinmodel))\n",
    "print('=====================')\n",
    "for r in oldnotinmodel:\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "249\n",
      "=====================\n",
      "Starch_synthesis\n",
      "     no rxn\n",
      "Starch_degredation\n",
      "     no rxn\n",
      "Rspont2\n",
      "     no rxn\n",
      "RPG161161\n",
      "     no rxn\n",
      "RPG161140\n",
      "     no rxn\n",
      "RPG160161\n",
      "     no rxn\n",
      "RPE161161\n",
      "     no rxn\n",
      "RPE161140\n",
      "     no rxn\n",
      "RPE160161\n",
      "     no rxn\n",
      "RPC161161\n",
      "     no rxn\n",
      "RPC161140\n",
      "     no rxn\n",
      "RPC160161\n",
      "     no rxn\n",
      "RCL161161\n",
      "     no rxn\n",
      "RCL161140\n",
      "     no rxn\n",
      "RCL160161\n",
      "     no rxn\n",
      "RC08362\n",
      "     no rxn\n",
      "R09380: acyl-ACP:sn-glycerol-3-phosphate 1-O-acyltransferase\n",
      "     https://www.kegg.jp/entry/R09380\n",
      "R09372: methaneselenol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00450+R09372      Selenocompound metabolism\n",
      "R08781\n",
      "     no rxn\n",
      "R08775\n",
      "     no rxn\n",
      "R08774\n",
      "     no rxn\n",
      "R08773\n",
      "     no rxn\n",
      "R08716: cobalt-precorrin 3 dehydrogenase\n",
      "     https://www.kegg.jp/entry/R08716\n",
      "R08676: ADP-glucose:D-fructose 2-alpha-D-glucosyltransferase\n",
      "     https://www.kegg.jp/entry/R08676\n",
      "R08575: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase\n",
      "     https://www.kegg.jp/entry/R08575\n",
      "R08549: 2-oxoglutarate:NAD+ 2-oxidoreductase (CoA-succinylating);\n",
      "     https://www.kegg.jp/entry/R08549\n",
      "R07410: L-aspartate:NAD+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00760+R07410      Nicotinate and nicotinamide metabolism\n",
      "R07407: L-aspartate:NADP+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00760+R07407      Nicotinate and nicotinamide metabolism\n",
      "R07280: 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R07280      Riboflavin metabolism\n",
      "R07035: glutathione: 15-HPETE oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00590+R07035      Arachidonic acid metabolism\n",
      "R07034: glutathione: 5-HPETE oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00590+R07034      Arachidonic acid metabolism\n",
      "R06859\n",
      "     no rxn\n",
      "R06146\n",
      "     no rxn\n",
      "R05809: \n",
      "     https://www.kegg.jp/entry/R05809\n",
      "R05705: FMNH2:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00740+R05705      Riboflavin metabolism\n",
      "     https://www.kegg.jp/pathway/rn01057+R05705      Biosynthesis of type II polyketide products\n",
      "R05639: 4,4-dimethyl-5a-cholesta-8,24-dien-3b-ol:NADP+ D14-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R05639      Steroid biosynthesis\n",
      "R05615: all-trans-octaprenyl-diphosphate:4-hydroxybenzoate 3-octaprenyltransferase\n",
      "     https://www.kegg.jp/entry/R05615\n",
      "R05614\n",
      "     no rxn\n",
      "R05591: Nitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R05591      Aminobenzoate degradation\n",
      "R04993\n",
      "     no rxn\n",
      "R04990\n",
      "     no rxn\n",
      "R04989\n",
      "     no rxn\n",
      "R04988: S-adenosyl-L-methionine:3-(all-trans-octaprenyl)benzene-1,2-diol 2-O-methyltransferase\n",
      "     https://www.kegg.jp/entry/R04988\n",
      "R04986: 3-octaprenyl-4-hydroxybenzoate carboxy-lyase\n",
      "     https://www.kegg.jp/entry/R04986\n",
      "R04326: \n",
      "     https://www.kegg.jp/pathway/rn00670+R04326      One carbon pool by folate\n",
      "R04325: 10-formyltetrahydrofolate:5'-phosphoribosylglycinamide formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04325      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R04325      One carbon pool by folate\n",
      "R04230: (R)-4'-phosphopantothenate:L-cysteine ligase (ATP-utilizing)\n",
      "     https://www.kegg.jp/pathway/rn00770+R04230      Pantothenate and CoA biosynthesis\n",
      "R03596: hydrogen selenide:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00450+R03596      Selenocompound metabolism\n",
      "R03542: alpha-Aminopropiononitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03542      Cyanoamino acid metabolism\n",
      "R03199: (S)-2,3-Epoxysqualene mutase (cyclizing, lanosterol-forming)\n",
      "     https://www.kegg.jp/pathway/rn00100+R03199      Steroid biosynthesis\n",
      "R03093: 3-Indoleacetonitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00380+R03093      Tryptophan metabolism\n",
      "R03051: 2,3-Dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/entry/R03051\n",
      "R03024: 4-nitrophenyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R03024      Aminobenzoate degradation\n",
      "R03013: L-histidinol-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03013      Histidine metabolism\n",
      "R02999: NADPH:cyanocob(III)alamin oxidoreductase (cyanide-eliminating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R02999      Porphyrin metabolism\n",
      "R02872: presqualene-diphosphate diphosphate-lyase (reducing, squalene-forming)\n",
      "     https://www.kegg.jp/pathway/rn00100+R02872      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R02872      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R02869: S-adenosylmethioninamine:spermidine 3-aminopropyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R02869      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R02869      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00480+R02869      Glutathione metabolism\n",
      "R02539: Phenylacetate:CoA ligase\n",
      "     https://www.kegg.jp/pathway/rn00360+R02539      Phenylalanine metabolism\n",
      "R02161\n",
      "     no rxn\n",
      "R02016: NADPH:oxidized-thioredoxin oxidoreductase\n",
      "     https://www.kegg.jp/entry/R02016\n",
      "R01974: 3-(indol-3-yl)pyruvate carboxy-lyase [(2-indol-3-yl)acetaldehyde-forming]\n",
      "     https://www.kegg.jp/pathway/rn00380+R01974      Tryptophan metabolism\n",
      "R01887: gamma-Amino-gamma-cyanobutanoate aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R01887      Cyanoamino acid metabolism\n",
      "R01858: dGTP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R01858\n",
      "R01771: ATP:L-homoserine O-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01771      Glycine, serine and threonine metabolism\n",
      "R01706: hexadecanoyl-[acyl-carrier protein] hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01706      Fatty acid biosynthesis\n",
      "R01698: Dihydrolipoamide:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/entry/R01698\n",
      "R01654: 10-Formyltetrahydrofolate:L-glutamate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/entry/R01654\n",
      "R01652: \n",
      "     https://www.kegg.jp/pathway/rn00290+R01652      Valine, leucine and isoleucine biosynthesis\n",
      "R01224: 5-methyltetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R01224      One carbon pool by folate\n",
      "R01216: 2-dehydropantoate formaldehyde-lyase (3-methyl-2-oxobutanoate-forming)\n",
      "     https://www.kegg.jp/entry/R01216\n",
      "R01209: 2,3-Dihydroxy-3-methylbutanoate hydro-lyase\n",
      "     https://www.kegg.jp/entry/R01209\n",
      "R01138: dATP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R01138\n",
      "R01106: nitrite:(acceptor) oxidoreductase\n",
      "     https://www.kegg.jp/entry/R01106\n",
      "R01067: D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase\n",
      "     https://www.kegg.jp/pathway/rn00710+R01067      Carbon fixation in photosynthetic organisms\n",
      "R01011: ATP:glycerone phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01011      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01011      Methane metabolism\n",
      "R00941: 10-formyltetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00941      One carbon pool by folate\n",
      "R00851: acyl-CoA:sn-glycerol-3-phosphate 1-O-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R00851      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R00851      Glycerophospholipid metabolism\n",
      "R00847: ATP:glycerol 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R00847      Glycerolipid metabolism\n",
      "R00835: D-glucose-6-phosphate:NADP+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00835\n",
      "R00806: UDP-glucose:D-fructose 2-alpha-D-glucosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00806      Starch and sucrose metabolism\n",
      "R00802: sucrose alpha-glucohydrolase\n",
      "     https://www.kegg.jp/entry/R00802\n",
      "R00704: (R)-Lactate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00704      Pyruvate metabolism\n",
      "R00568\n",
      "     no rxn\n",
      "R00528: sulfite:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00528      Sulfur metabolism\n",
      "R00483: L-aspartate:ammonia ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00483      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00460+R00483      Cyanoamino acid metabolism\n",
      "R00475: glycolate:oxygen 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00630+R00475      Glyoxylate and dicarboxylate metabolism\n",
      "R00430: GTP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R00430\n",
      "R00424: GTP 7,8-8,9-dihydrolase\n",
      "     https://www.kegg.jp/entry/R00424\n",
      "R00418: UDP-N-acetyl-D-glucosamine 4-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00418      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00418      O-Antigen nucleotide sugar biosynthesis\n",
      "R00287: UDP-glucose glucophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00287      Starch and sucrose metabolism\n",
      "R00256: L-glutamine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00256      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00256      Alanine, aspartate and glutamate metabolism\n",
      "R00217: oxaloacetate carboxy-lyase (pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00217      Pyruvate metabolism\n",
      "R00209: pyruvate:NAD+ 2-oxidoreductase (CoA-acetylating);\n",
      "     https://www.kegg.jp/entry/R00209\n",
      "R00149: Carbon-dioxide:ammonia ligase (ADP-forming,carbamate-phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00149      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00149      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00149      Nitrogen metabolism\n",
      "R00132: carbonate hydro-lyase (carbon-dioxide-forming);\n",
      "     https://www.kegg.jp/entry/R00132\n",
      "R00093: L-glutamate:NAD+ oxidoreductase (transaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00093      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00093      Nitrogen metabolism\n",
      "R00086: ATP phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00086      Purine metabolism\n",
      "R00082: ferrocytochrome c2:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00082\n",
      "R00026: cellobiose glucohydrolase\n",
      "     https://www.kegg.jp/entry/R00026\n",
      "R00006: pyruvate:pyruvate acetaldehydetransferase (decarboxylating);\n",
      "     https://www.kegg.jp/entry/R00006\n",
      "R00004: diphosphate phosphohydrolase;\n",
      "     https://www.kegg.jp/entry/R00004\n",
      "R_formylTHMPT_formate\n",
      "     no rxn\n",
      "NADH_DH_ubi\n",
      "     no rxn\n",
      "Maintenance_ATP\n",
      "     no rxn\n",
      "Cytochrome_c_oxidase\n",
      "     no rxn\n",
      "Biomass_reaction\n",
      "     no rxn\n",
      "ATPSynth\n",
      "     no rxn\n",
      "ATPase\n",
      "     no rxn\n",
      "R08159: tetradecanoyl-[acyl-carrier protein] hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00061+R08159      Fatty acid biosynthesis\n",
      "R02814: Oleoyl-[acyl-carrier-protein] hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00061+R02814      Fatty acid biosynthesis\n",
      "R00258: L-Alanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00258      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00258      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00258      Carbon fixation in photosynthetic organisms\n",
      "R00485: L-asparagine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00485      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00460+R00485      Cyanoamino acid metabolism\n",
      "R00576: L-Glutamine:pyruvate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00576      Alanine, aspartate and glutamate metabolism\n",
      "R01215: L-Valine:pyruvate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00290+R01215      Valine, leucine and isoleucine biosynthesis\n",
      "R00345: phosphate:oxaloacetate carboxy-lyase (adding phosphate;phosphoenolpyruvate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00620+R00345      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00345      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00345      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00345      Carbon fixation pathways in prokaryotes\n",
      "EX_Pyr\n",
      "     no rxn\n",
      "EX_Pi\n",
      "     no rxn\n",
      "EX_O2\n",
      "     no rxn\n",
      "EX_nitrite\n",
      "     no rxn\n",
      "EX_nitrate\n",
      "     no rxn\n",
      "EX_NH3\n",
      "     no rxn\n",
      "EX_N2\n",
      "     no rxn\n",
      "EX_Lact\n",
      "     no rxn\n",
      "EX_Heme\n",
      "     no rxn\n",
      "EX_H2O\n",
      "     no rxn\n",
      "EX_H\n",
      "     no rxn\n",
      "EX_Formate\n",
      "     no rxn\n",
      "EX_EtOH\n",
      "     no rxn\n",
      "EX_CO2\n",
      "     no rxn\n",
      "EX_CH4\n",
      "     no rxn\n",
      "EX_C00305\n",
      "     no rxn\n",
      "EX_C00175\n",
      "     no rxn\n",
      "EX_C00076\n",
      "     no rxn\n",
      "EX_C00070\n",
      "     no rxn\n",
      "EX_C00023\n",
      "     no rxn\n",
      "EX_Ac\n",
      "     no rxn\n",
      "EX_sulfate\n",
      "     no rxn\n",
      "EX_Suc\n",
      "     no rxn\n",
      "CCL160161_tx\n",
      "     no rxn\n",
      "CCL161140_tx\n",
      "     no rxn\n",
      "CCL161161_tx\n",
      "     no rxn\n",
      "CPC160161_tx\n",
      "     no rxn\n",
      "CPC161140_tx\n",
      "     no rxn\n",
      "CPC161161_tx\n",
      "     no rxn\n",
      "CPE160161_tx\n",
      "     no rxn\n",
      "CPE161140_tx\n",
      "     no rxn\n",
      "CPE161161_tx\n",
      "     no rxn\n",
      "CPG160161_tx\n",
      "     no rxn\n",
      "CPG161140_tx\n",
      "     no rxn\n",
      "CPG161161_tx\n",
      "     no rxn\n",
      "lipooligosaccharides_tx\n",
      "     no rxn\n",
      "lps_tx\n",
      "     no rxn\n",
      "putresine_tx\n",
      "     no rxn\n",
      "spermidine_tx\n",
      "     no rxn\n",
      "Urea_tx\n",
      "     no rxn\n",
      "Formate_tx_pc\n",
      "     no rxn\n",
      "Formate_tx_ep\n",
      "     no rxn\n",
      "Pyr_tx_pc\n",
      "     no rxn\n",
      "Pyr_tx_ep\n",
      "     no rxn\n",
      "Pi_tx_pc\n",
      "     no rxn\n",
      "Pi_tx_ec\n",
      "     no rxn\n",
      "O2_tx_pc\n",
      "     no rxn\n",
      "O2_tx_ep\n",
      "     no rxn\n",
      "nitrite_pc\n",
      "     no rxn\n",
      "nitrite_ep\n",
      "     no rxn\n",
      "nitrate_pc\n",
      "     no rxn\n",
      "nitrate_ep\n",
      "     no rxn\n",
      "NH3_tx_pc\n",
      "     no rxn\n",
      "NH3_tx_ep\n",
      "     no rxn\n",
      "N2_tx_pc\n",
      "     no rxn\n",
      "N2_tx_ep\n",
      "     no rxn\n",
      "Lact_tx_pc\n",
      "     no rxn\n",
      "Lact_tx_ep\n",
      "     no rxn\n",
      "Heme_tx_pc\n",
      "     no rxn\n",
      "Heme_tx_ep\n",
      "     no rxn\n",
      "H2O_tx_pc\n",
      "     no rxn\n",
      "H2O_tx_ep\n",
      "     no rxn\n",
      "EtOH_tx_pc\n",
      "     no rxn\n",
      "EtOH_tx_ep\n",
      "     no rxn\n",
      "CO2_tx_pc\n",
      "     no rxn\n",
      "CO2_tx_ep\n",
      "     no rxn\n",
      "CH4_tx_pc\n",
      "     no rxn\n",
      "CH4_tx_ep\n",
      "     no rxn\n",
      "C00305_tx_pc\n",
      "     no rxn\n",
      "C00305_tx_ep\n",
      "     no rxn\n",
      "C00175_tx_pc\n",
      "     no rxn\n",
      "C00175_tx_ep\n",
      "     no rxn\n",
      "C00076_tx_pc\n",
      "     no rxn\n",
      "C00076_tx_ep\n",
      "     no rxn\n",
      "C00070_tx_pc\n",
      "     no rxn\n",
      "C00070_tx_ep\n",
      "     no rxn\n",
      "C00023_tx_pc\n",
      "     no rxn\n",
      "C00023_tx_ep\n",
      "     no rxn\n",
      "Ac_tx_pc\n",
      "     no rxn\n",
      "Ac_tx_ep\n",
      "     no rxn\n",
      "sulfate_tx_pc\n",
      "     no rxn\n",
      "sulfate_tx_ep\n",
      "     no rxn\n",
      "Suc_tx_pc\n",
      "     no rxn\n",
      "Suc_tx_ep\n",
      "     no rxn\n",
      "EX_CCL160161\n",
      "     no rxn\n",
      "EX_CCL161140\n",
      "     no rxn\n",
      "EX_CCL161161\n",
      "     no rxn\n",
      "EX_CPC160161\n",
      "     no rxn\n",
      "EX_CPC161140\n",
      "     no rxn\n",
      "EX_CPC161161\n",
      "     no rxn\n",
      "EX_CPE160161\n",
      "     no rxn\n",
      "EX_CPE161140\n",
      "     no rxn\n",
      "EX_CPE161161\n",
      "     no rxn\n",
      "EX_CPG160161\n",
      "     no rxn\n",
      "EX_CPG161140\n",
      "     no rxn\n",
      "EX_CPG161161\n",
      "     no rxn\n",
      "EX_lipooligosaccharides\n",
      "     no rxn\n",
      "EX_lps\n",
      "     no rxn\n",
      "EX_putresine\n",
      "     no rxn\n",
      "EX_spermidine\n",
      "     no rxn\n",
      "EX_Urea\n",
      "     no rxn\n",
      "alanine_tx\n",
      "     no rxn\n",
      "Aspargine_tx\n",
      "     no rxn\n",
      "Aspartate_tx\n",
      "     no rxn\n",
      "Cysteine_tx\n",
      "     no rxn\n",
      "Glutamate_tx\n",
      "     no rxn\n",
      "Glutamine_tx\n",
      "     no rxn\n",
      "Valine_tx\n",
      "     no rxn\n",
      "mercaptopyruvate_tx\n",
      "     no rxn\n",
      "EX_mercaptopyruvate\n",
      "     no rxn\n",
      "methyloxobutanoicacid_tx\n",
      "     no rxn\n",
      "EX_methyloxobutanoicacid\n",
      "     no rxn\n",
      "OAA_tx\n",
      "     no rxn\n",
      "EX_OAA\n",
      "     no rxn\n",
      "oxoglutarate_tx\n",
      "     no rxn\n",
      "EX_oxoglutarate\n",
      "     no rxn\n",
      "EX_alanine\n",
      "     no rxn\n",
      "EX_Aspargine\n",
      "     no rxn\n",
      "EX_Aspartate\n",
      "     no rxn\n",
      "EX_Cysteine\n",
      "     no rxn\n",
      "EX_Glutamate\n",
      "     no rxn\n",
      "EX_Glutamine\n",
      "     no rxn\n",
      "EX_Valine\n",
      "     no rxn\n",
      "EX_methanol\n",
      "     no rxn\n",
      "methanol_tx\n",
      "     no rxn\n",
      "EX_formaldehyde\n",
      "     no rxn\n",
      "formaldehyde_tx\n",
      "     no rxn\n",
      "ubiquinol_tx_pc\n",
      "     no rxn\n",
      "ubiquinone_tx_pc\n",
      "     no rxn\n",
      "ferricytochrome_tx_pc\n",
      "     no rxn\n",
      "ferrochrome_tx_pc\n",
      "     no rxn\n",
      "formaldehyde_tx_pc\n",
      "     no rxn\n",
      "methanol_tx_cp\n",
      "     no rxn\n",
      "R02164_ub\n",
      "     no rxn\n"
     ]
    }
   ],
   "source": [
    "addedtomodel = [r for r in mcoosmodel_rxnlist if r not in oldkegglist]\n",
    "print(len(addedtomodel))\n",
    "print('=====================')\n",
    "for r in addedtomodel:\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "666\n",
      "=====================\n",
      "R00006: pyruvate:pyruvate acetaldehydetransferase (decarboxylating);\n",
      "     https://www.kegg.jp/entry/R00006\n",
      "R00014: pyruvate:thiamin diphosphate acetaldehydetransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00010+R00014      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R00014      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00014      Pyruvate metabolism\n",
      "R00022: chitobiose N-acetylglucosaminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00022      Amino sugar and nucleotide sugar metabolism\n",
      "R00026: cellobiose glucohydrolase\n",
      "     https://www.kegg.jp/entry/R00026\n",
      "R00028: maltose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00028      Starch and sucrose metabolism\n",
      "R00036: 5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing; porphobilinogen-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R00036      Porphyrin metabolism\n",
      "R00066: 6,7-Dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl)lumazine 2,3-butanediyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00066      Riboflavin metabolism\n",
      "R00078: Fe(II):oxygen oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00860+R00078      Porphyrin metabolism\n",
      "R00084: porphobilinogen:(4-[2-carboxyethyl]-3-[carboxymethyl]pyrrol-2-yl)methyltransferase (hydrolysing);\n",
      "     https://www.kegg.jp/pathway/rn00860+R00084      Porphyrin metabolism\n",
      "R00087: ATP diphosphohydrolase (diphosphate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00230+R00087      Purine metabolism\n",
      "R00104: ATP:NAD+ 2'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00104      Nicotinate and nicotinamide metabolism\n",
      "R00112: NADPH:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00112      Nicotinate and nicotinamide metabolism\n",
      "R00114: L-glutamate:NADP+ oxidoreductase (transaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00114      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00114      Nitrogen metabolism\n",
      "R00125: P1,P4-bis(5'-adenosyl)-tetraphosphate nucleotidebisphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00125      Purine metabolism\n",
      "R00127: ATP:AMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00127      Purine metabolism\n",
      "R00130: ATP:dephospho-CoA 3'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R00130      Pantothenate and CoA biosynthesis\n",
      "R00132: carbonate hydro-lyase (carbon-dioxide-forming);\n",
      "     https://www.kegg.jp/entry/R00132\n",
      "R00137: ATP:nicotinamide-nucleotide adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00137      Nicotinate and nicotinamide metabolism\n",
      "R00139: ATP:2'-deoxy-5-hydroxymethylcytidine-5'-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00139      Pyrimidine metabolism\n",
      "R00148: ammonia,ubiquinol:oxygen oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00910+R00148      Nitrogen metabolism\n",
      "R00156: ATP:UDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00156      Pyrimidine metabolism\n",
      "R00158: ATP:UMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00158      Pyrimidine metabolism\n",
      "R00161: ATP:FMN adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00161      Riboflavin metabolism\n",
      "R00177: ATP:L-methionine S-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00177      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00999+R00177      Biosynthesis of various plant secondary metabolites\n",
      "R00178: S-adenosyl-L-methionine carboxy-lyase [S-adenosyl 3-(methylsulfanyl)propylamine-forming]\n",
      "     https://www.kegg.jp/pathway/rn00270+R00178      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R00178      Arginine and proline metabolism\n",
      "R00183: adenosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00183      Purine metabolism\n",
      "R00185: ATP:adenosine 5'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00185      Purine metabolism\n",
      "R00190: AMP:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00190      Purine metabolism\n",
      "R00192: S-Adenosyl-L-homocysteine hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00192      Cysteine and methionine metabolism\n",
      "R00194: S-Adenosyl-L-homocysteine homocysteinylribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00194      Cysteine and methionine metabolism\n",
      "R00200: ATP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00200      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00200      Pyruvate metabolism\n",
      "R00206: ATP:pyruvate,phosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00206      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00206      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00206      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00206      Carbon fixation pathways in prokaryotes\n",
      "R00209: pyruvate:NAD+ 2-oxidoreductase (CoA-acetylating);\n",
      "     https://www.kegg.jp/entry/R00209\n",
      "R00214: (S)-malate:NAD+ oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00214      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00214      Carbon fixation in photosynthetic organisms\n",
      "R00220: L-serine ammonia-lyase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00220      Glycine, serine and threonine metabolism\n",
      "R00226: pyruvate:pyruvate acetaldehydetransferase (decarboxylating);\n",
      "     https://www.kegg.jp/pathway/rn00290+R00226      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00650+R00226      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R00226      C5-Branched dibasic acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00770+R00226      Pantothenate and CoA biosynthesis\n",
      "R00235: Acetate:CoA ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00010+R00235      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00630+R00235      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00235      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00235      Carbon fixation pathways in prokaryotes\n",
      "R00236: acetyl adenylate:CoA acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00236      Pyruvate metabolism\n",
      "R00239: ATP:L-glutamate 5-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R00239      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00332+R00239      Carbapenem biosynthesis\n",
      "R00243: L-glutamate:NAD+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00243      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00243      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00430+R00243      Taurine and hypotaurine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00243      Nitrogen metabolism\n",
      "R00248: L-glutamate:NADP+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00248      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00248      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00248      Nitrogen metabolism\n",
      "R00253: L-Glutamate:ammonia ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00253      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00253      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00253      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00253      Nitrogen metabolism\n",
      "R00256: L-glutamine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00256      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00256      Alanine, aspartate and glutamate metabolism\n",
      "R00257: Deamido-NAD+:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00760+R00257      Nicotinate and nicotinamide metabolism\n",
      "R00259: acetyl-CoA:L-glutamate N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00259      Arginine biosynthesis\n",
      "R00260: glutamate racemase\n",
      "     https://www.kegg.jp/pathway/rn00470+R00260      D-Amino acid metabolism\n",
      "R00271: acetyl-CoA:2-oxoglutarate C-acetyltransferase (thioester-hydrolysing, carboxymethyl forming)\n",
      "     https://www.kegg.jp/pathway/rn00300+R00271      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00271      Pyruvate metabolism\n",
      "R00274: glutathione:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00274      Glutathione metabolism\n",
      "R00286: UDP-glucose:NAD+ 6-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00040+R00286      Pentose and glucuronate interconversions\n",
      "     https://www.kegg.jp/pathway/rn00053+R00286      Ascorbate and aldarate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00286      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00286      O-Antigen nucleotide sugar biosynthesis\n",
      "R00287: UDP-glucose glucophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00287      Starch and sucrose metabolism\n",
      "R00289: UTP:alpha-D-glucose-1-phosphate uridylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00040+R00289      Pentose and glucuronate interconversions\n",
      "     https://www.kegg.jp/pathway/rn00052+R00289      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00500+R00289      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00289      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00289      O-Antigen nucleotide sugar biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00561+R00289      Glycerolipid metabolism\n",
      "R00291: UDP-glucose 4-epimerase;\n",
      "     https://www.kegg.jp/pathway/rn00052+R00291      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00291      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00291      O-Antigen nucleotide sugar biosynthesis\n",
      "R00294: nitrous-oxide:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00294      Nitrogen metabolism\n",
      "R00299: ATP:D-glucose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00299      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00521+R00299      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00524+R00299      Neomycin, kanamycin and gentamicin biosynthesis\n",
      "R00310: protoheme ferro-lyase (protoporphyrin-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R00310      Porphyrin metabolism\n",
      "R00315: ATP:acetate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00430+R00315      Taurine and hypotaurine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00620+R00315      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00315      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00315      Carbon fixation pathways in prokaryotes\n",
      "R00316: ATP:acetate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00316      Pyruvate metabolism\n",
      "R00317: Acetyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00317      Pyruvate metabolism\n",
      "R00330: ATP:GDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00330      Purine metabolism\n",
      "R00332: ATP:GMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00332      Purine metabolism\n",
      "R00336: guanosine-3',5'-bis(diphosphate) 3'-diphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00336      Purine metabolism\n",
      "R00342: (S)-malate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00020+R00342      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00342      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00342      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00342      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00342      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00342      Carbon fixation pathways in prokaryotes\n",
      "R00351: acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00351      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00351      Glyoxylate and dicarboxylate metabolism\n",
      "R00355: L-Aspartate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00355      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00355      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00355      Carbon fixation in photosynthetic organisms\n",
      "R00357: L-Aspartic acid:oxygen oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00357      Alanine, aspartate and glutamate metabolism\n",
      "R00369: L-Alanine:glyoxylate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00369      Alanine, aspartate and glutamate metabolism\n",
      "R00372: Glycine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00372      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00372      Glyoxylate and dicarboxylate metabolism\n",
      "R00396: L-alanine:NAD+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00396      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00430+R00396      Taurine and hypotaurine metabolism\n",
      "R00401: alanine racemase\n",
      "     https://www.kegg.jp/pathway/rn00470+R00401      D-Amino acid metabolism\n",
      "R00405: Succinate:CoA ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00405      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00640+R00405      Propanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00405      Carbon fixation pathways in prokaryotes\n",
      "R00416: UTP:N-acetyl-alpha-D-glucosamine-1-phosphate uridylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00416      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00416      O-Antigen nucleotide sugar biosynthesis\n",
      "R00424: GTP 7,8-8,9-dihydrolase\n",
      "     https://www.kegg.jp/entry/R00424\n",
      "R00425: GTP 7,8-8,9-dihydrolase (diphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00740+R00425      Riboflavin metabolism\n",
      "     https://www.kegg.jp/pathway/rn00790+R00425      Folate biosynthesis\n",
      "R00426: GTP diphosphohydrolase (diphosphate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00230+R00426      Purine metabolism\n",
      "R00428: GTP 8,9-hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00790+R00428      Folate biosynthesis\n",
      "R00429: ATP:GTP 3'-diphosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00429      Purine metabolism\n",
      "R00430: GTP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R00430\n",
      "R00451: meso-2,6-diaminoheptanedioate carboxy-lyase (L-lysine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00300+R00451      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00470+R00451      D-Amino acid metabolism\n",
      "R00470: 4-hydroxy-2-oxoglutarate glyoxylate-lyase (pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00470      Glyoxylate and dicarboxylate metabolism\n",
      "R00473: L-malyl-CoA glyoxylate-lyase (acetyl-CoA-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00473      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00473      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00473      Carbon fixation pathways in prokaryotes\n",
      "R00475: glycolate:oxygen 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00630+R00475      Glyoxylate and dicarboxylate metabolism\n",
      "R00480: ATP:L-aspartate 4-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00480      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00261+R00480      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00270+R00480      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R00480      Lysine biosynthesis\n",
      "R00481: L-aspartate:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00481      Nicotinate and nicotinamide metabolism\n",
      "R00489: L-aspartate 1-carboxy-lyase (beta-alanine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00410+R00489      beta-Alanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00770+R00489      Pantothenate and CoA biosynthesis\n",
      "R00494: glutathione gamma-glutamylaminopeptidase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00494      Glutathione metabolism\n",
      "R00497: gamma-L-glutamyl-L-cysteine:glycine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00497      Glutathione metabolism\n",
      "R00508: 3'-phospho-5'-adenylyl sulfate 3'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00508      Sulfur metabolism\n",
      "R00509: ATP:adenylylsulfate 3'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00509      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R00509      Sulfur metabolism\n",
      "R00511: cytidine-5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00511      Pyrimidine metabolism\n",
      "R00512: ATP:CMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00512      Pyrimidine metabolism\n",
      "R00515: CTP diphosphohydrolase (diphosphate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00240+R00515      Pyrimidine metabolism\n",
      "R00519: formate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00630+R00519      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00519      Methane metabolism\n",
      "R00529: ATP:sulfate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00529      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00261+R00529      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00920+R00529      Sulfur metabolism\n",
      "R00548: riboflavin-5-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00548      Riboflavin metabolism\n",
      "R00549: ATP:riboflavin 5'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00549      Riboflavin metabolism\n",
      "R00566: L-arginine carboxy-lyase (agmatine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00330+R00566      Arginine and proline metabolism\n",
      "R00570: ATP:CDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00570      Pyrimidine metabolism\n",
      "R00571: UTP:ammonia ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00571      Pyrimidine metabolism\n",
      "R00573: UTP:L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00573      Pyrimidine metabolism\n",
      "R00575: HCO3-:L-glutamine amido-ligase (ADP-forming, carbamate-phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00575      Pyrimidine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R00575      Alanine, aspartate and glutamate metabolism\n",
      "R00578: L-aspartate:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00578      Alanine, aspartate and glutamate metabolism\n",
      "R00582: O-phospho-L-serine phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00582      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00582      Methane metabolism\n",
      "R00585: L-Serine:pyruvate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00585      Glycine, serine and threonine metabolism\n",
      "R00586: acetyl-CoA:L-serine O-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00586      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R00586      Sulfur metabolism\n",
      "     https://www.kegg.jp/pathway/rn00998+R00586      Biosynthesis of various antibiotics\n",
      "R00588: L-Serine:glyoxylate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00588      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00588      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00588      Methane metabolism\n",
      "R00617: ATP:thiamin-phosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R00617      Thiamine metabolism\n",
      "R00621: \n",
      "     https://www.kegg.jp/pathway/rn00020+R00621      Citrate cycle (TCA cycle)\n",
      "R00658: 2-phospho-D-glycerate hydro-lyase (phosphoenolpyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00010+R00658      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00680+R00658      Methane metabolism\n",
      "R00660: Phosphoenolpyruvate:UDP-N-acetyl-D-glucosamine 1-carboxyvinyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00660      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R00660      Peptidoglycan biosynthesis\n",
      "R00662: uridine triphosphate pyrophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00662      Pyrimidine metabolism\n",
      "R00667: L-Ornithine:2-oxo-acid aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R00667      Arginine and proline metabolism\n",
      "R00669: N2-Acetyl-L-ornithine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00669      Arginine biosynthesis\n",
      "R00674: L-serine hydro-lyase (adding indole; L-tryptophan-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00674      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R00691: L-arogenate hydro-lyase (decarboxylating; L-phenylalanine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00691      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R00694: L-Phenylalanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00360+R00694      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00400+R00694      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00960+R00694      Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "R00702: farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00100+R00702      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00906+R00702      Carotenoid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R00702      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R00709: isocitrate:NAD+ oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00709      Citrate cycle (TCA cycle)\n",
      "R00713: succinate-semialdehyde:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00713      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00350+R00713      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R00713      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00760+R00713      Nicotinate and nicotinamide metabolism\n",
      "R00714: succinate-semialdehyde:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00714      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00350+R00714      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R00714      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00760+R00714      Nicotinate and nicotinamide metabolism\n",
      "R00720: Inosine 5'-triphosphate pyrophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00720      Purine metabolism\n",
      "R00722: ATP:IDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00722      Purine metabolism\n",
      "R00734: L-tyrosine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R00734      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00350+R00734      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00400+R00734      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00401+R00734      Novobiocin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00950+R00734      Isoquinoline alkaloid biosynthesis\n",
      "R00742: acetyl-CoA:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00061+R00742      Fatty acid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00254+R00742      Aflatoxin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00742      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00640+R00742      Propanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00742      Carbon fixation pathways in prokaryotes\n",
      "R00754: ethanol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00754      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00754      Pyruvate metabolism\n",
      "R00756: ATP:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00756      Methane metabolism\n",
      "R00768: L-glutamine:D-fructose-6-phosphate isomerase (deaminating);\n",
      "     https://www.kegg.jp/pathway/rn00250+R00768      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00768      Amino sugar and nucleotide sugar metabolism\n",
      "R00771: D-glucose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00771      Starch and sucrose metabolism\n",
      "R00782: L-cysteine hydrogen-sulfide-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R00782      Cysteine and methionine metabolism\n",
      "R00787: ammonia:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00787      Nitrogen metabolism\n",
      "R00801: sucrose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00052+R00801      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00500+R00801      Starch and sucrose metabolism\n",
      "R00802: sucrose alpha-glucohydrolase\n",
      "     https://www.kegg.jp/entry/R00802\n",
      "R00835: D-glucose-6-phosphate:NADP+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00835\n",
      "R00842: sn-Glycerol-3-phosphate:NAD+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00564+R00842      Glycerophospholipid metabolism\n",
      "R00844: sn-Glycerol-3-phosphate:NADP+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00564+R00844      Glycerophospholipid metabolism\n",
      "R00858: hydrogen-sulfide:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00858      Sulfur metabolism\n",
      "R00885: GTP:alpha-D-mannose-1-phosphate guanylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00051+R00885      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00885      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00885      O-Antigen nucleotide sugar biosynthesis\n",
      "R00888: GDP-alpha-D-mannose 4,6-hydro-lyase (GDP-4-dehydro-alpha-D-rhamnose-forming);\n",
      "     https://www.kegg.jp/pathway/rn00051+R00888      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00888      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00888      O-Antigen nucleotide sugar biosynthesis\n",
      "R00894: L-glutamate:L-cysteine gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00894      Glutathione metabolism\n",
      "R00896: L-cysteine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00896      Cysteine and methionine metabolism\n",
      "R00897: O3-acetyl-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00270+R00897      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R00897      Sulfur metabolism\n",
      "R00899: L-cysteinylglycine dipeptidase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00899      Glutathione metabolism\n",
      "R00926: Propionyladenylate:CoA propionyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R00926      Propanoate metabolism\n",
      "R00934: L-erythro-3-methylmalyl-CoA glyoxylate-lyase (propanoyl-CoA-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R00934      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R00934      C5-Branched dibasic acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00934      Carbon fixation pathways in prokaryotes\n",
      "R00936: 5,6,7,8-tetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00936      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00936      Folate biosynthesis\n",
      "R00937: 5,6,7,8-tetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00937      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00937      Folate biosynthesis\n",
      "R00939: 5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00939      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00939      Folate biosynthesis\n",
      "R00940: 5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00940      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R00940      Folate biosynthesis\n",
      "R00942: Tetrahydrofolate:L-glutamate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R00942      Folate biosynthesis\n",
      "R00943: Formate:tetrahydrofolate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00670+R00943      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R00943      Carbon fixation pathways in prokaryotes\n",
      "R00945: 5,10-Methylenetetrahydrofolate:glycine hydroxymethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R00945      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00460+R00945      Cyanoamino acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R00945      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R00945      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00680+R00945      Methane metabolism\n",
      "R00946: 5-methyltetrahydrofolate:L-homocysteine S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00946      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R00946      One carbon pool by folate\n",
      "R00948: ATP:alpha-D-glucose-1-phosphate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00948      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00948      Amino sugar and nucleotide sugar metabolism\n",
      "R00959: alpha-D-Glucose 1-phosphate 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00959      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00052+R00959      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00959      Amino sugar and nucleotide sugar metabolism\n",
      "R00963: uridine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00963      Pyrimidine metabolism\n",
      "R00965: orotidine-5'-phosphate carboxy-lyase (UMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00240+R00965      Pyrimidine metabolism\n",
      "R00966: UMP:diphosphate phospho-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00966      Pyrimidine metabolism\n",
      "R00974: Cytosine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00974      Pyrimidine metabolism\n",
      "R00985: chorismate pyruvate-lyase (amino-accepting; anthranilate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00985      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R00986: Chorismate pyruvate-lyase (amino-accepting)\n",
      "     https://www.kegg.jp/pathway/rn00400+R00986      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00405+R00986      Phenazine biosynthesis\n",
      "R00994: (2R,3S)-3-methylmalate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00290+R00994      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R00994      C5-Branched dibasic acid metabolism\n",
      "R00996: L-threonine ammonia-lyase (2-oxobutanoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R00996      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00290+R00996      Valine, leucine and isoleucine biosynthesis\n",
      "R01015: D-glyceraldehyde-3-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01015      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00051+R01015      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00562+R01015      Inositol phosphate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R01015      Carbon fixation in photosynthetic organisms\n",
      "R01030: sn-Glycero-3-phosphocholine glycerophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01030      Glycerophospholipid metabolism\n",
      "R01049: ATP:D-ribose-5-phosphate diphosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01049      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00230+R01049      Purine metabolism\n",
      "R01054: ADP-ribose ribophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01054      Purine metabolism\n",
      "R01056: D-ribose-5-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01056      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00710+R01056      Carbon fixation in photosynthetic organisms\n",
      "R01057: D-Ribose 1,5-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01057      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00230+R01057      Purine metabolism\n",
      "R01061: D-glyceraldehyde-3-phosphate:NAD+ oxidoreductase (phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00010+R01061      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00710+R01061      Carbon fixation in photosynthetic organisms\n",
      "R01067: D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase\n",
      "     https://www.kegg.jp/pathway/rn00710+R01067      Carbon fixation in photosynthetic organisms\n",
      "R01068: D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00680+R01068      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R01068      Carbon fixation in photosynthetic organisms\n",
      "R01070: beta-D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00010+R01070      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R01070      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R01070      Fructose and mannose metabolism\n",
      "R01071: 1-(5-phospho-D-ribosyl)-ATP:diphosphate phospho-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00340+R01071      Histidine metabolism\n",
      "R01072: 5-phosphoribosylamine:diphosphate phospho-alpha-D-ribosyltransferase (glutamate-amidating)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01072      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01072      Alanine, aspartate and glutamate metabolism\n",
      "R01073: N-(5-Phospho-D-ribosyl)anthranilate:pyrophosphate phosphoribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00400+R01073      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01074: biotin:CoA ligase (AMP-forming);\n",
      "     https://www.kegg.jp/pathway/rn00780+R01074      Biotin metabolism\n",
      "R01078: dethiobiotin:sulfur sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R01078      Biotin metabolism\n",
      "R01082: (S)-malate hydro-lyase (fumarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R01082      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R01082      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01082      Carbon fixation pathways in prokaryotes\n",
      "R01083: N6-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01083      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01083      Alanine, aspartate and glutamate metabolism\n",
      "R01085: 3-fumarylpyruvate fumarylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00350+R01085      Tyrosine metabolism\n",
      "R01086: 2-(Nomega-L-arginino)succinate arginine-lyase (fumarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R01086      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R01086      Alanine, aspartate and glutamate metabolism\n",
      "R01090: L-Leucine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00280+R01090      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00290+R01090      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00966+R01090      Glucosinolate biosynthesis\n",
      "R01106: nitrite:(acceptor) oxidoreductase\n",
      "     https://www.kegg.jp/entry/R01106\n",
      "R01126: inosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01126      Purine metabolism\n",
      "R01127: IMP 1,2-hydrolase (decyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01127      Purine metabolism\n",
      "R01130: IMP:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01130      Purine metabolism\n",
      "R01132: IMP:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01132      Purine metabolism\n",
      "R01135: IMP:L-aspartate ligase (GDP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01135      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01135      Alanine, aspartate and glutamate metabolism\n",
      "R01137: ATP:dADP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01137      Purine metabolism\n",
      "R01138: dATP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R01138\n",
      "R01142: Methane,NADH:oxygen oxidoreductase (hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00680+R01142      Methane metabolism\n",
      "R01146: methanol:ferricytochrome-c oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00680+R01146      Methane metabolism\n",
      "R01148: D-Alanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R01148      D-Amino acid metabolism\n",
      "R01150: D-alanine:D-alanine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00470+R01150      D-Amino acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R01150      Peptidoglycan biosynthesis\n",
      "R01152: N-Carbamoylputrescine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01152      Arginine and proline metabolism\n",
      "R01163: L-histidinal:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00340+R01163      Histidine metabolism\n",
      "R01167: L-histidine carboxy-lyase (histamine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00340+R01167      Histidine metabolism\n",
      "R01185: 1D-myo-inositol 1-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00562+R01185      Inositol phosphate metabolism\n",
      "R01186: 1D-myo-inositol 4-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00562+R01186      Inositol phosphate metabolism\n",
      "R01187: 1D-myo-inositol 3-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00521+R01187      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00562+R01187      Inositol phosphate metabolism\n",
      "R01196: pyruvate:ferredoxin 2-oxidoreductase (CoA-acetylating)\n",
      "     https://www.kegg.jp/pathway/rn00010+R01196      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R01196      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R01196      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R01196      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01196      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01196      Carbon fixation pathways in prokaryotes\n",
      "R01209: 2,3-Dihydroxy-3-methylbutanoate hydro-lyase\n",
      "     https://www.kegg.jp/entry/R01209\n",
      "R01213: acetyl-CoA:3-methyl-2-oxobutanoate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming);\n",
      "     https://www.kegg.jp/pathway/rn00290+R01213      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R01213      Pyruvate metabolism\n",
      "R01214: L-Valine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00280+R01214      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00290+R01214      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00770+R01214      Pantothenate and CoA biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00966+R01214      Glucosinolate biosynthesis\n",
      "R01221: glycine:NAD+ 2-oxidoreductase (tetrahydrofolate-methylene-adding);\n",
      "     https://www.kegg.jp/pathway/rn00630+R01221      Glyoxylate and dicarboxylate metabolism\n",
      "R01226: 5,10-Methylenetetrahydrofolate:3-methyl-2-oxobutanoate hydroxymethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R01226      Pantothenate and CoA biosynthesis\n",
      "R01227: guanosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01227      Purine metabolism\n",
      "R01229: GMP:diphosphate 5-phospho-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01229      Purine metabolism\n",
      "R01230: Xanthosine-5'-phosphate:ammonia ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01230      Purine metabolism\n",
      "R01231: Xanthosine-5'-phosphate:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R01231      Purine metabolism\n",
      "R01248: L-Proline:NAD+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01248      Arginine and proline metabolism\n",
      "R01251: L-Proline:NADP+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01251      Arginine and proline metabolism\n",
      "R01256: (S)-malate:CoA ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00630+R01256      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01256      Methane metabolism\n",
      "R01268: Nicotinamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R01268      Nicotinate and nicotinamide metabolism\n",
      "R01280: Palmitate:CoA ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00061+R01280      Fatty acid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00071+R01280      Fatty acid degradation\n",
      "R01286: L-cystathionine L-homocysteine-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R01286      Cysteine and methionine metabolism\n",
      "R01288: O4-succinyl-L-homoserine:hydrogen sulfide S-(3-amino-3-carboxypropyl)transferase;\n",
      "     https://www.kegg.jp/pathway/rn00270+R01288      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R01288      Sulfur metabolism\n",
      "R01302: chorismate pyruvate-lyase (4-hydroxybenzoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00130+R01302      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R01320: S-adenosyl-L-methionine:phosphatidyl-N-dimethylethanolamine N-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01320      Glycerophospholipid metabolism\n",
      "R01325: citrate hydro-lyase (cis-aconitate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R01325      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00630+R01325      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01325      Carbon fixation pathways in prokaryotes\n",
      "R01334: 2-phosphoglycolate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00630+R01334      Glyoxylate and dicarboxylate metabolism\n",
      "R01353: ATP:propanoate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R01353      Propanoate metabolism\n",
      "R01354: ATP:propanoate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R01354      Propanoate metabolism\n",
      "R01373: prephenate hydro-lyase (decarboxylating; phenylpyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01373      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01384: UDP-D-glucuronate carboxy-lyase (UDP-D-xylose-forming)\n",
      "     https://www.kegg.jp/pathway/rn00520+R01384      Amino sugar and nucleotide sugar metabolism\n",
      "R01385: UDP-glucuronate 4-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00053+R01385      Ascorbate and aldarate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01385      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R01385      O-Antigen nucleotide sugar biosynthesis\n",
      "R01397: carbamoyl-phosphate:L-aspartate carbamoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01397      Pyrimidine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00250+R01397      Alanine, aspartate and glutamate metabolism\n",
      "R01398: Carbamoyl-phosphate:L-ornithine carbamoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R01398      Arginine biosynthesis\n",
      "R01401: methylthioadenosine methylthioribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01401      Cysteine and methionine metabolism\n",
      "R01411: 5-Methylcytosine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01411      Pyrimidine metabolism\n",
      "R01421: Benzoyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R01421      Aminobenzoate degradation\n",
      "R01457: cholesterol:NADP+ Delta24-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R01457      Steroid biosynthesis\n",
      "R01466: O-phospho-L-homoserine phosphate-lyase (adding water;L-threonine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R01466      Glycine, serine and threonine metabolism\n",
      "R01470: sn-Glycero-3-phosphoethanolamine glycerophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01470      Glycerophospholipid metabolism\n",
      "R01492: ATP:cob(I)alamin Co-beta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R01492      Porphyrin metabolism\n",
      "R01512: ATP:3-phospho-D-glycerate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01512      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00710+R01512      Carbon fixation in photosynthetic organisms\n",
      "R01514: ATP:(R)-glycerate 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01514      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00561+R01514      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R01514      Glyoxylate and dicarboxylate metabolism\n",
      "R01518: D-phosphoglycerate 2,3-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01518      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00260+R01518      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01518      Methane metabolism\n",
      "R01528: 6-phospho-D-gluconate:NADP+ 2-oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00030+R01528      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00480+R01528      Glutathione metabolism\n",
      "R01529: D-Ribulose-5-phosphate 3-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01529      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00040+R01529      Pentose and glucuronate interconversions\n",
      "     https://www.kegg.jp/pathway/rn00710+R01529      Carbon fixation in photosynthetic organisms\n",
      "R01530: D-arabinose-5-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R01530      Lipopolysaccharide biosynthesis\n",
      "R01547: ATP:dAMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01547      Purine metabolism\n",
      "R01569: thymidylate 5'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01569      Pyrimidine metabolism\n",
      "R01570: thymidine:phosphate deoxy-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01570      Pyrimidine metabolism\n",
      "R01582: D-Phenylalanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00360+R01582      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00470+R01582      D-Amino acid metabolism\n",
      "R01600: ATP:beta-D-glucose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01600      Glycolysis / Gluconeogenesis\n",
      "R01621: D-xylulose 5-phosphate D-glyceraldehyde-3-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R01621      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00710+R01621      Carbon fixation in photosynthetic organisms\n",
      "R01626: Malonyl-CoA:[acyl-carrier-protein] S-malonyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01626      Fatty acid biosynthesis\n",
      "R01641: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01641      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00710+R01641      Carbon fixation in photosynthetic organisms\n",
      "R01655: 5,10-Methenyltetrahydrofolate 5-hydrolase (decyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00670+R01655      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R01655      Carbon fixation pathways in prokaryotes\n",
      "R01658: Dimethylallyl-diphosphate:isopentenyl-diphosphate dimethylallyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R01658      Terpenoid backbone biosynthesis\n",
      "R01664: 2'-deoxycytidine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01664      Pyrimidine metabolism\n",
      "R01665: ATP:dCMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01665      Pyrimidine metabolism\n",
      "R01698: Dihydrolipoamide:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/entry/R01698\n",
      "R01714: 5-O-(1-Carboxyvinyl)-3-phosphoshikimate phosphate-lyase (chorismate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01714      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01715: Chorismate pyruvatemutase\n",
      "     https://www.kegg.jp/pathway/rn00400+R01715      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01716: chorismate:L-glutamine aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R01716      Folate biosynthesis\n",
      "R01724: 5-phospho-alpha-D-ribose 1-diphosphate:nicotinate ligase (ADP, diphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00760+R01724      Nicotinate and nicotinamide metabolism\n",
      "R01728: prephenate:NAD+ oxidoreductase(decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01728      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00401+R01728      Novobiocin biosynthesis\n",
      "R01736: (R)-S-Lactoylglutathione hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00620+R01736      Pyruvate metabolism\n",
      "R01773: L-Homoserine:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01773      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R01773      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R01773      Lysine biosynthesis\n",
      "R01775: L-homoserine:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01775      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R01775      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R01775      Lysine biosynthesis\n",
      "R01777: Succinyl-CoA:L-homoserine O-succinyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01777      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00920+R01777      Sulfur metabolism\n",
      "R01786: ATP:alpha-D-glucose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R01786      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00052+R01786      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01786      Amino sugar and nucleotide sugar metabolism\n",
      "R01799: CTP:phosphatidate cytidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01799      Glycerophospholipid metabolism\n",
      "R01800: CDP-diacylglycerol:L-serine 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01800      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R01800      Glycerophospholipid metabolism\n",
      "R01801: CDP-diacylglycerol:sn-glycerol-3-phosphate 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R01801      Glycerophospholipid metabolism\n",
      "R01818: D-mannose 6-phosphate 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00051+R01818      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01818      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R01818      O-Antigen nucleotide sugar biosynthesis\n",
      "R01819: D-mannose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00051+R01819      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R01819      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R01819      O-Antigen nucleotide sugar biosynthesis\n",
      "R01826: Phosphoenolpyruvate:D-erythrose-4-phosphate C-(1-carboxyvinyl)transferase (phosphate hydrolysing, 2-carboxy-2-oxoethyl-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R01826      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R01827: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01827      Pentose phosphate pathway\n",
      "R01830: beta-D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01830      Pentose phosphate pathway\n",
      "R01857: ATP:dGDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01857      Purine metabolism\n",
      "R01858: dGTP:pyruvate 2-O-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R01858\n",
      "R01867: (S)-dihydroorotate:fumarate oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01867      Pyrimidine metabolism\n",
      "R01868: (S)-dihydroorotate:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01868      Pyrimidine metabolism\n",
      "R01870: Orotidine-5'-phosphate:diphosphate phospho-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01870      Pyrimidine metabolism\n",
      "R01900: isocitrate hydro-lyase (cis-aconitate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R01900      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00630+R01900      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R01900      Carbon fixation pathways in prokaryotes\n",
      "R01920: S-adenosylmethioninamine:putrescine 3-aminopropyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01920      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R01920      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00480+R01920      Glutathione metabolism\n",
      "R01931: thiosulfate:cyanide sulfurtranserase\n",
      "     https://www.kegg.jp/pathway/rn00920+R01931      Sulfur metabolism\n",
      "R01933: 2-oxoadipate dehydrogenase complex\n",
      "     https://www.kegg.jp/pathway/rn00310+R01933      Lysine degradation\n",
      "     https://www.kegg.jp/pathway/rn00380+R01933      Tryptophan metabolism\n",
      "R01954: L-Citrulline:L-aspartate ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R01954      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R01954      Alanine, aspartate and glutamate metabolism\n",
      "R01968: 2'-deoxyguanosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01968      Purine metabolism\n",
      "R01993: (S)-dihydroorotate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01993      Pyrimidine metabolism\n",
      "R02003: Geranyl-diphosphate:isopentenyl-diphosphate geranyltrans-transferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R02003      Terpenoid backbone biosynthesis\n",
      "R02017: 2'-deoxyadenosine 5'-diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02017      Purine metabolism\n",
      "R02018: 2'-deoxyuridine 5'-diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02018      Pyrimidine metabolism\n",
      "R02019: 2'-deoxyguanosine 5'-diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02019      Purine metabolism\n",
      "R02021: adenosine 3',5'-bisphosphate,sulfite:oxidized-thioredoxin oxidoreductase (3'-phosphoadenosine-5'-phosphosulfate -forming)\n",
      "     https://www.kegg.jp/pathway/rn00920+R02021      Sulfur metabolism\n",
      "R02024: 2'-deoxycytidine diphosphate:oxidized-thioredoxin 2'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02024      Pyrimidine metabolism\n",
      "R02029: Phosphatidylglycerophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02029      Glycerophospholipid metabolism\n",
      "R02030: CDP-diacylglycerol:phosphatidylglycerol 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02030      Glycerophospholipid metabolism\n",
      "R02035: 6-Phospho-D-glucono-1,5-lactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00030+R02035      Pentose phosphate pathway\n",
      "R02036: 6-Phospho-D-gluconate hydro-lyase(2-dehydro-3-deoxy-6-phospho-D-gluconate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R02036      Pentose phosphate pathway\n",
      "R02055: Phsophatidyl-L-serine carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02055      Glycerophospholipid metabolism\n",
      "R02056: S-adenosyl-L-methionine:phosphatidylethanolamine N-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02056      Glycerophospholipid metabolism\n",
      "R02060: D-Glucosamine 1-phosphate 1,6-phosphomutase;\n",
      "     https://www.kegg.jp/pathway/rn00520+R02060      Amino sugar and nucleotide sugar metabolism\n",
      "R02088: 2'-deoxyadenosine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02088      Purine metabolism\n",
      "R02090: ATP:dGMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02090      Purine metabolism\n",
      "R02093: ATP:dTDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02093      Pyrimidine metabolism\n",
      "R02094: ATP:dTMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02094      Pyrimidine metabolism\n",
      "R02098: ATP:dUMP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02098      Pyrimidine metabolism\n",
      "R02100: dUTP nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02100      Pyrimidine metabolism\n",
      "R02101: 5,10-Methylenetetrahydrofolate:dUMP C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02101      Pyrimidine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R02101      One carbon pool by folate\n",
      "R02102: 2'-deoxyuridine 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02102      Pyrimidine metabolism\n",
      "R02111: 1,4-alpha-D-Glucan:orthophosphate alpha-D-glucosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R02111      Starch and sucrose metabolism\n",
      "R02142: XMP:pyrophosphate phosphoribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02142      Purine metabolism\n",
      "R02164: succinate:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00020+R02164      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R02164      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00650+R02164      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R02164      Carbon fixation pathways in prokaryotes\n",
      "R02199: L-Isoleucine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00280+R02199      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00290+R02199      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00966+R02199      Glucosinolate biosynthesis\n",
      "R02235: dihydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R02235      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R02235      Folate biosynthesis\n",
      "R02236: dihydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R02236      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00790+R02236      Folate biosynthesis\n",
      "R02237: 7,8-dihydropteroate:L-glutamate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R02237      Folate biosynthesis\n",
      "R02240: ATP:1,2-diacylglycerol 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02240      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R02240      Glycerophospholipid metabolism\n",
      "R02241: acyl-CoA:1-acyl-sn-glycerol-3-phosphate 2-O-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02241      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R02241      Glycerophospholipid metabolism\n",
      "R02272: (S)-4-Amino-5-oxopentanoate 4,5-aminomutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R02272      Porphyrin metabolism\n",
      "R02282: N2-Acetyl-L-ornithine:L-glutamate N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R02282      Arginine biosynthesis\n",
      "R02283: N2-Acetyl-L-ornithine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R02283      Arginine biosynthesis\n",
      "R02291: L-Aspartate-4-semialdehyde:NADP+ oxidoreductase (phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00260+R02291      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R02291      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00300+R02291      Lysine biosynthesis\n",
      "R02300: S-Aminomethyldihydrolipoylprotein:(6S)-tetrahydrofolate aminomethyltransferase (ammonia-forming)\n",
      "     https://www.kegg.jp/pathway/rn00670+R02300      One carbon pool by folate\n",
      "R02301: 5-Formyltetrahydrofolate cyclo-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00670+R02301      One carbon pool by folate\n",
      "R02322: nicotinamide-D-ribonucleotide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R02322      Nicotinate and nicotinamide metabolism\n",
      "R02323: nicotinamide ribonucleotide phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R02323      Nicotinate and nicotinamide metabolism\n",
      "R02325: dCTP aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02325      Pyrimidine metabolism\n",
      "R02326: ATP:dCDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02326      Pyrimidine metabolism\n",
      "R02328: dTTP:alpha-D-glucose-1-phosphate thymidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00521+R02328      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R02328      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00525+R02328      Acarbose and validamycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R02328      O-Antigen nucleotide sugar biosynthesis\n",
      "R02331: ATP:dUDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02331      Pyrimidine metabolism\n",
      "R02340: (1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate D-glyceraldehyde-3-phosphate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00400+R02340      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00402+R02340      Benzoxazinoid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00999+R02340      Biosynthesis of various plant secondary metabolites\n",
      "R02401: glutarate-semialdehyde:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00310+R02401      Lysine degradation\n",
      "R02404: Itaconate:CoA ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00660+R02404      C5-Branched dibasic acid metabolism\n",
      "R02408: L-cystine thiocysteine-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R02408      Cysteine and methionine metabolism\n",
      "R02412: ATP:shikimate 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00400+R02412      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02413: Shikimate:NADP+ 3-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00400+R02413      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02433: L-cysteate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R02433      Cysteine and methionine metabolism\n",
      "R02472: (R)-pantoate:NADP+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00770+R02472      Pantothenate and CoA biosynthesis\n",
      "R02473: (R)-Pantoate:beta-alanine ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00410+R02473      beta-Alanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00770+R02473      Pantothenate and CoA biosynthesis\n",
      "R02484: deoxyuridine:orthophosphate 2-deoxy-D-ribosyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00240+R02484      Pyrimidine metabolism\n",
      "R02530: (R)-S-Lactoylglutathione methylglyoxal-lyase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00620+R02530      Pyruvate metabolism\n",
      "R02550: toluene, ferredoxin:oxygen oxidoreductase (methyl-hydroxylating);\n",
      "     https://www.kegg.jp/pathway/rn00623+R02550      Toluene degradation\n",
      "R02558: (R)-Prunasin beta-D-glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R02558      Cyanoamino acid metabolism\n",
      "R02568: D-fructose 1-phosphate D-glyceraldehyde-3-phosphate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00051+R02568      Fructose and mannose metabolism\n",
      "R02619: 3-sulfino-L-alanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R02619      Cysteine and methionine metabolism\n",
      "R02649: ATP:N-acetyl-L-glutamate 5-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R02649      Arginine biosynthesis\n",
      "R02719: xanthosine 5'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02719      Purine metabolism\n",
      "R02720: XTP pyrophosphohydrolase;\n",
      "     https://www.kegg.jp/pathway/rn00230+R02720      Purine metabolism\n",
      "R02722: L-serine hydro-lyase [adding 1-C-(indol-3-yl)glycerol 3-phosphate; L-tryptophan and glyceraldehyde-3-phosphate-forming]\n",
      "     https://www.kegg.jp/pathway/rn00260+R02722      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00400+R02722      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R02734: N-Succinyl-LL-2,6-diaminoheptanedioate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00300+R02734      Lysine biosynthesis\n",
      "R02735: LL-2,6-Diaminoheptanedioate 2-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00300+R02735      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00470+R02735      D-Amino acid metabolism\n",
      "R02736: beta-D-glucose-6-phosphate:NADP+ 1-oxoreductase\n",
      "     https://www.kegg.jp/pathway/rn00030+R02736      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00480+R02736      Glutathione metabolism\n",
      "R02739: alpha-D-Glucose 6-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02739      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R02739      Pentose phosphate pathway\n",
      "R02740: alpha-D-Glucose 6-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02740      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R02740      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00520+R02740      Amino sugar and nucleotide sugar metabolism\n",
      "R02777: dTDP-beta-L-rhamnose:NADP+ 4-oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00521+R02777      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R02777      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R02777      O-Antigen nucleotide sugar biosynthesis\n",
      "R02783: UDP-N-acetylmuramoyl-L-alanine:D-glutamate ligase(ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00470+R02783      D-Amino acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R02783      Peptidoglycan biosynthesis\n",
      "R02788: UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:(L)-meso-2,6-diaminoheptanedioate gamma-ligase (ADP-forming);\n",
      "     https://www.kegg.jp/pathway/rn00300+R02788      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00550+R02788      Peptidoglycan biosynthesis\n",
      "R02864: S-Adenosyl-L-methionine:uroporphyrin-III C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R02864      Porphyrin metabolism\n",
      "R02874: squalene,NADPH-hemoprotein:oxygen oxidoreductase (2,3-epoxidizing)\n",
      "     https://www.kegg.jp/pathway/rn00100+R02874      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R02874      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R02922: Creatinine iminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R02922      Arginine and proline metabolism\n",
      "R02948: (S)-2-hydroxy-2-methyl-3-oxobutanoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00650+R02948      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R02948      C5-Branched dibasic acid metabolism\n",
      "R02964: NADH:menaquinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00130+R02964      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R02971: ATP:pantothenate 4'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R02971      Pantothenate and CoA biosynthesis\n",
      "R02985: amygdalin beta-glucosidase\n",
      "     https://www.kegg.jp/pathway/rn00460+R02985      Cyanoamino acid metabolism\n",
      "R03005: ATP:nicotinamide-nucleotide adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00760+R03005      Nicotinate and nicotinamide metabolism\n",
      "R03012: L-Histidinol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03012      Histidine metabolism\n",
      "R03015: formylmethanofuran:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R03015      Methane metabolism\n",
      "R03018: ATP:pantothenate 4'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03018      Pantothenate and CoA biosynthesis\n",
      "R03035: ATP:pantetheine-4'-phosphate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03035      Pantothenate and CoA biosynthesis\n",
      "R03051: 2,3-Dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/entry/R03051\n",
      "R03066: 6-(hydroxymethyl)-7,8-dihydropterin:4-aminobenzoate 2-amino-4-hydroxydihydropteridine-6-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R03066      Folate biosynthesis\n",
      "R03067: (7,8-dihydropterin-6-yl)methyl diphosphate:4-aminobenzoate 2-amino-4-hydroxy-7,8-dihydropteridine-6-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R03067      Folate biosynthesis\n",
      "R03083: 2-Dehydro-3-deoxy-D-arabino-heptonate 7-phosphate phosphate-lyase (cyclyzing)\n",
      "     https://www.kegg.jp/pathway/rn00400+R03083      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03084: 3-Dehydroquinate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00400+R03084      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03105: 3-mercaptopyruvate:sulfide sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R03105      Cysteine and methionine metabolism\n",
      "R03132: O-acetyl-L-serine:thiosulfate 2-amino-2-carboxyethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R03132      Cysteine and methionine metabolism\n",
      "R03140: D-Ribulose 1,5-bisphosphate oxygenase\n",
      "     https://www.kegg.jp/pathway/rn00630+R03140      Glyoxylate and dicarboxylate metabolism\n",
      "R03165: Hydroxymethylbilane hydro-lyase(cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00860+R03165      Porphyrin metabolism\n",
      "R03182: 7,8-Diaminononanoate:carbon-dioxide cyclo-ligase\n",
      "     https://www.kegg.jp/pathway/rn00780+R03182      Biotin metabolism\n",
      "R03191: UDP-N-acetylmuramate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00520+R03191      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R03191      Peptidoglycan biosynthesis\n",
      "R03192: UDP-N-acetylmuramate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00520+R03192      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00550+R03192      Peptidoglycan biosynthesis\n",
      "R03193: UDP-N-acetylmuramate:L-alanine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00550+R03193      Peptidoglycan biosynthesis\n",
      "R03194: S-adenosyl-L-methionine:uroporphyrinogen-III C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03194      Porphyrin metabolism\n",
      "R03197: Uroporphyrinogen-III carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03197      Porphyrin metabolism\n",
      "R03210: pimeloyl-CoA:L-alanine C-carboxyhexanoyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00780+R03210      Biotin metabolism\n",
      "R03220: Coproporphyrinogen:oxygen oxidoreductase(decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R03220      Porphyrin metabolism\n",
      "R03223: 2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:4-methyl-5-(2-phosphoethyl)-thiazole 2-methyl-4-aminopyridine-5-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R03223      Thiamine metabolism\n",
      "R03231: S-Adenosyl-L-methionine:8-amino-7-oxononanoate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R03231      Biotin metabolism\n",
      "R03236: ATP:D-tagatose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03236      Galactose metabolism\n",
      "R03243: 5-Amino-2-oxopentanoate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03243      Histidine metabolism\n",
      "R03254: phosphoenolpyruvate:D-arabinose-5-phosphate C-(1-carboxyvinyl)transferase (phosphate-hydrolysing, 2-carboxy-2-oxoethyl-forming);\n",
      "     https://www.kegg.jp/pathway/rn00540+R03254      Lipopolysaccharide biosynthesis\n",
      "R03269: N-[(R)-4'-Phosphopantothenoyl]-L-cysteine carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03269      Pantothenate and CoA biosynthesis\n",
      "R03291: trans-4-Hydroxy-L-proline:NAD+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03291      Arginine and proline metabolism\n",
      "R03293: trans-4-Hydroxy-L-proline:NADP+ 5-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03293      Arginine and proline metabolism\n",
      "R03313: L-glutamate-5-semialdehyde:NADP+ 5-oxidoreductase (phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00330+R03313      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00332+R03313      Carbapenem biosynthesis\n",
      "R03321: beta-D-Glucose 6-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00010+R03321      Glycolysis / Gluconeogenesis\n",
      "R03346: nicotinate D-ribonucleotide phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R03346      Nicotinate and nicotinamide metabolism\n",
      "R03348: Nicotinate-nucleotide:pyrophosphate phosphoribosyltransferase (carboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00760+R03348      Nicotinate and nicotinamide metabolism\n",
      "R03350: 3-Deoxy-D-manno-octulosonate-8-phosphate 8-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R03350      Lipopolysaccharide biosynthesis\n",
      "R03351: CTP:3-deoxy-D-manno-octulosonate cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R03351      Lipopolysaccharide biosynthesis\n",
      "R03390: Formylmethanofuran:5,6,7,8-tetrahydromethanopterin 5-formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R03390      Methane metabolism\n",
      "R03409: guanosine 3'-diphosphate 5'-triphosphate 5'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R03409      Purine metabolism\n",
      "R03424: S-adenosyl-L-methionine:phosphatidyl-N-methylethanolamine N-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R03424      Glycerophospholipid metabolism\n",
      "R03443: N-acetyl-L-glutamate-5-semialdehyde:NADP+ 5-oxidoreductase (phosphrylating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R03443      Arginine biosynthesis\n",
      "R03457: D-erythro-1-(Imidazol-4-yl)glycerol 3-phosphate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03457      Histidine metabolism\n",
      "R03458: 5-amino-6-(5-phosphoribitylamino)uracil:NADP+ 1'-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00740+R03458      Riboflavin metabolism\n",
      "R03459: 2,5-Diamino-6-hydroxy-4-(5-phosphoribosylamino)-pyrimidine 2-aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R03459      Riboflavin metabolism\n",
      "R03460: phosphoenolpyruvate:3-phosphoshikimate 5-O-(1-carboxyvinyl)-transferase\n",
      "     https://www.kegg.jp/pathway/rn00400+R03460      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03464: 5,10-Methenyltetrahydromethanopterin 10-hydrolase (decyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00680+R03464      Methane metabolism\n",
      "R03471: ATP:4-amino-5-hydroxymethyl-2-methylpyrimidine 5-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R03471      Thiamine metabolism\n",
      "R03472: 5-amino-1-(5-phospho-D-ribosyl)imidazole formate-lyase (decarboxylating, 4-amino-2-methyl-5-phosphomethylpyrimidine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00730+R03472      Thiamine metabolism\n",
      "R03503: ATP:6-hydroxymethyl-7,8-dihydropterin 6'-diphosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R03503      Folate biosynthesis\n",
      "R03504: 7,8-dihydroneopterin glycolaldehyde-lyase [6-(hydroxymethyl)-7,8-dihydropterin-forming]\n",
      "     https://www.kegg.jp/pathway/rn00790+R03504      Folate biosynthesis\n",
      "R03508: 1-(2-Carboxyphenylamino)-1-deoxy-D-ribulose-5-phosphate carboxy-lyase(cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00400+R03508      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03509: N-(5-Phospho-beta-D-ribosyl)anthranilate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00400+R03509      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R03530: ATP:dIDP phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R03530      Purine metabolism\n",
      "R03531: 2'-deoxyinosine-5'-triphosphate pyrophosphohydrolase;\n",
      "     https://www.kegg.jp/pathway/rn00230+R03531      Purine metabolism\n",
      "R03562: Toluene,NADPH:oxogen oxidoreductase (3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00623+R03562      Toluene degradation\n",
      "R03599: L-selenocysteine selenide-lyase (L-alanine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00450+R03599      Selenocompound metabolism\n",
      "R03689: lanosterol delta24-reductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R03689      Steroid biosynthesis\n",
      "R03816: NADH:phylloquinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00130+R03816      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R03896: (R)-2-Methylmalate hydro-lyase (2-methylmaleate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00290+R03896      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R03896      C5-Branched dibasic acid metabolism\n",
      "R03898: 2-methylmaleate hydratase;\n",
      "     https://www.kegg.jp/pathway/rn00290+R03898      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R03898      C5-Branched dibasic acid metabolism\n",
      "R03916: (5-glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R03916      Glutathione metabolism\n",
      "R03947: S-adenosyl-L-methionine:uroporphyrin-III C-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03947      Porphyrin metabolism\n",
      "R03948: S-adenosyl-L-methionine:precorrin-4 C20-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03948      Porphyrin metabolism\n",
      "R03968: 2-Isopropylmalate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R03968      Valine, leucine and isoleucine biosynthesis\n",
      "R03970: (5-glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03970      Cyanoamino acid metabolism\n",
      "R03971: gamma-glutamyl-beta-aminopropiononitrile amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03971      Cyanoamino acid metabolism\n",
      "R04001: 3-Isopropylmalate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R04001      Valine, leucine and isoleucine biosynthesis\n",
      "R04035: Phosphoribosyl-ATP pyrophosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00340+R04035      Histidine metabolism\n",
      "R04037: 1-(5-phospho-D-ribosyl)-AMP 1,6-hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00340+R04037      Histidine metabolism\n",
      "R04144: 5-Phospho-D-ribosylamine:glycine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04144      Purine metabolism\n",
      "R04148: Nicotinate-nucleotide:dimethylbenzimidazole phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R04148      Porphyrin metabolism\n",
      "R04173: 3-Phosphoserine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R04173      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R04173      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R04173      Methane metabolism\n",
      "R04198: 2,3,4,5-tetrahydrodipicolinate:NAD+ 4-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00261+R04198      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00300+R04198      Lysine biosynthesis\n",
      "R04199: 2,3,4,5-tetrahydrodipicolinate:NADP+ 4-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00261+R04199      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00300+R04199      Lysine biosynthesis\n",
      "R04208: 2-(Formamido)-N1-(5-phosphoribosyl)acetamidine cyclo-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04208      Purine metabolism\n",
      "R04231: (R)-4'-phosphopantothenate:L-cysteine ligase\n",
      "     https://www.kegg.jp/pathway/rn00770+R04231      Pantothenate and CoA biosynthesis\n",
      "R04286: 2-amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl) dihydropteridine triphosphate lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R04286      Folate biosynthesis\n",
      "R04292: glycerone phosphate:iminosuccinate alkyltransferase (cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00760+R04292      Nicotinate and nicotinamide metabolism\n",
      "R04355: Acyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04355      Fatty acid biosynthesis\n",
      "R04365: Succinyl-CoA:2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R04365      Lysine biosynthesis\n",
      "R04378: 1-(5'-Phosphoribosyl)-5-amino-4-imidazolecarboxamide:pyrophosphate phosphoribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04378      Purine metabolism\n",
      "R04391: ATP:pantothenate 4'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R04391      Pantothenate and CoA biosynthesis\n",
      "R04405: 5-Methyltetrahydropteroyltri-L-glutamate:L-homocysteine S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04405      Cysteine and methionine metabolism\n",
      "R04420: 5-methylthio-5-deoxy-D-ribose-1-phosphate ketol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04420      Cysteine and methionine metabolism\n",
      "R04425: (2S,3R)-3-Hydroxybutane-1,2,3-tricarboxylate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00640+R04425      Propanoate metabolism\n",
      "R04426: (2R,3S)-3-isopropylmalate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00290+R04426      Valine, leucine and isoleucine biosynthesis\n",
      "R04428: (3R)-3-hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04428      Fatty acid biosynthesis\n",
      "R04429: butyryl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04429      Fatty acid biosynthesis\n",
      "R04430: butyryl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04430      Fatty acid biosynthesis\n",
      "R04440: (R)-2,3-Dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00290+R04440      Valine, leucine and isoleucine biosynthesis\n",
      "R04441: (R)-2,3-Dihydroxy-3-methylbutanoate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R04441      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00770+R04441      Pantothenate and CoA biosynthesis\n",
      "R04457: 5-amino-6-(D-ribitylamino)uracil butanedionetransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R04457      Riboflavin metabolism\n",
      "R04463: 5'-Phosphoribosylformylglycinamide:L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04463      Purine metabolism\n",
      "R04475: N-Succinyl-L-2,6-diaminoheptanedioate:2-oxoglutarate amino-transferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R04475      Lysine biosynthesis\n",
      "R04509: ATP:4-amino-2-methyl-5-phosphomethylpyrimidine phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R04509      Thiamine metabolism\n",
      "R04533: (3R)-3-hydroxybutanoyl-[acyl-carrier protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04533      Fatty acid biosynthesis\n",
      "R04534: (3R)-3-hydroxydecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04534      Fatty acid biosynthesis\n",
      "R04535: (3R)-3-hydroxydecanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04535      Fatty acid biosynthesis\n",
      "R04536: (3R)-3-hydroxyoctanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04536      Fatty acid biosynthesis\n",
      "R04537: (3R)-3-hydroxyoctanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04537      Fatty acid biosynthesis\n",
      "R04543: (3R)-3-hydroxypalmitoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04543      Fatty acid biosynthesis\n",
      "R04544: (3R)-3-hydroxypalmitoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04544      Fatty acid biosynthesis\n",
      "R04549: UDP-2,3-bis[(3R)-3-hydroxymyristoyl]-alpha-D-glucosamine 2,3-bis[(3R)-3-hydroxymyristoyl]-beta-D-glucosaminyl 1-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04549      Lipopolysaccharide biosynthesis\n",
      "R04550: (3R)-3-hydroxymyristoyl-[acyl-carrier protein]:UDP-3-O-[(3R)-3-hydroxymyristoyl]-alpha-D-glucosamine N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04550      Lipopolysaccharide biosynthesis\n",
      "R04558: 5-[(5-phospho-1-deoxy-D-ribulos-1-ylamino)methylideneamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate-lyase (L-glutamine-hydrolysing; 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00340+R04558      Histidine metabolism\n",
      "R04559: 1-(5'-Phosphoribosyl)-5-amino-4-(N-succinocarboxamide)-imidazole AMP-lyase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04559      Purine metabolism\n",
      "R04560: 10-Formyltetrahydrofolate:5'-phosphoribosyl-5-amino-4-imidazolecarboxamide formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04560      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R04560      One carbon pool by folate\n",
      "R04566: (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04566      Fatty acid biosynthesis\n",
      "R04567: (R)-3-Hydroxytetradecanoly-[acyl-carrier-protein]:UDP-N-acetyl-glucosamine 3-O-(3-hydroxytetradecanoyl)transferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04567      Lipopolysaccharide biosynthesis\n",
      "R04568: (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04568      Fatty acid biosynthesis\n",
      "R04573: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-L-lysine:alanyl-D-alanine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00550+R04573      Peptidoglycan biosynthesis\n",
      "R04587: UDP-3-O-[(3R)-3-hydroxymyristoyl]-N-acetylglucosamine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04587      Lipopolysaccharide biosynthesis\n",
      "R04591: 1-(5-Phosphoribosyl)-5-amino-4-carboxyimidazole:L-aspartate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R04591      Purine metabolism\n",
      "R04594: \n",
      "     https://www.kegg.jp/pathway/rn00860+R04594      Porphyrin metabolism\n",
      "R04606: UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine:2,3-bis-(3-hydroxytetradecanoyl)-alpha-D-glucosaminyl-1-phosphate 2,3-bis(3-hydroxytetradecanoyl)-glucosaminyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04606      Lipopolysaccharide biosynthesis\n",
      "R04617: UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminoheptanedioate:D-alanyl-D-alanine ligase(ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00300+R04617      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00550+R04617      Peptidoglycan biosynthesis\n",
      "R04620: 7,8-dihydroneopterin-3'-triphosphate phosphohydrolase (alkaline optimum)\n",
      "     https://www.kegg.jp/pathway/rn00790+R04620      Folate biosynthesis\n",
      "R04640: N-(5'-phospho-D-ribosylformimino)-5-amino-1- (5''-phospho-D-ribosyl)-4-imidazolecarboxamide ketol-isomerase;\n",
      "     https://www.kegg.jp/pathway/rn00340+R04640      Histidine metabolism\n",
      "R04657: ATP:2,3,2',3'-tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-beta-D-1,6-glucosaminyl-alpha-phosphate 4-O'-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04657      Lipopolysaccharide biosynthesis\n",
      "R04658: CMP-3-deoxy-D-manno-oct-2-ulosonate:lipid IVA 3-deoxy-D-manno-oct-2-ulosonate transferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R04658      Lipopolysaccharide biosynthesis\n",
      "R04724: dodecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04724      Fatty acid biosynthesis\n",
      "R04725: dodecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04725      Fatty acid biosynthesis\n",
      "R04726: dodecanoyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04726      Fatty acid biosynthesis\n",
      "R04779: ATP:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R04779      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R04779      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R04779      Fructose and mannose metabolism\n",
      "R04880: 3,4-dihydroxyphenylethyleneglycol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00350+R04880      Tyrosine metabolism\n",
      "R04929: ATP:selenate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00450+R04929      Selenocompound metabolism\n",
      "R04941: Selenocystathionine L-homocysteine-lyase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00450+R04941      Selenocompound metabolism\n",
      "R04952: butyryl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04952      Fatty acid biosynthesis\n",
      "R04953: (3R)-3-hydroxyhexanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04953      Fatty acid biosynthesis\n",
      "R04954: (3R)-3-hydroxyhexanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04954      Fatty acid biosynthesis\n",
      "R04955: hexanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04955      Fatty acid biosynthesis\n",
      "R04956: hexanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04956      Fatty acid biosynthesis\n",
      "R04957: hexanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04957      Fatty acid biosynthesis\n",
      "R04958: octanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04958      Fatty acid biosynthesis\n",
      "R04959: octanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04959      Fatty acid biosynthesis\n",
      "R04960: Octanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04960      Fatty acid biosynthesis\n",
      "R04961: decanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04961      Fatty acid biosynthesis\n",
      "R04962: decanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04962      Fatty acid biosynthesis\n",
      "R04963: Decanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04963      Fatty acid biosynthesis\n",
      "R04964: (3R)-3-hydroxydodecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04964      Fatty acid biosynthesis\n",
      "R04965: (3R)-3-hydroxydodecanoyl-[acyl-carrier-protein] hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04965      Fatty acid biosynthesis\n",
      "R04966: tetradecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04966      Fatty acid biosynthesis\n",
      "R04967: tetradecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04967      Fatty acid biosynthesis\n",
      "R04968: Tetradecanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R04968      Fatty acid biosynthesis\n",
      "R04969: hexadecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04969      Fatty acid biosynthesis\n",
      "R04970: hexadecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04970      Fatty acid biosynthesis\n",
      "R04972: Uroporphyrinogen I carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R04972      Porphyrin metabolism\n",
      "R04986: 3-octaprenyl-4-hydroxybenzoate carboxy-lyase\n",
      "     https://www.kegg.jp/entry/R04986\n",
      "R04988: S-adenosyl-L-methionine:3-(all-trans-octaprenyl)benzene-1,2-diol 2-O-methyltransferase\n",
      "     https://www.kegg.jp/entry/R04988\n",
      "R05032: UDP-N-acetyl-D-glucosamine:undecaprenyl-diphospho-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine 4-beta-N-acetylglucosaminlytransferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05032      Peptidoglycan biosynthesis\n",
      "R05046: formamidopyrimidine nucleoside triphosphate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00790+R05046      Folate biosynthesis\n",
      "R05052: L-erythro-4-hydroxyglutamate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R05052      Arginine and proline metabolism\n",
      "R05053: D-aspartate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00330+R05053      Arginine and proline metabolism\n",
      "R05068: (R)-2,3-Dihydroxy-3-methylpentanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00290+R05068      Valine, leucine and isoleucine biosynthesis\n",
      "R05069: (S)-2-Aceto-2-hydroxybutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00290+R05069      Valine, leucine and isoleucine biosynthesis\n",
      "R05070: (R)-2,3-Dihydroxy-3-methylpentanoate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00290+R05070      Valine, leucine and isoleucine biosynthesis\n",
      "R05071: (S)-2-Acetolactate methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00290+R05071      Valine, leucine and isoleucine biosynthesis\n",
      "R05074: CMP-3-deoxy-D-manno-oct-2-ulosonate:(KDO)-lipid IVA 3-deoxy-D-manno-oct-2-ulosonate transferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05074      Lipopolysaccharide biosynthesis\n",
      "R05085: O-Phospho-4-hydroxy-L-threonine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00750+R05085      Vitamin B6 metabolism\n",
      "R05086: O-Phospho-4-hydroxy-L-threonine phospho-lyase (adding water)\n",
      "     https://www.kegg.jp/pathway/rn00750+R05086      Vitamin B6 metabolism\n",
      "R05149: S-Adenosyl-L-methionine:1-precorrin-6Y C5,15-methyltransferase (C-12-decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05149      Porphyrin metabolism\n",
      "R05177: precorrin 8X 11,12-methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05177      Porphyrin metabolism\n",
      "R05180: S-adenosyl-L-methionine:precorrin-3B C17-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05180      Porphyrin metabolism\n",
      "R05181: S-adenosyl-L-methionine:precorrin-4 C11 methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05181      Porphyrin metabolism\n",
      "R05185: reduced ferredoxin:dinitrogen oxidoreductase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00910+R05185      Nitrogen metabolism\n",
      "R05220: ATP:cob(I)yrinic acid-a,c-diamide Cobeta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05220      Porphyrin metabolism\n",
      "R05221: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05221      Porphyrin metabolism\n",
      "R05222: GTP:adenosylcobinamide-phosphate guanylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05222      Porphyrin metabolism\n",
      "R05223: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05223      Porphyrin metabolism\n",
      "R05224: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05224      Porphyrin metabolism\n",
      "R05225: \n",
      "     https://www.kegg.jp/pathway/rn00860+R05225      Porphyrin metabolism\n",
      "R05233: trans-3-Chloro-2-propene-1-ol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00625+R05233      Chloroalkane and chloroalkene degradation\n",
      "R05234: cis-3-chloro-2-propene-1-ol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00625+R05234      Chloroalkane and chloroalkene degradation\n",
      "R05332: Acetyl-CoA:D-glucosamine-1-phosphate N-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R05332      Amino sugar and nucleotide sugar metabolism\n",
      "R05338: D-arabino-hex-3-ulose-6-phosphate formaldehyde-lyase (D-ribulose-5-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R05338      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00680+R05338      Methane metabolism\n",
      "R05339: D-arabino-hex-3-ulose-6-phosphate isomerase\n",
      "     https://www.kegg.jp/pathway/rn00680+R05339      Methane metabolism\n",
      "R05496: reduced ferredoxin:acetylene oxidoreductase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00625+R05496      Chloroalkane and chloroalkene degradation\n",
      "R05553: 4-amino-4-deoxychorismate pyruvate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R05553      Folate biosynthesis\n",
      "R05605: 2-dehydro-3-deoxy-6-phospho-D-gluconate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00030+R05605      Pentose phosphate pathway\n",
      "R05615: all-trans-octaprenyl-diphosphate:4-hydroxybenzoate 3-octaprenyltransferase\n",
      "     https://www.kegg.jp/entry/R05615\n",
      "R05627: undecaprenyl-diphosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05627      Peptidoglycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00552+R05627      Teichoic acid biosynthesis\n",
      "R05629: UDPMurAc(oyl-L-Ala-D-gamma-Glu-L-Lys-D-Ala-D-Ala):undecaprenyl-phosphate phospho-N-acetylmuramoyl-pentapeptide-transferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05629      Peptidoglycan biosynthesis\n",
      "R05630: UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine:undecaprenyl-phosphate phospho-N-acetylmuramoyl-pentapeptide-transferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05630      Peptidoglycan biosynthesis\n",
      "R05633: CTP: 2-C-Methyl-D-erythritol 4-phosphate cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05633      Terpenoid backbone biosynthesis\n",
      "R05634: ATP:4-(Cytidine 5'-diphospho)-2-C-methyl-D-erythritol 2-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05634      Terpenoid backbone biosynthesis\n",
      "R05636: 1-Deoxy-D-xylulose-5-phosphate pyruvate-lyase (carboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00730+R05636      Thiamine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00900+R05636      Terpenoid backbone biosynthesis\n",
      "R05637: 2-Phospho-4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol CMP-lyase (cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00900+R05637      Terpenoid backbone biosynthesis\n",
      "R05640: lanosterol,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R05640      Steroid biosynthesis\n",
      "R05644: ATP:D-glycero-beta-D-manno-heptose 1-phosphate adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05644      Lipopolysaccharide biosynthesis\n",
      "R05645: D-glycero-beta-D-manno-heptose-7-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05645      Lipopolysaccharide biosynthesis\n",
      "R05646: ATP:D-glycero-beta-D-manno-heptose 7-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05646      Lipopolysaccharide biosynthesis\n",
      "R05647: D-glycero-beta-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05647      Lipopolysaccharide biosynthesis\n",
      "R05662: UDP-N-acetyl-D-glucosamine:N-acetyl-alpha-D-muramyl(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-diphosphoundecaprenol 4-beta-N-acetylglucosaminlytransferase\n",
      "     https://www.kegg.jp/pathway/rn00550+R05662      Peptidoglycan biosynthesis\n",
      "R05681: NAD-dependent threonine 4-phosphate dehydrogenase\n",
      "     https://www.kegg.jp/pathway/rn00750+R05681      Vitamin B6 metabolism\n",
      "R05688: 1-Deoxy-D-xylulose-5-phosphate isomeroreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05688      Terpenoid backbone biosynthesis\n",
      "R05692: GDP-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing)\n",
      "     https://www.kegg.jp/pathway/rn00051+R05692      Fructose and mannose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R05692      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R05692      O-Antigen nucleotide sugar biosynthesis\n",
      "R05703: lanosterol D24-reductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R05703      Steroid biosynthesis\n",
      "R05731: obtusifoliol,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R05731      Steroid biosynthesis\n",
      "R05807: sirohydrochlorin cobalt-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05807      Porphyrin metabolism\n",
      "R05808: S-adenosyl-L-methionine:cobalt-factor-II C20-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05808      Porphyrin metabolism\n",
      "R05809: \n",
      "     https://www.kegg.jp/entry/R05809\n",
      "R05810: cobalt-precorrin-4 methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05810      Porphyrin metabolism\n",
      "R05814: precorrin-8 11,12-methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05814      Porphyrin metabolism\n",
      "R05815: cobyrinate:L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05815      Porphyrin metabolism\n",
      "R05838: pyridoxine 5'-phosphate synthase\n",
      "     https://www.kegg.jp/pathway/rn00750+R05838      Vitamin B6 metabolism\n",
      "R05884: isopentenyl-diphosphate:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R05884      Terpenoid backbone biosynthesis\n",
      "R06447: (2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate cistransferase (adding 8 isopentenyl units)\n",
      "     https://www.kegg.jp/pathway/rn00550+R06447      Peptidoglycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00900+R06447      Terpenoid backbone biosynthesis\n",
      "R06513: dTDPglucose 4,6-hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00521+R06513      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R06513      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00525+R06513      Acarbose and validamycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R06513      O-Antigen nucleotide sugar biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn01055+R06513      Biosynthesis of vancomycin group antibiotics\n",
      "R06514: dTDP-4-dehydro-6-deoxy-alpha-D-glucose 3,5-epimerase;\n",
      "     https://www.kegg.jp/pathway/rn00521+R06514      Streptomycin biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00523+R06514      Polyketide sugar unit biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00541+R06514      O-Antigen nucleotide sugar biosynthesis\n",
      "R06529: adenosylcobyric acid:(R)-1-aminopropan-2-yl phosphate ligase\n",
      "     https://www.kegg.jp/pathway/rn00860+R06529      Porphyrin metabolism\n",
      "R06558: \n",
      "     https://www.kegg.jp/pathway/rn00860+R06558      Porphyrin metabolism\n",
      "R06782: 3-phenylpropanoate,NADH:oxygen oxidoreductase (2,3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00360+R06782      Phenylalanine metabolism\n",
      "R06783: trans-cinnamate,NADH:oxygen oxidoreductase (2,3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00360+R06783      Phenylalanine metabolism\n",
      "R06895: coproporphyrinogen-III:S-adenosyl-L-methionine oxidoreductase(decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R06895      Porphyrin metabolism\n",
      "R06917: 1-hydroxymethylnaphthalene:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00626+R06917      Naphthalene degradation\n",
      "R06927: (2-naphthyl)methanol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00626+R06927      Naphthalene degradation\n",
      "R07136: (2R)-3-sulfolactate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00270+R07136      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R07136      Methane metabolism\n",
      "R07168: 5-methyltetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R07168      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R07168      Carbon fixation pathways in prokaryotes\n",
      "R07268: ATP:cobinamide Cobeta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07268      Porphyrin metabolism\n",
      "R07281: D-ribulose 5-phosphate formate-lyase (L-3,4-dihydroxybutan-2-one 4-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00740+R07281      Riboflavin metabolism\n",
      "R07302: adenosylcobyric acid:(R)-1-aminopropan-2-ol ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07302      Porphyrin metabolism\n",
      "R07322: squalene mutase (cyclizing)\n",
      "     https://www.kegg.jp/pathway/rn00909+R07322      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R07323: squalene mutase (cyclizing);\n",
      "     https://www.kegg.jp/pathway/rn00909+R07323      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R07363: 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one:oxygen oxidoreductase (formate- and CO-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R07363      Cysteine and methionine metabolism\n",
      "R07364: 1,2-dihydroxy-5-(methylthio)pent-1-en-3-one:oxygen oxidoreductase (formate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R07364      Cysteine and methionine metabolism\n",
      "R07392: S-methyl-5-thio-D-ribulose-1-phosphate hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00270+R07392      Cysteine and methionine metabolism\n",
      "R07395: 5-(methylthio)-2,3-dioxopentyl-phosphate phosphohydrolase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00270+R07395      Cysteine and methionine metabolism\n",
      "R07404: 5-amino-1-(5-phospho-D-ribosyl)imidazole:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R07404      Purine metabolism\n",
      "R07405: 5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole carboxymutase\n",
      "     https://www.kegg.jp/pathway/rn00230+R07405      Purine metabolism\n",
      "R07411: (2E,6E)-farnesyl-diphosphate:protoheme IX farnesyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07411      Porphyrin metabolism\n",
      "R07412: ferroheme o:acceptor C-81-oxidoreductase (heme a-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07412      Porphyrin metabolism\n",
      "R07463: glycine:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00730+R07463      Thiamine metabolism\n",
      "R07488: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07488      Steroid biosynthesis\n",
      "R07493: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07493      Steroid biosynthesis\n",
      "R07498: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07498      Steroid biosynthesis\n",
      "R07499: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07499      Steroid biosynthesis\n",
      "R07507: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07507      Steroid biosynthesis\n",
      "R07605: 7-aminomethyl-7-carbaguanine:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R07605      Folate biosynthesis\n",
      "R07639: decenoyl-[acyl-carrier-protein] Delta2-trans-Delta3-cis-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00061+R07639      Fatty acid biosynthesis\n",
      "R07762: \n",
      "     https://www.kegg.jp/pathway/rn00061+R07762      Fatty acid biosynthesis\n",
      "R07763: \n",
      "     https://www.kegg.jp/pathway/rn00061+R07763      Fatty acid biosynthesis\n",
      "R07764: \n",
      "     https://www.kegg.jp/pathway/rn00061+R07764      Fatty acid biosynthesis\n",
      "R07765: octadecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R07765      Fatty acid biosynthesis\n",
      "R07773: S-adenosyl-L-methionine:cobalt-precorrin-5B (C1)-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07773      Porphyrin metabolism\n",
      "R08034: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08034      Nitrotoluene degradation\n",
      "R08042: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08042      Nitrotoluene degradation\n",
      "R08058: 5,6,7,8-tetrahydromethanopterin hydro-lyase (formaldehyde-adding, 5,10-methylenetetrahydromethanopterin-forming)\n",
      "     https://www.kegg.jp/pathway/rn00680+R08058      Methane metabolism\n",
      "R08059: \n",
      "     https://www.kegg.jp/pathway/rn00680+R08059      Methane metabolism\n",
      "R08060: \n",
      "     https://www.kegg.jp/pathway/rn00680+R08060      Methane metabolism\n",
      "R08210: dimethylallyl diphosphate:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R08210      Terpenoid backbone biosynthesis\n",
      "R08359: S-adenosylmethioninamine:cadaverine 3-aminopropyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R08359      Glutathione metabolism\n",
      "R08549: 2-oxoglutarate:NAD+ 2-oxidoreductase (CoA-succinylating);\n",
      "     https://www.kegg.jp/entry/R08549\n",
      "R08639: alpha-D-glucose 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00500+R08639      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00521+R08639      Streptomycin biosynthesis\n",
      "R08648: pyruvate:2-oxobutanoate acetaldehydetransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00290+R08648      Valine, leucine and isoleucine biosynthesis\n",
      "R08689: (E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized ferredoxin oxidoreductase (hydrating)\n",
      "     https://www.kegg.jp/pathway/rn00900+R08689      Terpenoid backbone biosynthesis\n",
      "R08968: ATP:N-acetyl-D-glucosamine 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R08968      Amino sugar and nucleotide sugar metabolism\n",
      "R09030: D-allose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00051+R09030      Fructose and mannose metabolism\n",
      "R09081: carboxyspermidine carboxy-lyase (spermidine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00330+R09081      Arginine and proline metabolism\n",
      "R09082: carboxynorspermidine carboxy-lyase (norspermidine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00330+R09082      Arginine and proline metabolism\n",
      "R09099: 5,10-methylenetetrahydromethanopterin:glycine hydroxymethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R09099      Methane metabolism\n",
      "R09107: N-acetyl-L-citrulline amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R09107      Arginine biosynthesis\n",
      "R09127: ethanol:cytochrome c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R09127      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R09127      Pyruvate metabolism\n",
      "R09128: 2-chloroethanol:cytochrome c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00625+R09128      Chloroalkane and chloroalkene degradation\n",
      "R09248: (2E,6E)-farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase (adding 5 isopentenyl units)\n",
      "     https://www.kegg.jp/pathway/rn00900+R09248      Terpenoid backbone biosynthesis\n",
      "R09365: 5-methyltetrahydropteroyltri-L-glutamate:L-selenohomocysteine Se-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00450+R09365      Selenocompound metabolism\n",
      "R09394: GTP 3',8-cyclase [(8S)-3',8-cyclo-7,8-dihydroguanosine 5'-triphosphate-forming]\n",
      "     https://www.kegg.jp/pathway/rn00790+R09394      Folate biosynthesis\n",
      "R09518: methane,quinol:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R09518      Methane metabolism\n",
      "R09543: S-adenosyl-L-methionine:malonyl-[acyl-carrier protein] O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R09543      Biotin metabolism\n",
      "R09660: S-methyl-5'-thioadenosine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00270+R09660      Cysteine and methionine metabolism\n",
      "R09668: S-methyl-5'-thioinosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R09668      Cysteine and methionine metabolism\n",
      "R09725: pimeloyl-[acyl-carrier protein]-methyl-ester hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00780+R09725      Biotin metabolism\n",
      "R09769: D-glycero-alpha-D-manno-heptose 7-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R09769      Lipopolysaccharide biosynthesis\n",
      "R09771: D-glycero-alpha-D-manno-heptose 1,7-bisphosphate 7-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00540+R09771      Lipopolysaccharide biosynthesis\n",
      "R09780: D-arabino-hex-3-ulose-6-phosphate isomerase\n",
      "     https://www.kegg.jp/pathway/rn00030+R09780      Pentose phosphate pathway\n",
      "R09959: 7,8-dihydroneopterin 3'-triphosphate acetaldehyde-lyase (6-carboxy-5,6,7,8-tetrahydropterin and triphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09959      Folate biosynthesis\n",
      "R09978: 7-carboxy-7-carbaguanine:ammonia ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09978      Folate biosynthesis\n",
      "R10002: 6-carboxy-5,6,7,8-tetrahydropterin ammonia-lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R10002      Folate biosynthesis\n",
      "R10035: beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R10035      Cyanoamino acid metabolism\n",
      "R10039: beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R10039      Cyanoamino acid metabolism\n",
      "R10040: beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R10040      Cyanoamino acid metabolism\n",
      "R10092: carbonate hydro-lyase (carbon-dioxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00910+R10092      Nitrogen metabolism\n",
      "R10115: malonyl-[acyl-carrier protein]-methyl-ester:malonyl-[acyl-carrier protein] C-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10115      Biotin metabolism\n",
      "R10116: 3-hydroxyglutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10116      Biotin metabolism\n",
      "R10117: 3-hydroxyglutaryl-[acp]-methyl-ester hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10117      Biotin metabolism\n",
      "R10118: glutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10118      Biotin metabolism\n",
      "R10119: glutaryl-[acp]-methyl-ester:malonyl-[acyl-carrier protein] C-acyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00780+R10119      Biotin metabolism\n",
      "R10120: 3-hydroxypimeloyl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10120      Biotin metabolism\n",
      "R10121: 3-hydroxypimeloyl-[acp]-methyl-ester hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10121      Biotin metabolism\n",
      "R10122: pimelyl-[acyl-carrier protein]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10122      Biotin metabolism\n",
      "R10124: pimeloyl-[acyl-carrier protein]:L-alanine C-pimeloyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00780+R10124      Biotin metabolism\n",
      "R10147: L-aspartate-4-semialdehyde hydro-lyase [adding pyruvate and cyclizing; (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate-forming]\n",
      "     https://www.kegg.jp/pathway/rn00261+R10147      Monobactam biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00300+R10147      Lysine biosynthesis\n",
      "R10151: \n",
      "     https://www.kegg.jp/pathway/rn00920+R10151      Sulfur metabolism\n",
      "R10164: hydroxylamine:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R10164      Nitrogen metabolism\n",
      "R10221: 6-phospho-D-gluconate:NAD+ 2-oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00030+R10221      Pentose phosphate pathway\n",
      "R10707: acetyl-CoA:malonyl-[acyl-carrier protein] C-acetyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R10707      Fatty acid biosynthesis\n",
      "R10712: 2-methyl-4-amino-5-hydroxymethylpyrimidine-diphosphate:2-(2-carboxy-4-methylthiazol-5-yl)ethyl phosphate 2-methyl-4-aminopyrimidine-5-methenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R10712      Thiamine metabolism\n",
      "R10859: (E)-4-hydroxy-3-methylbut-2-en-1-yl-diphosphate:oxidized flavodoxin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00900+R10859      Terpenoid backbone biosynthesis\n",
      "R10866: pyruvate:flavodoxin 2-oxidoreductase (CoA-acetylating)\n",
      "     https://www.kegg.jp/pathway/rn00620+R10866      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R10866      Carbon fixation pathways in prokaryotes\n",
      "R10907: beta-D-glucose-6-phosphate:NAD+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00030+R10907      Pentose phosphate pathway\n",
      "R10991: 2-aminobutanoate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R10991      Cysteine and methionine metabolism\n",
      "R10993: L-glutamate:2-aminobutanoate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R10993      Cysteine and methionine metabolism\n",
      "R10994: gamma-L-glutamyl-L-2-aminobutyrate:glycine ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R10994      Cysteine and methionine metabolism\n",
      "R11024: ATP:N-acetyl-D-muramate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R11024      Amino sugar and nucleotide sugar metabolism\n",
      "R11025: UTP:N-acetyl-alpha-D-muramate-1-phosphate uridylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R11025      Amino sugar and nucleotide sugar metabolism\n",
      "R11096: \n",
      "     https://www.kegg.jp/pathway/rn00100+R11096      Steroid biosynthesis\n",
      "R01652: \n",
      "     https://www.kegg.jp/pathway/rn00290+R01652      Valine, leucine and isoleucine biosynthesis\n",
      "R04639: 2-amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl) dihydropteridine triphosphate hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00790+R04639      Folate biosynthesis\n",
      "R05048: 2,5-diaminopyrimidine nucleoside triphosphate mutase\n",
      "     https://www.kegg.jp/pathway/rn00790+R05048      Folate biosynthesis\n"
     ]
    }
   ],
   "source": [
    "inmodel=[r for r in  Mca_kegglist if r in mcoosmodel_rxnlist]\n",
    "print(len(inmodel))\n",
    "print('=====================')\n",
    "for r in inmodel:\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "508\n",
      "=====================\n",
      "R00024: 3-phospho-D-glycerate carboxy-lyase (dimerizing; D-ribulose-1,5-bisphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00710+R00024      Carbon fixation in photosynthetic organisms\n",
      "R00081: ferrocytochrome-c:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00081\n",
      "R00089: ATP diphosphate-lyase (cyclizing; 3',5'-cyclic-AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R00089      Purine metabolism\n",
      "R00103: NAD+ phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R00103      Nicotinate and nicotinamide metabolism\n",
      "R00134: formate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00134      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00134      Carbon fixation pathways in prokaryotes\n",
      "R00182: AMP phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00182      Purine metabolism\n",
      "R00188: adenosine-3',5'-bisphosphate 3'-phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R00188\n",
      "R00191: adenosine 3',5'-phosphate 5'-nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00191      Purine metabolism\n",
      "R00237: (3S)-citramalyl-CoA pyruvate-lyase (acetyl-CoA-forming)\n",
      "     https://www.kegg.jp/pathway/rn00660+R00237      C5-Branched dibasic acid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00237      Carbon fixation pathways in prokaryotes\n",
      "R00251: 5-oxo-L-proline amidohydrolase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00251      Glutathione metabolism\n",
      "R00306: cellobiose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00306      Starch and sucrose metabolism\n",
      "R00434: GTP diphosphate-lyase (cyclizing; 3',5'-cyclic-GMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R00434      Purine metabolism\n",
      "R00476: glycolate:acceptor 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00630+R00476      Glyoxylate and dicarboxylate metabolism\n",
      "R00510: cytidine-5'-monophosphate phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00510      Pyrimidine metabolism\n",
      "R00597: S-adenosyl-L-methionine:tRNA guanine N1-methyltransferase;\n",
      "     https://www.kegg.jp/entry/R00597\n",
      "R00615: Thiamin diphosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00730+R00615      Thiamine metabolism\n",
      "R00700: hydrogen:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00700\n",
      "R00746: Ethanol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00746      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00746      Pyruvate metabolism\n",
      "R00760: ATP:D-fructose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00760      Starch and sucrose metabolism\n",
      "R00761: D-fructose-6-phosphate D-erythrose-4-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00710+R00761      Carbon fixation in photosynthetic organisms\n",
      "R00764: diphosphate:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R00764\n",
      "R00789: ammonia:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00789      Nitrogen metabolism\n",
      "R00798: nitrite:acceptor oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00798      Nitrogen metabolism\n",
      "R00848: sn-Glycerol-3-phosphate:(acceptor) 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00564+R00848      Glycerophospholipid metabolism\n",
      "R00867: ATP:D-fructose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00051+R00867      Fructose and mannose metabolism\n",
      "R00925: Propanoate:CoA ligase (AMP-forming)\n",
      "     https://www.kegg.jp/entry/R00925\n",
      "R01009: GDPmannose:dolichyl-phosphate O-beta-D-mannosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00510+R01009      N-Glycan biosynthesis\n",
      "R01012: phosphoenolpyruvate:glycerone phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01012      Glycerolipid metabolism\n",
      "R01041: glycerol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01041      Glycerolipid metabolism\n",
      "R01122: delta2-isopentenyl-diphosphate:tRNA isopentenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00908+R01122      Zeatin biosynthesis\n",
      "R01158: L-histidinol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/entry/R01158\n",
      "R01220: 5,10-methylenetetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R01220      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R01220      Carbon fixation pathways in prokaryotes\n",
      "R01238: 4-hydroxybenzoate carboxy-lyase (phenol-forming)\n",
      "     https://www.kegg.jp/pathway/rn00627+R01238      Aminobenzoate degradation\n",
      "R01262: glutathione:L-amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R01262      Glutathione metabolism\n",
      "R01274: palmitoyl-CoA hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00062+R01274      Fatty acid elongation\n",
      "     https://www.kegg.jp/pathway/rn01040+R01274      Biosynthesis of unsaturated fatty acids\n",
      "R01324: citrate hydroxymutase\n",
      "     https://www.kegg.jp/entry/R01324\n",
      "R01344: D-alanine:2-oxo acid aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R01344      D-Amino acid metabolism\n",
      "R01347: Fatty acid,reduced rubredoxin:oxygen oxidoreductase (omega-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00071+R01347      Fatty acid degradation\n",
      "R01351: 1-acylglycerol acylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01351      Glycerolipid metabolism\n",
      "R01378: Phenylpyruvate keto-enol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00360+R01378      Phenylalanine metabolism\n",
      "R01395: ATP:carbamate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00910+R01395      Nitrogen metabolism\n",
      "R01402: S-methyl-5'-thioadenosine:phosphate S-methyl-5-thio-alpha-D-ribosyl-transferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01402      Cysteine and methionine metabolism\n",
      "R01403: acyl-[acyl-carrier-protein]:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01403      Fatty acid biosynthesis\n",
      "R01404: acyl-[acyl-carrier protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01404      Fatty acid biosynthesis\n",
      "R01429: D-xylose:NAD+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00040+R01429      Pentose and glucuronate interconversions\n",
      "R01430: D-xylose:NADP+ 1-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00040+R01430      Pentose and glucuronate interconversions\n",
      "R01515: 3-Phospho-D-glyceroyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R01515\n",
      "R01523: ATP:D-ribulose-5-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00710+R01523      Carbon fixation in photosynthetic organisms\n",
      "R01560: Adenosine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01560      Purine metabolism\n",
      "R01561: adenosine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01561      Purine metabolism\n",
      "R01562: adenosine 3'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01562      Purine metabolism\n",
      "R01625: CoA:apo-[acyl-carrier-protein] pantetheinephosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00770+R01625      Pantothenate and CoA biosynthesis\n",
      "R01676: Guanine aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01676      Purine metabolism\n",
      "R01687: (5-L-glutamyl)-peptide:taurine 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00430+R01687      Taurine and hypotaurine metabolism\n",
      "R01699: pyruvate:[dihydrolipoyllysine-residue acetyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-acetylating)\n",
      "     https://www.kegg.jp/entry/R01699\n",
      "R01700: 2-oxoglutarate:[dihydrolipoyllysine-residue succinyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-succinylating)\n",
      "     https://www.kegg.jp/entry/R01700\n",
      "R01776: Acetyl-CoA:L-homoserine O-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R01776      Cysteine and methionine metabolism\n",
      "R01812: 5,6,7,8-tetrahydrobiopterin:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R01812      Folate biosynthesis\n",
      "R01829: sedoheptulose 1,7-bisphosphate D-glyceraldehyde-3-phosphate-lyase\n",
      "     https://www.kegg.jp/pathway/rn00710+R01829      Carbon fixation in photosynthetic organisms\n",
      "R01851: Methanethiol:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R01851      Sulfur metabolism\n",
      "R01863: inosine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R01863      Purine metabolism\n",
      "R01876: uridine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01876      Pyrimidine metabolism\n",
      "R01877: uridine 3'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R01877      Pyrimidine metabolism\n",
      "R01939: L-2-aminoadipate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R01939      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00310+R01939      Lysine degradation\n",
      "R02000: rubredoxin:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00071+R02000      Fatty acid degradation\n",
      "R02061: trans,trans-Farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R02061      Terpenoid backbone biosynthesis\n",
      "R02073: diphosphate:beta-D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02073      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R02073      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R02073      Fructose and mannose metabolism\n",
      "R02110: 1,4-alpha-D-Glucan:1,4-alpha-D-glucan 6-alpha-D-(1,4-alpha-D-glucano)-transferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R02110      Starch and sucrose metabolism\n",
      "R02124: retinol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00830+R02124      Retinol metabolism\n",
      "R02135: thiamin monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00730+R02135      Thiamine metabolism\n",
      "R02147: guanosine:phosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02147      Purine metabolism\n",
      "R02148: guanosine 3'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02148      Purine metabolism\n",
      "R02222: steroyl-CoA,hydrogen-donor:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn01040+R02222      Biosynthesis of unsaturated fatty acids\n",
      "R02250: triacylglycerol acylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02250      Glycerolipid metabolism\n",
      "R02296: cytidine:orthophosphate alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02296      Pyrimidine metabolism\n",
      "R02297: xanthosine:orthophosphate ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R02297      Purine metabolism\n",
      "R02320: NTP:pyruvate O2-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00620+R02320      Pyruvate metabolism\n",
      "R02370: cytidine 3'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R02370      Pyrimidine metabolism\n",
      "R02459: D-Ornithine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R02459      D-Amino acid metabolism\n",
      "R02540: 2-phenylacetamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00360+R02540      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00643+R02540      Styrene degradation\n",
      "R02569: acetyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R02569      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R02569      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R02569      Pyruvate metabolism\n",
      "R02570: succinyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-succinyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00020+R02570      Citrate cycle (TCA cycle)\n",
      "R02571: Glutaryl-CoA:dihydrolipoamide S-succinyltransferase\n",
      "     https://www.kegg.jp/entry/R02571\n",
      "R02687: 1,2-diacyl-sn-glycerol acylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02687      Glycerolipid metabolism\n",
      "R02728: beta-D-Glucose 1-phosphate 1,6-phosphomutase\n",
      "     https://www.kegg.jp/entry/R02728\n",
      "R02851: D-Lysine:2-oxocarboxylate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00310+R02851      Lysine degradation\n",
      "     https://www.kegg.jp/pathway/rn00470+R02851      D-Amino acid metabolism\n",
      "R02918: L-Tyrosine:tRNA(Tyr) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R02918      Aminoacyl-tRNA biosynthesis\n",
      "R02924: D-Arginine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00470+R02924      D-Amino acid metabolism\n",
      "R02952: phenylacrylic acid decarboxylase\n",
      "     https://www.kegg.jp/pathway/rn00940+R02952      Phenylpropanoid biosynthesis\n",
      "R02984: dTDPglucose 4-epimerase\n",
      "     https://www.kegg.jp/entry/R02984\n",
      "R03004: Deamino-NAD+ nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00760+R03004      Nicotinate and nicotinamide metabolism\n",
      "R03028: glutaconyl-1-CoA carboxy-lyase;\n",
      "     https://www.kegg.jp/pathway/rn00362+R03028      Benzoate degradation\n",
      "     https://www.kegg.jp/pathway/rn00650+R03028      Butanoate metabolism\n",
      "R03036: Dephospho-CoA nucleotidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00770+R03036      Pantothenate and CoA biosynthesis\n",
      "R03038: L-Alanine:tRNA(Ala) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03038      Aminoacyl-tRNA biosynthesis\n",
      "R03050: 2-Acetolactate pyruvate-lyase (carboxylating)\n",
      "     https://www.kegg.jp/entry/R03050\n",
      "R03096: Indole-3-acetamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00380+R03096      Tryptophan metabolism\n",
      "R03180: 4-Guanidinobutanamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03180      Arginine and proline metabolism\n",
      "R03270: \n",
      "     https://www.kegg.jp/pathway/rn00010+R03270      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R03270      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R03270      Pyruvate metabolism\n",
      "R03316: \n",
      "     https://www.kegg.jp/pathway/rn00020+R03316      Citrate cycle (TCA cycle)\n",
      "R03342: 3-(4-Hydroxyphenyl)pyruvate keto-enol-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00350+R03342      Tyrosine metabolism\n",
      "R03367: \n",
      "     https://www.kegg.jp/pathway/rn00940+R03367      Phenylpropanoid biosynthesis\n",
      "R03425: glycine:lipoylprotein oxidoreductase (decarboxylating and acceptor-aminomethylating)\n",
      "     https://www.kegg.jp/pathway/rn00260+R03425      Glycine, serine and threonine metabolism\n",
      "R03470: 4-Carboxymuconolactone carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00362+R03470      Benzoate degradation\n",
      "R03522: RX:glutathione R-transferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R03522      Glutathione metabolism\n",
      "R03527: D-Glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R03527      Starch and sucrose metabolism\n",
      "R03601: O3-Acetyl-L-serine acetate-lyase (adding hydrogen sulfide)\n",
      "     https://www.kegg.jp/entry/R03601\n",
      "R03643: NADH:(quinone-acceptor) oxidoreductase\n",
      "     https://www.kegg.jp/entry/R03643\n",
      "R03646: L-Arginine:tRNA(Arg) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03646      Aminoacyl-tRNA biosynthesis\n",
      "R03650: L-Cysteine:tRNA(Cys) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03650      Aminoacyl-tRNA biosynthesis\n",
      "R03651: L-Glutamate:tRNA(Gln) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03651      Aminoacyl-tRNA biosynthesis\n",
      "R03654: glycine:tRNA(Gly) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03654      Aminoacyl-tRNA biosynthesis\n",
      "R03655: L-Histidine:tRNA(His) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03655      Aminoacyl-tRNA biosynthesis\n",
      "R03656: L-Isoleucine:tRNA(Ile) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03656      Aminoacyl-tRNA biosynthesis\n",
      "R03657: L-Leucine:tRNA(Leu) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03657      Aminoacyl-tRNA biosynthesis\n",
      "R03658: L-lysine:tRNALys ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03658      Aminoacyl-tRNA biosynthesis\n",
      "R03659: L-Methionine:tRNAMet ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03659      Aminoacyl-tRNA biosynthesis\n",
      "R03660: L-Phenylalanine:tRNA(Ala) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03660      Aminoacyl-tRNA biosynthesis\n",
      "R03661: L-proline:tRNA(Pro) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03661      Aminoacyl-tRNA biosynthesis\n",
      "R03662: L-Serine:tRNA(Ser) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03662      Aminoacyl-tRNA biosynthesis\n",
      "R03663: L-Threonine:tRNA(Thr) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03663      Aminoacyl-tRNA biosynthesis\n",
      "R03664: L-Tryptophan -tRNA(Trp) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03664      Aminoacyl-tRNA biosynthesis\n",
      "R03665: L-Valine:tRNAVal ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03665      Aminoacyl-tRNA biosynthesis\n",
      "R03813: L-leucyl-tRNA(Leu):protein leucyltransferase\n",
      "     https://www.kegg.jp/entry/R03813\n",
      "R03815: dihydrolipoylprotein:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R03815      Glycine, serine and threonine metabolism\n",
      "R03867: (5-Glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/entry/R03867\n",
      "R03905: Glu-tRNA(Gln):L-glutamine amido-ligase(ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03905      Aminoacyl-tRNA biosynthesis\n",
      "R03909: Monocarboxylic acid amide amidohydrolase\n",
      "     https://www.kegg.jp/entry/R03909\n",
      "R03920: ATP:D-fructose 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R03920      Amino sugar and nucleotide sugar metabolism\n",
      "R03940: 10-Formyltetrahydrofolate:L-methionyl-tRNA N-formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00670+R03940      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00970+R03940      Aminoacyl-tRNA biosynthesis\n",
      "R04072: dolichyl-beta-D-mannosyl-phosphate:protein-serine O-D-mannosyltransferase\n",
      "     https://www.kegg.jp/entry/R04072\n",
      "R04109: L-glutamate-semialdehyde: NADP+ oxidoreductase(L-glutamyl-tRNA(Glu)-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R04109      Porphyrin metabolism\n",
      "R04112: N-Acetylmuramoyl-Ala amidohydrolase\n",
      "     https://www.kegg.jp/entry/R04112\n",
      "R04125: [protein]-S8-aminomethyldihydrolipoyllysine:tetrahydrofolate aminomethyltransferase (ammonia-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R04125      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R04125      One carbon pool by folate\n",
      "R04212: Asp-tRNA(Asn):L-glutamine amido-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R04212      Aminoacyl-tRNA biosynthesis\n",
      "R04385: biotin-carboxyl-carrier-protein:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/entry/R04385\n",
      "R04386: carboxybiotin-carboxyl-carrier-protein:acetyl-CoA carboxytransferase\n",
      "     https://www.kegg.jp/entry/R04386\n",
      "R04439: (R)-2,3-dihydroxy-3-methylbutanoate:NADP+ oxidoreductase (isomerizing)\n",
      "     https://www.kegg.jp/pathway/rn00770+R04439      Pantothenate and CoA biosynthesis\n",
      "R04672: (S)-2-Acetolactate pyruvate-lyase (carboxylating)\n",
      "     https://www.kegg.jp/entry/R04672\n",
      "R04673: (S)-2-Aceto-2-hydroxybutanoate pyruvate-lyase (carboxylating)\n",
      "     https://www.kegg.jp/entry/R04673\n",
      "R04710: Protein-glycine,dihydroflavodoxin:S-adenosyl-L-methionine oxidoreductase (S-adenosyl-L-methionine cleaving)\n",
      "     https://www.kegg.jp/entry/R04710\n",
      "R04734: 4a-hydroxytetrahydrobiopterin hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00790+R04734      Folate biosynthesis\n",
      "R04771: ATP:L-selenomethione S-adenosyltransferase\n",
      "     https://www.kegg.jp/entry/R04771\n",
      "R04773: Selenomethionine:tRNAMet ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00450+R04773      Selenocompound metabolism\n",
      "R04858: S-adenosyl-L-methionine:DNA (cytosine-5-)-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04858      Cysteine and methionine metabolism\n",
      "R04859: O3-acetyl-L-serine:thiosulfate 2-amino-2-carboxyethyltransferase (reducing, L-cysteine-forming);\n",
      "     https://www.kegg.jp/entry/R04859\n",
      "R04928: ATP:adenylylsulfate 3'-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R04928\n",
      "R04935: (5-L-glutamyl)-peptide:Se-Methylselenocysteine 5-glutamyltransferase\n",
      "     https://www.kegg.jp/entry/R04935\n",
      "R04949: Cyanoglycoside glucohydrolase\n",
      "     https://www.kegg.jp/entry/R04949\n",
      "R04951: \n",
      "     https://www.kegg.jp/pathway/rn00480+R04951      Glutathione metabolism\n",
      "R04998: beta-D-Glucosyl-2-coumarinate glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00999+R04998      Biosynthesis of various plant secondary metabolites\n",
      "R05066: 3-hydroxy-2-methylpropanoate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00280+R05066      Valine, leucine and isoleucine degradation\n",
      "R05551: \n",
      "     https://www.kegg.jp/pathway/rn00643+R05551      Styrene degradation\n",
      "R05577: L-Aspartate:tRNA(Asp) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R05577      Aminoacyl-tRNA biosynthesis\n",
      "R05578: L-glutamate:tRNA(Glu) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05578      Porphyrin metabolism\n",
      "     https://www.kegg.jp/pathway/rn00970+R05578      Aminoacyl-tRNA biosynthesis\n",
      "R05590: Acylamide aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R05590      Aminobenzoate degradation\n",
      "R05666: toluene,NADH:oxygen oxidoreductase (4-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00623+R05666      Toluene degradation\n",
      "R05837: 4-(phosphonooxy)threonine:NAD oxidoreductase\n",
      "     https://www.kegg.jp/entry/R05837\n",
      "R06087: \n",
      "     https://www.kegg.jp/entry/R06087\n",
      "R06088: \n",
      "     https://www.kegg.jp/entry/R06088\n",
      "R06173: \n",
      "     https://www.kegg.jp/entry/R06173\n",
      "R06174: \n",
      "     https://www.kegg.jp/entry/R06174\n",
      "R06285: \n",
      "     https://www.kegg.jp/pathway/rn00540+R06285      Lipopolysaccharide biosynthesis\n",
      "R06363: Penicillin hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00311+R06363      Penicillin and cephalosporin biosynthesis\n",
      "R06590: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glycolaldehydetransferase;\n",
      "     https://www.kegg.jp/pathway/rn01051+R06590      Biosynthesis of ansamycins\n",
      "R06945: \n",
      "     https://www.kegg.jp/pathway/rn00930+R06945      Caprolactam degradation\n",
      "R06974: formate:N1-(5-phospho-beta-D-ribosyl)glycinamide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00230+R06974      Purine metabolism\n",
      "R07002: (1R)-hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07002      Metabolism of xenobiotics by cytochrome P450\n",
      "R07003: (1R)-glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07003      Metabolism of xenobiotics by cytochrome P450\n",
      "R07004: (1S)-hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07004      Metabolism of xenobiotics by cytochrome P450\n",
      "R07023: 1-nitro-7-hydroxy-8-glutathionyl-7,8-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07023      Metabolism of xenobiotics by cytochrome P450\n",
      "R07024: 1-nitro-7-glutathionyl-8-hydroxy-7,8-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07024      Metabolism of xenobiotics by cytochrome P450\n",
      "R07025: 1-nitro-5-hydroxy-6-glutathionyl-5,6-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07025      Metabolism of xenobiotics by cytochrome P450\n",
      "R07026: 1-nitro-5-glutathionyl-6-hydroxy-5,6-dihydronaphthalene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07026      Metabolism of xenobiotics by cytochrome P450\n",
      "R07069: 3,4-dihydro-3-hydroxy-4-S-glutathionyl bromobenzene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07069      Metabolism of xenobiotics by cytochrome P450\n",
      "R07070: 2,3-dihydro-2-S-glutathionyl-3-hydroxy bromobenzene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07070      Metabolism of xenobiotics by cytochrome P450\n",
      "R07083: 4,5-dihydro-4-hydroxy-5-S-glutathionyl-benzo[a]pyrene glutathione-lyase (epoxide-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07083      Metabolism of xenobiotics by cytochrome P450\n",
      "R07084: 7,8-dihydro-7-hydroxy-8-S-glutathionyl-benzo[a]pyrene hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00980+R07084      Metabolism of xenobiotics by cytochrome P450\n",
      "R07091: S-(2,2-dichloro-1-hydroxy)ethyl-glutathione 2,2-dichloroacetaldehyde-lyase (glutathione-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07091      Metabolism of xenobiotics by cytochrome P450\n",
      "R07092: 1,1-dichloroethylene-epoxide:glutathione S-(chloroepoxyethyl)transferase [2-(S-glutathionyl)acetyl-chloride-forming]\n",
      "     https://www.kegg.jp/pathway/rn00980+R07092      Metabolism of xenobiotics by cytochrome P450\n",
      "R07093: chloroacetyl-chloride:glutathione S-chloroacetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00980+R07093      Metabolism of xenobiotics by cytochrome P450\n",
      "R07094: 2-(S-glutathionyl)acetyl-chloride:glutathione 2-(S-glutathionyl)acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00980+R07094      Metabolism of xenobiotics by cytochrome P450\n",
      "R07100: trichloroethene:glutathione S-(1,2-dichlorovinyl)transferase\n",
      "     https://www.kegg.jp/pathway/rn00980+R07100      Metabolism of xenobiotics by cytochrome P450\n",
      "R07105: trichloroethanol:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00980+R07105      Metabolism of xenobiotics by cytochrome P450\n",
      "R07113: 1,2-dibromoethane:glutathione ethylenetransferase (episulfonium-forming)\n",
      "     https://www.kegg.jp/pathway/rn00980+R07113      Metabolism of xenobiotics by cytochrome P450\n",
      "R07116: 2-bromoacetaldehyde:glutathione S-(formylmethyl)transferase\n",
      "     https://www.kegg.jp/pathway/rn00980+R07116      Metabolism of xenobiotics by cytochrome P450\n",
      "R07274: O-phospho-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R07274      Cysteine and methionine metabolism\n",
      "R07459: ATP:[sulfur-carrier protein] adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R07459      Thiamine metabolism\n",
      "R07460: L-cysteine:[enzyme]-cysteine sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R07460      Thiamine metabolism\n",
      "R07618: enzyme N6-(dihydrolipoyl)lysine:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00010+R07618      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00020+R07618      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00280+R07618      Valine, leucine and isoleucine degradation\n",
      "     https://www.kegg.jp/pathway/rn00620+R07618      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00640+R07618      Propanoate metabolism\n",
      "R07620: \n",
      "     https://www.kegg.jp/pathway/rn00514+R07620      Other types of O-glycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00515+R07620      Mannose type O-glycan biosynthesis\n",
      "R07650: L-2,4-diaminobutanoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00260+R07650      Glycine, serine and threonine metabolism\n",
      "R07652: farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase (4,4'-diapophytoene forming)\n",
      "     https://www.kegg.jp/entry/R07652\n",
      "R07766: octanoyl-[acp]:protein N6-octanoyltransferase\n",
      "     https://www.kegg.jp/entry/R07766\n",
      "R07767: [protein]-N6-(octanoyl)-L-lysine:an [Fe-S] cluster scaffold protein carrying a [4Fe-4S]2+ cluster sulfurtransferase\n",
      "     https://www.kegg.jp/entry/R07767\n",
      "R07769: lipoyl-[acp]:protein N6-lipoyltransferase\n",
      "     https://www.kegg.jp/entry/R07769\n",
      "R07774: S-adenosyl-L-methionine:cobalt-precorrin-6B C15-methyltransferase (C12-decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07774      Porphyrin metabolism\n",
      "R07775: S-adenosyl-L-methionine:cobalt-precorrin-7 C5-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07775      Porphyrin metabolism\n",
      "R08014: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08014      Nitrotoluene degradation\n",
      "R08017: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08017      Nitrotoluene degradation\n",
      "R08057: GTP:GTP guanylyltransferase\n",
      "     https://www.kegg.jp/entry/R08057\n",
      "R08218: L-serine:tRNASec ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R08218      Aminoacyl-tRNA biosynthesis\n",
      "R08222: 5'-deoxy-5-fluorouridine:phosphate deoxy-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08222      Drug metabolism - other enzymes\n",
      "R08230: 5-fluorodeoxyuridine:phosphate deoxy-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08230      Drug metabolism - other enzymes\n",
      "R08236: S-adenosyl-L-methionine:6-mercaptopurin S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08236      Drug metabolism - other enzymes\n",
      "R08237: 6-thioinosine 5'-monophosphate:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08237      Drug metabolism - other enzymes\n",
      "R08238: 6-methylthiopurine 5'-monophosphate ribonucleotide:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08238      Drug metabolism - other enzymes\n",
      "R08239: S-adenosyl-L-methionine:6-thioinosine 5'-monophosphate S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08239      Drug metabolism - other enzymes\n",
      "R08240: 6-thioinosine 5'-monophosphate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08240      Drug metabolism - other enzymes\n",
      "R08243: inosine triphosphate pyrophosphatase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08243      Drug metabolism - other enzymes\n",
      "R08244: 6-thioxanthine 5'-monophosphate:L-glutamine amido-ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00983+R08244      Drug metabolism - other enzymes\n",
      "R08245: 6-thioguanosine monophosphate:diphosphate phospho-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08245      Drug metabolism - other enzymes\n",
      "R08246: S-adenosyl-L-methionine:6-thioguanosine monophosphate S-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08246      Drug metabolism - other enzymes\n",
      "R08280: 4-glutathionyl cyclophosphamide hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00982+R08280      Drug metabolism - cytochrome P450\n",
      "R08281: alcophosphamide:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00982+R08281      Drug metabolism - cytochrome P450\n",
      "R08306: 2-phenyl-1,3-propanediol monocarbamate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00982+R08306      Drug metabolism - cytochrome P450\n",
      "R08310: 4-hydroxy-5-phenyltetrahydro-1,3-oxazin-2-one:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00982+R08310      Drug metabolism - cytochrome P450\n",
      "R08620: 2-(2'-methylthio)ethylmalate hydroxymutase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08620      Glucosinolate biosynthesis\n",
      "R08622: homomethionine:2-oxo-acid aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08622      Glucosinolate biosynthesis\n",
      "R08624: 2-(3'-methylthio)propylmalate hydroxymutase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08624      Glucosinolate biosynthesis\n",
      "R08626: dihomomethionine:2-oxo-acid aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08626      Glucosinolate biosynthesis\n",
      "R08628: 2-(3'-methylthio)butylmalate hydroxymutase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08628      Glucosinolate biosynthesis\n",
      "R08634: 2-(3'-methylthio)pentylmalate hydroxymutase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08634      Glucosinolate biosynthesis\n",
      "R08641: 2-(3'-methylthio)hexylmalate hydroxymutase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08641      Glucosinolate biosynthesis\n",
      "R08645: 2-(3'-methylthio)heptylmalate hydroxymutase\n",
      "     https://www.kegg.jp/pathway/rn00966+R08645      Glucosinolate biosynthesis\n",
      "R08856: UDP-N-acetyl-D-glucosamine:undecaprenyl-phosphate N-acetyl-D-glucosamine phosphotransferase\n",
      "     https://www.kegg.jp/entry/R08856\n",
      "R09304: \n",
      "     https://www.kegg.jp/pathway/rn00514+R09304      Other types of O-glycan biosynthesis\n",
      "R09381: acyl-ACP:1-acyl-sn-glycerol-3-phosphate 2-O-acyltransferase\n",
      "     https://www.kegg.jp/entry/R09381\n",
      "R09382: CTP,CTP,ATP:tRNA cytidylyl,cytidylyl,adenylyltransferase\n",
      "     https://www.kegg.jp/entry/R09382\n",
      "R09383: CTP:tRNA cytidylyltransferase\n",
      "     https://www.kegg.jp/entry/R09383\n",
      "R09384: CTP:tRNA cytidylyltransferase\n",
      "     https://www.kegg.jp/entry/R09384\n",
      "R09386: ATP:tRNA adenylyltransferase\n",
      "     https://www.kegg.jp/entry/R09386\n",
      "R09395: thiocarboxylated molybdopterin synthase:cyclic pyranopterin phosphate sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R09395      Folate biosynthesis\n",
      "R09409: glutathione-S-transferase\n",
      "     https://www.kegg.jp/pathway/rn00980+R09409      Metabolism of xenobiotics by cytochrome P450\n",
      "R09499: hydrogen-sulfide:flavocytochrome c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R09499      Sulfur metabolism\n",
      "R09509: carboxybiotinyl-[protein] carboxy-lyase\n",
      "     https://www.kegg.jp/entry/R09509\n",
      "R09513: methanesulfonate,NADH:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R09513      Sulfur metabolism\n",
      "R09594: tetrahydromethanopterin:alpha-L-glutamate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09594      Folate biosynthesis\n",
      "R09597: L-lysine:[tRNA(Ile2)]-cytidine34 ligase (AMP-forming)\n",
      "     https://www.kegg.jp/entry/R09597\n",
      "R09675: ATP:3'-dephospho-CoA 5-triphospho-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/entry/R09675\n",
      "R09676: UDP-N-acetyl-D-glucosamine:protein-O-beta-N-acetyl-D-glucosaminyl transferase\n",
      "     https://www.kegg.jp/entry/R09676\n",
      "R09726: ATP:molybdopterin adenylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R09726      Folate biosynthesis\n",
      "R09735: adenylyl-molybdopterin:molybdate molybdate transferase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R09735      Folate biosynthesis\n",
      "R09736: S-adenosyl-L-methione:demethylmenaquinol methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R09736      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R09763: CMP-3-deoxy-D-manno-oct-2-ulosonate:(KDO)2-lipid IVA 3-deoxy-D-manno-oct-2-ulosonate transferase [(2->8) glycosidic bond-forming]\n",
      "     https://www.kegg.jp/entry/R09763\n",
      "R09764: \n",
      "     https://www.kegg.jp/entry/R09764\n",
      "R09766: CMP-3-deoxy-D-manno-oct-2-ulosonate:(KDO)3-lipid IVA 3-deoxy-D-manno-oct-2-ulosonate transferase [(2->4) glycosidic bond-forming]\n",
      "     https://www.kegg.jp/entry/R09766\n",
      "R09768: D-glycero-D-manno-heptose 7-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/entry/R09768\n",
      "R09793: presqualene-diphosphate diphosphate-lyase (15-cis-4,4'-diapophytoene-forming)\n",
      "     https://www.kegg.jp/pathway/rn00906+R09793      Carotenoid biosynthesis\n",
      "R09845: CAAX protease\n",
      "     https://www.kegg.jp/pathway/rn00900+R09845      Terpenoid backbone biosynthesis\n",
      "R10052: (2R,3S)-3-isopropylmalate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/entry/R10052\n",
      "R10137: UDP-N-acetyl-alpha-D-glucosamine:trans,octacis-decaprenyl-phosphate N-acetylglucosaminephosphotransferase\n",
      "     https://www.kegg.jp/entry/R10137\n",
      "R10170: (2R,3S)-3-isopropylmalate hydro-lyase (2-isopropylmaleate-forming)\n",
      "     https://www.kegg.jp/entry/R10170\n",
      "R10209: tRNA-guanosine34:7-aminomethyl-7-carbaguanine tRNA-D-ribosyltransferase\n",
      "     https://www.kegg.jp/entry/R10209\n",
      "R10220: S-adenosyl-L-methionine:7-aminomethyl-7-deazaguanosine ribosyltransferase (ribosyl isomerizing; L-methionine, adenine releasing)\n",
      "     https://www.kegg.jp/entry/R10220\n",
      "R10223: tRNA(adenine34) aminohydrolase\n",
      "     https://www.kegg.jp/entry/R10223\n",
      "R10247: 1-deoxy-D-xylulose 5-phosphate:thiol sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R10247      Thiamine metabolism\n",
      "R10337: 5-phospho-alpha-D-ribose 1-diphosphate:4-aminobenzoate 5-phospho-beta-D-ribofuranosyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/entry/R10337\n",
      "R10460: GDP-4-amino-4,6-dideoxy-alpha-D-mannose:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R10460      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R10460      O-Antigen nucleotide sugar biosynthesis\n",
      "R10463: ATP:L-threonyl,bicarbonate adenylyltransferase\n",
      "     https://www.kegg.jp/entry/R10463\n",
      "R10597: chorismate pyruvate-lyase (3-hydroxybenzoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R10597      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R10633: \n",
      "     https://www.kegg.jp/pathway/rn00760+R10633      Nicotinate and nicotinamide metabolism\n",
      "R10634: \n",
      "     https://www.kegg.jp/pathway/rn00760+R10634      Nicotinate and nicotinamide metabolism\n",
      "R10645: tRNA (N6-dimethylallyladenosine):sulfur-donor,S-adenosyl-L-methionine C2-methylthiotransferase\n",
      "     https://www.kegg.jp/entry/R10645\n",
      "R10646: \n",
      "     https://www.kegg.jp/entry/R10646\n",
      "R10647: \n",
      "     https://www.kegg.jp/entry/R10647\n",
      "R10649: tRNA (N6-L-threonylcarbamoyladenosine37):sulfur-donor,S-adenosyl-L-methionine C2-methylthiotransferase\n",
      "     https://www.kegg.jp/entry/R10649\n",
      "R10652: [ribosomal protein S12]-L-aspartate:sulfur-donor,S-adenosyl-L-methionine C3-methylthiotransferase\n",
      "     https://www.kegg.jp/entry/R10652\n",
      "R10713: primary alcohol:cytochrome cL oxidoreductase\n",
      "     https://www.kegg.jp/entry/R10713\n",
      "R10776: 2'-deoxyribonucleoside 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R10776\n",
      "R10806: S-adenosyl-L-methionine:[protein]-L-glutamine (N5-glutamine)-methyltransferase\n",
      "     https://www.kegg.jp/entry/R10806\n",
      "R10816: 5'-triphospho-[mRNA] diphosphatase\n",
      "     https://www.kegg.jp/entry/R10816\n",
      "R10831: \n",
      "     https://www.kegg.jp/pathway/rn00531+R10831      Glycosaminoglycan degradation\n",
      "R10902: tyramine:L-glutamate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00680+R10902      Methane metabolism\n",
      "R10935: \n",
      "     https://www.kegg.jp/pathway/rn00680+R10935      Methane metabolism\n",
      "R10938: L-alanyl-D-glutamate epimerase\n",
      "     https://www.kegg.jp/entry/R10938\n",
      "R10948: ATP:HCO3- phosphotransferase\n",
      "     https://www.kegg.jp/entry/R10948\n",
      "R10949: \n",
      "     https://www.kegg.jp/entry/R10949\n",
      "R10970: ATP:6-deoxy-6-sulfo-D-fructose 1-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R10970\n",
      "R11037: 7,8-dihydroneopterin:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11037      Folate biosynthesis\n",
      "R11039: [5-(aminomethyl)furan-3-yl]methyl phosphate kinase\n",
      "     https://www.kegg.jp/pathway/rn00680+R11039      Methane metabolism\n",
      "R11040: (4-{4-[2-(gamma-L-glutamylamino)ethyl]phenoxymethyl}furan-2-yl)methanamine synthase\n",
      "     https://www.kegg.jp/pathway/rn00680+R11040      Methane metabolism\n",
      "R11073: 7,8-dihydroneopterin 2'-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11073      Folate biosynthesis\n",
      "R11102: 5-phospho-alpha-D-ribose-1-diphosphate:4-hydroxybenzoate 5-phospho-beta-D-ribofuranosyltransferase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00790+R11102      Folate biosynthesis\n",
      "R11173: adenosylcobalamin-5'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R11173\n",
      "R11174: adenosylcobinamide-GDP:alpha-ribazole adenosylcobinamide-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R11174\n",
      "R11188: nucleoside-3',5'-bisphosphate 3'-phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R11188\n",
      "R11225: dimethylallyl-phosphate:FMNH2 prenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00740+R11225      Riboflavin metabolism\n",
      "R11310: beta-D-glucose 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00500+R11310      Starch and sucrose metabolism\n",
      "R11319: ADP:thiamin-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00730+R11319      Thiamine metabolism\n",
      "R11323: dTTP diphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R11323      Pyrimidine metabolism\n",
      "R11329: Fe-coproporphyrin-III ferro-lyase (coproporphyrin-III-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11329      Porphyrin metabolism\n",
      "R11372: (8S)-3',8-cyclo-7,8-dihydroguanosine 5'-triphosphate lyase (cyclic pyranopterin phosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R11372      Folate biosynthesis\n",
      "R11396: CTP:8-amino-3,8-dideoxy-D-manno-octulosonate cytidylyltransferase\n",
      "     https://www.kegg.jp/entry/R11396\n",
      "R11399: dolichyl-beta-D-mannosyl-phosphate:protein-threonine O-D-mannosyltransferase\n",
      "     https://www.kegg.jp/entry/R11399\n",
      "R11435: \n",
      "     https://www.kegg.jp/pathway/rn01059+R11435      Biosynthesis of enediyne antibiotics\n",
      "R11443: L-leucyl-tRNA(Leu):[protein] N-terminal L-lysine leucyltransferase\n",
      "     https://www.kegg.jp/entry/R11443\n",
      "R11444: L-leucyl-tRNA(Leu):[protein] N-terminal L-arginine leucyltransferase\n",
      "     https://www.kegg.jp/entry/R11444\n",
      "R11528: L-cysteine:sulfur-acceptor sulfurtransferase\n",
      "     https://www.kegg.jp/entry/R11528\n",
      "R11529: [enzyme]-S-sulfanylcysteine:sulfur-acceptor sulfurtransferase\n",
      "     https://www.kegg.jp/entry/R11529\n",
      "R11530: ADP-alpha-D-glucose:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (configuration-retaining)\n",
      "     https://www.kegg.jp/pathway/rn00500+R11530      Starch and sucrose metabolism\n",
      "R11547: L-leucyl-tRNA(Leu):[protein] N-terminal L-glutamate leucyltransferase\n",
      "     https://www.kegg.jp/entry/R11547\n",
      "R11548: L-leucyl-tRNA(Leu):[protein] N-terminal L-aspartate leucyltransferase\n",
      "     https://www.kegg.jp/entry/R11548\n",
      "R11580: S-adenosyl-L-methionine:cobalt-factor III 17-methyltransferase (ring contracting)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11580      Porphyrin metabolism\n",
      "R11581: GTP:molybdenum cofactor guanylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11581      Folate biosynthesis\n",
      "R11671: \n",
      "     https://www.kegg.jp/pathway/rn00333+R11671      Prodigiosin biosynthesis\n",
      "R11743: formylmethanofuran:ferredoxin oxidoreductase\n",
      "     https://www.kegg.jp/entry/R11743\n",
      "R11765: tetrahydrobiopterin:NADP+ oxidoreductase (7,8-dihydrobiopterin-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R11765      Folate biosynthesis\n",
      "R11786: (ribonucleotide)n-2',3'-cyclic phosphate 3'-nucleotidohydrolase\n",
      "     https://www.kegg.jp/entry/R11786\n",
      "R11816: protoheme,donor:oxygen oxidoreductase (biliverdin-IX-delta-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11816      Porphyrin metabolism\n",
      "R11817: protoheme,donor:oxygen oxidoreductase (biliverdin-IX-beta-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11817      Porphyrin metabolism\n",
      "R11894: ATP:5-fluorouridine-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R11894      Drug metabolism - other enzymes\n",
      "R11895: ATP:5-fluorodeoxyuridine-diphosphate phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R11895      Drug metabolism - other enzymes\n",
      "R11896: 5-fluorodeoxyuridine-triphosphate diphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00983+R11896      Drug metabolism - other enzymes\n",
      "R12024: N-(4-oxoglutaryl)-L-cysteinylglycine amidohydrolase\n",
      "     https://www.kegg.jp/entry/R12024\n",
      "R12049: \n",
      "     https://www.kegg.jp/pathway/rn00998+R12049      Biosynthesis of various antibiotics\n",
      "R12075: UDP-N-acetyl-alpha-D-glucosamine:trans,octacis-decaprenyl-phosphate N-acetylglucosaminephosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00572+R12075      Arabinogalactan biosynthesis - Mycobacterium\n",
      "R12098: S-adenosyl-L-methionine hydrolase (adenosine-forming)\n",
      "     https://www.kegg.jp/entry/R12098\n",
      "R12157: S-adenosyl-L-methionine:1,6-didemethyltoxoflavin N6-methyltransferase (reumycin-forming)\n",
      "     https://www.kegg.jp/entry/R12157\n",
      "R12158: S-adenosyl-L-methionine:reumycin N1-methyltransferase (toxoflavin-forming)\n",
      "     https://www.kegg.jp/entry/R12158\n",
      "R12173: palmitoyl-CoA,ferrocytochrome-b5:oxygen oxidoreductase (9,10-dehydrogenating)\n",
      "     https://www.kegg.jp/pathway/rn01040+R12173      Biosynthesis of unsaturated fatty acids\n",
      "R12184: ATP:Cob(II)yrinate diamide Cobeta-adenosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R12184      Porphyrin metabolism\n",
      "R12188: L-cysteine-S-conjugate thiol-lyase (deaminating; 2-aminoprop-2-enoate-forming)\n",
      "     https://www.kegg.jp/entry/R12188\n",
      "R12240: acyl-[acyl-carrier protein]:phosphate acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R12240      Glycerolipid metabolism\n",
      "R12241: acyl phosphoate:sn-glycerol 3-phosphate acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R12241      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R12241      Glycerophospholipid metabolism\n",
      "R12323: lanosterol,reduced ferredoxin:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R12323      Steroid biosynthesis\n",
      "R12423: [lipoyl-carrier protein E2]-N6-(octanoyl)-L-lysine:[Fe-S] cluster scaffold protein carrying a [4Fe-4S]2+ cluster sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12423      Lipoic acid metabolism\n",
      "R12424: [glycine cleavage system H]-N6-octanoyl-L-lysine:[Fe-S] cluster scaffold protein carrying a [4Fe-4S]2+ cluster sulfurtransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12424      Lipoic acid metabolism\n",
      "R12427: octanoyl-[acp]:lipoyl-carrier-protein-E2 N-octanoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12427      Lipoic acid metabolism\n",
      "R12428: octanoyl-[acp]:Glycine-cleavage-system-H N-octanoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00785+R12428      Lipoic acid metabolism\n",
      "R12500: queuosine34 in tRNA:acceptor oxidoreductase\n",
      "     https://www.kegg.jp/entry/R12500\n",
      "R12578: glutathione:hydroperoxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00480+R12578      Glutathione metabolism\n",
      "R12605: \n",
      "     https://www.kegg.jp/pathway/rn00860+R12605      Porphyrin metabolism\n",
      "R12644: 7,8-dihydrobiopterin:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R12644      Folate biosynthesis\n",
      "R12808: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12808      Teichoic acid biosynthesis\n",
      "R12814: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12814      Teichoic acid biosynthesis\n",
      "R12851: adenylosuccinate lyase\n",
      "     https://www.kegg.jp/pathway/rn00230+R12851      Purine metabolism\n",
      "R12852: guanylate kinase\n",
      "     https://www.kegg.jp/pathway/rn00230+R12852      Purine metabolism\n",
      "R12853: nucleoside-diphosphate kinase\n",
      "     https://www.kegg.jp/pathway/rn00230+R12853      Purine metabolism\n",
      "R12859: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12859      Teichoic acid biosynthesis\n",
      "R12885: \n",
      "     https://www.kegg.jp/pathway/rn00999+R12885      Biosynthesis of various plant secondary metabolites\n",
      "R12895: \n",
      "     https://www.kegg.jp/pathway/rn00900+R12895      Terpenoid backbone biosynthesis\n",
      "R12896: prenyl diphosphate diphosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R12896      Riboflavin metabolism\n",
      "R12933: \n",
      "     https://www.kegg.jp/entry/R12933\n",
      "R12934: \n",
      "     https://www.kegg.jp/pathway/rn00780+R12934      Biotin metabolism\n",
      "R12955: uridine 5'-monophosphate phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R12955      Pyrimidine metabolism\n",
      "R12956: thymidine 5'-monophosphate phosphodeoxyribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R12956      Pyrimidine metabolism\n",
      "R12958: dIMP phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R12958\n",
      "R13051: daidzein 7-O-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13051      Degradation of flavonoids\n",
      "R13052: genistein 7-O-beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13052      Degradation of flavonoids\n",
      "R13065: quercetin 3-O-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13065      Degradation of flavonoids\n",
      "R13081: naringenin 7-O-beta-D-glucoside glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00946+R13081      Degradation of flavonoids\n",
      "R13097: \n",
      "     https://www.kegg.jp/pathway/rn00470+R13097      D-Amino acid metabolism\n",
      "R00647: \n",
      "     https://www.kegg.jp/pathway/rn00053+R00647      Ascorbate and aldarate metabolism\n",
      "R02317: (S)-2-amino-6-oxohexanoate hydro-lyase (spontaneous)\n",
      "     https://www.kegg.jp/pathway/rn00310+R02317      Lysine degradation\n",
      "     https://www.kegg.jp/pathway/rn00960+R02317      Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "R02962: \n",
      "     https://www.kegg.jp/pathway/rn00350+R02962      Tyrosine metabolism\n",
      "R03166: \n",
      "     https://www.kegg.jp/pathway/rn00860+R03166      Porphyrin metabolism\n",
      "R03186: \n",
      "     https://www.kegg.jp/pathway/rn00053+R03186      Ascorbate and aldarate metabolism\n",
      "R03445: \n",
      "     https://www.kegg.jp/pathway/rn00380+R03445      Tryptophan metabolism\n",
      "R03672: \n",
      "     https://www.kegg.jp/pathway/rn00350+R03672      Tyrosine metabolism\n",
      "R03674: \n",
      "     https://www.kegg.jp/pathway/rn00350+R03674      Tyrosine metabolism\n",
      "R03758: \n",
      "     https://www.kegg.jp/pathway/rn00260+R03758      Glycine, serine and threonine metabolism\n",
      "R03954: \n",
      "     https://www.kegg.jp/pathway/rn00380+R03954      Tryptophan metabolism\n",
      "R04500: \n",
      "     https://www.kegg.jp/pathway/rn00361+R04500      Chlorocyclohexane and chlorobenzene degradation\n",
      "R04578: \n",
      "     https://www.kegg.jp/pathway/rn00902+R04578      Monoterpenoid biosynthesis\n",
      "R04701: \n",
      "     https://www.kegg.jp/pathway/rn00950+R04701      Isoquinoline alkaloid biosynthesis\n",
      "R04769: salutaridinol 7-O-acetate acetate-lyase (thebaine forming)\n",
      "     https://www.kegg.jp/pathway/rn00950+R04769      Isoquinoline alkaloid biosynthesis\n",
      "R05041: \n",
      "     https://www.kegg.jp/pathway/rn00830+R05041      Retinol metabolism\n",
      "R05098: \n",
      "     https://www.kegg.jp/pathway/rn00930+R05098      Caprolactam degradation\n",
      "R05455: \n",
      "     https://www.kegg.jp/pathway/rn00130+R05455      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R05483: \n",
      "     https://www.kegg.jp/pathway/rn00361+R05483      Chlorocyclohexane and chlorobenzene degradation\n",
      "R05484: \n",
      "     https://www.kegg.jp/pathway/rn00361+R05484      Chlorocyclohexane and chlorobenzene degradation\n",
      "R05824: \n",
      "     https://www.kegg.jp/pathway/rn00901+R05824      Indole alkaloid biosynthesis\n",
      "R05826: \n",
      "     https://www.kegg.jp/pathway/rn00901+R05826      Indole alkaloid biosynthesis\n",
      "R05844: \n",
      "     https://www.kegg.jp/pathway/rn00901+R05844      Indole alkaloid biosynthesis\n",
      "R05845: \n",
      "     https://www.kegg.jp/pathway/rn00901+R05845      Indole alkaloid biosynthesis\n",
      "R05846: \n",
      "     https://www.kegg.jp/pathway/rn00901+R05846      Indole alkaloid biosynthesis\n",
      "R05887: \n",
      "     https://www.kegg.jp/pathway/rn00950+R05887      Isoquinoline alkaloid biosynthesis\n",
      "R05889: \n",
      "     https://www.kegg.jp/pathway/rn00950+R05889      Isoquinoline alkaloid biosynthesis\n",
      "R06063: \n",
      "     https://www.kegg.jp/entry/R06063\n",
      "R06064: \n",
      "     https://www.kegg.jp/entry/R06064\n",
      "R06194: \n",
      "     https://www.kegg.jp/pathway/rn00830+R06194      Retinol metabolism\n",
      "R06650: \n",
      "     https://www.kegg.jp/pathway/rn01057+R06650      Biosynthesis of type II polyketide products\n",
      "R06830: \n",
      "     https://www.kegg.jp/entry/R06830\n",
      "R06982: S-(hydroxymethyl)glutathione formaldehyde-lyase (glutathione-forming);\n",
      "     https://www.kegg.jp/pathway/rn00680+R06982      Methane metabolism\n",
      "R07009: \n",
      "     https://www.kegg.jp/pathway/rn00980+R07009      Metabolism of xenobiotics by cytochrome P450\n",
      "R07010: \n",
      "     https://www.kegg.jp/pathway/rn00980+R07010      Metabolism of xenobiotics by cytochrome P450\n",
      "R07011: \n",
      "     https://www.kegg.jp/pathway/rn00980+R07011      Metabolism of xenobiotics by cytochrome P450\n",
      "R07012: \n",
      "     https://www.kegg.jp/pathway/rn00980+R07012      Metabolism of xenobiotics by cytochrome P450\n",
      "R07198: liquiritigenin,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (licodione-forming)\n",
      "     https://www.kegg.jp/entry/R07198\n",
      "R07316: \n",
      "     https://www.kegg.jp/pathway/rn00910+R07316      Nitrogen metabolism\n",
      "R07406: \n",
      "     https://www.kegg.jp/pathway/rn00750+R07406      Vitamin B6 metabolism\n",
      "R07679: \n",
      "     https://www.kegg.jp/pathway/rn00053+R07679      Ascorbate and aldarate metabolism\n",
      "R07947: \n",
      "     https://www.kegg.jp/pathway/rn00760+R07947      Nicotinate and nicotinamide metabolism\n",
      "R07948: \n",
      "     https://www.kegg.jp/pathway/rn00760+R07948      Nicotinate and nicotinamide metabolism\n",
      "R08020: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08020      Nitrotoluene degradation\n",
      "R08279: \n",
      "     https://www.kegg.jp/pathway/rn00982+R08279      Drug metabolism - cytochrome P450\n",
      "R08289: \n",
      "     https://www.kegg.jp/pathway/rn00982+R08289      Drug metabolism - cytochrome P450\n",
      "R08358: \n",
      "     https://www.kegg.jp/pathway/rn00480+R08358      Glutathione metabolism\n",
      "R08361: \n",
      "     https://www.kegg.jp/pathway/rn00480+R08361      Glutathione metabolism\n",
      "R08372: \n",
      "     https://www.kegg.jp/entry/R08372\n",
      "R08637: 2-Aminobut-2-enoate aminohydrolase (spontaneous)\n",
      "     https://www.kegg.jp/entry/R08637\n",
      "R08698: 2-aminoacrylate aminohydrolase\n",
      "     https://www.kegg.jp/entry/R08698\n",
      "R08763: 6-iminocyclohexa-2,4-dienone oxidoreductase (dimerizing)\n",
      "     https://www.kegg.jp/entry/R08763\n",
      "R08818: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08818      Betalain biosynthesis\n",
      "R08820: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08820      Betalain biosynthesis\n",
      "R08821: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08821      Betalain biosynthesis\n",
      "R08825: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08825      Betalain biosynthesis\n",
      "R08827: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08827      Betalain biosynthesis\n",
      "R08829: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08829      Betalain biosynthesis\n",
      "R08831: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08831      Betalain biosynthesis\n",
      "R08833: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08833      Betalain biosynthesis\n",
      "R08834: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08834      Betalain biosynthesis\n",
      "R08835: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08835      Betalain biosynthesis\n",
      "R08837: \n",
      "     https://www.kegg.jp/pathway/rn00965+R08837      Betalain biosynthesis\n",
      "R08976: \n",
      "     https://www.kegg.jp/entry/R08976\n",
      "R09142: \n",
      "     https://www.kegg.jp/pathway/rn00625+R09142      Chloroalkane and chloroalkene degradation\n",
      "R09143: \n",
      "     https://www.kegg.jp/pathway/rn00625+R09143      Chloroalkane and chloroalkene degradation\n",
      "R09144: \n",
      "     https://www.kegg.jp/pathway/rn00625+R09144      Chloroalkane and chloroalkene degradation\n",
      "R09145: \n",
      "     https://www.kegg.jp/pathway/rn00625+R09145      Chloroalkane and chloroalkene degradation\n",
      "R09149: \n",
      "     https://www.kegg.jp/pathway/rn00625+R09149      Chloroalkane and chloroalkene degradation\n",
      "R09274: \n",
      "     https://www.kegg.jp/pathway/rn00627+R09274      Aminobenzoate degradation\n",
      "R09367: \n",
      "     https://www.kegg.jp/pathway/rn00450+R09367      Selenocompound metabolism\n",
      "R09368: \n",
      "     https://www.kegg.jp/pathway/rn00450+R09368      Selenocompound metabolism\n",
      "R09391: \n",
      "     https://www.kegg.jp/pathway/rn00903+R09391      Limonene degradation\n",
      "R09392: \n",
      "     https://www.kegg.jp/pathway/rn00903+R09392      Limonene degradation\n",
      "R09431: \n",
      "     https://www.kegg.jp/pathway/rn00980+R09431      Metabolism of xenobiotics by cytochrome P450\n",
      "R09432: \n",
      "     https://www.kegg.jp/pathway/rn00980+R09432      Metabolism of xenobiotics by cytochrome P450\n",
      "R09433: \n",
      "     https://www.kegg.jp/pathway/rn00980+R09433      Metabolism of xenobiotics by cytochrome P450\n",
      "R09434: \n",
      "     https://www.kegg.jp/pathway/rn00980+R09434      Metabolism of xenobiotics by cytochrome P450\n",
      "R09885: \n",
      "     https://www.kegg.jp/pathway/rn00984+R09885      Steroid degradation\n",
      "R09981: \n",
      "     https://www.kegg.jp/pathway/rn00240+R09981      Pyrimidine metabolism\n",
      "R10013: \n",
      "     https://www.kegg.jp/pathway/rn00300+R10013      Lysine biosynthesis\n",
      "R10018: \n",
      "     https://www.kegg.jp/pathway/rn00950+R10018      Isoquinoline alkaloid biosynthesis\n",
      "R10022: \n",
      "     https://www.kegg.jp/entry/R10022\n",
      "R10030: \n",
      "     https://www.kegg.jp/pathway/rn00460+R10030      Cyanoamino acid metabolism\n",
      "R10034: \n",
      "     https://www.kegg.jp/pathway/rn00460+R10034      Cyanoamino acid metabolism\n",
      "R10041: \n",
      "     https://www.kegg.jp/pathway/rn00460+R10041      Cyanoamino acid metabolism\n",
      "R10078: \n",
      "     https://www.kegg.jp/entry/R10078\n",
      "R10082: \n",
      "     https://www.kegg.jp/entry/R10082\n",
      "R10097: \n",
      "     https://www.kegg.jp/entry/R10097\n",
      "R10245: \n",
      "     https://www.kegg.jp/entry/R10245\n",
      "R10302: \n",
      "     https://www.kegg.jp/entry/R10302\n",
      "R10792: 3alpha-hydroxy-3,5-dihydromonacolin L acid hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00997+R10792      Biosynthesis of various other secondary metabolites\n",
      "R10854: \n",
      "     https://www.kegg.jp/entry/R10854\n",
      "R10856: \n",
      "     https://www.kegg.jp/entry/R10856\n",
      "R10914: \n",
      "     https://www.kegg.jp/entry/R10914\n",
      "R11017: \n",
      "     https://www.kegg.jp/entry/R11017\n",
      "R11034: \n",
      "     https://www.kegg.jp/entry/R11034\n",
      "R11049: \n",
      "     https://www.kegg.jp/pathway/rn00760+R11049      Nicotinate and nicotinamide metabolism\n",
      "R11098: 2-iminobutanoate aminohydrolase\n",
      "     https://www.kegg.jp/entry/R11098\n",
      "R11099: 2-iminopropanoate aminohydrolase\n",
      "     https://www.kegg.jp/entry/R11099\n",
      "R11100: \n",
      "     https://www.kegg.jp/entry/R11100\n",
      "R11101: \n",
      "     https://www.kegg.jp/entry/R11101\n",
      "R11148: \n",
      "     https://www.kegg.jp/entry/R11148\n",
      "R11176: \n",
      "     https://www.kegg.jp/pathway/rn00760+R11176      Nicotinate and nicotinamide metabolism\n",
      "R11228: \n",
      "     https://www.kegg.jp/entry/R11228\n",
      "R11523: \n",
      "     https://www.kegg.jp/entry/R11523\n",
      "R11550: \n",
      "     https://www.kegg.jp/pathway/rn00999+R11550      Biosynthesis of various plant secondary metabolites\n",
      "R11570: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11570      Phenazine biosynthesis\n",
      "R11571: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11571      Phenazine biosynthesis\n",
      "R11572: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11572      Phenazine biosynthesis\n",
      "R11573: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11573      Phenazine biosynthesis\n",
      "R11574: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11574      Phenazine biosynthesis\n",
      "R11575: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11575      Phenazine biosynthesis\n",
      "R11576: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11576      Phenazine biosynthesis\n",
      "R11577: \n",
      "     https://www.kegg.jp/pathway/rn00405+R11577      Phenazine biosynthesis\n",
      "R11691: \n",
      "     https://www.kegg.jp/pathway/rn00950+R11691      Isoquinoline alkaloid biosynthesis\n",
      "R11695: \n",
      "     https://www.kegg.jp/pathway/rn00950+R11695      Isoquinoline alkaloid biosynthesis\n",
      "R11709: \n",
      "     https://www.kegg.jp/entry/R11709\n",
      "R11736: \n",
      "     https://www.kegg.jp/entry/R11736\n",
      "R11739: \n",
      "     https://www.kegg.jp/entry/R11739\n",
      "R11865: \n",
      "     https://www.kegg.jp/entry/R11865\n",
      "R11947: \n",
      "     https://www.kegg.jp/entry/R11947\n",
      "R12091: \n",
      "     https://www.kegg.jp/pathway/rn00901+R12091      Indole alkaloid biosynthesis\n",
      "R12094: \n",
      "     https://www.kegg.jp/pathway/rn00901+R12094      Indole alkaloid biosynthesis\n",
      "R12132: \n",
      "     https://www.kegg.jp/entry/R12132\n",
      "R12140: \n",
      "     https://www.kegg.jp/pathway/rn00592+R12140      alpha-Linolenic acid metabolism\n",
      "R12185: \n",
      "     https://www.kegg.jp/entry/R12185\n",
      "R12268: \n",
      "     https://www.kegg.jp/entry/R12268\n",
      "R12274: \n",
      "     https://www.kegg.jp/entry/R12274\n",
      "R12310: \n",
      "     https://www.kegg.jp/pathway/rn00999+R12310      Biosynthesis of various plant secondary metabolites\n",
      "R12335: \n",
      "     https://www.kegg.jp/pathway/rn00791+R12335      Atrazine degradation\n",
      "R12421: \n",
      "     https://www.kegg.jp/pathway/rn00997+R12421      Biosynthesis of various other secondary metabolites\n",
      "R12539: \n",
      "     https://www.kegg.jp/entry/R12539\n",
      "R12683: \n",
      "     https://www.kegg.jp/entry/R12683\n",
      "R12686: \n",
      "     https://www.kegg.jp/entry/R12686\n",
      "R12829: \n",
      "     https://www.kegg.jp/pathway/rn00901+R12829      Indole alkaloid biosynthesis\n",
      "R12886: \n",
      "     https://www.kegg.jp/pathway/rn00999+R12886      Biosynthesis of various plant secondary metabolites\n",
      "R12910: \n",
      "     https://www.kegg.jp/entry/R12910\n",
      "R12974: \n",
      "     https://www.kegg.jp/entry/R12974\n",
      "R13023: \n",
      "     https://www.kegg.jp/entry/R13023\n",
      "R13091: \n",
      "     https://www.kegg.jp/entry/R13091\n",
      "R13113: \n",
      "     https://www.kegg.jp/entry/R13113\n",
      "R13114: \n",
      "     https://www.kegg.jp/entry/R13114\n",
      "R13140: \n",
      "     https://www.kegg.jp/entry/R13140\n",
      "R13144: \n",
      "     https://www.kegg.jp/pathway/rn00051+R13144      Fructose and mannose metabolism\n",
      "R00382: (deoxyribonucleotide)n:5'-phospho-(deoxyribonucleotide)m ligase (AMP-forming, NMN-forming)\n",
      "     https://www.kegg.jp/entry/R00382\n",
      "R02887: 1,4-beta-D-Glucan glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R02887      Starch and sucrose metabolism\n",
      "R04241: tetrahydropteroyl-gamma-polyglutamate:L-glutamate gamma-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00790+R04241      Folate biosynthesis\n",
      "R05196: 1,4-alpha-D-Glucan:1,4-alpha-D-glucan 4-alpha-D-glycosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R05196      Starch and sucrose metabolism\n",
      "R06177: \n",
      "     https://www.kegg.jp/entry/R06177\n",
      "R09994: alpha-maltose 1-phosphate:(1->4)-alpha-D-glucan 4-alpha-D-maltosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R09994      Starch and sucrose metabolism\n",
      "R10152: sulfide:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R10152      Sulfur metabolism\n",
      "R11261: \n",
      "     https://www.kegg.jp/pathway/rn00500+R11261      Starch and sucrose metabolism\n",
      "R08700: [enzyme]-S-sulfanylcysteine:tRNA-uridine sulfurtransferase (ATP-hydrolysing)\n",
      "     https://www.kegg.jp/entry/R08700\n",
      "R08701: tRNA uridine 5-carboxymethylaminomethyl modification\n",
      "     https://www.kegg.jp/entry/R08701\n",
      "R09083: \n",
      "     https://www.kegg.jp/pathway/rn00740+R09083      Riboflavin metabolism\n",
      "R11905: \n",
      "     https://www.kegg.jp/pathway/rn00983+R11905      Drug metabolism - other enzymes\n"
     ]
    }
   ],
   "source": [
    "notlinmodel=[r for r in  Mca_kegglist if r not in mcoosmodel_rxnlist]\n",
    "print(len(notlinmodel))\n",
    "print('=====================')\n",
    "for r in notlinmodel:\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "233\n",
      "=====================\n",
      "Starch_synthesis\n",
      "     no rxn\n",
      "Starch_degredation\n",
      "     no rxn\n",
      "Rspont2\n",
      "     no rxn\n",
      "RPG161161\n",
      "     no rxn\n",
      "RPG161140\n",
      "     no rxn\n",
      "RPG160161\n",
      "     no rxn\n",
      "RPE161161\n",
      "     no rxn\n",
      "RPE161140\n",
      "     no rxn\n",
      "RPE160161\n",
      "     no rxn\n",
      "RPC161161\n",
      "     no rxn\n",
      "RPC161140\n",
      "     no rxn\n",
      "RPC160161\n",
      "     no rxn\n",
      "RCL161161\n",
      "     no rxn\n",
      "RCL161140\n",
      "     no rxn\n",
      "RCL160161\n",
      "     no rxn\n",
      "RC08362\n",
      "     no rxn\n",
      "R09380: acyl-ACP:sn-glycerol-3-phosphate 1-O-acyltransferase\n",
      "     https://www.kegg.jp/entry/R09380\n",
      "R09372: methaneselenol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00450+R09372      Selenocompound metabolism\n",
      "R08781\n",
      "     no rxn\n",
      "R08775\n",
      "     no rxn\n",
      "R08774\n",
      "     no rxn\n",
      "R08773\n",
      "     no rxn\n",
      "R08769: S-adenosyl-L-methionine:2-polyprenyl-6-hydroxyphenol 6-O-methyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00130+R08769      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R08716: cobalt-precorrin 3 dehydrogenase\n",
      "     https://www.kegg.jp/entry/R08716\n",
      "R08676: ADP-glucose:D-fructose 2-alpha-D-glucosyltransferase\n",
      "     https://www.kegg.jp/entry/R08676\n",
      "R08575: sedoheptulose-7-phosphate:D-glyceraldehyde-3-phosphate glyceronetransferase\n",
      "     https://www.kegg.jp/entry/R08575\n",
      "R07410: L-aspartate:NAD+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00760+R07410      Nicotinate and nicotinamide metabolism\n",
      "R07407: L-aspartate:NADP+ oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00760+R07407      Nicotinate and nicotinamide metabolism\n",
      "R07280: 5-amino-6-(5-phospho-D-ribitylamino)uracil phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R07280      Riboflavin metabolism\n",
      "R07035: glutathione: 15-HPETE oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00590+R07035      Arachidonic acid metabolism\n",
      "R07034: glutathione: 5-HPETE oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00590+R07034      Arachidonic acid metabolism\n",
      "R06859\n",
      "     no rxn\n",
      "R06146\n",
      "     no rxn\n",
      "R05705: FMNH2:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00740+R05705      Riboflavin metabolism\n",
      "     https://www.kegg.jp/pathway/rn01057+R05705      Biosynthesis of type II polyketide products\n",
      "R05639: 4,4-dimethyl-5a-cholesta-8,24-dien-3b-ol:NADP+ D14-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R05639      Steroid biosynthesis\n",
      "R05614\n",
      "     no rxn\n",
      "R05591: Nitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R05591      Aminobenzoate degradation\n",
      "R05000: polyisopentenylpyrolinate:4-hydroxybenzoate nonaprenyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R05000      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R04993\n",
      "     no rxn\n",
      "R04990\n",
      "     no rxn\n",
      "R04989\n",
      "     no rxn\n",
      "R04985: 4-hydroxy-3-polyprenylbenzoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00130+R04985      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R04326: \n",
      "     https://www.kegg.jp/pathway/rn00670+R04326      One carbon pool by folate\n",
      "R04325: 10-formyltetrahydrofolate:5'-phosphoribosylglycinamide formyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00230+R04325      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00670+R04325      One carbon pool by folate\n",
      "R04230: (R)-4'-phosphopantothenate:L-cysteine ligase (ATP-utilizing)\n",
      "     https://www.kegg.jp/pathway/rn00770+R04230      Pantothenate and CoA biosynthesis\n",
      "R03596: hydrogen selenide:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00450+R03596      Selenocompound metabolism\n",
      "R03542: alpha-Aminopropiononitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03542      Cyanoamino acid metabolism\n",
      "R03239: ITP:D-tagatose 6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03239      Galactose metabolism\n",
      "R03238: UTP:D-tagatose 6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03238      Galactose metabolism\n",
      "R03237: CTP:D-tagatose 6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03237      Galactose metabolism\n",
      "R03199: (S)-2,3-Epoxysqualene mutase (cyclizing, lanosterol-forming)\n",
      "     https://www.kegg.jp/pathway/rn00100+R03199      Steroid biosynthesis\n",
      "R03093: 3-Indoleacetonitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00380+R03093      Tryptophan metabolism\n",
      "R03024: 4-nitrophenyl phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R03024      Aminobenzoate degradation\n",
      "R03013: L-histidinol-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00340+R03013      Histidine metabolism\n",
      "R02999: NADPH:cyanocob(III)alamin oxidoreductase (cyanide-eliminating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R02999      Porphyrin metabolism\n",
      "R02872: presqualene-diphosphate diphosphate-lyase (reducing, squalene-forming)\n",
      "     https://www.kegg.jp/pathway/rn00100+R02872      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R02872      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R02869: S-adenosylmethioninamine:spermidine 3-aminopropyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R02869      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R02869      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00480+R02869      Glutathione metabolism\n",
      "R02539: Phenylacetate:CoA ligase\n",
      "     https://www.kegg.jp/pathway/rn00360+R02539      Phenylalanine metabolism\n",
      "R02421\n",
      "     no rxn\n",
      "R02161\n",
      "     no rxn\n",
      "R02016: NADPH:oxidized-thioredoxin oxidoreductase\n",
      "     https://www.kegg.jp/entry/R02016\n",
      "R01974: 3-(indol-3-yl)pyruvate carboxy-lyase [(2-indol-3-yl)acetaldehyde-forming]\n",
      "     https://www.kegg.jp/pathway/rn00380+R01974      Tryptophan metabolism\n",
      "R01887: gamma-Amino-gamma-cyanobutanoate aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R01887      Cyanoamino acid metabolism\n",
      "R01771: ATP:L-homoserine O-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01771      Glycine, serine and threonine metabolism\n",
      "R01706: hexadecanoyl-[acyl-carrier protein] hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01706      Fatty acid biosynthesis\n",
      "R01654: 10-Formyltetrahydrofolate:L-glutamate ligase (ADP-forming)\n",
      "     https://www.kegg.jp/entry/R01654\n",
      "R01224: 5-methyltetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R01224      One carbon pool by folate\n",
      "R01216: 2-dehydropantoate formaldehyde-lyase (3-methyl-2-oxobutanoate-forming)\n",
      "     https://www.kegg.jp/entry/R01216\n",
      "R01011: ATP:glycerone phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R01011      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01011      Methane metabolism\n",
      "R00941: 10-formyltetrahydrofolate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R00941      One carbon pool by folate\n",
      "R00851: acyl-CoA:sn-glycerol-3-phosphate 1-O-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R00851      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00564+R00851      Glycerophospholipid metabolism\n",
      "R00847: ATP:glycerol 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R00847      Glycerolipid metabolism\n",
      "R00806: UDP-glucose:D-fructose 2-alpha-D-glucosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00806      Starch and sucrose metabolism\n",
      "R00704: (R)-Lactate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00620+R00704      Pyruvate metabolism\n",
      "R00568\n",
      "     no rxn\n",
      "R00528: sulfite:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00528      Sulfur metabolism\n",
      "R00483: L-aspartate:ammonia ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00483      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00460+R00483      Cyanoamino acid metabolism\n",
      "R00418: UDP-N-acetyl-D-glucosamine 4-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00418      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00418      O-Antigen nucleotide sugar biosynthesis\n",
      "R00217: oxaloacetate carboxy-lyase (pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00217      Pyruvate metabolism\n",
      "R00149: Carbon-dioxide:ammonia ligase (ADP-forming,carbamate-phosphorylating)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00149      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00149      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00149      Nitrogen metabolism\n",
      "R00093: L-glutamate:NAD+ oxidoreductase (transaminating)\n",
      "     https://www.kegg.jp/pathway/rn00250+R00093      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00910+R00093      Nitrogen metabolism\n",
      "R00086: ATP phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00086      Purine metabolism\n",
      "R00082: ferrocytochrome c2:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00082\n",
      "R00004: diphosphate phosphohydrolase;\n",
      "     https://www.kegg.jp/entry/R00004\n",
      "R_formylTHMPT_formate\n",
      "     no rxn\n",
      "NADH_DH_ubi\n",
      "     no rxn\n",
      "Maintenance_ATP\n",
      "     no rxn\n",
      "Cytochrome_c_oxidase\n",
      "     no rxn\n",
      "Biomass_reaction\n",
      "     no rxn\n",
      "ATPSynth\n",
      "     no rxn\n",
      "ATPase\n",
      "     no rxn\n",
      "R08159: tetradecanoyl-[acyl-carrier protein] hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00061+R08159      Fatty acid biosynthesis\n",
      "R03370: stearoyl-[acyl-carrier protein],reduced-ferredoxin:oxygen oxidoreductase (9,10 cis-dehydrogenating)\n",
      "     https://www.kegg.jp/pathway/rn00061+R03370      Fatty acid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn01040+R03370      Biosynthesis of unsaturated fatty acids\n",
      "R02814: Oleoyl-[acyl-carrier-protein] hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00061+R02814      Fatty acid biosynthesis\n",
      "R00258: L-Alanine:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00258      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00250+R00258      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00258      Carbon fixation in photosynthetic organisms\n",
      "R00485: L-asparagine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00485      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00460+R00485      Cyanoamino acid metabolism\n",
      "R00576: L-Glutamine:pyruvate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00576      Alanine, aspartate and glutamate metabolism\n",
      "R01215: L-Valine:pyruvate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00290+R01215      Valine, leucine and isoleucine biosynthesis\n",
      "R00345: phosphate:oxaloacetate carboxy-lyase (adding phosphate;phosphoenolpyruvate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00620+R00345      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00345      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00345      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00345      Carbon fixation pathways in prokaryotes\n",
      "EX_Pyr\n",
      "     no rxn\n",
      "EX_Pi\n",
      "     no rxn\n",
      "EX_O2\n",
      "     no rxn\n",
      "EX_nitrite\n",
      "     no rxn\n",
      "EX_nitrate\n",
      "     no rxn\n",
      "EX_NH3\n",
      "     no rxn\n",
      "EX_N2\n",
      "     no rxn\n",
      "EX_Lact\n",
      "     no rxn\n",
      "EX_Heme\n",
      "     no rxn\n",
      "EX_H2O\n",
      "     no rxn\n",
      "EX_H\n",
      "     no rxn\n",
      "EX_Formate\n",
      "     no rxn\n",
      "EX_EtOH\n",
      "     no rxn\n",
      "EX_CO2\n",
      "     no rxn\n",
      "EX_CH4\n",
      "     no rxn\n",
      "EX_C00305\n",
      "     no rxn\n",
      "EX_C00175\n",
      "     no rxn\n",
      "EX_C00076\n",
      "     no rxn\n",
      "EX_C00070\n",
      "     no rxn\n",
      "EX_C00023\n",
      "     no rxn\n",
      "EX_Ac\n",
      "     no rxn\n",
      "EX_sulfate\n",
      "     no rxn\n",
      "EX_Suc\n",
      "     no rxn\n",
      "CCL160161_tx\n",
      "     no rxn\n",
      "CCL161140_tx\n",
      "     no rxn\n",
      "CCL161161_tx\n",
      "     no rxn\n",
      "CPC160161_tx\n",
      "     no rxn\n",
      "CPC161140_tx\n",
      "     no rxn\n",
      "CPC161161_tx\n",
      "     no rxn\n",
      "CPE160161_tx\n",
      "     no rxn\n",
      "CPE161140_tx\n",
      "     no rxn\n",
      "CPE161161_tx\n",
      "     no rxn\n",
      "CPG160161_tx\n",
      "     no rxn\n",
      "CPG161140_tx\n",
      "     no rxn\n",
      "CPG161161_tx\n",
      "     no rxn\n",
      "lipooligosaccharides_tx\n",
      "     no rxn\n",
      "lps_tx\n",
      "     no rxn\n",
      "putresine_tx\n",
      "     no rxn\n",
      "spermidine_tx\n",
      "     no rxn\n",
      "Urea_tx\n",
      "     no rxn\n",
      "Formate_tx_pc\n",
      "     no rxn\n",
      "Formate_tx_ep\n",
      "     no rxn\n",
      "Pyr_tx_pc\n",
      "     no rxn\n",
      "Pyr_tx_ep\n",
      "     no rxn\n",
      "Pi_tx_pc\n",
      "     no rxn\n",
      "Pi_tx_ec\n",
      "     no rxn\n",
      "O2_tx_pc\n",
      "     no rxn\n",
      "O2_tx_ep\n",
      "     no rxn\n",
      "nitrite_pc\n",
      "     no rxn\n",
      "nitrite_ep\n",
      "     no rxn\n",
      "nitrate_pc\n",
      "     no rxn\n",
      "nitrate_ep\n",
      "     no rxn\n",
      "NH3_tx_pc\n",
      "     no rxn\n",
      "NH3_tx_ep\n",
      "     no rxn\n",
      "N2_tx_pc\n",
      "     no rxn\n",
      "N2_tx_ep\n",
      "     no rxn\n",
      "Lact_tx_pc\n",
      "     no rxn\n",
      "Lact_tx_ep\n",
      "     no rxn\n",
      "Heme_tx_pc\n",
      "     no rxn\n",
      "Heme_tx_ep\n",
      "     no rxn\n",
      "H2O_tx_pc\n",
      "     no rxn\n",
      "H2O_tx_ep\n",
      "     no rxn\n",
      "EtOH_tx_pc\n",
      "     no rxn\n",
      "EtOH_tx_ep\n",
      "     no rxn\n",
      "CO2_tx_pc\n",
      "     no rxn\n",
      "CO2_tx_ep\n",
      "     no rxn\n",
      "CH4_tx_pc\n",
      "     no rxn\n",
      "CH4_tx_ep\n",
      "     no rxn\n",
      "C00305_tx_pc\n",
      "     no rxn\n",
      "C00305_tx_ep\n",
      "     no rxn\n",
      "C00175_tx_pc\n",
      "     no rxn\n",
      "C00175_tx_ep\n",
      "     no rxn\n",
      "C00076_tx_pc\n",
      "     no rxn\n",
      "C00076_tx_ep\n",
      "     no rxn\n",
      "C00070_tx_pc\n",
      "     no rxn\n",
      "C00070_tx_ep\n",
      "     no rxn\n",
      "C00023_tx_pc\n",
      "     no rxn\n",
      "C00023_tx_ep\n",
      "     no rxn\n",
      "Ac_tx_pc\n",
      "     no rxn\n",
      "Ac_tx_ep\n",
      "     no rxn\n",
      "sulfate_tx_pc\n",
      "     no rxn\n",
      "sulfate_tx_ep\n",
      "     no rxn\n",
      "Suc_tx_pc\n",
      "     no rxn\n",
      "Suc_tx_ep\n",
      "     no rxn\n",
      "EX_CCL160161\n",
      "     no rxn\n",
      "EX_CCL161140\n",
      "     no rxn\n",
      "EX_CCL161161\n",
      "     no rxn\n",
      "EX_CPC160161\n",
      "     no rxn\n",
      "EX_CPC161140\n",
      "     no rxn\n",
      "EX_CPC161161\n",
      "     no rxn\n",
      "EX_CPE160161\n",
      "     no rxn\n",
      "EX_CPE161140\n",
      "     no rxn\n",
      "EX_CPE161161\n",
      "     no rxn\n",
      "EX_CPG160161\n",
      "     no rxn\n",
      "EX_CPG161140\n",
      "     no rxn\n",
      "EX_CPG161161\n",
      "     no rxn\n",
      "EX_lipooligosaccharides\n",
      "     no rxn\n",
      "EX_lps\n",
      "     no rxn\n",
      "EX_putresine\n",
      "     no rxn\n",
      "EX_spermidine\n",
      "     no rxn\n",
      "EX_Urea\n",
      "     no rxn\n",
      "alanine_tx\n",
      "     no rxn\n",
      "Aspargine_tx\n",
      "     no rxn\n",
      "Aspartate_tx\n",
      "     no rxn\n",
      "Cysteine_tx\n",
      "     no rxn\n",
      "Glutamate_tx\n",
      "     no rxn\n",
      "Glutamine_tx\n",
      "     no rxn\n",
      "Valine_tx\n",
      "     no rxn\n",
      "mercaptopyruvate_tx\n",
      "     no rxn\n",
      "EX_mercaptopyruvate\n",
      "     no rxn\n",
      "methyloxobutanoicacid_tx\n",
      "     no rxn\n",
      "EX_methyloxobutanoicacid\n",
      "     no rxn\n",
      "OAA_tx\n",
      "     no rxn\n",
      "EX_OAA\n",
      "     no rxn\n",
      "oxoglutarate_tx\n",
      "     no rxn\n",
      "EX_oxoglutarate\n",
      "     no rxn\n",
      "EX_alanine\n",
      "     no rxn\n",
      "EX_Aspargine\n",
      "     no rxn\n",
      "EX_Aspartate\n",
      "     no rxn\n",
      "EX_Cysteine\n",
      "     no rxn\n",
      "EX_Glutamate\n",
      "     no rxn\n",
      "EX_Glutamine\n",
      "     no rxn\n",
      "EX_Valine\n",
      "     no rxn\n",
      "EX_methanol\n",
      "     no rxn\n",
      "methanol_tx\n",
      "     no rxn\n",
      "EX_formaldehyde\n",
      "     no rxn\n",
      "formaldehyde_tx\n",
      "     no rxn\n",
      "ubiquinol_tx_pc\n",
      "     no rxn\n",
      "ubiquinone_tx_pc\n",
      "     no rxn\n",
      "ferricytochrome_tx_pc\n",
      "     no rxn\n",
      "ferrochrome_tx_pc\n",
      "     no rxn\n",
      "formaldehyde_tx_pc\n",
      "     no rxn\n",
      "methanol_tx_cp\n",
      "     no rxn\n",
      "R02164_ub\n",
      "     no rxn\n"
     ]
    }
   ],
   "source": [
    "addedinmodel=[r for r in mcoosmodel_rxnlist if r not in Mca_kegglist]\n",
    "print(len(addedinmodel))\n",
    "print('=====================')\n",
    "for r in addedinmodel:\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "157 132\n"
     ]
    }
   ],
   "source": [
    "notinmac = [r for r in spec_kegglist if r not in Mca_kegglist]\n",
    "notinspec = [r for r in Mca_kegglist if r not in spec_kegglist]\n",
    "print(len(notinmac), len(notinspec))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Arginine biosynthesis', 'Atrazine degradation'], dtype=object)"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rxn_raven_df[rxn_raven_df['id']==r]['subsystem'].values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "R00005: urea-1-carboxylate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00005      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00791+R00005      Atrazine degradation\n",
      "R00131: Urea amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00131      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00230+R00131      Purine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00791+R00131      Atrazine degradation\n",
      "R00143: ammonia:NAD+ oxidoreductase;\n",
      "     https://www.kegg.jp/pathway/rn00910+R00143      Nitrogen metabolism\n",
      "R00199: ATP:pyruvate,water phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R00199      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00620+R00199      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00199      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00199      Carbon fixation pathways in prokaryotes\n",
      "R00230: acetyl-CoA:phosphate acetyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00430+R00230      Taurine and hypotaurine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00620+R00230      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00230      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00230      Carbon fixation pathways in prokaryotes\n",
      "R00245: L-glutamate gamma-semialdehyde:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R00245      Arginine and proline metabolism\n",
      "R00267: Isocitrate:NADP+ oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00480+R00267      Glutathione metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00267      Carbon fixation pathways in prokaryotes\n",
      "R00268: oxalosuccinate carboxy-lyase (2-oxoglutarate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00268      Citrate cycle (TCA cycle)\n",
      "R00344: Pyruvate:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00020+R00344      Citrate cycle (TCA cycle)\n",
      "     https://www.kegg.jp/pathway/rn00620+R00344      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00344      Carbon fixation pathways in prokaryotes\n",
      "R00345: phosphate:oxaloacetate carboxy-lyase (adding phosphate;phosphoenolpyruvate-forming);\n",
      "     https://www.kegg.jp/pathway/rn00620+R00345      Pyruvate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R00345      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00345      Carbon fixation in photosynthetic organisms\n",
      "     https://www.kegg.jp/pathway/rn00720+R00345      Carbon fixation pathways in prokaryotes\n",
      "R00411: N-succinyl-L-glutamate amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R00411      Arginine and proline metabolism\n",
      "R00420: UDP-N-acetyl-D-glucosamine 2-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00420      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00420      O-Antigen nucleotide sugar biosynthesis\n",
      "R00421: UDP-N-acetyl-D-glucosamine:NAD+ 6-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00520+R00421      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00421      O-Antigen nucleotide sugar biosynthesis\n",
      "R00602: methanol:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/entry/R00602\n",
      "R00707: (S)-1-pyrroline-5-carboxylate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00707      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R00707      Arginine and proline metabolism\n",
      "R00708: (S)-1-pyrroline-5-carboxylate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00250+R00708      Alanine, aspartate and glutamate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00330+R00708      Arginine and proline metabolism\n",
      "R00749: ethanolamine ammonia-lyase (acetaldehyde-forming)\n",
      "     https://www.kegg.jp/pathway/rn00564+R00749      Glycerophospholipid metabolism\n",
      "R00751: L-threonine acetaldehyde-lyase (glycine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R00751      Glycine, serine and threonine metabolism\n",
      "R00762: D-Fructose-1,6-bisphosphate 1-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00762      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00710+R00762      Carbon fixation in photosynthetic organisms\n",
      "R00774: Urea:carbon-dioxide ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00220+R00774      Arginine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00791+R00774      Atrazine degradation\n",
      "R00783: nitric-oxide:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00783      Nitrogen metabolism\n",
      "R00785: nitric-oxide:oxidized azurin oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R00785      Nitrogen metabolism\n",
      "R00921: propanoyl-CoA:phosphate propanoyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00640+R00921      Propanoate metabolism\n",
      "R00956: CTP:alpha-D-glucose-1-phosphate cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00956      Starch and sucrose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00520+R00956      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R00956      O-Antigen nucleotide sugar biosynthesis\n",
      "R00997: 1-aminocyclopropane-1-carboxylate aminohydrolase (isomerizing);\n",
      "     https://www.kegg.jp/pathway/rn00270+R00997      Cysteine and methionine metabolism\n",
      "R01168: L-histidine ammonia-lyase (urocanate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00340+R01168      Histidine metabolism\n",
      "R01171: butanoyl-CoA:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00310+R01171      Lysine degradation\n",
      "     https://www.kegg.jp/pathway/rn00650+R01171      Butanoate metabolism\n",
      "R01253: L-proline:quinone oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R01253      Arginine and proline metabolism\n",
      "R01394: hydroxypyruvate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00630+R01394      Glyoxylate and dicarboxylate metabolism\n",
      "R01406: long-chain-fatty-acid:[acyl-carrier-protein] ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00071+R01406      Fatty acid degradation\n",
      "R01459: Cholesterol:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00984+R01459      Steroid degradation\n",
      "R01498: D-glucosyl-N-acylsphingosine glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00600+R01498      Sphingolipid metabolism\n",
      "R01513: 3-Phospho-D-glycerate:NAD+ 2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01513      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00270+R01513      Cysteine and methionine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R01513      Methane metabolism\n",
      "R01519: D-Glucono-1,5-lactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00030+R01519      Pentose phosphate pathway\n",
      "R01701: 3-methyl-2-oxobutanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating)\n",
      "     https://www.kegg.jp/entry/R01701\n",
      "R01702: 4-methyl-2-oxopentanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating)\n",
      "     https://www.kegg.jp/entry/R01702\n",
      "R01887: gamma-Amino-gamma-cyanobutanoate aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R01887      Cyanoamino acid metabolism\n",
      "R01899: Isocitrate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00020+R01899      Citrate cycle (TCA cycle)\n",
      "R02016: NADPH:oxidized-thioredoxin oxidoreductase\n",
      "     https://www.kegg.jp/entry/R02016\n",
      "R02173: Tryptamine:oxygen oxidoreductase(deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00380+R02173      Tryptophan metabolism\n",
      "R02382: tyramine:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R02382      Tyrosine metabolism\n",
      "R02426: CDPglucose 4,6-hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00520+R02426      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R02426      O-Antigen nucleotide sugar biosynthesis\n",
      "R02529: aminoacetone:oxygen oxidoreductase(deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00260+R02529      Glycine, serine and threonine metabolism\n",
      "R02532: 4-[(1R)-2-Amino-1-hydroxyethyl]-1,2-benzenediol:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R02532      Tyrosine metabolism\n",
      "R02596: Donor:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00940+R02596      Phenylpropanoid biosynthesis\n",
      "R02613: Phenethylamine:oxygen oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00360+R02613      Phenylalanine metabolism\n",
      "R02670: \n",
      "     https://www.kegg.jp/pathway/rn00380+R02670      Tryptophan metabolism\n",
      "R02689: UDP-alpha-D-glucose:1,2-diacyl-sn-glycerol 3-beta-D-glucosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R02689      Glycerolipid metabolism\n",
      "R02718: D-Alanine:poly(ribitol phosphate) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00470+R02718      D-Amino acid metabolism\n",
      "R02908: 5-Hydroxytryptamine:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00380+R02908      Tryptophan metabolism\n",
      "R02919: 4-[(1R)-1-Hydroxy-2-(methylamino)ethyl]-1,2-benzenediol:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R02919      Tyrosine metabolism\n",
      "R02933: L-Gulono-1,4-lactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00053+R02933      Ascorbate and aldarate metabolism\n",
      "R03062: Cephalosporin-C acetylhdrolase\n",
      "     https://www.kegg.jp/pathway/rn00311+R03062      Penicillin and cephalosporin biosynthesis\n",
      "R03093: 3-Indoleacetonitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00380+R03093      Tryptophan metabolism\n",
      "R03332: 1-Phosphatidyl-D-myo-inositol inositolphosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00562+R03332      Inositol phosphate metabolism\n",
      "R03542: alpha-Aminopropiononitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03542      Cyanoamino acid metabolism\n",
      "R03546: carbamate hydro-lyase;\n",
      "     https://www.kegg.jp/pathway/rn00910+R03546      Nitrogen metabolism\n",
      "R03596: hydrogen selenide:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00450+R03596      Selenocompound metabolism\n",
      "R03652: L-Glutamine:tRNA(Gln) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03652      Aminoacyl-tRNA biosynthesis\n",
      "R03751: epsilon-Caprolactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00930+R03751      Caprolactam degradation\n",
      "R03893: 4-Carboxymethylenebut-2-en-4-olide lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00361+R03893      Chlorocyclohexane and chlorobenzene degradation\n",
      "R03919: Donor:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00940+R03919      Phenylpropanoid biosynthesis\n",
      "R04007: Donor:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00940+R04007      Phenylpropanoid biosynthesis\n",
      "R04025: N-Acetylputrescine:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00330+R04025      Arginine and proline metabolism\n",
      "R04225: (S)-3-Methyl-2-oxopentanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating)\n",
      "     https://www.kegg.jp/entry/R04225\n",
      "R04300: 4-(2-aminoethyl)-1,2-benzenediol:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R04300      Tyrosine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00950+R04300      Isoquinoline alkaloid biosynthesis\n",
      "R04377: UDP-alpha-D-glucose:1,2-diacyl-3-O-(beta-D-glucopyranosyl)-sn-glycerol 6-glucosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00561+R04377      Glycerolipid metabolism\n",
      "R04444: L-1-pyrroline-3-hydroxy-5-carboxylate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R04444      Arginine and proline metabolism\n",
      "R04445: L-1-pyrroline-3-hydroxy-5-carboxylate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R04445      Arginine and proline metabolism\n",
      "R04674: N-methylhistamine:oxygen oxidoreductase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00340+R04674      Histidine metabolism\n",
      "R04780: beta-D-Fructose 1,6-bisphosphate 1-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00010+R04780      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R04780      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R04780      Fructose and mannose metabolism\n",
      "R04864: 2-Acyl-sn-glycero-3-phosphoethanolamine O-acyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R04864      Glycerophospholipid metabolism\n",
      "R04890: 3-Methoxytyramine:oxygen oxidoreductase (deaminating) (copper-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R04890      Tyrosine metabolism\n",
      "R04893: L-Normetanephrine:oxygen oxidoreductase (deaminating) (copper-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R04893      Tyrosine metabolism\n",
      "R04894: L-Metanephrine:oxygen oxidoreductase (deaminating) (flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00350+R04894      Tyrosine metabolism\n",
      "R04907: 3-Hydroxykynurenamine:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00380+R04907      Tryptophan metabolism\n",
      "R05051: L-erythro-4-Hydroxyglutamate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R05051      Arginine and proline metabolism\n",
      "R05176: ADP-D-glycero-D-manno-heptose 6-epimerase\n",
      "     https://www.kegg.jp/pathway/rn00540+R05176      Lipopolysaccharide biosynthesis\n",
      "R05227: hydrogenobyrinic-acid-a,c-diamide:cobalt cobalt-ligase (ADP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05227      Porphyrin metabolism\n",
      "R05510: Protoanemonin lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00361+R05510      Chlorocyclohexane and chlorobenzene degradation\n",
      "R05511: \n",
      "     https://www.kegg.jp/pathway/rn00361+R05511      Chlorocyclohexane and chlorobenzene degradation\n",
      "R05591: Nitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R05591      Aminobenzoate degradation\n",
      "R06171: L-allo-threonine acetaldehyde-lyase (glycine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00260+R06171      Glycine, serine and threonine metabolism\n",
      "R06835: 4-carboxymethylenebut-2-en-4-olide lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00361+R06835      Chlorocyclohexane and chlorobenzene degradation\n",
      "R06838: 4-carboxymethylenebut-2-en-4-olide lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00361+R06838      Chlorocyclohexane and chlorobenzene degradation\n",
      "R06894: \n",
      "     https://www.kegg.jp/pathway/rn00520+R06894      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R06894      O-Antigen nucleotide sugar biosynthesis\n",
      "R07236: S-adenosyl-L-methionine:gamma-tocopherol 5-O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R07236      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R07399: \n",
      "     https://www.kegg.jp/pathway/rn00290+R07399      Valine, leucine and isoleucine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00660+R07399      C5-Branched dibasic acid metabolism\n",
      "R07443: donor:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00940+R07443      Phenylpropanoid biosynthesis\n",
      "R07504: S-adenosyl-L-methionine:delta-tocopherol 5-O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R07504      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R07599: \n",
      "     https://www.kegg.jp/pathway/rn00280+R07599      Valine, leucine and isoleucine degradation\n",
      "R07600: \n",
      "     https://www.kegg.jp/pathway/rn00280+R07600      Valine, leucine and isoleucine degradation\n",
      "R07601: \n",
      "     https://www.kegg.jp/pathway/rn00280+R07601      Valine, leucine and isoleucine degradation\n",
      "R07602: \n",
      "     https://www.kegg.jp/pathway/rn00280+R07602      Valine, leucine and isoleucine degradation\n",
      "R07603: \n",
      "     https://www.kegg.jp/pathway/rn00280+R07603      Valine, leucine and isoleucine degradation\n",
      "R07604: \n",
      "     https://www.kegg.jp/pathway/rn00280+R07604      Valine, leucine and isoleucine degradation\n",
      "R07653: 4,4'-diapophytoene:FAD oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00906+R07653      Carotenoid biosynthesis\n",
      "R07654: 4,4'-diaponeurosporene,AH2:oxygen 4-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00906+R07654      Carotenoid biosynthesis\n",
      "R07855: phenylacetonitrile aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00643+R07855      Styrene degradation\n",
      "R08120: 5-fluoromuconolactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00364+R08120      Fluorobenzoate degradation\n",
      "R08121: 4-fluoromuconolactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00364+R08121      Fluorobenzoate degradation\n",
      "R08346: Citalopram:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00982+R08346      Drug metabolism - cytochrome P450\n",
      "R08347: Demethylcitalopram:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00982+R08347      Drug metabolism - cytochrome P450\n",
      "R08348: didemethylcitalopram:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "     https://www.kegg.jp/pathway/rn00982+R08348      Drug metabolism - cytochrome P450\n",
      "R08572: ATP:(R)-glycerate 2-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00030+R08572      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00260+R08572      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00561+R08572      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R08572      Glyoxylate and dicarboxylate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00680+R08572      Methane metabolism\n",
      "R08678: S-sulfanylglutathione:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R08678      Sulfur metabolism\n",
      "R09124: trimethylamine:coenzyme M methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R09124      Methane metabolism\n",
      "R09136: 2,3,5-trichlorodienelactone lactonohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00361+R09136      Chlorocyclohexane and chlorobenzene degradation\n",
      "R09220: dienelactone hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00623+R09220      Toluene degradation\n",
      "R09222: dienelactone hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00623+R09222      Toluene degradation\n",
      "R09372: methaneselenol:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00450+R09372      Selenocompound metabolism\n",
      "R09640: ATP:1,6-anhydro-N-acetylmuramate 6-phosphotransferase\n",
      "     https://www.kegg.jp/entry/R09640\n",
      "R09671: 4,4'-diapolycopene,AH2:oxygen oxidoreductase (4,4'-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00906+R09671      Carotenoid biosynthesis\n",
      "R09690: 4,4'-diapophytoene:FAD oxidoreductase\n",
      "     https://www.kegg.jp/entry/R09690\n",
      "R09697: UDP-N-acetylglucosamine hydro-lyase (inverting; UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-hexos-4-ulose-forming)\n",
      "     https://www.kegg.jp/pathway/rn00520+R09697      Amino sugar and nucleotide sugar metabolism\n",
      "     https://www.kegg.jp/pathway/rn00541+R09697      O-Antigen nucleotide sugar biosynthesis\n",
      "R09707: (5S,6S)-6-amino-5-hydroxycyclohexa-1,3-diene-1-carboxyate isomerase\n",
      "     https://www.kegg.jp/pathway/rn00405+R09707      Phenazine biosynthesis\n",
      "R09719: 4,4'-diapophytoene:FAD oxidoreductase\n",
      "     https://www.kegg.jp/entry/R09719\n",
      "R09727: 4,4'-diaponeurosporene,AH2:oxygen 4-oxidoreductase\n",
      "     https://www.kegg.jp/entry/R09727\n",
      "R09728: 4,4'-diaponeurosporen-4-al,AH2:oxygen 4-oxidoreductase\n",
      "     https://www.kegg.jp/entry/R09728\n",
      "R09744: 4,4'-diapophytoene:FAD oxidoreductase\n",
      "     https://www.kegg.jp/entry/R09744\n",
      "R09745: 4,4'-diapophytoene:FAD oxidoreductase\n",
      "     https://www.kegg.jp/entry/R09745\n",
      "R09746: 4,4'-diapophytoene:FAD oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00906+R09746      Carotenoid biosynthesis\n",
      "R10079: carbamate hydro-lyase;\n",
      "     https://www.kegg.jp/entry/R10079\n",
      "R10182: CTP:N,N'-diacetyllegionaminate cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R10182      Amino sugar and nucleotide sugar metabolism\n",
      "R10187: UDP-N,N'-diacetylbacillosamine hydrolase (2-epimerising)\n",
      "     https://www.kegg.jp/pathway/rn00520+R10187      Amino sugar and nucleotide sugar metabolism\n",
      "R10486: maltose 6'-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R10486      Starch and sucrose metabolism\n",
      "R10491: S-adenosyl-L-methionine:gamma-tocotrienol 5-O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R10491      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R10492: S-adenosyl-L-methionine:delta-tocotrienol 5-O-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00130+R10492      Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "R10507: L-proline:(acceptor) oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00330+R10507      Arginine and proline metabolism\n",
      "     https://www.kegg.jp/pathway/rn00332+R10507      Carbapenem biosynthesis\n",
      "R10802: 5,6,7,8-tetrahydromethanopterin:acceptor 5,6-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00790+R10802      Folate biosynthesis\n",
      "R10817: D-psicose 3-epimerase\n",
      "     https://www.kegg.jp/entry/R10817\n",
      "R10818: D-tagatose 3-epimerase\n",
      "     https://www.kegg.jp/entry/R10818\n",
      "R10880: \n",
      "     https://www.kegg.jp/pathway/rn00261+R10880      Monobactam biosynthesis\n",
      "R10939: (4S)-4-hydroxy-5-phosphooxypentane-2,3-dione aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/entry/R10939\n",
      "R10996: \n",
      "     https://www.kegg.jp/pathway/rn00640+R10996      Propanoate metabolism\n",
      "R10997: \n",
      "     https://www.kegg.jp/pathway/rn00640+R10997      Propanoate metabolism\n",
      "R11524: [CysO sulfur-carrier protein]-L-cysteine carboxypeptidase\n",
      "     https://www.kegg.jp/entry/R11524\n",
      "R11582: CTP:molybdenum cofactor cytidylyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00790+R11582      Folate biosynthesis\n",
      "R11785: N-acetyl-D-muramate 6-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00520+R11785      Amino sugar and nucleotide sugar metabolism\n",
      "R11906: isoniazid:hydrogen-peroxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00983+R11906      Drug metabolism - other enzymes\n",
      "R11914: 4,4'-diapolycopene-4,4'-dial,donor:oxygen oxidoreductase (4,4'-diapolycopene-4,4'-dioate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00906+R11914      Carotenoid biosynthesis\n",
      "R12000: siroheme carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R12000      Porphyrin metabolism\n",
      "R12390: O-acetyl-ADP-D-ribose carboxylesterase\n",
      "     https://www.kegg.jp/entry/R12390\n",
      "R12391: O-acetyl-ADP-D-ribose carboxylesterase\n",
      "     https://www.kegg.jp/entry/R12391\n",
      "R12554: \n",
      "     https://www.kegg.jp/pathway/rn00541+R12554      O-Antigen nucleotide sugar biosynthesis\n",
      "R12755: ATP:1,6-anhydro-N-acetyl-beta-muramate 6-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00520+R12755      Amino sugar and nucleotide sugar metabolism\n",
      "R12812: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12812      Teichoic acid biosynthesis\n",
      "R12863: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12863      Teichoic acid biosynthesis\n",
      "R12867: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12867      Teichoic acid biosynthesis\n",
      "R12871: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12871      Teichoic acid biosynthesis\n",
      "R12873: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12873      Teichoic acid biosynthesis\n",
      "R12875: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12875      Teichoic acid biosynthesis\n",
      "R12898: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12898      Teichoic acid biosynthesis\n",
      "R12899: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12899      Teichoic acid biosynthesis\n",
      "R12904: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12904      Teichoic acid biosynthesis\n",
      "R00590: L-serine hydro-lyase\n",
      "     https://www.kegg.jp/pathway/rn00270+R00590      Cysteine and methionine metabolism\n",
      "R00698: L-phenylalanine:oxygen oxidoreductase (decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00360+R00698      Phenylalanine metabolism\n"
     ]
    }
   ],
   "source": [
    "notinmacsystemlist =[]\n",
    "for r in notinmac:\n",
    "    systems = rxn_raven_df[rxn_raven_df['id']==r]['subsystem'].values\n",
    "    for s in systems:\n",
    "        notinmacsystemlist.append(s)\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "19 \t \n",
      "10 \t Amino sugar and nucleotide sugar metabolism\n",
      "10 \t Arginine and proline metabolism\n",
      "9 \t Teichoic acid biosynthesis\n",
      "7 \t Methane metabolism\n",
      "7 \t O-Antigen nucleotide sugar biosynthesis\n",
      "7 \t Tyrosine metabolism\n",
      "6 \t Valine, leucine and isoleucine degradation\n",
      "6 \t Chlorocyclohexane and chlorobenzene degradation\n",
      "5 \t Carbon fixation pathways in prokaryotes\n",
      "5 \t Carotenoid biosynthesis\n",
      "5 \t Glycine, serine and threonine metabolism\n",
      "5 \t Tryptophan metabolism\n",
      "4 \t Pyruvate metabolism\n",
      "4 \t Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "4 \t Nitrogen metabolism\n",
      "4 \t Phenylpropanoid biosynthesis\n",
      "3 \t Pentose phosphate pathway\n",
      "3 \t Drug metabolism - cytochrome P450\n",
      "3 \t Glycerolipid metabolism\n",
      "3 \t Atrazine degradation\n",
      "3 \t Arginine biosynthesis\n",
      "3 \t Citrate cycle (TCA cycle)\n",
      "3 \t Cysteine and methionine metabolism\n",
      "3 \t Propanoate metabolism\n",
      "2 \t Histidine metabolism\n",
      "2 \t Glycerophospholipid metabolism\n",
      "2 \t Carbon fixation in photosynthetic organisms\n",
      "2 \t Fluorobenzoate degradation\n",
      "2 \t Selenocompound metabolism\n",
      "2 \t Toluene degradation\n",
      "2 \t Glycolysis / Gluconeogenesis\n",
      "2 \t Phenylalanine metabolism\n",
      "2 \t Alanine, aspartate and glutamate metabolism\n",
      "2 \t Starch and sucrose metabolism\n",
      "2 \t Cyanoamino acid metabolism\n",
      "2 \t Folate biosynthesis\n",
      "2 \t Glyoxylate and dicarboxylate metabolism\n",
      "2 \t Porphyrin metabolism\n",
      "1 \t Phenazine biosynthesis\n",
      "1 \t Carbapenem biosynthesis\n",
      "1 \t Sulfur metabolism\n",
      "1 \t Glutathione metabolism\n",
      "1 \t Monobactam biosynthesis\n",
      "1 \t Styrene degradation\n",
      "1 \t Drug metabolism - other enzymes\n",
      "1 \t C5-Branched dibasic acid metabolism\n",
      "1 \t Valine, leucine and isoleucine biosynthesis\n",
      "1 \t Purine metabolism\n",
      "1 \t Aminobenzoate degradation\n",
      "1 \t Lipopolysaccharide biosynthesis\n",
      "1 \t Fructose and mannose metabolism\n",
      "1 \t Isoquinoline alkaloid biosynthesis\n",
      "1 \t Lysine degradation\n",
      "1 \t Caprolactam degradation\n",
      "1 \t Aminoacyl-tRNA biosynthesis\n",
      "1 \t Inositol phosphate metabolism\n",
      "1 \t Penicillin and cephalosporin biosynthesis\n",
      "1 \t Ascorbate and aldarate metabolism\n",
      "1 \t Taurine and hypotaurine metabolism\n",
      "1 \t Sphingolipid metabolism\n",
      "1 \t Steroid degradation\n",
      "1 \t Fatty acid degradation\n",
      "1 \t Butanoate metabolism\n",
      "1 \t D-Amino acid metabolism\n"
     ]
    }
   ],
   "source": [
    "count1 = pd.value_counts(notinmacsystemlist)\n",
    "for i in range(len(count1)):\n",
    "    print(count1[i],'\\t',count1.index[i],)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>name</th>\n",
       "      <th>eccode</th>\n",
       "      <th>reversibility</th>\n",
       "      <th>subsystem</th>\n",
       "      <th>subsystemid</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>R00005</td>\n",
       "      <td>urea-1-carboxylate amidohydrolase</td>\n",
       "      <td>3.5.1.54</td>\n",
       "      <td>1</td>\n",
       "      <td>Arginine biosynthesis</td>\n",
       "      <td>rn00220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>R00005</td>\n",
       "      <td>urea-1-carboxylate amidohydrolase</td>\n",
       "      <td>3.5.1.54</td>\n",
       "      <td>1</td>\n",
       "      <td>Atrazine degradation</td>\n",
       "      <td>rn00791</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>R00131</td>\n",
       "      <td>Urea amidohydrolase</td>\n",
       "      <td>3.5.1.5</td>\n",
       "      <td>1</td>\n",
       "      <td>Arginine biosynthesis</td>\n",
       "      <td>rn00220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>R00131</td>\n",
       "      <td>Urea amidohydrolase</td>\n",
       "      <td>3.5.1.5</td>\n",
       "      <td>1</td>\n",
       "      <td>Purine metabolism</td>\n",
       "      <td>rn00230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>R00131</td>\n",
       "      <td>Urea amidohydrolase</td>\n",
       "      <td>3.5.1.5</td>\n",
       "      <td>1</td>\n",
       "      <td>Atrazine degradation</td>\n",
       "      <td>rn00791</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11309</th>\n",
       "      <td>R12898</td>\n",
       "      <td></td>\n",
       "      <td>2.4.1.315</td>\n",
       "      <td>0</td>\n",
       "      <td>Teichoic acid biosynthesis</td>\n",
       "      <td>rn00552\\n</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11310</th>\n",
       "      <td>R12899</td>\n",
       "      <td></td>\n",
       "      <td>2.4.1.315</td>\n",
       "      <td>0</td>\n",
       "      <td>Teichoic acid biosynthesis</td>\n",
       "      <td>rn00552\\n</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11314</th>\n",
       "      <td>R12904</td>\n",
       "      <td></td>\n",
       "      <td>6.1.1.13</td>\n",
       "      <td>0</td>\n",
       "      <td>Teichoic acid biosynthesis</td>\n",
       "      <td>rn00552\\n</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11808</th>\n",
       "      <td>R00590</td>\n",
       "      <td>L-serine hydro-lyase</td>\n",
       "      <td>4.3.1.17</td>\n",
       "      <td>1</td>\n",
       "      <td>Cysteine and methionine metabolism</td>\n",
       "      <td>rn00270\\n</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11810</th>\n",
       "      <td>R00698</td>\n",
       "      <td>L-phenylalanine:oxygen oxidoreductase (decarbo...</td>\n",
       "      <td>1.11.1.21</td>\n",
       "      <td>1</td>\n",
       "      <td>Phenylalanine metabolism</td>\n",
       "      <td>rn00360\\n</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>195 rows  6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           id                                               name     eccode  \\\n",
       "2      R00005                  urea-1-carboxylate amidohydrolase   3.5.1.54   \n",
       "3      R00005                  urea-1-carboxylate amidohydrolase   3.5.1.54   \n",
       "111    R00131                                Urea amidohydrolase    3.5.1.5   \n",
       "112    R00131                                Urea amidohydrolase    3.5.1.5   \n",
       "113    R00131                                Urea amidohydrolase    3.5.1.5   \n",
       "...       ...                                                ...        ...   \n",
       "11309  R12898                                                     2.4.1.315   \n",
       "11310  R12899                                                     2.4.1.315   \n",
       "11314  R12904                                                      6.1.1.13   \n",
       "11808  R00590                               L-serine hydro-lyase   4.3.1.17   \n",
       "11810  R00698  L-phenylalanine:oxygen oxidoreductase (decarbo...  1.11.1.21   \n",
       "\n",
       "      reversibility                           subsystem subsystemid  \n",
       "2                 1               Arginine biosynthesis     rn00220  \n",
       "3                 1                Atrazine degradation     rn00791  \n",
       "111               1               Arginine biosynthesis     rn00220  \n",
       "112               1                   Purine metabolism     rn00230  \n",
       "113               1                Atrazine degradation     rn00791  \n",
       "...             ...                                 ...         ...  \n",
       "11309             0          Teichoic acid biosynthesis   rn00552\\n  \n",
       "11310             0          Teichoic acid biosynthesis   rn00552\\n  \n",
       "11314             0          Teichoic acid biosynthesis   rn00552\\n  \n",
       "11808             1  Cysteine and methionine metabolism   rn00270\\n  \n",
       "11810             1            Phenylalanine metabolism   rn00360\\n  \n",
       "\n",
       "[195 rows x 6 columns]"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rxn_raven_df[rxn_raven_df['id'].isin(notinmac)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "R00024: 3-phospho-D-glycerate carboxy-lyase (dimerizing; D-ribulose-1,5-bisphosphate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00710+R00024      Carbon fixation in photosynthetic organisms\n",
      "R00028: maltose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00500+R00028      Starch and sucrose metabolism\n",
      "R00134: formate:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00134      Methane metabolism\n",
      "     https://www.kegg.jp/pathway/rn00720+R00134      Carbon fixation pathways in prokaryotes\n",
      "R00182: AMP phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00230+R00182      Purine metabolism\n",
      "R00494: glutathione gamma-glutamylaminopeptidase\n",
      "     https://www.kegg.jp/pathway/rn00480+R00494      Glutathione metabolism\n",
      "R00508: 3'-phospho-5'-adenylyl sulfate 3'-phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00920+R00508      Sulfur metabolism\n",
      "R00510: cytidine-5'-monophosphate phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R00510      Pyrimidine metabolism\n",
      "R00548: riboflavin-5-phosphate phosphohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00740+R00548      Riboflavin metabolism\n",
      "R00669: N2-Acetyl-L-ornithine amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R00669      Arginine biosynthesis\n",
      "R00756: ATP:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00680+R00756      Methane metabolism\n",
      "R00782: L-cysteine hydrogen-sulfide-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R00782      Cysteine and methionine metabolism\n",
      "R00801: sucrose glucohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00052+R00801      Galactose metabolism\n",
      "     https://www.kegg.jp/pathway/rn00500+R00801      Starch and sucrose metabolism\n",
      "R00802: sucrose alpha-glucohydrolase\n",
      "     https://www.kegg.jp/entry/R00802\n",
      "R01085: 3-fumarylpyruvate fumarylhydrolase\n",
      "     https://www.kegg.jp/pathway/rn00350+R01085      Tyrosine metabolism\n",
      "R01142: Methane,NADH:oxygen oxidoreductase (hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00680+R01142      Methane metabolism\n",
      "R01167: L-histidine carboxy-lyase (histamine-forming)\n",
      "     https://www.kegg.jp/pathway/rn00340+R01167      Histidine metabolism\n",
      "R01262: glutathione:L-amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R01262      Glutathione metabolism\n",
      "R01286: L-cystathionine L-homocysteine-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R01286      Cysteine and methionine metabolism\n",
      "R01347: Fatty acid,reduced rubredoxin:oxygen oxidoreductase (omega-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00071+R01347      Fatty acid degradation\n",
      "R01384: UDP-D-glucuronate carboxy-lyase (UDP-D-xylose-forming)\n",
      "     https://www.kegg.jp/pathway/rn00520+R01384      Amino sugar and nucleotide sugar metabolism\n",
      "R01404: acyl-[acyl-carrier protein]:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R01404      Fatty acid biosynthesis\n",
      "R01457: cholesterol:NADP+ Delta24-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R01457      Steroid biosynthesis\n",
      "R01514: ATP:(R)-glycerate 3-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00260+R01514      Glycine, serine and threonine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00561+R01514      Glycerolipid metabolism\n",
      "     https://www.kegg.jp/pathway/rn00630+R01514      Glyoxylate and dicarboxylate metabolism\n",
      "R01523: ATP:D-ribulose-5-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00710+R01523      Carbon fixation in photosynthetic organisms\n",
      "R01687: (5-L-glutamyl)-peptide:taurine 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00430+R01687      Taurine and hypotaurine metabolism\n",
      "R01939: L-2-aminoadipate:2-oxoglutarate aminotransferase\n",
      "     https://www.kegg.jp/pathway/rn00300+R01939      Lysine biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00310+R01939      Lysine degradation\n",
      "R02000: rubredoxin:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00071+R02000      Fatty acid degradation\n",
      "R02030: CDP-diacylglycerol:phosphatidylglycerol 3-phosphatidyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00564+R02030      Glycerophospholipid metabolism\n",
      "R02061: trans,trans-Farnesyl-diphosphate:isopentenyl-diphosphate farnesyltranstransferase\n",
      "     https://www.kegg.jp/pathway/rn00900+R02061      Terpenoid backbone biosynthesis\n",
      "R02222: steroyl-CoA,hydrogen-donor:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn01040+R02222      Biosynthesis of unsaturated fatty acids\n",
      "R02408: L-cystine thiocysteine-lyase (deaminating; pyruvate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00270+R02408      Cysteine and methionine metabolism\n",
      "R02484: deoxyuridine:orthophosphate 2-deoxy-D-ribosyltransferase;\n",
      "     https://www.kegg.jp/pathway/rn00240+R02484      Pyrimidine metabolism\n",
      "R02540: 2-phenylacetamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00360+R02540      Phenylalanine metabolism\n",
      "     https://www.kegg.jp/pathway/rn00643+R02540      Styrene degradation\n",
      "R02550: toluene, ferredoxin:oxygen oxidoreductase (methyl-hydroxylating);\n",
      "     https://www.kegg.jp/pathway/rn00623+R02550      Toluene degradation\n",
      "R02728: beta-D-Glucose 1-phosphate 1,6-phosphomutase\n",
      "     https://www.kegg.jp/entry/R02728\n",
      "R02874: squalene,NADPH-hemoprotein:oxygen oxidoreductase (2,3-epoxidizing)\n",
      "     https://www.kegg.jp/pathway/rn00100+R02874      Steroid biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00909+R02874      Sesquiterpenoid and triterpenoid biosynthesis\n",
      "R02948: (S)-2-hydroxy-2-methyl-3-oxobutanoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00650+R02948      Butanoate metabolism\n",
      "     https://www.kegg.jp/pathway/rn00660+R02948      C5-Branched dibasic acid metabolism\n",
      "R03096: Indole-3-acetamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00380+R03096      Tryptophan metabolism\n",
      "R03140: D-Ribulose 1,5-bisphosphate oxygenase\n",
      "     https://www.kegg.jp/pathway/rn00630+R03140      Glyoxylate and dicarboxylate metabolism\n",
      "R03180: 4-Guanidinobutanamide amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00330+R03180      Arginine and proline metabolism\n",
      "R03236: ATP:D-tagatose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00052+R03236      Galactose metabolism\n",
      "R03470: 4-Carboxymuconolactone carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00362+R03470      Benzoate degradation\n",
      "R03562: Toluene,NADPH:oxogen oxidoreductase (3-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00623+R03562      Toluene degradation\n",
      "R03651: L-Glutamate:tRNA(Gln) ligase (AMP-forming)\n",
      "     https://www.kegg.jp/pathway/rn00970+R03651      Aminoacyl-tRNA biosynthesis\n",
      "R03689: lanosterol delta24-reductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R03689      Steroid biosynthesis\n",
      "R03867: (5-Glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/entry/R03867\n",
      "R03909: Monocarboxylic acid amide amidohydrolase\n",
      "     https://www.kegg.jp/entry/R03909\n",
      "R03916: (5-glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00480+R03916      Glutathione metabolism\n",
      "R03948: S-adenosyl-L-methionine:precorrin-4 C20-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R03948      Porphyrin metabolism\n",
      "R03970: (5-glutamyl)-peptide:amino-acid 5-glutamyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03970      Cyanoamino acid metabolism\n",
      "R03971: gamma-glutamyl-beta-aminopropiononitrile amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00460+R03971      Cyanoamino acid metabolism\n",
      "R04072: dolichyl-beta-D-mannosyl-phosphate:protein-serine O-D-mannosyltransferase\n",
      "     https://www.kegg.jp/entry/R04072\n",
      "R04430: butyryl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04430      Fatty acid biosynthesis\n",
      "R04725: dodecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04725      Fatty acid biosynthesis\n",
      "R04779: ATP:D-fructose-6-phosphate 1-phosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00010+R04779      Glycolysis / Gluconeogenesis\n",
      "     https://www.kegg.jp/pathway/rn00030+R04779      Pentose phosphate pathway\n",
      "     https://www.kegg.jp/pathway/rn00051+R04779      Fructose and mannose metabolism\n",
      "R04858: S-adenosyl-L-methionine:DNA (cytosine-5-)-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00270+R04858      Cysteine and methionine metabolism\n",
      "R04935: (5-L-glutamyl)-peptide:Se-Methylselenocysteine 5-glutamyltransferase\n",
      "     https://www.kegg.jp/entry/R04935\n",
      "R04941: Selenocystathionine L-homocysteine-lyase (deaminating)\n",
      "     https://www.kegg.jp/pathway/rn00450+R04941      Selenocompound metabolism\n",
      "R04956: hexanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04956      Fatty acid biosynthesis\n",
      "R04959: octanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04959      Fatty acid biosynthesis\n",
      "R04962: decanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04962      Fatty acid biosynthesis\n",
      "R04967: tetradecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04967      Fatty acid biosynthesis\n",
      "R04970: hexadecanoyl-[acp]:NADP+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R04970      Fatty acid biosynthesis\n",
      "R05149: S-Adenosyl-L-methionine:1-precorrin-6Y C5,15-methyltransferase (C-12-decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R05149      Porphyrin metabolism\n",
      "R05177: precorrin 8X 11,12-methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05177      Porphyrin metabolism\n",
      "R05180: S-adenosyl-L-methionine:precorrin-3B C17-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05180      Porphyrin metabolism\n",
      "R05181: S-adenosyl-L-methionine:precorrin-4 C11 methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05181      Porphyrin metabolism\n",
      "R05551: \n",
      "     https://www.kegg.jp/pathway/rn00643+R05551      Styrene degradation\n",
      "R05590: Acylamide aminohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00627+R05590      Aminobenzoate degradation\n",
      "R05640: lanosterol,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R05640      Steroid biosynthesis\n",
      "R05666: toluene,NADH:oxygen oxidoreductase (4-hydroxylating)\n",
      "     https://www.kegg.jp/pathway/rn00623+R05666      Toluene degradation\n",
      "R05703: lanosterol D24-reductase\n",
      "     https://www.kegg.jp/pathway/rn00100+R05703      Steroid biosynthesis\n",
      "R05731: obtusifoliol,[reduced NADPH---hemoprotein reductase]:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R05731      Steroid biosynthesis\n",
      "R05807: sirohydrochlorin cobalt-lyase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05807      Porphyrin metabolism\n",
      "R05808: S-adenosyl-L-methionine:cobalt-factor-II C20-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05808      Porphyrin metabolism\n",
      "R05809: \n",
      "     https://www.kegg.jp/entry/R05809\n",
      "R05810: cobalt-precorrin-4 methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05810      Porphyrin metabolism\n",
      "R05814: precorrin-8 11,12-methylmutase\n",
      "     https://www.kegg.jp/pathway/rn00860+R05814      Porphyrin metabolism\n",
      "R06087: \n",
      "     https://www.kegg.jp/entry/R06087\n",
      "R06088: \n",
      "     https://www.kegg.jp/entry/R06088\n",
      "R06363: Penicillin hydrolase\n",
      "     https://www.kegg.jp/pathway/rn00311+R06363      Penicillin and cephalosporin biosynthesis\n",
      "R06945: \n",
      "     https://www.kegg.jp/pathway/rn00930+R06945      Caprolactam degradation\n",
      "R07168: 5-methyltetrahydrofolate:NAD+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00670+R07168      One carbon pool by folate\n",
      "     https://www.kegg.jp/pathway/rn00720+R07168      Carbon fixation pathways in prokaryotes\n",
      "R07488: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07488      Steroid biosynthesis\n",
      "R07493: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07493      Steroid biosynthesis\n",
      "R07498: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07498      Steroid biosynthesis\n",
      "R07499: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07499      Steroid biosynthesis\n",
      "R07507: \n",
      "     https://www.kegg.jp/pathway/rn00100+R07507      Steroid biosynthesis\n",
      "R07620: \n",
      "     https://www.kegg.jp/pathway/rn00514+R07620      Other types of O-glycan biosynthesis\n",
      "     https://www.kegg.jp/pathway/rn00515+R07620      Mannose type O-glycan biosynthesis\n",
      "R07650: L-2,4-diaminobutanoate carboxy-lyase\n",
      "     https://www.kegg.jp/pathway/rn00260+R07650      Glycine, serine and threonine metabolism\n",
      "R07765: octadecanoyl-[acp]:NAD+ trans-2-oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00061+R07765      Fatty acid biosynthesis\n",
      "R07773: S-adenosyl-L-methionine:cobalt-precorrin-5B (C1)-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07773      Porphyrin metabolism\n",
      "R07774: S-adenosyl-L-methionine:cobalt-precorrin-6B C15-methyltransferase (C12-decarboxylating)\n",
      "     https://www.kegg.jp/pathway/rn00860+R07774      Porphyrin metabolism\n",
      "R07775: S-adenosyl-L-methionine:cobalt-precorrin-7 C5-methyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00860+R07775      Porphyrin metabolism\n",
      "R08014: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08014      Nitrotoluene degradation\n",
      "R08017: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08017      Nitrotoluene degradation\n",
      "R08042: \n",
      "     https://www.kegg.jp/pathway/rn00633+R08042      Nitrotoluene degradation\n",
      "R08057: GTP:GTP guanylyltransferase\n",
      "     https://www.kegg.jp/entry/R08057\n",
      "R08222: 5'-deoxy-5-fluorouridine:phosphate deoxy-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08222      Drug metabolism - other enzymes\n",
      "R08230: 5-fluorodeoxyuridine:phosphate deoxy-alpha-D-ribosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00983+R08230      Drug metabolism - other enzymes\n",
      "R08856: UDP-N-acetyl-D-glucosamine:undecaprenyl-phosphate N-acetyl-D-glucosamine phosphotransferase\n",
      "     https://www.kegg.jp/entry/R08856\n",
      "R09030: D-allose-6-phosphate aldose-ketose-isomerase\n",
      "     https://www.kegg.jp/pathway/rn00051+R09030      Fructose and mannose metabolism\n",
      "R09107: N-acetyl-L-citrulline amidohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00220+R09107      Arginine biosynthesis\n",
      "R09513: methanesulfonate,NADH:oxygen oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00920+R09513      Sulfur metabolism\n",
      "R10118: glutaryl-[acp]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10118      Biotin metabolism\n",
      "R10122: pimelyl-[acyl-carrier protein]-methyl-ester:NADP+ oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00780+R10122      Biotin metabolism\n",
      "R10137: UDP-N-acetyl-alpha-D-glucosamine:trans,octacis-decaprenyl-phosphate N-acetylglucosaminephosphotransferase\n",
      "     https://www.kegg.jp/entry/R10137\n",
      "R10164: hydroxylamine:ferricytochrome-c oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00910+R10164      Nitrogen metabolism\n",
      "R10597: chorismate pyruvate-lyase (3-hydroxybenzoate-forming)\n",
      "     https://www.kegg.jp/pathway/rn00400+R10597      Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "R10633: \n",
      "     https://www.kegg.jp/pathway/rn00760+R10633      Nicotinate and nicotinamide metabolism\n",
      "R10634: \n",
      "     https://www.kegg.jp/pathway/rn00760+R10634      Nicotinate and nicotinamide metabolism\n",
      "R10776: 2'-deoxyribonucleoside 5'-monophosphate phosphohydrolase\n",
      "     https://www.kegg.jp/entry/R10776\n",
      "R10938: L-alanyl-D-glutamate epimerase\n",
      "     https://www.kegg.jp/entry/R10938\n",
      "R11096: \n",
      "     https://www.kegg.jp/pathway/rn00100+R11096      Steroid biosynthesis\n",
      "R11310: beta-D-glucose 1,6-phosphomutase\n",
      "     https://www.kegg.jp/pathway/rn00500+R11310      Starch and sucrose metabolism\n",
      "R11399: dolichyl-beta-D-mannosyl-phosphate:protein-threonine O-D-mannosyltransferase\n",
      "     https://www.kegg.jp/entry/R11399\n",
      "R11435: \n",
      "     https://www.kegg.jp/pathway/rn01059+R11435      Biosynthesis of enediyne antibiotics\n",
      "R11530: ADP-alpha-D-glucose:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (configuration-retaining)\n",
      "     https://www.kegg.jp/pathway/rn00500+R11530      Starch and sucrose metabolism\n",
      "R11580: S-adenosyl-L-methionine:cobalt-factor III 17-methyltransferase (ring contracting)\n",
      "     https://www.kegg.jp/pathway/rn00860+R11580      Porphyrin metabolism\n",
      "R12075: UDP-N-acetyl-alpha-D-glucosamine:trans,octacis-decaprenyl-phosphate N-acetylglucosaminephosphotransferase\n",
      "     https://www.kegg.jp/pathway/rn00572+R12075      Arabinogalactan biosynthesis - Mycobacterium\n",
      "R12098: S-adenosyl-L-methionine hydrolase (adenosine-forming)\n",
      "     https://www.kegg.jp/entry/R12098\n",
      "R12157: S-adenosyl-L-methionine:1,6-didemethyltoxoflavin N6-methyltransferase (reumycin-forming)\n",
      "     https://www.kegg.jp/entry/R12157\n",
      "R12158: S-adenosyl-L-methionine:reumycin N1-methyltransferase (toxoflavin-forming)\n",
      "     https://www.kegg.jp/entry/R12158\n",
      "R12173: palmitoyl-CoA,ferrocytochrome-b5:oxygen oxidoreductase (9,10-dehydrogenating)\n",
      "     https://www.kegg.jp/pathway/rn01040+R12173      Biosynthesis of unsaturated fatty acids\n",
      "R12188: L-cysteine-S-conjugate thiol-lyase (deaminating; 2-aminoprop-2-enoate-forming)\n",
      "     https://www.kegg.jp/entry/R12188\n",
      "R12323: lanosterol,reduced ferredoxin:oxygen oxidoreductase (14-methyl cleaving)\n",
      "     https://www.kegg.jp/pathway/rn00100+R12323      Steroid biosynthesis\n",
      "R12578: glutathione:hydroperoxide oxidoreductase\n",
      "     https://www.kegg.jp/pathway/rn00480+R12578      Glutathione metabolism\n",
      "R12859: \n",
      "     https://www.kegg.jp/pathway/rn00552+R12859      Teichoic acid biosynthesis\n",
      "R12955: uridine 5'-monophosphate phosphoribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R12955      Pyrimidine metabolism\n",
      "R12956: thymidine 5'-monophosphate phosphodeoxyribohydrolase\n",
      "     https://www.kegg.jp/pathway/rn00240+R12956      Pyrimidine metabolism\n",
      "R13097: \n",
      "     https://www.kegg.jp/pathway/rn00470+R13097      D-Amino acid metabolism\n",
      "R09994: alpha-maltose 1-phosphate:(1->4)-alpha-D-glucan 4-alpha-D-maltosyltransferase\n",
      "     https://www.kegg.jp/pathway/rn00500+R09994      Starch and sucrose metabolism\n"
     ]
    }
   ],
   "source": [
    "notinspecsystemlist =[]\n",
    "for r in notinspec:\n",
    "    systems = rxn_raven_df[rxn_raven_df['id']==r]['subsystem'].values\n",
    "    for s in systems:\n",
    "        notinspecsystemlist.append(s)\n",
    "    printkeggrxn(r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "19 \t \n",
      "13 \t Steroid biosynthesis\n",
      "13 \t Porphyrin metabolism\n",
      "9 \t Fatty acid biosynthesis\n",
      "5 \t Starch and sucrose metabolism\n",
      "4 \t Cysteine and methionine metabolism\n",
      "4 \t Glutathione metabolism\n",
      "4 \t Pyrimidine metabolism\n",
      "3 \t Toluene degradation\n",
      "3 \t Nitrotoluene degradation\n",
      "3 \t Methane metabolism\n",
      "2 \t Glyoxylate and dicarboxylate metabolism\n",
      "2 \t Cyanoamino acid metabolism\n",
      "2 \t Drug metabolism - other enzymes\n",
      "2 \t Biotin metabolism\n",
      "2 \t Nicotinate and nicotinamide metabolism\n",
      "2 \t Styrene degradation\n",
      "2 \t Biosynthesis of unsaturated fatty acids\n",
      "2 \t Fructose and mannose metabolism\n",
      "2 \t Carbon fixation in photosynthetic organisms\n",
      "2 \t Sulfur metabolism\n",
      "2 \t Carbon fixation pathways in prokaryotes\n",
      "2 \t Galactose metabolism\n",
      "2 \t Arginine biosynthesis\n",
      "2 \t Fatty acid degradation\n",
      "2 \t Glycine, serine and threonine metabolism\n",
      "1 \t Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "1 \t Nitrogen metabolism\n",
      "1 \t Purine metabolism\n",
      "1 \t Biosynthesis of enediyne antibiotics\n",
      "1 \t Arabinogalactan biosynthesis - Mycobacterium\n",
      "1 \t Riboflavin metabolism\n",
      "1 \t Mannose type O-glycan biosynthesis\n",
      "1 \t Other types of O-glycan biosynthesis\n",
      "1 \t One carbon pool by folate\n",
      "1 \t Caprolactam degradation\n",
      "1 \t Penicillin and cephalosporin biosynthesis\n",
      "1 \t Aminobenzoate degradation\n",
      "1 \t Selenocompound metabolism\n",
      "1 \t Teichoic acid biosynthesis\n",
      "1 \t Glycerolipid metabolism\n",
      "1 \t Glycolysis / Gluconeogenesis\n",
      "1 \t Pentose phosphate pathway\n",
      "1 \t Tyrosine metabolism\n",
      "1 \t Taurine and hypotaurine metabolism\n",
      "1 \t Lysine biosynthesis\n",
      "1 \t Lysine degradation\n",
      "1 \t Glycerophospholipid metabolism\n",
      "1 \t Terpenoid backbone biosynthesis\n",
      "1 \t Amino sugar and nucleotide sugar metabolism\n",
      "1 \t Phenylalanine metabolism\n",
      "1 \t Histidine metabolism\n",
      "1 \t Sesquiterpenoid and triterpenoid biosynthesis\n",
      "1 \t Butanoate metabolism\n",
      "1 \t C5-Branched dibasic acid metabolism\n",
      "1 \t Tryptophan metabolism\n",
      "1 \t Arginine and proline metabolism\n",
      "1 \t Benzoate degradation\n",
      "1 \t Aminoacyl-tRNA biosynthesis\n",
      "1 \t D-Amino acid metabolism\n"
     ]
    }
   ],
   "source": [
    "count2 = pd.value_counts(notinspecsystemlist)\n",
    "for i in range(len(count2)):\n",
    "    print(count2[i],'\\t',count2.index[i],)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## mapping KEGG to BIGG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "df_kegg_reaction  = pd.read_excel(org_name+'_model\\\\'+org_name+'_clean_kegg_reaction.xlsx', index_col=0) \n",
    "oldkegglist=df_kegg_reaction['Kegg_ID'].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "348\n",
      "23\n"
     ]
    }
   ],
   "source": [
    "new = [r for r in kegglist if r not in oldkegglist]\n",
    "print(len(new))\n",
    "old = [r for r in oldkegglist if r not in kegglist]\n",
    "print(len(old))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "def select_rxn(unimodel, rxnlist):\n",
    "    if not isinstance(rxnlist,list):\n",
    "        unirxn = unimodel.reactions.get_by_id(rxnlist)\n",
    "        comlist=[]\n",
    "        for key, value in unirxn.metabolites.items():\n",
    "            comlist.append(key.id.split('_')[-1])\n",
    "        if set(comlist).issubset(set(['c','e','p'])):\n",
    "            return rxnlist\n",
    "        else:\n",
    "            return None\n",
    "    else:\n",
    "        newrxnlist = []\n",
    "        for rxn in rxnlist:\n",
    "            unirxn = unimodel.reactions.get_by_id(rxn)\n",
    "            comlist=[]\n",
    "            for key, value in unirxn.metabolites.items():\n",
    "                comlist.append(key.id.split('_')[-1])\n",
    "            if set(comlist).issubset(set(['c','e','p'])):\n",
    "                newrxnlist.append(rxn)\n",
    "        if len(newrxnlist) == 0:\n",
    "            return rxnlist\n",
    "        elif len(newrxnlist) > 1:\n",
    "            return min(newrxnlist, key=len)\n",
    "        else:\n",
    "            return newrxnlist[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "R00014 ['PDHam1hi', 'PDHam1mi']\n",
      "R00997 ['ACCAHh', 'ACCAHm']\n",
      "R01070 ['FBAf', 'FBAm', 'FBAh']\n",
      "R04672 ['APLh', 'APLm']\n",
      "581/1199\n"
     ]
    }
   ],
   "source": [
    "kegginbigg ={}\n",
    "for keggr in kegglist:\n",
    "    if KEGG_BIGG_dir.get(keggr):\n",
    "        selr = select_rxn(unimodel, KEGG_BIGG_dir.get(keggr))\n",
    "        if selr:\n",
    "            kegginbigg[keggr] = selr\n",
    "            if type(selr) is list:\n",
    "                print(keggr, selr)\n",
    "print('%d/%d'%(len(kegginbigg),len(kegglist)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "kegginbigg['R01070'] = 'FBA'\n",
    "kegginbigg['R00014'] = 'PDH'\n",
    "del kegginbigg['R00997']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "R01070 FBA\n",
      "R00746 ALCD2y\n",
      "R00754 ALCD2x\n",
      "R00014 PDH\n",
      "R00200 PYK\n",
      "R00658 ENO\n",
      "R01061 GAPD\n",
      "R01015 TPI\n",
      "R04780 BFBP\n",
      "R02740 PGIA\n",
      "R00959 PGCM\n",
      "R03321 G6PI3\n",
      "R02739 G6PI_1\n",
      "R01786 GLUKA\n",
      "R01512 PGK\n",
      "R01196 POR_syn\n",
      "R00235 ACS\n",
      "R01518 PGM\n",
      "R00206 PPDK\n",
      "R00199 PPS\n",
      "R00405 SUCOAS\n",
      "R00268 r0082\n",
      "R00709 ICDHx\n",
      "R01899 r0422\n",
      "R01082 FUM\n",
      "R01900 ACONTb\n",
      "R01325 ACONTa\n",
      "R00351 CS\n",
      "R00342 MDH\n",
      "R00344 PC\n",
      "R05605 EDA\n",
      "R01529 RPE\n",
      "R01830 TKT2\n",
      "R01528 GND\n",
      "R02035 PGL\n",
      "R02736 G6PBDH\n",
      "R02036 EDD\n",
      "R01827 TALA\n",
      "R08572 GLYCK2\n",
      "R00289 GALUi\n",
      "R00286 UDPGD\n",
      "R01430 XYLOR\n",
      "R00867 HEX7\n",
      "R01819 MPAKI\n",
      "R01818 PMANM\n",
      "R05692 GFUCS\n",
      "R00888 GMAND\n",
      "R00885 MAN1PT\n",
      "R02568 FBA2\n",
      "R00291 UDPG4E\n",
      "R01385 UGE\n",
      "R07764 3HAD180\n",
      "R04543 3OAR160\n",
      "R04566 3OAR140\n",
      "R04964 3OAR120\n",
      "R04534 3OAR100\n",
      "R04536 3OAR80\n",
      "R04953 3OAR60\n",
      "R01626 MCOATA\n",
      "R00742 ACCOAC\n",
      "R04355 KAS14\n",
      "R10707 KAS15\n",
      "R04954 3HAD60\n",
      "R04537 3HAD80\n",
      "R04535 3HAD100\n",
      "R04965 3HAD120\n",
      "R04568 3HAD140\n",
      "R04544 3HAD160\n",
      "R01280 FACOAL160\n",
      "R01302 CHRPL\n",
      "R07504 TOCOPHOM2\n",
      "R07236 TOCOPHOM1\n",
      "R00253 GLNS\n",
      "R01398 OCT\n",
      "R01954 ARGSS\n",
      "R01086 ARGSL\n",
      "R00259 ACGS\n",
      "R02649 ACGK\n",
      "R03443 AGPR\n",
      "R02283 ACOTA\n",
      "R02282 ORNTAC\n",
      "R00774 URCB\n",
      "R00005 ALPHNH\n",
      "R00131 UREA\n",
      "R00243 GDH1\n",
      "R00355 ASPTA\n",
      "R02019 RNDR2\n",
      "R01072 GLUPRT\n",
      "R04144 PPRGL\n",
      "R04560 AICART\n",
      "R04559 ADSL2r\n",
      "R04591 PRASCSi\n",
      "R01127 IMPC\n",
      "R00089 ADNCYC\n",
      "R02017 RNDR1\n",
      "R00127 ADK1\n",
      "R01083 ADSL1r\n",
      "R01135 ADSS\n",
      "R00183 NTD7\n",
      "R00190 ADPT\n",
      "R02088 NTD6\n",
      "R02297 PUNP7\n",
      "R01126 NTD11\n",
      "R01560 ADA\n",
      "R00722 NDPK9\n",
      "R01130 IMPD\n",
      "R01230 GMPS\n",
      "R01132 HXPRT\n",
      "R01561 PUNP1\n",
      "R01227 NTD9\n",
      "R02142 XPPT\n",
      "R02719 NTD10\n",
      "R01229 GPAR\n",
      "R00332 GK1\n",
      "R01676 GUAD\n",
      "R00336 PPGPPDP\n",
      "R03409 GDTP\n",
      "R02090 DGK1\n",
      "R01857 NDPK5\n",
      "R01968 NTD8\n",
      "R02147 PUNP3\n",
      "R00429 GTPDPK\n",
      "R00125 AP4AH\n",
      "R00529 SADT\n",
      "R00509 ADSK\n",
      "R00720 NTPP9\n",
      "R02720 NTPP11\n",
      "R01562 3NTD7pp\n",
      "R02148 3NTD9pp\n",
      "R01547 DADK\n",
      "R03530 NDPK10\n",
      "R00434 GUAC\n",
      "R04378 ADPT2\n",
      "R07405 AIRC3\n",
      "R00191 PDE1\n",
      "R03531 NTPP10\n",
      "R00087 NTPP6\n",
      "R00426 NTPP2\n",
      "R06974 GART\n",
      "R00330 NDPK1\n",
      "R01137 NDPK8\n",
      "R00185 ADNK1\n",
      "R01863 PUNP5\n",
      "R02018 RNDR4\n",
      "R00158 UMPK\n",
      "R00575 CBPS\n",
      "R01397 ASPCT\n",
      "R01993 DHORTS\n",
      "R01867 DHORDfum\n",
      "R01870 ORPT\n",
      "R00156 NDPK2\n",
      "R00571 CTPS1\n",
      "R00966 UPPRT\n",
      "R00963 NTD2\n",
      "R00570 NDPK3\n",
      "R00511 NTD4\n",
      "R00974 CSND\n",
      "R02024 RNDR3\n",
      "R02326 NDPK7\n",
      "R02325 DCTPD\n",
      "R01664 NTD3\n",
      "R02331 NDPK6\n",
      "R02098 URIDK2r\n",
      "R02101 TMDS\n",
      "R02093 NDPK4\n",
      "R02094 DTMPK\n",
      "R01569 NTD5\n",
      "R01411 MCSNAH\n",
      "R00965 OMPDC\n",
      "R01877 3NTD2pp\n",
      "R00662 NTPP8\n",
      "R02370 3NTD4pp\n",
      "R02102 NTD1\n",
      "R11323 NTPP7\n",
      "R00515 NTPP4\n",
      "R02100 DUTPDP\n",
      "R01665 CYTK2\n",
      "R00512 CYTK1\n",
      "R02296 PYNP1\n",
      "R01570 TMDPP\n",
      "R01876 PYNP2r\n",
      "R00578 ASNS1\n",
      "R00357 ASPO1\n",
      "R00369 AGTi\n",
      "R00396 ALAD_L\n",
      "R00713 SSALx\n",
      "R00114 GLUSy\n",
      "R00707 P5CD\n",
      "R00714 SSALy\n",
      "R00588 SPT_syn\n",
      "R00585 SPTc\n",
      "R01513 PGCD\n",
      "R00480 ASPK\n",
      "R04173 PSERT\n",
      "R00582 PSP_L\n",
      "R00945 GHMT2r\n",
      "R00996 THRD_L\n",
      "R00751 THRA\n",
      "R01466 THRS\n",
      "R02291 ASAD\n",
      "R00220 SERD_L\n",
      "R03425 GCCa\n",
      "R01773 HSDxi\n",
      "R02529 AACTOOR\n",
      "R04125 GCCb\n",
      "R03815 GCCc\n",
      "R02722 TRPS1\n",
      "R00372 GLYTA\n",
      "R01288 SHSL2\n",
      "R07364 ARD\n",
      "R07363 ARD1\n",
      "R07392 MDRPD\n",
      "R04420 MTRI\n",
      "R01402 MTAP\n",
      "R01401 MTAN\n",
      "R01920 SPMS\n",
      "R00178 ADMDC\n",
      "R00946 METS\n",
      "R04405 MHPGLUT\n",
      "R00177 METAT\n",
      "R00192 AHCi\n",
      "R00194 AHCYSNS\n",
      "R07395 ENOPH\n",
      "R00897 CYSS\n",
      "R00586 SERAT\n",
      "R03132 SLCYSS\n",
      "R00590 SERHL\n",
      "R02433 LCYSTAT\n",
      "R00896 CYSTA\n",
      "R01776 HSERTA\n",
      "R01777 HSST\n",
      "R10991 RE2034C\n",
      "R01214 VALTA\n",
      "R02199 ILETA\n",
      "R01090 LEUTA\n",
      "R03898 CITCIb\n",
      "R03968 IPPMIb\n",
      "R04426 IPMD\n",
      "R04001 IPPMIa\n",
      "R01213 IPPS\n",
      "R04441 DHAD1\n",
      "R05070 DHAD2\n",
      "R08648 ACHBS\n",
      "R00226 ACLS\n",
      "R04365 THDPS\n",
      "R04475 SDPTA\n",
      "R02734 SDPDS\n",
      "R02735 DAPE\n",
      "R00451 DAPDC\n",
      "R02401 OXPTNDH\n",
      "R04025 APRTO2\n",
      "R09081 CSPMDDC\n",
      "R00566 ARGDC\n",
      "R00411 SGDS\n",
      "R00667 ORNTA\n",
      "R01248 P5CRx\n",
      "R00245 PUTA3\n",
      "R00239 GLU5K\n",
      "R03313 G5SD\n",
      "R03291 HPROa\n",
      "R04444 PHCD\n",
      "R05051 4HGLSD\n",
      "R01168 HISDr\n",
      "R01071 ATPPRT\n",
      "R03243 HSTPT\n",
      "R03457 IGPDH\n",
      "R04640 PRMICI\n",
      "R04037 PRAMPC\n",
      "R04035 PRATPP\n",
      "R04558 IG3PS\n",
      "R02532 41R2A1H12BOOX\n",
      "R04880 34DHOXPEGOX\n",
      "R04300 42A12BOOX\n",
      "R00734 TYRTA\n",
      "R03342 34HPPYRI\n",
      "R04890 3MOXTYROX\n",
      "R02382 TYROXDAc\n",
      "R02613 PEAMNO\n",
      "R00694 PHETA1\n",
      "R06783 CINNDO\n",
      "R04907 3HXKYNOXDA\n",
      "R02908 5HOXINOXDA\n",
      "R02173 TRYPTAOX\n",
      "R03093 NTRLASE\n",
      "R01826 DDPA\n",
      "R01714 CHORS\n",
      "R02340 TRPS3\n",
      "R00674 TRPS2\n",
      "R03508 IGPS\n",
      "R03509 PRAIi\n",
      "R01073 ANPRT\n",
      "R00985 ANS2\n",
      "R01715 CHORM\n",
      "R01373 PPNDH\n",
      "R00691 AROH\n",
      "R01728 PPND\n",
      "R03460 PSCVT\n",
      "R02412 SHKK\n",
      "R02413 SHK3Dr\n",
      "R03084 DHQTi\n",
      "R03083 DHQS\n",
      "R00986 ANS\n",
      "R02473 PBAL\n",
      "R00489 ASP1DC\n",
      "R00230 PTAr\n",
      "R00315 ACKr\n",
      "R03542 NTRLASE2\n",
      "R01887 NTRLASE3\n",
      "R00260 GLUR\n",
      "R01150 ALAALAr\n",
      "R00401 ALAR\n",
      "R02783 UAMAGS\n",
      "R01148 ALATA_D\n",
      "R00274 GTHPe\n",
      "R00267 ICDHyr\n",
      "R00497 GTHS\n",
      "R00894 GLUCYS\n",
      "R00899 AMPTASECG\n",
      "R00251 OPAH\n",
      "R08359 APCS\n",
      "R00306 BGLA\n",
      "R00956 G1PCTYT\n",
      "R00948 GLGC\n",
      "R08639 PGMT\n",
      "R00299 HEX1\n",
      "R02060 PGAMT\n",
      "R00420 UAG2E\n",
      "R00660 UAGCVT\n",
      "R00022 HXAD\n",
      "R03920 ABFPT\n",
      "R02426 CDPGLC46DH\n",
      "R01187 MI3PP\n",
      "R02777 TDPDRR\n",
      "R06514 TDPDRE\n",
      "R06513 TDPGDH\n",
      "R02328 G1PTT\n",
      "R04657 TDSK\n",
      "R04606 LPADSS\n",
      "R05645 S7PI\n",
      "R05644 GMHEPAT\n",
      "R05074 MOAT2\n",
      "R04658 MOAT\n",
      "R05176 AGMHE\n",
      "R04549 USHD\n",
      "R03254 KDOPS\n",
      "R04550 U23GAAT\n",
      "R04587 UHGADA\n",
      "R04567 UAGAAT\n",
      "R01530 A5PISO\n",
      "R06285 MOAT2\n",
      "R05662 UAGPT2\n",
      "R03193 UAMAS\n",
      "R05032 UAGPT3\n",
      "R01041 ALCD19y\n",
      "R01012 DHAPT\n",
      "R01186 MI4PP\n",
      "R01185 MI1PP\n",
      "R01470 GPDDA2\n",
      "R00842 G3PD1ir\n",
      "R00848 G3PD\n",
      "R00749 ETHAAL\n",
      "R01320 RE3511C\n",
      "R01030 GPDDA1\n",
      "R03424 RE0066C\n",
      "R00345 PPC\n",
      "R02530 LGTHL\n",
      "R01736 GLYOX\n",
      "R00214 ME1\n",
      "R00236 r0068\n",
      "R00316 r0097\n",
      "R00317 ACYP_2\n",
      "R10866 PFOR\n",
      "R01394 HPI\n",
      "R00519 FDH\n",
      "R01334 PGLYCP\n",
      "R01221 GLYCL\n",
      "R00470 DDPGA\n",
      "R00921 PTA2\n",
      "R04425 MICITDr\n",
      "R01171 ACOAD1\n",
      "R00936 DHFR2i\n",
      "R02235 DHFOR\n",
      "R01655 MTHFC\n",
      "R01220 MTHFD\n",
      "R02301 FTHFCL\n",
      "R02300 THFAT\n",
      "R03940 FMETTRS\n",
      "R00943 FTHFLi\n",
      "R00937 THFOR1\n",
      "R09124 TMAMT\n",
      "R01068 FBA\n",
      "R00617 TMPK\n",
      "R05636 DXPS\n",
      "R03223 TMPPP\n",
      "R00615 TDP\n",
      "R07281 DB4PS\n",
      "R00161 AFAT\n",
      "R00549 RBFK\n",
      "R00425 GTPCII2\n",
      "R03459 DHPPDA2\n",
      "R03458 APRAUR\n",
      "R04457 RBFSa\n",
      "R00066 RBFSb\n",
      "R05086 4HTHRS\n",
      "R05085 OHPBAT\n",
      "R00481 ASPO6\n",
      "R04292 QULNS\n",
      "R03348 NNDPR\n",
      "R03004 DNADDP\n",
      "R03005 NNATr\n",
      "R00257 NADS2\n",
      "R02322 NMNDA\n",
      "R01724 NAMNPP\n",
      "R01268 NNAM\n",
      "R00103 NPH\n",
      "R00137 NMNAT\n",
      "R00104 NADK\n",
      "R00112 NADTRHD\n",
      "R02323 NMNR\n",
      "R03346 NT5C\n",
      "R00130 DPCOAK\n",
      "R03036 DPCOAPP\n",
      "R02971 APNPT\n",
      "R03269 PPCDC\n",
      "R04391 APCPT\n",
      "R04231 PPNCL2\n",
      "R03018 PNTK\n",
      "R02472 DPR\n",
      "R01226 MOHMT\n",
      "R04439 KARA1\n",
      "R03035 APPAT\n",
      "R03182 DBTS\n",
      "R03231 AMAOTr\n",
      "R03210 AOXSr\n",
      "R09725 PMEACPE\n",
      "R10124 AOXSr2\n",
      "R10117 OGMEACPD\n",
      "R10121 OPMEACPD\n",
      "R01716 ADCS\n",
      "R05553 ADCL\n",
      "R00942 THFGLUS\n",
      "R02237 DHFS\n",
      "R03067 DHPS2\n",
      "R03503 HPPK2\n",
      "R03504 DHNPA2r\n",
      "R04639 r0708\n",
      "R05048 r0777\n",
      "R05046 r0775\n",
      "R04620 AKP1\n",
      "R04286 PTHPS\n",
      "R09959 CPH4S\n",
      "R10002 CDGS\n",
      "R09978 CCGS\n",
      "R07605 CDGR\n",
      "R09594 H4MPTGL_atp\n",
      "R09726 MPTAT\n",
      "R00428 r0120\n",
      "R06895 CPPPGO2\n",
      "R03220 CPPPGO\n",
      "R07411 HEMEOS\n",
      "R00310 FCLT\n",
      "R00078 FEROc\n",
      "R03197 UPPDC1\n",
      "R03165 UPP3S\n",
      "R03194 UPP3MT\n",
      "R04972 UPPDC2\n",
      "R00084 HMBS\n",
      "R00036 PPBNGS\n",
      "R02272 G1SAT\n",
      "R04109 GLUTRR\n",
      "R05578 GLUTRS\n",
      "R02864 SHCHF\n",
      "R03947 SHCHD2\n",
      "R05227 COCHL\n",
      "R05221 ADOCBIK\n",
      "R01492 CBLAT\n",
      "R05222 ACBIPGT\n",
      "R05223 ADOCBLS\n",
      "R04594 RZ5PP\n",
      "R04148 NNDMBRT\n",
      "R07412 HEMEAS\n",
      "R07302 ADCPS1\n",
      "R06529 ADCPS2\n",
      "R05688 DXPRIi\n",
      "R05633 MEPCT\n",
      "R05634 CDPMEK\n",
      "R05637 MECDPS\n",
      "R01658 DMATT\n",
      "R02003 GRTT\n",
      "R09248 OCTDPS\n",
      "R00702 PSPPS\n",
      "R07322 SQLC\n",
      "R07323 SQLC2\n",
      "R00143 HAMR\n",
      "R00787 NTRIR2x\n",
      "R10092 HCO3E\n",
      "R00858 SULR\n",
      "R02021 PAPSR\n",
      "R01931 CYANST\n",
      "R03656 ILETRS\n",
      "R03655 HISTRS\n",
      "R03660 PHETRS\n",
      "R03646 ARGTRS\n",
      "R03652 GLNTRS\n",
      "R03650 CYSTRS\n",
      "R03654 GLYTRS\n",
      "R05577 ASPTRS\n",
      "R03662 SERTRS\n",
      "R03659 METTRS\n",
      "R03665 VALTRS\n",
      "R03658 LYSTRS\n",
      "R03657 LEUTRS\n",
      "R03663 THRTRS\n",
      "R03664 TRPTRS\n",
      "R02918 TYRTRS\n",
      "R03661 PROTRS\n",
      "R03905 GLNTRAT\n",
      "R04212 ASNTRAT\n",
      "R04998 BDG2HCGHD\n",
      "R01231 GMPS2\n",
      "R00573 CTPS2\n",
      "R01775 HSDy\n",
      "R01251 P5CR\n",
      "R10507 PROD2\n",
      "R03293 HPROb\n",
      "R03192 UAPGR\n",
      "R00844 G3PD2\n",
      "R00939 DHFR\n",
      "R02236 FOLR2\n",
      "R00940 THFOR2\n",
      "R00789 NTRIR2y\n"
     ]
    }
   ],
   "source": [
    "keggtobigglist = []\n",
    "for keggr, biggr in kegginbigg.items():\n",
    "    keggtobigglist.append(biggr)\n",
    "    print(keggr, biggr)\n",
    "    \n",
    "kbigglist = list(set(keggtobigglist))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## mapping pathwaytool to BIGG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__49 \"RXN2B4Q-49\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__49 \"RXN2B4Q-49\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20407 \"RXN-20407\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20407 \"RXN-20407\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20401 \"RXN-20401\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20401 \"RXN-20401\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11865 \"RXN-11865\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11865 \"RXN-11865\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ISOLEUCINE__45____45__TRNA__45__LIGASE__45__RXN \"ISOLEUCINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ISOLEUCINE__45____45__TRNA__45__LIGASE__45__RXN \"ISOLEUCINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23229__45__GLYCOLLATE__47__NADP__47____47__GLYOX__47__NADPH__47__PROTON__46__36__46__ \"RXN-23229-GLYCOLLATE/NADP//GLYOX/NADPH/PROTON.36.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23229__45__GLYCOLLATE__47__NADP__47____47__GLYOX__47__NADPH__47__PROTON__46__36__46__ \"RXN-23229-GLYCOLLATE/NADP//GLYOX/NADPH/PROTON.36.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23229__45__GLYCOLLATE__47__NAD__47____47__GLYOX__47__NADH__47__PROTON__46__34__46__ \"RXN-23229-GLYCOLLATE/NAD//GLYOX/NADH/PROTON.34.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23229__45__GLYCOLLATE__47__NAD__47____47__GLYOX__47__NADH__47__PROTON__46__34__46__ \"RXN-23229-GLYCOLLATE/NAD//GLYOX/NADH/PROTON.34.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HYDROXYPYRUVATE__45__REDUCTASE__45__RXN__45__GLYCERATE__47__NADP__47____47__OH__45__PYR__47__NADPH__47__PROTON__46__36__46__ \"HYDROXYPYRUVATE-REDUCTASE-RXN-GLYCERATE/NADP//OH-PYR/NADPH/PROTON.36.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HYDROXYPYRUVATE__45__REDUCTASE__45__RXN__45__GLYCERATE__47__NADP__47____47__OH__45__PYR__47__NADPH__47__PROTON__46__36__46__ \"HYDROXYPYRUVATE-REDUCTASE-RXN-GLYCERATE/NADP//OH-PYR/NADPH/PROTON.36.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYCERATE__45__DEHYDROGENASE__45__RXN \"GLYCERATE-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYCERATE__45__DEHYDROGENASE__45__RXN \"GLYCERATE-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22623 \"RXN-22623\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22623 \"RXN-22623\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GUANYL__45__KIN__45__RXN \"GUANYL-KIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GUANYL__45__KIN__45__RXN \"GUANYL-KIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction N__45__ACETYLNEURAMINATE__45__SYNTHASE__45__RXN__45__PHOSPHO__45__ENOL__45__PYRUVATE__47__CPD__45__13462__47__WATER__47____47__CPD0__45__1123__47__Pi__46__52__46__ \"N-ACETYLNEURAMINATE-SYNTHASE-RXN-PHOSPHO-ENOL-PYRUVATE/CPD-13462/WATER//CPD0-1123/Pi.52.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction N__45__ACETYLNEURAMINATE__45__SYNTHASE__45__RXN__45__PHOSPHO__45__ENOL__45__PYRUVATE__47__CPD__45__13462__47__WATER__47____47__CPD0__45__1123__47__Pi__46__52__46__ \"N-ACETYLNEURAMINATE-SYNTHASE-RXN-PHOSPHO-ENOL-PYRUVATE/CPD-13462/WATER//CPD0-1123/Pi.52.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction N__45__ACETYLNEURAMINATE__45__SYNTHASE__45__RXN \"N-ACETYLNEURAMINATE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction N__45__ACETYLNEURAMINATE__45__SYNTHASE__45__RXN \"N-ACETYLNEURAMINATE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MALATE__45____45__COA__45__LIGASE__45__RXN \"MALATE--COA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MALATE__45____45__COA__45__LIGASE__45__RXN \"MALATE--COA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPNACETYLMURAMATEDEHYDROG__45__RXN \"UDPNACETYLMURAMATEDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPNACETYLMURAMATEDEHYDROG__45__RXN \"UDPNACETYLMURAMATEDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11860 \"RXN-11860\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11860 \"RXN-11860\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__747 \"RXN0-747\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__747 \"RXN0-747\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5414 \"RXN0-5414\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5414 \"RXN0-5414\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDP__45__NACMUR__45__ALA__45__LIG__45__RXN \"UDP-NACMUR-ALA-LIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDP__45__NACMUR__45__ALA__45__LIG__45__RXN \"UDP-NACMUR-ALA-LIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SPERMINE__45__SYNTHASE__45__RXN \"SPERMINE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SPERMINE__45__SYNTHASE__45__RXN \"SPERMINE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYOHMETRANS__45__RXN \"GLYOHMETRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYOHMETRANS__45__RXN \"GLYOHMETRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction OHMETHYLBILANESYN__45__RXN \"OHMETHYLBILANESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction OHMETHYLBILANESYN__45__RXN \"OHMETHYLBILANESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2883__45__METHYLENETETRAHYDROMETHANOPTERIN__47__NADP__47____47__METHENYLTETRAHYDROMETHANOPTERIN__47__NADPH__46__77__46__ \"RXN-2883-METHYLENETETRAHYDROMETHANOPTERIN/NADP//METHENYLTETRAHYDROMETHANOPTERIN/NADPH.77.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2883__45__METHYLENETETRAHYDROMETHANOPTERIN__47__NADP__47____47__METHENYLTETRAHYDROMETHANOPTERIN__47__NADPH__46__77__46__ \"RXN-2883-METHYLENETETRAHYDROMETHANOPTERIN/NADP//METHENYLTETRAHYDROMETHANOPTERIN/NADPH.77.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2883__45__METHYLENETETRAHYDROMETHANOPTERIN__47__NAD__47____47__METHENYLTETRAHYDROMETHANOPTERIN__47__NADH__46__75__46__ \"RXN-2883-METHYLENETETRAHYDROMETHANOPTERIN/NAD//METHENYLTETRAHYDROMETHANOPTERIN/NADH.75.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2883__45__METHYLENETETRAHYDROMETHANOPTERIN__47__NAD__47____47__METHENYLTETRAHYDROMETHANOPTERIN__47__NADH__46__75__46__ \"RXN-2883-METHYLENETETRAHYDROMETHANOPTERIN/NAD//METHENYLTETRAHYDROMETHANOPTERIN/NADH.75.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__5__46__5__46__1__45__RXN__45__2__45__HYDROXY__45__2__45__METHYLPROPANENITRILE__47__WATER__47____47__CPD__45__335__47__AMMONIUM__46__57__46__ \"3.5.5.1-RXN-2-HYDROXY-2-METHYLPROPANENITRILE/WATER//CPD-335/AMMONIUM.57.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__5__46__5__46__1__45__RXN__45__2__45__HYDROXY__45__2__45__METHYLPROPANENITRILE__47__WATER__47____47__CPD__45__335__47__AMMONIUM__46__57__46__ \"3.5.5.1-RXN-2-HYDROXY-2-METHYLPROPANENITRILE/WATER//CPD-335/AMMONIUM.57.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__5__46__5__46__1__45__RXN__45__HCN__47__WATER__47____47__FORMATE__47__AMMONIUM__46__28__46__ \"3.5.5.1-RXN-HCN/WATER//FORMATE/AMMONIUM.28.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__5__46__5__46__1__45__RXN__45__HCN__47__WATER__47____47__FORMATE__47__AMMONIUM__46__28__46__ \"3.5.5.1-RXN-HCN/WATER//FORMATE/AMMONIUM.28.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18201 \"RXN-18201\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18201 \"RXN-18201\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRNA__45__S__45__TRANSFERASE__45__RXN \"TRNA-S-TRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRNA__45__S__45__TRANSFERASE__45__RXN \"TRNA-S-TRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__1404 \"RXN-1404\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__1404 \"RXN-1404\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15878 \"RXN-15878\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15878 \"RXN-15878\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LIPIDXSYNTHESIS__45__RXN \"LIPIDXSYNTHESIS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LIPIDXSYNTHESIS__45__RXN \"LIPIDXSYNTHESIS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20786 \"RXN-20786\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20786 \"RXN-20786\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22239 \"RXN-22239\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22239 \"RXN-22239\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__51 \"RXN2B4Q-51\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__51 \"RXN2B4Q-51\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__7 \"TRANS-RXN2PP0-7\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__7 \"TRANS-RXN2PP0-7\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7008 \"RXN0-7008\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7008 \"RXN0-7008\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17607 \"RXN-17607\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17607 \"RXN-17607\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8999 \"RXN-8999\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8999 \"RXN-8999\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2DNV__45__1 \"RXN2DNV-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2DNV__45__1 \"RXN2DNV-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHENYLALANINE__45____45__TRNA__45__LIGASE__45__RXN \"PHENYLALANINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHENYLALANINE__45____45__TRNA__45__LIGASE__45__RXN \"PHENYLALANINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23220 \"RXN-23220\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23220 \"RXN-23220\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__10__46__1__45__RXN \"2.7.10.1-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__10__46__1__45__RXN \"2.7.10.1-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16665 \"RXN-16665\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16665 \"RXN-16665\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19302 \"RXN-19302\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19302 \"RXN-19302\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11484 \"RXN-11484\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11484 \"RXN-11484\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19318 \"RXN-19318\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19318 \"RXN-19318\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5401 \"RXN0-5401\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5401 \"RXN0-5401\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22736 \"RXN-22736\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22736 \"RXN-22736\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16660 \"RXN-16660\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16660 \"RXN-16660\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction COBALAMIN5PSYN__45__RXN \"COBALAMIN5PSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction COBALAMIN5PSYN__45__RXN \"COBALAMIN5PSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12610 \"RXN-12610\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12610 \"RXN-12610\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19308 \"RXN-19308\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19308 \"RXN-19308\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19236 \"RXN-19236\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19236 \"RXN-19236\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16759 \"RXN-16759\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16759 \"RXN-16759\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__2__46__13__45__RXN \"2.3.2.13-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__2__46__13__45__RXN \"2.3.2.13-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2PP0__45__1 \"RXN2PP0-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2PP0__45__1 \"RXN2PP0-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4__46__2__46__1__46__99__45__RXN \"4.2.1.99-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4__46__2__46__1__46__99__45__RXN \"4.2.1.99-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETOLACTREDUCTOISOM__45__RXN \"ACETOLACTREDUCTOISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETOLACTREDUCTOISOM__45__RXN \"ACETOLACTREDUCTOISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11481 \"RXN-11481\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11481 \"RXN-11481\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBONUCLEOSIDE__45__DIP__45__REDUCTII__45__RXN \"RIBONUCLEOSIDE-DIP-REDUCTII-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBONUCLEOSIDE__45__DIP__45__REDUCTII__45__RXN \"RIBONUCLEOSIDE-DIP-REDUCTII-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GART__45__RXN \"GART-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GART__45__RXN \"GART-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DEOXYADENPHOSPHOR__45__RXN \"DEOXYADENPHOSPHOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DEOXYADENPHOSPHOR__45__RXN \"DEOXYADENPHOSPHOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DMBPPRIBOSYLTRANS__45__RXN \"DMBPPRIBOSYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DMBPPRIBOSYLTRANS__45__RXN \"DMBPPRIBOSYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DEOXYINOPHOSPHOR__45__RXN \"DEOXYINOPHOSPHOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DEOXYINOPHOSPHOR__45__RXN \"DEOXYINOPHOSPHOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19295 \"RXN-19295\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19295 \"RXN-19295\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19298 \"RXN-19298\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19298 \"RXN-19298\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AICARTRANSFORM__45__RXN \"AICARTRANSFORM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AICARTRANSFORM__45__RXN \"AICARTRANSFORM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DEOXYGUANPHOSPHOR__45__RXN \"DEOXYGUANPHOSPHOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DEOXYGUANPHOSPHOR__45__RXN \"DEOXYGUANPHOSPHOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__4821 \"RXN-4821\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__4821 \"RXN-4821\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19306 \"RXN-19306\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19306 \"RXN-19306\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13908 \"RXN-13908\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13908 \"RXN-13908\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENPHOSPHOR__45__RXN \"ADENPHOSPHOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENPHOSPHOR__45__RXN \"ADENPHOSPHOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19301 \"RXN-19301\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19301 \"RXN-19301\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction INOPHOSPHOR__45__RXN \"INOPHOSPHOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction INOPHOSPHOR__45__RXN \"INOPHOSPHOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19299 \"RXN-19299\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19299 \"RXN-19299\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUC1PADENYLTRANS__45__RXN \"GLUC1PADENYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUC1PADENYLTRANS__45__RXN \"GLUC1PADENYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11591 \"RXN-11591\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11591 \"RXN-11591\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19316 \"RXN-19316\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19316 \"RXN-19316\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction XANTHOSINEPHOSPHORY__45__RXN \"XANTHOSINEPHOSPHORY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction XANTHOSINEPHOSPHORY__45__RXN \"XANTHOSINEPHOSPHORY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19300 \"RXN-19300\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19300 \"RXN-19300\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5199 \"RXN0-5199\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5199 \"RXN0-5199\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19319 \"RXN-19319\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19319 \"RXN-19319\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19340 \"RXN-19340\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19340 \"RXN-19340\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction URA__45__PHOSPH__45__RXN \"URA-PHOSPH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction URA__45__PHOSPH__45__RXN \"URA-PHOSPH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8991 \"RXN-8991\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8991 \"RXN-8991\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction COBINAMIDEKIN__45__RXN \"COBINAMIDEKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction COBINAMIDEKIN__45__RXN \"COBINAMIDEKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction URPHOS__45__RXN \"URPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction URPHOS__45__RXN \"URPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SAICARSYN__45__RXN \"SAICARSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SAICARSYN__45__RXN \"SAICARSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23038 \"RXN-23038\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23038 \"RXN-23038\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDP__45__NACMURALA__45__GLU__45__LIG__45__RXN \"UDP-NACMURALA-GLU-LIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDP__45__NACMURALA__45__GLU__45__LIG__45__RXN \"UDP-NACMURALA-GLU-LIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14063 \"RXN-14063\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14063 \"RXN-14063\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THYM__45__PHOSPH__45__RXN \"THYM-PHOSPH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THYM__45__PHOSPH__45__RXN \"THYM-PHOSPH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__882 \"RXN0-882\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__882 \"RXN0-882\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__1__46__156__45__RXN \"2.7.1.156-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__1__46__156__45__RXN \"2.7.1.156-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__21__46__92__45__RXN__45__General__45__Protein__45__Substrates__47__ATP__47__WATER__47____47__Peptides__45__holder__47__ADP__47__Pi__47__PROTON__46__68__46__ \"3.4.21.92-RXN-General-Protein-Substrates/ATP/WATER//Peptides-holder/ADP/Pi/PROTON.68.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__21__46__92__45__RXN__45__General__45__Protein__45__Substrates__47__ATP__47__WATER__47____47__Peptides__45__holder__47__ADP__47__Pi__47__PROTON__46__68__46__ \"3.4.21.92-RXN-General-Protein-Substrates/ATP/WATER//Peptides-holder/ADP/Pi/PROTON.68.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12515 \"RXN-12515\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12515 \"RXN-12515\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLURS__45__RXN \"GLURS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLURS__45__RXN \"GLURS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10857 \"RXN-10857\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10857 \"RXN-10857\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22734 \"RXN-22734\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22734 \"RXN-22734\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__21__46__53__45__RXN__45__General__45__Protein__45__Substrates__47__ATP__47__WATER__47____47__Peptides__45__holder__47__ADP__47__Pi__47__PROTON__46__68__46__ \"3.4.21.53-RXN-General-Protein-Substrates/ATP/WATER//Peptides-holder/ADP/Pi/PROTON.68.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__21__46__53__45__RXN__45__General__45__Protein__45__Substrates__47__ATP__47__WATER__47____47__Peptides__45__holder__47__ADP__47__Pi__47__PROTON__46__68__46__ \"3.4.21.53-RXN-General-Protein-Substrates/ATP/WATER//Peptides-holder/ADP/Pi/PROTON.68.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16649 \"RXN-16649\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16649 \"RXN-16649\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12002 \"RXN-12002\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12002 \"RXN-12002\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRNA__45__GUANINE__45__N7__45____45__METHYLTRANSFERASE__45__RXN \"TRNA-GUANINE-N7--METHYLTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRNA__45__GUANINE__45__N7__45____45__METHYLTRANSFERASE__45__RXN \"TRNA-GUANINE-N7--METHYLTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDP__45__GLUCOSIDASE__45__RXN__45__GDP__45__D__45__GLUCOSE__47__WATER__47____47__GDP__47__ALPHA__45__GLUCOSE__47__PROTON__46__46__46__ \"GDP-GLUCOSIDASE-RXN-GDP-D-GLUCOSE/WATER//GDP/ALPHA-GLUCOSE/PROTON.46.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDP__45__GLUCOSIDASE__45__RXN__45__GDP__45__D__45__GLUCOSE__47__WATER__47____47__GDP__47__ALPHA__45__GLUCOSE__47__PROTON__46__46__46__ \"GDP-GLUCOSIDASE-RXN-GDP-D-GLUCOSE/WATER//GDP/ALPHA-GLUCOSE/PROTON.46.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDP__45__GLUCOSIDASE__45__RXN__45__GDP__45__D__45__GLUCOSE__47__WATER__47____47__GDP__47__GLC__47__PROTON__46__36__46__ \"GDP-GLUCOSIDASE-RXN-GDP-D-GLUCOSE/WATER//GDP/GLC/PROTON.36.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDP__45__GLUCOSIDASE__45__RXN__45__GDP__45__D__45__GLUCOSE__47__WATER__47____47__GDP__47__GLC__47__PROTON__46__36__46__ \"GDP-GLUCOSIDASE-RXN-GDP-D-GLUCOSE/WATER//GDP/GLC/PROTON.36.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDP__45__GLUCOSIDASE__45__RXN \"GDP-GLUCOSIDASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDP__45__GLUCOSIDASE__45__RXN \"GDP-GLUCOSIDASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1YI0__45__23 \"RXN1YI0-23\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1YI0__45__23 \"RXN1YI0-23\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DXPREDISOM__45__RXN \"DXPREDISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DXPREDISOM__45__RXN \"DXPREDISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12865 \"RXN-12865\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12865 \"RXN-12865\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12867 \"RXN-12867\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12867 \"RXN-12867\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8348 \"RXN-8348\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8348 \"RXN-8348\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12864 \"RXN-12864\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12864 \"RXN-12864\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THI__45__P__45__SYN__45__RXN \"THI-P-SYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THI__45__P__45__SYN__45__RXN \"THI-P-SYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5422 \"RXN0-5422\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5422 \"RXN0-5422\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12721 \"RXN-12721\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12721 \"RXN-12721\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRUVFORMLY__45__RXN \"PYRUVFORMLY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRUVFORMLY__45__RXN \"PYRUVFORMLY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21539__45__CPD__45__1302__47__Methionine__45__synthase__45__cob__45__I__45__alamins__47__PROTON__47____47__CPD__45__1301__47__METHIONINE__45__SYNTHASE__45__METHYLCOBALAMIN__46__96__46__ \"RXN-21539-CPD-1302/Methionine-synthase-cob-I-alamins/PROTON//CPD-1301/METHIONINE-SYNTHASE-METHYLCOBALAMIN.96.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21539__45__CPD__45__1302__47__Methionine__45__synthase__45__cob__45__I__45__alamins__47__PROTON__47____47__CPD__45__1301__47__METHIONINE__45__SYNTHASE__45__METHYLCOBALAMIN__46__96__46__ \"RXN-21539-CPD-1302/Methionine-synthase-cob-I-alamins/PROTON//CPD-1301/METHIONINE-SYNTHASE-METHYLCOBALAMIN.96.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21538 \"RXN-21538\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21538 \"RXN-21538\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4__46__2__46__99__46__18__45__RXN \"4.2.99.18-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4__46__2__46__99__46__18__45__RXN \"4.2.99.18-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22554 \"RXN-22554\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22554 \"RXN-22554\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYCINE__45____45__TRNA__45__LIGASE__45__RXN \"GLYCINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYCINE__45____45__TRNA__45__LIGASE__45__RXN \"GLYCINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11029 \"RXN-11029\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11029 \"RXN-11029\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22611 \"RXN-22611\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22611 \"RXN-22611\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8976 \"RXN-8976\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8976 \"RXN-8976\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11346 \"RXN-11346\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11346 \"RXN-11346\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NACGLCTRANS__45__RXN \"NACGLCTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NACGLCTRANS__45__RXN \"NACGLCTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__6__46__3__46__16__45__RXN \"3.6.3.16-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__6__46__3__46__16__45__RXN \"3.6.3.16-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HOLO__45__ACP__45__SYNTH__45__RXN \"HOLO-ACP-SYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HOLO__45__ACP__45__SYNTH__45__RXN \"HOLO-ACP-SYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22366 \"RXN-22366\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22366 \"RXN-22366\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22389 \"RXN-22389\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22389 \"RXN-22389\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20482 \"RXN-20482\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20482 \"RXN-20482\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENTDB__45__RXN \"ENTDB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENTDB__45__RXN \"ENTDB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DEPHOSPHOCOAKIN__45__RXN \"DEPHOSPHOCOAKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DEPHOSPHOCOAKIN__45__RXN \"DEPHOSPHOCOAKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19477 \"RXN-19477\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19477 \"RXN-19477\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21566 \"RXN-21566\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21566 \"RXN-21566\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19736 \"RXN-19736\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19736 \"RXN-19736\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PTAALT__45__RXN \"PTAALT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PTAALT__45__RXN \"PTAALT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15122 \"RXN-15122\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15122 \"RXN-15122\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN66__45__1 \"RXN66-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN66__45__1 \"RXN66-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HOMOSERDEHYDROG__45__RXN__45__HOMO__45__SER__47__NADP__47____47__L__45__ASPARTATE__45__SEMIALDEHYDE__47__NADPH__47__PROTON__46__53__46__ \"HOMOSERDEHYDROG-RXN-HOMO-SER/NADP//L-ASPARTATE-SEMIALDEHYDE/NADPH/PROTON.53.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HOMOSERDEHYDROG__45__RXN__45__HOMO__45__SER__47__NADP__47____47__L__45__ASPARTATE__45__SEMIALDEHYDE__47__NADPH__47__PROTON__46__53__46__ \"HOMOSERDEHYDROG-RXN-HOMO-SER/NADP//L-ASPARTATE-SEMIALDEHYDE/NADPH/PROTON.53.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23610 \"RXN-23610\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23610 \"RXN-23610\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CATAL__45__RXN \"CATAL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CATAL__45__RXN \"CATAL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction KETOBUTFORMLY__45__RXN \"KETOBUTFORMLY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction KETOBUTFORMLY__45__RXN \"KETOBUTFORMLY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14189 \"RXN-14189\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14189 \"RXN-14189\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSACETYLTRANS__45__RXN \"PHOSACETYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSACETYLTRANS__45__RXN \"PHOSACETYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__an__45__oxidized__45__NrfB__45__protein__47__WATER__47____47__NITRITE__47__a__45__reduced__45__NrfB__45__protein__47__PROTON__46__83__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/an-oxidized-NrfB-protein/WATER//NITRITE/a-reduced-NrfB-protein/PROTON.83.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__an__45__oxidized__45__NrfB__45__protein__47__WATER__47____47__NITRITE__47__a__45__reduced__45__NrfB__45__protein__47__PROTON__46__83__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/an-oxidized-NrfB-protein/WATER//NITRITE/a-reduced-NrfB-protein/PROTON.83.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__cytochromes__45__C4__47__WATER__47____47__NITRITE__47__Reduced__45__cytochromes__45__C4__47__PROTON__46__82__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-cytochromes-C4/WATER//NITRITE/Reduced-cytochromes-C4/PROTON.82.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__cytochromes__45__C4__47__WATER__47____47__NITRITE__47__Reduced__45__cytochromes__45__C4__47__PROTON__46__82__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-cytochromes-C4/WATER//NITRITE/Reduced-cytochromes-C4/PROTON.82.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__CycA1__45__cytochromes__47__WATER__47____47__NITRITE__47__Reduced__45__CycA1__45__cytochromes__47__PROTON__46__88__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-CycA1-cytochromes/WATER//NITRITE/Reduced-CycA1-cytochromes/PROTON.88.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__CycA1__45__cytochromes__47__WATER__47____47__NITRITE__47__Reduced__45__CycA1__45__cytochromes__47__PROTON__46__88__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-CycA1-cytochromes/WATER//NITRITE/Reduced-CycA1-cytochromes/PROTON.88.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__NapC__45__proteins__47__WATER__47____47__NITRITE__47__Reduced__45__NapC__45__proteins__47__PROTON__46__80__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-NapC-proteins/WATER//NITRITE/Reduced-NapC-proteins/PROTON.80.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__NapC__45__proteins__47__WATER__47____47__NITRITE__47__Reduced__45__NapC__45__proteins__47__PROTON__46__80__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-NapC-proteins/WATER//NITRITE/Reduced-NapC-proteins/PROTON.80.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__Cytochromes__45__C6__47__WATER__47____47__NITRITE__47__Reduced__45__Cytochromes__45__C6__47__PROTON__46__82__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-Cytochromes-C6/WATER//NITRITE/Reduced-Cytochromes-C6/PROTON.82.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Oxidized__45__Cytochromes__45__C6__47__WATER__47____47__NITRITE__47__Reduced__45__Cytochromes__45__C6__47__PROTON__46__82__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Oxidized-Cytochromes-C6/WATER//NITRITE/Reduced-Cytochromes-C6/PROTON.82.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Cytochromes__45__CL__45__Ox__47__WATER__47____47__NITRITE__47__Cytochromes__45__CL__45__Red__47__PROTON__46__72__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Cytochromes-CL-Ox/WATER//NITRITE/Cytochromes-CL-Red/PROTON.72.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN__45__NITRIC__45__OXIDE__47__Cytochromes__45__CL__45__Ox__47__WATER__47____47__NITRITE__47__Cytochromes__45__CL__45__Red__47__PROTON__46__72__46__ \"NITRITE-REDUCTASE-CYTOCHROME-RXN-NITRIC-OXIDE/Cytochromes-CL-Ox/WATER//NITRITE/Cytochromes-CL-Red/PROTON.72.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN \"NITRITE-REDUCTASE-CYTOCHROME-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITRITE__45__REDUCTASE__45__CYTOCHROME__45__RXN \"NITRITE-REDUCTASE-CYTOCHROME-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__OCTAPRENYLPHENOL__45__HYDROX__45__RXN \"2-OCTAPRENYLPHENOL-HYDROX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__OCTAPRENYLPHENOL__45__HYDROX__45__RXN \"2-OCTAPRENYLPHENOL-HYDROX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14291 \"RXN-14291\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14291 \"RXN-14291\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNQT__45__4171 \"RXNQT-4171\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNQT__45__4171 \"RXNQT-4171\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__5__46__1__46__39__45__RXN \"2.5.1.39-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__5__46__1__46__39__45__RXN \"2.5.1.39-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9230 \"RXN-9230\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9230 \"RXN-9230\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22709 \"RXN-22709\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22709 \"RXN-22709\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9222 \"RXN-9222\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9222 \"RXN-9222\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__5__46__1__46__88__45__RXN \"3.5.1.88-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__5__46__1__46__88__45__RXN \"3.5.1.88-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9003 \"RXN-9003\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9003 \"RXN-9003\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACSERLY__45__RXN \"ACSERLY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACSERLY__45__RXN \"ACSERLY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19329 \"RXN-19329\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19329 \"RXN-19329\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22610 \"RXN-22610\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22610 \"RXN-22610\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12726 \"RXN-12726\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12726 \"RXN-12726\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11028 \"RXN-11028\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11028 \"RXN-11028\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15946 \"RXN-15946\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15946 \"RXN-15946\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTAMINE__45____45__TRNA__45__LIGASE__45__RXN \"GLUTAMINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTAMINE__45____45__TRNA__45__LIGASE__45__RXN \"GLUTAMINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRIBFAICARPISOM__45__RXN \"PRIBFAICARPISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRIBFAICARPISOM__45__RXN \"PRIBFAICARPISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21601 \"RXN-21601\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21601 \"RXN-21601\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSNACMURPENTATRANS__45__RXN \"PHOSNACMURPENTATRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSNACMURPENTATRANS__45__RXN \"PHOSNACMURPENTATRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTCYCLOHYD__45__RXN \"HISTCYCLOHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTCYCLOHYD__45__RXN \"HISTCYCLOHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SUCCINATE__45__DEHYDROGENASE__45__UBIQUINONE__45__RXN \"SUCCINATE-DEHYDROGENASE-UBIQUINONE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SUCCINATE__45__DEHYDROGENASE__45__UBIQUINONE__45__RXN \"SUCCINATE-DEHYDROGENASE-UBIQUINONE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8975 \"RXN-8975\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8975 \"RXN-8975\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction IGPSYN__45__RXN \"IGPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction IGPSYN__45__RXN \"IGPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1YI0__45__14 \"RXN1YI0-14\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1YI0__45__14 \"RXN1YI0-14\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRTRANS__45__RXN \"PRTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRTRANS__45__RXN \"PRTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6259 \"RXN0-6259\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6259 \"RXN0-6259\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PEPTIDYLPROLYL__45__ISOMERASE__45__RXN \"PEPTIDYLPROLYL-ISOMERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PEPTIDYLPROLYL__45__ISOMERASE__45__RXN \"PEPTIDYLPROLYL-ISOMERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2TRANSKETO__45__RXN \"2TRANSKETO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2TRANSKETO__45__RXN \"2TRANSKETO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIOREDOXIN__45__RXN \"THIOREDOXIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIOREDOXIN__45__RXN \"THIOREDOXIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8675 \"RXN-8675\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8675 \"RXN-8675\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__ALPHA__45__GLUCOSE__46__40__46__ \"GLUCOSYLCERAMIDASE-RXN-CPD-25367/WATER//CPD66-41/ALPHA-GLUCOSE.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__ALPHA__45__GLUCOSE__46__40__46__ \"GLUCOSYLCERAMIDASE-RXN-CPD-25367/WATER//CPD66-41/ALPHA-GLUCOSE.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__GLC__46__30__46__ \"GLUCOSYLCERAMIDASE-RXN-CPD-25367/WATER//CPD66-41/GLC.30.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__GLC__46__30__46__ \"GLUCOSYLCERAMIDASE-RXN-CPD-25367/WATER//CPD66-41/GLC.30.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__Glucosyl__45__acyl__45__sphingosines__47__WATER__47____47__N__45__Acylsphingosine__47__ALPHA__45__GLUCOSE__46__66__46__ \"GLUCOSYLCERAMIDASE-RXN-Glucosyl-acyl-sphingosines/WATER//N-Acylsphingosine/ALPHA-GLUCOSE.66.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__Glucosyl__45__acyl__45__sphingosines__47__WATER__47____47__N__45__Acylsphingosine__47__ALPHA__45__GLUCOSE__46__66__46__ \"GLUCOSYLCERAMIDASE-RXN-Glucosyl-acyl-sphingosines/WATER//N-Acylsphingosine/ALPHA-GLUCOSE.66.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__Glucosyl__45__acyl__45__sphingosines__47__WATER__47____47__N__45__Acylsphingosine__47__GLC__46__56__46__ \"GLUCOSYLCERAMIDASE-RXN-Glucosyl-acyl-sphingosines/WATER//N-Acylsphingosine/GLC.56.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN__45__Glucosyl__45__acyl__45__sphingosines__47__WATER__47____47__N__45__Acylsphingosine__47__GLC__46__56__46__ \"GLUCOSYLCERAMIDASE-RXN-Glucosyl-acyl-sphingosines/WATER//N-Acylsphingosine/GLC.56.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN \"GLUCOSYLCERAMIDASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOSYLCERAMIDASE__45__RXN \"GLUCOSYLCERAMIDASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1TRANSKETO__45__RXN \"1TRANSKETO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1TRANSKETO__45__RXN \"1TRANSKETO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HEMEOSYN__45__RXN \"HEMEOSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HEMEOSYN__45__RXN \"HEMEOSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10002 \"RXN-10002\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10002 \"RXN-10002\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20645 \"RXN-20645\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20645 \"RXN-20645\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__128__45__RXN \"2.3.1.128-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__128__45__RXN \"2.3.1.128-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22897__45__CPD__45__25369__47__WATER__47____47__CPD66__45__40__47__ALPHA__45__D__45__GALACTOSE__46__44__46__ \"RXN-22897-CPD-25369/WATER//CPD66-40/ALPHA-D-GALACTOSE.44.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22897__45__CPD__45__25369__47__WATER__47____47__CPD66__45__40__47__ALPHA__45__D__45__GALACTOSE__46__44__46__ \"RXN-22897-CPD-25369/WATER//CPD66-40/ALPHA-D-GALACTOSE.44.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22897__45__CPD__45__25369__47__WATER__47____47__CPD66__45__40__47__GALACTOSE__46__36__46__ \"RXN-22897-CPD-25369/WATER//CPD66-40/GALACTOSE.36.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22897__45__CPD__45__25369__47__WATER__47____47__CPD66__45__40__47__GALACTOSE__46__36__46__ \"RXN-22897-CPD-25369/WATER//CPD66-40/GALACTOSE.36.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22897 \"RXN-22897\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22897 \"RXN-22897\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10001 \"RXN-10001\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10001 \"RXN-10001\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22 \"RXN-22\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22 \"RXN-22\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20065 \"RXN-20065\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20065 \"RXN-20065\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22895__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__ALPHA__45__GLUCOSE__46__40__46__ \"RXN-22895-CPD-25367/WATER//CPD66-41/ALPHA-GLUCOSE.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22895__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__ALPHA__45__GLUCOSE__46__40__46__ \"RXN-22895-CPD-25367/WATER//CPD66-41/ALPHA-GLUCOSE.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22895__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__GLC__46__30__46__ \"RXN-22895-CPD-25367/WATER//CPD66-41/GLC.30.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22895__45__CPD__45__25367__47__WATER__47____47__CPD66__45__41__47__GLC__46__30__46__ \"RXN-22895-CPD-25367/WATER//CPD66-41/GLC.30.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22895 \"RXN-22895\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22895 \"RXN-22895\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ARGSUCCINLYA__45__RXN \"ARGSUCCINLYA-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ARGSUCCINLYA__45__RXN \"ARGSUCCINLYA-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__2__46__2__46__24__45__RXN \"3.2.2.24-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__2__46__2__46__24__45__RXN \"3.2.2.24-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADDALT__45__RXN \"ADDALT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADDALT__45__RXN \"ADDALT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21524 \"RXN-21524\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21524 \"RXN-21524\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN0__45__277 \"TRANS-RXN0-277\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN0__45__277 \"TRANS-RXN0-277\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2OXOGLUTARATEDEH__45__RXN \"2OXOGLUTARATEDEH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2OXOGLUTARATEDEH__45__RXN \"2OXOGLUTARATEDEH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIAMINOPIMEPIM__45__RXN \"DIAMINOPIMEPIM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIAMINOPIMEPIM__45__RXN \"DIAMINOPIMEPIM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACONITATEHYDR__45__RXN \"ACONITATEHYDR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACONITATEHYDR__45__RXN \"ACONITATEHYDR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6515 \"RXN0-6515\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6515 \"RXN0-6515\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACONITATEDEHYDR__45__RXN \"ACONITATEDEHYDR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACONITATEDEHYDR__45__RXN \"ACONITATEDEHYDR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6726 \"RXN0-6726\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6726 \"RXN0-6726\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14047 \"RXN-14047\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14047 \"RXN-14047\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PANTEPADENYLYLTRAN__45__RXN \"PANTEPADENYLYLTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PANTEPADENYLYLTRAN__45__RXN \"PANTEPADENYLYLTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7271__45__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47__Reduced__45__NrdH__45__Proteins__47____47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47__Oxidized__45__NrdH__45__Proteins__46__117__46__ \"RXN0-7271-Oxidized-Disulfide-Carrier-Proteins/Reduced-NrdH-Proteins//Reduced-Disulfide-Carrier-Proteins/Oxidized-NrdH-Proteins.117.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7271__45__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47__Reduced__45__NrdH__45__Proteins__47____47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47__Oxidized__45__NrdH__45__Proteins__46__117__46__ \"RXN0-7271-Oxidized-Disulfide-Carrier-Proteins/Reduced-NrdH-Proteins//Reduced-Disulfide-Carrier-Proteins/Oxidized-NrdH-Proteins.117.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7271__45__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47__Red__45__Glutaredoxins__47____47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47__Ox__45__Glutaredoxins__46__107__46__ \"RXN0-7271-Oxidized-Disulfide-Carrier-Proteins/Red-Glutaredoxins//Reduced-Disulfide-Carrier-Proteins/Ox-Glutaredoxins.107.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7271__45__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47__Red__45__Glutaredoxins__47____47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47__Ox__45__Glutaredoxins__46__107__46__ \"RXN0-7271-Oxidized-Disulfide-Carrier-Proteins/Red-Glutaredoxins//Reduced-Disulfide-Carrier-Proteins/Ox-Glutaredoxins.107.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7271__45__Protein__45__Ox__45__Disulfides__47__Reduced__45__NrdH__45__Proteins__47____47__Protein__45__Red__45__Disulfides__47__Oxidized__45__NrdH__45__Proteins__46__91__46__ \"RXN0-7271-Protein-Ox-Disulfides/Reduced-NrdH-Proteins//Protein-Red-Disulfides/Oxidized-NrdH-Proteins.91.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7271__45__Protein__45__Ox__45__Disulfides__47__Reduced__45__NrdH__45__Proteins__47____47__Protein__45__Red__45__Disulfides__47__Oxidized__45__NrdH__45__Proteins__46__91__46__ \"RXN0-7271-Protein-Ox-Disulfides/Reduced-NrdH-Proteins//Protein-Red-Disulfides/Oxidized-NrdH-Proteins.91.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7271 \"RXN0-7271\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7271 \"RXN0-7271\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__A__45__RXN__45__GLUTATHIONE__47__Oxidized__45__NrdH__45__Proteins__47____47__OXIDIZED__45__GLUTATHIONE__47__Reduced__45__NrdH__45__Proteins__46__79__46__ \"PRODISULFREDUCT-A-RXN-GLUTATHIONE/Oxidized-NrdH-Proteins//OXIDIZED-GLUTATHIONE/Reduced-NrdH-Proteins.79.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__A__45__RXN__45__GLUTATHIONE__47__Oxidized__45__NrdH__45__Proteins__47____47__OXIDIZED__45__GLUTATHIONE__47__Reduced__45__NrdH__45__Proteins__46__79__46__ \"PRODISULFREDUCT-A-RXN-GLUTATHIONE/Oxidized-NrdH-Proteins//OXIDIZED-GLUTATHIONE/Reduced-NrdH-Proteins.79.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__A__45__RXN \"PRODISULFREDUCT-A-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__A__45__RXN \"PRODISULFREDUCT-A-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__RXN__45__GLUTATHIONE__47__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47____47__OXIDIZED__45__GLUTATHIONE__47__Reduced__45__Disulfide__45__Carrier__45__Proteins__46__105__46__ \"PRODISULFREDUCT-RXN-GLUTATHIONE/Oxidized-Disulfide-Carrier-Proteins//OXIDIZED-GLUTATHIONE/Reduced-Disulfide-Carrier-Proteins.105.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__RXN__45__GLUTATHIONE__47__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47____47__OXIDIZED__45__GLUTATHIONE__47__Reduced__45__Disulfide__45__Carrier__45__Proteins__46__105__46__ \"PRODISULFREDUCT-RXN-GLUTATHIONE/Oxidized-Disulfide-Carrier-Proteins//OXIDIZED-GLUTATHIONE/Reduced-Disulfide-Carrier-Proteins.105.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__RXN \"PRODISULFREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRODISULFREDUCT__45__RXN \"PRODISULFREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4__46__1__46__1__46__81__45__RXN \"4.1.1.81-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4__46__1__46__1__46__81__45__RXN \"4.1.1.81-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9384 \"RXN-9384\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9384 \"RXN-9384\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASNSYNB__45__RXN \"ASNSYNB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASNSYNB__45__RXN \"ASNSYNB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5055 \"RXN0-5055\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5055 \"RXN0-5055\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DAHPSYN__45__RXN \"DAHPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DAHPSYN__45__RXN \"DAHPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETOOHBUTREDUCTOISOM__45__RXN \"ACETOOHBUTREDUCTOISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETOOHBUTREDUCTOISOM__45__RXN \"ACETOOHBUTREDUCTOISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MANNPISOM__45__RXN \"MANNPISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MANNPISOM__45__RXN \"MANNPISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UNDECAPRENYL__45__DIPHOSPHATASE__45__RXN \"UNDECAPRENYL-DIPHOSPHATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UNDECAPRENYL__45__DIPHOSPHATASE__45__RXN \"UNDECAPRENYL-DIPHOSPHATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11791 \"RXN-11791\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11791 \"RXN-11791\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction F16BDEPHOS__45__RXN \"F16BDEPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction F16BDEPHOS__45__RXN \"F16BDEPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRIMSYN3__45__RXN \"PYRIMSYN3-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRIMSYN3__45__RXN \"PYRIMSYN3-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIOREDOXIN__45__REDUCT__45__NADPH__45__RXN \"THIOREDOXIN-REDUCT-NADPH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIOREDOXIN__45__REDUCT__45__NADPH__45__RXN \"THIOREDOXIN-REDUCT-NADPH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10642 \"RXN-10642\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10642 \"RXN-10642\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15376 \"RXN-15376\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15376 \"RXN-15376\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction OHMETPYRKIN__45__RXN \"OHMETPYRKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction OHMETPYRKIN__45__RXN \"OHMETPYRKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UROGENDECARBOX__45__RXN \"UROGENDECARBOX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UROGENDECARBOX__45__RXN \"UROGENDECARBOX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SUPEROX__45__DISMUT__45__RXN \"SUPEROX-DISMUT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SUPEROX__45__DISMUT__45__RXN \"SUPEROX-DISMUT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4__45__NITROPHENYLPHOSPHATASE__45__RXN \"4-NITROPHENYLPHOSPHATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4__45__NITROPHENYLPHOSPHATASE__45__RXN \"4-NITROPHENYLPHOSPHATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2PGADEHYDRAT__45__RXN \"2PGADEHYDRAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2PGADEHYDRAT__45__RXN \"2PGADEHYDRAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20674 \"RXN-20674\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20674 \"RXN-20674\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18319 \"RXN-18319\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18319 \"RXN-18319\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20673 \"RXN-20673\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20673 \"RXN-20673\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11480 \"RXN-11480\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11480 \"RXN-11480\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__DEHYDROQUINATE__45__SYNTHASE__45__RXN \"3-DEHYDROQUINATE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__DEHYDROQUINATE__45__SYNTHASE__45__RXN \"3-DEHYDROQUINATE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction OROTPDECARB__45__RXN \"OROTPDECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction OROTPDECARB__45__RXN \"OROTPDECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTKIN__45__RXN \"GLUTKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTKIN__45__RXN \"GLUTKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18446 \"RXN-18446\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18446 \"RXN-18446\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9999 \"RXN-9999\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9999 \"RXN-9999\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20165 \"RXN-20165\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20165 \"RXN-20165\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18447 \"RXN-18447\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18447 \"RXN-18447\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETATE__45____45__COA__45__LIGASE__45__RXN \"ACETATE--COA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETATE__45____45__COA__45__LIGASE__45__RXN \"ACETATE--COA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5510 \"RXN0-5510\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5510 \"RXN0-5510\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20667 \"RXN-20667\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20667 \"RXN-20667\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1863 \"RXN0-1863\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1863 \"RXN0-1863\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20666 \"RXN-20666\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20666 \"RXN-20666\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11347 \"RXN-11347\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11347 \"RXN-11347\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9998 \"RXN-9998\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9998 \"RXN-9998\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10994 \"RXN-10994\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10994 \"RXN-10994\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUC1PURIDYLTRANS__45__RXN \"GLUC1PURIDYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUC1PURIDYLTRANS__45__RXN \"GLUC1PURIDYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMYLOMALT__45__RXN \"AMYLOMALT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMYLOMALT__45__RXN \"AMYLOMALT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMYLOMALT__45__RXN__45__MALTOTETRAOSE__47__GLC__47____47__MALTOTRIOSE__47__MALTOSE__46__39__46__ \"AMYLOMALT-RXN-MALTOTETRAOSE/GLC//MALTOTRIOSE/MALTOSE.39.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMYLOMALT__45__RXN__45__MALTOTETRAOSE__47__GLC__47____47__MALTOTRIOSE__47__MALTOSE__46__39__46__ \"AMYLOMALT-RXN-MALTOTETRAOSE/GLC//MALTOTRIOSE/MALTOSE.39.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMYLOMALT__45__RXN__45__MALTOTETRAOSE__47__ALPHA__45__GLUCOSE__47____47__MALTOTRIOSE__47__MALTOSE__46__49__46__ \"AMYLOMALT-RXN-MALTOTETRAOSE/ALPHA-GLUCOSE//MALTOTRIOSE/MALTOSE.49.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMYLOMALT__45__RXN__45__MALTOTETRAOSE__47__ALPHA__45__GLUCOSE__47____47__MALTOTRIOSE__47__MALTOSE__46__49__46__ \"AMYLOMALT-RXN-MALTOTETRAOSE/ALPHA-GLUCOSE//MALTOTRIOSE/MALTOSE.49.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19324 \"RXN-19324\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19324 \"RXN-19324\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTAMATE__45__N__45__ACETYLTRANSFERASE__45__RXN \"GLUTAMATE-N-ACETYLTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTAMATE__45__N__45__ACETYLTRANSFERASE__45__RXN \"GLUTAMATE-N-ACETYLTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__78__45__RXN \"2.3.1.78-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__78__45__RXN \"2.3.1.78-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7115 \"RXN0-7115\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7115 \"RXN0-7115\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11564 \"RXN-11564\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11564 \"RXN-11564\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__4301 \"RXN0-4301\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__4301 \"RXN0-4301\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AIRCARBOXY__45__RXN \"AIRCARBOXY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AIRCARBOXY__45__RXN \"AIRCARBOXY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9380 \"RXN-9380\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9380 \"RXN-9380\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12752 \"RXN-12752\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12752 \"RXN-12752\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GTP__45__CYCLOHYDRO__45__II__45__RXN \"GTP-CYCLOHYDRO-II-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GTP__45__CYCLOHYDRO__45__II__45__RXN \"GTP-CYCLOHYDRO-II-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13142 \"RXN-13142\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13142 \"RXN-13142\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15889 \"RXN-15889\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15889 \"RXN-15889\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5292 \"RXN0-5292\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5292 \"RXN0-5292\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HOMOSERKIN__45__RXN \"HOMOSERKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HOMOSERKIN__45__RXN \"HOMOSERKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11832 \"RXN-11832\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11832 \"RXN-11832\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13141 \"RXN-13141\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13141 \"RXN-13141\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18511 \"RXN-18511\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18511 \"RXN-18511\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6727 \"RXN0-6727\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6727 \"RXN0-6727\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14143 \"RXN-14143\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14143 \"RXN-14143\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11476 \"RXN-11476\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11476 \"RXN-11476\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FADSYN__45__RXN \"FADSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FADSYN__45__RXN \"FADSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19326 \"RXN-19326\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19326 \"RXN-19326\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__5__46__1__46__19__45__RXN \"2.5.1.19-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__5__46__1__46__19__45__RXN \"2.5.1.19-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ABC__45__19__45__RXN \"ABC-19-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ABC__45__19__45__RXN \"ABC-19-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADPREDUCT__45__RXN \"ADPREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADPREDUCT__45__RXN \"ADPREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10035 \"RXN-10035\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10035 \"RXN-10035\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PEPCARBOXYKIN__45__RXN \"PEPCARBOXYKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PEPCARBOXYKIN__45__RXN \"PEPCARBOXYKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPREDUCT__45__RXN \"GDPREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPREDUCT__45__RXN \"GDPREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTAMINOTRANS__45__RXN \"HISTAMINOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTAMINOTRANS__45__RXN \"HISTAMINOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7080 \"RXN0-7080\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7080 \"RXN0-7080\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14142 \"RXN-14142\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14142 \"RXN-14142\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15511 \"RXN-15511\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15511 \"RXN-15511\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15510 \"RXN-15510\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15510 \"RXN-15510\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15513 \"RXN-15513\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15513 \"RXN-15513\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPREDUCT__45__RXN \"UDPREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPREDUCT__45__RXN \"UDPREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PREPHENATEDEHYDRAT__45__RXN \"PREPHENATEDEHYDRAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PREPHENATEDEHYDRAT__45__RXN \"PREPHENATEDEHYDRAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THYMIDYLATE__45__5__45__PHOSPHATASE__45__RXN \"THYMIDYLATE-5-PHOSPHATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THYMIDYLATE__45__5__45__PHOSPHATASE__45__RXN \"THYMIDYLATE-5-PHOSPHATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7913 \"RXN-7913\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7913 \"RXN-7913\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9772 \"RXN-9772\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9772 \"RXN-9772\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__2__46__1__46__9__45__RXN \"1.2.1.9-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__2__46__1__46__9__45__RXN \"1.2.1.9-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction METHIONYL__45__TRNA__45__FORMYLTRANSFERASE__45__RXN \"METHIONYL-TRNA-FORMYLTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction METHIONYL__45__TRNA__45__FORMYLTRANSFERASE__45__RXN \"METHIONYL-TRNA-FORMYLTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _6PFRUCTPHOS__45__RXN \"6PFRUCTPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _6PFRUCTPHOS__45__RXN \"6PFRUCTPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14161 \"RXN-14161\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14161 \"RXN-14161\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12197 \"RXN-12197\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12197 \"RXN-12197\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12196 \"RXN-12196\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12196 \"RXN-12196\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12198 \"RXN-12198\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12198 \"RXN-12198\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PSERTRANSAMPYR__45__RXN \"PSERTRANSAMPYR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PSERTRANSAMPYR__45__RXN \"PSERTRANSAMPYR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8748 \"RXN-8748\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8748 \"RXN-8748\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PSERTRANSAM__45__RXN \"PSERTRANSAM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PSERTRANSAM__45__RXN \"PSERTRANSAM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FGAMSYN__45__RXN \"FGAMSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FGAMSYN__45__RXN \"FGAMSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18229 \"RXN-18229\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18229 \"RXN-18229\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CHORISMATE__45__SYNTHASE__45__RXN \"CHORISMATE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CHORISMATE__45__SYNTHASE__45__RXN \"CHORISMATE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__6__46__1__46__41__45__RXN \"3.6.1.41-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__6__46__1__46__41__45__RXN \"3.6.1.41-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18584 \"RXN-18584\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18584 \"RXN-18584\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8764 \"RXN-8764\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8764 \"RXN-8764\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15829 \"RXN-15829\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15829 \"RXN-15829\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDRODIPICSYN__45__RXN \"DIHYDRODIPICSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDRODIPICSYN__45__RXN \"DIHYDRODIPICSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14107 \"RXN-14107\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14107 \"RXN-14107\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SAMDECARB__45__RXN \"SAMDECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SAMDECARB__45__RXN \"SAMDECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UREASE__45__RXN \"UREASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UREASE__45__RXN \"UREASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15816 \"RXN-15816\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15816 \"RXN-15816\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18604 \"RXN-18604\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18604 \"RXN-18604\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction L__45__ASPARTATE__45__OXID__45__RXN \"L-ASPARTATE-OXID-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction L__45__ASPARTATE__45__OXID__45__RXN \"L-ASPARTATE-OXID-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THRESYN__45__RXN \"THRESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THRESYN__45__RXN \"THRESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PEROXID__45__RXN \"PEROXID-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PEROXID__45__RXN \"PEROXID-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7716 \"RXN-7716\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7716 \"RXN-7716\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6377__45__AMMONIUM__47__NADP__47__WATER__47____47__NITRITE__47__NADPH__47__PROTON__46__42__46__ \"RXN0-6377-AMMONIUM/NADP/WATER//NITRITE/NADPH/PROTON.42.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6377__45__AMMONIUM__47__NADP__47__WATER__47____47__NITRITE__47__NADPH__47__PROTON__46__42__46__ \"RXN0-6377-AMMONIUM/NADP/WATER//NITRITE/NADPH/PROTON.42.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6377__45__AMMONIUM__47__NAD__47__WATER__47____47__NITRITE__47__NADH__47__PROTON__46__40__46__ \"RXN0-6377-AMMONIUM/NAD/WATER//NITRITE/NADH/PROTON.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6377__45__AMMONIUM__47__NAD__47__WATER__47____47__NITRITE__47__NADH__47__PROTON__46__40__46__ \"RXN0-6377-AMMONIUM/NAD/WATER//NITRITE/NADH/PROTON.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMPSYN__45__RXN \"AMPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMPSYN__45__RXN \"AMPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22606 \"RXN-22606\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22606 \"RXN-22606\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16804 \"RXN-16804\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16804 \"RXN-16804\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DARAB5PISOM__45__RXN \"DARAB5PISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DARAB5PISOM__45__RXN \"DARAB5PISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__8__46__1__46__4__45__RXN \"1.8.1.4-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__8__46__1__46__4__45__RXN \"1.8.1.4-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction KDO__45__8PPHOSPHAT__45__RXN \"KDO-8PPHOSPHAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction KDO__45__8PPHOSPHAT__45__RXN \"KDO-8PPHOSPHAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction METHYLENETHFDEHYDROG__45__NADP__45__RXN \"METHYLENETHFDEHYDROG-NADP-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction METHYLENETHFDEHYDROG__45__NADP__45__RXN \"METHYLENETHFDEHYDROG-NADP-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8629 \"RXN-8629\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8629 \"RXN-8629\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18797 \"RXN-18797\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18797 \"RXN-18797\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction METHENYLTHFCYCLOHYDRO__45__RXN \"METHENYLTHFCYCLOHYDRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction METHENYLTHFCYCLOHYDRO__45__RXN \"METHENYLTHFCYCLOHYDRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7719 \"RXN-7719\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7719 \"RXN-7719\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1132 \"RXN0-1132\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1132 \"RXN0-1132\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22438 \"RXN-22438\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22438 \"RXN-22438\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__52 \"RXN2B4Q-52\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__52 \"RXN2B4Q-52\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ABC__45__70__45__RXN \"ABC-70-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ABC__45__70__45__RXN \"ABC-70-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18796 \"RXN-18796\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18796 \"RXN-18796\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction L__45__GLN__45__FRUCT__45__6__45__P__45__AMINOTRANS__45__RXN \"L-GLN-FRUCT-6-P-AMINOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction L__45__GLN__45__FRUCT__45__6__45__P__45__AMINOTRANS__45__RXN \"L-GLN-FRUCT-6-P-AMINOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTRACE__45__RXN \"GLUTRACE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTRACE__45__RXN \"GLUTRACE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11952 \"RXN-11952\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11952 \"RXN-11952\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11951 \"RXN-11951\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11951 \"RXN-11951\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15123 \"RXN-15123\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15123 \"RXN-15123\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19310 \"RXN-19310\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19310 \"RXN-19310\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ARYLSULFAT__45__RXN \"ARYLSULFAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ARYLSULFAT__45__RXN \"ARYLSULFAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ABC__45__7__45__RXN \"ABC-7-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ABC__45__7__45__RXN \"ABC-7-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15121 \"RXN-15121\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15121 \"RXN-15121\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDLIPACETRANS__45__RXN \"DIHYDLIPACETRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDLIPACETRANS__45__RXN \"DIHYDLIPACETRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THREDEHYD__45__RXN \"THREDEHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THREDEHYD__45__RXN \"THREDEHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NAG1P__45__URIDYLTRANS__45__RXN \"NAG1P-URIDYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NAG1P__45__URIDYLTRANS__45__RXN \"NAG1P-URIDYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__157__45__RXN \"2.3.1.157-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__157__45__RXN \"2.3.1.157-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16765 \"RXN-16765\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16765 \"RXN-16765\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIB5PISOM__45__RXN \"RIB5PISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIB5PISOM__45__RXN \"RIB5PISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FPPSYN__45__RXN \"FPPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FPPSYN__45__RXN \"FPPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R524__45__RXN \"R524-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R524__45__RXN \"R524-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRNUTRANSHYDROGEN__45__RXN \"PYRNUTRANSHYDROGEN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRNUTRANSHYDROGEN__45__RXN \"PYRNUTRANSHYDROGEN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYCDEH__45__RXN \"GLYCDEH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYCDEH__45__RXN \"GLYCDEH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NQOR__45__RXN \"NQOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NQOR__45__RXN \"NQOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22497 \"RXN-22497\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22497 \"RXN-22497\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTALDEHYD__45__RXN \"HISTALDEHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTALDEHYD__45__RXN \"HISTALDEHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1281 \"RXN0-1281\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1281 \"RXN0-1281\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2OXOGLUTDECARB__45__RXN \"2OXOGLUTDECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2OXOGLUTDECARB__45__RXN \"2OXOGLUTDECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C16__45__H32__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C16__45__H32__47__NADPH__47__PROTON__46__88__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C16-H32/NADP//POLYMER-INST-B-KETOACYL-ACP-C16-H32/NADPH/PROTON.88.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C16__45__H32__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C16__45__H32__47__NADPH__47__PROTON__46__88__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C16-H32/NADP//POLYMER-INST-B-KETOACYL-ACP-C16-H32/NADPH/PROTON.88.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__R__45__3__45__hydroxystearoyl__45__ACPs__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C14__45__H28__47__NADPH__47__PROTON__46__80__46__ \"3-OXOACYL-ACP-REDUCT-RXN-R-3-hydroxystearoyl-ACPs/NADP//POLYMER-INST-B-KETOACYL-ACP-C14-H28/NADPH/PROTON.80.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__R__45__3__45__hydroxystearoyl__45__ACPs__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C14__45__H28__47__NADPH__47__PROTON__46__80__46__ \"3-OXOACYL-ACP-REDUCT-RXN-R-3-hydroxystearoyl-ACPs/NADP//POLYMER-INST-B-KETOACYL-ACP-C14-H28/NADPH/PROTON.80.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__R__45__3__45__HYDROXYPALMITOYL__45__ACP__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C12__45__H24__47__NADPH__47__PROTON__46__80__46__ \"3-OXOACYL-ACP-REDUCT-RXN-R-3-HYDROXYPALMITOYL-ACP/NADP//POLYMER-INST-B-KETOACYL-ACP-C12-H24/NADPH/PROTON.80.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__R__45__3__45__HYDROXYPALMITOYL__45__ACP__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C12__45__H24__47__NADPH__47__PROTON__46__80__46__ \"3-OXOACYL-ACP-REDUCT-RXN-R-3-HYDROXYPALMITOYL-ACP/NADP//POLYMER-INST-B-KETOACYL-ACP-C12-H24/NADPH/PROTON.80.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__R__45__3__45__hydroxymyristoyl__45__ACPs__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C10__45__H20__47__NADPH__47__PROTON__46__81__46__ \"3-OXOACYL-ACP-REDUCT-RXN-R-3-hydroxymyristoyl-ACPs/NADP//POLYMER-INST-B-KETOACYL-ACP-C10-H20/NADPH/PROTON.81.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__R__45__3__45__hydroxymyristoyl__45__ACPs__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C10__45__H20__47__NADPH__47__PROTON__46__81__46__ \"3-OXOACYL-ACP-REDUCT-RXN-R-3-hydroxymyristoyl-ACPs/NADP//POLYMER-INST-B-KETOACYL-ACP-C10-H20/NADPH/PROTON.81.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C8__45__H16__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C8__45__H16__47__NADPH__47__PROTON__46__86__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C8-H16/NADP//POLYMER-INST-B-KETOACYL-ACP-C8-H16/NADPH/PROTON.86.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C8__45__H16__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C8__45__H16__47__NADPH__47__PROTON__46__86__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C8-H16/NADP//POLYMER-INST-B-KETOACYL-ACP-C8-H16/NADPH/PROTON.86.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__Beta__45__hydroxydecanoyl__45__ACPs__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C6__45__H12__47__NADPH__47__PROTON__46__80__46__ \"3-OXOACYL-ACP-REDUCT-RXN-Beta-hydroxydecanoyl-ACPs/NADP//POLYMER-INST-B-KETOACYL-ACP-C6-H12/NADPH/PROTON.80.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__Beta__45__hydroxydecanoyl__45__ACPs__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C6__45__H12__47__NADPH__47__PROTON__46__80__46__ \"3-OXOACYL-ACP-REDUCT-RXN-Beta-hydroxydecanoyl-ACPs/NADP//POLYMER-INST-B-KETOACYL-ACP-C6-H12/NADPH/PROTON.80.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C4__45__H8__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C4__45__H8__47__NADPH__47__PROTON__46__84__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C4-H8/NADP//POLYMER-INST-B-KETOACYL-ACP-C4-H8/NADPH/PROTON.84.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C4__45__H8__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C4__45__H8__47__NADPH__47__PROTON__46__84__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C4-H8/NADP//POLYMER-INST-B-KETOACYL-ACP-C4-H8/NADPH/PROTON.84.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C2__45__H4__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C2__45__H4__47__NADPH__47__PROTON__46__84__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C2-H4/NADP//POLYMER-INST-B-KETOACYL-ACP-C2-H4/NADPH/PROTON.84.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C2__45__H4__47__NADP__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C2__45__H4__47__NADPH__47__PROTON__46__84__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C2-H4/NADP//POLYMER-INST-B-KETOACYL-ACP-C2-H4/NADPH/PROTON.84.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C0__45__H0__47__NADP__47____47__Acetoacetyl__45__ACPs__47__NADPH__47__PROTON__46__67__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C0-H0/NADP//Acetoacetyl-ACPs/NADPH/PROTON.67.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__REDUCT__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C0__45__H0__47__NADP__47____47__Acetoacetyl__45__ACPs__47__NADPH__47__PROTON__46__67__46__ \"3-OXOACYL-ACP-REDUCT-RXN-POLYMER-INST-OH-ACYL-ACP-C0-H0/NADP//Acetoacetyl-ACPs/NADPH/PROTON.67.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1147 \"RXN0-1147\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1147 \"RXN0-1147\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTOLDEHYD__45__RXN \"HISTOLDEHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTOLDEHYD__45__RXN \"HISTOLDEHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5052 \"RXN0-5052\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5052 \"RXN0-5052\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8001 \"RXN-8001\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8001 \"RXN-8001\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21027 \"RXN-21027\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21027 \"RXN-21027\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21026 \"RXN-21026\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21026 \"RXN-21026\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ATPPHOSPHORIBOSYLTRANS__45__RXN \"ATPPHOSPHORIBOSYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ATPPHOSPHORIBOSYLTRANS__45__RXN \"ATPPHOSPHORIBOSYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21024 \"RXN-21024\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21024 \"RXN-21024\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THI__45__P__45__KIN__45__RXN \"THI-P-KIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THI__45__P__45__KIN__45__RXN \"THI-P-KIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__982 \"RXN-982\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__982 \"RXN-982\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16691 \"RXN-16691\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16691 \"RXN-16691\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction IMP__45__DEHYDROG__45__RXN \"IMP-DEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction IMP__45__DEHYDROG__45__RXN \"IMP-DEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16687 \"RXN-16687\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16687 \"RXN-16687\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPNACETYLGLUCOSAMENOLPYRTRANS__45__RXN \"UDPNACETYLGLUCOSAMENOLPYRTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPNACETYLGLUCOSAMENOLPYRTRANS__45__RXN \"UDPNACETYLGLUCOSAMENOLPYRTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GMP__45__SYN__45__NH3__45__RXN \"GMP-SYN-NH3-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GMP__45__SYN__45__NH3__45__RXN \"GMP-SYN-NH3-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GMP__45__SYN__45__GLUT__45__RXN \"GMP-SYN-GLUT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GMP__45__SYN__45__GLUT__45__RXN \"GMP-SYN-GLUT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16690 \"RXN-16690\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16690 \"RXN-16690\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19334 \"RXN-19334\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19334 \"RXN-19334\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LUMAZINESYN__45__RXN \"LUMAZINESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LUMAZINESYN__45__RXN \"LUMAZINESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16689 \"RXN-16689\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16689 \"RXN-16689\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIOHBUTANONEPSYN__45__RXN \"DIOHBUTANONEPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIOHBUTANONEPSYN__45__RXN \"DIOHBUTANONEPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRYPSYN__45__RXN \"TRYPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRYPSYN__45__RXN \"TRYPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIOPURINE__45__S__45__METHYLTRANSFERASE__45__RXN \"THIOPURINE-S-METHYLTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIOPURINE__45__S__45__METHYLTRANSFERASE__45__RXN \"THIOPURINE-S-METHYLTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16688 \"RXN-16688\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16688 \"RXN-16688\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTSEMIALDEHYDROG__45__RXN \"GLUTSEMIALDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTSEMIALDEHYDROG__45__RXN \"GLUTSEMIALDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16685 \"RXN-16685\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16685 \"RXN-16685\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction OXALODECARB__45__RXN \"OXALODECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction OXALODECARB__45__RXN \"OXALODECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DISULFOXRED__45__RXN__45__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47____47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47__Oxidized__45__Disulfide__45__Carrier__45__Proteins__46__143__46__ \"DISULFOXRED-RXN-Oxidized-Disulfide-Carrier-Proteins/Reduced-Disulfide-Carrier-Proteins//Reduced-Disulfide-Carrier-Proteins/Oxidized-Disulfide-Carrier-Proteins.143.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DISULFOXRED__45__RXN__45__Oxidized__45__Disulfide__45__Carrier__45__Proteins__47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47____47__Reduced__45__Disulfide__45__Carrier__45__Proteins__47__Oxidized__45__Disulfide__45__Carrier__45__Proteins__46__143__46__ \"DISULFOXRED-RXN-Oxidized-Disulfide-Carrier-Proteins/Reduced-Disulfide-Carrier-Proteins//Reduced-Disulfide-Carrier-Proteins/Oxidized-Disulfide-Carrier-Proteins.143.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DISULFOXRED__45__RXN \"DISULFOXRED-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DISULFOXRED__45__RXN \"DISULFOXRED-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MALATE__45__DEH__45__RXN \"MALATE-DEH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MALATE__45__DEH__45__RXN \"MALATE-DEH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16664 \"RXN-16664\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16664 \"RXN-16664\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15881 \"RXN-15881\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15881 \"RXN-15881\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10058 \"RXN-10058\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10058 \"RXN-10058\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__1__46__1__46__39__45__RXN \"1.1.1.39-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__1__46__1__46__39__45__RXN \"1.1.1.39-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBOFLAVINSYNDEAM__45__RXN \"RIBOFLAVINSYNDEAM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBOFLAVINSYNDEAM__45__RXN \"RIBOFLAVINSYNDEAM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CHORISMATEMUT__45__RXN \"CHORISMATEMUT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CHORISMATEMUT__45__RXN \"CHORISMATEMUT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13329 \"RXN-13329\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13329 \"RXN-13329\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5330 \"RXN0-5330\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5330 \"RXN0-5330\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MALIC__45__NADP__45__RXN \"MALIC-NADP-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MALIC__45__NADP__45__RXN \"MALIC-NADP-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRPPAMIDOTRANS__45__RXN \"PRPPAMIDOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRPPAMIDOTRANS__45__RXN \"PRPPAMIDOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12303 \"RXN-12303\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12303 \"RXN-12303\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21522 \"RXN-21522\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21522 \"RXN-21522\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBOFLAVINSYNREDUC__45__RXN \"RIBOFLAVINSYNREDUC-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBOFLAVINSYNREDUC__45__RXN \"RIBOFLAVINSYNREDUC-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21025 \"RXN-21025\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21025 \"RXN-21025\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN185E__45__16 \"RXN185E-16\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN185E__45__16 \"RXN185E-16\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10674 \"RXN-10674\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10674 \"RXN-10674\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C0__45__H0__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C0__45__H0__47__WATER__46__76__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C0-H0//POLYMER-INST-TRANS-D2-ENOYL-ACP-C0-H0/WATER.76.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C0__45__H0__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C0__45__H0__47__WATER__46__76__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C0-H0//POLYMER-INST-TRANS-D2-ENOYL-ACP-C0-H0/WATER.76.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C2__45__H4__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C2__45__H4__47__WATER__46__76__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C2-H4//POLYMER-INST-TRANS-D2-ENOYL-ACP-C2-H4/WATER.76.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C2__45__H4__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C2__45__H4__47__WATER__46__76__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C2-H4//POLYMER-INST-TRANS-D2-ENOYL-ACP-C2-H4/WATER.76.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C4__45__H8__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C4__45__H8__47__WATER__46__76__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C4-H8//POLYMER-INST-TRANS-D2-ENOYL-ACP-C4-H8/WATER.76.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C4__45__H8__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C4__45__H8__47__WATER__46__76__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C4-H8//POLYMER-INST-TRANS-D2-ENOYL-ACP-C4-H8/WATER.76.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__Beta__45__hydroxydecanoyl__45__ACPs__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C6__45__H12__47__WATER__46__72__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-Beta-hydroxydecanoyl-ACPs//POLYMER-INST-TRANS-D2-ENOYL-ACP-C6-H12/WATER.72.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__Beta__45__hydroxydecanoyl__45__ACPs__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C6__45__H12__47__WATER__46__72__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-Beta-hydroxydecanoyl-ACPs//POLYMER-INST-TRANS-D2-ENOYL-ACP-C6-H12/WATER.72.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C8__45__H16__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C8__45__H16__47__WATER__46__78__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C8-H16//POLYMER-INST-TRANS-D2-ENOYL-ACP-C8-H16/WATER.78.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C8__45__H16__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C8__45__H16__47__WATER__46__78__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C8-H16//POLYMER-INST-TRANS-D2-ENOYL-ACP-C8-H16/WATER.78.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__R__45__3__45__hydroxymyristoyl__45__ACPs__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C10__45__H20__47__WATER__46__73__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-R-3-hydroxymyristoyl-ACPs//POLYMER-INST-TRANS-D2-ENOYL-ACP-C10-H20/WATER.73.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__R__45__3__45__hydroxymyristoyl__45__ACPs__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C10__45__H20__47__WATER__46__73__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-R-3-hydroxymyristoyl-ACPs//POLYMER-INST-TRANS-D2-ENOYL-ACP-C10-H20/WATER.73.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__R__45__3__45__HYDROXYPALMITOYL__45__ACP__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C12__45__H24__47__WATER__46__72__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-R-3-HYDROXYPALMITOYL-ACP//POLYMER-INST-TRANS-D2-ENOYL-ACP-C12-H24/WATER.72.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__R__45__3__45__HYDROXYPALMITOYL__45__ACP__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C12__45__H24__47__WATER__46__72__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-R-3-HYDROXYPALMITOYL-ACP//POLYMER-INST-TRANS-D2-ENOYL-ACP-C12-H24/WATER.72.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__R__45__3__45__hydroxystearoyl__45__ACPs__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C14__45__H28__47__WATER__46__72__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-R-3-hydroxystearoyl-ACPs//POLYMER-INST-TRANS-D2-ENOYL-ACP-C14-H28/WATER.72.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__R__45__3__45__hydroxystearoyl__45__ACPs__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C14__45__H28__47__WATER__46__72__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-R-3-hydroxystearoyl-ACPs//POLYMER-INST-TRANS-D2-ENOYL-ACP-C14-H28/WATER.72.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C16__45__H32__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C16__45__H32__47__WATER__46__80__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C16-H32//POLYMER-INST-TRANS-D2-ENOYL-ACP-C16-H32/WATER.80.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__HYDROXYDECANOYL__45__ACP__45__DEHYDR__45__RXN__45__POLYMER__45__INST__45__OH__45__ACYL__45__ACP__45__C16__45__H32__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C16__45__H32__47__WATER__46__80__46__ \"3-HYDROXYDECANOYL-ACP-DEHYDR-RXN-POLYMER-INST-OH-ACYL-ACP-C16-H32//POLYMER-INST-TRANS-D2-ENOYL-ACP-C16-H32/WATER.80.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20400 \"RXN-20400\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20400 \"RXN-20400\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MALONYL__45__COA__45__ACP__45__TRANSACYL__45__RXN \"MALONYL-COA-ACP-TRANSACYL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MALONYL__45__COA__45__ACP__45__TRANSACYL__45__RXN \"MALONYL-COA-ACP-TRANSACYL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20498 \"RXN-20498\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20498 \"RXN-20498\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14569 \"RXN-14569\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14569 \"RXN-14569\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20481 \"RXN-20481\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20481 \"RXN-20481\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9590 \"RXN-9590\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9590 \"RXN-9590\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRAISOM__45__RXN \"PRAISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRAISOM__45__RXN \"PRAISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CPM__45__KDOSYNTH__45__RXN \"CPM-KDOSYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CPM__45__KDOSYNTH__45__RXN \"CPM-KDOSYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19366 \"RXN-19366\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19366 \"RXN-19366\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10814 \"RXN-10814\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10814 \"RXN-10814\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17025 \"RXN-17025\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17025 \"RXN-17025\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1YI0__45__16 \"RXN1YI0-16\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1YI0__45__16 \"RXN1YI0-16\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16427 \"RXN-16427\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16427 \"RXN-16427\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19364 \"RXN-19364\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19364 \"RXN-19364\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12456 \"RXN-12456\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12456 \"RXN-12456\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction COBALADENOSYLTRANS__45__RXN \"COBALADENOSYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction COBALADENOSYLTRANS__45__RXN \"COBALADENOSYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__262 \"RXN0-262\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__262 \"RXN0-262\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19349 \"RXN-19349\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19349 \"RXN-19349\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11319 \"RXN-11319\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11319 \"RXN-11319\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21295 \"RXN-21295\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21295 \"RXN-21295\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PGLUCISOM__45__RXN \"PGLUCISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PGLUCISOM__45__RXN \"PGLUCISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__53 \"RXN2B4Q-53\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__53 \"RXN2B4Q-53\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction QUEUOSINE__45__TRNA__45__RIBOSYLTRANSFERASE__45__RXN \"QUEUOSINE-TRNA-RIBOSYLTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction QUEUOSINE__45__TRNA__45__RIBOSYLTRANSFERASE__45__RXN \"QUEUOSINE-TRNA-RIBOSYLTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TETRAACYLDISACC4KIN__45__RXN \"TETRAACYLDISACC4KIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TETRAACYLDISACC4KIN__45__RXN \"TETRAACYLDISACC4KIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9788 \"RXN-9788\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9788 \"RXN-9788\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2G6Z__45__1 \"RXN2G6Z-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2G6Z__45__1 \"RXN2G6Z-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7386__45__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__NADH__47__PROTON__47____47__CPD__45__12831__47__NAD__46__63__46__ \"RXN0-7386-2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/NADH/PROTON//CPD-12831/NAD.63.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7386__45__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__NADH__47__PROTON__47____47__CPD__45__12831__47__NAD__46__63__46__ \"RXN0-7386-2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/NADH/PROTON//CPD-12831/NAD.63.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7386__45__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__NADPH__47__PROTON__47____47__CPD__45__12831__47__NADP__46__65__46__ \"RXN0-7386-2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/NADPH/PROTON//CPD-12831/NADP.65.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7386__45__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__NADPH__47__PROTON__47____47__CPD__45__12831__47__NADP__46__65__46__ \"RXN0-7386-2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/NADPH/PROTON//CPD-12831/NADP.65.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7386__45__CPD__45__9720__47__NADH__47__PROTON__47____47__CPD__45__12126__47__NAD__46__36__46__ \"RXN0-7386-CPD-9720/NADH/PROTON//CPD-12126/NAD.36.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7386__45__CPD__45__9720__47__NADH__47__PROTON__47____47__CPD__45__12126__47__NAD__46__36__46__ \"RXN0-7386-CPD-9720/NADH/PROTON//CPD-12126/NAD.36.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7386__45__CPD__45__9720__47__NADPH__47__PROTON__47____47__CPD__45__12126__47__NADP__46__38__46__ \"RXN0-7386-CPD-9720/NADPH/PROTON//CPD-12126/NADP.38.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7386__45__CPD__45__9720__47__NADPH__47__PROTON__47____47__CPD__45__12126__47__NADP__46__38__46__ \"RXN0-7386-CPD-9720/NADPH/PROTON//CPD-12126/NADP.38.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9789 \"RXN-9789\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9789 \"RXN-9789\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction VALINE__45____45__TRNA__45__LIGASE__45__RXN \"VALINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction VALINE__45____45__TRNA__45__LIGASE__45__RXN \"VALINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19343 \"RXN-19343\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19343 \"RXN-19343\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20784 \"RXN-20784\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20784 \"RXN-20784\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16456 \"RXN-16456\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16456 \"RXN-16456\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19344 \"RXN-19344\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19344 \"RXN-19344\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16457 \"RXN-16457\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16457 \"RXN-16457\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIAZOLSYN2__45__RXN \"THIAZOLSYN2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIAZOLSYN2__45__RXN \"THIAZOLSYN2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19338 \"RXN-19338\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19338 \"RXN-19338\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GTP__45__CYCLOHYDRO__45__I__45__RXN \"GTP-CYCLOHYDRO-I-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GTP__45__CYCLOHYDRO__45__I__45__RXN \"GTP-CYCLOHYDRO-I-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FORMATEDEHYDROG__45__RXN \"FORMATEDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FORMATEDEHYDROG__45__RXN \"FORMATEDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CITSYN__45__RXN \"CITSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CITSYN__45__RXN \"CITSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R343__45__RXN \"R343-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R343__45__RXN \"R343-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11586 \"RXN-11586\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11586 \"RXN-11586\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5225 \"RXN0-5225\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5225 \"RXN0-5225\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction BTUR2__45__RXN \"BTUR2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction BTUR2__45__RXN \"BTUR2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PROTOHEMEFERROCHELAT__45__RXN \"PROTOHEMEFERROCHELAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PROTOHEMEFERROCHELAT__45__RXN \"PROTOHEMEFERROCHELAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19369 \"RXN-19369\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19369 \"RXN-19369\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19335 \"RXN-19335\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19335 \"RXN-19335\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12611 \"RXN-12611\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12611 \"RXN-12611\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22246 \"RXN-22246\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22246 \"RXN-22246\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22238 \"RXN-22238\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22238 \"RXN-22238\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6427 \"RXN0-6427\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6427 \"RXN0-6427\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19342 \"RXN-19342\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19342 \"RXN-19342\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14192 \"RXN-14192\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14192 \"RXN-14192\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9280 \"RXN-9280\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9280 \"RXN-9280\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACYLPHOSPHATASE__45__RXN__45__ACETYL__45__P__47__WATER__47____47__ACET__47__Pi__47__PROTON__46__31__46__ \"ACYLPHOSPHATASE-RXN-ACETYL-P/WATER//ACET/Pi/PROTON.31.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACYLPHOSPHATASE__45__RXN__45__ACETYL__45__P__47__WATER__47____47__ACET__47__Pi__47__PROTON__46__31__46__ \"ACYLPHOSPHATASE-RXN-ACETYL-P/WATER//ACET/Pi/PROTON.31.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPPYPHOSKIN__45__RXN \"GDPPYPHOSKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPPYPHOSKIN__45__RXN \"GDPPYPHOSKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14207 \"RXN-14207\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14207 \"RXN-14207\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9283 \"RXN-9283\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9283 \"RXN-9283\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14517 \"RXN-14517\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14517 \"RXN-14517\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SERINE__45____45__TRNA__45__LIGASE__45__RXN \"SERINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SERINE__45____45__TRNA__45__LIGASE__45__RXN \"SERINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19339 \"RXN-19339\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19339 \"RXN-19339\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PPPGPPHYDRO__45__RXN \"PPPGPPHYDRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PPPGPPHYDRO__45__RXN \"PPPGPPHYDRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12458 \"RXN-12458\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12458 \"RXN-12458\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9277 \"RXN-9277\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9277 \"RXN-9277\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__2161 \"RXN0-2161\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__2161 \"RXN0-2161\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PEPDEPHOS__45__RXN \"PEPDEPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PEPDEPHOS__45__RXN \"PEPDEPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20490 \"RXN-20490\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20490 \"RXN-20490\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction OCTAPRENYL__45__METHYL__45__METHOXY__45__BENZOQ__45__OH__45__RXN \"OCTAPRENYL-METHYL-METHOXY-BENZOQ-OH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction OCTAPRENYL__45__METHYL__45__METHOXY__45__BENZOQ__45__OH__45__RXN \"OCTAPRENYL-METHYL-METHOXY-BENZOQ-OH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNI__45__27 \"RXNI-27\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNI__45__27 \"RXNI-27\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14117 \"RXN-14117\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14117 \"RXN-14117\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20787 \"RXN-20787\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20787 \"RXN-20787\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__63__45__RXN \"2.1.1.63-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__63__45__RXN \"2.1.1.63-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1__45__PHOSPHATE__45__SYNTHASE__45__RXN__45__ALPHA__45__GLC__45__6__45__P__47____47__1__45__L__45__MYO__45__INOSITOL__45__1__45__P__46__36__46__ \"MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN-ALPHA-GLC-6-P//1-L-MYO-INOSITOL-1-P.36.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1__45__PHOSPHATE__45__SYNTHASE__45__RXN__45__ALPHA__45__GLC__45__6__45__P__47____47__1__45__L__45__MYO__45__INOSITOL__45__1__45__P__46__36__46__ \"MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN-ALPHA-GLC-6-P//1-L-MYO-INOSITOL-1-P.36.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1__45__PHOSPHATE__45__SYNTHASE__45__RXN__45__GLC__45__6__45__P__47____47__1__45__L__45__MYO__45__INOSITOL__45__1__45__P__46__30__46__ \"MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN-GLC-6-P//1-L-MYO-INOSITOL-1-P.30.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1__45__PHOSPHATE__45__SYNTHASE__45__RXN__45__GLC__45__6__45__P__47____47__1__45__L__45__MYO__45__INOSITOL__45__1__45__P__46__30__46__ \"MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN-GLC-6-P//1-L-MYO-INOSITOL-1-P.30.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1__45__PHOSPHATE__45__SYNTHASE__45__RXN \"MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1__45__PHOSPHATE__45__SYNTHASE__45__RXN \"MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__50 \"RXN2B4Q-50\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__50 \"RXN2B4Q-50\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23221 \"RXN-23221\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23221 \"RXN-23221\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPARTATE__45____45__TRNA__45__LIGASE__45__RXN \"ASPARTATE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPARTATE__45____45__TRNA__45__LIGASE__45__RXN \"ASPARTATE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22245 \"RXN-22245\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22245 \"RXN-22245\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22482 \"RXN-22482\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22482 \"RXN-22482\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPHYDROXYMYRGLUCOSAMNACETYLTRANS__45__RXN \"UDPHYDROXYMYRGLUCOSAMNACETYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPHYDROXYMYRGLUCOSAMNACETYLTRANS__45__RXN \"UDPHYDROXYMYRGLUCOSAMNACETYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__55 \"RXN2B4Q-55\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__55 \"RXN2B4Q-55\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22241 \"RXN-22241\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22241 \"RXN-22241\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22498 \"RXN-22498\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22498 \"RXN-22498\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction H2PTEROATESYNTH__45__RXN \"H2PTEROATESYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction H2PTEROATESYNTH__45__RXN \"H2PTEROATESYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction BIOTIN__45__CARBOXYL__45__RXN \"BIOTIN-CARBOXYL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction BIOTIN__45__CARBOXYL__45__RXN \"BIOTIN-CARBOXYL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction QUINOLINATE__45__SYNTHA__45__RXN \"QUINOLINATE-SYNTHA-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction QUINOLINATE__45__SYNTHA__45__RXN \"QUINOLINATE-SYNTHA-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2DNV__45__3 \"RXN2DNV-3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2DNV__45__3 \"RXN2DNV-3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRUVATEORTHOPHOSPHATE__45__DIKINASE__45__RXN \"PYRUVATEORTHOPHOSPHATE-DIKINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRUVATEORTHOPHOSPHATE__45__DIKINASE__45__RXN \"PYRUVATEORTHOPHOSPHATE-DIKINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23513 \"RXN-23513\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23513 \"RXN-23513\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GSAAMINOTRANS__45__RXN \"GSAAMINOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GSAAMINOTRANS__45__RXN \"GSAAMINOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12754 \"RXN-12754\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12754 \"RXN-12754\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12753 \"RXN-12753\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12753 \"RXN-12753\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__2382 \"RXN0-2382\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__2382 \"RXN0-2382\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1133 \"RXN0-1133\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1133 \"RXN0-1133\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__2__46__2__46__23__45__RXN \"3.2.2.23-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__2__46__2__46__23__45__RXN \"3.2.2.23-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__20 \"RXN0-20\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__20 \"RXN0-20\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PEPCARBOX__45__RXN \"PEPCARBOX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PEPCARBOX__45__RXN \"PEPCARBOX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTIDINE__45____45__TRNA__45__LIGASE__45__RXN \"HISTIDINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTIDINE__45____45__TRNA__45__LIGASE__45__RXN \"HISTIDINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18435 \"RXN-18435\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18435 \"RXN-18435\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16165 \"RXN-16165\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16165 \"RXN-16165\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSPHAGLYPSYN__45__RXN \"PHOSPHAGLYPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSPHAGLYPSYN__45__RXN \"PHOSPHAGLYPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17824 \"RXN-17824\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17824 \"RXN-17824\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__1961 \"RXN-1961\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__1961 \"RXN-1961\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5398 \"RXN0-5398\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5398 \"RXN0-5398\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__11__46__9__45__RXN \"3.4.11.9-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__11__46__9__45__RXN \"3.4.11.9-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22699 \"RXN-22699\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22699 \"RXN-22699\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11483 \"RXN-11483\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11483 \"RXN-11483\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20757 \"RXN-20757\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20757 \"RXN-20757\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11475 \"RXN-11475\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11475 \"RXN-11475\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20756 \"RXN-20756\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20756 \"RXN-20756\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20755 \"RXN-20755\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20755 \"RXN-20755\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18709 \"RXN-18709\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18709 \"RXN-18709\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20456 \"RXN-20456\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20456 \"RXN-20456\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ORNCARBAMTRANSFER__45__RXN \"ORNCARBAMTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ORNCARBAMTRANSFER__45__RXN \"ORNCARBAMTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20457 \"RXN-20457\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20457 \"RXN-20457\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5144 \"RXN0-5144\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5144 \"RXN0-5144\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13826 \"RXN-13826\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13826 \"RXN-13826\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13824 \"RXN-13824\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13824 \"RXN-13824\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7081 \"RXN0-7081\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7081 \"RXN0-7081\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CHORPYRLY__45__RXN \"CHORPYRLY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CHORPYRLY__45__RXN \"CHORPYRLY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21230 \"RXN-21230\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21230 \"RXN-21230\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R341__45__RXN \"R341-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R341__45__RXN \"R341-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETYL__45__COA__45__CARBOXYLTRANSFER__45__RXN \"ACETYL-COA-CARBOXYLTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETYL__45__COA__45__CARBOXYLTRANSFER__45__RXN \"ACETYL-COA-CARBOXYLTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__3__46__1__46__54__45__RXN \"1.3.1.54-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__3__46__1__46__54__45__RXN \"1.3.1.54-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDROXYISOVALDEHYDRAT__45__RXN \"DIHYDROXYISOVALDEHYDRAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDROXYISOVALDEHYDRAT__45__RXN \"DIHYDROXYISOVALDEHYDRAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDROXYMETVALDEHYDRAT__45__RXN \"DIHYDROXYMETVALDEHYDRAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDROXYMETVALDEHYDRAT__45__RXN \"DIHYDROXYMETVALDEHYDRAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R322__45__RXN \"R322-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R322__45__RXN \"R322-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R321__45__RXN \"R321-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R321__45__RXN \"R321-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8770 \"RXN-8770\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8770 \"RXN-8770\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DCDPKIN__45__RXN \"DCDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DCDPKIN__45__RXN \"DCDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DUDPKIN__45__RXN \"DUDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DUDPKIN__45__RXN \"DUDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14481 \"RXN-14481\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14481 \"RXN-14481\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14480 \"RXN-14480\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14480 \"RXN-14480\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIAZOLSYN3__45__RXN \"THIAZOLSYN3-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIAZOLSYN3__45__RXN \"THIAZOLSYN3-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction N__45__ACETYLTRANSFER__45__RXN \"N-ACETYLTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction N__45__ACETYLTRANSFER__45__RXN \"N-ACETYLTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5063 \"RXN0-5063\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5063 \"RXN0-5063\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6948 \"RXN0-6948\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6948 \"RXN0-6948\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MALATE__45__DEHYDROGENASE__45__NADP__43____45__RXN \"MALATE-DEHYDROGENASE-NADP+-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MALATE__45__DEHYDROGENASE__45__NADP__43____45__RXN \"MALATE-DEHYDROGENASE-NADP+-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DTDPKIN__45__RXN \"DTDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DTDPKIN__45__RXN \"DTDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DADPKIN__45__RXN \"DADPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DADPKIN__45__RXN \"DADPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPKIN__45__RXN \"UDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPKIN__45__RXN \"UDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__72__45__RXN \"2.1.1.72-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__72__45__RXN \"2.1.1.72-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12093 \"RXN-12093\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12093 \"RXN-12093\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SIROHEME__45__FERROCHELAT__45__RXN \"SIROHEME-FERROCHELAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SIROHEME__45__FERROCHELAT__45__RXN \"SIROHEME-FERROCHELAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6575 \"RXN0-6575\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6575 \"RXN0-6575\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPKIN__45__RXN \"GDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPKIN__45__RXN \"GDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CDPKIN__45__RXN \"CDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CDPKIN__45__RXN \"CDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYCERONE__45__KINASE__45__RXN \"GLYCERONE-KINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYCERONE__45__KINASE__45__RXN \"GLYCERONE-KINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__8__46__25__45__RXN \"2.7.8.25-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__8__46__25__45__RXN \"2.7.8.25-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2DNV__45__2 \"RXN2DNV-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2DNV__45__2 \"RXN2DNV-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UROPORIIIMETHYLTRANSA__45__RXN \"UROPORIIIMETHYLTRANSA-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UROPORIIIMETHYLTRANSA__45__RXN \"UROPORIIIMETHYLTRANSA-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DGDPKIN__45__RXN \"DGDPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DGDPKIN__45__RXN \"DGDPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13403 \"RXN-13403\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13403 \"RXN-13403\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ARGSUCCINSYN__45__RXN \"ARGSUCCINSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ARGSUCCINSYN__45__RXN \"ARGSUCCINSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10 \"RXN-10\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10 \"RXN-10\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PEPSYNTH__45__RXN \"PEPSYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PEPSYNTH__45__RXN \"PEPSYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23219 \"RXN-23219\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23219 \"RXN-23219\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSPHASERDECARB__45__RXN \"PHOSPHASERDECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSPHASERDECARB__45__RXN \"PHOSPHASERDECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FUC4NACTRANS__45__RXN \"FUC4NACTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FUC4NACTRANS__45__RXN \"FUC4NACTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__4621 \"RXN0-4621\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__4621 \"RXN0-4621\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPARAGHYD__45__RXN \"ASPARAGHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPARAGHYD__45__RXN \"ASPARAGHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14396 \"RXN-14396\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14396 \"RXN-14396\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _5__46__1__46__3__46__20__45__RXN \"5.1.3.20-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _5__46__1__46__3__46__20__45__RXN \"5.1.3.20-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22237 \"RXN-22237\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22237 \"RXN-22237\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21562 \"RXN-21562\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21562 \"RXN-21562\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15139 \"RXN-15139\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15139 \"RXN-15139\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__4022 \"RXN0-4022\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__4022 \"RXN0-4022\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction N__45__ACETYLGLUTPREDUCT__45__RXN \"N-ACETYLGLUTPREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction N__45__ACETYLGLUTPREDUCT__45__RXN \"N-ACETYLGLUTPREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15635 \"RXN-15635\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15635 \"RXN-15635\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LIPIDADISACCHARIDESYNTH__45__RXN \"LIPIDADISACCHARIDESYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LIPIDADISACCHARIDESYNTH__45__RXN \"LIPIDADISACCHARIDESYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9190 \"RXN-9190\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9190 \"RXN-9190\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11633 \"RXN-11633\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11633 \"RXN-11633\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENYLOSUCCINATE__45__SYNTHASE__45__RXN \"ADENYLOSUCCINATE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENYLOSUCCINATE__45__SYNTHASE__45__RXN \"ADENYLOSUCCINATE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBOPHOSPHAT__45__RXN \"RIBOPHOSPHAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBOPHOSPHAT__45__RXN \"RIBOPHOSPHAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23525 \"RXN-23525\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23525 \"RXN-23525\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _5__45__FORMYL__45__THF__45__CYCLO__45__LIGASE__45__RXN \"5-FORMYL-THF-CYCLO-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _5__45__FORMYL__45__THF__45__CYCLO__45__LIGASE__45__RXN \"5-FORMYL-THF-CYCLO-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CHEBDEAMID__45__RXN \"CHEBDEAMID-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CHEBDEAMID__45__RXN \"CHEBDEAMID-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FORMATETHFLIG__45__RXN \"FORMATETHFLIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FORMATETHFLIG__45__RXN \"FORMATETHFLIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ERYTHRON4PDEHYDROG__45__RXN \"ERYTHRON4PDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ERYTHRON4PDEHYDROG__45__RXN \"ERYTHRON4PDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6274 \"RXN0-6274\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6274 \"RXN0-6274\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ERYTH4PDEHYDROG__45__RXN \"ERYTH4PDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ERYTH4PDEHYDROG__45__RXN \"ERYTH4PDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__1__46__90__45__RXN \"2.7.1.90-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__1__46__90__45__RXN \"2.7.1.90-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PNPOXI__45__RXN \"PNPOXI-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PNPOXI__45__RXN \"PNPOXI-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15125 \"RXN-15125\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15125 \"RXN-15125\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21730 \"RXN-21730\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21730 \"RXN-21730\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DTDPDEHYDRHAMEPIM__45__RXN \"DTDPDEHYDRHAMEPIM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DTDPDEHYDRHAMEPIM__45__RXN \"DTDPDEHYDRHAMEPIM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LYSINE__45____45__TRNA__45__LIGASE__45__RXN \"LYSINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LYSINE__45____45__TRNA__45__LIGASE__45__RXN \"LYSINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4__46__3__46__1__46__17__45__RXN \"4.3.1.17-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4__46__3__46__1__46__17__45__RXN \"4.3.1.17-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14727 \"RXN-14727\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14727 \"RXN-14727\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16998 \"RXN-16998\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16998 \"RXN-16998\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16998__45__ALPHA__45__GLUCOSE__45__16__45__BISPHOSPHATE__47__Phosphoglucomutase__47____47__GLC__45__6__45__P__47__Phosphorylated__45__phosphoglucomutase__46__92__46__ \"RXN-16998-ALPHA-GLUCOSE-16-BISPHOSPHATE/Phosphoglucomutase//GLC-6-P/Phosphorylated-phosphoglucomutase.92.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16998__45__ALPHA__45__GLUCOSE__45__16__45__BISPHOSPHATE__47__Phosphoglucomutase__47____47__GLC__45__6__45__P__47__Phosphorylated__45__phosphoglucomutase__46__92__46__ \"RXN-16998-ALPHA-GLUCOSE-16-BISPHOSPHATE/Phosphoglucomutase//GLC-6-P/Phosphorylated-phosphoglucomutase.92.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16998__45__ALPHA__45__GLUCOSE__45__16__45__BISPHOSPHATE__47__Phosphoglucomutase__47____47__ALPHA__45__GLC__45__6__45__P__47__Phosphorylated__45__phosphoglucomutase__46__98__46__ \"RXN-16998-ALPHA-GLUCOSE-16-BISPHOSPHATE/Phosphoglucomutase//ALPHA-GLC-6-P/Phosphorylated-phosphoglucomutase.98.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16998__45__ALPHA__45__GLUCOSE__45__16__45__BISPHOSPHATE__47__Phosphoglucomutase__47____47__ALPHA__45__GLC__45__6__45__P__47__Phosphorylated__45__phosphoglucomutase__46__98__46__ \"RXN-16998-ALPHA-GLUCOSE-16-BISPHOSPHATE/Phosphoglucomutase//ALPHA-GLC-6-P/Phosphorylated-phosphoglucomutase.98.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16997 \"RXN-16997\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16997 \"RXN-16997\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12104 \"RXN-12104\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12104 \"RXN-12104\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPARAGINE__45____45__TRNA__45__LIGASE__45__RXN \"ASPARAGINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPARAGINE__45____45__TRNA__45__LIGASE__45__RXN \"ASPARAGINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11477 \"RXN-11477\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11477 \"RXN-11477\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSPHOGLUCMUT__45__RXN__45__GLC__45__1__45__P__47____47__ALPHA__45__GLC__45__6__45__P__46__23__46__ \"PHOSPHOGLUCMUT-RXN-GLC-1-P//ALPHA-GLC-6-P.23.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSPHOGLUCMUT__45__RXN__45__GLC__45__1__45__P__47____47__ALPHA__45__GLC__45__6__45__P__46__23__46__ \"PHOSPHOGLUCMUT-RXN-GLC-1-P//ALPHA-GLC-6-P.23.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSPHOGLUCMUT__45__RXN__45__GLC__45__1__45__P__47____47__GLC__45__6__45__P__46__17__46__ \"PHOSPHOGLUCMUT-RXN-GLC-1-P//GLC-6-P.17.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSPHOGLUCMUT__45__RXN__45__GLC__45__1__45__P__47____47__GLC__45__6__45__P__46__17__46__ \"PHOSPHOGLUCMUT-RXN-GLC-1-P//GLC-6-P.17.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSPHOGLUCMUT__45__RXN \"PHOSPHOGLUCMUT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSPHOGLUCMUT__45__RXN \"PHOSPHOGLUCMUT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDROOROT__45__RXN \"DIHYDROOROT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDROOROT__45__RXN \"DIHYDROOROT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HYDROG__45__RXN \"HYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HYDROG__45__RXN \"HYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5114 \"RXN0-5114\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5114 \"RXN0-5114\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1321 \"RXN0-1321\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1321 \"RXN0-1321\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15127 \"RXN-15127\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15127 \"RXN-15127\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SULFATE__45__ADENYLYLTRANS__45__RXN \"SULFATE-ADENYLYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SULFATE__45__ADENYLYLTRANS__45__RXN \"SULFATE-ADENYLYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14257 \"RXN-14257\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14257 \"RXN-14257\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DTDPGLUCOSEPP__45__RXN \"DTDPGLUCOSEPP-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DTDPGLUCOSEPP__45__RXN \"DTDPGLUCOSEPP-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14256 \"RXN-14256\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14256 \"RXN-14256\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1483 \"RXN0-1483\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1483 \"RXN0-1483\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14971 \"RXN-14971\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14971 \"RXN-14971\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIMETHUROPORDEHYDROG__45__RXN \"DIMETHUROPORDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIMETHUROPORDEHYDROG__45__RXN \"DIMETHUROPORDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8769 \"RXN-8769\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8769 \"RXN-8769\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MALATE__45__DEHYDROGENASE__45__ACCEPTOR__45__RXN \"MALATE-DEHYDROGENASE-ACCEPTOR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MALATE__45__DEHYDROGENASE__45__ACCEPTOR__45__RXN \"MALATE-DEHYDROGENASE-ACCEPTOR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction INORGPYROPHOSPHAT__45__RXN \"INORGPYROPHOSPHAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction INORGPYROPHOSPHAT__45__RXN \"INORGPYROPHOSPHAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__ALPHA__45__GLUCOSE__47__ATP__47____47__ALPHA__45__GLC__45__6__45__P__47__ADP__47__PROTON__46__44__46__ \"GLUCOKIN-RXN-ALPHA-GLUCOSE/ATP//ALPHA-GLC-6-P/ADP/PROTON.44.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__ALPHA__45__GLUCOSE__47__ATP__47____47__ALPHA__45__GLC__45__6__45__P__47__ADP__47__PROTON__46__44__46__ \"GLUCOKIN-RXN-ALPHA-GLUCOSE/ATP//ALPHA-GLC-6-P/ADP/PROTON.44.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__ALPHA__45__GLUCOSE__47__ATP__47____47__GLC__45__6__45__P__47__ADP__47__PROTON__46__38__46__ \"GLUCOKIN-RXN-ALPHA-GLUCOSE/ATP//GLC-6-P/ADP/PROTON.38.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__ALPHA__45__GLUCOSE__47__ATP__47____47__GLC__45__6__45__P__47__ADP__47__PROTON__46__38__46__ \"GLUCOKIN-RXN-ALPHA-GLUCOSE/ATP//GLC-6-P/ADP/PROTON.38.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__ALPHA__45__GLUCOSE__47__ATP__47____47__D__45__glucopyranose__45__6__45__phosphate__47__ADP__47__PROTON__46__58__46__ \"GLUCOKIN-RXN-ALPHA-GLUCOSE/ATP//D-glucopyranose-6-phosphate/ADP/PROTON.58.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__ALPHA__45__GLUCOSE__47__ATP__47____47__D__45__glucopyranose__45__6__45__phosphate__47__ADP__47__PROTON__46__58__46__ \"GLUCOKIN-RXN-ALPHA-GLUCOSE/ATP//D-glucopyranose-6-phosphate/ADP/PROTON.58.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__GLC__47__ATP__47____47__ALPHA__45__GLC__45__6__45__P__47__ADP__47__PROTON__46__34__46__ \"GLUCOKIN-RXN-GLC/ATP//ALPHA-GLC-6-P/ADP/PROTON.34.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__GLC__47__ATP__47____47__ALPHA__45__GLC__45__6__45__P__47__ADP__47__PROTON__46__34__46__ \"GLUCOKIN-RXN-GLC/ATP//ALPHA-GLC-6-P/ADP/PROTON.34.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__GLC__47__ATP__47____47__GLC__45__6__45__P__47__ADP__47__PROTON__46__28__46__ \"GLUCOKIN-RXN-GLC/ATP//GLC-6-P/ADP/PROTON.28.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__GLC__47__ATP__47____47__GLC__45__6__45__P__47__ADP__47__PROTON__46__28__46__ \"GLUCOKIN-RXN-GLC/ATP//GLC-6-P/ADP/PROTON.28.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__GLC__47__ATP__47____47__D__45__glucopyranose__45__6__45__phosphate__47__ADP__47__PROTON__46__48__46__ \"GLUCOKIN-RXN-GLC/ATP//D-glucopyranose-6-phosphate/ADP/PROTON.48.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__GLC__47__ATP__47____47__D__45__glucopyranose__45__6__45__phosphate__47__ADP__47__PROTON__46__48__46__ \"GLUCOKIN-RXN-GLC/ATP//D-glucopyranose-6-phosphate/ADP/PROTON.48.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__Glucopyranose__47__ATP__47____47__ALPHA__45__GLC__45__6__45__P__47__ADP__47__PROTON__46__44__46__ \"GLUCOKIN-RXN-Glucopyranose/ATP//ALPHA-GLC-6-P/ADP/PROTON.44.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__Glucopyranose__47__ATP__47____47__ALPHA__45__GLC__45__6__45__P__47__ADP__47__PROTON__46__44__46__ \"GLUCOKIN-RXN-Glucopyranose/ATP//ALPHA-GLC-6-P/ADP/PROTON.44.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__Glucopyranose__47__ATP__47____47__GLC__45__6__45__P__47__ADP__47__PROTON__46__38__46__ \"GLUCOKIN-RXN-Glucopyranose/ATP//GLC-6-P/ADP/PROTON.38.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN__45__Glucopyranose__47__ATP__47____47__GLC__45__6__45__P__47__ADP__47__PROTON__46__38__46__ \"GLUCOKIN-RXN-Glucopyranose/ATP//GLC-6-P/ADP/PROTON.38.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN \"GLUCOKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOKIN__45__RXN \"GLUCOKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11837 \"RXN-11837\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11837 \"RXN-11837\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18392 \"RXN-18392\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18392 \"RXN-18392\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALANINE__45____45__TRNA__45__LIGASE__45__RXN \"ALANINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALANINE__45____45__TRNA__45__LIGASE__45__RXN \"ALANINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14882 \"RXN-14882\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14882 \"RXN-14882\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__OCTAPRENYL__45__METHOXY__45__BENZOQ__45__METH__45__RXN \"2-OCTAPRENYL-METHOXY-BENZOQ-METH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__OCTAPRENYL__45__METHOXY__45__BENZOQ__45__METH__45__RXN \"2-OCTAPRENYL-METHOXY-BENZOQ-METH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FUMHYDR__45__RXN \"FUMHYDR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FUMHYDR__45__RXN \"FUMHYDR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPARTATEKIN__45__RXN \"ASPARTATEKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPARTATEKIN__45__RXN \"ASPARTATEKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18032 \"RXN-18032\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18032 \"RXN-18032\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11845 \"RXN-11845\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11845 \"RXN-11845\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11291 \"RXN-11291\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11291 \"RXN-11291\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11637 \"RXN-11637\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11637 \"RXN-11637\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12720 \"RXN-12720\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12720 \"RXN-12720\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18448 \"RXN-18448\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18448 \"RXN-18448\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NICOTINATEPRIBOSYLTRANS__45__RXN \"NICOTINATEPRIBOSYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NICOTINATEPRIBOSYLTRANS__45__RXN \"NICOTINATEPRIBOSYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14118 \"RXN-14118\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14118 \"RXN-14118\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20083 \"RXN-20083\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20083 \"RXN-20083\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11348 \"RXN-11348\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11348 \"RXN-11348\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DGTPTRIPHYDRO__45__RXN \"DGTPTRIPHYDRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DGTPTRIPHYDRO__45__RXN \"DGTPTRIPHYDRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11350 \"RXN-11350\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11350 \"RXN-11350\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11409 \"RXN-11409\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11409 \"RXN-11409\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6435 \"RXN0-6435\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6435 \"RXN0-6435\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11478 \"RXN-11478\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11478 \"RXN-11478\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5359 \"RXN0-5359\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5359 \"RXN0-5359\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__5__46__1__46__20__45__RXN \"1.5.1.20-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__5__46__1__46__20__45__RXN \"1.5.1.20-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__8__46__4__46__13__45__RXN \"1.8.4.13-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__8__46__4__46__13__45__RXN \"1.8.4.13-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__6081 \"RXN-6081\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__6081 \"RXN-6081\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11482 \"RXN-11482\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11482 \"RXN-11482\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11410 \"RXN-11410\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11410 \"RXN-11410\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADCLY__45__RXN \"ADCLY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADCLY__45__RXN \"ADCLY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PDXJ__45__RXN \"PDXJ-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PDXJ__45__RXN \"PDXJ-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8315 \"RXN-8315\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8315 \"RXN-8315\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9191 \"RXN-9191\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9191 \"RXN-9191\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN3O__45__1983 \"RXN3O-1983\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN3O__45__1983 \"RXN3O-1983\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20785 \"RXN-20785\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20785 \"RXN-20785\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction H2NEOPTERINP3PYROPHOSPHOHYDRO__45__RXN \"H2NEOPTERINP3PYROPHOSPHOHYDRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction H2NEOPTERINP3PYROPHOSPHOHYDRO__45__RXN \"H2NEOPTERINP3PYROPHOSPHOHYDRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14409__45__CPD__45__15590__47____47__ALPHA__45__D__45__GALACTOSE__46__29__46__ \"RXN-14409-CPD-15590//ALPHA-D-GALACTOSE.29.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14409__45__CPD__45__15590__47____47__ALPHA__45__D__45__GALACTOSE__46__29__46__ \"RXN-14409-CPD-15590//ALPHA-D-GALACTOSE.29.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14409__45__CPD__45__15590__47____47__GALACTOSE__46__21__46__ \"RXN-14409-CPD-15590//GALACTOSE.21.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14409__45__CPD__45__15590__47____47__GALACTOSE__46__21__46__ \"RXN-14409-CPD-15590//GALACTOSE.21.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14409 \"RXN-14409\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14409 \"RXN-14409\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14002 \"RXN-14002\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14002 \"RXN-14002\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C0__45__H0__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C0__45__H0__47__NADH__47__PROTON__46__100__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C0-H0/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C0-H0/NADH/PROTON.100.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C0__45__H0__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C0__45__H0__47__NADH__47__PROTON__46__100__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C0-H0/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C0-H0/NADH/PROTON.100.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C2__45__H4__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C2__45__H4__47__NADH__47__PROTON__46__100__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C2-H4/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C2-H4/NADH/PROTON.100.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C2__45__H4__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C2__45__H4__47__NADH__47__PROTON__46__100__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C2-H4/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C2-H4/NADH/PROTON.100.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__Octanoyl__45__ACPs__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C4__45__H8__47__NADH__47__PROTON__46__69__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-Octanoyl-ACPs/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C4-H8/NADH/PROTON.69.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__Octanoyl__45__ACPs__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C4__45__H8__47__NADH__47__PROTON__46__69__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-Octanoyl-ACPs/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C4-H8/NADH/PROTON.69.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C6__45__H12__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C6__45__H12__47__NADH__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C6-H12/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C6-H12/NADH/PROTON.102.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C6__45__H12__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C6__45__H12__47__NADH__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C6-H12/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C6-H12/NADH/PROTON.102.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C8__45__H16__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C8__45__H16__47__NADH__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C8-H16/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C8-H16/NADH/PROTON.102.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C8__45__H16__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C8__45__H16__47__NADH__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C8-H16/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C8-H16/NADH/PROTON.102.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C10__45__H20__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C10__45__H20__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C10-H20/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C10-H20/NADH/PROTON.104.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C10__45__H20__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C10__45__H20__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C10-H20/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C10-H20/NADH/PROTON.104.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C12__45__H24__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C12__45__H24__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C12-H24/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C12-H24/NADH/PROTON.104.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C12__45__H24__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C12__45__H24__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C12-H24/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C12-H24/NADH/PROTON.104.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C14__45__H28__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C14__45__H28__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C14-H28/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C14-H28/NADH/PROTON.104.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C14__45__H28__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C14__45__H28__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C14-H28/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C14-H28/NADH/PROTON.104.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C16__45__H32__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C16__45__H32__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C16-H32/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C16-H32/NADH/PROTON.104.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C16__45__H32__47__NAD__47____47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C16__45__H32__47__NADH__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C16-H32/NAD//POLYMER-INST-TRANS-D2-ENOYL-ACP-C16-H32/NADH/PROTON.104.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2961 \"RXN-2961\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2961 \"RXN-2961\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C0__45__H0__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C0__45__H0__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C0-H0/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C0-H0/PROTON.102.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C0__45__H0__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C0__45__H0__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C0-H0/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C0-H0/PROTON.102.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C2__45__H4__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C2__45__H4__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C2-H4/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C2-H4/PROTON.102.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C2__45__H4__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C2__45__H4__47__PROTON__46__102__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C2-H4/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C2-H4/PROTON.102.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__Octanoyl__45__ACPs__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C4__45__H8__47__PROTON__46__71__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-Octanoyl-ACPs/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C4-H8/PROTON.71.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__Octanoyl__45__ACPs__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C4__45__H8__47__PROTON__46__71__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-Octanoyl-ACPs/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C4-H8/PROTON.71.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C6__45__H12__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C6__45__H12__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C6-H12/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C6-H12/PROTON.104.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C6__45__H12__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C6__45__H12__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C6-H12/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C6-H12/PROTON.104.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C8__45__H16__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C8__45__H16__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C8-H16/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C8-H16/PROTON.104.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C8__45__H16__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C8__45__H16__47__PROTON__46__104__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C8-H16/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C8-H16/PROTON.104.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C10__45__H20__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C10__45__H20__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C10-H20/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C10-H20/PROTON.106.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C10__45__H20__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C10__45__H20__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C10-H20/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C10-H20/PROTON.106.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C12__45__H24__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C12__45__H24__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C12-H24/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C12-H24/PROTON.106.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C12__45__H24__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C12__45__H24__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C12-H24/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C12-H24/PROTON.106.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C14__45__H28__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C14__45__H28__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C14-H28/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C14-H28/PROTON.106.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C14__45__H28__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C14__45__H28__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C14-H28/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C14-H28/PROTON.106.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C16__45__H32__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C16__45__H32__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C16-H32/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C16-H32/PROTON.106.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ENOYL__45__ACP__45__REDUCT__45__NADPH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C16__45__H32__47__NADP__47____47__NADPH__47__POLYMER__45__INST__45__TRANS__45__D2__45__ENOYL__45__ACP__45__C16__45__H32__47__PROTON__46__106__46__ \"ENOYL-ACP-REDUCT-NADPH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C16-H32/NADP//NADPH/POLYMER-INST-TRANS-D2-ENOYL-ACP-C16-H32/PROTON.106.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22700 \"RXN-22700\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22700 \"RXN-22700\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DHHB__45__METHYLTRANSFER__45__RXN \"DHHB-METHYLTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DHHB__45__METHYLTRANSFER__45__RXN \"DHHB-METHYLTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R83__45__RXN \"R83-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R83__45__RXN \"R83-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15140 \"RXN-15140\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15140 \"RXN-15140\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R82__45__RXN \"R82-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R82__45__RXN \"R82-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__1__46__3__46__77__45__RXN \"3.1.3.77-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__1__46__3__46__77__45__RXN \"3.1.3.77-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDRONEOPTERIN__45__MONO__45__P__45__DEPHOS__45__RXN \"DIHYDRONEOPTERIN-MONO-P-DEPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDRONEOPTERIN__45__MONO__45__P__45__DEPHOS__45__RXN \"DIHYDRONEOPTERIN-MONO-P-DEPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14552 \"RXN-14552\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14552 \"RXN-14552\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19671__45__NADH__47__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__PROTON__47____47__NAD__47__CPD__45__12831__46__63__46__ \"RXN-19671-NADH/2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/PROTON//NAD/CPD-12831.63.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19671__45__NADH__47__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__PROTON__47____47__NAD__47__CPD__45__12831__46__63__46__ \"RXN-19671-NADH/2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/PROTON//NAD/CPD-12831.63.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19671__45__NADPH__47__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__PROTON__47____47__NADP__47__CPD__45__12831__46__65__46__ \"RXN-19671-NADPH/2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/PROTON//NADP/CPD-12831.65.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19671__45__NADPH__47__2__45__METHYL__45__3__45__PHYTYL__45__14__45__NAPHTHOQUINONE__47__PROTON__47____47__NADP__47__CPD__45__12831__46__65__46__ \"RXN-19671-NADPH/2-METHYL-3-PHYTYL-14-NAPHTHOQUINONE/PROTON//NADP/CPD-12831.65.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__189 \"TRANS-RXN-189\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__189 \"TRANS-RXN-189\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21315 \"RXN-21315\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21315 \"RXN-21315\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22450 \"RXN-22450\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22450 \"RXN-22450\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _5__46__3__46__1__46__23__45__RXN \"5.3.1.23-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _5__46__3__46__1__46__23__45__RXN \"5.3.1.23-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__505 \"TRANS-RXN-505\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__505 \"TRANS-RXN-505\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTPRATPHYD__45__RXN \"HISTPRATPHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTPRATPHYD__45__RXN \"HISTPRATPHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__214 \"TRANS-RXN-214\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__214 \"TRANS-RXN-214\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GABATRANSAM__45__RXN \"GABATRANSAM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GABATRANSAM__45__RXN \"GABATRANSAM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9220 \"RXN-9220\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9220 \"RXN-9220\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__11__46__1__45__RXN \"3.4.11.1-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__11__46__1__45__RXN \"3.4.11.1-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22710 \"RXN-22710\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22710 \"RXN-22710\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ETHAMLY__45__RXN \"ETHAMLY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ETHAMLY__45__RXN \"ETHAMLY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__250 \"TRANS-RXN-250\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__250 \"TRANS-RXN-250\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22440 \"RXN-22440\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22440 \"RXN-22440\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALDOSE__45__1__45__EPIMERASE__45__RXN \"ALDOSE-1-EPIMERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALDOSE__45__1__45__EPIMERASE__45__RXN \"ALDOSE-1-EPIMERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ARGININE__45____45__TRNA__45__LIGASE__45__RXN \"ARGININE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ARGININE__45____45__TRNA__45__LIGASE__45__RXN \"ARGININE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9205 \"RXN-9205\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9205 \"RXN-9205\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SULFITE__45__REDUCT__45__RXN \"SULFITE-REDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SULFITE__45__REDUCT__45__RXN \"SULFITE-REDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADOMET__45__DMK__45__METHYLTRANSFER__45__RXN \"ADOMET-DMK-METHYLTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADOMET__45__DMK__45__METHYLTRANSFER__45__RXN \"ADOMET-DMK-METHYLTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2B4Q__45__128 \"TRANS-RXN2B4Q-128\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2B4Q__45__128 \"TRANS-RXN2B4Q-128\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22326 \"RXN-22326\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22326 \"RXN-22326\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GSDEADENYLATION__45__RXN \"GSDEADENYLATION-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GSDEADENYLATION__45__RXN \"GSDEADENYLATION-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PGLUCONDEHYDRAT__45__RXN \"PGLUCONDEHYDRAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PGLUCONDEHYDRAT__45__RXN \"PGLUCONDEHYDRAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17878 \"RXN-17878\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17878 \"RXN-17878\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9952 \"RXN-9952\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9952 \"RXN-9952\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18031 \"RXN-18031\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18031 \"RXN-18031\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17873 \"RXN-17873\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17873 \"RXN-17873\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22319 \"RXN-22319\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22319 \"RXN-22319\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBULP3EPIM__45__RXN \"RIBULP3EPIM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBULP3EPIM__45__RXN \"RIBULP3EPIM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22715 \"RXN-22715\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22715 \"RXN-22715\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22286 \"RXN-22286\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22286 \"RXN-22286\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22604 \"RXN-22604\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22604 \"RXN-22604\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__101__45__RXN \"2.3.1.101-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__101__45__RXN \"2.3.1.101-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AICARSYN__45__RXN \"AICARSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AICARSYN__45__RXN \"AICARSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17875 \"RXN-17875\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17875 \"RXN-17875\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17890 \"RXN-17890\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17890 \"RXN-17890\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18208 \"RXN-18208\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18208 \"RXN-18208\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23514 \"RXN-23514\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23514 \"RXN-23514\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AIRS__45__RXN \"AIRS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AIRS__45__RXN \"AIRS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__237 \"TRANS-RXN-237\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__237 \"TRANS-RXN-237\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17888 \"RXN-17888\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17888 \"RXN-17888\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18205 \"RXN-18205\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18205 \"RXN-18205\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1R65__45__51 \"RXN1R65-51\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1R65__45__51 \"RXN1R65-51\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17877 \"RXN-17877\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17877 \"RXN-17877\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__2 \"TRANS-RXN-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__2 \"TRANS-RXN-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18204 \"RXN-18204\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18204 \"RXN-18204\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENYLYLSULFKIN__45__RXN \"ADENYLYLSULFKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENYLYLSULFKIN__45__RXN \"ADENYLYLSULFKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FTRMBARK__45__RXN \"FTRMBARK-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FTRMBARK__45__RXN \"FTRMBARK-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17891 \"RXN-17891\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17891 \"RXN-17891\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7800 \"RXN-7800\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7800 \"RXN-7800\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22328 \"RXN-22328\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22328 \"RXN-22328\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17473 \"RXN-17473\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17473 \"RXN-17473\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NAD__45__SYNTH__45__GLN__45__RXN \"NAD-SYNTH-GLN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NAD__45__SYNTH__45__GLN__45__RXN \"NAD-SYNTH-GLN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17874 \"RXN-17874\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17874 \"RXN-17874\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__ISOPROPYLMALDEHYDROG__45__RXN \"3-ISOPROPYLMALDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__ISOPROPYLMALDEHYDROG__45__RXN \"3-ISOPROPYLMALDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22913 \"RXN-22913\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22913 \"RXN-22913\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYRIBONUCSYN__45__RXN \"GLYRIBONUCSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYRIBONUCSYN__45__RXN \"GLYRIBONUCSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTAMINESYN__45__RXN \"GLUTAMINESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTAMINESYN__45__RXN \"GLUTAMINESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23617 \"RXN-23617\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23617 \"RXN-23617\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__DEHYDROPANTOATE__45__REDUCT__45__RXN \"2-DEHYDROPANTOATE-REDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__DEHYDROPANTOATE__45__REDUCT__45__RXN \"2-DEHYDROPANTOATE-REDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17608 \"RXN-17608\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17608 \"RXN-17608\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18211 \"RXN-18211\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18211 \"RXN-18211\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17876 \"RXN-17876\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17876 \"RXN-17876\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18210 \"RXN-18210\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18210 \"RXN-18210\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2881 \"RXN-2881\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2881 \"RXN-2881\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23614 \"RXN-23614\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23614 \"RXN-23614\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__10__46__2__46__2__45__RXN \"1.10.2.2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__10__46__2__46__2__45__RXN \"1.10.2.2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__11__46__18__45__RXN \"3.4.11.18-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__11__46__18__45__RXN \"3.4.11.18-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15348 \"RXN-15348\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15348 \"RXN-15348\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GUANIDINOBUTANAMIDE__45__NH3__45__RXN \"GUANIDINOBUTANAMIDE-NH3-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GUANIDINOBUTANAMIDE__45__NH3__45__RXN \"GUANIDINOBUTANAMIDE-NH3-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__2381 \"RXN0-2381\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__2381 \"RXN0-2381\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7101 \"RXN-7101\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7101 \"RXN-7101\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13037 \"RXN-13037\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13037 \"RXN-13037\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19341 \"RXN-19341\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19341 \"RXN-19341\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19612 \"RXN-19612\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19612 \"RXN-19612\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19611 \"RXN-19611\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19611 \"RXN-19611\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11598 \"RXN-11598\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11598 \"RXN-11598\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20524 \"RXN-20524\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20524 \"RXN-20524\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19609 \"RXN-19609\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19609 \"RXN-19609\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8899 \"RXN-8899\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8899 \"RXN-8899\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R311__45__RXN \"R311-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R311__45__RXN \"R311-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12514 \"RXN-12514\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12514 \"RXN-12514\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18202 \"RXN-18202\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18202 \"RXN-18202\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTRNAREDUCT__45__RXN \"GLUTRNAREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTRNAREDUCT__45__RXN \"GLUTRNAREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5515 \"RXN0-5515\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5515 \"RXN0-5515\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17652 \"RXN-17652\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17652 \"RXN-17652\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNQT__45__4174 \"RXNQT-4174\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNQT__45__4174 \"RXNQT-4174\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CDPDIGLYSYN__45__RXN \"CDPDIGLYSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CDPDIGLYSYN__45__RXN \"CDPDIGLYSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17663 \"RXN-17663\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17663 \"RXN-17663\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LEUCYLTRANSFERASE__45__RXN \"LEUCYLTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LEUCYLTRANSFERASE__45__RXN \"LEUCYLTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNQT__45__4178 \"RXNQT-4178\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNQT__45__4178 \"RXNQT-4178\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__3482 \"RXN-3482\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__3482 \"RXN-3482\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__ISOPROPYLMALATESYN__45__RXN \"2-ISOPROPYLMALATESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__ISOPROPYLMALATESYN__45__RXN \"2-ISOPROPYLMALATESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__11__46__1__46__12__45__RXN \"1.11.1.12-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__11__46__1__46__12__45__RXN \"1.11.1.12-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23257 \"RXN-23257\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23257 \"RXN-23257\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17848 \"RXN-17848\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17848 \"RXN-17848\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DALADALALIG__45__RXN \"DALADALALIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DALADALALIG__45__RXN \"DALADALALIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYSTEINE__45____45__TRNA__45__LIGASE__45__RXN \"CYSTEINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYSTEINE__45____45__TRNA__45__LIGASE__45__RXN \"CYSTEINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN0__45__286 \"TRANS-RXN0-286\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN0__45__286 \"TRANS-RXN0-286\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14986 \"RXN-14986\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14986 \"RXN-14986\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8667 \"RXN-8667\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8667 \"RXN-8667\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PPGPPSYN__45__RXN \"PPGPPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PPGPPSYN__45__RXN \"PPGPPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14959 \"RXN-14959\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14959 \"RXN-14959\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GTPPYPHOSKIN__45__RXN \"GTPPYPHOSKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GTPPYPHOSKIN__45__RXN \"GTPPYPHOSKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14950 \"RXN-14950\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14950 \"RXN-14950\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14957 \"RXN-14957\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14957 \"RXN-14957\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7188 \"RXN0-7188\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7188 \"RXN0-7188\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14188 \"RXN-14188\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14188 \"RXN-14188\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22608 \"RXN-22608\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22608 \"RXN-22608\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17847 \"RXN-17847\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17847 \"RXN-17847\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12460 \"RXN-12460\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12460 \"RXN-12460\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _6__46__3__46__2__46__10__45__RXN \"6.3.2.10-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _6__46__3__46__2__46__10__45__RXN \"6.3.2.10-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMINOCYL__45__TRNA__45__HYDROLASE__45__RXN \"AMINOCYL-TRNA-HYDROLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMINOCYL__45__TRNA__45__HYDROLASE__45__RXN \"AMINOCYL-TRNA-HYDROLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21019 \"RXN-21019\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21019 \"RXN-21019\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDP__45__NACMURALGLDAPAALIG__45__RXN \"UDP-NACMURALGLDAPAALIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDP__45__NACMURALGLDAPAALIG__45__RXN \"UDP-NACMURALGLDAPAALIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22316 \"RXN-22316\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22316 \"RXN-22316\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14531 \"RXN-14531\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14531 \"RXN-14531\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16637 \"RXN-16637\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16637 \"RXN-16637\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction KDPGALDOL__45__RXN \"KDPGALDOL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction KDPGALDOL__45__RXN \"KDPGALDOL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14461 \"RXN-14461\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14461 \"RXN-14461\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7605 \"RXN-7605\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7605 \"RXN-7605\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23515 \"RXN-23515\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23515 \"RXN-23515\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11601 \"RXN-11601\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11601 \"RXN-11601\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DTDPDEHYRHAMREDUCT__45__RXN \"DTDPDEHYRHAMREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DTDPDEHYRHAMREDUCT__45__RXN \"DTDPDEHYRHAMREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMONITRO__45__RXN \"AMONITRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMONITRO__45__RXN \"AMONITRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNQT__45__4165 \"RXNQT-4165\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNQT__45__4165 \"RXNQT-4165\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22729 \"RXN-22729\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22729 \"RXN-22729\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTCYSLIG__45__RXN \"GLUTCYSLIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTCYSLIG__45__RXN \"GLUTCYSLIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6359 \"RXN0-6359\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6359 \"RXN0-6359\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7167 \"RXN0-7167\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7167 \"RXN0-7167\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN490__45__3616 \"RXN490-3616\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN490__45__3616 \"RXN490-3616\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6385 \"RXN0-6385\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6385 \"RXN0-6385\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20406 \"RXN-20406\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20406 \"RXN-20406\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5468 \"RXN0-5468\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5468 \"RXN0-5468\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRNA__45__PSEUDOURIDINE__45__SYNTHASE__45__I__45__RXN \"TRNA-PSEUDOURIDINE-SYNTHASE-I-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRNA__45__PSEUDOURIDINE__45__SYNTHASE__45__I__45__RXN \"TRNA-PSEUDOURIDINE-SYNTHASE-I-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13158 \"RXN-13158\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13158 \"RXN-13158\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19040 \"RXN-19040\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19040 \"RXN-19040\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12893 \"RXN-12893\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12893 \"RXN-12893\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIOSULFATE__45__SULFURTRANSFERASE__45__RXN \"THIOSULFATE-SULFURTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIOSULFATE__45__SULFURTRANSFERASE__45__RXN \"THIOSULFATE-SULFURTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__8__46__4__46__8__45__RXN \"1.8.4.8-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__8__46__4__46__8__45__RXN \"1.8.4.8-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19307 \"RXN-19307\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19307 \"RXN-19307\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11811 \"RXN-11811\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11811 \"RXN-11811\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17846 \"RXN-17846\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17846 \"RXN-17846\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTAMIN__45__RXN \"GLUTAMIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTAMIN__45__RXN \"GLUTAMIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19954 \"RXN-19954\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19954 \"RXN-19954\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10462 \"RXN-10462\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10462 \"RXN-10462\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22619 \"RXN-22619\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22619 \"RXN-22619\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHENYLPYRUVATE__45__TAUTOMERASE__45__RXN \"PHENYLPYRUVATE-TAUTOMERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHENYLPYRUVATE__45__TAUTOMERASE__45__RXN \"PHENYLPYRUVATE-TAUTOMERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPNACETYLGLUCOSAMACYLTRANS__45__RXN \"UDPNACETYLGLUCOSAMACYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPNACETYLGLUCOSAMACYLTRANS__45__RXN \"UDPNACETYLGLUCOSAMACYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__HALOACID__45__DEHALOGENASE__45__RXN \"2-HALOACID-DEHALOGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__HALOACID__45__DEHALOGENASE__45__RXN \"2-HALOACID-DEHALOGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18200 \"RXN-18200\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18200 \"RXN-18200\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22620__45__Red__45__Thioredoxin__47__HYDROGEN__45__PEROXIDE__47____47__Ox__45__Thioredoxin__47__WATER__46__56__46__ \"RXN-22620-Red-Thioredoxin/HYDROGEN-PEROXIDE//Ox-Thioredoxin/WATER.56.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22620__45__Red__45__Thioredoxin__47__HYDROGEN__45__PEROXIDE__47____47__Ox__45__Thioredoxin__47__WATER__46__56__46__ \"RXN-22620-Red-Thioredoxin/HYDROGEN-PEROXIDE//Ox-Thioredoxin/WATER.56.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20789 \"RXN-20789\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20789 \"RXN-20789\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14812__45__FRUCTOSE__45__6P__47____47__FRUCTOSE__45__6P__46__25__46__ \"RXN-14812-FRUCTOSE-6P//FRUCTOSE-6P.25.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14812__45__FRUCTOSE__45__6P__47____47__FRUCTOSE__45__6P__46__25__46__ \"RXN-14812-FRUCTOSE-6P//FRUCTOSE-6P.25.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14812 \"RXN-14812\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14812 \"RXN-14812\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22480 \"RXN-22480\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22480 \"RXN-22480\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__6264 \"RXN-6264\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__6264 \"RXN-6264\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22244 \"RXN-22244\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22244 \"RXN-22244\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRAZIN__45__RXN \"PYRAZIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRAZIN__45__RXN \"PYRAZIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__6263 \"RXN-6263\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__6263 \"RXN-6263\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23516 \"RXN-23516\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23516 \"RXN-23516\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__2__46__2__46__15__45__RXN \"3.2.2.15-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__2__46__2__46__15__45__RXN \"3.2.2.15-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__8__46__1__46__11__45__RXN \"3.8.1.11-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__8__46__1__46__11__45__RXN \"3.8.1.11-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTATHIONE__45__PEROXIDASE__45__RXN \"GLUTATHIONE-PEROXIDASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTATHIONE__45__PEROXIDASE__45__RXN \"GLUTATHIONE-PEROXIDASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22285 \"RXN-22285\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22285 \"RXN-22285\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ISPH2__45__RXN \"ISPH2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ISPH2__45__RXN \"ISPH2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14290 \"RXN-14290\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14290 \"RXN-14290\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLU6PDEHYDROG__45__RXN__45__ALPHA__45__GLC__45__6__45__P__47__NADP__47____47__D__45__6__45__P__45__GLUCONO__45__DELTA__45__LACTONE__47__NADPH__47__PROTON__46__61__46__ \"GLU6PDEHYDROG-RXN-ALPHA-GLC-6-P/NADP//D-6-P-GLUCONO-DELTA-LACTONE/NADPH/PROTON.61.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLU6PDEHYDROG__45__RXN__45__ALPHA__45__GLC__45__6__45__P__47__NADP__47____47__D__45__6__45__P__45__GLUCONO__45__DELTA__45__LACTONE__47__NADPH__47__PROTON__46__61__46__ \"GLU6PDEHYDROG-RXN-ALPHA-GLC-6-P/NADP//D-6-P-GLUCONO-DELTA-LACTONE/NADPH/PROTON.61.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLU6PDEHYDROG__45__RXN__45__GLC__45__6__45__P__47__NADP__47____47__D__45__6__45__P__45__GLUCONO__45__DELTA__45__LACTONE__47__NADPH__47__PROTON__46__55__46__ \"GLU6PDEHYDROG-RXN-GLC-6-P/NADP//D-6-P-GLUCONO-DELTA-LACTONE/NADPH/PROTON.55.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLU6PDEHYDROG__45__RXN__45__GLC__45__6__45__P__47__NADP__47____47__D__45__6__45__P__45__GLUCONO__45__DELTA__45__LACTONE__47__NADPH__47__PROTON__46__55__46__ \"GLU6PDEHYDROG-RXN-GLC-6-P/NADP//D-6-P-GLUCONO-DELTA-LACTONE/NADPH/PROTON.55.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLU6PDEHYDROG__45__RXN \"GLU6PDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLU6PDEHYDROG__45__RXN \"GLU6PDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__7__46__13__45__RXN \"2.7.7.13-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__7__46__13__45__RXN \"2.7.7.13-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22616 \"RXN-22616\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22616 \"RXN-22616\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20694 \"RXN-20694\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20694 \"RXN-20694\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22617 \"RXN-22617\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22617 \"RXN-22617\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ANTHRANSYN__45__RXN \"ANTHRANSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ANTHRANSYN__45__RXN \"ANTHRANSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CDP__45__GLUCOSE__45__46__45__DEHYDRATASE__45__RXN \"CDP-GLUCOSE-46-DEHYDRATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CDP__45__GLUCOSE__45__46__45__DEHYDRATASE__45__RXN \"CDP-GLUCOSE-46-DEHYDRATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__7__46__33__45__RXN \"2.7.7.33-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__7__46__33__45__RXN \"2.7.7.33-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5408 \"RXN0-5408\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5408 \"RXN0-5408\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22618 \"RXN-22618\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22618 \"RXN-22618\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__884 \"RXN0-884\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__884 \"RXN0-884\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__267__45__Red__45__Thioredoxin__47__HYDROGEN__45__PEROXIDE__47____47__Ox__45__Thioredoxin__47__WATER__46__56__46__ \"RXN0-267-Red-Thioredoxin/HYDROGEN-PEROXIDE//Ox-Thioredoxin/WATER.56.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__267__45__Red__45__Thioredoxin__47__HYDROGEN__45__PEROXIDE__47____47__Ox__45__Thioredoxin__47__WATER__46__56__46__ \"RXN0-267-Red-Thioredoxin/HYDROGEN-PEROXIDE//Ox-Thioredoxin/WATER.56.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7253 \"RXN-7253\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7253 \"RXN-7253\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17881 \"RXN-17881\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17881 \"RXN-17881\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1OR__45__4__45__MONOPHOSPHATASE__45__RXN \"MYO-INOSITOL-1OR-4-MONOPHOSPHATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MYO__45__INOSITOL__45__1OR__45__4__45__MONOPHOSPHATASE__45__RXN \"MYO-INOSITOL-1OR-4-MONOPHOSPHATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__1__46__148__45__RXN \"2.7.1.148-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__1__46__148__45__RXN \"2.7.1.148-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19953 \"RXN-19953\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19953 \"RXN-19953\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5256 \"RXN0-5256\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5256 \"RXN0-5256\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__77__45__RXN \"2.1.1.77-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__77__45__RXN \"2.1.1.77-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__11__46__1__46__15__45__RXN \"1.11.1.15-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__11__46__1__46__15__45__RXN \"1.11.1.15-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19022 \"RXN-19022\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19022 \"RXN-19022\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ATPSYN__45__RXN \"ATPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ATPSYN__45__RXN \"ATPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17889 \"RXN-17889\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17889 \"RXN-17889\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DTDPGLUCDEHYDRAT__45__RXN \"DTDPGLUCDEHYDRAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DTDPGLUCDEHYDRAT__45__RXN \"DTDPGLUCDEHYDRAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__1381 \"RXN-1381\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__1381 \"RXN-1381\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPMANDEHYDRA__45__RXN \"GDPMANDEHYDRA-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPMANDEHYDRA__45__RXN \"GDPMANDEHYDRA-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NAD__45__KIN__45__RXN \"NAD-KIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NAD__45__KIN__45__RXN \"NAD-KIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__743 \"RXN0-743\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__743 \"RXN0-743\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22315 \"RXN-22315\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22315 \"RXN-22315\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CTPSYN__45__RXN \"CTPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CTPSYN__45__RXN \"CTPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__131__45__RXN \"2.1.1.131-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__131__45__RXN \"2.1.1.131-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__366 \"RXN0-366\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__366 \"RXN0-366\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction D__45__ALANINE__45__AMINOTRANSFERASE__45__RXN \"D-ALANINE-AMINOTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction D__45__ALANINE__45__AMINOTRANSFERASE__45__RXN \"D-ALANINE-AMINOTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18203 \"RXN-18203\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18203 \"RXN-18203\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__742 \"RXN0-742\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__742 \"RXN0-742\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__130__45__RXN \"2.1.1.130-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__130__45__RXN \"2.1.1.130-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction H2PTERIDINEPYROPHOSPHOKIN__45__RXN \"H2PTERIDINEPYROPHOSPHOKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction H2PTERIDINEPYROPHOSPHOKIN__45__RXN \"H2PTERIDINEPYROPHOSPHOKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__133__45__RXN \"2.1.1.133-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__133__45__RXN \"2.1.1.133-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12540 \"RXN-12540\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12540 \"RXN-12540\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20739 \"RXN-20739\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20739 \"RXN-20739\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__11__46__5__45__RXN \"3.4.11.5-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__11__46__5__45__RXN \"3.4.11.5-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15733 \"RXN-15733\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15733 \"RXN-15733\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SERINE__45__O__45__ACETTRAN__45__RXN \"SERINE-O-ACETTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SERINE__45__O__45__ACETTRAN__45__RXN \"SERINE-O-ACETTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20190 \"RXN-20190\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20190 \"RXN-20190\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20743 \"RXN-20743\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20743 \"RXN-20743\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__141 \"TRANS-RXN-141\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__141 \"TRANS-RXN-141\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDROOROTATE__45__DEHYDROGENASE__45__RXN \"DIHYDROOROTATE-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDROOROTATE__45__DEHYDROGENASE__45__RXN \"DIHYDROOROTATE-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23223 \"RXN-23223\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23223 \"RXN-23223\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GCVT__45__RXN \"GCVT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GCVT__45__RXN \"GCVT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CDPREDUCT__45__RXN \"CDPREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CDPREDUCT__45__RXN \"CDPREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ARYLESTERASE__45__RXN \"ARYLESTERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ARYLESTERASE__45__RXN \"ARYLESTERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SPERMIDINESYN__45__RXN \"SPERMIDINESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SPERMIDINESYN__45__RXN \"SPERMIDINESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GCVP__45__RXN \"GCVP-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GCVP__45__RXN \"GCVP-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6554 \"RXN0-6554\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6554 \"RXN0-6554\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18207 \"RXN-18207\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18207 \"RXN-18207\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22388 \"RXN-22388\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22388 \"RXN-22388\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14728 \"RXN-14728\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14728 \"RXN-14728\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20738 \"RXN-20738\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20738 \"RXN-20738\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6491 \"RXN0-6491\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6491 \"RXN0-6491\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5507 \"RXN0-5507\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5507 \"RXN0-5507\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _6PGLUCONOLACT__45__RXN \"6PGLUCONOLACT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _6PGLUCONOLACT__45__RXN \"6PGLUCONOLACT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALANINE__45__AMINOTRANSFERASE__45__RXN \"ALANINE-AMINOTRANSFERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALANINE__45__AMINOTRANSFERASE__45__RXN \"ALANINE-AMINOTRANSFERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12019 \"RXN-12019\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12019 \"RXN-12019\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13698 \"RXN-13698\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13698 \"RXN-13698\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__3341 \"RXN-3341\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__3341 \"RXN-3341\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1081 \"RXN0-1081\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1081 \"RXN0-1081\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22317 \"RXN-22317\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22317 \"RXN-22317\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17472 \"RXN-17472\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17472 \"RXN-17472\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SEDOBISALDOL__45__RXN \"SEDOBISALDOL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SEDOBISALDOL__45__RXN \"SEDOBISALDOL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALCOHOL__45__DEHYDROG__45__RXN \"ALCOHOL-DEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALCOHOL__45__DEHYDROG__45__RXN \"ALCOHOL-DEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7139 \"RXN0-7139\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7139 \"RXN0-7139\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11588 \"RXN-11588\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11588 \"RXN-11588\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5305 \"RXN0-5305\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5305 \"RXN0-5305\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETALD__45__DEHYDROG__45__RXN \"ACETALD-DEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETALD__45__DEHYDROG__45__RXN \"ACETALD-DEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7192 \"RXN0-7192\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7192 \"RXN0-7192\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7169 \"RXN0-7169\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7169 \"RXN0-7169\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16424 \"RXN-16424\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16424 \"RXN-16424\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10851 \"RXN-10851\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10851 \"RXN-10851\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11181 \"RXN-11181\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11181 \"RXN-11181\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _5__46__4__46__2__46__10__45__RXN \"5.4.2.10-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _5__46__4__46__2__46__10__45__RXN \"5.4.2.10-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17882 \"RXN-17882\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17882 \"RXN-17882\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14196 \"RXN-14196\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14196 \"RXN-14196\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__8__46__1__46__6__45__RXN \"2.8.1.6-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__8__46__1__46__6__45__RXN \"2.8.1.6-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20894 \"RXN-20894\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20894 \"RXN-20894\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1A0__45__6303 \"RXN1A0-6303\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1A0__45__6303 \"RXN1A0-6303\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22735 \"RXN-22735\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22735 \"RXN-22735\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTATHIONE__45__SYN__45__RXN \"GLUTATHIONE-SYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTATHIONE__45__SYN__45__RXN \"GLUTATHIONE-SYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12723 \"RXN-12723\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12723 \"RXN-12723\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRYPTOPHAN__45____45__TRNA__45__LIGASE__45__RXN \"TRYPTOPHAN--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRYPTOPHAN__45____45__TRNA__45__LIGASE__45__RXN \"TRYPTOPHAN--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19138 \"RXN-19138\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19138 \"RXN-19138\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _7KAPSYN__45__RXN \"7KAPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _7KAPSYN__45__RXN \"7KAPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16909 \"RXN-16909\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16909 \"RXN-16909\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__2421 \"RXN0-2421\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__2421 \"RXN0-2421\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__8__46__4__46__12__45__RXN \"1.8.4.12-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__8__46__4__46__12__45__RXN \"1.8.4.12-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14883 \"RXN-14883\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14883 \"RXN-14883\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12722 \"RXN-12722\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12722 \"RXN-12722\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14570 \"RXN-14570\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14570 \"RXN-14570\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDP__45__NACMURALGLDAPLIG__45__RXN \"UDP-NACMURALGLDAPLIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDP__45__NACMURALGLDAPLIG__45__RXN \"UDP-NACMURALGLDAPLIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10665 \"RXN-10665\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10665 \"RXN-10665\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23276 \"RXN-23276\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23276 \"RXN-23276\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10643 \"RXN-10643\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10643 \"RXN-10643\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1342 \"RXN0-1342\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1342 \"RXN0-1342\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11638 \"RXN-11638\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11638 \"RXN-11638\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__1__46__3__46__16__45__RXN \"3.1.3.16-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__1__46__3__46__16__45__RXN \"3.1.3.16-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1ZN3__45__10 \"RXN1ZN3-10\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1ZN3__45__10 \"RXN1ZN3-10\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17883 \"RXN-17883\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17883 \"RXN-17883\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction F16ALDOLASE__45__RXN \"F16ALDOLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction F16ALDOLASE__45__RXN \"F16ALDOLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HOMOCITRATE__45__SYNTHASE__45__RXN \"HOMOCITRATE-SYNTHASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HOMOCITRATE__45__SYNTHASE__45__RXN \"HOMOCITRATE-SYNTHASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16055 \"RXN-16055\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16055 \"RXN-16055\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__302 \"RXN0-302\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__302 \"RXN0-302\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22603 \"RXN-22603\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22603 \"RXN-22603\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1ZN3__45__9 \"RXN1ZN3-9\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1ZN3__45__9 \"RXN1ZN3-9\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTAMATE__45__DEHYDROGENASE__45__RXN \"GLUTAMATE-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTAMATE__45__DEHYDROGENASE__45__RXN \"GLUTAMATE-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DETHIOBIOTIN__45__SYN__45__RXN \"DETHIOBIOTIN-SYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DETHIOBIOTIN__45__SYN__45__RXN \"DETHIOBIOTIN-SYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19916 \"RXN-19916\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19916 \"RXN-19916\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__7__46__60__45__RXN \"2.7.7.60-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__7__46__60__45__RXN \"2.7.7.60-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9223 \"RXN-9223\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9223 \"RXN-9223\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9238 \"RXN-9238\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9238 \"RXN-9238\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OCTAPRENYL__45__4__45__OHBENZOATE__45__DECARBOX__45__RXN \"3-OCTAPRENYL-4-OHBENZOATE-DECARBOX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OCTAPRENYL__45__4__45__OHBENZOATE__45__DECARBOX__45__RXN \"3-OCTAPRENYL-4-OHBENZOATE-DECARBOX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__168__45__RXN \"2.3.1.168-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__168__45__RXN \"2.3.1.168-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__2__46__4__46__4__45__RXN \"1.2.4.4-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__2__46__4__46__4__45__RXN \"1.2.4.4-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18206 \"RXN-18206\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18206 \"RXN-18206\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9231 \"RXN-9231\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9231 \"RXN-9231\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19911 \"RXN-19911\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19911 \"RXN-19911\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBOFLAVINKIN__45__RXN \"RIBOFLAVINKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBOFLAVINKIN__45__RXN \"RIBOFLAVINKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ABC__45__27__45__RXN \"ABC-27-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ABC__45__27__45__RXN \"ABC-27-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7013 \"RXN0-7013\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7013 \"RXN0-7013\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__CH3__45__2__45__OXOBUTANOATE__45__OH__45__CH3__45__XFER__45__RXN \"3-CH3-2-OXOBUTANOATE-OH-CH3-XFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__CH3__45__2__45__OXOBUTANOATE__45__OH__45__CH3__45__XFER__45__RXN \"3-CH3-2-OXOBUTANOATE-OH-CH3-XFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction KDO__45__8PSYNTH__45__RXN \"KDO-8PSYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction KDO__45__8PSYNTH__45__RXN \"KDO-8PSYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14325 \"RXN-14325\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14325 \"RXN-14325\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12816 \"RXN-12816\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12816 \"RXN-12816\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5462 \"RXN0-5462\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5462 \"RXN0-5462\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__44__45__RXN \"2.1.1.44-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__44__45__RXN \"2.1.1.44-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7022 \"RXN0-7022\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7022 \"RXN0-7022\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNQT__45__4168 \"RXNQT-4168\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNQT__45__4168 \"RXNQT-4168\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14437 \"RXN-14437\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14437 \"RXN-14437\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction IPPISOM__45__RXN \"IPPISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction IPPISOM__45__RXN \"IPPISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14438 \"RXN-14438\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14438 \"RXN-14438\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8340 \"RXN-8340\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8340 \"RXN-8340\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2861 \"RXN-2861\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2861 \"RXN-2861\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14431 \"RXN-14431\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14431 \"RXN-14431\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__2361 \"RXN0-2361\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__2361 \"RXN0-2361\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14315 \"RXN-14315\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14315 \"RXN-14315\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MOXXANAU__45__RXN \"MOXXANAU-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MOXXANAU__45__RXN \"MOXXANAU-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10000 \"RXN-10000\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10000 \"RXN-10000\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction IMPCYCLOHYDROLASE__45__RXN \"IMPCYCLOHYDROLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction IMPCYCLOHYDROLASE__45__RXN \"IMPCYCLOHYDROLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CARBPSYN__45__RXN \"CARBPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CARBPSYN__45__RXN \"CARBPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11332__45__ETOH__47__Cytochromes__45__CL__45__Ox__47____47__ACETALD__47__Cytochromes__45__CL__45__Red__47__PROTON__46__58__46__ \"RXN-11332-ETOH/Cytochromes-CL-Ox//ACETALD/Cytochromes-CL-Red/PROTON.58.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11332__45__ETOH__47__Cytochromes__45__CL__45__Ox__47____47__ACETALD__47__Cytochromes__45__CL__45__Red__47__PROTON__46__58__46__ \"RXN-11332-ETOH/Cytochromes-CL-Ox//ACETALD/Cytochromes-CL-Red/PROTON.58.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTATHIONE__45__REDUCT__45__NADPH__45__RXN \"GLUTATHIONE-REDUCT-NADPH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTATHIONE__45__REDUCT__45__NADPH__45__RXN \"GLUTATHIONE-REDUCT-NADPH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__1__46__1__46__151__45__RXN \"2.1.1.151-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__1__46__1__46__151__45__RXN \"2.1.1.151-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17809 \"RXN-17809\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17809 \"RXN-17809\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8668 \"RXN-8668\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8668 \"RXN-8668\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4__46__99__46__1__46__3__45__RXN \"4.99.1.3-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4__46__99__46__1__46__3__45__RXN \"4.99.1.3-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PANTOATE__45__BETA__45__ALANINE__45__LIG__45__RXN \"PANTOATE-BETA-ALANINE-LIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PANTOATE__45__BETA__45__ALANINE__45__LIG__45__RXN \"PANTOATE-BETA-ALANINE-LIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18708 \"RXN-18708\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18708 \"RXN-18708\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15745 \"RXN-15745\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15745 \"RXN-15745\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GCVMULTI__45__RXN \"GCVMULTI-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GCVMULTI__45__RXN \"GCVMULTI-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12508 \"RXN-12508\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12508 \"RXN-12508\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8766 \"RXN-8766\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8766 \"RXN-8766\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12583 \"RXN-12583\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12583 \"RXN-12583\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRIOSEPISOMERIZATION__45__RXN \"TRIOSEPISOMERIZATION-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRIOSEPISOMERIZATION__45__RXN \"TRIOSEPISOMERIZATION-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14025 \"RXN-14025\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14025 \"RXN-14025\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1134 \"RXN0-1134\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1134 \"RXN0-1134\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18510 \"RXN-18510\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18510 \"RXN-18510\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8762 \"RXN-8762\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8762 \"RXN-8762\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R310__45__RXN \"R310-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R310__45__RXN \"R310-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14026 \"RXN-14026\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14026 \"RXN-14026\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14014__45__DELTA1__45__PIPERIDEINE__45__2__45__6__45__DICARBOXYLATE__47__NADP__47__WATER__47____47__CPD__45__14443__47__NADPH__47__PROTON__46__72__46__ \"RXN-14014-DELTA1-PIPERIDEINE-2-6-DICARBOXYLATE/NADP/WATER//CPD-14443/NADPH/PROTON.72.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14014__45__DELTA1__45__PIPERIDEINE__45__2__45__6__45__DICARBOXYLATE__47__NADP__47__WATER__47____47__CPD__45__14443__47__NADPH__47__PROTON__46__72__46__ \"RXN-14014-DELTA1-PIPERIDEINE-2-6-DICARBOXYLATE/NADP/WATER//CPD-14443/NADPH/PROTON.72.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14014__45__DELTA1__45__PIPERIDEINE__45__2__45__6__45__DICARBOXYLATE__47__NAD__47__WATER__47____47__CPD__45__14443__47__NADH__47__PROTON__46__70__46__ \"RXN-14014-DELTA1-PIPERIDEINE-2-6-DICARBOXYLATE/NAD/WATER//CPD-14443/NADH/PROTON.70.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14014__45__DELTA1__45__PIPERIDEINE__45__2__45__6__45__DICARBOXYLATE__47__NAD__47__WATER__47____47__CPD__45__14443__47__NADH__47__PROTON__46__70__46__ \"RXN-14014-DELTA1-PIPERIDEINE-2-6-DICARBOXYLATE/NAD/WATER//CPD-14443/NADH/PROTON.70.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8761 \"RXN-8761\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8761 \"RXN-8761\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPDECARBOX__45__RXN \"ASPDECARBOX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPDECARBOX__45__RXN \"ASPDECARBOX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7609 \"RXN-7609\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7609 \"RXN-7609\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8768 \"RXN-8768\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8768 \"RXN-8768\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17753 \"RXN-17753\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17753 \"RXN-17753\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction XMPXAN__45__RXN \"XMPXAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction XMPXAN__45__RXN \"XMPXAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8765 \"RXN-8765\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8765 \"RXN-8765\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12626 \"RXN-12626\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12626 \"RXN-12626\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPGLCNACEPIM__45__RXN \"UDPGLCNACEPIM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPGLCNACEPIM__45__RXN \"UDPGLCNACEPIM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LEUCINE__45____45__TRNA__45__LIGASE__45__RXN \"LEUCINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LEUCINE__45____45__TRNA__45__LIGASE__45__RXN \"LEUCINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15915 \"RXN-15915\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15915 \"RXN-15915\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8763 \"RXN-8763\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8763 \"RXN-8763\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PROLINE__45____45__TRNA__45__LIGASE__45__RXN \"PROLINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PROLINE__45____45__TRNA__45__LIGASE__45__RXN \"PROLINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__1 \"TRANS-RXN2PP0-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__1 \"TRANS-RXN2PP0-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__5841 \"RXN-5841\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__5841 \"RXN-5841\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5226 \"RXN0-5226\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5226 \"RXN0-5226\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22647 \"RXN-22647\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22647 \"RXN-22647\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21432 \"RXN-21432\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21432 \"RXN-21432\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14227 \"RXN-14227\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14227 \"RXN-14227\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _6__46__3__46__5__46__7__45__RXN \"6.3.5.7-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _6__46__3__46__5__46__7__45__RXN \"6.3.5.7-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5268 \"RXN0-5268\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5268 \"RXN0-5268\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5266 \"RXN0-5266\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5266 \"RXN0-5266\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__6883 \"RXN-6883\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__6883 \"RXN-6883\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17943 \"RXN-17943\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17943 \"RXN-17943\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__2__46__1__46__52__45__RXN \"3.2.1.52-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__2__46__1__46__52__45__RXN \"3.2.1.52-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THYMIDYLATESYN__45__RXN \"THYMIDYLATESYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THYMIDYLATESYN__45__RXN \"THYMIDYLATESYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20993 \"RXN-20993\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20993 \"RXN-20993\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction URACIL__45__PRIBOSYLTRANS__45__RXN \"URACIL-PRIBOSYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction URACIL__45__PRIBOSYLTRANS__45__RXN \"URACIL-PRIBOSYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20992 \"RXN-20992\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20992 \"RXN-20992\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__2 \"TRANS-RXN2PP0-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__2 \"TRANS-RXN2PP0-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13997 \"RXN-13997\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13997 \"RXN-13997\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9771 \"RXN-9771\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9771 \"RXN-9771\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21392 \"RXN-21392\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21392 \"RXN-21392\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22591 \"RXN-22591\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22591 \"RXN-22591\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2882 \"RXN-2882\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2882 \"RXN-2882\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14260 \"RXN-14260\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14260 \"RXN-14260\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14260__45__MALTOPENTAOSE__47__GLC__47____47__MALTOTETRAOSE__47__MALTOSE__46__41__46__ \"RXN-14260-MALTOPENTAOSE/GLC//MALTOTETRAOSE/MALTOSE.41.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14260__45__MALTOPENTAOSE__47__GLC__47____47__MALTOTETRAOSE__47__MALTOSE__46__41__46__ \"RXN-14260-MALTOPENTAOSE/GLC//MALTOTETRAOSE/MALTOSE.41.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14260__45__MALTOPENTAOSE__47__ALPHA__45__GLUCOSE__47____47__MALTOTETRAOSE__47__MALTOSE__46__51__46__ \"RXN-14260-MALTOPENTAOSE/ALPHA-GLUCOSE//MALTOTETRAOSE/MALTOSE.51.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14260__45__MALTOPENTAOSE__47__ALPHA__45__GLUCOSE__47____47__MALTOTETRAOSE__47__MALTOSE__46__51__46__ \"RXN-14260-MALTOPENTAOSE/ALPHA-GLUCOSE//MALTOTETRAOSE/MALTOSE.51.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22612 \"RXN-22612\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22612 \"RXN-22612\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8767 \"RXN-8767\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8767 \"RXN-8767\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8344 \"RXN-8344\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8344 \"RXN-8344\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12625 \"RXN-12625\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12625 \"RXN-12625\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14261 \"RXN-14261\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14261 \"RXN-14261\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14261__45__MALTOHEXAOSE__47__GLC__47____47__MALTOPENTAOSE__47__MALTOSE__46__40__46__ \"RXN-14261-MALTOHEXAOSE/GLC//MALTOPENTAOSE/MALTOSE.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14261__45__MALTOHEXAOSE__47__GLC__47____47__MALTOPENTAOSE__47__MALTOSE__46__40__46__ \"RXN-14261-MALTOHEXAOSE/GLC//MALTOPENTAOSE/MALTOSE.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14261__45__MALTOHEXAOSE__47__ALPHA__45__GLUCOSE__47____47__MALTOPENTAOSE__47__MALTOSE__46__50__46__ \"RXN-14261-MALTOHEXAOSE/ALPHA-GLUCOSE//MALTOPENTAOSE/MALTOSE.50.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14261__45__MALTOHEXAOSE__47__ALPHA__45__GLUCOSE__47____47__MALTOPENTAOSE__47__MALTOSE__46__50__46__ \"RXN-14261-MALTOHEXAOSE/ALPHA-GLUCOSE//MALTOPENTAOSE/MALTOSE.50.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _5__46__4__46__1__46__2__45__RXN \"5.4.1.2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _5__46__4__46__1__46__2__45__RXN \"5.4.1.2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11479 \"RXN-11479\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11479 \"RXN-11479\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HYPOXANPRIBOSYLTRAN__45__RXN \"HYPOXANPRIBOSYLTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HYPOXANPRIBOSYLTRAN__45__RXN \"HYPOXANPRIBOSYLTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20081 \"RXN-20081\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20081 \"RXN-20081\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20080 \"RXN-20080\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20080 \"RXN-20080\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GUANPRIBOSYLTRAN__45__RXN \"GUANPRIBOSYLTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GUANPRIBOSYLTRAN__45__RXN \"GUANPRIBOSYLTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _6__46__3__46__5__46__6__45__RXN \"6.3.5.6-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _6__46__3__46__5__46__6__45__RXN \"6.3.5.6-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__413 \"TRANS-RXN-413\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__413 \"TRANS-RXN-413\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8771 \"RXN-8771\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8771 \"RXN-8771\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12241 \"RXN-12241\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12241 \"RXN-12241\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GSADENYLATION__45__RXN \"GSADENYLATION-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GSADENYLATION__45__RXN \"GSADENYLATION-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HEMN__45__RXN \"HEMN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HEMN__45__RXN \"HEMN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17393 \"RXN-17393\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17393 \"RXN-17393\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12473 \"RXN-12473\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12473 \"RXN-12473\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction OROPRIBTRANS__45__RXN \"OROPRIBTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction OROPRIBTRANS__45__RXN \"OROPRIBTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8342 \"RXN-8342\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8342 \"RXN-8342\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17392 \"RXN-17392\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17392 \"RXN-17392\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__3 \"TRANS-RXN2PP0-3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__3 \"TRANS-RXN2PP0-3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18232 \"RXN-18232\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18232 \"RXN-18232\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5190 \"RXN0-5190\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5190 \"RXN0-5190\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15124 \"RXN-15124\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15124 \"RXN-15124\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13313 \"RXN-13313\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13313 \"RXN-13313\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22439 \"RXN-22439\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22439 \"RXN-22439\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction COBINPGUANYLYLTRANS__45__RXN \"COBINPGUANYLYLTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction COBINPGUANYLYLTRANS__45__RXN \"COBINPGUANYLYLTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CHER__45__RXN \"CHER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CHER__45__RXN \"CHER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTIDEAM2__45__RXN \"CYTIDEAM2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTIDEAM2__45__RXN \"CYTIDEAM2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN1YI0__45__22__45__Plastocyanin__45__Reduced__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__Oxidized__45__Plastocyanins__47__PROTON__47__WATER__46__81__46__ \"RXN1YI0-22-Plastocyanin-Reduced/OXYGEN-MOLECULE/PROTON//Oxidized-Plastocyanins/PROTON/WATER.81.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN1YI0__45__22__45__Plastocyanin__45__Reduced__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__Oxidized__45__Plastocyanins__47__PROTON__47__WATER__46__81__46__ \"RXN1YI0-22-Plastocyanin-Reduced/OXYGEN-MOLECULE/PROTON//Oxidized-Plastocyanins/PROTON/WATER.81.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTIDEAM__45__RXN \"CYTIDEAM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTIDEAM__45__RXN \"CYTIDEAM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RIBOFLAVIN__45__SYN__45__RXN \"RIBOFLAVIN-SYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RIBOFLAVIN__45__SYN__45__RXN \"RIBOFLAVIN-SYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction METHIONINE__45____45__TRNA__45__LIGASE__45__RXN \"METHIONINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction METHIONINE__45____45__TRNA__45__LIGASE__45__RXN \"METHIONINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__a__45__reduced__45__NrfB__45__protein__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__an__45__oxidized__45__NrfB__45__protein__47__PROTON__47__WATER__46__85__46__ \"CYTOCHROME-C-OXIDASE-RXN-a-reduced-NrfB-protein/PROTON/OXYGEN-MOLECULE//an-oxidized-NrfB-protein/PROTON/WATER.85.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__a__45__reduced__45__NrfB__45__protein__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__an__45__oxidized__45__NrfB__45__protein__47__PROTON__47__WATER__46__85__46__ \"CYTOCHROME-C-OXIDASE-RXN-a-reduced-NrfB-protein/PROTON/OXYGEN-MOLECULE//an-oxidized-NrfB-protein/PROTON/WATER.85.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__NapC__45__proteins__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__NapC__45__proteins__47__PROTON__47__WATER__46__82__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-NapC-proteins/PROTON/OXYGEN-MOLECULE//Oxidized-NapC-proteins/PROTON/WATER.82.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__NapC__45__proteins__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__NapC__45__proteins__47__PROTON__47__WATER__46__82__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-NapC-proteins/PROTON/OXYGEN-MOLECULE//Oxidized-NapC-proteins/PROTON/WATER.82.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__cytochromes__45__C4__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__cytochromes__45__C4__47__PROTON__47__WATER__46__84__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-cytochromes-C4/PROTON/OXYGEN-MOLECULE//Oxidized-cytochromes-C4/PROTON/WATER.84.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__cytochromes__45__C4__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__cytochromes__45__C4__47__PROTON__47__WATER__46__84__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-cytochromes-C4/PROTON/OXYGEN-MOLECULE//Oxidized-cytochromes-C4/PROTON/WATER.84.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__CycA1__45__cytochromes__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__CycA1__45__cytochromes__47__PROTON__47__WATER__46__90__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-CycA1-cytochromes/PROTON/OXYGEN-MOLECULE//Oxidized-CycA1-cytochromes/PROTON/WATER.90.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__CycA1__45__cytochromes__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__CycA1__45__cytochromes__47__PROTON__47__WATER__46__90__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-CycA1-cytochromes/PROTON/OXYGEN-MOLECULE//Oxidized-CycA1-cytochromes/PROTON/WATER.90.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Cytochromes__45__CL__45__Red__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Cytochromes__45__CL__45__Ox__47__PROTON__47__WATER__46__74__46__ \"CYTOCHROME-C-OXIDASE-RXN-Cytochromes-CL-Red/PROTON/OXYGEN-MOLECULE//Cytochromes-CL-Ox/PROTON/WATER.74.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Cytochromes__45__CL__45__Red__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Cytochromes__45__CL__45__Ox__47__PROTON__47__WATER__46__74__46__ \"CYTOCHROME-C-OXIDASE-RXN-Cytochromes-CL-Red/PROTON/OXYGEN-MOLECULE//Cytochromes-CL-Ox/PROTON/WATER.74.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__Cytochromes__45__C6__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__Cytochromes__45__C6__47__PROTON__47__WATER__46__84__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-Cytochromes-C6/PROTON/OXYGEN-MOLECULE//Oxidized-Cytochromes-C6/PROTON/WATER.84.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Reduced__45__Cytochromes__45__C6__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Oxidized__45__Cytochromes__45__C6__47__PROTON__47__WATER__46__84__46__ \"CYTOCHROME-C-OXIDASE-RXN-Reduced-Cytochromes-C6/PROTON/OXYGEN-MOLECULE//Oxidized-Cytochromes-C6/PROTON/WATER.84.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Cytochromes__45__C__45__Reduced__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Cytochromes__45__C__45__Oxidized__47__PROTON__47__WATER__46__82__46__ \"CYTOCHROME-C-OXIDASE-RXN-Cytochromes-C-Reduced/PROTON/OXYGEN-MOLECULE//Cytochromes-C-Oxidized/PROTON/WATER.82.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction CYTOCHROME__45__C__45__OXIDASE__45__RXN__45__Cytochromes__45__C__45__Reduced__47__PROTON__47__OXYGEN__45__MOLECULE__47____47__Cytochromes__45__C__45__Oxidized__47__PROTON__47__WATER__46__82__46__ \"CYTOCHROME-C-OXIDASE-RXN-Cytochromes-C-Reduced/PROTON/OXYGEN-MOLECULE//Cytochromes-C-Oxidized/PROTON/WATER.82.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20084 \"RXN-20084\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20084 \"RXN-20084\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NAD__45__SYNTH__45__NH3__45__RXN \"NAD-SYNTH-NH3-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NAD__45__SYNTH__45__NH3__45__RXN \"NAD-SYNTH-NH3-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21237__45__Reduced__45__Cytochromes__45__C6__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__Oxidized__45__Cytochromes__45__C6__47__PROTON__47__WATER__46__84__46__ \"RXN-21237-Reduced-Cytochromes-C6/OXYGEN-MOLECULE/PROTON//Oxidized-Cytochromes-C6/PROTON/WATER.84.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21237__45__Reduced__45__Cytochromes__45__C6__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__Oxidized__45__Cytochromes__45__C6__47__PROTON__47__WATER__46__84__46__ \"RXN-21237-Reduced-Cytochromes-C6/OXYGEN-MOLECULE/PROTON//Oxidized-Cytochromes-C6/PROTON/WATER.84.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__4 \"TRANS-RXN2PP0-4\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__4 \"TRANS-RXN2PP0-4\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DCTP__45__DEAM__45__RXN \"DCTP-DEAM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DCTP__45__DEAM__45__RXN \"DCTP-DEAM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7567 \"RXN-7567\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7567 \"RXN-7567\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__4361 \"RXN0-4361\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__4361 \"RXN0-4361\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21299__45__General__45__Protein__45__Substrates__47__ATP__47__WATER__47____47__Peptides__45__holder__47__ADP__47__Pi__47__PROTON__46__68__46__ \"RXN-21299-General-Protein-Substrates/ATP/WATER//Peptides-holder/ADP/Pi/PROTON.68.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21299__45__General__45__Protein__45__Substrates__47__ATP__47__WATER__47____47__Peptides__45__holder__47__ADP__47__Pi__47__PROTON__46__68__46__ \"RXN-21299-General-Protein-Substrates/ATP/WATER//Peptides-holder/ADP/Pi/PROTON.68.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__745 \"RXN-745\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__745 \"RXN-745\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXNN__45__404 \"RXNN-404\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXNN__45__404 \"RXNN-404\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16910 \"RXN-16910\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16910 \"RXN-16910\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13202 \"RXN-13202\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13202 \"RXN-13202\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12353 \"RXN-12353\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12353 \"RXN-12353\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R344__45__RXN \"R344-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R344__45__RXN \"R344-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2DNV__45__43 \"RXN2DNV-43\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2DNV__45__43 \"RXN2DNV-43\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22002 \"RXN-22002\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22002 \"RXN-22002\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction BIOTINLIG__45__RXN \"BIOTINLIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction BIOTINLIG__45__RXN \"BIOTINLIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22643 \"RXN-22643\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22643 \"RXN-22643\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIHYDROFOLATESYNTH__45__RXN \"DIHYDROFOLATESYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIHYDROFOLATESYNTH__45__RXN \"DIHYDROFOLATESYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17026 \"RXN-17026\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17026 \"RXN-17026\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__5 \"TRANS-RXN2PP0-5\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__5 \"TRANS-RXN2PP0-5\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20082 \"RXN-20082\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20082 \"RXN-20082\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PANTOTHENATE__45__KIN__45__RXN \"PANTOTHENATE-KIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PANTOTHENATE__45__KIN__45__RXN \"PANTOTHENATE-KIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__11__46__2__45__RXN \"3.4.11.2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__11__46__2__45__RXN \"3.4.11.2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17823 \"RXN-17823\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17823 \"RXN-17823\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21228 \"RXN-21228\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21228 \"RXN-21228\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7187 \"RXN0-7187\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7187 \"RXN0-7187\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSMANMUT__45__RXN \"PHOSMANMUT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSMANMUT__45__RXN \"PHOSMANMUT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17900 \"RXN-17900\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17900 \"RXN-17900\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PROTEIN__45__KINASE__45__RXN \"PROTEIN-KINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PROTEIN__45__KINASE__45__RXN \"PROTEIN-KINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUTAMIDOTRANS__45__RXN \"GLUTAMIDOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUTAMIDOTRANS__45__RXN \"GLUTAMIDOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20459 \"RXN-20459\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20459 \"RXN-20459\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14136 \"RXN-14136\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14136 \"RXN-14136\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20556 \"RXN-20556\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20556 \"RXN-20556\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1241 \"RXN0-1241\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1241 \"RXN0-1241\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11574 \"RXN-11574\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11574 \"RXN-11574\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21021 \"RXN-21021\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21021 \"RXN-21021\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10708 \"RXN-10708\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10708 \"RXN-10708\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14160 \"RXN-14160\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14160 \"RXN-14160\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIOESTER__45__RXN__45__PROPIONYL__45__COA__47__WATER__47____47__PROPIONATE__47__CO__45__A__47__PROTON__46__44__46__ \"THIOESTER-RXN-PROPIONYL-COA/WATER//PROPIONATE/CO-A/PROTON.44.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIOESTER__45__RXN__45__PROPIONYL__45__COA__47__WATER__47____47__PROPIONATE__47__CO__45__A__47__PROTON__46__44__46__ \"THIOESTER-RXN-PROPIONYL-COA/WATER//PROPIONATE/CO-A/PROTON.44.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14073 \"RXN-14073\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14073 \"RXN-14073\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22697 \"RXN-22697\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22697 \"RXN-22697\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACYL__45__COA__45__HYDROLASE__45__RXN__45__PROPIONYL__45__COA__47__WATER__47____47__PROPIONATE__47__CO__45__A__47__PROTON__46__44__46__ \"ACYL-COA-HYDROLASE-RXN-PROPIONYL-COA/WATER//PROPIONATE/CO-A/PROTON.44.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACYL__45__COA__45__HYDROLASE__45__RXN__45__PROPIONYL__45__COA__47__WATER__47____47__PROPIONATE__47__CO__45__A__47__PROTON__46__44__46__ \"ACYL-COA-HYDROLASE-RXN-PROPIONYL-COA/WATER//PROPIONATE/CO-A/PROTON.44.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10662 \"RXN-10662\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10662 \"RXN-10662\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11838 \"RXN-11838\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11838 \"RXN-11838\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18840 \"RXN-18840\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18840 \"RXN-18840\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SHIKIMATE__45__KINASE__45__RXN \"SHIKIMATE-KINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SHIKIMATE__45__KINASE__45__RXN \"SHIKIMATE-KINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11813 \"RXN-11813\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11813 \"RXN-11813\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12621 \"RXN-12621\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12621 \"RXN-12621\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SUCCINATE__45__SEMIALDEHYDE__45__DEHYDROGENASE__45__RXN \"SUCCINATE-SEMIALDEHYDE-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SUCCINATE__45__SEMIALDEHYDE__45__DEHYDROGENASE__45__RXN \"SUCCINATE-SEMIALDEHYDE-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18707 \"RXN-18707\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18707 \"RXN-18707\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21565 \"RXN-21565\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21565 \"RXN-21565\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8992 \"RXN-8992\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8992 \"RXN-8992\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11841 \"RXN-11841\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11841 \"RXN-11841\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__748 \"RXN0-748\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__748 \"RXN0-748\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__OCTAPRENYL__45__6__45__METHOXYPHENOL__45__HYDROX__45__RXN \"2-OCTAPRENYL-6-METHOXYPHENOL-HYDROX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__OCTAPRENYL__45__6__45__METHOXYPHENOL__45__HYDROX__45__RXN \"2-OCTAPRENYL-6-METHOXYPHENOL-HYDROX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DCTP__45__PYROPHOSPHATASE__45__RXN \"DCTP-PYROPHOSPHATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DCTP__45__PYROPHOSPHATASE__45__RXN \"DCTP-PYROPHOSPHATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYC3PDEHYDROGBIOSYN__45__RXN__45__GLYCEROL__45__3P__47__NADP__47____47__DIHYDROXY__45__ACETONE__45__PHOSPHATE__47__NADPH__47__PROTON__46__59__46__ \"GLYC3PDEHYDROGBIOSYN-RXN-GLYCEROL-3P/NADP//DIHYDROXY-ACETONE-PHOSPHATE/NADPH/PROTON.59.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYC3PDEHYDROGBIOSYN__45__RXN__45__GLYCEROL__45__3P__47__NADP__47____47__DIHYDROXY__45__ACETONE__45__PHOSPHATE__47__NADPH__47__PROTON__46__59__46__ \"GLYC3PDEHYDROGBIOSYN-RXN-GLYCEROL-3P/NADP//DIHYDROXY-ACETONE-PHOSPHATE/NADPH/PROTON.59.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYC3PDEHYDROGBIOSYN__45__RXN__45__GLYCEROL__45__3P__47__NAD__47____47__DIHYDROXY__45__ACETONE__45__PHOSPHATE__47__NADH__47__PROTON__46__57__46__ \"GLYC3PDEHYDROGBIOSYN-RXN-GLYCEROL-3P/NAD//DIHYDROXY-ACETONE-PHOSPHATE/NADH/PROTON.57.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYC3PDEHYDROGBIOSYN__45__RXN__45__GLYCEROL__45__3P__47__NAD__47____47__DIHYDROXY__45__ACETONE__45__PHOSPHATE__47__NADH__47__PROTON__46__57__46__ \"GLYC3PDEHYDROGBIOSYN-RXN-GLYCEROL-3P/NAD//DIHYDROXY-ACETONE-PHOSPHATE/NADH/PROTON.57.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__722 \"RXN0-722\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__722 \"RXN0-722\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13179 \"RXN-13179\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13179 \"RXN-13179\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11839 \"RXN-11839\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11839 \"RXN-11839\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRIMSYN1__45__RXN \"PYRIMSYN1-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRIMSYN1__45__RXN \"PYRIMSYN1-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TYROSINE__45____45__TRNA__45__LIGASE__45__RXN \"TYROSINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TYROSINE__45____45__TRNA__45__LIGASE__45__RXN \"TYROSINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENYL__45__KIN__45__RXN \"ADENYL-KIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENYL__45__KIN__45__RXN \"ADENYL-KIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7068 \"RXN0-7068\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7068 \"RXN0-7068\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__ISOPROPYLMALISOM__45__RXN \"3-ISOPROPYLMALISOM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__ISOPROPYLMALISOM__45__RXN \"3-ISOPROPYLMALISOM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__308 \"RXN0-308\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__308 \"RXN0-308\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6366 \"RXN0-6366\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6366 \"RXN0-6366\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18710 \"RXN-18710\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18710 \"RXN-18710\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13163 \"RXN-13163\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13163 \"RXN-13163\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21080 \"RXN-21080\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21080 \"RXN-21080\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18209 \"RXN-18209\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18209 \"RXN-18209\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSGLYPHOS__45__RXN \"PHOSGLYPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSGLYPHOS__45__RXN \"PHOSGLYPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6950 \"RXN0-6950\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6950 \"RXN0-6950\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPARTATE__45__SEMIALDEHYDE__45__DEHYDROGENASE__45__RXN \"ASPARTATE-SEMIALDEHYDE-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPARTATE__45__SEMIALDEHYDE__45__DEHYDROGENASE__45__RXN \"ASPARTATE-SEMIALDEHYDE-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__7__46__1__46__121__45__RXN \"2.7.1.121-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__7__46__1__46__121__45__RXN \"2.7.1.121-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11852 \"RXN-11852\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11852 \"RXN-11852\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETYLORNTRANSAM__45__RXN \"ACETYLORNTRANSAM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETYLORNTRANSAM__45__RXN \"ACETYLORNTRANSAM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9787 \"RXN-9787\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9787 \"RXN-9787\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__2__46__1__46__17__45__RXN \"3.2.1.17-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__2__46__1__46__17__45__RXN \"3.2.1.17-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21560 \"RXN-21560\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21560 \"RXN-21560\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PGLYCDEHYDROG__45__RXN \"PGLYCDEHYDROG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PGLYCDEHYDROG__45__RXN \"PGLYCDEHYDROG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20037 \"RXN-20037\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20037 \"RXN-20037\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPMANMANHYDRO__45__RXN__45__GDP__45__MANNOSE__47__WATER__47____47__CPD__45__12601__47__GDP__47__PROTON__46__40__46__ \"GDPMANMANHYDRO-RXN-GDP-MANNOSE/WATER//CPD-12601/GDP/PROTON.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPMANMANHYDRO__45__RXN__45__GDP__45__MANNOSE__47__WATER__47____47__CPD__45__12601__47__GDP__47__PROTON__46__40__46__ \"GDPMANMANHYDRO-RXN-GDP-MANNOSE/WATER//CPD-12601/GDP/PROTON.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPMANMANHYDRO__45__RXN__45__GDP__45__MANNOSE__47__WATER__47____47__CPD__45__13559__47__GDP__47__PROTON__46__40__46__ \"GDPMANMANHYDRO-RXN-GDP-MANNOSE/WATER//CPD-13559/GDP/PROTON.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPMANMANHYDRO__45__RXN__45__GDP__45__MANNOSE__47__WATER__47____47__CPD__45__13559__47__GDP__47__PROTON__46__40__46__ \"GDPMANMANHYDRO-RXN-GDP-MANNOSE/WATER//CPD-13559/GDP/PROTON.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GDPMANMANHYDRO__45__RXN \"GDPMANMANHYDRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GDPMANMANHYDRO__45__RXN \"GDPMANMANHYDRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19774 \"RXN-19774\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19774 \"RXN-19774\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__17363 \"RXN-17363\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__17363 \"RXN-17363\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22698 \"RXN-22698\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22698 \"RXN-22698\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12588 \"RXN-12588\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12588 \"RXN-12588\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1061 \"RXN0-1061\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1061 \"RXN0-1061\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C0__45__H0__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C2__45__H4__47__CARBON__45__DIOXIDE__47__ACP__46__118__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C0-H0/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C2-H4/CARBON-DIOXIDE/ACP.118.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C0__45__H0__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C2__45__H4__47__CARBON__45__DIOXIDE__47__ACP__46__118__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C0-H0/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C2-H4/CARBON-DIOXIDE/ACP.118.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C2__45__H4__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C4__45__H8__47__CARBON__45__DIOXIDE__47__ACP__46__118__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C2-H4/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C4-H8/CARBON-DIOXIDE/ACP.118.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C2__45__H4__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C4__45__H8__47__CARBON__45__DIOXIDE__47__ACP__46__118__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C2-H4/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C4-H8/CARBON-DIOXIDE/ACP.118.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__Octanoyl__45__ACPs__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C6__45__H12__47__CARBON__45__DIOXIDE__47__ACP__46__88__46__ \"3-OXOACYL-ACP-SYNTH-RXN-Octanoyl-ACPs/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C6-H12/CARBON-DIOXIDE/ACP.88.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__Octanoyl__45__ACPs__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C6__45__H12__47__CARBON__45__DIOXIDE__47__ACP__46__88__46__ \"3-OXOACYL-ACP-SYNTH-RXN-Octanoyl-ACPs/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C6-H12/CARBON-DIOXIDE/ACP.88.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C6__45__H12__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C8__45__H16__47__CARBON__45__DIOXIDE__47__ACP__46__120__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C6-H12/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C8-H16/CARBON-DIOXIDE/ACP.120.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C6__45__H12__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C8__45__H16__47__CARBON__45__DIOXIDE__47__ACP__46__120__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C6-H12/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C8-H16/CARBON-DIOXIDE/ACP.120.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C8__45__H16__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C10__45__H20__47__CARBON__45__DIOXIDE__47__ACP__46__121__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C8-H16/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C10-H20/CARBON-DIOXIDE/ACP.121.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C8__45__H16__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C10__45__H20__47__CARBON__45__DIOXIDE__47__ACP__46__121__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C8-H16/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C10-H20/CARBON-DIOXIDE/ACP.121.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C10__45__H20__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C12__45__H24__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C10-H20/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C12-H24/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C10__45__H20__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C12__45__H24__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C10-H20/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C12-H24/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C12__45__H24__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C14__45__H28__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C12-H24/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C14-H28/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C12__45__H24__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C14__45__H28__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C12-H24/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C14-H28/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C14__45__H28__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C16__45__H32__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C14-H28/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C16-H32/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C14__45__H28__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C16__45__H32__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C14-H28/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C16-H32/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C16__45__H32__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C18__45__H36__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C16-H32/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C18-H36/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__OXOACYL__45__ACP__45__SYNTH__45__RXN__45__POLYMER__45__INST__45__Saturated__45__Fatty__45__Acyl__45__ACPs__45__C16__45__H32__47__MALONYL__45__ACP__47__PROTON__47____47__POLYMER__45__INST__45__B__45__KETOACYL__45__ACP__45__C18__45__H36__47__CARBON__45__DIOXIDE__47__ACP__46__122__46__ \"3-OXOACYL-ACP-SYNTH-RXN-POLYMER-INST-Saturated-Fatty-Acyl-ACPs-C16-H32/MALONYL-ACP/PROTON//POLYMER-INST-B-KETOACYL-ACP-C18-H36/CARBON-DIOXIDE/ACP.122.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11361 \"RXN-11361\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11361 \"RXN-11361\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5205 \"RXN0-5205\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5205 \"RXN0-5205\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__45__DEHYDROQUINATE__45__DEHYDRATASE__45__RXN \"3-DEHYDROQUINATE-DEHYDRATASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__45__DEHYDROQUINATE__45__DEHYDRATASE__45__RXN \"3-DEHYDROQUINATE-DEHYDRATASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UREA__45__CARBOXYLASE__45__RXN \"UREA-CARBOXYLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UREA__45__CARBOXYLASE__45__RXN \"UREA-CARBOXYLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__2884 \"RXN-2884\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__2884 \"RXN-2884\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18798 \"RXN-18798\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18798 \"RXN-18798\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__10003 \"RXN-10003\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__10003 \"RXN-10003\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2G6Z__45__2 \"RXN2G6Z-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2G6Z__45__2 \"RXN2G6Z-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction KDOTRANS__45__RXN \"KDOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction KDOTRANS__45__RXN \"KDOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2B4Q__45__54 \"RXN2B4Q-54\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2B4Q__45__54 \"RXN2B4Q-54\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15446 \"RXN-15446\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15446 \"RXN-15446\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13430 \"RXN-13430\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13430 \"RXN-13430\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5222 \"RXN0-5222\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5222 \"RXN0-5222\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20790 \"RXN-20790\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20790 \"RXN-20790\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11178 \"RXN-11178\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11178 \"RXN-11178\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5390 \"RXN0-5390\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5390 \"RXN0-5390\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6721 \"RXN0-6721\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6721 \"RXN0-6721\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11179 \"RXN-11179\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11179 \"RXN-11179\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THIKIN__45__RXN \"THIKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THIKIN__45__RXN \"THIKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13435 \"RXN-13435\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13435 \"RXN-13435\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11180 \"RXN-11180\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11180 \"RXN-11180\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22248 \"RXN-22248\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22248 \"RXN-22248\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11474 \"RXN-11474\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11474 \"RXN-11474\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN0__45__498 \"TRANS-RXN0-498\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN0__45__498 \"TRANS-RXN0-498\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11176 \"RXN-11176\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11176 \"RXN-11176\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16659 \"RXN-16659\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16659 \"RXN-16659\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__7908 \"RXN-7908\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__7908 \"RXN-7908\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11578 \"RXN-11578\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11578 \"RXN-11578\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23036 \"RXN-23036\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23036 \"RXN-23036\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN2DNV__45__4 \"RXN2DNV-4\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN2DNV__45__4 \"RXN2DNV-4\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__45__ACYLGLYCEROL__45__3__45__P__45__ACYLTRANSFER__45__RXN \"1-ACYLGLYCEROL-3-P-ACYLTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__45__ACYLGLYCEROL__45__3__45__P__45__ACYLTRANSFER__45__RXN \"1-ACYLGLYCEROL-3-P-ACYLTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14903 \"RXN-14903\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14903 \"RXN-14903\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__1623 \"RXN-1623\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__1623 \"RXN-1623\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__2__46__1__46__2__45__RXN \"1.2.1.2-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__2__46__1__46__2__45__RXN \"1.2.1.2-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8941__45__CPD__45__9372__47__NADH__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__CPD__45__9373__47__NAD__47__WATER__46__57__46__ \"RXN-8941-CPD-9372/NADH/OXYGEN-MOLECULE/PROTON//CPD-9373/NAD/WATER.57.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8941__45__CPD__45__9372__47__NADH__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__CPD__45__9373__47__NAD__47__WATER__46__57__46__ \"RXN-8941-CPD-9372/NADH/OXYGEN-MOLECULE/PROTON//CPD-9373/NAD/WATER.57.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8941__45__CPD__45__9372__47__NADPH__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__CPD__45__9373__47__NADP__47__WATER__46__59__46__ \"RXN-8941-CPD-9372/NADPH/OXYGEN-MOLECULE/PROTON//CPD-9373/NADP/WATER.59.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8941__45__CPD__45__9372__47__NADPH__47__OXYGEN__45__MOLECULE__47__PROTON__47____47__CPD__45__9373__47__NADP__47__WATER__46__59__46__ \"RXN-8941-CPD-9372/NADPH/OXYGEN-MOLECULE/PROTON//CPD-9373/NADP/WATER.59.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14116 \"RXN-14116\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14116 \"RXN-14116\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21000 \"RXN-21000\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21000 \"RXN-21000\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__4342 \"RXN0-4342\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__4342 \"RXN0-4342\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__3221 \"RXN0-3221\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__3221 \"RXN0-3221\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__4341 \"RXN0-4341\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__4341 \"RXN0-4341\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14382 \"RXN-14382\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14382 \"RXN-14382\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9386 \"RXN-9386\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9386 \"RXN-9386\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FORMYLMETHANOFURAN__45__DEHYDROGENASE__45__RXN \"FORMYLMETHANOFURAN-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FORMYLMETHANOFURAN__45__DEHYDROGENASE__45__RXN \"FORMYLMETHANOFURAN-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6998 \"RXN0-6998\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6998 \"RXN0-6998\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MCPMETEST__45__RXN \"MCPMETEST-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MCPMETEST__45__RXN \"MCPMETEST-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14267 \"RXN-14267\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14267 \"RXN-14267\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19368 \"RXN-19368\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19368 \"RXN-19368\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN66__45__546__45__4__45__HYDROXY__45__L__45__PROLINE__47__NADP__47____47__PYRROLINE__45__HYDROXY__45__CARBOXYLATE__47__NADPH__47__PROTON__46__69__46__ \"RXN66-546-4-HYDROXY-L-PROLINE/NADP//PYRROLINE-HYDROXY-CARBOXYLATE/NADPH/PROTON.69.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN66__45__546__45__4__45__HYDROXY__45__L__45__PROLINE__47__NADP__47____47__PYRROLINE__45__HYDROXY__45__CARBOXYLATE__47__NADPH__47__PROTON__46__69__46__ \"RXN66-546-4-HYDROXY-L-PROLINE/NADP//PYRROLINE-HYDROXY-CARBOXYLATE/NADPH/PROTON.69.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN66__45__546__45__4__45__HYDROXY__45__L__45__PROLINE__47__NAD__47____47__PYRROLINE__45__HYDROXY__45__CARBOXYLATE__47__NADH__47__PROTON__46__67__46__ \"RXN66-546-4-HYDROXY-L-PROLINE/NAD//PYRROLINE-HYDROXY-CARBOXYLATE/NADH/PROTON.67.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN66__45__546__45__4__45__HYDROXY__45__L__45__PROLINE__47__NAD__47____47__PYRROLINE__45__HYDROXY__45__CARBOXYLATE__47__NADH__47__PROTON__46__67__46__ \"RXN66-546-4-HYDROXY-L-PROLINE/NAD//PYRROLINE-HYDROXY-CARBOXYLATE/NADH/PROTON.67.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRROLINECARBREDUCT__45__RXN__45__PRO__47__NADP__47____47__L__45__DELTA1__45__PYRROLINE_5__45__CARBOXYLATE__47__NADPH__47__PROTON__46__56__46__ \"PYRROLINECARBREDUCT-RXN-PRO/NADP//L-DELTA1-PYRROLINE_5-CARBOXYLATE/NADPH/PROTON.56.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRROLINECARBREDUCT__45__RXN__45__PRO__47__NADP__47____47__L__45__DELTA1__45__PYRROLINE_5__45__CARBOXYLATE__47__NADPH__47__PROTON__46__56__46__ \"PYRROLINECARBREDUCT-RXN-PRO/NADP//L-DELTA1-PYRROLINE_5-CARBOXYLATE/NADPH/PROTON.56.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PYRROLINECARBREDUCT__45__RXN__45__PRO__47__NAD__47____47__L__45__DELTA1__45__PYRROLINE_5__45__CARBOXYLATE__47__NADH__47__PROTON__46__54__46__ \"PYRROLINECARBREDUCT-RXN-PRO/NAD//L-DELTA1-PYRROLINE_5-CARBOXYLATE/NADH/PROTON.54.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PYRROLINECARBREDUCT__45__RXN__45__PRO__47__NAD__47____47__L__45__DELTA1__45__PYRROLINE_5__45__CARBOXYLATE__47__NADH__47__PROTON__46__54__46__ \"PYRROLINECARBREDUCT-RXN-PRO/NAD//L-DELTA1-PYRROLINE_5-CARBOXYLATE/NADH/PROTON.54.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NITROGENASE__45__RXN \"NITROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NITROGENASE__45__RXN \"NITROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETYLGLUTKIN__45__RXN \"ACETYLGLUTKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETYLGLUTKIN__45__RXN \"ACETYLGLUTKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12195 \"RXN-12195\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12195 \"RXN-12195\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction MCHMBARK__45__RXN \"MCHMBARK-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction MCHMBARK__45__RXN \"MCHMBARK-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SPONTPRO__45__RXN \"SPONTPRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SPONTPRO__45__RXN \"SPONTPRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DUTP__45__PYROP__45__RXN \"DUTP-PYROP-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DUTP__45__PYROP__45__RXN \"DUTP-PYROP-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PROPKIN__45__RXN \"PROPKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PROPKIN__45__RXN \"PROPKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__5__46__4__46__27__45__RXN \"3.5.4.27-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__5__46__4__46__27__45__RXN \"3.5.4.27-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETATEKIN__45__RXN \"ACETATEKIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETATEKIN__45__RXN \"ACETATEKIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _4OHBENZOATE__45__OCTAPRENYLTRANSFER__45__RXN \"4OHBENZOATE-OCTAPRENYLTRANSFER-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _4OHBENZOATE__45__OCTAPRENYLTRANSFER__45__RXN \"4OHBENZOATE-OCTAPRENYLTRANSFER-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction P__45__PANTOCYSDECARB__45__RXN \"P-PANTOCYSDECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction P__45__PANTOCYSDECARB__45__RXN \"P-PANTOCYSDECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANSALDOL__45__RXN \"TRANSALDOL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANSALDOL__45__RXN \"TRANSALDOL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R12__45__RXN \"R12-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R12__45__RXN \"R12-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PABASYN__45__RXN \"PABASYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PABASYN__45__RXN \"PABASYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15952 \"RXN-15952\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15952 \"RXN-15952\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22613 \"RXN-22613\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22613 \"RXN-22613\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction P__45__PANTOCYSLIG__45__RXN \"P-PANTOCYSLIG-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction P__45__PANTOCYSLIG__45__RXN \"P-PANTOCYSLIG-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22713 \"RXN-22713\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22713 \"RXN-22713\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7067 \"RXN0-7067\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7067 \"RXN0-7067\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HISTIDPHOS__45__RXN \"HISTIDPHOS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HISTIDPHOS__45__RXN \"HISTIDPHOS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3__46__4__46__24__46__32__45__RXN__45__C6__47__WATER__47____47__CPD__45__15797__47__CPD0__45__1085__46__30__46__ \"3.4.24.32-RXN-C6/WATER//CPD-15797/CPD0-1085.30.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3__46__4__46__24__46__32__45__RXN__45__C6__47__WATER__47____47__CPD__45__15797__47__CPD0__45__1085__46__30__46__ \"3.4.24.32-RXN-C6/WATER//CPD-15797/CPD0-1085.30.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALLOPHANATE__45__HYDROLASE__45__RXN \"ALLOPHANATE-HYDROLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALLOPHANATE__45__HYDROLASE__45__RXN \"ALLOPHANATE-HYDROLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22914 \"RXN-22914\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22914 \"RXN-22914\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7066 \"RXN0-7066\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7066 \"RXN0-7066\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UROGENIIISYN__45__RXN \"UROGENIIISYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UROGENIIISYN__45__RXN \"UROGENIIISYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THREONINE__45____45__TRNA__45__LIGASE__45__RXN \"THREONINE--TRNA-LIGASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THREONINE__45____45__TRNA__45__LIGASE__45__RXN \"THREONINE--TRNA-LIGASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SUCCDIAMINOPIMDESUCC__45__RXN \"SUCCDIAMINOPIMDESUCC-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SUCCDIAMINOPIMDESUCC__45__RXN \"SUCCDIAMINOPIMDESUCC-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12587 \"RXN-12587\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12587 \"RXN-12587\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FMNREDUCT__45__RXN__45__FMNH2__47__NADP__47____47__FMN__47__NADPH__47__PROTON__46__29__46__ \"FMNREDUCT-RXN-FMNH2/NADP//FMN/NADPH/PROTON.29.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FMNREDUCT__45__RXN__45__FMNH2__47__NADP__47____47__FMN__47__NADPH__47__PROTON__46__29__46__ \"FMNREDUCT-RXN-FMNH2/NADP//FMN/NADPH/PROTON.29.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FMNREDUCT__45__RXN__45__FMNH2__47__NAD__47____47__FMN__47__NADH__47__PROTON__46__27__46__ \"FMNREDUCT-RXN-FMNH2/NAD//FMN/NADH/PROTON.27.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FMNREDUCT__45__RXN__45__FMNH2__47__NAD__47____47__FMN__47__NADH__47__PROTON__46__27__46__ \"FMNREDUCT-RXN-FMNH2/NAD//FMN/NADH/PROTON.27.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__45__OCTAPRENYL__45__6__45__OHPHENOL__45__METHY__45__RXN \"2-OCTAPRENYL-6-OHPHENOL-METHY-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__45__OCTAPRENYL__45__6__45__OHPHENOL__45__METHY__45__RXN \"2-OCTAPRENYL-6-OHPHENOL-METHY-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__12541 \"RXN-12541\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__12541 \"RXN-12541\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALANINE__45__DEHYDROGENASE__45__RXN \"ALANINE-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALANINE__45__DEHYDROGENASE__45__RXN \"ALANINE-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SUCCINYLDIAMINOPIMTRANS__45__RXN \"SUCCINYLDIAMINOPIMTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SUCCINYLDIAMINOPIMTRANS__45__RXN \"SUCCINYLDIAMINOPIMTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HOMSUCTRAN__45__RXN \"HOMSUCTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HOMSUCTRAN__45__RXN \"HOMSUCTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TETHYDPICSUCC__45__RXN \"TETHYDPICSUCC-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TETHYDPICSUCC__45__RXN \"TETHYDPICSUCC-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DTMPKI__45__RXN \"DTMPKI-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DTMPKI__45__RXN \"DTMPKI-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__4__46__2__46__37__45__RXN \"2.4.2.37-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__4__46__2__46__37__45__RXN \"2.4.2.37-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22287 \"RXN-22287\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22287 \"RXN-22287\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16938 \"RXN-16938\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16938 \"RXN-16938\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DAPASYN__45__RXN \"DAPASYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DAPASYN__45__RXN \"DAPASYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PSEUDOURIDINE__45__KINASE__45__RXN \"PSEUDOURIDINE-KINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PSEUDOURIDINE__45__KINASE__45__RXN \"PSEUDOURIDINE-KINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction QUINOPRIBOTRANS__45__RXN \"QUINOPRIBOTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction QUINOPRIBOTRANS__45__RXN \"QUINOPRIBOTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction AMIDASE__45__RXN__45__CPD__45__14534__47__WATER__47____47__GLYCOLLATE__47__AMMONIUM__46__37__46__ \"AMIDASE-RXN-CPD-14534/WATER//GLYCOLLATE/AMMONIUM.37.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction AMIDASE__45__RXN__45__CPD__45__14534__47__WATER__47____47__GLYCOLLATE__47__AMMONIUM__46__37__46__ \"AMIDASE-RXN-CPD-14534/WATER//GLYCOLLATE/AMMONIUM.37.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22974 \"RXN-22974\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22974 \"RXN-22974\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENOSYLHOMOCYSTEINASE__45__RXN \"ADENOSYLHOMOCYSTEINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENOSYLHOMOCYSTEINASE__45__RXN \"ADENOSYLHOMOCYSTEINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHENYLSERINE__45__ALDOLASE__45__RXN \"PHENYLSERINE-ALDOLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHENYLSERINE__45__ALDOLASE__45__RXN \"PHENYLSERINE-ALDOLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction S__45__ADENMETSYN__45__RXN \"S-ADENMETSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction S__45__ADENMETSYN__45__RXN \"S-ADENMETSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction THREONINE__45__ALDOLASE__45__RXN \"THREONINE-ALDOLASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction THREONINE__45__ALDOLASE__45__RXN \"THREONINE-ALDOLASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SUCCCOASYN__45__RXN \"SUCCCOASYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SUCCCOASYN__45__RXN \"SUCCCOASYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6563 \"RXN0-6563\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6563 \"RXN0-6563\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction LTAA__45__RXN \"LTAA-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction LTAA__45__RXN \"LTAA-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11592 \"RXN-11592\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11592 \"RXN-11592\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADPSUGPPHOSPHAT__45__RXN__45__ADP__45__D__45__GLUCOSE__47__WATER__47____47__AMP__47__GLC__45__1__45__P__47__PROTON__46__40__46__ \"ADPSUGPPHOSPHAT-RXN-ADP-D-GLUCOSE/WATER//AMP/GLC-1-P/PROTON.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADPSUGPPHOSPHAT__45__RXN__45__ADP__45__D__45__GLUCOSE__47__WATER__47____47__AMP__47__GLC__45__1__45__P__47__PROTON__46__40__46__ \"ADPSUGPPHOSPHAT-RXN-ADP-D-GLUCOSE/WATER//AMP/GLC-1-P/PROTON.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14992 \"RXN-14992\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14992 \"RXN-14992\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _3PGAREARR__45__RXN \"3PGAREARR-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _3PGAREARR__45__RXN \"3PGAREARR-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13528 \"RXN-13528\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13528 \"RXN-13528\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8642 \"RXN-8642\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8642 \"RXN-8642\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction H2NEOPTERINALDOL__45__RXN \"H2NEOPTERINALDOL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction H2NEOPTERINALDOL__45__RXN \"H2NEOPTERINALDOL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GPPSYN__45__RXN \"GPPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GPPSYN__45__RXN \"GPPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19365 \"RXN-19365\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19365 \"RXN-19365\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9951 \"RXN-9951\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9951 \"RXN-9951\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PRPPSYN__45__RXN \"PRPPSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PRPPSYN__45__RXN \"PRPPSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R345__45__RXN \"R345-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R345__45__RXN \"R345-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22711 \"RXN-22711\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22711 \"RXN-22711\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__7229 \"RXN0-7229\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__7229 \"RXN0-7229\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ISOCITDEH__45__RXN \"ISOCITDEH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ISOCITDEH__45__RXN \"ISOCITDEH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PPENTOMUT__45__RXN \"PPENTOMUT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PPENTOMUT__45__RXN \"PPENTOMUT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R10__45__RXN \"R10-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R10__45__RXN \"R10-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYCOLATEDEHYDRO__45__RXN \"GLYCOLATEDEHYDRO-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYCOLATEDEHYDRO__45__RXN \"GLYCOLATEDEHYDRO-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction BRANCHED__45__CHAINAMINOTRANSFERVAL__45__RXN \"BRANCHED-CHAINAMINOTRANSFERVAL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction BRANCHED__45__CHAINAMINOTRANSFERVAL__45__RXN \"BRANCHED-CHAINAMINOTRANSFERVAL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction HOMOCYSMET__45__RXN \"HOMOCYSMET-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction HOMOCYSMET__45__RXN \"HOMOCYSMET-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__901 \"RXN0-901\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__901 \"RXN0-901\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PORPHOBILSYNTH__45__RXN \"PORPHOBILSYNTH-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PORPHOBILSYNTH__45__RXN \"PORPHOBILSYNTH-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ARGDECARBOX__45__RXN \"ARGDECARBOX-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ARGDECARBOX__45__RXN \"ARGDECARBOX-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENODEAMIN__45__RXN \"ADENODEAMIN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENODEAMIN__45__RXN \"ADENODEAMIN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14904 \"RXN-14904\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14904 \"RXN-14904\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GUANINE__45__DEAMINASE__45__RXN \"GUANINE-DEAMINASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GUANINE__45__DEAMINASE__45__RXN \"GUANINE-DEAMINASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPGLUCEPIM__45__RXN \"UDPGLUCEPIM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPGLUCEPIM__45__RXN \"UDPGLUCEPIM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19728 \"RXN-19728\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19728 \"RXN-19728\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction BRANCHED__45__CHAINAMINOTRANSFERILEU__45__RXN \"BRANCHED-CHAINAMINOTRANSFERILEU-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction BRANCHED__45__CHAINAMINOTRANSFERILEU__45__RXN \"BRANCHED-CHAINAMINOTRANSFERILEU-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21550 \"RXN-21550\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21550 \"RXN-21550\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__180__45__RXN \"2.3.1.180-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__180__45__RXN \"2.3.1.180-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLUCOSE__45__6__45__PHOSPHATE__45__1__45__EPIMERASE__45__RXN \"GLUCOSE-6-PHOSPHATE-1-EPIMERASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLUCOSE__45__6__45__PHOSPHATE__45__1__45__EPIMERASE__45__RXN \"GLUCOSE-6-PHOSPHATE-1-EPIMERASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction BRANCHED__45__CHAINAMINOTRANSFERLEU__45__RXN \"BRANCHED-CHAINAMINOTRANSFERLEU-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction BRANCHED__45__CHAINAMINOTRANSFERLEU__45__RXN \"BRANCHED-CHAINAMINOTRANSFERLEU-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _6__46__3__46__1__46__10__45__RXN \"6.3.1.10-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _6__46__3__46__1__46__10__45__RXN \"6.3.1.10-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__15041 \"RXN-15041\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__15041 \"RXN-15041\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__6261 \"RXN-6261\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__6261 \"RXN-6261\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GAPOXNPHOSPHN__45__RXN \"GAPOXNPHOSPHN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GAPOXNPHOSPHN__45__RXN \"GAPOXNPHOSPHN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction QUINATE__45__5__45__DEHYDROGENASE__45__RXN__45__QUINATE__47__NADP__47____47__DEHYDROQUINATE__47__NADPH__47__PROTON__46__42__46__ \"QUINATE-5-DEHYDROGENASE-RXN-QUINATE/NADP//DEHYDROQUINATE/NADPH/PROTON.42.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction QUINATE__45__5__45__DEHYDROGENASE__45__RXN__45__QUINATE__47__NADP__47____47__DEHYDROQUINATE__47__NADPH__47__PROTON__46__42__46__ \"QUINATE-5-DEHYDROGENASE-RXN-QUINATE/NADP//DEHYDROQUINATE/NADPH/PROTON.42.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction QUINATE__45__5__45__DEHYDROGENASE__45__RXN__45__QUINATE__47__NAD__47____47__DEHYDROQUINATE__47__NADH__47__PROTON__46__40__46__ \"QUINATE-5-DEHYDROGENASE-RXN-QUINATE/NAD//DEHYDROQUINATE/NADH/PROTON.40.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction QUINATE__45__5__45__DEHYDROGENASE__45__RXN__45__QUINATE__47__NAD__47____47__DEHYDROQUINATE__47__NADH__47__PROTON__46__40__46__ \"QUINATE-5-DEHYDROGENASE-RXN-QUINATE/NAD//DEHYDROQUINATE/NADH/PROTON.40.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R342__45__RXN \"R342-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R342__45__RXN \"R342-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19729 \"RXN-19729\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19729 \"RXN-19729\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__6 \"TRANS-RXN2PP0-6\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN2PP0__45__6 \"TRANS-RXN2PP0-6\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction SHIKIMATE__45__5__45__DEHYDROGENASE__45__RXN \"SHIKIMATE-5-DEHYDROGENASE-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction SHIKIMATE__45__5__45__DEHYDROGENASE__45__RXN \"SHIKIMATE-5-DEHYDROGENASE-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__11174__45__SHIKIMATE__47__NAD__47____47__3__45__DEHYDRO__45__SHIKIMATE__47__NADH__47__PROTON__46__47__46__ \"RXN-11174-SHIKIMATE/NAD//3-DEHYDRO-SHIKIMATE/NADH/PROTON.47.\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__11174__45__SHIKIMATE__47__NAD__47____47__3__45__DEHYDRO__45__SHIKIMATE__47__NADH__47__PROTON__46__47__46__ \"RXN-11174-SHIKIMATE/NAD//3-DEHYDRO-SHIKIMATE/NADH/PROTON.47.\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19726 \"RXN-19726\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19726 \"RXN-19726\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FCLREDUCT__45__RXN \"FCLREDUCT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FCLREDUCT__45__RXN \"FCLREDUCT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DXS__45__RXN \"DXS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DXS__45__RXN \"DXS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction FCLEPIM__45__RXN \"FCLEPIM-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction FCLEPIM__45__RXN \"FCLEPIM-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _1__46__1__46__1__46__271__45__RXN \"1.1.1.271-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _1__46__1__46__1__46__271__45__RXN \"1.1.1.271-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__19727 \"RXN-19727\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__19727 \"RXN-19727\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13252 \"RXN-13252\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13252 \"RXN-13252\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__8631 \"RXN-8631\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__8631 \"RXN-8631\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NMNNUCLEOSID__45__RXN \"NMNNUCLEOSID-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NMNNUCLEOSID__45__RXN \"NMNNUCLEOSID-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PHOSPHASERSYN__45__RXN \"PHOSPHASERSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PHOSPHASERSYN__45__RXN \"PHOSPHASERSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction DIAMINOPIMDECARB__45__RXN \"DIAMINOPIMDECARB-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction DIAMINOPIMDECARB__45__RXN \"DIAMINOPIMDECARB-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NADPYROPHOSPHAT__45__RXN \"NADPYROPHOSPHAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NADPYROPHOSPHAT__45__RXN \"NADPYROPHOSPHAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21525 \"RXN-21525\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21525 \"RXN-21525\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction TRANS__45__RXN__45__468 \"TRANS-RXN-468\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction TRANS__45__RXN__45__468 \"TRANS-RXN-468\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _5__46__4__46__99__46__17__45__RXN \"5.4.99.17-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _5__46__4__46__99__46__17__45__RXN \"5.4.99.17-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21523 \"RXN-21523\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21523 \"RXN-21523\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__16820 \"RXN-16820\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__16820 \"RXN-16820\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NICOTINAMID__45__RXN \"NICOTINAMID-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NICOTINAMID__45__RXN \"NICOTINAMID-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21527 \"RXN-21527\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21527 \"RXN-21527\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__6999 \"RXN0-6999\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__6999 \"RXN0-6999\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction PROTOPORGENOXI__45__RXN \"PROTOPORGENOXI-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction PROTOPORGENOXI__45__RXN \"PROTOPORGENOXI-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__1461 \"RXN0-1461\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__1461 \"RXN0-1461\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5224 \"RXN0-5224\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5224 \"RXN0-5224\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ADENPRIBOSYLTRAN__45__RXN \"ADENPRIBOSYLTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ADENPRIBOSYLTRAN__45__RXN \"ADENPRIBOSYLTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__18693 \"RXN-18693\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__18693 \"RXN-18693\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13871 \"RXN-13871\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13871 \"RXN-13871\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETOOHBUTSYN__45__RXN \"ACETOOHBUTSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETOOHBUTSYN__45__RXN \"ACETOOHBUTSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__14270 \"RXN-14270\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__14270 \"RXN-14270\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__947 \"RXN0-947\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__947 \"RXN0-947\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__13870 \"RXN-13870\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__13870 \"RXN-13870\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__2023 \"RXN0-2023\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__2023 \"RXN0-2023\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction _2__46__3__46__1__46__118__45__RXN \"2.3.1.118-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction _2__46__3__46__1__46__118__45__RXN \"2.3.1.118-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__9987 \"RXN-9987\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__9987 \"RXN-9987\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction XANPRIBOSYLTRAN__45__RXN \"XANPRIBOSYLTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction XANPRIBOSYLTRAN__45__RXN \"XANPRIBOSYLTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ACETOLACTSYN__45__RXN \"ACETOLACTSYN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ACETOLACTSYN__45__RXN \"ACETOLACTSYN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NADH__45__DEHYDROG__45__A__45__RXN \"NADH-DEHYDROG-A-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NADH__45__DEHYDROG__45__A__45__RXN \"NADH-DEHYDROG-A-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21415 \"RXN-21415\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21415 \"RXN-21415\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction IMIDPHOSDEHYD__45__RXN \"IMIDPHOSDEHYD-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction IMIDPHOSDEHYD__45__RXN \"IMIDPHOSDEHYD-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ASPCARBTRANS__45__RXN \"ASPCARBTRANS-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ASPCARBTRANS__45__RXN \"ASPCARBTRANS-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction GLYOXII__45__RXN \"GLYOXII-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction GLYOXII__45__RXN \"GLYOXII-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction UDPACYLGLCNACDEACETYL__45__RXN \"UDPACYLGLCNACDEACETYL-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction UDPACYLGLCNACDEACETYL__45__RXN \"UDPACYLGLCNACDEACETYL-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22247 \"RXN-22247\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22247 \"RXN-22247\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__20788 \"RXN-20788\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__20788 \"RXN-20788\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22242 \"RXN-22242\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22242 \"RXN-22242\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction NICONUCADENYLYLTRAN__45__RXN \"NICONUCADENYLYLTRAN-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction NICONUCADENYLYLTRAN__45__RXN \"NICONUCADENYLYLTRAN-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__23222 \"RXN-23222\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__23222 \"RXN-23222\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__21526 \"RXN-21526\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__21526 \"RXN-21526\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN0__45__5101 \"RXN0-5101\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN0__45__5101 \"RXN0-5101\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction ALARACECAT__45__RXN \"ALARACECAT-RXN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction ALARACECAT__45__RXN \"ALARACECAT-RXN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction RXN__45__22915 \"RXN-22915\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction RXN__45__22915 \"RXN-22915\">'\n",
      "No objective in listOfObjectives\n",
      "No objective coefficients in model. Unclear what should be optimized\n",
      "Missing flux bounds on reactions set to default bounds.As best practise and to avoid confusion flux bounds should be set explicitly on all reactions.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>FW12</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>1f8266f9f70</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>1644</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>1449</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>830</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>CCO-CYTOSOL, CCO-IN, CCO-PERI-BAC, CCO-OUT, CCO-MIDDLE, CCO-MEMBRANE</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model FW12 at 0x1f8266f9f70>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import cobra\n",
    "pathwaytool_model = cobra.io.read_sbml_model(org_name+'_model\\\\'+org_name+'pathwaytool.xml')\n",
    "pathwaytool_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "for rxn in pathwaytool_model.reactions:\n",
    "    biocyc = rxn.annotation.get('biocyc')\n",
    "    if biocyc:\n",
    "        if isinstance(biocyc, list):\n",
    "            rxn.annotation['biocyc'] = ['META:'+i.split(':')[-1] for i in biocyc]\n",
    "        else:\n",
    "            rxn.annotation['biocyc'] = 'META:'+biocyc.split(':')[-1]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "META:F16ALDOLASE-RXN ['FBAf', 'FBAm', 'FBAh']\n",
      "META:RXN-12583 ['PDHam1hi', 'PDHam1mi']\n",
      "447/1449\n"
     ]
    }
   ],
   "source": [
    "ptmodelinbigg ={}\n",
    "for rxn in pathwaytool_model.reactions:\n",
    "    biocyc = rxn.annotation.get('biocyc')\n",
    "    if cyc_BIGG_dir.get(biocyc):\n",
    "        selr = select_rxn(unimodel, cyc_BIGG_dir.get(biocyc))\n",
    "        if selr:\n",
    "            ptmodelinbigg[biocyc] = selr\n",
    "            if type(selr) is list:\n",
    "                print(biocyc, selr)\n",
    "print('%d/%d'%(len(ptmodelinbigg),len(pathwaytool_model.reactions)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "ptmodelinbigg['META:F16ALDOLASE-RXN'] = 'FBA'\n",
    "ptmodelinbigg['META:RXN-12583'] = 'PDH'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "META:GLYCERATE-DEHYDROGENASE-RXN HPYRRx\n",
      "META:GUANYL-KIN-RXN GK1\n",
      "META:N-ACETYLNEURAMINATE-SYNTHASE-RXN ACNPLYS\n",
      "META:UDPNACETYLMURAMATEDEHYDROG-RXN UAPGR\n",
      "META:UDP-NACMUR-ALA-LIG-RXN UAMAS\n",
      "META:SPERMINE-SYNTHASE-RXN SPRMS\n",
      "META:OHMETHYLBILANESYN-RXN HMBS\n",
      "META:RXN-1404 NTRLASE\n",
      "META:LIPIDXSYNTHESIS-RXN USHD\n",
      "META:RXN-11484 AOXSr2\n",
      "META:4.2.1.99-RXN MICITDr\n",
      "META:ACETOLACTREDUCTOISOM-RXN KARA1\n",
      "META:RXN-11481 OPMEACPD\n",
      "META:DEOXYADENPHOSPHOR-RXN PUNP2\n",
      "META:DMBPPRIBOSYLTRANS-RXN NNDMBRT\n",
      "META:DEOXYINOPHOSPHOR-RXN PUNP6\n",
      "META:DEOXYGUANPHOSPHOR-RXN PUNP4\n",
      "META:ADENPHOSPHOR-RXN PUNP1\n",
      "META:INOPHOSPHOR-RXN PUNP5\n",
      "META:GLUC1PADENYLTRANS-RXN GLGC\n",
      "META:XANTHOSINEPHOSPHORY-RXN PUNP7\n",
      "META:RXN0-5199 PUNP3\n",
      "META:URA-PHOSPH-RXN DURIPP\n",
      "META:RXN-8991 IPPMIa\n",
      "META:COBINAMIDEKIN-RXN ADOCBIK\n",
      "META:URPHOS-RXN PYNP2r\n",
      "META:SAICARSYN-RXN PRASCSi\n",
      "META:UDP-NACMURALA-GLU-LIG-RXN UAMAGS\n",
      "META:THYM-PHOSPH-RXN TMDPP\n",
      "META:RXN-12002 UMPK\n",
      "META:DXPREDISOM-RXN DXPRIi\n",
      "META:RXN-12864 SELGTHR\n",
      "META:THI-P-SYN-RXN TMPPP\n",
      "META:PYRUVFORMLY-RXN PFL\n",
      "META:RXN-8976 UAGPT2\n",
      "META:NACGLCTRANS-RXN UAGPT3\n",
      "META:3.6.3.16-RXN ASO3t8pp\n",
      "META:DEPHOSPHOCOAKIN-RXN DPCOAK\n",
      "META:PTAALT-RXN PTA2\n",
      "META:CATAL-RXN CAT\n",
      "META:KETOBUTFORMLY-RXN OBTFL\n",
      "META:RXN-14189 PRDX\n",
      "META:PHOSACETYLTRANS-RXN PTAr\n",
      "META:ACSERLY-RXN CYSS\n",
      "META:RXN-12726 ACSERL\n",
      "META:PRIBFAICARPISOM-RXN PRMICI\n",
      "META:HISTCYCLOHYD-RXN PRAMPC\n",
      "META:IGPSYN-RXN IGPS\n",
      "META:PRTRANS-RXN ANPRT\n",
      "META:2TRANSKETO-RXN TKT2\n",
      "META:ARGSUCCINLYA-RXN ARGSL\n",
      "META:ADDALT-RXN DADA\n",
      "META:2OXOGLUTARATEDEH-RXN AKGDH\n",
      "META:DIAMINOPIMEPIM-RXN DAPE\n",
      "META:ACONITATEDEHYDR-RXN ACONTa\n",
      "META:PANTEPADENYLYLTRAN-RXN APPAT\n",
      "META:4.1.1.81-RXN THRPDC\n",
      "META:RXN-9384 SHSL2\n",
      "META:ASNSYNB-RXN ASNS1\n",
      "META:DAHPSYN-RXN DDPA\n",
      "META:ACETOOHBUTREDUCTOISOM-RXN KARA2\n",
      "META:MANNPISOM-RXN MPAKI\n",
      "META:F16BDEPHOS-RXN BFBP\n",
      "META:THIOREDOXIN-REDUCT-NADPH-RXN TRDR\n",
      "META:RXN-10642 UPPDC2\n",
      "META:UROGENDECARBOX-RXN UPPDC1\n",
      "META:SUPEROX-DISMUT-RXN SPODM\n",
      "META:2PGADEHYDRAT-RXN ENO\n",
      "META:3-DEHYDROQUINATE-SYNTHASE-RXN DHQS\n",
      "META:OROTPDECARB-RXN OMPDC\n",
      "META:GLUTKIN-RXN GLU5K\n",
      "META:ACETATE--COA-LIGASE-RXN ACS\n",
      "META:RXN0-1863 UDPKAAT\n",
      "META:GLUC1PURIDYLTRANS-RXN GALUi\n",
      "META:AMYLOMALT-RXN AMALT1\n",
      "META:GLUTAMATE-N-ACETYLTRANSFERASE-RXN ORNTAC\n",
      "META:RXN-11564 CSPMDDC\n",
      "META:RXN0-4301 S7PI\n",
      "META:GTP-CYCLOHYDRO-II-RXN GTPCII2\n",
      "META:RXN0-5292 NTD3\n",
      "META:HOMOSERKIN-RXN HSK\n",
      "META:RXN-11832 CYTK1\n",
      "META:RXN-14143 NTD1\n",
      "META:FADSYN-RXN AFAT\n",
      "META:2.5.1.19-RXN PSCVT\n",
      "META:ABC-19-RXN MOBDabc\n",
      "META:ADPREDUCT-RXN RNDR1\n",
      "META:PEPCARBOXYKIN-RXN PPCK\n",
      "META:GDPREDUCT-RXN RNDR2\n",
      "META:HISTAMINOTRANS-RXN HSTPT\n",
      "META:RXN-14142 NTD8\n",
      "META:RXN-15513 PGM\n",
      "META:UDPREDUCT-RXN RNDR4\n",
      "META:PREPHENATEDEHYDRAT-RXN PPNDH\n",
      "META:THYMIDYLATE-5-PHOSPHATASE-RXN NTD5\n",
      "META:RXN-7913 CYTK2\n",
      "META:RXN-9772 ASPO5\n",
      "META:RXN-14161 NTD6\n",
      "META:RXN-12197 NDP7\n",
      "META:RXN-12196 NTP7\n",
      "META:RXN-12198 CDPPH\n",
      "META:PSERTRANSAMPYR-RXN OHPBAT\n",
      "META:PSERTRANSAM-RXN PSERT\n",
      "META:CHORISMATE-SYNTHASE-RXN CHORS\n",
      "META:3.6.1.41-RXN AP4AH\n",
      "META:SAMDECARB-RXN ADMDC\n",
      "META:UREASE-RXN UREA\n",
      "META:L-ASPARTATE-OXID-RXN ASPO6\n",
      "META:THRESYN-RXN THRS\n",
      "META:AMPSYN-RXN ADSL1r\n",
      "META:DARAB5PISOM-RXN A5PISO\n",
      "META:KDO-8PPHOSPHAT-RXN KDOPP\n",
      "META:ABC-70-RXN SULabc\n",
      "META:L-GLN-FRUCT-6-P-AMINOTRANS-RXN GF6PTA\n",
      "META:GLUTRACE-RXN GLUR\n",
      "META:ABC-7-RXN TSULabc\n",
      "META:DIHYDLIPACETRANS-RXN PDHbr\n",
      "META:THREDEHYD-RXN THRD_L\n",
      "META:FPPSYN-RXN GRTT\n",
      "META:PYRNUTRANSHYDROGEN-RXN NADTRHD\n",
      "META:GLYCDEH-RXN GLYCDx\n",
      "META:RXN-8001 HDH\n",
      "META:ATPPHOSPHORIBOSYLTRANS-RXN ATPPRT\n",
      "META:THI-P-KIN-RXN TMPK\n",
      "META:IMP-DEHYDROG-RXN IMPD\n",
      "META:UDPNACETYLGLUCOSAMENOLPYRTRANS-RXN UAGCVT\n",
      "META:GMP-SYN-NH3-RXN GMPS\n",
      "META:GMP-SYN-GLUT-RXN GMPS2\n",
      "META:TRYPSYN-RXN TRPS1\n",
      "META:GLUTSEMIALDEHYDROG-RXN G5SD\n",
      "META:OXALODECARB-RXN OAADC\n",
      "META:MALATE-DEH-RXN MDH\n",
      "META:1.1.1.39-RXN ME1\n",
      "META:RIBOFLAVINSYNDEAM-RXN DHPPDA2\n",
      "META:CHORISMATEMUT-RXN CHORM\n",
      "META:MALIC-NADP-RXN ME2\n",
      "META:PRPPAMIDOTRANS-RXN GLUPRT\n",
      "META:RIBOFLAVINSYNREDUC-RXN APRAUR\n",
      "META:MALONYL-COA-ACP-TRANSACYL-RXN MCOATA\n",
      "META:PRAISOM-RXN PRAIi\n",
      "META:CPM-KDOSYNTH-RXN KDOCT2\n",
      "META:RXN-10814 PHETA1\n",
      "META:COBALADENOSYLTRANS-RXN CBLAT\n",
      "META:RXN-11319 TYRL\n",
      "META:PGLUCISOM-RXN PGI\n",
      "META:TETRAACYLDISACC4KIN-RXN TDSK\n",
      "META:GTP-CYCLOHYDRO-I-RXN GTPCI\n",
      "META:CITSYN-RXN CS\n",
      "META:R343-RXN CYRDAR2\n",
      "META:BTUR2-RXN CBIAT\n",
      "META:PROTOHEMEFERROCHELAT-RXN FCLT\n",
      "META:RXN0-6427 GDPTPDP\n",
      "META:RXN-14192 r0280\n",
      "META:GDPPYPHOSKIN-RXN GDPDPK\n",
      "META:RXN-14207 AGPOP\n",
      "META:PPPGPPHYDRO-RXN GDTP\n",
      "META:PEPDEPHOS-RXN PYK\n",
      "META:RXN-14117 PYK3\n",
      "META:MYO-INOSITOL-1-PHOSPHATE-SYNTHASE-RXN MI3PS\n",
      "META:UDPHYDROXYMYRGLUCOSAMNACETYLTRANS-RXN U23GAAT\n",
      "META:H2PTEROATESYNTH-RXN DHPS2\n",
      "META:QUINOLINATE-SYNTHA-RXN QULNS\n",
      "META:PYRUVATEORTHOPHOSPHATE-DIKINASE-RXN PPDK\n",
      "META:GSAAMINOTRANS-RXN G1SAT\n",
      "META:RXN0-2382 TRPS2\n",
      "META:RXN-11483 PMEACPE\n",
      "META:ORNCARBAMTRANSFER-RXN OCT\n",
      "META:CHORPYRLY-RXN CHRPL\n",
      "META:ACETYL-COA-CARBOXYLTRANSFER-RXN ACCOAC\n",
      "META:1.3.1.54-RXN PC6AR\n",
      "META:DIHYDROXYISOVALDEHYDRAT-RXN DHAD1\n",
      "META:DIHYDROXYMETVALDEHYDRAT-RXN DHAD2\n",
      "META:R322-RXN R05219\n",
      "META:DCDPKIN-RXN NDPK7\n",
      "META:DUDPKIN-RXN NDPK6\n",
      "META:THIAZOLSYN3-RXN HETZK\n",
      "META:N-ACETYLTRANSFER-RXN ACGS\n",
      "META:MALATE-DEHYDROGENASE-NADP+-RXN MDHy\n",
      "META:DTDPKIN-RXN NDPK4\n",
      "META:DADPKIN-RXN NDPK8\n",
      "META:UDPKIN-RXN NDPK2\n",
      "META:RXN-12093 CCGS\n",
      "META:SIROHEME-FERROCHELAT-RXN SHCHF\n",
      "META:RXN0-6575 CDGS\n",
      "META:GDPKIN-RXN NDPK1\n",
      "META:CDPKIN-RXN NDPK3\n",
      "META:GLYCERONE-KINASE-RXN DHAK\n",
      "META:2.7.8.25-RXN TPRDCOAS\n",
      "META:DGDPKIN-RXN NDPK5\n",
      "META:RXN-13403 UPP3MT\n",
      "META:ARGSUCCINSYN-RXN ARGSS\n",
      "META:PEPSYNTH-RXN PPS\n",
      "META:FUC4NACTRANS-RXN AADDGT\n",
      "META:ASPARAGHYD-RXN ASNN\n",
      "META:5.1.3.20-RXN AGMHE\n",
      "META:RXN0-4022 CDGR\n",
      "META:N-ACETYLGLUTPREDUCT-RXN AGPR\n",
      "META:LIPIDADISACCHARIDESYNTH-RXN LPADSS\n",
      "META:ADENYLOSUCCINATE-SYNTHASE-RXN ADSS\n",
      "META:RIBOPHOSPHAT-RXN PMDPHT\n",
      "META:ERYTHRON4PDEHYDROG-RXN PERD\n",
      "META:ERYTH4PDEHYDROG-RXN E4PD\n",
      "META:PNPOXI-RXN PDX5POi\n",
      "META:RXN-15125 SERHL\n",
      "META:DTDPDEHYDRHAMEPIM-RXN TDPDRE\n",
      "META:4.3.1.17-RXN SERD_L\n",
      "META:RXN-11477 OGMEACPD\n",
      "META:PHOSPHOGLUCMUT-RXN PGMT\n",
      "META:DIHYDROOROT-RXN DHORTS\n",
      "META:HYDROG-RXN HYDA\n",
      "META:RXN0-5114 PSP_L\n",
      "META:SULFATE-ADENYLYLTRANS-RXN SADT\n",
      "META:DTDPGLUCOSEPP-RXN G1PTT\n",
      "META:RXN0-1483 FEROc\n",
      "META:DIMETHUROPORDEHYDROG-RXN SHCHD2\n",
      "META:INORGPYROPHOSPHAT-RXN PPA\n",
      "META:GLUCOKIN-RXN HEX1\n",
      "META:2-OCTAPRENYL-METHOXY-BENZOQ-METH-RXN OMBZLM\n",
      "META:FUMHYDR-RXN FUM\n",
      "META:ASPARTATEKIN-RXN ASPK\n",
      "META:NICOTINATEPRIBOSYLTRANS-RXN NAMNPP\n",
      "META:RXN-14118 DCTPD2\n",
      "META:DGTPTRIPHYDRO-RXN NTPTP1\n",
      "META:ADCLY-RXN ADCL\n",
      "META:H2NEOPTERINP3PYROPHOSPHOHYDRO-RXN DNTPPA\n",
      "META:RXN-2961 FALGTHLs\n",
      "META:DHHB-METHYLTRANSFER-RXN DMQMT\n",
      "META:R83-RXN 2OH3K5MPPISO\n",
      "META:R82-RXN ACRS\n",
      "META:3.1.3.77-RXN ENOPH\n",
      "META:DIHYDRONEOPTERIN-MONO-P-DEPHOS-RXN DNMPPA\n",
      "META:TRANS-RXN-189 NAt\n",
      "META:5.3.1.23-RXN MTRI\n",
      "META:HISTPRATPHYD-RXN PRATPP\n",
      "META:GABATRANSAM-RXN ABTA\n",
      "META:ETHAMLY-RXN ETHAAL\n",
      "META:TRANS-RXN-250 MG2abc\n",
      "META:SULFITE-REDUCT-RXN SULR\n",
      "META:ADOMET-DMK-METHYLTRANSFER-RXN AMMQLT8\n",
      "META:PGLUCONDEHYDRAT-RXN EDD\n",
      "META:RXN-9952 GND\n",
      "META:RIBULP3EPIM-RXN RPE\n",
      "META:AICARSYN-RXN ADSL2r\n",
      "META:TRANS-RXN-2 HKt\n",
      "META:ADENYLYLSULFKIN-RXN ADSK\n",
      "META:RXN-7800 OMCDC\n",
      "META:NAD-SYNTH-GLN-RXN NADS2\n",
      "META:3-ISOPROPYLMALDEHYDROG-RXN IPMD\n",
      "META:GLYRIBONUCSYN-RXN PPRGL\n",
      "META:GLUTAMINESYN-RXN GLNS\n",
      "META:2-DEHYDROPANTOATE-REDUCT-RXN DPR\n",
      "META:GUANIDINOBUTANAMIDE-NH3-RXN AMID\n",
      "META:RXN0-2381 TRPS3\n",
      "META:RXN-8899 2AMACHYD\n",
      "META:R311-RXN AMID5\n",
      "META:2-ISOPROPYLMALATESYN-RXN IPPS\n",
      "META:DALADALALIG-RXN ALAALAr\n",
      "META:TRANS-RXN0-286 UAAGMDt\n",
      "META:PPGPPSYN-RXN PPGPPDP\n",
      "META:GTPPYPHOSKIN-RXN GTPDPK\n",
      "META:RXN0-7188 METOX2s\n",
      "META:KDPGALDOL-RXN EDA\n",
      "META:DTDPDEHYRHAMREDUCT-RXN TDPDRR\n",
      "META:GLUTCYSLIG-RXN GLUCYS\n",
      "META:RXN490-3616 ASPTRS_Asn\n",
      "META:RXN-12893 CYNTAH\n",
      "META:THIOSULFATE-SULFURTRANSFERASE-RXN CYANST\n",
      "META:1.8.4.8-RXN PAPSR\n",
      "META:RXN-11811 NH3c\n",
      "META:GLUTAMIN-RXN GLUN\n",
      "META:UDPNACETYLGLUCOSAMACYLTRANS-RXN UAGAAT\n",
      "META:GLUTATHIONE-PEROXIDASE-RXN GTHPe\n",
      "META:GLU6PDEHYDROG-RXN G6PDH2r\n",
      "META:2.7.7.13-RXN MAN1PT\n",
      "META:ANTHRANSYN-RXN ANS\n",
      "META:CDP-GLUCOSE-46-DEHYDRATASE-RXN CDPGLC46DH\n",
      "META:2.7.7.33-RXN G1PCTYT\n",
      "META:RXN0-5408 MI1PP\n",
      "META:MYO-INOSITOL-1OR-4-MONOPHOSPHATASE-RXN MI3PP\n",
      "META:2.7.1.148-RXN CDPMEK\n",
      "META:ATPSYN-RXN ATPS4r\n",
      "META:DTDPGLUCDEHYDRAT-RXN TDPGDH\n",
      "META:GDPMANDEHYDRA-RXN GMAND\n",
      "META:NAD-KIN-RXN NADK\n",
      "META:RXN0-743 AIRC3\n",
      "META:CTPSYN-RXN CTPS2\n",
      "META:2.1.1.131-RXN PC17M\n",
      "META:RXN0-366 GNNUC\n",
      "META:D-ALANINE-AMINOTRANSFERASE-RXN ALATA_D\n",
      "META:2.1.1.130-RXN PC20M\n",
      "META:H2PTERIDINEPYROPHOSPHOKIN-RXN HPPK2\n",
      "META:2.1.1.133-RXN PC11M\n",
      "META:SERINE-O-ACETTRAN-RXN SERAT\n",
      "META:TRANS-RXN-141 MGt5\n",
      "META:CDPREDUCT-RXN RNDR3\n",
      "META:SPERMIDINESYN-RXN SPMS\n",
      "META:RXN-14728 AMID4\n",
      "META:RXN0-5507 CPH4S\n",
      "META:6PGLUCONOLACT-RXN PGL\n",
      "META:ALANINE-AMINOTRANSFERASE-RXN ALATA_L\n",
      "META:RXN-12019 APSR\n",
      "META:RXN-13698 ALATA_L\n",
      "META:SEDOBISALDOL-RXN FBA3\n",
      "META:ALCOHOL-DEHYDROG-RXN ALCD2x\n",
      "META:ACETALD-DEHYDROG-RXN ACALD\n",
      "META:5.4.2.10-RXN PGAMT\n",
      "META:GLUTATHIONE-SYN-RXN GTHS\n",
      "META:7KAPSYN-RXN AOXSr\n",
      "META:RXN0-2421 FE2t2pp\n",
      "META:F16ALDOLASE-RXN FBA\n",
      "META:RXN0-302 MECDPS\n",
      "META:GLUTAMATE-DEHYDROGENASE-RXN GDH1\n",
      "META:DETHIOBIOTIN-SYN-RXN DBTS\n",
      "META:2.7.7.60-RXN MEPCT\n",
      "META:3-OCTAPRENYL-4-OHBENZOATE-DECARBOX-RXN OPHBDC\n",
      "META:RIBOFLAVINKIN-RXN RBFK\n",
      "META:ABC-27-RXN PIabc\n",
      "META:RXN-14325 CTPS1\n",
      "META:RXN0-5462 NTP3\n",
      "META:IPPISOM-RXN IPDDI\n",
      "META:IMPCYCLOHYDROLASE-RXN IMPC\n",
      "META:CARBPSYN-RXN CBPS\n",
      "META:GLUTATHIONE-REDUCT-NADPH-RXN GTHOr\n",
      "META:PANTOATE-BETA-ALANINE-LIG-RXN PBAL\n",
      "META:RXN-12583 PDH\n",
      "META:TRIOSEPISOMERIZATION-RXN TPI\n",
      "META:RXN-14025 NTD2\n",
      "META:RXN-8762 CODSCL4MT\n",
      "META:RXN-14026 NTD4\n",
      "META:RXN-8761 CPC3MT\n",
      "META:ASPDECARBOX-RXN ASP1DC\n",
      "META:RXN-7609 NTD9\n",
      "META:RXN-8768 CPC8MM\n",
      "META:XMPXAN-RXN NTD10\n",
      "META:UDPGLCNACEPIM-RXN UAG2E\n",
      "META:RXN-8763 CODSCL5DA\n",
      "META:RXN-5841 NMNR\n",
      "META:RXN-14227 NT5C\n",
      "META:URACIL-PRIBOSYLTRANS-RXN UPPRT\n",
      "META:RXN-14260 AMALT2\n",
      "META:RXN-8344 MPTAT\n",
      "META:RXN-12625 HXAD\n",
      "META:RXN-14261 AMALT3\n",
      "META:5.4.1.2-RXN PC8XM\n",
      "META:HYPOXANPRIBOSYLTRAN-RXN HXPRT\n",
      "META:GUANPRIBOSYLTRAN-RXN GPAR\n",
      "META:HEMN-RXN CPPPGO2\n",
      "META:OROPRIBTRANS-RXN ORPT\n",
      "META:COBINPGUANYLYLTRANS-RXN ACBIPGT\n",
      "META:CYTIDEAM2-RXN CYTD\n",
      "META:CYTIDEAM-RXN DCYTD\n",
      "META:RIBOFLAVIN-SYN-RXN RBFSb\n",
      "META:NAD-SYNTH-NH3-RXN NADS1\n",
      "META:DCTP-DEAM-RXN DCTPD\n",
      "META:RXNN-404 AMID3\n",
      "META:DIHYDROFOLATESYNTH-RXN DHFS\n",
      "META:PANTOTHENATE-KIN-RXN PNTK\n",
      "META:PHOSMANMUT-RXN PMANM\n",
      "META:GLUTAMIDOTRANS-RXN IG3PS\n",
      "META:RXN-14136 GPDDA3\n",
      "META:RXN-14160 GPDDA2\n",
      "META:RXN-14073 GPDDA4\n",
      "META:SHIKIMATE-KINASE-RXN SHKK\n",
      "META:SUCCINATE-SEMIALDEHYDE-DEHYDROGENASE-RXN SSALx\n",
      "META:RXN-8992 OCTDPS\n",
      "META:DCTP-PYROPHOSPHATASE-RXN NTPP3\n",
      "META:ADENYL-KIN-RXN ADK1\n",
      "META:3-ISOPROPYLMALISOM-RXN IPPMIb\n",
      "META:PHOSGLYPHOS-RXN PGK\n",
      "META:ASPARTATE-SEMIALDEHYDE-DEHYDROGENASE-RXN ASAD\n",
      "META:2.7.1.121-RXN DHAPT\n",
      "META:ACETYLORNTRANSAM-RXN ACOTA\n",
      "META:PGLYCDEHYDROG-RXN PGCD\n",
      "META:RXN0-5205 ZN2abcpp\n",
      "META:3-DEHYDROQUINATE-DEHYDRATASE-RXN DHQTi\n",
      "META:KDOTRANS-RXN MOAT\n",
      "META:RXN0-5222 CBMD\n",
      "META:1.2.1.2-RXN FDH\n",
      "META:RXN-14116 PUTA3\n",
      "META:RXN0-4342 GMHEPAT\n",
      "META:RXN-9386 GLUTRS_Gln\n",
      "META:ACETYLGLUTKIN-RXN ACGK\n",
      "META:RXN-12195 NTP5\n",
      "META:SPONTPRO-RXN G5SADs\n",
      "META:DUTP-PYROP-RXN DUTPDP\n",
      "META:ACETATEKIN-RXN ACKr\n",
      "META:4OHBENZOATE-OCTAPRENYLTRANSFER-RXN HBZOPT\n",
      "META:P-PANTOCYSDECARB-RXN PPCDC\n",
      "META:TRANSALDOL-RXN TALA\n",
      "META:PABASYN-RXN ADCS\n",
      "META:P-PANTOCYSLIG-RXN PPNCL2\n",
      "META:HISTIDPHOS-RXN HP\n",
      "META:ALLOPHANATE-HYDROLASE-RXN ALPHNH\n",
      "META:UROGENIIISYN-RXN UPP3S\n",
      "META:SUCCDIAMINOPIMDESUCC-RXN SDPDS\n",
      "META:2-OCTAPRENYL-6-OHPHENOL-METHY-RXN OHPHM\n",
      "META:ALANINE-DEHYDROGENASE-RXN ALAD_L\n",
      "META:SUCCINYLDIAMINOPIMTRANS-RXN SDPTA\n",
      "META:HOMSUCTRAN-RXN HSST\n",
      "META:TETHYDPICSUCC-RXN THDPS\n",
      "META:DTMPKI-RXN DTMPK\n",
      "META:DAPASYN-RXN AMAOTr\n",
      "META:QUINOPRIBOTRANS-RXN NNDPR\n",
      "META:ADENOSYLHOMOCYSTEINASE-RXN AHCi\n",
      "META:S-ADENMETSYN-RXN METAT\n",
      "META:THREONINE-ALDOLASE-RXN THRA\n",
      "META:SUCCCOASYN-RXN SUCOAS\n",
      "META:3PGAREARR-RXN PGM\n",
      "META:RXN-8642 r0082\n",
      "META:H2NEOPTERINALDOL-RXN DHNPA2r\n",
      "META:GPPSYN-RXN DMATT\n",
      "META:BRANCHED-CHAINAMINOTRANSFERVAL-RXN VALTA\n",
      "META:HOMOCYSMET-RXN MHPGLUT\n",
      "META:RXN0-901 XAND\n",
      "META:PORPHOBILSYNTH-RXN PPBNGS\n",
      "META:ARGDECARBOX-RXN ARGDC\n",
      "META:ADENODEAMIN-RXN ADA\n",
      "META:GUANINE-DEAMINASE-RXN GUAD\n",
      "META:UDPGLUCEPIM-RXN UDPG4E\n",
      "META:BRANCHED-CHAINAMINOTRANSFERILEU-RXN ILETA\n",
      "META:2.3.1.180-RXN KAS15\n",
      "META:GLUCOSE-6-PHOSPHATE-1-EPIMERASE-RXN G6PI_1\n",
      "META:BRANCHED-CHAINAMINOTRANSFERLEU-RXN LEUTA\n",
      "META:6.3.1.10-RXN ADCPS1\n",
      "META:RXN-6261 ADCPS2\n",
      "META:GAPOXNPHOSPHN-RXN GAPD\n",
      "META:R342-RXN COCHL\n",
      "META:SHIKIMATE-5-DEHYDROGENASE-RXN SHK3Dr\n",
      "META:DXS-RXN DXPS\n",
      "META:1.1.1.271-RXN GFUCS\n",
      "META:DIAMINOPIMDECARB-RXN DAPDC\n",
      "META:NADPYROPHOSPHAT-RXN NPH\n",
      "META:5.4.99.17-RXN SQLC\n",
      "META:NICOTINAMID-RXN NNAM\n",
      "META:RXN0-1461 CPPPGO\n",
      "META:RXN0-5224 HCO3E\n",
      "META:ADENPRIBOSYLTRAN-RXN ADPT\n",
      "META:ACETOOHBUTSYN-RXN ACHBS\n",
      "META:RXN-14270 ADPT2\n",
      "META:XANPRIBOSYLTRAN-RXN XPPT\n",
      "META:ACETOLACTSYN-RXN ACLS\n",
      "META:IMIDPHOSDEHYD-RXN IGPDH\n",
      "META:ASPCARBTRANS-RXN ASPCT\n",
      "META:GLYOXII-RXN GLYOX\n",
      "META:UDPACYLGLCNACDEACETYL-RXN UHGADA\n",
      "META:NICONUCADENYLYLTRAN-RXN NNATr\n",
      "META:ALARACECAT-RXN ALAR\n"
     ]
    }
   ],
   "source": [
    "pttobigglist = []\n",
    "for keggr, biggr in ptmodelinbigg.items():\n",
    "    pttobigglist.append(biggr)\n",
    "    print(keggr, biggr)\n",
    "    \n",
    "pbigglist = list(set(pttobigglist))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## mapping carveme to BIGG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>mko_carveme_model</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>1f82baf4e50</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>1291</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>1909</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>882</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>1.0*Growth - 1.0*Growth_reverse_699ae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>cytosol, periplasm, extracellular space</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model mko_carveme_model at 0x1f82baf4e50>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import cobra\n",
    "carveme_model = cobra.io.read_sbml_model(org_name+'_model\\\\'+org_name+'_carveme_model.xml')\n",
    "carveme_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "carveme_rxnlist = [r.id for r in carveme_model.reactions]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3SALATAi\n",
      "      3-sulfino-alanine transaminase (irreversible)\n",
      "ACOTA\n",
      "      Acetylornithine transaminase\n",
      "ADPT\n",
      "      Adenine phosphoribosyltransferase\n",
      "AGTi\n",
      "      Alanine glyoxylate aminotransferase  irreversible\n",
      "AICART\n",
      "      Phosphoribosylaminoimidazolecarboxamide formyltransferase\n",
      "ALATA_D\n",
      "      D-alanine transaminase\n",
      "ALATA_L\n",
      "      L-alanine transaminase\n",
      "AMAOTr\n",
      "      Adenosylmethionine-8-amino-7-oxononanoate transaminase\n",
      "ANPRT\n",
      "      Anthranilate phosphoribosyltransferase\n",
      "ASO3tex\n",
      "      Arsenite transport via diffusion (extracellular to periplasm)\n",
      "ASPCT\n",
      "      Aspartate carbamoyltransferase\n",
      "ASPTA\n",
      "      Aspartate transaminase\n",
      "ASPTA6\n",
      "      E L-alanine-alpha-keto acid aminotransferase\n",
      "ASPt2_2pp\n",
      "      Aspartate transport via proton symport (2 H) (periplasm)\n",
      "ATPPRT\n",
      "      ATP phosphoribosyltransferase\n",
      "CA2abcpp\n",
      "      Calcium transport via ABC system (periplasm)\n",
      "CA2tex\n",
      "      Calcium (Ca+2) transport via diffusion (extracellular to periplasm)\n",
      "CD2t3pp\n",
      "      Cadmium (Cd+2) transport out via proton antiport (periplasm)\n",
      "CD2t4pp\n",
      "      Cadmium (Cd+2) transport out via proton antiport (periplasm)\n",
      "CD2tex\n",
      "      Cadmium (Cd+2) transport via diffusion (extracellular to periplasm)\n",
      "CD2tpp\n",
      "      Cadmium (+2) transport in via permease (no H+)\n",
      "CLt3_2pp\n",
      "      Chloride transport out via proton antiport (2:1) (periplasm)\n",
      "CLtex\n",
      "      Chloride (Cl-1) transport via diffusion (extracellular to periplasm)\n",
      "CO2t\n",
      "      CO2 transporter via diffusion\n",
      "COBALT2t3pp\n",
      "      Cobalt (Co+2) transport out via proton antiport (periplasm)\n",
      "COBALT2tex\n",
      "      Cobalt (Co+2) transport via diffusion (extracellular to periplasm)\n",
      "COBALT2tpp\n",
      "      Cobalt transport in via permease (no H+)\n",
      "CU2abcpp\n",
      "      Copper (Cu+2) ABC transporter (periplasm)\n",
      "CU2tex\n",
      "      Copper (Cu+2) transport via diffusion (extracellular to periplasm)\n",
      "CU2tpp\n",
      "      Copper transport in via permease (no H+)\n",
      "CYSTA\n",
      "      Cysteine transaminase\n",
      "DMATT\n",
      "      Dimethylallyltranstransferase\n",
      "FALDtpp\n",
      "      Formaldehyde transport via diffusion (periplasm)\n",
      "FE2Gabcpp\n",
      "      Iron (II) transport via ABC system (GTP) (periplasm)\n",
      "FE2tex\n",
      "      Iron (II) transport via diffusion (extracellular to periplasm)\n",
      "FE2tpp\n",
      "      Iron (+2) transport in via permease (no H+)\n",
      "FMNAT\n",
      "      FMN adenylyltransferase\n",
      "FOMETRi\n",
      "      Aminomethyltransferase\n",
      "FUMt2_2pp\n",
      "      Fumarate transport via proton symport (2 H) (periplasm)\n",
      "G1PACT\n",
      "      Glucosamine-1-phosphate N-acetyltransferase\n",
      "G1SAT\n",
      "      Glutamate-1-semialdehyde aminotransferase\n",
      "GALUi\n",
      "      UTP-glucose-1-phosphate uridylyltransferase (irreversible)\n",
      "GARFT\n",
      "      Phosphoribosylglycinamide formyltransferase\n",
      "GF6PTA\n",
      "      Glutamine-fructose-6-phosphate transaminase\n",
      "GHMT2r\n",
      "      Glycine hydroxymethyltransferase, reversible\n",
      "GLGC\n",
      "      Glucose-1-phosphate adenylyltransferase\n",
      "GLUPRT\n",
      "      Glutamine phosphoribosyldiphosphate amidotransferase\n",
      "GLYCtex\n",
      "      Glycerol transport via diffusion (extracellular to periplasm)\n",
      "GLYTA\n",
      "      Glycine transaminase\n",
      "GRTT\n",
      "      Geranyltranstransferase\n",
      "GUAPRT\n",
      "      Guanine phosphoribosyltransferase\n",
      "GUAtex\n",
      "      Guanine transport via diffusion (extracellular to periplasm)\n",
      "H2Otex\n",
      "      H2O transport via diffusion (extracellular to periplasm)\n",
      "H2Otpp\n",
      "      H2O transport via diffusion (periplasm)\n",
      "HBZOPT\n",
      "      Hydroxybenzoate octaprenyltransferase\n",
      "HCYSMT\n",
      "      Homocysteine S-methyltransferase\n",
      "HSST\n",
      "      Homoserine O-succinyltransferase\n",
      "HSTPT\n",
      "      Histidinol-phosphate transaminase\n",
      "HXPRT\n",
      "      Hypoxanthine phosphoribosyltransferase (Hypoxanthine)\n",
      "Htex\n",
      "      Proton transport via diffusion (extracellular to periplasm)\n",
      "ILETA\n",
      "      Isoleucine transaminase\n",
      "Kabcpp\n",
      "      Potassium ABC transporter (periplasm)\n",
      "Kt2pp\n",
      "      Potassium transport in via proton symport (periplasm)\n",
      "Kt3pp\n",
      "      Potassium transport out via proton antiport (periplasm)\n",
      "Ktex\n",
      "      Potassium transport via diffusion (extracellular to periplasm)\n",
      "LEUTA\n",
      "      Leucine transaminase\n",
      "MALt2_2pp\n",
      "      Malate transport via proton symport (2 H) (periplasm)\n",
      "MAN1PT\n",
      "      Mannose 1 phosphate guanylyltransferase\n",
      "MAN1PT2\n",
      "      Mannose-1-phosphate guanylyltransferase (GDP)\n",
      "MCOATA\n",
      "      Malonyl-CoA-ACP transacylase\n",
      "MEOHtex\n",
      "      Methanol transport via diffusion (extracellular to periplasm)\n",
      "MEPCT\n",
      "      2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase\n",
      "METAT\n",
      "      Methionine adenosyltransferase\n",
      "MG2tex\n",
      "      Magnesium (Mg+2) transport via diffusion (extracellular to periplasm)\n",
      "MG2tpp\n",
      "      Magnesium (+2) transport in via diffusion\n",
      "MG2uabcpp\n",
      "      Magnesium (Mg+2) ABC transporter (ubtake, periplasm)\n",
      "MN2t3pp\n",
      "      Manganese (Mn+2) transport out via proton antiport (periplasm)\n",
      "MN2tpp\n",
      "      Manganese transport in via permease (no H+)\n",
      "MOHMT\n",
      "      3-methyl-2-oxobutanoate hydroxymethyltransferase\n",
      "N2Otex\n",
      "      Nitrious oxide transport via diffusion (extracellular to periplasm)\n",
      "NADS2\n",
      "      Nicotinate-mononucleotide adenylyltransferase\n",
      "NADTRHD\n",
      "      NAD transhydrogenase\n",
      "NH4tex\n",
      "      Ammonia transport via diffusion (extracellular to periplasm)\n",
      "NH4tpp\n",
      "      Ammonia reversible transport (periplasm)\n",
      "NI2t3pp\n",
      "      Nickle (Ni+2) transport out via proton antiport (periplasm)\n",
      "NI2tex\n",
      "      Nickel transport via diffusion (extracellular to periplasm)\n",
      "NI2uabcpp\n",
      "      Nickel transport via ABC system (uptake, periplasm)\n",
      "NMNAT\n",
      "      Nicotinamide-nucleotide adenylyltransferase\n",
      "NO2tex\n",
      "      Nitrite transport via diffusion (extracellular to periplasm)\n",
      "NO3tex\n",
      "      Nitrate transport via diffusion (extracellular to periplasm)\n",
      "O2tex\n",
      "      Oxygen transport via diffusion (extracellular to periplasm)\n",
      "O2tpp\n",
      "      O2 transport via diffusion (periplasm)\n",
      "OHPBAT\n",
      "      O-Phospho-4-hydroxy-L-threonine:2-oxoglutarate aminotransferase\n",
      "ORNTAC\n",
      "      Ornithine transacetylase\n",
      "OROTt2_2pp\n",
      "      Orotate transport via proton symport (2 H) (periplasm)\n",
      "OROTtex\n",
      "      Orotate transport via diffusion (extracellular to periplasm)\n",
      "ORPT\n",
      "      Orotate phosphoribosyltransferase\n",
      "PAPPT3\n",
      "      Phospho-N-acetylmuramoyl-pentapeptide-transferase (meso-2,6-diaminopimelate)\n",
      "PHEMEabcpp\n",
      "      Protoheme transport via ABC system (periplasm)\n",
      "PHETA1\n",
      "      Phenylalanine transaminase\n",
      "PIt2rpp\n",
      "      Phosphate reversible transport via symport (periplasm)\n",
      "PItex\n",
      "      Phosphate transport via diffusion (extracellular to periplasm)\n",
      "PIuabcpp\n",
      "      Phosphate transport via ABC system (uptake, periplasm)\n",
      "PSCVT\n",
      "      3-phosphoshikimate 1-carboxyvinyltransferase\n",
      "PSERT\n",
      "      Phosphoserine transaminase\n",
      "PTPATi\n",
      "      Pantetheine-phosphate adenylyltransferase\n",
      "PTRCabcpp\n",
      "      Putrescine transport via ABC system (periplasm)\n",
      "PYRtex\n",
      "      Pyruvate transport via diffusion (extracellular to periplasm)\n",
      "SADT\n",
      "      Sulfate adenylyltransferase\n",
      "SDPTA\n",
      "      Succinyldiaminopimelate transaminase\n",
      "SELADT\n",
      "      Selenate adenylyltransferase\n",
      "SERAT\n",
      "      Serine O-acetyltransferase\n",
      "SO4tex\n",
      "      Sulfate transport via diffusion (extracellular to periplasm)\n",
      "SUCCt2_2pp\n",
      "      Succinate transport via proton symport (2 H) (periplasm)\n",
      "SULabcpp\n",
      "      Sulfate transport via ABC system (periplasm)\n",
      "TSULabcpp\n",
      "      Thiosulfate transport via ABC system (periplasm)\n",
      "TSULtex\n",
      "      Thiosulfate transport via diffusion (extracellular to periplasm)\n",
      "TYRTA\n",
      "      Tyrosine transaminase\n",
      "UAGCVT\n",
      "      UDP-N-acetylglucosamine 1-carboxyvinyltransferase\n",
      "UAGPT3\n",
      "      UDP-N-acetylglucosamine-N-acetylmuramyl-(pentapeptide)pyrophosphoryl-undecaprenol N-acetylglucosamine transferase\n",
      "UPP3MT\n",
      "      Uroporphyrinogen methyltransferase\n",
      "UPPRT\n",
      "      Uracil phosphoribosyltransferase\n",
      "UREAabcpp\n",
      "      Urea transport via ABC system (periplasm)\n",
      "UREAtex\n",
      "      Urea transport via diffusion (extracellular to periplasm)\n",
      "XPPT\n",
      "      Xanthine phosphoribosyltransferase\n",
      "ZN2t3pp\n",
      "      Zinc (Zn+2) transport out via proton antiport (periplasm)\n",
      "ZN2t4pp\n",
      "      Zinc transport out via antiport (periplasm)\n",
      "ZN2tpp\n",
      "      Zinc transport in via permease (no H+)\n"
     ]
    }
   ],
   "source": [
    "for r in carveme_model.reactions:\n",
    "    if 'trans' in r.name:\n",
    "        if r.id in iMcBath_rxnlist:\n",
    "            print(r.id)\n",
    "            print('     ',r.name)\n",
    "        #print(r.id)\n",
    "        #print('     ',r.name)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.save_json_model(carveme_model, org_name+'_model\\\\'+org_name+'_carveme_model.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "rxngenedir = {}\n",
    "for rxn in carveme_model.reactions:\n",
    "    if rxn.gene_reaction_rule:\n",
    "        add_to_dir(rxngenedir, rxn.gene_reaction_rule, rxn.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "for key, value in rxngenedir.items():\n",
    "    if type(value) is list:\n",
    "        rxngenedir[key] = select_rxn(unimodel, value)\n",
    "    if type(rxngenedir[key]) is list:\n",
    "        print(key, rxngenedir[key])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "cbigglist = list(set([value for key, value in rxngenedir.items() if value]))"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## add KEGG and pathway tool to Carveme Model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "145 87 394 316 268 242 200\n"
     ]
    }
   ],
   "source": [
    "kalone = [i for i in kbigglist if i not in pbigglist+cbigglist]\n",
    "palone = [i for i in pbigglist if i not in kbigglist+cbigglist]\n",
    "calone = [i for i in cbigglist if i not in kbigglist+pbigglist]\n",
    "kandp = [i for i in kbigglist if i in pbigglist]\n",
    "kandc = [i for i in kbigglist if i in cbigglist]\n",
    "pandc = [i for i in pbigglist if i in cbigglist]\n",
    "kandpandc = [i for i in kbigglist if i in pbigglist and i in cbigglist]\n",
    "\n",
    "print(len(kalone), len(palone), len(calone), len(kandp), len(kandc), len(pandc), len(kandpandc))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [],
   "source": [
    "calist = []\n",
    "for rxn in carveme_model.reactions:\n",
    "    calist.append(rxn.id)\n",
    "\n",
    "addlist = [i for i in kbigglist+pbigglist if i not in calist]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "metadata": {},
   "outputs": [],
   "source": [
    "for rxn in list(set(addlist)):\n",
    "    reaction = unimodel.reactions.get_by_id(rxn)\n",
    "    carveme_model.add_reactions([reaction.copy()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [],
   "source": [
    "for met in carveme_model.metabolites:\n",
    "    if met.compartment == None:\n",
    "        comlet = met.id.split('_')[-1]\n",
    "        if comlet in ['c','e','p']:\n",
    "            met.compartment = comlet\n",
    "        else:\n",
    "            print(met.id,'may not in prokaryotes!')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.save_json_model(carveme_model, org_name+'_model\\\\'+org_name+'_added_carveme_model.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1547 2147\n",
      "313 913\n"
     ]
    }
   ],
   "source": [
    "carveme_rxnlist = [i.id for i in carveme_model.reactions]\n",
    "iMcBath_rxnlist = [i.id for i in iMcBath_model.reactions]\n",
    "carvealone = [i for i in carveme_rxnlist if i not in iMcBath_rxnlist]\n",
    "iMcBathalone = [i for i in iMcBath_rxnlist if i not in carveme_rxnlist]\n",
    "carceandimc = [i for i in carveme_rxnlist if i in iMcBath_rxnlist]\n",
    "print(len(carvealone), len(carveme_rxnlist))\n",
    "print(len(iMcBathalone), len(iMcBath_rxnlist))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>Methylomonas_koyamae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>25c27ec0070</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>652</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>600</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>c, e, p</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model Methylomonas_koyamae at 0x25c27ec0070>"
      ]
     },
     "execution_count": 135,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_model = cobra.Model(org_full_name)\n",
    "for rxn in carceandimc:\n",
    "    reaction = unimodel.reactions.get_by_id(rxn)\n",
    "    new_model.add_reactions([reaction.copy()])\n",
    "\n",
    "for met in new_model.metabolites:\n",
    "    comlet = met.id.split('_')[-1]\n",
    "    if comlet in ['c','e','p']:\n",
    "        met.compartment = comlet\n",
    "    else:\n",
    "        print(met.id,'may not in prokaryotes!')\n",
    "\n",
    "new_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.save_json_model(new_model, org_name+'_model\\\\'+org_name+'_mapped_iMc_model.json')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## add KEGG and pathway tool reaction to new model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "kpnoti = [i for i in kbigglist+pbigglist if i not in iMcBath_rxnlist]\n",
    "inotkp = [i for i in iMcBath_rxnlist if i not in kbigglist+pbigglist]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "inotkpinc = [i for i in inotkp if i in carveme_rxnlist]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>Methylomonas_koyamae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>1f8331e7d90</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>819</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>839</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>c, p, e</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model Methylomonas_koyamae at 0x1f8331e7d90>"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_model = cobra.Model(org_full_name)\n",
    "for rxn in list(set(kbigglist+pbigglist+inotkpinc)):\n",
    "    reaction = unimodel.reactions.get_by_id(rxn)\n",
    "    new_model.add_reactions([reaction.copy()])\n",
    "\n",
    "for met in new_model.metabolites:\n",
    "    comlet = met.id.split('_')[-1]\n",
    "    if comlet in ['c','e','p']:\n",
    "        met.compartment = comlet\n",
    "    else:\n",
    "        print(met.id,'may not in prokaryotes!')\n",
    "\n",
    "new_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>Methylomonas_koyamae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>25c264f11f0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>995</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>1052</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>c, e, p</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model Methylomonas_koyamae at 0x25c264f11f0>"
      ]
     },
     "execution_count": 113,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_model = cobra.Model(org_full_name)\n",
    "for rxn in list(set(kbigglist+pbigglist+cbigglist)):\n",
    "    reaction = unimodel.reactions.get_by_id(rxn)\n",
    "    new_model.add_reactions([reaction.copy()])\n",
    "\n",
    "for met in new_model.metabolites:\n",
    "    comlet = met.id.split('_')[-1]\n",
    "    if comlet in ['c','e','p']:\n",
    "        met.compartment = comlet\n",
    "    else:\n",
    "        print(met.id,'may not in prokaryotes!')\n",
    "\n",
    "new_model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 159,
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.write_sbml_model(new_model, org_name+'_model\\\\'+org_name+'_bigg_model.xml')\n",
    "cobra.io.save_json_model(new_model, org_name+'_model\\\\'+org_name+'_bigg_model.json')"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## other"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "def printmyrxn(model, rxn):\n",
    "    if type(rxn) == str:\n",
    "        rxn = model.reactions.get_by_id(rxn)\n",
    "    print(rxn.id,',', rxn.name)\n",
    "    #print('\\t https://modelseed.org/biochem/reactions/'+rxn.id.split('_')[0])\n",
    "    print('\\t',rxn.reaction)\n",
    "    for meta, sto in rxn.metabolites.items():\n",
    "        metablic = model.metabolites.get_by_id(meta.id)\n",
    "        print('\\t\\t',metablic.id,'\\t', metablic.name,'\\t', sto)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "ename": "KeyError",
     "evalue": "'R00001'",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[18], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m printmyrxn(model, \u001b[39m'\u001b[39;49m\u001b[39mR00001\u001b[39;49m\u001b[39m'\u001b[39;49m)\n",
      "Cell \u001b[1;32mIn[17], line 3\u001b[0m, in \u001b[0;36mprintmyrxn\u001b[1;34m(model, rxn)\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mprintmyrxn\u001b[39m(model, rxn):\n\u001b[0;32m      2\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mtype\u001b[39m(rxn) \u001b[39m==\u001b[39m \u001b[39mstr\u001b[39m:\n\u001b[1;32m----> 3\u001b[0m         rxn \u001b[39m=\u001b[39m model\u001b[39m.\u001b[39;49mreactions\u001b[39m.\u001b[39;49mget_by_id(rxn)\n\u001b[0;32m      4\u001b[0m     \u001b[39mprint\u001b[39m(rxn\u001b[39m.\u001b[39mid,\u001b[39m'\u001b[39m\u001b[39m,\u001b[39m\u001b[39m'\u001b[39m, rxn\u001b[39m.\u001b[39mname)\n\u001b[0;32m      5\u001b[0m     \u001b[39m#print('\\t https://modelseed.org/biochem/reactions/'+rxn.id.split('_')[0])\u001b[39;00m\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\core\\dictlist.py:75\u001b[0m, in \u001b[0;36mDictList.get_by_id\u001b[1;34m(self, id)\u001b[0m\n\u001b[0;32m     73\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mget_by_id\u001b[39m(\u001b[39mself\u001b[39m, \u001b[39mid\u001b[39m: Union[Object, \u001b[39mstr\u001b[39m]) \u001b[39m-\u001b[39m\u001b[39m>\u001b[39m Object:\n\u001b[0;32m     74\u001b[0m \u001b[39m    \u001b[39m\u001b[39m\"\"\"Return the element with a matching id.\"\"\"\u001b[39;00m\n\u001b[1;32m---> 75\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mlist\u001b[39m\u001b[39m.\u001b[39m\u001b[39m__getitem__\u001b[39m(\u001b[39mself\u001b[39m, \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_dict[\u001b[39mid\u001b[39;49m])\n",
      "\u001b[1;31mKeyError\u001b[0m: 'R00001'"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1PPDCRc , L PipecolateNADP 2 oxidoreductase\n",
      "\t R02203\n",
      "34HPPOR , 4 Hydroxyphenylpyruvateoxygen oxidoreductase\n",
      "\t R02521\n",
      "3HAD80 , 3-hydroxyacyl-[acyl-carrier-protein] dehydratase (n-C8:0)\n",
      "\t R04537\n",
      "3OADPCOAT , 3-oxoadipate CoA-transferase\n",
      "\t R02990\n",
      "3OXCOAT , 3-oxoadipyl-CoA thiolase\n",
      "\t R00829\n",
      "4ABUTD , 4-Aminobutyraldehyde:NAD+ oxidoreductase\n",
      "\t R01986\n",
      "4HGLSD , 4HGLSD\n",
      "\t R05051\n",
      "4HTHRS , 4-Hydroxy-L-threonine synthase\n",
      "\t R05086\n",
      "6HNACMO , 6 hydroxynicotinic acid monooxygenase\n",
      "\t R08764\n",
      "AABTN , 4 acetamidobutyrate deacetylase\n",
      "\t R01987\n",
      "AACOAT , Acetoacetyl-CoA:acetate CoA-transferase\n",
      "\t R01357\n",
      "AADSACYCL , AAPSAS\n",
      "\t R02317\n",
      "AASAD3 , L-aminoadipate-semialdehyde dehydrogenase (NADH)\n",
      "\t R03102\n",
      "AATA , 2 aminoadipate transaminase\n",
      "\t R01939\n",
      "ABTA , 4-aminobutyrate transaminase\n",
      "\t R01648\n",
      "ABUTD , Aminobutyraldehyde dehydrogenase\n",
      "\t R02549\n",
      "ACACCT , Acetyl-CoA:acetoacetyl-CoA transferase\n",
      "\t R01359\n",
      "ACACT1r , Acetyl-CoA C-acetyltransferase\n",
      "\t R00238\n",
      "ACACT2r , Acetyl-CoA C-acyltransferase (butanoyl-CoA) (r)\n",
      "\t R01177\n",
      "ACACT4r , Acetyl-CoA C-acyltransferase (octanoyl-CoA) (r)\n",
      "\t R03778\n",
      "ACACT6r , Acetyl-CoA C-acyltransferase (dodecanoyl-CoA) (r)\n",
      "\t R03858\n",
      "ACACT7r , Acetyl-CoA C-acyltransferase (tetradecanoyl-CoA) (r)\n",
      "\t R03991\n",
      "ACCOAC , Acetyl-CoA carboxylase\n",
      "\t R00742\n",
      "ACGAMK , N-acetylglucosamine kinase\n",
      "\t R01201\n",
      "ACGK , Acetylglutamate kinase\n",
      "\t R02649\n",
      "ACGS , N-acetylglutamate synthase\n",
      "\t R00259\n",
      "ACHBS , 2-aceto-2-hydroxybutanoate synthase\n",
      "\t R08648\n",
      "ACKr , Acetate kinase\n",
      "\t R00315\n",
      "ACLS , Acetolactate synthase\n",
      "\t R00226\n",
      "ACM6PH , N-acetylmuramate 6-phosphate hydrolase\n",
      "\t R08555\n",
      "ACOAD1 , Acyl-CoA dehydrogenase (butanoyl-CoA)\n",
      "\t R01171\n",
      "ACOAD1f , Acyl-CoA dehydrogenase (butanoyl-CoA)\n",
      "\t R01175\n",
      "ACOAD3f , Acyl-CoA dehydrogenase (octanoyl-CoA)\n",
      "\t R03777\n",
      "ACOAD5f , Acyl-CoA dehydrogenase (dodecanoyl-CoA)\n",
      "\t R03857\n",
      "ACOAD6f , Acyl-CoA dehydrogenase (tetradecanoyl-CoA)\n",
      "\t R03990\n",
      "ACOAD7f , Acyl-CoA dehydrogenase (hexadecanoyl-CoA)\n",
      "\t R01279\n",
      "ACODA , Acetylornithine deacetylase\n",
      "\t R00669\n",
      "ACONT , Aconitate hydratase\n",
      "\t R01324\n",
      "ACONTa , Aconitase (half-reaction A, Citrate hydro-lyase)\n",
      "\t R01325\n",
      "ACONTb , Aconitase (half-reaction B, Isocitrate hydro-lyase)\n",
      "\t R01900\n",
      "ACOTA , Acetylornithine transaminase\n",
      "\t R02283\n",
      "ACP1_FMN , Acid phosphatase (FMN)\n",
      "\t R00548\n",
      "ACS , Acetyl-CoA synthetase\n",
      "\t R00235\n",
      "ACSERL , O3-Acetyl-L-serine acetate-lyase (adding hydrogen sulfide), cytosol\n",
      "\t R03601\n",
      "ACTD , Acetoin dehydrogenase\n",
      "\t R02855\n",
      "ACTDa , Acetoin dehydrogenase\n",
      "\t R02856\n",
      "ACYP_2 , Acylphosphatase (2)\n",
      "\t R00317\n",
      "ADA , Adenosine deaminase\n",
      "\t R01560\n",
      "ADCL , 4-aminobenzoate synthase\n",
      "\t R05553\n",
      "ADCPS1 , Adenosylcobinamide-phosphate synthase\n",
      "\t R07302\n",
      "ADCS , 4-amino-4-deoxychorismate synthase\n",
      "\t R01716\n",
      "ADD , Adenine deaminase\n",
      "\t R01244\n",
      "ADK1 , Adenylate kinase\n",
      "\t R00127\n",
      "ADMDC , Adenosylmethionine decarboxylase\n",
      "\t R00178\n",
      "ADNCYC , Adenylate cyclase\n",
      "\t R00089\n",
      "ADNK1 , Adenosine kinase\n",
      "\t R00185\n",
      "ADOCBIAH , MNXR85327\n",
      "\t R05226\n",
      "ADOCBIK , Adenosyl cobinamide kinase\n",
      "\t R05221\n",
      "ADPGLC , ADPglucose diphosphatase\n",
      "\t R00951\n",
      "ADPT , Adenine phosphoribosyltransferase\n",
      "\t R00190\n",
      "ADSK , Adenylyl-sulfate kinase\n",
      "\t R00509\n",
      "ADSL1r , Adenylsuccinate lyase\n",
      "\t R01083\n",
      "ADSL2r , Adenylosuccinate lyase\n",
      "\t R04559\n",
      "ADSS , Adenylosuccinate synthase\n",
      "\t R01135\n",
      "AGDC , N-acetylglucosamine-6-phosphate deacetylase\n",
      "\t R02059\n",
      "AGPOP , DGTP:pyruvate 2-O-phosphotransferase, cytosol\n",
      "\t R01858\n",
      "AGPR , N-acetyl-g-glutamyl-phosphate reductase\n",
      "\t R03443\n",
      "AGTi , Alanine glyoxylate aminotransferase  irreversible\n",
      "\t R00369\n",
      "AHCi , Adenosylhomocysteinase\n",
      "\t R00192\n",
      "AHSERL2 , O acetylhomoserine  thiol  lyase\n",
      "\t R01287\n",
      "AHSERL4 , Cysteine synthase  Thiosulfate\n",
      "\t R04859\n",
      "AICART , Phosphoribosylaminoimidazolecarboxamide formyltransferase\n",
      "\t R04560\n",
      "AIRC3 , Phosphoribosylaminoimidazole carboxylase (mutase rxn)\n",
      "\t R07405\n",
      "AKGDH , 2-Oxogluterate dehydrogenase\n",
      "\t R08549\n",
      "ALAALAD_1 , D alanine D alanine dipeptidase\n",
      "\t R07651\n",
      "ALAALAr , D-alanine-D-alanine ligase (reversible)\n",
      "\t R01150\n",
      "ALAD_L , L-alanine dehydrogenase\n",
      "\t R00396\n",
      "ALAGLUE , L-alanyl-gamma-glutamate epimerase\n",
      "\t R10938\n",
      "ALAR , Alanine racemase\n",
      "\t R00401\n",
      "ALATA_D2 , D-alanine transaminase\n",
      "\t R01147\n",
      "ALATA_L , L-alanine transaminase\n",
      "\t R00258\n",
      "ALCD19 , Alcohol dehydrogenase (glycerol)\n",
      "\t R01036\n",
      "ALCD19y , Alcohol dehydrogenase  glycerol  NADP\n",
      "\t R01041\n",
      "ALCD2ir , Alcohol dehydrogenase  reverse rxn  acetaldehyde   ethanol\n",
      "\t R00754\n",
      "ALCD2y , Alcohol dehydrogenase (ethanol, NADP)\n",
      "\t R00746\n",
      "ALDD19xr , Aldehyde dehydrogenase (phenylacetaldehyde, NAD)\n",
      "\t R02536\n",
      "ALDD20x , Aldehyde dehydrogenase (indole-3-acetaldehyde, NAD)\n",
      "\t R02678\n",
      "ALDD2x , Aldehyde dehydrogenase (acetaldehyde, NAD)\n",
      "\t R00710\n",
      "ALDD2y , Aldehyde dehydrogenase (acetaldehyde, NADP)\n",
      "\t R00711\n",
      "ALKP , Alkaline phosphatase\n",
      "\t R01010\n",
      "ALPHNH , Allophanate hydrolase\n",
      "\t R00005\n",
      "ALR2 , Aldose reductase (methylglyoxal)\n",
      "\t R02528\n",
      "AMANAPEr , N-acetylmannosamine 6-phosphate epimerase\n",
      "\t R02087\n",
      "AMAOTr , Adenosylmethionine-8-amino-7-oxononanoate transaminase\n",
      "\t R03231\n",
      "AMID2 , Amidase\n",
      "\t R02540\n",
      "AMPTASECG , Alanyl aminopeptidase (cys-gly)\n",
      "\t R00899\n",
      "ANPRT , Anthranilate phosphoribosyltransferase\n",
      "\t R01073\n",
      "ANS , Anthranilate synthase\n",
      "\t R00986\n",
      "ANS2 , Anthranilate synthase 2\n",
      "\t R00985\n",
      "AOXHEXCYCL , PPD2CSPp spontaneous cyclization reaction\n",
      "\t R04175\n",
      "AOXSr , 8-amino-7-oxononanoate synthase\n",
      "\t R03210\n",
      "APPAT , ATP:pantetheine-4'-phosphate adenylyltransferase\n",
      "\t R03035\n",
      "APRAUR , 5-amino-6-(5-phosphoribosylamino)uracil reductase\n",
      "\t R03458\n",
      "APRTO2 , N acetylputrescine oxygen oxireductase  deaminating\n",
      "\t R04025\n",
      "APSR , Adenosine 5'-phosphosulfate reductase;\n",
      "\t R07176\n",
      "APTNAT , 5-aminopentanoate:2-oxoglutarate aminotransferase\n",
      "\t R02274\n",
      "ARD , 1;2-dihydroxy-3-keto-5-methylthiopentene dioxygenase\n",
      "\t R07364\n",
      "ARGSL , Argininosuccinate lyase\n",
      "\t R01086\n",
      "ARGSS , Argininosuccinate synthase\n",
      "\t R01954\n",
      "AROH , L-Arogenate hydro-lyase\n",
      "\t R00691\n",
      "ASAD , Aspartate-semialdehyde dehydrogenase\n",
      "\t R02291\n",
      "ASCBPL , L-ascorbate 6-phosphate lactonase\n",
      "\t R07677\n",
      "ASNS1 , Asparagine synthase (glutamine-hydrolysing)\n",
      "\t R00578\n",
      "ASP1DC , Aspartate 1-decarboxylase\n",
      "\t R00489\n",
      "ASPCT , Aspartate carbamoyltransferase\n",
      "\t R01397\n",
      "ASPK , Aspartate kinase\n",
      "\t R00480\n",
      "ASPO1 , L-aspartate oxidase\n",
      "\t R00357\n",
      "ASPO6 , L-aspartate oxidase\n",
      "\t R00481\n",
      "ASPTA , Aspartate transaminase\n",
      "\t R00355\n",
      "ASPTA6 , E L-alanine-alpha-keto acid aminotransferase\n",
      "\t R00400\n",
      "ASR2 , Arsenate reductase (thioredoxin)\n",
      "\t R10927\n",
      "ATHRDHr , L-allo-threonine dehydrogenase\n",
      "\t R10851\n",
      "ATPHs , ATP amine hydrolysis (spontaneous)\n",
      "\t R00088\n",
      "ATPPRT , ATP phosphoribosyltransferase\n",
      "\t R01071\n",
      "BCOALIG , Benzoate CoA ligase\n",
      "\t R01422\n",
      "BDH , 3-HYDROXYBUTYRATE-DEHYDROGENASE-RXN\n",
      "\t R01361\n",
      "BETALDHy , Betaine-aldehyde dehydrogenase\n",
      "\t R02566\n",
      "BPNT , 3',5'-bisphosphate nucleotidase\n",
      "\t R00188\n",
      "BPNT2 , 3'-Phospho-5'-adenylyl sulfate 3'-phosphohydrolase\n",
      "\t R00508\n",
      "BTS , Butanol synthase\n",
      "\t R03544\n",
      "BTS_nadph , Butanol synthase (NADPH)\n",
      "\t R03545\n",
      "CAT , Catalase\n",
      "\t R00009\n",
      "CBPS , Carbamoyl-phosphate synthase (glutamine-hydrolysing)\n",
      "\t R00575\n",
      "CDPMEK , 4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol kinase\n",
      "\t R05634\n",
      "CEPA , Cellobiose phosphorylase\n",
      "\t ['R00952', 'R06056']\n",
      "CHCOAL , 6-carboxyhexanoate-CoA ligase\n",
      "\t R03209\n",
      "CHOLD , Choline dehydrogenase\n",
      "\t R08557\n",
      "CHOLSH , CHOLINE-SULFATASE-RXN\n",
      "\t R01028\n",
      "CHORM , Chorismate mutase\n",
      "\t R01715\n",
      "CHORS , Chorismate synthase\n",
      "\t R01714\n",
      "CITL , Citrate lyase\n",
      "\t R00362\n",
      "CMCMSAD , 4 carboxy 2 hydroxymuconate 6 semialdehyde dehydrogenase\n",
      "\t R04418\n",
      "CMHMI , 5 carboxymethyl 2 hydroxymuconate delta isomerase\n",
      "\t R04379\n",
      "CMLDC , 4 carboxymuconolactone decarboxylase\n",
      "\t R03470\n",
      "CPPPGO , Coproporphyrinogen oxidase (O2 required)\n",
      "\t R03220\n",
      "CPPPGO2 , Oxygen Independent coproporphyrinogen-III oxidase\n",
      "\t R06895\n",
      "CS , Citrate synthase\n",
      "\t R00351\n",
      "CSND , Cytosine deaminase\n",
      "\t R00974\n",
      "CTPS1 , CTP synthase  NH3\n",
      "\t R00571\n",
      "CTPS2 , CTP synthase (glutamine)\n",
      "\t R00573\n",
      "CYANST , Cyanide sulfurtransferase\n",
      "\t R01931\n",
      "CYNTAH , Cyanate aminohydrolase\n",
      "\t R10079\n",
      "CYSDS , Cysteine Desulfhydrase\n",
      "\t R00782\n",
      "CYSS , Cysteine synthase\n",
      "\t R00897\n",
      "CYSTA , Cysteine transaminase\n",
      "\t ['R00895', 'R00896']\n",
      "CYSTL , Cystathionine b-lyase\n",
      "\t R01286\n",
      "CYTK1 , Cytidylate kinase (CMP)\n",
      "\t R00512\n",
      "CYTK2 , Cytidylate kinase (dCMP)\n",
      "\t R01665\n",
      "DABAAT , DABAAT\n",
      "\t R06977\n",
      "DADA , Deoxyadenosine deaminase\n",
      "\t R02556\n",
      "DADK , Deoxyadenylate kinase\n",
      "\t R01547\n",
      "DALAOX , D amino acid oxidase\n",
      "\t R05861\n",
      "DAPDC , Diaminopimelate decarboxylase\n",
      "\t R00451\n",
      "DAPE , Diaminopimelate epimerase\n",
      "\t R02735\n",
      "DB4PS , 3,4-Dihydroxy-2-butanone-4-phosphate synthase\n",
      "\t R07281\n",
      "DBTS , Dethiobiotin synthase\n",
      "\t R03182\n",
      "DCTPD , DCTP deaminase\n",
      "\t R02325\n",
      "DCTPD2 , DCTP deaminase\n",
      "\t R00568\n",
      "DDPA , 3-deoxy-D-arabino-heptulosonate 7-phosphate synthetase\n",
      "\t R01826\n",
      "DDPGA , 2-dehydro-3-deoxy-phosphogluconate aldolase\n",
      "\t R00470\n",
      "DGK1 , Deoxyguanylate kinase (dGMP:ATP)\n",
      "\t R02090\n",
      "DGNSK , Deoxyguanosine kinase\n",
      "\t R01967\n",
      "DHAD1 , Dihydroxy-acid dehydratase (2,3-dihydroxy-3-methylbutanoate)\n",
      "\t R04441\n",
      "DHAD2 , Dihydroxy-acid dehydratase (2,3-dihydroxy-3-methylpentanoate)\n",
      "\t R05070\n",
      "DHAPT , Dihydroxyacetone phosphotransferase\n",
      "\t R01012\n",
      "DHBD , 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase\n",
      "\t R01505\n",
      "DHEDAA , 2,4-dihydroxyhept-2-ene-1,7-dioic acid aldolase\n",
      "\t R01647\n",
      "DHFR , Dihydrofolate reductase\n",
      "\t R00939\n",
      "DHFS , Dihydrofolate synthase\n",
      "\t R02237\n",
      "DHNPA2r , Dihydroneopterin aldolase reversible\n",
      "\t R03504\n",
      "DHORTS , Dihydroorotase\n",
      "\t R01993\n",
      "DHPDO , 3 4 dihydroxyphenylacetate 2 3 dioxygenase\n",
      "\t R03303\n",
      "DHPPDA2 , Diaminohydroxyphosphoribosylaminopryrimidine deaminase (25drapp)\n",
      "\t R03459\n",
      "DHPS2 , Dihydropteroate synthase\n",
      "\t R03067\n",
      "DHQS , 3-dehydroquinate synthase\n",
      "\t R03083\n",
      "DHQTi , 3-dehydroquinate dehydratase, irreversible\n",
      "\t R03084\n",
      "DMATT , Dimethylallyltranstransferase\n",
      "\t R01658\n",
      "DNMPPA , Dihydroneopterin monophosphate dephosphorylase\n",
      "\t R04621\n",
      "DNTPPA , Dihydroneopterin triphosphate pyrophosphatase\n",
      "\t R04638\n",
      "DPCOAK , Dephospho-CoA kinase\n",
      "\t R00130\n",
      "DPPS , Decaprenyl synthase\n",
      "\t R09249\n",
      "DPR , 2-dehydropantoate 2-reductase\n",
      "\t R02472\n",
      "DRBK , Deoxyribokinase\n",
      "\t R02750\n",
      "DRPA , Deoxyribose-phosphate aldolase\n",
      "\t R01066\n",
      "DTARTD , D(-)-tartrate dehydratase\n",
      "\t R00340\n",
      "DTMPK , DTMP kinase\n",
      "\t R02094\n",
      "DURIK1 , Deoxyuridine kinase (ATP:Deoxyuridine)\n",
      "\t R02099\n",
      "DUTCP , DUTP:cytidine 5'-phosphotransferase\n",
      "\t R02372\n",
      "DUTPDP , DUTP diphosphatase\n",
      "\t R02100\n",
      "DXPRIi , 1-deoxy-D-xylulose reductoisomerase\n",
      "\t R05688\n",
      "DXPS , 1-deoxy-D-xylulose 5-phosphate synthase\n",
      "\t R05636\n",
      "E4PD , Erythrose 4-phosphate dehydrogenase\n",
      "\t R01825\n",
      "ECOAH1 , 3-hydroxyacyl-CoA dehydratase (3-hydroxybutanoyl-CoA)\n",
      "\t R03026\n",
      "ECOAH12 , 3-hydroxyacyl-CoA dehydratase (3-hydroxyisobutyryl-CoA) (mitochondria)\n",
      "\t R04224\n",
      "ECOAH2 , 3-hydroxyacyl-CoA dehydratase (3-hydroxyhexanoyl-CoA)\n",
      "\t R04749\n",
      "ECOAH3 , 3-hydroxyacyl-CoA dehydratase (3-hydroxyoctanoyl-CoA)\n",
      "\t R04746\n",
      "ECOAH4 , 3-hydroxyacyl-CoA dehydratase (3-hydroxydecanoyl-CoA)\n",
      "\t R04744\n",
      "ECOAH5 , 3-hydroxyacyl-CoA dehydratase (3-hydroxydodecanoyl-CoA)\n",
      "\t R04170\n",
      "ECOAH6 , 3-hydroxyacyl-CoA dehydratase (3-hydroxytetradecanoyl-CoA)\n",
      "\t R04740\n",
      "ECOAH7 , 3-hydroxyacyl-CoA dehydratase (3-hydroxyhexadecanoyl-CoA)\n",
      "\t R04738\n",
      "ECOAH9ir , 2 Methylprop 2 enoyl CoA  2 Methylbut 2 enoyl CoA\n",
      "\t R04204\n",
      "ENO , Enolase\n",
      "\t R00658\n",
      "ENOPH , Enolase-phosphatase E1\n",
      "\t R07395\n",
      "ETHAAL , Ethanolamine ammonia-lyase\n",
      "\t R00749\n",
      "FACOAE160 , Fatty-acid-CoA thioesterase (hexadecanoate)\n",
      "\t R01274\n",
      "FACOAE180 , Fatty-acid-CoA thioesterase (octadecanoate)\n",
      "\t R08174\n",
      "FACOAE1829Z12Z , Linoleoyl-CoA hydrolase (18:2(9Z,12Z))\n",
      "\t R08177\n",
      "FACOAL160 , Fatty acid  CoA ligase  hexadecanoate\n",
      "\t R01280\n",
      "FBA , Fructose-bisphosphate aldolase\n",
      "\t R01068\n",
      "FBA2 , D Fructose 1 phosphate D glyceraldehyde 3 phosphate lyase\n",
      "\t R02568\n",
      "FBA3 , Sedoheptulose 1,7-bisphosphate D-glyceraldehyde-3-phosphate-lyase\n",
      "\t R01829\n",
      "FCLT , Ferrochelatase\n",
      "\t R00310\n",
      "FDH , Formate dehydrogenase\n",
      "\t R00519\n",
      "FDMO2 , FMNH2-dependent monooxygenase (methanesulfonate)\n",
      "\t R10206\n",
      "FEROpp , Ferroxidase\n",
      "\t R00078\n",
      "FFSD , Beta-fructofuranosidase\n",
      "\t R03921\n",
      "FMNAT , FMN adenylyltransferase\n",
      "\t R00161\n",
      "FMNRx , FMN reductase\n",
      "\t R05705\n",
      "FOLR2 , Folate reductase\n",
      "\t R02236\n",
      "FOMETRi , Aminomethyltransferase\n",
      "\t R02300\n",
      "FORAMD , FORAMD\n",
      "\t R00524\n",
      "FRTT , Farnesyltranstransferase\n",
      "\t R02061\n",
      "FTCD , Formimidoyltransferase cyclodeaminase reversible\n",
      "\t R02302\n",
      "FTHFCL , 5 formethyltetrahydrofolate cyclo ligase\n",
      "\t R02301\n",
      "FTHFLi , Formate-tetrahydrofolate ligase\n",
      "\t R00943\n",
      "FUM , Fumarase\n",
      "\t R01082\n",
      "FUMAC , Fumarylacetoacetase\n",
      "\t R01364\n",
      "G1PP , Glucose-1-phosphatase\n",
      "\t R00304\n",
      "G1SAT , Glutamate-1-semialdehyde aminotransferase\n",
      "\t R02272\n",
      "G3PD1ir , Glycerol 3 phosphate dehydrogenase  NAD\n",
      "\t R00842\n",
      "G3PD2 , Glycerol-3-phosphate dehydrogenase (NADP)\n",
      "\t R00844\n",
      "G3PT , Glycerol-3-phosphatase\n",
      "\t R00841\n",
      "G5SADs , L-glutamate 5-semialdehyde dehydratase (spontaneous)\n",
      "\t R03314\n",
      "G5SD , Glutamate-5-semialdehyde dehydrogenase\n",
      "\t R03313\n",
      "G6PBDH , Beta-D-Glucose-6-phosphate NADP+ 1-oxoreductase\n",
      "\t R02736\n",
      "G6PDH2r , Glucose 6-phosphate dehydrogenase\n",
      "\t R00835\n",
      "G6PI3 , Glucose 6 phosphate isomerase\n",
      "\t R03321\n",
      "GALCTD , Galactarate dehydratase\n",
      "\t R05608\n",
      "GALKr , Galactokinase\n",
      "\t R01092\n",
      "GALT , Galactose 1 phosphate uridylyltransferase\n",
      "\t R00502\n",
      "GALUi , UTP-glucose-1-phosphate uridylyltransferase (irreversible)\n",
      "\t R00289\n",
      "GAPD , Glyceraldehyde-3-phosphate dehydrogenase\n",
      "\t R01061\n",
      "GARFT , Phosphoribosylglycinamide formyltransferase\n",
      "\t R04325\n",
      "GCALDD , Glycolaldehyde dehydrogenase\n",
      "\t R01333\n",
      "GCCa , Glycine-cleavage complex\n",
      "\t R03425\n",
      "GCCb , Glycine cleavage complex\n",
      "\t R04125\n",
      "GCCc , Glycine-cleavage complex\n",
      "\t R03815\n",
      "GDPMNP , GDP-mannose phyrophosphatase\n",
      "\t R00881\n",
      "GHMT2r , Glycine hydroxymethyltransferase, reversible\n",
      "\t R00945\n",
      "GK1 , Guanylate kinase (GMP:ATP)\n",
      "\t R00332\n",
      "GLCOAS , Glutaryl-CoA synthetase\n",
      "\t R02402\n",
      "GLCRAL , 5-dehydro-4-deoxyglucarate aldolase\n",
      "\t R02754\n",
      "GLGC , Glucose-1-phosphate adenylyltransferase\n",
      "\t R00948\n",
      "GLNS , Glutamine synthetase\n",
      "\t R00253\n",
      "GLTPD , Galactitol-1-phosphate dehydrogenase\n",
      "\t R05571\n",
      "GLU5K , Glutamate 5-kinase\n",
      "\t R00239\n",
      "GLUCYS , Gamma-glutamylcysteine synthetase\n",
      "\t R00894\n",
      "GLUDxi , Glutamate dehydrogenase  NAD\n",
      "\t R00243\n",
      "GLUFT , 5-Formyltetrahydrofolate\n",
      "\t R03189\n",
      "GLUN , Glutaminase\n",
      "\t R00256\n",
      "GLUPRT , Glutamine phosphoribosyldiphosphate amidotransferase\n",
      "\t R01072\n",
      "GLUSx , Glutamate synthase  NADH2\n",
      "\t R00093\n",
      "GLUSy , Glutamate synthase (NADPH)\n",
      "\t R00114\n",
      "GLUTRR , Glutamyl-tRNA reductase\n",
      "\t R04109\n",
      "GLUTRS , Glutamyl-tRNA synthetase\n",
      "\t R05578\n",
      "GLXCBL , Glyoxalate carboligase\n",
      "\t R00013\n",
      "GLXCL , Glyoxalate carboligase\n",
      "\t R00013\n",
      "GLYALDDr , D-Glyceraldehyde dehydrogenase\n",
      "\t R01752\n",
      "GLYAT , Glycine C-acetyltransferase\n",
      "\t R00371\n",
      "GLYCK2 , Glycerate kinase\n",
      "\t R08572\n",
      "GLYCL , Glycine Cleavage System\n",
      "\t R01221\n",
      "GLYCLTDx , Glycolate dehydrogenase (NAD)\n",
      "\t R00717\n",
      "GLYCLTDy , Glycolate dehydrogenase (NADP)\n",
      "\t R00465\n",
      "GLYCTO1 , Glycolate oxidase\n",
      "\t R00475\n",
      "GLYK , Glycerol kinase\n",
      "\t R00847\n",
      "GLYO1 , Glycine oxidase\n",
      "\t R00366\n",
      "GLYOX , Hydroxyacylglutathione hydrolase\n",
      "\t R01736\n",
      "GLYOX_1 , Glyoxalase I; Ni-dependent\n",
      "\t R02530\n",
      "GLYTA , Glycine transaminase\n",
      "\t R00372\n",
      "GMPS , GMP synthase\n",
      "\t R01230\n",
      "GMPS2 , GMP synthase\n",
      "\t R01231\n",
      "GND , Phosphogluconate dehydrogenase\n",
      "\t R01528\n",
      "GPDDA1 , Glycerophosphodiester phosphodiesterase (Glycerophosphocholine)\n",
      "\t R01030\n",
      "GPDDA2 , Glycerophosphodiester phosphodiesterase (Glycerophosphoethanolamine)\n",
      "\t R01470\n",
      "GRTT , Geranyltranstransferase\n",
      "\t R02003\n",
      "GTHOr , Glutathione oxidoreductase\n",
      "\t R00115\n",
      "GTHPe , Glutathione peroxidase\n",
      "\t R00274\n",
      "GTHPi , Glutathione peridoxase\n",
      "\t R00274\n",
      "GTHRDH_syn , Glutathione hydralase (periplasmic)\n",
      "\t R00494\n",
      "GTHS , Glutathione synthetase\n",
      "\t R00497\n",
      "GTPCI , GTP cyclohydrolase I\n",
      "\t R00424\n",
      "GTPCII2 , GTP cyclohydrolase II (25drapp)\n",
      "\t R00425\n",
      "GTPDPK , GTP diphosphokinase\n",
      "\t R00429\n",
      "GUACYC , Guanylate cyclase\n",
      "\t R00434\n",
      "GUAD , Guanine deaminase\n",
      "\t R01676\n",
      "GUAPRT , Guanine phosphoribosyltransferase\n",
      "\t R01229\n",
      "GluForTx , Formimidoyltransferase cyclodeaminase\n",
      "\t R02287\n",
      "H2ASE_syn , Bidirectional [NiFe] Hydrogenase\n",
      "\t R07181\n",
      "HACD1 , 3-hydroxyacyl-CoA dehydrogenase (acetoacetyl-CoA)\n",
      "\t R01975\n",
      "HACD1_1 , 3 hydroxyacyl CoA dehydrogenase  acetoacetyl CoA\n",
      "\t R01975\n",
      "HACD1i , 3-hydroxyacyl-CoA dehydrogenase (acetoacetyl-CoA)\n",
      "\t R01975\n",
      "HACD2 , 3-hydroxyacyl-CoA dehydrogenase (3-oxohexanoyl-CoA)\n",
      "\t R04748\n",
      "HACD3 , 3-hydroxyacyl-CoA dehydrogenase (3-oxooctanoyl-CoA)\n",
      "\t R04745\n",
      "HACD4 , 3-hydroxyacyl-CoA dehydrogenase (3-oxodecanoyl-CoA)\n",
      "\t R04743\n",
      "HACD5 , 3-hydroxyacyl-CoA dehydrogenase (3-oxododecanoyl-CoA)\n",
      "\t R04741\n",
      "HACD6 , 3-hydroxyacyl-CoA dehydrogenase (3-oxotetradecanoyl-CoA)\n",
      "\t R04739\n",
      "HACD7 , 3-hydroxyacyl-CoA dehydrogenase (3-oxohexadecanoyl-CoA)\n",
      "\t R04737\n",
      "HACD9 , 3 hydroxyacyl CoA dehydrogenase  2 Methylacetoacetyl CoA\n",
      "\t R04203\n",
      "HBCO_nadp , 3-Hydroxybutanoyl-CoA:NADP+ oxidoreductase\n",
      "\t R01976\n",
      "HBZOPT , Hydroxybenzoate octaprenyltransferase\n",
      "\t R05615\n",
      "HCO3E , HCO3 equilibration reaction\n",
      "\t ['R00132', 'R10092']\n",
      "HCYSMT , Homocysteine S-methyltransferase\n",
      "\t R00650\n",
      "HDH , Histidinol dehydrogenase, cytosol\n",
      "\t R01158\n",
      "HEMEOS , Heme O synthase\n",
      "\t R07411\n",
      "HETZK , Hydroxyethylthiazole kinase\n",
      "\t R04448\n",
      "HEX1 , Hexokinase (D-glucose:ATP)\n",
      "\t R00299\n",
      "HEX7 , Hexokinase (D-fructose:ATP)\n",
      "\t R00867\n",
      "HGNTOR , Homogentisateoxygen 1 2 oxidoreductase  decyclizing\n",
      "\t R02519\n",
      "HISDr , Histidase r\n",
      "\t R01168\n",
      "HISTD , Histidinol dehydrogenase\n",
      "\t R01158\n",
      "HISTP , Histidinol-phosphatase\n",
      "\t R03013\n",
      "HOXPRx , 2 hydroxy 3 oxopropionate reductase  NAD\n",
      "\t R01745\n",
      "HPPK2 , 6-hydroxymethyl-dihydropterin pyrophosphokinase\n",
      "\t R03503\n",
      "HPROb , L hydroxyproline reductase  NADP\n",
      "\t R03293\n",
      "HPYRI , Hydroxypyruvate isomerase\n",
      "\t R01394\n",
      "HPYRRx , Hydroxypyruvate reductase (NADH)\n",
      "\t R01388\n",
      "HPYRRy , Hydroxypyruvate reductase (NADPH)\n",
      "\t R01392\n",
      "HSDxi , Homoserine dehydrogenase  NADH   irreversible\n",
      "\t R01773\n",
      "HSDy , Homoserine dehydrogenase (NADPH)\n",
      "\t R01775\n",
      "HSERTA , Homoserine O trans acetylase\n",
      "\t R01776\n",
      "HSK , Homoserine kinase\n",
      "\t R01771\n",
      "HSST , Homoserine O-succinyltransferase\n",
      "\t R01777\n",
      "HSTPT , Histidinol-phosphate transaminase\n",
      "\t R03243\n",
      "HSTPTr , Histidinol phosphate transaminase\n",
      "\t R03243\n",
      "HXPRT , Hypoxanthine phosphoribosyltransferase (Hypoxanthine)\n",
      "\t R01132\n",
      "ICDHyr , Isocitrate dehydrogenase (NADP)\n",
      "\t R00267\n",
      "ICHORS , Isochorismate synthase\n",
      "\t R01717\n",
      "ICHORT , Isochorismatase\n",
      "\t R03037\n",
      "IG3PS , Imidazole-glycerol-3-phosphate synthase\n",
      "\t R04558\n",
      "IGPDH , Imidazoleglycerol-phosphate dehydratase\n",
      "\t R03457\n",
      "IGPS , Indole-3-glycerol-phosphate synthase\n",
      "\t R03508\n",
      "ILETA , Isoleucine transaminase\n",
      "\t R02199\n",
      "IMPC , IMP cyclohydrolase\n",
      "\t R01127\n",
      "IMPD , IMP dehydrogenase\n",
      "\t R01130\n",
      "IPMD , 3-isopropylmalate dehydrogenase\n",
      "\t R04426\n",
      "IPPMIa , 3-isopropylmalate dehydratase\n",
      "\t R04001\n",
      "IPPMIb , 2-isopropylmalate hydratase\n",
      "\t R03968\n",
      "IPPS , 2-isopropylmalate synthase\n",
      "\t R01213\n",
      "IZPN , Imidazolonepropionase\n",
      "\t R02288\n",
      "KARA1 , Ketol-acid reductoisomerase (2,3-dihydroxy-3-methylbutanoate)\n",
      "\t R04439\n",
      "KAT1 , 3-ketoacyl-CoA thiolase\n",
      "\t R00238\n",
      "KG6PDC , 3-keto-L-gulonate 6-phosphate decarboxylase\n",
      "\t R07125\n",
      "LACZ , B-galactosidase\n",
      "\t ['R01100', 'R06114']\n",
      "LACZpp , B-galactosidase\n",
      "\t ['R01100', 'R06114']\n",
      "LALDO , D-Lactaldehyde NAD+ oxidoreductase (glutathione-formylating)\n",
      "\t R03082\n",
      "LCADi , Lactaldehyde dehydrogenase\n",
      "\t R01446\n",
      "LCARS , Lacaldehyde reductase (S-propane-1,2-diol forming)\n",
      "\t R02258\n",
      "LDH_L , L-lactate dehydrogenase\n",
      "\t R00703\n",
      "LEUTA , Leucine transaminase\n",
      "\t R01090\n",
      "LGTHL , Lactoylglutathione lyase\n",
      "\t R02530\n",
      "LPCOXc , L Pipecolateoxygen 1 6 oxidoreductase\n",
      "\t R02204\n",
      "LSERDHr , L-serine dehydrogenase\n",
      "\t R06126\n",
      "LTDCL , L-Tryptophan decarboxy-lyase\n",
      "\t R00685\n",
      "LTHRK , L threonine kinase\n",
      "\t R06531\n",
      "LYSDC , Lysine decarboxylase\n",
      "\t R00462\n",
      "MACACI , Maleylacetoacetate isomerase\n",
      "\t R03181\n",
      "MACCOAT , 2-Methylacetyl-CoA thiolase\n",
      "\t R00927\n",
      "MAL6PG , Maltose 6  phosphate glucosidase\n",
      "\t ['R00838', 'R06115']\n",
      "MALEI , Maleate cis trans isomerase\n",
      "\t R01087\n",
      "MAN1PT , Mannose 1 phosphate guanylyltransferase\n",
      "\t R00885\n",
      "MAN1PT2 , Mannose-1-phosphate guanylyltransferase (GDP)\n",
      "\t R00883\n",
      "MAN6PI , Mannose-6-phosphate isomerase\n",
      "\t R01819\n",
      "MBCOAi , 3-Methylbutanoyl-CoA:(acceptor) 2,3-oxidoreductase\n",
      "\t R04095\n",
      "MCCC , Methylcrotonoyl-CoA carboxylase\n",
      "\t R04138\n",
      "MCITD , 2-methylcitrate dehydratase\n",
      "\t R04424\n",
      "MCITL2 , Methylisocitrate lyase\n",
      "\t R00409\n",
      "MCITS , 2-methylcitrate synthase\n",
      "\t R00931\n",
      "MCOATA , Malonyl-CoA-ACP transacylase\n",
      "\t R01626\n",
      "MCPST , 3-mercaptopyruvate sulfurtransferase\n",
      "\t R03106\n",
      "MCSNAH , 5-Methylcytosine aminohydrolase\n",
      "\t R01411\n",
      "MDH , Malate dehydrogenase\n",
      "\t R00342\n",
      "MDRPD , 5-Methylthio-5-deoxy-D-ribulose 1-phosphate dehydratase\n",
      "\t R07392\n",
      "ME1 , Malic enzyme (NAD)\n",
      "\t R00214\n",
      "ME2 , Malic enzyme (NADP)\n",
      "\t R00216\n",
      "MECDPS , 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase\n",
      "\t R05637\n",
      "MEPCT , 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase\n",
      "\t R05633\n",
      "METAT , Methionine adenosyltransferase\n",
      "\t R00177\n",
      "METGL , Methionine g lyase\n",
      "\t R00654\n",
      "METS , Methionine synthase\n",
      "\t R00946\n",
      "METSOXR1 , L-methionine-S-oxide reductase\n",
      "\t R02025\n",
      "METSOXR2 , L-methionine-R-sulfoxide reductase\n",
      "\t R07608\n",
      "MGSA , Methylglyoxal synthase\n",
      "\t R01016\n",
      "MI1PP , Myo-inositol 1-phosphatase\n",
      "\t R01185\n",
      "MICITDr , 2-methylisocitrate dehydratase\n",
      "\t R04425\n",
      "MMCD , Methylmalonyl-CoA decarboxylase\n",
      "\t R00923\n",
      "MME , Methylmalonyl-CoA epimerase\n",
      "\t R02765\n",
      "MMM2 , Methylmalonyl-CoA mutase\n",
      "\t R00833\n",
      "MMSAD1 , Methylmalonate semialdehyde dehydrogenase\n",
      "\t R00922\n",
      "MOHMT , 3-methyl-2-oxobutanoate hydroxymethyltransferase\n",
      "\t R01226\n",
      "MTAP , 5  methylthioadenosine phosphorylase\n",
      "\t R01402\n",
      "MTHFC , Methenyltetrahydrofolate cyclohydrolase\n",
      "\t R01655\n",
      "MTHFD , Methylenetetrahydrofolate dehydrogenase (NADP)\n",
      "\t R01220\n",
      "MTHFD2i , Methylenetetrahydrofolate dehydrogenase  NAD\n",
      "\t R01218\n",
      "MTHFR2 , 5,10-methylenetetrahydrofolate reductase (NADH)\n",
      "\t R07168\n",
      "MTI , Maltose trehalose isomerase\n",
      "\t ['R01557', 'R06218']\n",
      "MTRI , 5-methylthioribose-1-phosphate isomerase\n",
      "\t R04420\n",
      "MUCCY_kt , 3 carboxy cis cis muconate cycloisomerase\n",
      "\t R03307\n",
      "NABTNO , N4 AcetylaminobutanalNAD oxidoreductase\n",
      "\t R05050\n",
      "NADDP , NAD diphosphatase\n",
      "\t R00103\n",
      "NADHPO , NADH peroxidase\n",
      "\t R00090\n",
      "NADK , NAD kinase\n",
      "\t R00104\n",
      "NADN , NAD nucleosidase\n",
      "\t R00102\n",
      "NADS2 , Nicotinate-mononucleotide adenylyltransferase\n",
      "\t R00257\n",
      "NADTRHD , NAD transhydrogenase\n",
      "\t R00112\n",
      "NDPK1 , Nucleoside-diphosphate kinase (ATP:GDP)\n",
      "\t R00330\n",
      "NDPK2 , Nucleoside-diphosphate kinase (ATP:UDP)\n",
      "\t R00156\n",
      "NDPK3 , Nucleoside-diphosphate kinase (ATP:CDP)\n",
      "\t R00570\n",
      "NDPK4 , Nucleoside-diphosphate kinase (ATP:dTDP)\n",
      "\t R02093\n",
      "NDPK5 , Nucleoside-diphosphate kinase (ATP:dGDP)\n",
      "\t R01857\n",
      "NDPK6 , Nucleoside-diphosphate kinase (ATP:dUDP)\n",
      "\t R02331\n",
      "NDPK7 , Nucleoside-diphosphate kinase (ATP:dCDP)\n",
      "\t R02326\n",
      "NDPK8 , Nucleoside-diphosphate kinase (ATP:dADP)\n",
      "\t R01137\n",
      "NDPK9 , Nucleoside diphosphate kinase  ATPIDP\n",
      "\t R00722\n",
      "NFMLDF , N Formyl maleate deformylase\n",
      "\t R09126\n",
      "NHFRBO , NADH:flavorubredoxin oxidoreductase\n",
      "\t R02492\n",
      "NMNAT , Nicotinamide-nucleotide adenylyltransferase\n",
      "\t R00137\n",
      "NMNDA , Nicotinamide-nucleotide amidase\n",
      "\t R02322\n",
      "NMNHYD , Nicotinamide ribonucleotide phosphohydrolase\n",
      "\t R02323\n",
      "NMNR , Nmn hydrolysis\n",
      "\t R02323\n",
      "NNAM , Nicotinamidase\n",
      "\t R01268\n",
      "NNATr , Nicotinate-nucleotide adenylyltransferase\n",
      "\t R03005\n",
      "NNDPR , Nicotinate-nucleotide diphosphorylase (carboxylating)\n",
      "\t R03348\n",
      "NPHS , Naphthoate synthase\n",
      "\t R04150\n",
      "NT5C , 5'-nucleotidase\n",
      "\t R03346\n",
      "NTD10 , 5'-nucleotidase (XMP)\n",
      "\t R02719\n",
      "NTD11 , 5'-nucleotidase (IMP)\n",
      "\t R01126\n",
      "NTD2 , 5'-nucleotidase (UMP)\n",
      "\t R00963\n",
      "NTD4 , 5'-nucleotidase (CMP)\n",
      "\t R00511\n",
      "NTD6 , 5'-nucleotidase (dAMP)\n",
      "\t R02088\n",
      "NTD7 , 5'-nucleotidase (AMP)\n",
      "\t R00183\n",
      "NTD9 , 5'-nucleotidase (GMP)\n",
      "\t R01227\n",
      "NTP1 , Nucleoside-triphosphatase (ATP)\n",
      "\t R00086\n",
      "NTP3 , Nucleoside-triphosphatase (GTP)\n",
      "\t R00335\n",
      "NTP5 , Nucleoside-triphosphatase (CTP)\n",
      "\t R00569\n",
      "NTPP1 , Nucleoside triphosphate pyrophosphorylase (dgtp)\n",
      "\t R01855\n",
      "NTPP10 , Nucleoside triphosphate pyrophosphorylase (ditp)\n",
      "\t R03531\n",
      "NTPP11 , Nucleoside triphosphate pyrophosphorylase (xtp)\n",
      "\t R02720\n",
      "NTPP2 , Nucleoside triphosphate pyrophosphorylase (gtp)\n",
      "\t R00426\n",
      "NTPP3 , Nucleoside triphosphate pyrophosphorylase (dctp)\n",
      "\t R01668\n",
      "NTPP4 , Nucleoside triphosphate pyrophosphorylase (ctp)\n",
      "\t R00515\n",
      "NTPP6 , Nucleoside triphosphate pyrophosphorylase (atp)\n",
      "\t R00087\n",
      "NTPP7 , Nucleoside triphosphate pyrophosphorylase (dttp)\n",
      "\t R11323\n",
      "NTPP8 , Nucleoside triphosphate pyrophosphorylase (utp)\n",
      "\t R00662\n",
      "NTPP9 , Nucleoside triphosphate pyrophosphorylase (itp)\n",
      "\t R00720\n",
      "NTPTP1 , Nucleoside triphosphate tripolyhydrolase\n",
      "\t R01856\n",
      "NTRIR2x , Nitrite Reductase (NADH)\n",
      "\t R00787\n",
      "NTRIR2y , Nitrite Reductase (NADPH)\n",
      "\t R00789\n",
      "OCBT , Ornithine carbamoyltransferase\n",
      "\t R01398\n",
      "OCBT_1 , Ornithine carbamoyltransferase\n",
      "\t R01398\n",
      "OCOAT1 , 3-oxoacid CoA-transferase (Succinyl-CoA: acetoacetate)\n",
      "\t R00410\n",
      "OCTDPS , Octaprenyl pyrophosphate synthase\n",
      "\t R09248\n",
      "OHEDH , 2-oxo-hept-3-ene-1,7-dioate hydratase\n",
      "\t R04132\n",
      "OIVD1r , 2-oxoisovalerate dehydrogenase (acylating; 4-methyl-2-oxopentaoate)\n",
      "\t R01651\n",
      "OIVD2 , 2-oxoisovalerate dehydrogenase (acylating; 3-methyl-2-oxobutanoate)\n",
      "\t R01210\n",
      "OMCDC , 2-Oxo-4-methyl-3-carboxypentanoate decarboxylation\n",
      "\t R01652\n",
      "OMPDC , Orotidine-5'-phosphate decarboxylase\n",
      "\t R00965\n",
      "OPHBDC , Octaprenyl-hydroxybenzoate decarboxylase\n",
      "\t R04986\n",
      "ORNDC , Ornithine Decarboxylase\n",
      "\t R00670\n",
      "ORNTA , Ornithine transaminase\n",
      "\t R00667\n",
      "ORNTAC , Ornithine transacetylase\n",
      "\t R02282\n",
      "ORNTA_1 , Ornithine transaminase\n",
      "\t R00667\n",
      "ORPT , Orotate phosphoribosyltransferase\n",
      "\t R01870\n",
      "OXADC , Oxalate decarboxylase\n",
      "\t R00522\n",
      "OXOAEL , 3-oxoadipate enol-lactone hydrolase\n",
      "\t R02991\n",
      "OXPTNDH , Glutarate-semialdehyde:NAD+ oxidoreductase\n",
      "\t R02401\n",
      "P5CD , 1-pyrroline-5-carboxylate dehydrogenase\n",
      "\t R00707\n",
      "P5CR , Pyrroline-5-carboxylate reductase\n",
      "\t R01251\n",
      "PANTS , Pantothenate synthase\n",
      "\t R02473\n",
      "PAPSR , Phosphoadenylyl-sulfate reductase (thioredoxin)\n",
      "\t R02021\n",
      "PBUTT , Phosphate butyryltransferase\n",
      "\t R01174\n",
      "PCADYOX , Protocatechuate 3 4 dioxygenase\n",
      "\t R01631\n",
      "PCNO , Propanoyl-CoA:NADP+ 2-oxidoreductase\n",
      "\t R00919\n",
      "PDE1 , 3',5'-cyclic-nucleotide phosphodiesterase\n",
      "\t R00191\n",
      "PDE4 , 3',5'-cyclic-nucleotide phosphodiesterase\n",
      "\t R01234\n",
      "PDH , Pyruvate dehydrogenase\n",
      "\t R00209\n",
      "PDHcr , Pyruvate dehydrogenase (dihydrolipoamide dehydrogenase) reversible\n",
      "\t R01698\n",
      "PDX5POi , Pyridoxine 5'-phosphate oxidase\n",
      "\t R00278\n",
      "PEAMNO , Phenethylamine oxidase\n",
      "\t R02613\n",
      "PEPCK_re , Phosphoenolpyruvate carboxykinase (GTP)\n",
      "\t R00431\n",
      "PFK_3 , Phosphofructokinase (s7p)\n",
      "\t R01843\n",
      "PGAMT , Phosphoglucosamine mutase\n",
      "\t R02060\n",
      "PGCD , Phosphoglycerate dehydrogenase\n",
      "\t R01513\n",
      "PGK , Phosphoglycerate kinase\n",
      "\t R01512\n",
      "PGL , 6-phosphogluconolactonase\n",
      "\t R02035\n",
      "PGLYCP , Phosphoglycolate phosphatase\n",
      "\t R01334\n",
      "PGM , Phosphoglycerate mutase\n",
      "\t R01518\n",
      "PGMT , Phosphoglucomutase\n",
      "\t R08639\n",
      "PHCD , L-1-pyrroline-3-hydroxy-5-carboxylate dehydrogenase\n",
      "\t R04444\n",
      "PHCHGS , PHCHGS\n",
      "\t R04443\n",
      "PHETA1 , Phenylalanine transaminase\n",
      "\t R00694\n",
      "PHYCBOXL , L-Phenylalanine carboxy-lyase\n",
      "\t R00699\n",
      "PMANM , Phosphomannomutase\n",
      "\t R01818\n",
      "PMDPHT , Pyrimidine phosphatase\n",
      "\t R07280\n",
      "PNCDC , PNCDC\n",
      "\t R02972\n",
      "PNP , Purine nucleoside phosphorylase\n",
      "\t R02294\n",
      "PNTK , Pantothenate kinase\n",
      "\t R03018\n",
      "PPA , Inorganic diphosphatase\n",
      "\t R00004\n",
      "PPA2 , Inorganic triphosphatase\n",
      "\t R00138\n",
      "PPA_1pp , Inorganic diphosphatase (one proton translocation) (periplasm)\n",
      "\t R00004\n",
      "PPBNGS , Porphobilinogen synthase\n",
      "\t R00036\n",
      "PPC , Phosphoenolpyruvate carboxylase\n",
      "\t R00345\n",
      "PPCDC , Phosphopantothenoylcysteine decarboxylase\n",
      "\t R03269\n",
      "PPCK , Phosphoenolpyruvate carboxykinase\n",
      "\t R00341\n",
      "PPDK , Pyruvate,phosphate dikinase.\n",
      "\t R00206\n",
      "PPGPPDP , Guanosine-3',5'-bis(diphosphate) 3'-diphosphatase\n",
      "\t R00336\n",
      "PPNCL2 , Phosphopantothenate-cysteine ligase\n",
      "\t R04231\n",
      "PPND , Prephenate dehydrogenase\n",
      "\t R01728\n",
      "PPNDH , Prephenate dehydratase\n",
      "\t R01373\n",
      "PPOR , Phenylpyruvate:oxygen oxidoreductase (hydroxylating,decarboxylating)\n",
      "\t R01372\n",
      "PPS , Phosphoenolpyruvate synthase\n",
      "\t R00199\n",
      "PRAGSr , Phosphoribosylglycinamide synthase\n",
      "\t R04144\n",
      "PRAIi , Phosphoribosylanthranilate isomerase (irreversible)\n",
      "\t R03509\n",
      "PRAMPC , Phosphoribosyl-AMP cyclohydrolase\n",
      "\t R04037\n",
      "PRASCSi , Phosphoribosylaminoimidazolesuccinocarboxamide synthase\n",
      "\t R04591\n",
      "PRATPP , Phosphoribosyl-ATP pyrophosphatase\n",
      "\t R04035\n",
      "PRMICI , 1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino)imidazole-4-carboxamide isomerase\n",
      "\t R04640\n",
      "PROD2 , Proline dehydrogenase\n",
      "\t R10507\n",
      "PSCVT , 3-phosphoshikimate 1-carboxyvinyltransferase\n",
      "\t R03460\n",
      "PSERT , Phosphoserine transaminase\n",
      "\t R04173\n",
      "PSP_L , Phosphoserine phosphatase (L-serine)\n",
      "\t R00582\n",
      "PTA2 , Phosphate acetyltransferase\n",
      "\t R00921\n",
      "PTAr , Phosphotransacetylase\n",
      "\t R00230\n",
      "PTPATi , Pantetheine-phosphate adenylyltransferase\n",
      "\t R03035\n",
      "PTRCAT1 , Putrescine acetyltransferase\n",
      "\t R01154\n",
      "PTRCTA , Putrescine Transaminase\n",
      "\t R01155\n",
      "PUNP3 , Purine-nucleoside phosphorylase (Guanosine)\n",
      "\t R02147\n",
      "PUNP6 , Purine-nucleoside phosphorylase (Deoxyinosine)\n",
      "\t R02748\n",
      "PUNP7 , Purine-nucleoside phosphorylase (Xanthosine)\n",
      "\t R02297\n",
      "PUTA3 , PUTA3\n",
      "\t R00245\n",
      "PYAM5PO , Pyridoxamine 5'-phosphate oxidase\n",
      "\t R00277\n",
      "PYDXNO , Pyridoxine oxidase\n",
      "\t R01711\n",
      "PYDXPP , Pyridoxal 5-phosphate phosphatase\n",
      "\t R00173\n",
      "PYDXS , Pyridoxal 5-phosphate synthase\n",
      "\t R07456\n",
      "PYK , Pyruvate kinase\n",
      "\t R00200\n",
      "PYK2 , Pyruvate kinase(2)\n",
      "\t R00659\n",
      "PYK5 , Pyruvate kinase(5)\n",
      "\t R00724\n",
      "PYNP1 , Pyrimidine-nucleoside phosphorylase (cytosine)\n",
      "\t R02296\n",
      "PYNP2r , Pyrimidine-nucleoside phosphorylase (uracil)\n",
      "\t R01876\n",
      "PYRDOX , 2 5 dihydroxypyridine dioxygenase\n",
      "\t R09125\n",
      "PYRZAM , Pyrazinamidase\n",
      "\t R03540\n",
      "QULNS , Quinolinate synthase\n",
      "\t R04292\n",
      "RBFK , Riboflavin kinase\n",
      "\t R00549\n",
      "RBFSa , Riboflavin synthase\n",
      "\t R04457\n",
      "RBFSb , Riboflavin synthase\n",
      "\t R00066\n",
      "RBP4E , L-ribulose-phosphate 4-epimerase\n",
      "\t R05850\n",
      "RNDR1 , Ribonucleoside-diphosphate reductase (ADP)\n",
      "\t R02017\n",
      "RNDR2 , Ribonucleoside-diphosphate reductase (GDP)\n",
      "\t R02019\n",
      "RNDR3 , Ribonucleoside-diphosphate reductase (CDP)\n",
      "\t R02024\n",
      "RNDR4 , Ribonucleoside-diphosphate reductase (UDP)\n",
      "\t R02018\n",
      "RNTR1 , Ribonucleoside triphosphate reductase  ATP\n",
      "\t R02014\n",
      "RNTR2 , Ribonucleoside triphosphate reductase  GTP\n",
      "\t R02020\n",
      "RNTR3 , Ribonucleoside triphosphate reductase  CTP\n",
      "\t R02022\n",
      "RNTR4 , Ribonucleoside triphosphate reductase  UTP\n",
      "\t R02023\n",
      "RPE , Ribulose 5-phosphate 3-epimerase\n",
      "\t R01529\n",
      "SADT , Sulfate adenylyltransferase\n",
      "\t R00529\n",
      "SALADC2 , Sulfinoalanine decarboxylase\n",
      "\t R01682\n",
      "SARCOX , Sarcosine oxidase\n",
      "\t R00610\n",
      "SBTD_D2 , D sorbitol dehydrogenase  D fructose producing\n",
      "\t R00875\n",
      "SBTPD , Sorbitol-6-phosphate dehydrogenase\n",
      "\t R05607\n",
      "SDPDS , Succinyl-diaminopimelate desuccinylase\n",
      "\t R02734\n",
      "SDPTA , Succinyldiaminopimelate transaminase\n",
      "\t R04475\n",
      "SERAT , Serine O-acetyltransferase\n",
      "\t R00586\n",
      "SHCHD2 , Sirohydrochlorin dehydrogenase (NAD)\n",
      "\t R03947\n",
      "SHCHF , Sirohydrochlorin ferrochelatase\n",
      "\t R02864\n",
      "SHK3Dr , Shikimate dehydrogenase\n",
      "\t R02413\n",
      "SHKK , Shikimate kinase\n",
      "\t R02412\n",
      "SHSL1 , O-succinylhomoserine lyase (L-cysteine)\n",
      "\t R03260\n",
      "SHSL2r , O-succinylhomoserine lyase (H2S)\n",
      "\t R01288\n",
      "SLCYSS , O-acetyl-L-serine sulfhydrylase\n",
      "\t R03132\n",
      "SPMS , Spermidine synthase\n",
      "\t R01920\n",
      "SPODM , Superoxide dismutase\n",
      "\t R00275\n",
      "SPT_syn , Serine-glyoxilate aminotransferase\n",
      "\t R00588\n",
      "SPTc , Serine-pyruvate aminotransferase\n",
      "\t R00585\n",
      "SSALx , Succinate-semialdehyde dehydrogenase (NAD)\n",
      "\t R00713\n",
      "SSALy , Succinate-semialdehyde dehydrogenase (NADP)\n",
      "\t R00714\n",
      "SSCOARy , Succinyl-coA reductase (NADPH)\n",
      "\t R09280\n",
      "SUCBZL , O-succinylbenzoate-CoA ligase\n",
      "\t R04030\n",
      "SUCBZS , O-succinylbenzoate-CoA synthase\n",
      "\t R04031\n",
      "SUCOAS , Succinyl-CoA synthetase (ADP-forming)\n",
      "\t R00405\n",
      "SULR , Sulfite reductase (NADPH2)\n",
      "\t R00858\n",
      "TALA , Transaldolase\n",
      "\t R01827\n",
      "TDP , Thiamin pyrophosphatase\n",
      "\t R00615\n",
      "TGBPA , Tagatose-bisphosphate aldolase\n",
      "\t R01069\n",
      "THDPS , Tetrahydrodipicolinate succinylase\n",
      "\t R04365\n",
      "THFAT , Tetrahydrofolate aminomethyltransferase\n",
      "\t R02300\n",
      "THRA , Threonine aldolase\n",
      "\t R00751\n",
      "THRD , L-threonine dehydrogenase\n",
      "\t R01465\n",
      "THRS , Threonine synthase\n",
      "\t R01466\n",
      "TKT2 , Transketolase\n",
      "\t R01830\n",
      "TMDPP , Thymidine phosphorylase\n",
      "\t R01570\n",
      "TMDS , Thymidylate synthase\n",
      "\t R02101\n",
      "TMPK , Thiamine-phosphate kinase\n",
      "\t R00617\n",
      "TMPPP , Thiamine-phosphate diphosphorylase\n",
      "\t R03223\n",
      "TPI , Triose-phosphate isomerase\n",
      "\t R01015\n",
      "TRDR , Thioredoxin reductase (NADPH)\n",
      "\t R02016\n",
      "TRE6PH , Trehalose-6-phosphate hydrolase\n",
      "\t ['R00837', 'R06113']\n",
      "TRE6PP , Trehalose-phosphatase\n",
      "\t ['R02778', 'R06228']\n",
      "TRE6PS , Alpha,alpha-trehalose-phosphate synthase (UDP-forming)\n",
      "\t ['R00836', 'R06043']\n",
      "TRPAS2 , Tryptophanase (L-tryptophan)\n",
      "\t R00673\n",
      "TRPS2 , Tryptophan synthase (indole)\n",
      "\t R00674\n",
      "TRPS3 , Tryptophan synthase (indoleglycerol phosphate)\n",
      "\t R02340\n",
      "TRYPTAOX , Tryptamine:oxygen oxidoreductase(deaminating)(flavin-containing)\n",
      "\t R02173\n",
      "TYRTA , Tyrosine transaminase\n",
      "\t R00734\n",
      "TYRTAi , Tyrosine transaminase  irreversible\n",
      "\t R00734\n",
      "UAG4Ei , UDP-N-acetylglucosamine 4-epimerase\n",
      "\t R00418\n",
      "UAGCVT , UDP-N-acetylglucosamine 1-carboxyvinyltransferase\n",
      "\t R00660\n",
      "UAGPT3 , UDP-N-acetylglucosamine-N-acetylmuramyl-(pentapeptide)pyrophosphoryl-undecaprenol N-acetylglucosamine transferase\n",
      "\t ['R05032', 'R06174']\n",
      "UAMAGS , UDP-N-acetylmuramoyl-L-alanyl-D-glutamate synthetase\n",
      "\t R02783\n",
      "UAMAS , UDP-N-acetylmuramoyl-L-alanine synthetase\n",
      "\t R03193\n",
      "UAPGR , UDP-N-acetylenolpyruvoylglucosamine reductase\n",
      "\t R03192\n",
      "UDPG4E , UDPglucose 4-epimerase\n",
      "\t R00291\n",
      "UGLT , UDPglucose--hexose-1-phosphate uridylyltransferase\n",
      "\t R00955\n",
      "UMPK , UMP kinase\n",
      "\t R00158\n",
      "UPP3MT , Uroporphyrinogen methyltransferase\n",
      "\t R03194\n",
      "UPP3S , Uroporphyrinogen-III synthase\n",
      "\t R03165\n",
      "UPPDC1 , Uroporphyrinogen decarboxylase (uroporphyrinogen III)\n",
      "\t R03197\n",
      "URCN , Urocanase\n",
      "\t R02914\n",
      "UREASE , Urea carboxylase\n",
      "\t R00774\n",
      "URIDK2r , Uridylate kinase (dUMP)\n",
      "\t R02098\n",
      "VACOAI , Vinylacetyl-CoA Delta-isomerase\n",
      "\t R03031\n",
      "VALTA , Valine transaminase\n",
      "\t R01214\n",
      "VNTDM , Vanillateoxygen oxidoreductase  demethylating\n",
      "\t R05274\n",
      "VPAMTr , Valine-pyruvate aminotransferase\n",
      "\t R01215\n",
      "X5PL3E , L-xylulose 5-phosphate 3-epimerase\n",
      "\t R03244\n",
      "XPPT , Xanthine phosphoribosyltransferase\n",
      "\t R02142\n",
      "ALATA_D , D-alanine transaminase\n",
      "\t R01148\n"
     ]
    }
   ],
   "source": [
    "BIGGinKEGGlist = []\n",
    "BIGGinKEGGdir = {}\n",
    "for rxn in carveme_model.reactions:\n",
    "    if rxn.annotation.get('kegg.reaction'):\n",
    "        if type(rxn.annotation.get('kegg.reaction')) == list:\n",
    "            for kegg in rxn.annotation.get('kegg.reaction'):\n",
    "                BIGGinKEGGlist.append(kegg)\n",
    "                BIGGinKEGGdir[kegg] = rxn.id\n",
    "        else:\n",
    "            BIGGinKEGGlist.append(rxn.annotation.get('kegg.reaction'))\n",
    "            BIGGinKEGGdir[rxn.annotation.get('kegg.reaction')] = rxn.id\n",
    "        print(rxn.id,',', rxn.name)\n",
    "        print('\\t',rxn.annotation.get('kegg.reaction'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "584\n",
      "249\n",
      "408\n"
     ]
    }
   ],
   "source": [
    "keggalone = [r for r in kegglist if r not in BIGGinKEGGlist]\n",
    "print(len(keggalone))\n",
    "biggalone = [r for r in BIGGinKEGGlist if r not in kegglist]\n",
    "print(len(biggalone))\n",
    "bigggood = [r for r in BIGGinKEGGlist if r in kegglist]\n",
    "print(len(bigggood))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "R02521 34HPPOR\n",
      "R04537 3HAD80\n",
      "R01986 4ABUTD\n",
      "R05051 4HGLSD\n",
      "R05086 4HTHRS\n",
      "R01357 AACOAT\n",
      "R02549 ABUTD\n",
      "R01359 ACACCT\n",
      "R00238 KAT1\n",
      "R01177 ACACT2r\n",
      "R00742 ACCOAC\n",
      "R02649 ACGK\n",
      "R00259 ACGS\n",
      "R08648 ACHBS\n",
      "R00315 ACKr\n",
      "R00226 ACLS\n",
      "R00669 ACODA\n",
      "R01325 ACONTa\n",
      "R01900 ACONTb\n",
      "R02283 ACOTA\n",
      "R00235 ACS\n",
      "R00317 ACYP_2\n",
      "R01560 ADA\n",
      "R05553 ADCL\n",
      "R07302 ADCPS1\n",
      "R01716 ADCS\n",
      "R01244 ADD\n",
      "R00127 ADK1\n",
      "R00089 ADNCYC\n",
      "R05221 ADOCBIK\n",
      "R00190 ADPT\n",
      "R00509 ADSK\n",
      "R01083 ADSL1r\n",
      "R04559 ADSL2r\n",
      "R01135 ADSS\n",
      "R03443 AGPR\n",
      "R00369 AGTi\n",
      "R00192 AHCi\n",
      "R01287 AHSERL2\n",
      "R04560 AICART\n",
      "R07405 AIRC3\n",
      "R01150 ALAALAr\n",
      "R00396 ALAD_L\n",
      "R00401 ALAR\n",
      "R01041 ALCD19y\n",
      "R00754 ALCD2ir\n",
      "R00746 ALCD2y\n",
      "R02678 ALDD20x\n",
      "R00710 ALDD2x\n",
      "R00711 ALDD2y\n",
      "R03231 AMAOTr\n",
      "R02540 AMID2\n",
      "R00899 AMPTASECG\n",
      "R01073 ANPRT\n",
      "R00986 ANS\n",
      "R00985 ANS2\n",
      "R03210 AOXSr\n",
      "R03035 PTPATi\n",
      "R03458 APRAUR\n",
      "R04025 APRTO2\n",
      "R01086 ARGSL\n",
      "R01954 ARGSS\n",
      "R00691 AROH\n",
      "R02291 ASAD\n",
      "R00578 ASNS1\n",
      "R01397 ASPCT\n",
      "R00480 ASPK\n",
      "R00357 ASPO1\n",
      "R00481 ASPO6\n",
      "R00355 ASPTA\n",
      "R01071 ATPPRT\n",
      "R01361 BDH\n",
      "R00508 BPNT2\n",
      "R00575 CBPS\n",
      "R05634 CDPMEK\n",
      "R01714 CHORS\n",
      "R03470 CMLDC\n",
      "R06895 CPPPGO2\n",
      "R00351 CS\n",
      "R00571 CTPS1\n",
      "R00573 CTPS2\n",
      "R01931 CYANST\n",
      "R00782 CYSDS\n",
      "R00897 CYSS\n",
      "R00896 CYSTA\n",
      "R01286 CYSTL\n",
      "R00512 CYTK1\n",
      "R01665 CYTK2\n",
      "R06977 DABAAT\n",
      "R01547 DADK\n",
      "R00451 DAPDC\n",
      "R02735 DAPE\n",
      "R07281 DB4PS\n",
      "R03182 DBTS\n",
      "R02325 DCTPD\n",
      "R01826 DDPA\n",
      "R02090 DGK1\n",
      "R04441 DHAD1\n",
      "R05070 DHAD2\n",
      "R00939 DHFR\n",
      "R02237 DHFS\n",
      "R01993 DHORTS\n",
      "R03459 DHPPDA2\n",
      "R03067 DHPS2\n",
      "R03083 DHQS\n",
      "R03084 DHQTi\n",
      "R01658 DMATT\n",
      "R00130 DPCOAK\n",
      "R02472 DPR\n",
      "R02094 DTMPK\n",
      "R02100 DUTPDP\n",
      "R05688 DXPRIi\n",
      "R05636 DXPS\n",
      "R03026 ECOAH1\n",
      "R04224 ECOAH12\n",
      "R04749 ECOAH2\n",
      "R04746 ECOAH3\n",
      "R04744 ECOAH4\n",
      "R04170 ECOAH5\n",
      "R04740 ECOAH6\n",
      "R04738 ECOAH7\n",
      "R04204 ECOAH9ir\n",
      "R00658 ENO\n",
      "R07395 ENOPH\n",
      "R00749 ETHAAL\n",
      "R01274 FACOAE160\n",
      "R01280 FACOAL160\n",
      "R01068 FBA\n",
      "R02568 FBA2\n",
      "R01829 FBA3\n",
      "R00310 FCLT\n",
      "R00519 FDH\n",
      "R10206 FDMO2\n",
      "R00078 FEROpp\n",
      "R00161 FMNAT\n",
      "R02236 FOLR2\n",
      "R02300 THFAT\n",
      "R02301 FTHFCL\n",
      "R00943 FTHFLi\n",
      "R01082 FUM\n",
      "R01364 FUMAC\n",
      "R02272 G1SAT\n",
      "R00842 G3PD1ir\n",
      "R00844 G3PD2\n",
      "R03313 G5SD\n",
      "R02736 G6PBDH\n",
      "R03321 G6PI3\n",
      "R01092 GALKr\n",
      "R00289 GALUi\n",
      "R01061 GAPD\n",
      "R04325 GARFT\n",
      "R03425 GCCa\n",
      "R04125 GCCb\n",
      "R03815 GCCc\n",
      "R00945 GHMT2r\n",
      "R00332 GK1\n",
      "R00253 GLNS\n",
      "R00239 GLU5K\n",
      "R00894 GLUCYS\n",
      "R00243 GLUDxi\n",
      "R00256 GLUN\n",
      "R01072 GLUPRT\n",
      "R00114 GLUSy\n",
      "R05578 GLUTRS\n",
      "R01752 GLYALDDr\n",
      "R00371 GLYAT\n",
      "R08572 GLYCK2\n",
      "R01221 GLYCL\n",
      "R00717 GLYCLTDx\n",
      "R00847 GLYK\n",
      "R01736 GLYOX\n",
      "R02530 LGTHL\n",
      "R00372 GLYTA\n",
      "R01230 GMPS\n",
      "R01231 GMPS2\n",
      "R01528 GND\n",
      "R01030 GPDDA1\n",
      "R01470 GPDDA2\n",
      "R02003 GRTT\n",
      "R00115 GTHOr\n",
      "R00274 GTHPi\n",
      "R00274 GTHPi\n",
      "R00497 GTHS\n",
      "R00425 GTPCII2\n",
      "R00429 GTPDPK\n",
      "R00434 GUACYC\n",
      "R01676 GUAD\n",
      "R01229 GUAPRT\n",
      "R01975 HACD1i\n",
      "R01975 HACD1i\n",
      "R01975 HACD1i\n",
      "R04748 HACD2\n",
      "R04745 HACD3\n",
      "R04743 HACD4\n",
      "R04741 HACD5\n",
      "R04739 HACD6\n",
      "R04737 HACD7\n",
      "R04203 HACD9\n",
      "R01976 HBCO_nadp\n",
      "R10092 HCO3E\n",
      "R07411 HEMEOS\n",
      "R00299 HEX1\n",
      "R00867 HEX7\n",
      "R02519 HGNTOR\n",
      "R03503 HPPK2\n",
      "R03293 HPROb\n",
      "R01388 HPYRRx\n",
      "R01773 HSDxi\n",
      "R01775 HSDy\n",
      "R01776 HSERTA\n",
      "R01771 HSK\n",
      "R01777 HSST\n",
      "R03243 HSTPTr\n",
      "R03243 HSTPTr\n",
      "R01132 HXPRT\n",
      "R00267 ICDHyr\n",
      "R04558 IG3PS\n",
      "R03457 IGPDH\n",
      "R03508 IGPS\n",
      "R02199 ILETA\n",
      "R01127 IMPC\n",
      "R01130 IMPD\n",
      "R04426 IPMD\n",
      "R04001 IPPMIa\n",
      "R03968 IPPMIb\n",
      "R01213 IPPS\n",
      "R04439 KARA1\n",
      "R00238 KAT1\n",
      "R01090 LEUTA\n",
      "R02530 LGTHL\n",
      "R00685 LTDCL\n",
      "R00885 MAN1PT\n",
      "R01819 MAN6PI\n",
      "R04095 MBCOAi\n",
      "R04138 MCCC\n",
      "R01626 MCOATA\n",
      "R00342 MDH\n",
      "R00216 ME2\n",
      "R05637 MECDPS\n",
      "R05633 MEPCT\n",
      "R00177 METAT\n",
      "R00654 METGL\n",
      "R00946 METS\n",
      "R01185 MI1PP\n",
      "R00923 MMCD\n",
      "R02765 MME\n",
      "R00833 MMM2\n",
      "R00922 MMSAD1\n",
      "R01226 MOHMT\n",
      "R01402 MTAP\n",
      "R01655 MTHFC\n",
      "R01220 MTHFD\n",
      "R07168 MTHFR2\n",
      "R04420 MTRI\n",
      "R05050 NABTNO\n",
      "R00103 NADDP\n",
      "R00104 NADK\n",
      "R00257 NADS2\n",
      "R00112 NADTRHD\n",
      "R00330 NDPK1\n",
      "R00156 NDPK2\n",
      "R00570 NDPK3\n",
      "R02093 NDPK4\n",
      "R01857 NDPK5\n",
      "R02331 NDPK6\n",
      "R02326 NDPK7\n",
      "R01137 NDPK8\n",
      "R00722 NDPK9\n",
      "R00137 NMNAT\n",
      "R02322 NMNDA\n",
      "R02323 NMNR\n",
      "R02323 NMNR\n",
      "R01268 NNAM\n",
      "R03005 NNATr\n",
      "R03348 NNDPR\n",
      "R03346 NT5C\n",
      "R02719 NTD10\n",
      "R01126 NTD11\n",
      "R00963 NTD2\n",
      "R00511 NTD4\n",
      "R02088 NTD6\n",
      "R00183 NTD7\n",
      "R01227 NTD9\n",
      "R03531 NTPP10\n",
      "R02720 NTPP11\n",
      "R00426 NTPP2\n",
      "R00515 NTPP4\n",
      "R00087 NTPP6\n",
      "R11323 NTPP7\n",
      "R00662 NTPP8\n",
      "R00720 NTPP9\n",
      "R01856 NTPTP1\n",
      "R00787 NTRIR2x\n",
      "R00789 NTRIR2y\n",
      "R01398 OCBT_1\n",
      "R01398 OCBT_1\n",
      "R09248 OCTDPS\n",
      "R00965 OMPDC\n",
      "R00670 ORNDC\n",
      "R00667 ORNTA_1\n",
      "R02282 ORNTAC\n",
      "R00667 ORNTA_1\n",
      "R01870 ORPT\n",
      "R00522 OXADC\n",
      "R02401 OXPTNDH\n",
      "R00707 P5CD\n",
      "R01251 P5CR\n",
      "R02473 PANTS\n",
      "R02021 PAPSR\n",
      "R00919 PCNO\n",
      "R00191 PDE1\n",
      "R00278 PDX5POi\n",
      "R02613 PEAMNO\n",
      "R02060 PGAMT\n",
      "R01513 PGCD\n",
      "R01512 PGK\n",
      "R02035 PGL\n",
      "R01334 PGLYCP\n",
      "R01518 PGM\n",
      "R08639 PGMT\n",
      "R04444 PHCD\n",
      "R00694 PHETA1\n",
      "R00699 PHYCBOXL\n",
      "R01818 PMANM\n",
      "R03018 PNTK\n",
      "R00036 PPBNGS\n",
      "R00345 PPC\n",
      "R03269 PPCDC\n",
      "R00341 PPCK\n",
      "R00206 PPDK\n",
      "R00336 PPGPPDP\n",
      "R04231 PPNCL2\n",
      "R01728 PPND\n",
      "R01373 PPNDH\n",
      "R01372 PPOR\n",
      "R00199 PPS\n",
      "R04144 PRAGSr\n",
      "R03509 PRAIi\n",
      "R04037 PRAMPC\n",
      "R04591 PRASCSi\n",
      "R04035 PRATPP\n",
      "R04640 PRMICI\n",
      "R10507 PROD2\n",
      "R03460 PSCVT\n",
      "R04173 PSERT\n",
      "R00582 PSP_L\n",
      "R00921 PTA2\n",
      "R00230 PTAr\n",
      "R03035 PTPATi\n",
      "R02147 PUNP3\n",
      "R00245 PUTA3\n",
      "R00277 PYAM5PO\n",
      "R01711 PYDXNO\n",
      "R00200 PYK\n",
      "R04292 QULNS\n",
      "R00549 RBFK\n",
      "R04457 RBFSa\n",
      "R00066 RBFSb\n",
      "R02017 RNDR1\n",
      "R02019 RNDR2\n",
      "R02024 RNDR3\n",
      "R02018 RNDR4\n",
      "R01529 RPE\n",
      "R00529 SADT\n",
      "R02734 SDPDS\n",
      "R04475 SDPTA\n",
      "R00586 SERAT\n",
      "R03947 SHCHD2\n",
      "R02864 SHCHF\n",
      "R02413 SHK3Dr\n",
      "R02412 SHKK\n",
      "R01288 SHSL2r\n",
      "R00588 SPT_syn\n",
      "R00585 SPTc\n",
      "R00713 SSALx\n",
      "R00714 SSALy\n",
      "R00405 SUCOAS\n",
      "R00858 SULR\n",
      "R01827 TALA\n",
      "R04365 THDPS\n",
      "R02300 THFAT\n",
      "R01466 THRS\n",
      "R01830 TKT2\n",
      "R02101 TMDS\n",
      "R00617 TMPK\n",
      "R03223 TMPPP\n",
      "R01015 TPI\n",
      "R02778 TRE6PP\n",
      "R00674 TRPS2\n",
      "R02340 TRPS3\n",
      "R02173 TRYPTAOX\n",
      "R00734 TYRTAi\n",
      "R00734 TYRTAi\n",
      "R00418 UAG4Ei\n",
      "R00660 UAGCVT\n",
      "R05032 UAGPT3\n",
      "R02783 UAMAGS\n",
      "R03193 UAMAS\n",
      "R03192 UAPGR\n",
      "R00291 UDPG4E\n",
      "R00955 UGLT\n",
      "R00158 UMPK\n",
      "R03194 UPP3MT\n",
      "R03165 UPP3S\n",
      "R03197 UPPDC1\n",
      "R02098 URIDK2r\n",
      "R01214 VALTA\n",
      "R02142 XPPT\n"
     ]
    }
   ],
   "source": [
    "for ke in bigggood:\n",
    "    print(ke, BIGGinKEGGdir[ke])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "rxn = carveme_model.reactions.get_by_id('ALATA_D')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'sbo': 'SBO:0000176',\n",
       " 'rhea': ['15869', '15870', '15871', '15872'],\n",
       " 'ec-code': '2.6.1.21',\n",
       " 'biocyc': 'META:D-ALANINE-AMINOTRANSFERASE-RXN',\n",
       " 'metanetx.reaction': 'MNXR95696',\n",
       " 'kegg.reaction': 'R01148',\n",
       " 'seed.reaction': 'rxn00849'}"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rxn.annotation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'R01148'"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rxn.annotation.get('kegg.reaction')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for rxn in carveme_model.reactions:\n",
    "    rxn.annotation['kegg.reaction']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "Annotation must be a dict",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[8], line 2\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[39mimport\u001b[39;00m \u001b[39mcobra\u001b[39;00m\n\u001b[1;32m----> 2\u001b[0m uni_model \u001b[39m=\u001b[39m cobra\u001b[39m.\u001b[39;49mio\u001b[39m.\u001b[39;49mload_json_model(\u001b[39m'\u001b[39;49m\u001b[39muniversal_model.json\u001b[39;49m\u001b[39m'\u001b[39;49m)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\json.py:149\u001b[0m, in \u001b[0;36mload_json_model\u001b[1;34m(filename)\u001b[0m\n\u001b[0;32m    147\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39misinstance\u001b[39m(filename, (\u001b[39mstr\u001b[39m, Path)):\n\u001b[0;32m    148\u001b[0m     \u001b[39mwith\u001b[39;00m \u001b[39mopen\u001b[39m(filename, \u001b[39m\"\u001b[39m\u001b[39mr\u001b[39m\u001b[39m\"\u001b[39m) \u001b[39mas\u001b[39;00m file_handle:\n\u001b[1;32m--> 149\u001b[0m         \u001b[39mreturn\u001b[39;00m model_from_dict(json\u001b[39m.\u001b[39;49mload(file_handle))\n\u001b[0;32m    150\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m    151\u001b[0m     \u001b[39mreturn\u001b[39;00m model_from_dict(json\u001b[39m.\u001b[39mload(filename))\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\dict.py:401\u001b[0m, in \u001b[0;36mmodel_from_dict\u001b[1;34m(obj)\u001b[0m\n\u001b[0;32m    398\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mValueError\u001b[39;00m(\u001b[39m\"\u001b[39m\u001b[39mObject has no .reactions attribute. Cannot load.\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[0;32m    399\u001b[0m model \u001b[39m=\u001b[39m Model()\n\u001b[0;32m    400\u001b[0m model\u001b[39m.\u001b[39madd_metabolites(\n\u001b[1;32m--> 401\u001b[0m     [_metabolite_from_dict(metabolite) \u001b[39mfor\u001b[39;00m metabolite \u001b[39min\u001b[39;00m obj[\u001b[39m\"\u001b[39m\u001b[39mmetabolites\u001b[39m\u001b[39m\"\u001b[39m]]\n\u001b[0;32m    402\u001b[0m )\n\u001b[0;32m    403\u001b[0m model\u001b[39m.\u001b[39mgenes\u001b[39m.\u001b[39mextend([gene_from_dict(gene) \u001b[39mfor\u001b[39;00m gene \u001b[39min\u001b[39;00m obj[\u001b[39m\"\u001b[39m\u001b[39mgenes\u001b[39m\u001b[39m\"\u001b[39m]])\n\u001b[0;32m    404\u001b[0m model\u001b[39m.\u001b[39madd_reactions(\n\u001b[0;32m    405\u001b[0m     [_reaction_from_dict(reaction, model) \u001b[39mfor\u001b[39;00m reaction \u001b[39min\u001b[39;00m obj[\u001b[39m\"\u001b[39m\u001b[39mreactions\u001b[39m\u001b[39m\"\u001b[39m]]\n\u001b[0;32m    406\u001b[0m )\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\dict.py:401\u001b[0m, in \u001b[0;36m<listcomp>\u001b[1;34m(.0)\u001b[0m\n\u001b[0;32m    398\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mValueError\u001b[39;00m(\u001b[39m\"\u001b[39m\u001b[39mObject has no .reactions attribute. Cannot load.\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[0;32m    399\u001b[0m model \u001b[39m=\u001b[39m Model()\n\u001b[0;32m    400\u001b[0m model\u001b[39m.\u001b[39madd_metabolites(\n\u001b[1;32m--> 401\u001b[0m     [_metabolite_from_dict(metabolite) \u001b[39mfor\u001b[39;00m metabolite \u001b[39min\u001b[39;00m obj[\u001b[39m\"\u001b[39m\u001b[39mmetabolites\u001b[39m\u001b[39m\"\u001b[39m]]\n\u001b[0;32m    402\u001b[0m )\n\u001b[0;32m    403\u001b[0m model\u001b[39m.\u001b[39mgenes\u001b[39m.\u001b[39mextend([gene_from_dict(gene) \u001b[39mfor\u001b[39;00m gene \u001b[39min\u001b[39;00m obj[\u001b[39m\"\u001b[39m\u001b[39mgenes\u001b[39m\u001b[39m\"\u001b[39m]])\n\u001b[0;32m    404\u001b[0m model\u001b[39m.\u001b[39madd_reactions(\n\u001b[0;32m    405\u001b[0m     [_reaction_from_dict(reaction, model) \u001b[39mfor\u001b[39;00m reaction \u001b[39min\u001b[39;00m obj[\u001b[39m\"\u001b[39m\u001b[39mreactions\u001b[39m\u001b[39m\"\u001b[39m]]\n\u001b[0;32m    406\u001b[0m )\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\dict.py:189\u001b[0m, in \u001b[0;36m_metabolite_from_dict\u001b[1;34m(metabolite)\u001b[0m\n\u001b[0;32m    187\u001b[0m new_metabolite \u001b[39m=\u001b[39m Metabolite()\n\u001b[0;32m    188\u001b[0m \u001b[39mfor\u001b[39;00m k, v \u001b[39min\u001b[39;00m metabolite\u001b[39m.\u001b[39mitems():\n\u001b[1;32m--> 189\u001b[0m     \u001b[39msetattr\u001b[39;49m(new_metabolite, k, v)\n\u001b[0;32m    190\u001b[0m \u001b[39mreturn\u001b[39;00m new_metabolite\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\core\\object.py:97\u001b[0m, in \u001b[0;36mObject.annotation\u001b[1;34m(self, annotation)\u001b[0m\n\u001b[0;32m     85\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"Set annotation.\u001b[39;00m\n\u001b[0;32m     86\u001b[0m \n\u001b[0;32m     87\u001b[0m \u001b[39mParameters\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     94\u001b[0m \u001b[39mTypeError if annotation not a dict.\u001b[39;00m\n\u001b[0;32m     95\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[0;32m     96\u001b[0m \u001b[39mif\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39misinstance\u001b[39m(annotation, \u001b[39mdict\u001b[39m):\n\u001b[1;32m---> 97\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mTypeError\u001b[39;00m(\u001b[39m\"\u001b[39m\u001b[39mAnnotation must be a dict\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[0;32m     98\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m     99\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_annotation \u001b[39m=\u001b[39m annotation\n",
      "\u001b[1;31mTypeError\u001b[0m: Annotation must be a dict"
     ]
    }
   ],
   "source": [
    "import cobra\n",
    "uni_model = cobra.io.load_json_model('universal_model.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "import cobra\n",
    "carveme_model = cobra.io.read_sbml_model('mbry_carveme_model.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "ename": "CobraSBMLError",
     "evalue": "Something went wrong reading the SBML model. Most likely the SBML model is not valid. Please check that your model is valid using the `cobra.io.sbml.validate_sbml_model` function or via the online validator at https://sbml.org/validator_servlet/ .\n\t`(model, errors) = validate_sbml_model(filename)`\nIf the model is valid and cannot be read please open an issue at https://github.com/opencobra/cobrapy/issues .",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\sbml.py:987\u001b[0m, in \u001b[0;36m_sbml_to_model\u001b[1;34m(doc, number, f_replace, set_missing_bounds, **kwargs)\u001b[0m\n\u001b[0;32m    986\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[1;32m--> 987\u001b[0m     objective_reaction \u001b[39m=\u001b[39m cobra_model\u001b[39m.\u001b[39;49mreactions\u001b[39m.\u001b[39;49mget_by_id(rid)\n\u001b[0;32m    988\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mKeyError\u001b[39;00m:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\core\\dictlist.py:75\u001b[0m, in \u001b[0;36mDictList.get_by_id\u001b[1;34m(self, id)\u001b[0m\n\u001b[0;32m     74\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"Return the element with a matching id.\"\"\"\u001b[39;00m\n\u001b[1;32m---> 75\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mlist\u001b[39m\u001b[39m.\u001b[39m\u001b[39m__getitem__\u001b[39m(\u001b[39mself\u001b[39m, \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_dict[\u001b[39mid\u001b[39;49m])\n",
      "\u001b[1;31mKeyError\u001b[0m: ''",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mCobraSBMLError\u001b[0m                            Traceback (most recent call last)",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\sbml.py:458\u001b[0m, in \u001b[0;36mread_sbml_model\u001b[1;34m(filename, number, f_replace, **kwargs)\u001b[0m\n\u001b[0;32m    457\u001b[0m     doc \u001b[39m=\u001b[39m _get_doc_from_filename(filename)\n\u001b[1;32m--> 458\u001b[0m     \u001b[39mreturn\u001b[39;00m _sbml_to_model(doc, number\u001b[39m=\u001b[39mnumber, f_replace\u001b[39m=\u001b[39mf_replace, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mkwargs)\n\u001b[0;32m    459\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mIOError\u001b[39;00m \u001b[39mas\u001b[39;00m e:\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\sbml.py:989\u001b[0m, in \u001b[0;36m_sbml_to_model\u001b[1;34m(doc, number, f_replace, set_missing_bounds, **kwargs)\u001b[0m\n\u001b[0;32m    988\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mKeyError\u001b[39;00m:\n\u001b[1;32m--> 989\u001b[0m     \u001b[39mraise\u001b[39;00m CobraSBMLError(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mObjective reaction \u001b[39m\u001b[39m'\u001b[39m\u001b[39m{\u001b[39;00mrid\u001b[39m}\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m not found\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[0;32m    990\u001b[0m \u001b[39mtry\u001b[39;00m:\n",
      "\u001b[1;31mCobraSBMLError\u001b[0m: Objective reaction '' not found",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[1;31mCobraSBMLError\u001b[0m                            Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[3], line 3\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[39mimport\u001b[39;00m \u001b[39mcobra\u001b[39;00m\n\u001b[1;32m----> 3\u001b[0m kegg_model \u001b[39m=\u001b[39m cobra\u001b[39m.\u001b[39;49mio\u001b[39m.\u001b[39;49mread_sbml_model(\u001b[39m'\u001b[39;49m\u001b[39mmbry_raven_model.xml\u001b[39;49m\u001b[39m'\u001b[39;49m)\n",
      "File \u001b[1;32mc:\\Users\\vickenlee\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\cobra\\io\\sbml.py:472\u001b[0m, in \u001b[0;36mread_sbml_model\u001b[1;34m(filename, number, f_replace, **kwargs)\u001b[0m\n\u001b[0;32m    462\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mException\u001b[39;00m \u001b[39mas\u001b[39;00m original_error:\n\u001b[0;32m    463\u001b[0m     cobra_error \u001b[39m=\u001b[39m CobraSBMLError(\n\u001b[0;32m    464\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39mSomething went wrong reading the SBML model. Most likely the SBML\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[0;32m    465\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39m model is not valid. Please check that your model is valid using \u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m    470\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39mat https://github.com/opencobra/cobrapy/issues .\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[0;32m    471\u001b[0m     )\n\u001b[1;32m--> 472\u001b[0m     \u001b[39mraise\u001b[39;00m cobra_error \u001b[39mfrom\u001b[39;00m \u001b[39moriginal_error\u001b[39;00m\n",
      "\u001b[1;31mCobraSBMLError\u001b[0m: Something went wrong reading the SBML model. Most likely the SBML model is not valid. Please check that your model is valid using the `cobra.io.sbml.validate_sbml_model` function or via the online validator at https://sbml.org/validator_servlet/ .\n\t`(model, errors) = validate_sbml_model(filename)`\nIf the model is valid and cannot be read please open an issue at https://github.com/opencobra/cobrapy/issues ."
     ]
    }
   ],
   "source": [
    "import cobra\n",
    "\n",
    "kegg_model = cobra.io.read_sbml_model('mbry_raven_model.xml')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.11"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
